{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "# Abstract\n",
    "\n",
    "**Author:** [Charles Tapley Hoyt](https://github.com/cthoyt)\n",
    "\n",
    "**Estimated Run Time:** 4 minutes\n",
    "\n",
    "This notebooks demonstrates the usage of a variant on the [Network Perturbation Amplitude](http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-6-54) algorithm to analyze an BEL knowledge assembly in the context of a differential gene expression experiment. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "import os\n",
    "\n",
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "from matplotlib_venn import venn2\n",
    "import logging\n",
    "import pybel\n",
    "from pybel.constants import *\n",
    "from pybel.canonicalize import calculate_canonical_name\n",
    "import pybel_tools as pbt\n",
    "from pybel_tools.visualization import to_jupyter\n",
    "import pybel_tools.analysis.npa as npa\n",
    "\n",
    "%config InlineBackend.figure_format = 'svg'\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.4.1'"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel.__version__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.1.3-dev'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pbt.__version__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "graph_vis_width = 1000\n",
    "graph_vis_height = 650"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "bms_base = os.environ['BMS_BASE']\n",
    "owncloud_base = os.environ['OWNCLOUD_BASE']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "# Data\n",
    "\n",
    "## Alzheimer's Disease Knowledge Assembly\n",
    "\n",
    "The Alzheimer's Disease Knowledge Assembly has been precompiled with the following command line script, and will be loaded from this format for improved performance. In general, derived data, such as the gpickle representation of a BEL script, are not saved under version control to ensure that the most up-to-date data is always used.\n",
    "\n",
    "```sh\n",
    "pybel convert --path \"$BMS_BASE/aetionomy/alzheimers.bel\" --pickle \"$BMS_BASE/aetionomy/alzheimers.gpickle\"\n",
    "```\n",
    "\n",
    "The BEL script can also be compiled from inside this notebook with the following python code:\n",
    "\n",
    "```python\n",
    ">>> import os\n",
    ">>> import pybel\n",
    ">>> # Input from BEL script\n",
    ">>> bel_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.bel')\n",
    ">>> graph = pybel.from_path(bel_path)\n",
    ">>> # Output to gpickle for fast loading later\n",
    ">>> pickle_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.gpickle')\n",
    ">>> pybel.to_pickle(graph, pickle_path)\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "pickle_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.gpickle')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "graph = pybel.from_pickle(pickle_path)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "All orthologies are discared before analysis."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "pbt.mutation.remove_nodes_by_namespace(graph, GENE, 'MGI')\n",
    "pbt.mutation.remove_nodes_by_namespace(graph, RNA, 'MGI')\n",
    "pbt.mutation.remove_nodes_by_namespace(graph, PROTEIN, 'MGI')\n",
    "\n",
    "pbt.mutation.remove_nodes_by_namespace(graph, GENE, 'RGD')\n",
    "pbt.mutation.remove_nodes_by_namespace(graph, RNA, 'RGD')\n",
    "pbt.mutation.remove_nodes_by_namespace(graph, PROTEIN, 'RGD')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "To merge the differential gene expression annotations, the entire graph must be collapsed to genes using `pbt.mutation.opening_by_central_dogma_to_genes`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "pbt.mutation.opening_by_central_dogma_to_genes(graph)\n",
    "pbt.mutation.collapse_variants_to_genes(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Name: \n",
      "Number of nodes: 9269\n",
      "Number of edges: 58301\n",
      "Network density: 0.0006786675695545498\n",
      "Number weakly connected components: 55\n",
      "Average in-degree: 6.289891034631568\n",
      "Average out-degree: 6.289891034631568\n"
     ]
    }
   ],
   "source": [
    "pbt.summary.print_summary(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"280pt\" version=\"1.1\" viewBox=\"0 0 712 280\" width=\"712pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 280.8 \n",
       "L 712.8 280.8 \n",
       "L 712.8 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 89.335068 257.744323 \n",
       "L 332.680527 257.744323 \n",
       "L 332.680527 21.50218 \n",
       "L 89.335068 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#pfc2dfe20f4)\" d=\"M -23954.127403 250.361756 \n",
       "L 100.396225 250.361756 \n",
       "L 100.396225 235.596622 \n",
       "L -23954.127403 235.596622 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#pfc2dfe20f4)\" d=\"M -23954.127403 220.831488 \n",
       "L 154.460217 220.831488 \n",
       "L 154.460217 206.066354 \n",
       "L -23954.127403 206.066354 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#pfc2dfe20f4)\" d=\"M -23954.127403 191.30122 \n",
       "L 176.954601 191.30122 \n",
       "L 176.954601 176.536086 \n",
       "L -23954.127403 176.536086 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#pfc2dfe20f4)\" d=\"M -23954.127403 161.770953 \n",
       "L 203.164858 161.770953 \n",
       "L 203.164858 147.005819 \n",
       "L -23954.127403 147.005819 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#pfc2dfe20f4)\" d=\"M -23954.127403 132.240685 \n",
       "L 224.413221 132.240685 \n",
       "L 224.413221 117.475551 \n",
       "L -23954.127403 117.475551 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#pfc2dfe20f4)\" d=\"M -23954.127403 102.710417 \n",
       "L 225.372665 102.710417 \n",
       "L 225.372665 87.945283 \n",
       "L -23954.127403 87.945283 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#pfc2dfe20f4)\" d=\"M -23954.127403 73.180149 \n",
       "L 251.785593 73.180149 \n",
       "L 251.785593 58.415015 \n",
       "L -23954.127403 58.415015 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#pfc2dfe20f4)\" d=\"M -23954.127403 43.649881 \n",
       "L 321.61937 43.649881 \n",
       "L 321.61937 28.884747 \n",
       "L -23954.127403 28.884747 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"m2a62420e08\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"94.070091\" xlink:href=\"#m2a62420e08\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- $10^{1}$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "\" id=\"CMR17-48\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(87.103137 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-49\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"173.964435\" xlink:href=\"#m2a62420e08\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- $10^{2}$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(166.997481 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"253.858779\" xlink:href=\"#m2a62420e08\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- $10^{3}$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 22.09375 33.703125 \n",
       "C 31 33.703125 34.90625 25.96875 34.90625 17.03125 \n",
       "C 34.90625 5 28.5 0.390625 22.703125 0.390625 \n",
       "C 17.40625 0.390625 8.796875 3.03125 6.09375 10.828125 \n",
       "C 6.59375 10.625 7.09375 10.625 7.59375 10.625 \n",
       "C 10 10.625 11.796875 12.21875 11.796875 14.8125 \n",
       "C 11.796875 17.703125 9.59375 19 7.59375 19 \n",
       "C 5.90625 19 3.296875 18.203125 3.296875 14.46875 \n",
       "C 3.296875 5.234375 12.296875 -2 22.90625 -2 \n",
       "C 34 -2 42.5 6.71875 42.5 16.953125 \n",
       "C 42.5 26.671875 34.5 33.703125 25 34.796875 \n",
       "C 32.59375 36.375 39.90625 43.0625 39.90625 52.015625 \n",
       "C 39.90625 59.6875 32 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 59.828125 5.90625 52.015625 \n",
       "C 5.90625 48.59375 8.5 48 9.796875 48 \n",
       "C 11.90625 48 13.703125 49.296875 13.703125 51.890625 \n",
       "C 13.703125 54.46875 11.90625 55.765625 9.796875 55.765625 \n",
       "C 9.40625 55.765625 8.90625 55.765625 8.5 55.578125 \n",
       "C 11.40625 61.859375 19.296875 63 22.796875 63 \n",
       "C 26.296875 63 32.90625 61.328125 32.90625 51.875 \n",
       "C 32.90625 49.109375 32.5 44.1875 29.09375 39.84375 \n",
       "C 26.09375 36 22.703125 36 19.40625 35.6875 \n",
       "C 18.90625 35.6875 16.59375 35.453125 16.203125 35.453125 \n",
       "C 15.5 35.359375 15.09375 35.265625 15.09375 34.5 \n",
       "C 15.09375 33.796875 15.203125 33.703125 17.203125 33.703125 \n",
       "z\n",
       "\" id=\"CMR17-51\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(246.891825 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-51\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 2 \n",
       "\" id=\"m676e07c72e\" style=\"stroke:#000000;stroke-width:0.6;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"90.414327\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"118.120685\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"132.189381\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_7\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"142.171279\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_8\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"149.913841\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_9\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"156.239975\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_10\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"161.588645\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_11\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"166.221873\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_12\">\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"170.308671\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_13\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"198.015029\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_14\">\n",
       "     <g id=\"line2d_14\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"212.083725\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_15\">\n",
       "     <g id=\"line2d_15\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"222.065623\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_16\">\n",
       "     <g id=\"line2d_16\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"229.808185\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_17\">\n",
       "     <g id=\"line2d_17\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"236.134319\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_18\">\n",
       "     <g id=\"line2d_18\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"241.482989\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_19\">\n",
       "     <g id=\"line2d_19\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"246.116217\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_20\">\n",
       "     <g id=\"line2d_20\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"250.203014\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_21\">\n",
       "     <g id=\"line2d_21\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"277.909373\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_22\">\n",
       "     <g id=\"line2d_22\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"291.978069\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_23\">\n",
       "     <g id=\"line2d_23\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"301.959967\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_24\">\n",
       "     <g id=\"line2d_24\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"309.702529\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_25\">\n",
       "     <g id=\"line2d_25\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"316.028663\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_26\">\n",
       "     <g id=\"line2d_26\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"321.377332\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_27\">\n",
       "     <g id=\"line2d_27\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"326.010561\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_28\">\n",
       "     <g id=\"line2d_28\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"330.097358\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_29\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"md5943156f4\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#md5943156f4\" y=\"242.979189\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- Reaction -->\n",
       "      <defs>\n",
       "       <path d=\"M 39.203125 33.53125 \n",
       "C 48.703125 35.4375 56.703125 41.8125 56.703125 49.78125 \n",
       "C 56.703125 59.140625 46.09375 67.59375 32.09375 67.59375 \n",
       "L 5.203125 67.59375 \n",
       "L 5.203125 65 \n",
       "C 12.09375 65 13.203125 65 13.203125 60.515625 \n",
       "L 13.203125 6.484375 \n",
       "C 13.203125 2 12.09375 2 5.203125 2 \n",
       "L 5.203125 -0.40625 \n",
       "C 8.09375 -0.40625 13.296875 -0.40625 16.40625 -0.40625 \n",
       "C 19.5 -0.40625 24.703125 -0.40625 27.59375 -0.40625 \n",
       "L 27.59375 2 \n",
       "C 20.703125 2 19.59375 2 19.59375 6.515625 \n",
       "L 19.59375 33 \n",
       "L 31.296875 33 \n",
       "C 37 33 40.203125 30.6875 41.59375 29.390625 \n",
       "C 45.59375 25.5 45.59375 22.796875 45.59375 16 \n",
       "C 45.59375 9.296875 45.59375 5.890625 48.59375 2.5 \n",
       "C 52.40625 -1.59375 57.5 -2 59.703125 -2 \n",
       "C 67.09375 -2 68.203125 6.1875 68.203125 7.859375 \n",
       "C 68.203125 8.4375 68.203125 9.140625 67.296875 9.140625 \n",
       "C 66.5 9.140625 66.40625 8.546875 66.40625 8.15625 \n",
       "C 65.90625 1.75 63 -0.40625 59.90625 -0.40625 \n",
       "C 54.40625 -0.40625 53.796875 5.859375 53.203125 11.9375 \n",
       "C 52.90625 14.03125 52.703125 15.515625 52.5 17.703125 \n",
       "C 51.90625 22.6875 51.09375 29.953125 39.203125 33.53125 \n",
       "M 31 34.59375 \n",
       "L 19.59375 34.59375 \n",
       "L 19.59375 61.140625 \n",
       "C 19.59375 64.515625 19.796875 65 23.796875 65 \n",
       "L 31.09375 65 \n",
       "C 39.40625 65 49.09375 62.140625 49.09375 49.84375 \n",
       "C 49.09375 37.0625 38.5 34.59375 31 34.59375 \n",
       "\" id=\"CMR17-82\"/>\n",
       "       <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "       <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "       <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "\" id=\"CMR17-99\"/>\n",
       "       <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "       <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "       <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "\" id=\"CMR17-111\"/>\n",
       "       <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(44.00651 246.438439)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-82\"/>\n",
       "       <use transform=\"translate(67.791102 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(108.276762 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(153.96724 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(194.4529 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(229.733668 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(254.604726 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(300.295203 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_30\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#md5943156f4\" y=\"213.448921\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- Composite -->\n",
       "      <defs>\n",
       "       <path d=\"M 62.09375 67.015625 \n",
       "C 62.09375 68.796875 62 68.90625 61.40625 68.90625 \n",
       "C 61 68.90625 60.90625 68.796875 60.203125 67.609375 \n",
       "L 55.796875 59.25 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 12.90625 20.09375 -2 37.296875 -2 \n",
       "C 52.59375 -2 62.09375 11.40625 62.09375 22.828125 \n",
       "C 62.09375 23.8125 62.09375 24.21875 61.203125 24.21875 \n",
       "C 60.40625 24.21875 60.40625 23.921875 60.296875 23.03125 \n",
       "C 59.5 9.140625 49.5 0.59375 38.703125 0.59375 \n",
       "C 28.5 0.59375 12.203125 7.734375 12.203125 33.546875 \n",
       "C 12.203125 59.453125 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.6875 59.90625 42.890625 \n",
       "C 60.09375 41.6875 60.09375 41.484375 61 41.484375 \n",
       "C 62.09375 41.484375 62.09375 41.6875 62.09375 43.484375 \n",
       "z\n",
       "\" id=\"CMR17-67\"/>\n",
       "       <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "       <path d=\"M 22 -16.609375 \n",
       "C 15.59375 -16.609375 15.203125 -16.203125 15.203125 -12.328125 \n",
       "L 15.203125 6.28125 \n",
       "C 18.09375 2.09375 22.296875 -0.59375 27.59375 -0.59375 \n",
       "C 37.59375 -0.59375 47.703125 8.171875 47.703125 21.6875 \n",
       "C 47.703125 34.234375 39 43.78125 28.703125 43.78125 \n",
       "C 22.703125 43.78125 17.796875 40.5 15 36.53125 \n",
       "L 15 43.78125 \n",
       "L 3.09375 42.6875 \n",
       "L 3.09375 40.109375 \n",
       "C 9 40.109375 9.90625 39.5 9.90625 34.734375 \n",
       "L 9.90625 -12.328125 \n",
       "C 9.90625 -16.109375 9.5 -16.609375 3.09375 -16.609375 \n",
       "L 3.09375 -19.1875 \n",
       "C 5.5 -19 9.90625 -19 12.5 -19 \n",
       "C 15.09375 -19 19.59375 -19 22 -19.1875 \n",
       "z\n",
       "M 15.203125 31.4375 \n",
       "C 15.203125 32.84375 15.203125 34.921875 19.09375 38.609375 \n",
       "C 19.59375 39.015625 23 42 28 42 \n",
       "C 35.296875 42 41.296875 32.9375 41.296875 21.59375 \n",
       "C 41.296875 10.25 34.90625 1 27 1 \n",
       "C 23.40625 1 19.5 2.6875 16.5 7.375 \n",
       "C 15.203125 9.5625 15.203125 10.15625 15.203125 11.75 \n",
       "z\n",
       "\" id=\"CMR17-112\"/>\n",
       "       <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(36.064035 216.908171)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(66.509881 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(112.200358 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(189.119948 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(242.617756 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(288.308233 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(324.109571 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(348.980629 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(384.261397 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_31\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#md5943156f4\" y=\"183.918653\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_6\">\n",
       "      <!-- Pathology -->\n",
       "      <defs>\n",
       "       <path d=\"M 19.796875 31.90625 \n",
       "L 35.796875 31.90625 \n",
       "C 48.703125 31.90625 58.09375 40.28125 58.09375 49.75 \n",
       "C 58.09375 59.234375 48.90625 68 35.796875 68 \n",
       "L 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.921875 \n",
       "L 13.203125 6.96875 \n",
       "C 13.203125 2.5 12.09375 2.5 5.203125 2.5 \n",
       "L 5.203125 0 \n",
       "C 8.09375 0 13.40625 0 16.5 0 \n",
       "C 19.59375 0 24.90625 0 27.796875 0 \n",
       "L 27.796875 2.5 \n",
       "C 20.90625 2.5 19.796875 2.5 19.796875 6.96875 \n",
       "z\n",
       "M 19.59375 34 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 33.90625 65.40625 \n",
       "C 46.796875 65.40625 50.5 58.03125 50.5 49.75 \n",
       "C 50.5 40.6875 46.09375 34 33.90625 34 \n",
       "z\n",
       "\" id=\"CMR17-80\"/>\n",
       "       <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.5 43.59375 16.5 37 15.296875 33.703125 \n",
       "L 15.203125 33.703125 \n",
       "L 15.203125 68.6875 \n",
       "L 3.296875 67.59375 \n",
       "L 3.296875 65 \n",
       "C 9.203125 65 10.09375 64.40625 10.09375 59.640625 \n",
       "L 10.09375 6.65625 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-104\"/>\n",
       "       <path d=\"M 15.296875 68.6875 \n",
       "L 3.40625 67.59375 \n",
       "L 3.40625 65 \n",
       "C 9.296875 65 10.203125 64.40625 10.203125 59.640625 \n",
       "L 10.203125 6.65625 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.203125 0 12.703125 0 \n",
       "C 15.296875 0 19.703125 0 22.09375 -0.203125 \n",
       "L 22.09375 2.390625 \n",
       "C 15.703125 2.390625 15.296875 2.78125 15.296875 6.65625 \n",
       "z\n",
       "\" id=\"CMR17-108\"/>\n",
       "       <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "\" id=\"CMR17-103\"/>\n",
       "       <path d=\"M 39 34.328125 \n",
       "C 41.5 40.40625 46 40.40625 47.40625 40.40625 \n",
       "L 47.40625 43 \n",
       "C 45.5 43 43 43 41.09375 43 \n",
       "C 39 43 35.703125 43 33.703125 43.09375 \n",
       "L 33.703125 40.5 \n",
       "C 37.5 40.203125 37.59375 37.40625 37.59375 36.609375 \n",
       "C 37.59375 35.625 37.40625 35.125 36.90625 33.921875 \n",
       "L 26 7.015625 \n",
       "L 14.09375 36.03125 \n",
       "C 13.59375 37.21875 13.59375 37.90625 13.59375 38.015625 \n",
       "C 13.59375 40.203125 15.90625 40.40625 18.296875 40.40625 \n",
       "L 18.296875 43.09375 \n",
       "C 15.90625 43 11.59375 43 9.09375 43 \n",
       "C 6.40625 43 3.203125 43 1 43.09375 \n",
       "L 1 40.40625 \n",
       "C 6.40625 40.40625 7 39.90625 8.296875 36.71875 \n",
       "L 23.203125 0.234375 \n",
       "C 18.703125 -11.53125 16.09375 -18.40625 9.796875 -18.40625 \n",
       "C 8.703125 -18.40625 6.203125 -18.109375 4.40625 -16.3125 \n",
       "C 6.703125 -16.109375 7.703125 -14.71875 7.703125 -13.015625 \n",
       "C 7.703125 -11.328125 6.5 -9.828125 4.5 -9.828125 \n",
       "C 2.296875 -9.828125 1.203125 -11.328125 1.203125 -13.125 \n",
       "C 1.203125 -17.109375 5.296875 -20 9.796875 -20 \n",
       "C 15.59375 -20 19.203125 -14.40625 21.203125 -9.4375 \n",
       "z\n",
       "\" id=\"CMR17-121\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(38.471647 187.377903)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(59.983847 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(105.674324 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(140.955092 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(191.850461 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(237.540938 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(262.411997 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(308.102474 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(353.792951 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_32\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#md5943156f4\" y=\"154.388386\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- Protein -->\n",
       "      <defs>\n",
       "       <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(50.067111 157.847635)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(62.586301 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(97.867068 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(143.557545 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(178.838313 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(219.323974 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(244.195032 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_33\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#md5943156f4\" y=\"124.858118\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- Abundance -->\n",
       "      <defs>\n",
       "       <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "       <path d=\"M 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 -0.203125 \n",
       "L 11.703125 -0.203125 \n",
       "L 14.59375 6.890625 \n",
       "C 17.5 2.203125 21.796875 -1 27.59375 -1 \n",
       "C 37.59375 -1 47.703125 7.8125 47.703125 21.40625 \n",
       "C 47.703125 34.1875 38.703125 43.59375 28.59375 43.59375 \n",
       "C 22.40625 43.59375 18 40.296875 15 36.40625 \n",
       "z\n",
       "M 15.203125 31.3125 \n",
       "C 15.203125 33.09375 15.203125 33.40625 16.59375 35.5 \n",
       "C 20.5 41.515625 26.09375 42 28 42 \n",
       "C 31 42 41.296875 40.390625 41.296875 21.5 \n",
       "C 41.296875 1.6875 29.5 0.59375 27.09375 0.59375 \n",
       "C 24 0.59375 19.5 1.78125 16.296875 7.59375 \n",
       "C 15.203125 9.5 15.203125 9.703125 15.203125 11.484375 \n",
       "z\n",
       "\" id=\"CMR17-98\"/>\n",
       "       <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "       <path d=\"M 29.203125 67.59375 \n",
       "L 29.203125 65 \n",
       "C 35.09375 65 36 64.40625 36 59.6875 \n",
       "L 36 36.640625 \n",
       "C 35.59375 37.140625 31.5 43.59375 23.40625 43.59375 \n",
       "C 13.203125 43.59375 3.296875 34.5 3.296875 21.3125 \n",
       "C 3.296875 8.203125 12.59375 -1 22.40625 -1 \n",
       "C 30.90625 -1 35.296875 5.265625 35.796875 5.9375 \n",
       "L 35.796875 -0.984375 \n",
       "L 47.90625 -0.984375 \n",
       "L 47.90625 2 \n",
       "C 42 2 41.09375 2.59375 41.09375 7.390625 \n",
       "L 41.09375 68.6875 \n",
       "z\n",
       "M 35.796875 11.796875 \n",
       "C 35.796875 8.796875 34 6.09375 31.703125 4.09375 \n",
       "C 28.296875 1.09375 24.90625 0.59375 23 0.59375 \n",
       "C 20.09375 0.59375 9.703125 2.09375 9.703125 21.1875 \n",
       "C 9.703125 40.8125 21.296875 42 23.90625 42 \n",
       "C 28.5 42 32.203125 39.40625 34.5 35.796875 \n",
       "C 35.796875 33.703125 35.796875 33.40625 35.796875 31.609375 \n",
       "z\n",
       "\" id=\"CMR17-100\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(33.351976 128.317367)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "       <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "       <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "       <use transform=\"translate(170.863169 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(221.758539 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(272.653909 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(318.344386 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(369.239756 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(409.725416 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_34\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#md5943156f4\" y=\"95.32785\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- BiologicalProcess -->\n",
       "      <defs>\n",
       "       <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 39.59375 0 \n",
       "C 52.40625 0 60.703125 9.109375 60.703125 18.1875 \n",
       "C 60.703125 27.203125 52.5 34.890625 41.796875 34.890625 \n",
       "C 51.40625 36.8125 58.09375 43.296875 58.09375 51 \n",
       "C 58.09375 59.40625 49.90625 68 37 68 \n",
       "z\n",
       "M 19.5 36 \n",
       "L 19.5 61.515625 \n",
       "C 19.5 64.921875 19.703125 65.40625 23.703125 65.40625 \n",
       "L 36.5 65.40625 \n",
       "C 46.40625 65.40625 50.796875 57.40625 50.796875 51.203125 \n",
       "C 50.796875 43.609375 44.703125 36 33.90625 36 \n",
       "z\n",
       "M 23.703125 2.59375 \n",
       "C 19.703125 2.59375 19.5 3.078125 19.5 6.453125 \n",
       "L 19.5 34.40625 \n",
       "L 38 34.40625 \n",
       "C 48.296875 34.40625 53.296875 25.390625 53.296875 18.15625 \n",
       "C 53.296875 10.921875 47.59375 2.59375 36.703125 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-66\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(7.2 98.787099)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-66\"/>\n",
       "       <use transform=\"translate(65.188754 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(90.059813 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(135.75029 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(160.621349 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(206.311826 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(252.002303 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(276.873361 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(317.359022 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(363.049499 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(387.920557 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(450.506858 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(485.787626 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(534.080541 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(574.566202 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(615.051862 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(650.8532 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_7\">\n",
       "     <g id=\"line2d_35\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#md5943156f4\" y=\"65.797582\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- Complex -->\n",
       "      <defs>\n",
       "       <path d=\"M 26 23.078125 \n",
       "C 29 26.96875 32.40625 31.34375 35.203125 34.625 \n",
       "C 38.5 38.40625 42.296875 40 46.703125 40 \n",
       "L 46.703125 42.59375 \n",
       "C 45 42.59375 41.5 42.59375 39.703125 42.59375 \n",
       "C 37 42.59375 33.90625 42.59375 31.59375 42.6875 \n",
       "L 31.59375 40.09375 \n",
       "C 33 39.890625 33.796875 38.890625 33.796875 37.5 \n",
       "C 33.796875 35.703125 32.796875 34.609375 32.296875 33.921875 \n",
       "L 24.90625 24.578125 \n",
       "L 16.09375 36.03125 \n",
       "C 15.203125 37.125 15.203125 37.3125 15.203125 37.8125 \n",
       "C 15.203125 39.015625 16.203125 40 17.90625 40 \n",
       "L 17.90625 42.6875 \n",
       "C 15.5 42.59375 11.203125 42.59375 8.703125 42.59375 \n",
       "C 6 42.59375 2.796875 42.59375 0.59375 42.6875 \n",
       "L 0.59375 40 \n",
       "C 6 40 7.203125 39.59375 9.59375 36.609375 \n",
       "L 21.09375 21.796875 \n",
       "C 21.296875 21.59375 21.796875 21 21.796875 20.703125 \n",
       "C 21.796875 20.296875 13.09375 9.671875 12 8.28125 \n",
       "C 7.5 2.890625 2.796875 2.5 0.09375 2.5 \n",
       "L 0.09375 0 \n",
       "C 1.796875 0 5.296875 0 7.09375 0 \n",
       "C 9.09375 0 13.5 0 15.296875 -0.09375 \n",
       "L 15.296875 2.5 \n",
       "C 14.703125 2.59375 13.09375 2.796875 13.09375 5.078125 \n",
       "C 13.09375 6.875 13.90625 7.96875 14.796875 9.15625 \n",
       "L 23.09375 19.3125 \n",
       "L 31.796875 8 \n",
       "C 32.59375 6.90625 33.796875 5.40625 33.796875 4.703125 \n",
       "C 33.796875 3.296875 32.59375 2.5 31 2.5 \n",
       "L 31 -0.09375 \n",
       "C 33.40625 0 37.703125 0 40.203125 0 \n",
       "C 42.90625 0 46.09375 0 48.296875 -0.09375 \n",
       "L 48.296875 2.5 \n",
       "C 43.296875 2.5 41.796875 2.6875 39.40625 5.78125 \n",
       "z\n",
       "\" id=\"CMR17-120\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(43.868113 69.256831)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(66.509881 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(112.200358 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(189.119948 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(240.015317 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(264.886376 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(305.372036 0)scale(0.996264)\" xlink:href=\"#CMR17-120\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_8\">\n",
       "     <g id=\"line2d_36\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#md5943156f4\" y=\"36.267314\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- Gene -->\n",
       "      <defs>\n",
       "       <path d=\"M 62.09375 19.265625 \n",
       "C 62.09375 23.015625 62.40625 23.515625 68.59375 23.515625 \n",
       "L 68.59375 26.09375 \n",
       "C 66.203125 26 60.796875 26 58.09375 26 \n",
       "C 55.296875 26 47.5 26 43.703125 26 \n",
       "L 43.703125 23.515625 \n",
       "L 46.703125 23.515625 \n",
       "C 55.296875 23.515625 55.5 22.40625 55.5 18.765625 \n",
       "L 55.5 12.09375 \n",
       "C 55.5 0.984375 42.5 0.984375 39.40625 0.984375 \n",
       "C 29.09375 0.984375 12.203125 7.984375 12.203125 33.75 \n",
       "C 12.203125 59.5 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.65625 59.90625 42.828125 \n",
       "C 60.09375 41.625 60.09375 41.421875 61 41.421875 \n",
       "C 62.09375 41.421875 62.09375 41.625 62.09375 43.421875 \n",
       "L 62.09375 67 \n",
       "C 62.09375 68.78125 62 68.890625 61.40625 68.890625 \n",
       "C 61 68.890625 60.90625 68.78125 60.203125 67.59375 \n",
       "L 55.796875 59.234375 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 13.203125 19.796875 -2 37.40625 -2 \n",
       "C 45.5 -2 53.40625 0.875 56.5 6.515625 \n",
       "C 58.203125 2.46875 61 -0.3125 61.5 -0.3125 \n",
       "C 62 -0.3125 62.09375 -0.203125 62.09375 1.5625 \n",
       "z\n",
       "\" id=\"CMR17-71\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(60.126662 39.726563)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "       <use transform=\"translate(72.355346 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(112.841006 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(163.736376 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path d=\"M 89.335068 257.744323 \n",
       "L 89.335068 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path d=\"M 332.680527 257.744323 \n",
       "L 332.680527 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 89.335068 257.744323 \n",
       "L 332.680527 257.744323 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 89.335068 21.50218 \n",
       "L 332.680527 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_12\">\n",
       "    <!-- Number of nodes: 9269 -->\n",
       "    <defs>\n",
       "     <path d=\"M 20.5 66.40625 \n",
       "C 19.796875 67.5 19.703125 67.5 17.796875 67.5 \n",
       "L 5.09375 67.5 \n",
       "L 5.09375 65 \n",
       "L 7 65 \n",
       "C 11 65 12.796875 64.515625 13.09375 64.40625 \n",
       "L 13.09375 9.53125 \n",
       "C 13.09375 7.046875 13.09375 2.390625 5.09375 2.390625 \n",
       "L 5.09375 -0.203125 \n",
       "C 7.40625 0 11.59375 0 14.09375 0 \n",
       "C 16.59375 0 20.796875 0 23.09375 -0.203125 \n",
       "L 23.09375 2.390625 \n",
       "C 15.09375 2.390625 15.09375 7.046875 15.09375 9.53125 \n",
       "L 15.09375 63.015625 \n",
       "C 15.796875 62.421875 15.796875 62.21875 16.5 61.234375 \n",
       "L 53.59375 1.1875 \n",
       "C 54.296875 0 54.59375 0 55.09375 0 \n",
       "C 56.09375 0 56.09375 0.296875 56.09375 2.171875 \n",
       "L 56.09375 57.890625 \n",
       "C 56.09375 60.359375 56.09375 65 64.09375 65 \n",
       "L 64.09375 67.59375 \n",
       "C 61.796875 67.5 57.59375 67.5 55.09375 67.5 \n",
       "C 52.59375 67.5 48.40625 67.5 46.09375 67.59375 \n",
       "L 46.09375 65 \n",
       "C 54.09375 65 54.09375 60.34375 54.09375 57.859375 \n",
       "L 54.09375 11.921875 \n",
       "z\n",
       "\" id=\"CMR17-78\"/>\n",
       "     <path d=\"M 15.296875 40 \n",
       "L 26.703125 40 \n",
       "L 26.703125 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 54.59375 \n",
       "C 15.09375 62.84375 19.296875 68 23.59375 68 \n",
       "C 24.90625 68 26.40625 67.609375 27.40625 66.90625 \n",
       "C 26.59375 66.703125 24.5 66.015625 24.5 63.546875 \n",
       "C 24.5 60.953125 26.5 60.15625 27.90625 60.15625 \n",
       "C 29.296875 60.15625 31.296875 60.953125 31.296875 63.546875 \n",
       "C 31.296875 67.3125 27.703125 69.59375 23.703125 69.59375 \n",
       "C 18 69.59375 10.203125 64.9375 10.203125 54.390625 \n",
       "L 10.203125 42.578125 \n",
       "L 2.203125 42.578125 \n",
       "L 2.203125 40 \n",
       "L 10.203125 40 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.40625 0 13 0 \n",
       "C 15.90625 0 21 0 23.703125 -0.203125 \n",
       "L 23.703125 2.390625 \n",
       "C 16.5 2.390625 15.296875 2.390625 15.296875 6.859375 \n",
       "z\n",
       "\" id=\"CMR17-102\"/>\n",
       "     <path d=\"M 16.59375 38.921875 \n",
       "C 16.59375 41.40625 14.59375 43 12.5 43 \n",
       "C 10.5 43 8.40625 41.40625 8.40625 38.921875 \n",
       "C 8.40625 36.4375 10.40625 34.84375 12.5 34.84375 \n",
       "C 14.5 34.84375 16.59375 36.4375 16.59375 38.921875 \n",
       "M 16.59375 4.078125 \n",
       "C 16.59375 6.5625 14.59375 8.15625 12.5 8.15625 \n",
       "C 10.5 8.15625 8.40625 6.5625 8.40625 4.078125 \n",
       "C 8.40625 1.59375 10.40625 0 12.5 0 \n",
       "C 14.5 0 16.59375 1.59375 16.59375 4.078125 \n",
       "\" id=\"CMR17-58\"/>\n",
       "     <path d=\"M 35.203125 29.125 \n",
       "C 35.203125 7.25 26.09375 0.484375 18.59375 0.484375 \n",
       "C 16.296875 0.484375 10.703125 0.484375 8.40625 4.59375 \n",
       "C 11 4.1875 12.90625 5.578125 12.90625 8.09375 \n",
       "C 12.90625 10.8125 10.703125 11.703125 9.296875 11.703125 \n",
       "C 8.40625 11.703125 5.703125 11.296875 5.703125 7.90625 \n",
       "C 5.703125 1.109375 11.59375 -2 18.796875 -2 \n",
       "C 30.296875 -2 42 10.53125 42 32.390625 \n",
       "C 42 59.640625 30.796875 65.296875 23.203125 65.296875 \n",
       "C 13.296875 65.296875 3.796875 56.6875 3.796875 43.40625 \n",
       "C 3.796875 31.109375 11.90625 22 22.09375 22 \n",
       "C 30.5 22 34.203125 29.734375 35.203125 33 \n",
       "z\n",
       "M 22.296875 23.59375 \n",
       "C 19.59375 23.59375 15.796875 24.09375 12.703125 30.03125 \n",
       "C 10.59375 33.890625 10.59375 38.453125 10.59375 43.296875 \n",
       "C 10.59375 49.15625 10.59375 53.203125 13.40625 57.5625 \n",
       "C 14.796875 59.640625 17.40625 63 23.203125 63 \n",
       "C 35 63 35 45.078125 35 41.125 \n",
       "C 35 34.09375 31.796875 23.59375 22.296875 23.59375 \n",
       "\" id=\"CMR17-57\"/>\n",
       "     <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "\" id=\"CMR17-54\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(151.309372 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "     <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(194.284936 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "     <use transform=\"translate(247.782744 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(288.268404 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(353.625139 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(399.315616 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(456.865095 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(507.760464 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(556.05338 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "     <use transform=\"translate(606.94875 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(647.43441 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(683.235748 0)scale(0.996264)\" xlink:href=\"#CMR17-58\"/>\n",
       "     <use transform=\"translate(748.592467 0)scale(0.996264)\" xlink:href=\"#CMR17-57\"/>\n",
       "     <use transform=\"translate(794.282944 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "     <use transform=\"translate(839.973421 0)scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "     <use transform=\"translate(885.663898 0)scale(0.996264)\" xlink:href=\"#CMR17-57\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"legend_1\">\n",
       "    <g id=\"patch_15\">\n",
       "     <path d=\"M 288.699192 252.744323 \n",
       "L 325.680527 252.744323 \n",
       "Q 327.680527 252.744323 327.680527 250.744323 \n",
       "L 327.680527 237.807145 \n",
       "Q 327.680527 235.807145 325.680527 235.807145 \n",
       "L 288.699192 235.807145 \n",
       "Q 286.699192 235.807145 286.699192 237.807145 \n",
       "L 286.699192 250.744323 \n",
       "Q 286.699192 252.744323 288.699192 252.744323 \n",
       "z\n",
       "\" style=\"fill:#ffffff;opacity:0.8;stroke:#cccccc;stroke-linejoin:miter;\"/>\n",
       "    </g>\n",
       "    <g id=\"patch_16\">\n",
       "     <path d=\"M 290.699192 246.807145 \n",
       "L 310.699192 246.807145 \n",
       "L 310.699192 239.807145 \n",
       "L 290.699192 239.807145 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "    </g>\n",
       "    <g id=\"text_13\">\n",
       "     <!-- 0 -->\n",
       "     <g transform=\"translate(318.699192 246.807145)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_2\">\n",
       "   <g id=\"patch_17\">\n",
       "    <path d=\"M 458.754541 257.744323 \n",
       "L 702.1 257.744323 \n",
       "L 702.1 21.50218 \n",
       "L 458.754541 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_18\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 254.270174 \n",
       "L 469.815698 254.270174 \n",
       "L 469.815698 247.321876 \n",
       "L -14450.723697 247.321876 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_19\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 240.373577 \n",
       "L 469.815698 240.373577 \n",
       "L 469.815698 233.425279 \n",
       "L -14450.723697 233.425279 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_20\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 226.476981 \n",
       "L 528.308751 226.476981 \n",
       "L 528.308751 219.528682 \n",
       "L -14450.723697 219.528682 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_21\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 212.580384 \n",
       "L 534.522596 212.580384 \n",
       "L 534.522596 205.632086 \n",
       "L -14450.723697 205.632086 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_22\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 198.683787 \n",
       "L 541.790293 198.683787 \n",
       "L 541.790293 191.735489 \n",
       "L -14450.723697 191.735489 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_23\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 184.787191 \n",
       "L 557.520021 184.787191 \n",
       "L 557.520021 177.838892 \n",
       "L -14450.723697 177.838892 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_24\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 170.890594 \n",
       "L 558.609312 170.890594 \n",
       "L 558.609312 163.942296 \n",
       "L -14450.723697 163.942296 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_25\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 156.993997 \n",
       "L 561.581891 156.993997 \n",
       "L 561.581891 150.045699 \n",
       "L -14450.723697 150.045699 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_26\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 143.097401 \n",
       "L 571.540105 143.097401 \n",
       "L 571.540105 136.149102 \n",
       "L -14450.723697 136.149102 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_27\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 129.200804 \n",
       "L 582.806038 129.200804 \n",
       "L 582.806038 122.252506 \n",
       "L -14450.723697 122.252506 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_28\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 115.304207 \n",
       "L 587.924119 115.304207 \n",
       "L 587.924119 108.355909 \n",
       "L -14450.723697 108.355909 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_29\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 101.407611 \n",
       "L 615.913536 101.407611 \n",
       "L 615.913536 94.459312 \n",
       "L -14450.723697 94.459312 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_30\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 87.511014 \n",
       "L 618.891103 87.511014 \n",
       "L 618.891103 80.562716 \n",
       "L -14450.723697 80.562716 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_31\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 73.614417 \n",
       "L 634.239837 73.614417 \n",
       "L 634.239837 66.666119 \n",
       "L -14450.723697 66.666119 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_32\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 59.717821 \n",
       "L 671.430466 59.717821 \n",
       "L 671.430466 52.769522 \n",
       "L -14450.723697 52.769522 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_33\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 45.821224 \n",
       "L 676.413803 45.821224 \n",
       "L 676.413803 38.872926 \n",
       "L -14450.723697 38.872926 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_34\">\n",
       "    <path clip-path=\"url(#p88917dd0da)\" d=\"M -14450.723697 31.924627 \n",
       "L 691.038843 31.924627 \n",
       "L 691.038843 24.976329 \n",
       "L -14450.723697 24.976329 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_3\">\n",
       "    <g id=\"xtick_29\">\n",
       "     <g id=\"line2d_37\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"469.815698\" xlink:href=\"#m2a62420e08\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_14\">\n",
       "      <!-- $10^{0}$ -->\n",
       "      <g transform=\"translate(462.848744 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_30\">\n",
       "     <g id=\"line2d_38\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"519.550829\" xlink:href=\"#m2a62420e08\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- $10^{1}$ -->\n",
       "      <g transform=\"translate(512.583875 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-49\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_31\">\n",
       "     <g id=\"line2d_39\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"569.28596\" xlink:href=\"#m2a62420e08\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- $10^{2}$ -->\n",
       "      <g transform=\"translate(562.319006 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_32\">\n",
       "     <g id=\"line2d_40\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"619.021092\" xlink:href=\"#m2a62420e08\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- $10^{3}$ -->\n",
       "      <g transform=\"translate(612.054138 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-51\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_33\">\n",
       "     <g id=\"line2d_41\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"668.756223\" xlink:href=\"#m2a62420e08\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_18\">\n",
       "      <!-- $10^{4}$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(661.789269 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_34\">\n",
       "     <g id=\"line2d_42\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"458.782021\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_35\">\n",
       "     <g id=\"line2d_43\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"462.111629\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_36\">\n",
       "     <g id=\"line2d_44\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"464.995866\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_37\">\n",
       "     <g id=\"line2d_45\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"467.539943\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_38\">\n",
       "     <g id=\"line2d_46\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"484.787464\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_39\">\n",
       "     <g id=\"line2d_47\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"493.545386\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_40\">\n",
       "     <g id=\"line2d_48\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"499.759231\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_41\">\n",
       "     <g id=\"line2d_49\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"504.579063\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_42\">\n",
       "     <g id=\"line2d_50\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"508.517152\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_43\">\n",
       "     <g id=\"line2d_51\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"511.84676\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_44\">\n",
       "     <g id=\"line2d_52\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"514.730997\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_45\">\n",
       "     <g id=\"line2d_53\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"517.275074\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_46\">\n",
       "     <g id=\"line2d_54\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"534.522596\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_47\">\n",
       "     <g id=\"line2d_55\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"543.280517\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_48\">\n",
       "     <g id=\"line2d_56\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"549.494362\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_49\">\n",
       "     <g id=\"line2d_57\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"554.314194\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_50\">\n",
       "     <g id=\"line2d_58\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"558.252284\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_51\">\n",
       "     <g id=\"line2d_59\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"561.581891\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_52\">\n",
       "     <g id=\"line2d_60\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"564.466128\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_53\">\n",
       "     <g id=\"line2d_61\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"567.010206\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_54\">\n",
       "     <g id=\"line2d_62\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"584.257727\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_55\">\n",
       "     <g id=\"line2d_63\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"593.015649\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_56\">\n",
       "     <g id=\"line2d_64\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"599.229493\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_57\">\n",
       "     <g id=\"line2d_65\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"604.049325\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_58\">\n",
       "     <g id=\"line2d_66\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"607.987415\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_59\">\n",
       "     <g id=\"line2d_67\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"611.317022\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_60\">\n",
       "     <g id=\"line2d_68\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"614.20126\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_61\">\n",
       "     <g id=\"line2d_69\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"616.745337\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_62\">\n",
       "     <g id=\"line2d_70\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"633.992858\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_63\">\n",
       "     <g id=\"line2d_71\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"642.75078\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_64\">\n",
       "     <g id=\"line2d_72\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"648.964625\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_65\">\n",
       "     <g id=\"line2d_73\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"653.784457\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_66\">\n",
       "     <g id=\"line2d_74\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"657.722546\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_67\">\n",
       "     <g id=\"line2d_75\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"661.052154\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_68\">\n",
       "     <g id=\"line2d_76\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"663.936391\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_69\">\n",
       "     <g id=\"line2d_77\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"666.480468\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_70\">\n",
       "     <g id=\"line2d_78\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"683.727989\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_71\">\n",
       "     <g id=\"line2d_79\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"692.485911\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_72\">\n",
       "     <g id=\"line2d_80\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"698.699756\" xlink:href=\"#m676e07c72e\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_4\">\n",
       "    <g id=\"ytick_9\">\n",
       "     <g id=\"line2d_81\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"250.796025\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_19\">\n",
       "      <!-- prognosticBiomarkerFor -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.09375 26.140625 \n",
       "C 26 26.34375 25.5 26.9375 25.5 27.234375 \n",
       "C 25.5 27.4375 32.59375 33.921875 33.5 34.71875 \n",
       "C 39.203125 39.5 42.796875 40 45.40625 40 \n",
       "L 45.40625 42.703125 \n",
       "C 44.796875 42.703125 44.59375 42.703125 44.5 42.5 \n",
       "L 39.59375 42.5 \n",
       "C 36.796875 42.5 32.5 42.5 29.796875 42.703125 \n",
       "L 29.796875 40.09375 \n",
       "C 31.203125 39.984375 32.203125 39.1875 32.203125 37.796875 \n",
       "C 32.203125 35.796875 30.203125 33.90625 30.09375 33.90625 \n",
       "L 15 20.140625 \n",
       "L 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 6.65625 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 9.90625 0 12.40625 0 \n",
       "C 14.90625 0 19.296875 0 21.703125 -0.203125 \n",
       "L 21.703125 2.390625 \n",
       "C 15.296875 2.390625 14.90625 2.78125 14.90625 6.6875 \n",
       "L 14.90625 17.65625 \n",
       "L 21.796875 23.9375 \n",
       "L 31.90625 8.96875 \n",
       "C 33.09375 7.171875 34.09375 5.765625 34.09375 4.46875 \n",
       "C 34.09375 2.6875 32.296875 2.390625 31.09375 2.390625 \n",
       "L 31.09375 -0.203125 \n",
       "C 33.5 0 37.796875 0 40.296875 0 \n",
       "C 42.09375 0 45.796875 0 47.5 0 \n",
       "L 47.5 2.390625 \n",
       "C 42.796875 2.390625 41.703125 3.28125 38.703125 7.671875 \n",
       "z\n",
       "\" id=\"CMR17-107\"/>\n",
       "       <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(346.980527 254.255274)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(86.176138 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(131.866615 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(177.557092 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(228.452462 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(274.142939 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(309.944276 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(345.225044 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(370.096103 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(410.581763 0)scale(0.996264)\" xlink:href=\"#CMR17-66\"/>\n",
       "       <use transform=\"translate(475.770518 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(500.641576 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(546.332053 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(623.251643 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(668.94212 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(704.222888 0)scale(0.996264)\" xlink:href=\"#CMR17-107\"/>\n",
       "       <use transform=\"translate(749.91335 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(790.39901 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(825.679778 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "       <use transform=\"translate(877.856369 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(923.546846 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_10\">\n",
       "     <g id=\"line2d_82\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"236.899428\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_20\">\n",
       "      <!-- rateLimitingStepOf -->\n",
       "      <defs>\n",
       "       <path d=\"M 54.203125 25.234375 \n",
       "L 52.40625 25.234375 \n",
       "C 51.59375 15.359375 50.296875 2.59375 33.203125 2.59375 \n",
       "L 23.90625 2.59375 \n",
       "C 19.90625 2.59375 19.703125 3.078125 19.703125 6.46875 \n",
       "L 19.703125 60.90625 \n",
       "C 19.703125 64.515625 19.90625 65.5 27.5 65.5 \n",
       "L 29.703125 65.5 \n",
       "L 29.703125 68 \n",
       "C 25.59375 68 20.796875 68 16.703125 68 \n",
       "C 13.59375 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 7.0625 \n",
       "C 13.09375 2.59375 12 2.59375 5.09375 2.59375 \n",
       "L 5.09375 0 \n",
       "L 51.796875 0 \n",
       "z\n",
       "\" id=\"CMR17-76\"/>\n",
       "       <path d=\"M 43.796875 67.03125 \n",
       "C 43.796875 68.8125 43.703125 68.921875 43.09375 68.921875 \n",
       "C 42.5 68.921875 42.296875 68.515625 41.90625 67.625 \n",
       "L 38.90625 61.046875 \n",
       "C 35.09375 66.515625 29.296875 69 23 69 \n",
       "C 12.796875 69 4.59375 60.953125 4.59375 50.828125 \n",
       "C 4.59375 47.75 5.296875 43.375 8.703125 39.21875 \n",
       "C 12.40625 34.65625 15.90625 33.859375 23.90625 31.765625 \n",
       "C 26.90625 30.96875 31.90625 29.671875 32.90625 29.28125 \n",
       "C 38.59375 26.90625 41.59375 20.953125 41.59375 15.390625 \n",
       "C 41.59375 8.234375 36.5 0.59375 27.796875 0.59375 \n",
       "C 17.09375 0.59375 7.09375 6.546875 6.40625 20.484375 \n",
       "C 6.296875 22.078125 6.203125 22.171875 5.5 22.171875 \n",
       "C 4.703125 22.171875 4.59375 22.078125 4.59375 20.078125 \n",
       "L 4.59375 -0.015625 \n",
       "C 4.59375 -1.796875 4.703125 -1.90625 5.296875 -1.90625 \n",
       "C 6 -1.90625 6.203125 -1.40625 7.59375 1.6875 \n",
       "C 7.703125 1.984375 7.703125 2.1875 9.59375 5.953125 \n",
       "C 14.59375 -0.609375 23 -2 27.796875 -2 \n",
       "C 38.703125 -2 46.40625 7.046875 46.40625 17.5625 \n",
       "C 46.40625 22.71875 44.5 26.6875 43.40625 28.484375 \n",
       "C 39.203125 34.84375 34.703125 36.03125 29.90625 37.21875 \n",
       "C 29.203125 37.53125 29 37.53125 23.40625 38.90625 \n",
       "C 18 40.296875 15.703125 41 13.203125 43.6875 \n",
       "C 10.40625 46.75 9.40625 49.9375 9.40625 53.109375 \n",
       "C 9.40625 59.5625 14.59375 66.609375 23.09375 66.609375 \n",
       "C 33.703125 66.609375 40.296875 59.15625 41.796875 46.765625 \n",
       "C 42.09375 44.96875 42.203125 44.875 42.90625 44.875 \n",
       "C 43.796875 44.875 43.796875 45.171875 43.796875 46.859375 \n",
       "z\n",
       "\" id=\"CMR17-83\"/>\n",
       "       <path d=\"M 67.296875 33.359375 \n",
       "C 67.296875 53.703125 52.59375 69 36 69 \n",
       "C 19 69 4.59375 53.5 4.59375 33.359375 \n",
       "C 4.59375 13.203125 19.203125 -2 35.90625 -2 \n",
       "C 53 -2 67.296875 13.5 67.296875 33.359375 \n",
       "M 36 -0.203125 \n",
       "C 24.59375 -0.203125 12.203125 11.109375 12.203125 34.640625 \n",
       "C 12.203125 58.0625 25.796875 67.203125 35.90625 67.203125 \n",
       "C 46.5 67.203125 59.703125 57.78125 59.703125 34.640625 \n",
       "C 59.703125 10.71875 47 -0.203125 36 -0.203125 \n",
       "\" id=\"CMR17-79\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(367.182571 240.358677)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(35.280768 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(80.971245 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(116.252013 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(156.737673 0)scale(0.996264)\" xlink:href=\"#CMR17-76\"/>\n",
       "       <use transform=\"translate(214.119081 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(238.99014 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(315.909729 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(340.780788 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(376.061556 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(400.932614 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(451.827984 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(497.518461 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "       <use transform=\"translate(548.413831 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(583.694599 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(624.180259 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(675.075629 0)scale(0.996264)\" xlink:href=\"#CMR17-79\"/>\n",
       "       <use transform=\"translate(746.790417 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_11\">\n",
       "     <g id=\"line2d_83\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"223.002832\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_21\">\n",
       "      <!-- hasProduct -->\n",
       "      <g transform=\"translate(401.996568 226.462081)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(194.973485 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(230.254253 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(278.547169 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(329.442538 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "       <use transform=\"translate(380.337908 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(420.823569 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_12\">\n",
       "     <g id=\"line2d_84\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"209.106235\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_22\">\n",
       "      <!-- hasReactant -->\n",
       "      <g transform=\"translate(398.149909 212.565484)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-82\"/>\n",
       "       <use transform=\"translate(200.178287 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(240.663947 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(286.354424 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(326.840085 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(362.120852 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(407.811329 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(456.104245 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_13\">\n",
       "     <g id=\"line2d_85\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"195.209638\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_23\">\n",
       "      <!-- biomarkerFor -->\n",
       "      <g transform=\"translate(393.41761 198.668887)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(121.456905 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(198.376495 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(244.066972 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(279.34774 0)scale(0.996264)\" xlink:href=\"#CMR17-107\"/>\n",
       "       <use transform=\"translate(325.038202 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(365.523863 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(400.80463 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "       <use transform=\"translate(452.981222 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(498.671699 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_14\">\n",
       "     <g id=\"line2d_86\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"181.313042\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_24\">\n",
       "      <!-- hasMember -->\n",
       "      <defs>\n",
       "       <path d=\"M 20.90625 66 \n",
       "C 20.296875 67.5 20.203125 67.5 18.09375 67.5 \n",
       "L 5.296875 67.5 \n",
       "L 5.296875 65 \n",
       "C 12.203125 65 13.296875 65 13.296875 60.546875 \n",
       "L 13.296875 9.53125 \n",
       "C 13.296875 7.046875 13.296875 2.390625 5.296875 2.390625 \n",
       "L 5.296875 -0.203125 \n",
       "C 7.59375 0 11.796875 0 14.296875 0 \n",
       "C 16.796875 0 21 0 23.296875 -0.203125 \n",
       "L 23.296875 2.390625 \n",
       "C 15.296875 2.390625 15.296875 7.046875 15.296875 9.53125 \n",
       "L 15.296875 64.609375 \n",
       "L 15.40625 64.609375 \n",
       "L 38.90625 1.390625 \n",
       "C 39.296875 0.390625 39.5 -0.203125 40.296875 -0.203125 \n",
       "C 41.09375 -0.203125 41.296875 0.390625 41.703125 1.390625 \n",
       "L 65.40625 65.09375 \n",
       "L 65.5 65.09375 \n",
       "L 65.5 6.84375 \n",
       "C 65.5 2.390625 64.40625 2.390625 57.5 2.390625 \n",
       "L 57.5 -0.203125 \n",
       "C 60.203125 0 65.59375 0 68.5 0 \n",
       "C 71.40625 0 76.90625 0 79.59375 -0.203125 \n",
       "L 79.59375 2.390625 \n",
       "C 72.703125 2.390625 71.59375 2.390625 71.59375 6.84375 \n",
       "L 71.59375 60.546875 \n",
       "C 71.59375 65 72.703125 65 79.59375 65 \n",
       "L 79.59375 67.5 \n",
       "L 66.90625 67.5 \n",
       "C 64.796875 67.5 64.703125 67.40625 64.09375 65.90625 \n",
       "L 42.5 7.84375 \n",
       "z\n",
       "\" id=\"CMR17-77\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(401.581483 184.772291)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-77\"/>\n",
       "       <use transform=\"translate(217.074125 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(257.559786 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(331.876922 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "       <use transform=\"translate(385.37473 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(425.86039 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_15\">\n",
       "     <g id=\"line2d_87\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"167.416445\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_25\">\n",
       "      <!-- isA -->\n",
       "      <g transform=\"translate(437.58543 170.875694)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(24.871058 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(60.672396 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_16\">\n",
       "     <g id=\"line2d_88\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"153.519848\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_26\">\n",
       "      <!-- directlyDecreases -->\n",
       "      <defs>\n",
       "       <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 37 0 \n",
       "C 53.09375 0 66 14.734375 66 33.453125 \n",
       "C 66 52.28125 53.296875 68 37 68 \n",
       "z\n",
       "M 23.796875 2.59375 \n",
       "C 19.796875 2.59375 19.59375 3.078125 19.59375 6.46875 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 35 65.40625 \n",
       "C 43.296875 65.40625 58.59375 60.53125 58.59375 33.453125 \n",
       "C 58.59375 19.40625 54.5 13.734375 52.59375 11.046875 \n",
       "C 50.203125 7.96875 44.5 2.59375 35 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-68\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(376.730233 156.979097)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(111.047196 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(151.532857 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(192.018517 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(227.299285 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(252.170343 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "       <use transform=\"translate(300.463259 0)scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "       <use transform=\"translate(370.856815 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(411.342475 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(451.828136 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(487.108904 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(527.594564 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(573.285041 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(609.086379 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(649.572039 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_17\">\n",
       "     <g id=\"line2d_89\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"139.623252\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_27\">\n",
       "      <!-- causesNoChange -->\n",
       "      <g transform=\"translate(379.414579 143.082501)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(40.48566 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(86.176138 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "       <use transform=\"translate(137.071507 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(172.872845 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(213.358505 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(249.159843 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "       <use transform=\"translate(318.232273 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(363.92275 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(430.432631 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(481.328 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(527.018477 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(577.913847 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(623.604324 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_18\">\n",
       "     <g id=\"line2d_90\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"125.726655\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_28\">\n",
       "      <!-- directlyIncreases -->\n",
       "      <defs>\n",
       "       <path d=\"M 19.703125 61.015625 \n",
       "C 19.703125 65.5 20.90625 65.5 28.09375 65.5 \n",
       "L 28.09375 68 \n",
       "C 24.90625 68 19.796875 68 16.40625 68 \n",
       "C 13 68 7.90625 68 4.703125 68 \n",
       "L 4.703125 65.5 \n",
       "C 11.90625 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 6.984375 \n",
       "C 13.09375 2.5 11.90625 2.5 4.703125 2.5 \n",
       "L 4.703125 0 \n",
       "C 7.90625 0 13 0 16.40625 0 \n",
       "C 19.796875 0 24.90625 0 28.09375 0 \n",
       "L 28.09375 2.5 \n",
       "C 20.90625 2.5 19.703125 2.5 19.703125 6.984375 \n",
       "z\n",
       "\" id=\"CMR17-73\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(379.635993 129.185904)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(111.047196 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(151.532857 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(192.018517 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(227.299285 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(252.170343 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "       <use transform=\"translate(300.463259 0)scale(0.996264)\" xlink:href=\"#CMR17-73\"/>\n",
       "       <use transform=\"translate(333.101668 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(383.997038 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(424.482699 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(459.763466 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(500.249127 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(545.939604 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(581.740942 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(622.226602 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_19\">\n",
       "     <g id=\"line2d_91\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"111.830058\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_29\">\n",
       "      <!-- hasVariant -->\n",
       "      <defs>\n",
       "       <path d=\"M 59 58.515625 \n",
       "C 59.90625 60.71875 61.5 65 68.203125 65 \n",
       "L 68.203125 67.6875 \n",
       "C 65.90625 67.484375 63.296875 67.484375 60.796875 67.484375 \n",
       "C 58.40625 67.484375 54.296875 67.484375 52 67.6875 \n",
       "L 52 65 \n",
       "C 57 65 57.203125 61.5 57.203125 60.90625 \n",
       "C 57.203125 59.796875 56.90625 59.109375 56.59375 58.3125 \n",
       "L 36.90625 6.96875 \n",
       "L 16.296875 60.71875 \n",
       "C 16 61.421875 15.796875 61.921875 15.796875 62.40625 \n",
       "C 15.796875 65 19.796875 65 21.90625 65 \n",
       "L 21.90625 67.484375 \n",
       "C 18.703125 67.484375 14 67.484375 10.703125 67.484375 \n",
       "C 8.09375 67.484375 3.40625 67.484375 1 67.6875 \n",
       "L 1 65 \n",
       "C 6.796875 65 7.90625 64.609375 9.09375 61.421875 \n",
       "L 32.90625 -0.40625 \n",
       "C 33.40625 -1.796875 33.5 -2 34.59375 -2 \n",
       "C 35.703125 -2 35.796875 -1.796875 36.296875 -0.40625 \n",
       "z\n",
       "\" id=\"CMR17-86\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(404.902358 115.289307)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-86\"/>\n",
       "       <use transform=\"translate(193.652359 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(239.342836 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(274.623604 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(299.494662 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(345.185139 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(393.478055 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_20\">\n",
       "     <g id=\"line2d_92\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"97.933462\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_30\">\n",
       "      <!-- negativeCorrelation -->\n",
       "      <defs>\n",
       "       <path d=\"M 39.09375 33.796875 \n",
       "C 40 36 41.59375 40 47.40625 40 \n",
       "L 47.40625 42.59375 \n",
       "C 45.5 42.59375 43 42.59375 41.09375 42.59375 \n",
       "C 39 42.59375 35.703125 42.59375 33.703125 42.6875 \n",
       "L 33.703125 40.09375 \n",
       "C 36.203125 39.890625 37.59375 38.703125 37.59375 36.109375 \n",
       "C 37.59375 35.203125 37.59375 35 37 33.609375 \n",
       "L 26 5.890625 \n",
       "L 14.09375 35.71875 \n",
       "C 13.5 37.015625 13.5 37.203125 13.5 37.609375 \n",
       "C 13.5 40 16.796875 40 18.296875 40 \n",
       "L 18.296875 42.6875 \n",
       "C 15.90625 42.59375 11.5 42.59375 9 42.59375 \n",
       "C 6.296875 42.59375 3.203125 42.59375 1 42.6875 \n",
       "L 1 40 \n",
       "C 4.796875 40 6.90625 40 8.09375 36.90625 \n",
       "L 22.59375 0.6875 \n",
       "C 23.203125 -0.796875 23.296875 -1 24.203125 -1 \n",
       "C 25.09375 -1 25.203125 -0.796875 25.796875 0.6875 \n",
       "z\n",
       "\" id=\"CMR17-118\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(366.463086 101.392711)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(91.38103 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(137.071507 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(182.761984 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(218.042752 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(242.913811 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "       <use transform=\"translate(288.604273 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(329.089933 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(395.599814 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(441.290291 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(476.571059 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(511.851826 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(552.337487 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(577.208545 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(622.899022 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(658.17979 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(683.050849 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(728.741326 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_21\">\n",
       "     <g id=\"line2d_93\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"84.036865\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_31\">\n",
       "      <!-- positiveCorrelation -->\n",
       "      <g transform=\"translate(368.898396 87.496114)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(53.497809 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(99.188286 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(134.989623 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(159.860682 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(195.14145 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(220.012508 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "       <use transform=\"translate(265.70297 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(306.18863 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(372.698511 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(418.388988 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(453.669756 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(488.950524 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(529.436184 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(554.307243 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(599.99772 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(635.278488 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(660.149546 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(705.840023 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_22\">\n",
       "     <g id=\"line2d_94\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"70.140268\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_32\">\n",
       "      <!-- hasComponent -->\n",
       "      <g transform=\"translate(386.941878 73.599517)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(198.897065 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(244.587542 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(321.507132 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(375.004941 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(420.695418 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(471.590787 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(512.076448 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(560.369364 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_23\">\n",
       "     <g id=\"line2d_95\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"56.243672\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_33\">\n",
       "      <!-- increases -->\n",
       "      <g transform=\"translate(413.426013 59.702921)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(24.871058 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(116.252089 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(151.532857 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(192.018517 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(237.708994 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(273.510332 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(313.995992 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_24\">\n",
       "     <g id=\"line2d_96\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"42.347075\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_34\">\n",
       "      <!-- decreases -->\n",
       "      <g transform=\"translate(411.765583 45.806324)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(91.38103 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(131.866691 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(167.147459 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(207.633119 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(253.323596 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(289.124934 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(329.610594 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_25\">\n",
       "     <g id=\"line2d_97\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"458.754541\" xlink:href=\"#md5943156f4\" y=\"28.450478\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_35\">\n",
       "      <!-- association -->\n",
       "      <g transform=\"translate(404.321194 31.909727)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(81.491815 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(117.293152 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(165.586068 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(206.071729 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(230.942787 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(276.633264 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(311.914032 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(336.785091 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(382.475568 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_35\">\n",
       "    <path d=\"M 458.754541 257.744323 \n",
       "L 458.754541 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_36\">\n",
       "    <path d=\"M 702.1 257.744323 \n",
       "L 702.1 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_37\">\n",
       "    <path d=\"M 458.754541 257.744323 \n",
       "L 702.1 257.744323 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_38\">\n",
       "    <path d=\"M 458.754541 21.50218 \n",
       "L 702.1 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_36\">\n",
       "    <!-- Number of edges: 58301 -->\n",
       "    <defs>\n",
       "     <path d=\"M 11.40625 57.6875 \n",
       "C 12.40625 57.28125 16.5 56 20.703125 56 \n",
       "C 30 56 35.09375 61.109375 38 64.0625 \n",
       "C 38 64.953125 38 65.5 37.40625 65.5 \n",
       "C 37.296875 65.5 37.09375 65.5 36.296875 65.046875 \n",
       "C 32.796875 63.40625 28.703125 62.078125 23.703125 62.078125 \n",
       "C 20.703125 62.078125 16.203125 62.484375 11.296875 64.671875 \n",
       "C 10.203125 65.171875 10 65.171875 9.90625 65.171875 \n",
       "C 9.40625 65.171875 9.296875 65.0625 9.296875 63.078125 \n",
       "L 9.296875 34.421875 \n",
       "C 9.296875 32.640625 9.296875 32.140625 10.296875 32.140625 \n",
       "C 10.796875 32.140625 11 32.34375 11.5 33.03125 \n",
       "C 14.703125 37.515625 19.09375 39.40625 24.09375 39.40625 \n",
       "C 27.59375 39.40625 35.09375 37.21875 35.09375 20.140625 \n",
       "C 35.09375 16.953125 35.09375 11.171875 32.09375 6.578125 \n",
       "C 29.59375 2.484375 25.703125 0.390625 21.40625 0.390625 \n",
       "C 14.796875 0.390625 8.09375 5 6.296875 12.71875 \n",
       "C 6.703125 12.609375 7.5 12.421875 7.90625 12.421875 \n",
       "C 9.203125 12.421875 11.703125 13.125 11.703125 16.21875 \n",
       "C 11.703125 18.90625 9.796875 20 7.90625 20 \n",
       "C 5.59375 20 4.09375 18.59375 4.09375 15.796875 \n",
       "C 4.09375 7.09375 11 -2 21.59375 -2 \n",
       "C 31.90625 -2 41.703125 6.875 41.703125 19.75 \n",
       "C 41.703125 31.71875 33.90625 41 24.203125 41 \n",
       "C 19.09375 41 14.796875 39.109375 11.40625 35.53125 \n",
       "z\n",
       "\" id=\"CMR17-53\"/>\n",
       "     <path d=\"M 27.203125 35.375 \n",
       "C 33.5 38.546875 39.90625 43.328125 39.90625 50.984375 \n",
       "C 39.90625 60.015625 31.09375 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 58.734375 5.90625 49.6875 \n",
       "C 5.90625 47.203125 6.5 42.921875 10.40625 39.140625 \n",
       "C 11.40625 38.15625 15.59375 35.171875 18.296875 33.28125 \n",
       "C 13.796875 30.984375 3.296875 25.53125 3.296875 14.59375 \n",
       "C 3.296875 4.359375 13.09375 -2 22.796875 -2 \n",
       "C 33.5 -2 42.5 5.65625 42.5 15.78125 \n",
       "C 42.5 24.84375 36.40625 29.015625 32.40625 31.703125 \n",
       "z\n",
       "M 14.09375 44.09375 \n",
       "C 13.296875 44.59375 9.296875 47.671875 9.296875 52.359375 \n",
       "C 9.296875 58.421875 15.59375 63 22.796875 63 \n",
       "C 30.703125 63 36.5 57.4375 36.5 50.96875 \n",
       "C 36.5 41.703125 26.09375 36.421875 25.59375 36.421875 \n",
       "C 25.5 36.421875 25.40625 36.421875 24.59375 37.03125 \n",
       "z\n",
       "M 32.5 23.734375 \n",
       "C 34 22.640625 38.796875 19.375 38.796875 13.296875 \n",
       "C 38.796875 5.953125 31.40625 0.390625 23 0.390625 \n",
       "C 13.90625 0.390625 7 6.84375 7 14.6875 \n",
       "C 7 22.546875 13.09375 29.09375 20 32.1875 \n",
       "z\n",
       "\" id=\"CMR17-56\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(518.615265 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "     <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(194.284936 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "     <use transform=\"translate(247.782744 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(288.268404 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(353.625139 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(399.315616 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(456.865095 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(497.350755 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "     <use transform=\"translate(548.246125 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(593.936602 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(634.422262 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(670.2236 0)scale(0.996264)\" xlink:href=\"#CMR17-58\"/>\n",
       "     <use transform=\"translate(735.580319 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "     <use transform=\"translate(781.270796 0)scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "     <use transform=\"translate(826.961273 0)scale(0.996264)\" xlink:href=\"#CMR17-51\"/>\n",
       "     <use transform=\"translate(872.65175 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     <use transform=\"translate(918.342227 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"legend_2\">\n",
       "    <g id=\"patch_39\">\n",
       "     <path d=\"M 658.118665 252.744323 \n",
       "L 695.1 252.744323 \n",
       "Q 697.1 252.744323 697.1 250.744323 \n",
       "L 697.1 237.807145 \n",
       "Q 697.1 235.807145 695.1 235.807145 \n",
       "L 658.118665 235.807145 \n",
       "Q 656.118665 235.807145 656.118665 237.807145 \n",
       "L 656.118665 250.744323 \n",
       "Q 656.118665 252.744323 658.118665 252.744323 \n",
       "z\n",
       "\" style=\"fill:#ffffff;opacity:0.8;stroke:#cccccc;stroke-linejoin:miter;\"/>\n",
       "    </g>\n",
       "    <g id=\"patch_40\">\n",
       "     <path d=\"M 660.118665 246.807145 \n",
       "L 680.118665 246.807145 \n",
       "L 680.118665 239.807145 \n",
       "L 660.118665 239.807145 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "    </g>\n",
       "    <g id=\"text_37\">\n",
       "     <!-- 0 -->\n",
       "     <g transform=\"translate(688.118665 246.807145)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"pfc2dfe20f4\">\n",
       "   <rect height=\"236.242143\" width=\"243.345459\" x=\"89.335068\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p88917dd0da\">\n",
       "   <rect height=\"236.242143\" width=\"243.345459\" x=\"458.754541\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x117a90d30>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pbt.summary.plot_summary(graph, plt, figsize=(10, 4))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "## Differential Gene Expression\n",
    "\n",
    "Differential gene expression data can be obtained from many sources, including ADNI and other large clinical studies. This analysis is concerned with the log-fold-changes on each gene, and not necessarily the p-value. This is better as a data-driven process becuase it does not require a model or multiple hypothesis testing on raw data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "data_path = os.path.join(owncloud_base, 'SevAD.csv')\n",
    "target_columns = ['Gene.symbol', 'logFC']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene.symbol</th>\n",
       "      <th>logFC</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ZNF616</td>\n",
       "      <td>-4.244691</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DEFB125</td>\n",
       "      <td>3.974393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>SNAP23</td>\n",
       "      <td>3.337636</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PHLDB2</td>\n",
       "      <td>3.192559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>LOC389895</td>\n",
       "      <td>-4.296850</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/latex": [
       "\\begin{center}{\\begin{tabular}{llr}\n",
       "\\toprule\n",
       "{} & Gene.symbol &     logFC \\\\\n",
       "\\midrule\n",
       "0 &      ZNF616 & -4.244691 \\\\\n",
       "1 &     DEFB125 &  3.974393 \\\\\n",
       "3 &      SNAP23 &  3.337636 \\\\\n",
       "4 &      PHLDB2 &  3.192559 \\\\\n",
       "5 &   LOC389895 & -4.296850 \\\\\n",
       "\\bottomrule\n",
       "\\end{tabular}\n",
       "}\\end{center}"
      ],
      "text/plain": [
       "  Gene.symbol     logFC\n",
       "0      ZNF616 -4.244691\n",
       "1     DEFB125  3.974393\n",
       "3      SNAP23  3.337636\n",
       "4      PHLDB2  3.192559\n",
       "5   LOC389895 -4.296850"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.read_csv(data_path)\n",
    "df = df.loc[df['Gene.symbol'].notnull(), target_columns]\n",
    "df.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "A histogram of the log-fold-changes shows that the data are normally distributed, as expected for differential gene expression data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "# max_fc = {}\n",
    "# for row, symbol, fc in df.itertuples():\n",
    "#    if symbol not in max_fc or abs(max_fc[symbol]) < abs(fc):\n",
    "#        max_fc[symbol] = fc\n",
    "        \n",
    "data = {k: v for _, k, v in df.itertuples()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"274pt\" version=\"1.1\" viewBox=\"0 0 397 274\" width=\"397pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 274.853533 \n",
       "L 397.462353 274.853533 \n",
       "L 397.462353 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 51.962353 238.94218 \n",
       "L 386.762353 238.94218 \n",
       "L 386.762353 21.50218 \n",
       "L 51.962353 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#p135a16a4ee)\" d=\"M 67.180535 238.94218 \n",
       "L 97.616898 238.94218 \n",
       "L 97.616898 238.904231 \n",
       "L 67.180535 238.904231 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#p135a16a4ee)\" d=\"M 97.616898 238.94218 \n",
       "L 128.053262 238.94218 \n",
       "L 128.053262 238.657565 \n",
       "L 97.616898 238.657565 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#p135a16a4ee)\" d=\"M 128.053262 238.94218 \n",
       "L 158.489626 238.94218 \n",
       "L 158.489626 234.122703 \n",
       "L 128.053262 234.122703 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#p135a16a4ee)\" d=\"M 158.489626 238.94218 \n",
       "L 188.925989 238.94218 \n",
       "L 188.925989 212.397108 \n",
       "L 158.489626 212.397108 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#p135a16a4ee)\" d=\"M 188.925989 238.94218 \n",
       "L 219.362353 238.94218 \n",
       "L 219.362353 101.017857 \n",
       "L 188.925989 101.017857 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#p135a16a4ee)\" d=\"M 219.362353 238.94218 \n",
       "L 249.798717 238.94218 \n",
       "L 249.798717 31.856466 \n",
       "L 219.362353 31.856466 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#p135a16a4ee)\" d=\"M 249.798717 238.94218 \n",
       "L 280.23508 238.94218 \n",
       "L 280.23508 201.373029 \n",
       "L 249.798717 201.373029 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#p135a16a4ee)\" d=\"M 280.23508 238.94218 \n",
       "L 310.671444 238.94218 \n",
       "L 310.671444 233.060141 \n",
       "L 280.23508 233.060141 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#p135a16a4ee)\" d=\"M 310.671444 238.94218 \n",
       "L 341.107808 238.94218 \n",
       "L 341.107808 238.221156 \n",
       "L 310.671444 238.221156 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#p135a16a4ee)\" d=\"M 341.107808 238.94218 \n",
       "L 371.544171 238.94218 \n",
       "L 371.544171 238.828334 \n",
       "L 341.107808 238.828334 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"m883568acbb\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"77.978435\" xlink:href=\"#m883568acbb\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- $-4$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 65.90625 23 \n",
       "C 67.59375 23 69.40625 23 69.40625 25 \n",
       "C 69.40625 27 67.59375 27 65.90625 27 \n",
       "L 11.796875 27 \n",
       "C 10.09375 27 8.296875 27 8.296875 25 \n",
       "C 8.296875 23 10.09375 23 11.796875 23 \n",
       "z\n",
       "\" id=\"CMSY10-0\"/>\n",
       "       <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(71.613396 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"150.728385\" xlink:href=\"#m883568acbb\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- $-2$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(144.363345 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"223.478335\" xlink:href=\"#m883568acbb\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- $0$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "\" id=\"CMR17-48\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(220.987667 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"296.228284\" xlink:href=\"#m883568acbb\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- $2$ -->\n",
       "      <g transform=\"translate(293.737617 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"368.978234\" xlink:href=\"#m883568acbb\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- $4$ -->\n",
       "      <g transform=\"translate(366.487567 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_6\">\n",
       "     <!-- Log-Fold-Change -->\n",
       "     <defs>\n",
       "      <path d=\"M 54.203125 25.234375 \n",
       "L 52.40625 25.234375 \n",
       "C 51.59375 15.359375 50.296875 2.59375 33.203125 2.59375 \n",
       "L 23.90625 2.59375 \n",
       "C 19.90625 2.59375 19.703125 3.078125 19.703125 6.46875 \n",
       "L 19.703125 60.90625 \n",
       "C 19.703125 64.515625 19.90625 65.5 27.5 65.5 \n",
       "L 29.703125 65.5 \n",
       "L 29.703125 68 \n",
       "C 25.59375 68 20.796875 68 16.703125 68 \n",
       "C 13.59375 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 7.0625 \n",
       "C 13.09375 2.59375 12 2.59375 5.09375 2.59375 \n",
       "L 5.09375 0 \n",
       "L 51.796875 0 \n",
       "z\n",
       "\" id=\"CMR17-76\"/>\n",
       "      <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "\" id=\"CMR17-111\"/>\n",
       "      <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "\" id=\"CMR17-103\"/>\n",
       "      <path d=\"M 25.5 19 \n",
       "L 25.5 23.375 \n",
       "L 0.40625 23.375 \n",
       "L 0.40625 19 \n",
       "z\n",
       "\" id=\"CMR17-45\"/>\n",
       "      <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      <path d=\"M 15.296875 68.6875 \n",
       "L 3.40625 67.59375 \n",
       "L 3.40625 65 \n",
       "C 9.296875 65 10.203125 64.40625 10.203125 59.640625 \n",
       "L 10.203125 6.65625 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.203125 0 12.703125 0 \n",
       "C 15.296875 0 19.703125 0 22.09375 -0.203125 \n",
       "L 22.09375 2.390625 \n",
       "C 15.703125 2.390625 15.296875 2.78125 15.296875 6.65625 \n",
       "z\n",
       "\" id=\"CMR17-108\"/>\n",
       "      <path d=\"M 29.203125 67.59375 \n",
       "L 29.203125 65 \n",
       "C 35.09375 65 36 64.40625 36 59.6875 \n",
       "L 36 36.640625 \n",
       "C 35.59375 37.140625 31.5 43.59375 23.40625 43.59375 \n",
       "C 13.203125 43.59375 3.296875 34.5 3.296875 21.3125 \n",
       "C 3.296875 8.203125 12.59375 -1 22.40625 -1 \n",
       "C 30.90625 -1 35.296875 5.265625 35.796875 5.9375 \n",
       "L 35.796875 -0.984375 \n",
       "L 47.90625 -0.984375 \n",
       "L 47.90625 2 \n",
       "C 42 2 41.09375 2.59375 41.09375 7.390625 \n",
       "L 41.09375 68.6875 \n",
       "z\n",
       "M 35.796875 11.796875 \n",
       "C 35.796875 8.796875 34 6.09375 31.703125 4.09375 \n",
       "C 28.296875 1.09375 24.90625 0.59375 23 0.59375 \n",
       "C 20.09375 0.59375 9.703125 2.09375 9.703125 21.1875 \n",
       "C 9.703125 40.8125 21.296875 42 23.90625 42 \n",
       "C 28.5 42 32.203125 39.40625 34.5 35.796875 \n",
       "C 35.796875 33.703125 35.796875 33.40625 35.796875 31.609375 \n",
       "z\n",
       "\" id=\"CMR17-100\"/>\n",
       "      <path d=\"M 62.09375 67.015625 \n",
       "C 62.09375 68.796875 62 68.90625 61.40625 68.90625 \n",
       "C 61 68.90625 60.90625 68.796875 60.203125 67.609375 \n",
       "L 55.796875 59.25 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 12.90625 20.09375 -2 37.296875 -2 \n",
       "C 52.59375 -2 62.09375 11.40625 62.09375 22.828125 \n",
       "C 62.09375 23.8125 62.09375 24.21875 61.203125 24.21875 \n",
       "C 60.40625 24.21875 60.40625 23.921875 60.296875 23.03125 \n",
       "C 59.5 9.140625 49.5 0.59375 38.703125 0.59375 \n",
       "C 28.5 0.59375 12.203125 7.734375 12.203125 33.546875 \n",
       "C 12.203125 59.453125 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.6875 59.90625 42.890625 \n",
       "C 60.09375 41.6875 60.09375 41.484375 61 41.484375 \n",
       "C 62.09375 41.484375 62.09375 41.6875 62.09375 43.484375 \n",
       "z\n",
       "\" id=\"CMR17-67\"/>\n",
       "      <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.5 43.59375 16.5 37 15.296875 33.703125 \n",
       "L 15.203125 33.703125 \n",
       "L 15.203125 68.6875 \n",
       "L 3.296875 67.59375 \n",
       "L 3.296875 65 \n",
       "C 9.203125 65 10.09375 64.40625 10.09375 59.640625 \n",
       "L 10.09375 6.65625 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-104\"/>\n",
       "      <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "      <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "      <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(182.140743 265.716355)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-76\"/>\n",
       "      <use transform=\"translate(57.381408 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(103.071885 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(148.762362 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "      <use transform=\"translate(178.838328 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(231.01492 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(276.705397 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "      <use transform=\"translate(301.576455 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "      <use transform=\"translate(352.471825 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "      <use transform=\"translate(382.547791 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "      <use transform=\"translate(449.057672 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(499.953042 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(545.643519 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      <use transform=\"translate(596.538889 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(642.229366 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"mab9f1d015f\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mab9f1d015f\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(39.981018 242.401429)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mab9f1d015f\" y=\"200.993542\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- $2000$ -->\n",
       "      <g transform=\"translate(25.037012 204.452792)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mab9f1d015f\" y=\"163.044905\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- $4000$ -->\n",
       "      <g transform=\"translate(25.037012 166.504154)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mab9f1d015f\" y=\"125.096268\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- $6000$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "\" id=\"CMR17-54\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(25.037012 128.555517)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mab9f1d015f\" y=\"87.14763\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- $8000$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 27.203125 35.375 \n",
       "C 33.5 38.546875 39.90625 43.328125 39.90625 50.984375 \n",
       "C 39.90625 60.015625 31.09375 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 58.734375 5.90625 49.6875 \n",
       "C 5.90625 47.203125 6.5 42.921875 10.40625 39.140625 \n",
       "C 11.40625 38.15625 15.59375 35.171875 18.296875 33.28125 \n",
       "C 13.796875 30.984375 3.296875 25.53125 3.296875 14.59375 \n",
       "C 3.296875 4.359375 13.09375 -2 22.796875 -2 \n",
       "C 33.5 -2 42.5 5.65625 42.5 15.78125 \n",
       "C 42.5 24.84375 36.40625 29.015625 32.40625 31.703125 \n",
       "z\n",
       "M 14.09375 44.09375 \n",
       "C 13.296875 44.59375 9.296875 47.671875 9.296875 52.359375 \n",
       "C 9.296875 58.421875 15.59375 63 22.796875 63 \n",
       "C 30.703125 63 36.5 57.4375 36.5 50.96875 \n",
       "C 36.5 41.703125 26.09375 36.421875 25.59375 36.421875 \n",
       "C 25.5 36.421875 25.40625 36.421875 24.59375 37.03125 \n",
       "z\n",
       "M 32.5 23.734375 \n",
       "C 34 22.640625 38.796875 19.375 38.796875 13.296875 \n",
       "C 38.796875 5.953125 31.40625 0.390625 23 0.390625 \n",
       "C 13.90625 0.390625 7 6.84375 7 14.6875 \n",
       "C 7 22.546875 13.09375 29.09375 20 32.1875 \n",
       "z\n",
       "\" id=\"CMR17-56\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(25.037012 90.606879)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mab9f1d015f\" y=\"49.198993\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_12\">\n",
       "      <!-- $10000$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(20.055677 52.658242)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(182.761908 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_13\">\n",
       "     <!-- Frequency -->\n",
       "     <defs>\n",
       "      <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "      <path d=\"M 41.09375 43.78125 \n",
       "L 39.5 43.78125 \n",
       "L 36.5 33.9375 \n",
       "C 34.203125 39.203125 30 43.78125 23.296875 43.78125 \n",
       "C 13.09375 43.78125 3.296875 34.734375 3.296875 21.609375 \n",
       "C 3.296875 8.5625 12.5 -0.59375 22.5 -0.59375 \n",
       "C 30.203125 -0.59375 34.5 5.28125 35.796875 7.46875 \n",
       "L 35.796875 -12.328125 \n",
       "C 35.796875 -16.109375 35.40625 -16.609375 29 -16.609375 \n",
       "L 29 -19.1875 \n",
       "C 31.40625 -19 35.796875 -19 38.40625 -19 \n",
       "C 41 -19 45.5 -19 47.90625 -19.1875 \n",
       "L 47.90625 -16.609375 \n",
       "C 41.5 -16.609375 41.09375 -16.203125 41.09375 -12.328125 \n",
       "z\n",
       "M 36 13.84375 \n",
       "C 36 9.5625 32.40625 5.578125 32.09375 5.28125 \n",
       "C 29 1.796875 25.40625 1 23 1 \n",
       "C 15.5 1 9.703125 10.453125 9.703125 21.484375 \n",
       "C 9.703125 33.140625 16.296875 42 24 42 \n",
       "C 32 42 36 32.9375 36 27.765625 \n",
       "z\n",
       "\" id=\"CMR17-113\"/>\n",
       "      <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "      <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "\" id=\"CMR17-99\"/>\n",
       "      <path d=\"M 39 34.328125 \n",
       "C 41.5 40.40625 46 40.40625 47.40625 40.40625 \n",
       "L 47.40625 43 \n",
       "C 45.5 43 43 43 41.09375 43 \n",
       "C 39 43 35.703125 43 33.703125 43.09375 \n",
       "L 33.703125 40.5 \n",
       "C 37.5 40.203125 37.59375 37.40625 37.59375 36.609375 \n",
       "C 37.59375 35.625 37.40625 35.125 36.90625 33.921875 \n",
       "L 26 7.015625 \n",
       "L 14.09375 36.03125 \n",
       "C 13.59375 37.21875 13.59375 37.90625 13.59375 38.015625 \n",
       "C 13.59375 40.203125 15.90625 40.40625 18.296875 40.40625 \n",
       "L 18.296875 43.09375 \n",
       "C 15.90625 43 11.59375 43 9.09375 43 \n",
       "C 6.40625 43 3.203125 43 1 43.09375 \n",
       "L 1 40.40625 \n",
       "C 6.40625 40.40625 7 39.90625 8.296875 36.71875 \n",
       "L 23.203125 0.234375 \n",
       "C 18.703125 -11.53125 16.09375 -18.40625 9.796875 -18.40625 \n",
       "C 8.703125 -18.40625 6.203125 -18.109375 4.40625 -16.3125 \n",
       "C 6.703125 -16.109375 7.703125 -14.71875 7.703125 -13.015625 \n",
       "C 7.703125 -11.328125 6.5 -9.828125 4.5 -9.828125 \n",
       "C 2.296875 -9.828125 1.203125 -11.328125 1.203125 -13.125 \n",
       "C 1.203125 -17.109375 5.296875 -20 9.796875 -20 \n",
       "C 15.59375 -20 19.203125 -14.40625 21.203125 -9.4375 \n",
       "z\n",
       "\" id=\"CMR17-121\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(14.118498 152.444442)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(52.176591 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(87.457359 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(127.943019 0)scale(0.996264)\" xlink:href=\"#CMR17-113\"/>\n",
       "      <use transform=\"translate(176.235935 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(227.131305 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(267.616966 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      <use transform=\"translate(318.512335 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(358.997996 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 51.962353 238.94218 \n",
       "L 51.962353 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 386.762353 238.94218 \n",
       "L 386.762353 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 51.962353 238.94218 \n",
       "L 386.762353 238.94218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 51.962353 21.50218 \n",
       "L 386.762353 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_14\">\n",
       "    <!-- Distribution of Log-Fold-Changes -->\n",
       "    <defs>\n",
       "     <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 37 0 \n",
       "C 53.09375 0 66 14.734375 66 33.453125 \n",
       "C 66 52.28125 53.296875 68 37 68 \n",
       "z\n",
       "M 23.796875 2.59375 \n",
       "C 19.796875 2.59375 19.59375 3.078125 19.59375 6.46875 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 35 65.40625 \n",
       "C 43.296875 65.40625 58.59375 60.53125 58.59375 33.453125 \n",
       "C 58.59375 19.40625 54.5 13.734375 52.59375 11.046875 \n",
       "C 50.203125 7.96875 44.5 2.59375 35 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-68\"/>\n",
       "     <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "     <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "     <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "     <path d=\"M 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 -0.203125 \n",
       "L 11.703125 -0.203125 \n",
       "L 14.59375 6.890625 \n",
       "C 17.5 2.203125 21.796875 -1 27.59375 -1 \n",
       "C 37.59375 -1 47.703125 7.8125 47.703125 21.40625 \n",
       "C 47.703125 34.1875 38.703125 43.59375 28.59375 43.59375 \n",
       "C 22.40625 43.59375 18 40.296875 15 36.40625 \n",
       "z\n",
       "M 15.203125 31.3125 \n",
       "C 15.203125 33.09375 15.203125 33.40625 16.59375 35.5 \n",
       "C 20.5 41.515625 26.09375 42 28 42 \n",
       "C 31 42 41.296875 40.390625 41.296875 21.5 \n",
       "C 41.296875 1.6875 29.5 0.59375 27.09375 0.59375 \n",
       "C 24 0.59375 19.5 1.78125 16.296875 7.59375 \n",
       "C 15.203125 9.5 15.203125 9.703125 15.203125 11.484375 \n",
       "z\n",
       "\" id=\"CMR17-98\"/>\n",
       "     <path d=\"M 15.296875 40 \n",
       "L 26.703125 40 \n",
       "L 26.703125 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 54.59375 \n",
       "C 15.09375 62.84375 19.296875 68 23.59375 68 \n",
       "C 24.90625 68 26.40625 67.609375 27.40625 66.90625 \n",
       "C 26.59375 66.703125 24.5 66.015625 24.5 63.546875 \n",
       "C 24.5 60.953125 26.5 60.15625 27.90625 60.15625 \n",
       "C 29.296875 60.15625 31.296875 60.953125 31.296875 63.546875 \n",
       "C 31.296875 67.3125 27.703125 69.59375 23.703125 69.59375 \n",
       "C 18 69.59375 10.203125 64.9375 10.203125 54.390625 \n",
       "L 10.203125 42.578125 \n",
       "L 2.203125 42.578125 \n",
       "L 2.203125 40 \n",
       "L 10.203125 40 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.40625 0 13 0 \n",
       "C 15.90625 0 21 0 23.703125 -0.203125 \n",
       "L 23.703125 2.390625 \n",
       "C 16.5 2.390625 15.296875 2.390625 15.296875 6.859375 \n",
       "z\n",
       "\" id=\"CMR17-102\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(133.537855 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "     <use transform=\"translate(70.393556 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(95.264614 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(131.065952 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(166.34672 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(201.627488 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(226.498546 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "     <use transform=\"translate(277.393916 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "     <use transform=\"translate(328.289286 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(363.570053 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(388.441112 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(434.131589 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(515.102925 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(560.793402 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(618.342881 0)scale(0.996264)\" xlink:href=\"#CMR17-76\"/>\n",
       "     <use transform=\"translate(675.724289 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(721.414766 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(767.105243 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(797.18121 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "     <use transform=\"translate(849.357801 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(895.048278 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "     <use transform=\"translate(919.919336 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "     <use transform=\"translate(970.814706 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(1000.890672 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "     <use transform=\"translate(1067.400553 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "     <use transform=\"translate(1118.295923 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(1163.9864 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(1214.88177 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(1260.572247 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(1301.057907 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"p135a16a4ee\">\n",
       "   <rect height=\"217.44\" width=\"334.8\" x=\"51.962353\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x119295160>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.title('Distribution of Log-Fold-Changes')\n",
    "plt.xlabel('Log-Fold-Change')\n",
    "plt.ylabel('Frequency')\n",
    "plt.hist(list(data.values()))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "## Overlap Summary"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "The number of shared elements in the knowledge assembly and differential gene data set are counted."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"249pt\" version=\"1.1\" viewBox=\"0 0 308 249\" width=\"308pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 249.64218 \n",
       "L 308.81016 249.64218 \n",
       "L 308.81016 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path clip-path=\"url(#pbb38949aee)\" d=\"M 104.403102 166.110542 \n",
       "C 99.607351 154.753748 97.1363 142.550034 97.1363 130.22218 \n",
       "C 97.1363 117.894326 99.607351 105.690611 104.403102 94.333817 \n",
       "C 94.985587 103.899769 89.70177 116.798432 89.70177 130.22218 \n",
       "C 89.70177 143.645928 94.985587 156.544591 104.403102 166.110542 \n",
       "\" style=\"fill:#ff0000;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#pbb38949aee)\" d=\"M 104.403102 94.333817 \n",
       "C 107.944544 90.736556 111.999025 87.683226 116.434562 85.273222 \n",
       "C 120.870099 82.863218 125.638034 81.122976 130.583158 80.109147 \n",
       "C 135.528283 79.095318 140.59635 78.819023 145.622381 79.289255 \n",
       "C 150.648412 79.759488 155.577272 80.97109 160.248514 82.88462 \n",
       "C 164.919757 84.798151 169.282139 87.392619 173.193653 90.583569 \n",
       "C 177.105168 93.774518 180.522907 97.526944 183.335614 101.718697 \n",
       "C 186.148321 105.910449 188.325141 110.495544 189.795213 115.324725 \n",
       "C 191.265285 120.153907 192.012482 125.174199 192.012482 130.22218 \n",
       "C 192.012482 135.27016 191.265285 140.290453 189.795213 145.119634 \n",
       "C 188.325141 149.948816 186.148321 154.533911 183.335614 158.725663 \n",
       "C 180.522907 162.917415 177.105168 166.669841 173.193653 169.860791 \n",
       "C 169.282139 173.05174 164.919757 175.646209 160.248514 177.559739 \n",
       "C 155.577272 179.47327 150.648412 180.684872 145.622381 181.155105 \n",
       "C 140.59635 181.625337 135.528283 181.349042 130.583158 180.335213 \n",
       "C 125.638034 179.321383 120.870099 177.581142 116.434562 175.171138 \n",
       "C 111.999025 172.761134 107.944544 169.707804 104.403102 166.110542 \n",
       "C 108.513398 175.844115 114.265491 184.799608 121.407978 192.585612 \n",
       "C 128.550466 200.371615 136.977753 206.87302 146.321516 211.805674 \n",
       "C 155.665278 216.738329 165.787377 220.029308 176.245415 221.534777 \n",
       "C 186.703452 223.040245 197.342815 222.737946 207.698498 220.641091 \n",
       "C 218.054182 218.544236 227.973086 214.683825 237.021694 209.228583 \n",
       "C 246.070302 203.77334 254.114839 196.803917 260.803708 188.624919 \n",
       "C 267.492577 180.445921 272.72689 171.178268 276.277876 161.227012 \n",
       "C 279.828861 151.275757 281.64402 140.78802 281.64402 130.22218 \n",
       "C 281.64402 119.65634 279.828861 109.168603 276.277876 99.217347 \n",
       "C 272.72689 89.266092 267.492577 79.998439 260.803708 71.819441 \n",
       "C 254.114839 63.640443 246.070302 56.671019 237.021694 51.215777 \n",
       "C 227.973086 45.760534 218.054182 41.900124 207.698498 39.803269 \n",
       "C 197.342815 37.706414 186.703452 37.404114 176.245415 38.909583 \n",
       "C 165.787377 40.415051 155.665278 43.706031 146.321516 48.638685 \n",
       "C 136.977753 53.57134 128.550466 60.072744 121.407978 67.858748 \n",
       "C 114.265491 75.644751 108.513398 84.600244 104.403102 94.333817 \n",
       "\" style=\"fill:#008000;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#pbb38949aee)\" d=\"M 104.403102 166.110542 \n",
       "C 107.944544 169.707804 111.999025 172.761134 116.434562 175.171138 \n",
       "C 120.870099 177.581142 125.638034 179.321383 130.583158 180.335213 \n",
       "C 135.528283 181.349042 140.59635 181.625337 145.622381 181.155105 \n",
       "C 150.648412 180.684872 155.577272 179.47327 160.248514 177.559739 \n",
       "C 164.919757 175.646209 169.282139 173.05174 173.193653 169.860791 \n",
       "C 177.105168 166.669841 180.522907 162.917415 183.335614 158.725663 \n",
       "C 186.148321 154.533911 188.325141 149.948816 189.795213 145.119634 \n",
       "C 191.265285 140.290453 192.012482 135.27016 192.012482 130.22218 \n",
       "C 192.012482 125.174199 191.265285 120.153907 189.795213 115.324725 \n",
       "C 188.325141 110.495544 186.148321 105.910449 183.335614 101.718697 \n",
       "C 180.522907 97.526944 177.105168 93.774518 173.193653 90.583569 \n",
       "C 169.282139 87.392619 164.919757 84.798151 160.248514 82.88462 \n",
       "C 155.577272 80.97109 150.648412 79.759488 145.622381 79.289255 \n",
       "C 140.59635 78.819023 135.528283 79.095318 130.583158 80.109147 \n",
       "C 125.638034 81.122976 120.870099 82.863218 116.434562 85.273222 \n",
       "C 111.999025 87.683226 107.944544 90.736556 104.403102 94.333817 \n",
       "C 99.607351 105.690611 97.1363 117.894326 97.1363 130.22218 \n",
       "C 97.1363 142.550034 99.607351 154.753748 104.403102 166.110542 \n",
       "\" style=\"fill:#b25900;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_1\">\n",
       "    <!-- 312 -->\n",
       "    <defs>\n",
       "     <path d=\"M 22.09375 33.703125 \n",
       "C 31 33.703125 34.90625 25.96875 34.90625 17.03125 \n",
       "C 34.90625 5 28.5 0.390625 22.703125 0.390625 \n",
       "C 17.40625 0.390625 8.796875 3.03125 6.09375 10.828125 \n",
       "C 6.59375 10.625 7.09375 10.625 7.59375 10.625 \n",
       "C 10 10.625 11.796875 12.21875 11.796875 14.8125 \n",
       "C 11.796875 17.703125 9.59375 19 7.59375 19 \n",
       "C 5.90625 19 3.296875 18.203125 3.296875 14.46875 \n",
       "C 3.296875 5.234375 12.296875 -2 22.90625 -2 \n",
       "C 34 -2 42.5 6.71875 42.5 16.953125 \n",
       "C 42.5 26.671875 34.5 33.703125 25 34.796875 \n",
       "C 32.59375 36.375 39.90625 43.0625 39.90625 52.015625 \n",
       "C 39.90625 59.6875 32 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 59.828125 5.90625 52.015625 \n",
       "C 5.90625 48.59375 8.5 48 9.796875 48 \n",
       "C 11.90625 48 13.703125 49.296875 13.703125 51.890625 \n",
       "C 13.703125 54.46875 11.90625 55.765625 9.796875 55.765625 \n",
       "C 9.40625 55.765625 8.90625 55.765625 8.5 55.578125 \n",
       "C 11.40625 61.859375 19.296875 63 22.796875 63 \n",
       "C 26.296875 63 32.90625 61.328125 32.90625 51.875 \n",
       "C 32.90625 49.109375 32.5 44.1875 29.09375 39.84375 \n",
       "C 26.09375 36 22.703125 36 19.40625 35.6875 \n",
       "C 18.90625 35.6875 16.59375 35.453125 16.203125 35.453125 \n",
       "C 15.5 35.359375 15.09375 35.265625 15.09375 34.5 \n",
       "C 15.09375 33.796875 15.203125 33.703125 17.203125 33.703125 \n",
       "z\n",
       "\" id=\"CMR17-51\"/>\n",
       "     <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "     <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(85.947032 132.71284)scale(0.1 -0.1)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-51\"/>\n",
       "     <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "     <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_2\">\n",
       "    <!-- 15677 -->\n",
       "    <defs>\n",
       "     <path d=\"M 11.40625 57.6875 \n",
       "C 12.40625 57.28125 16.5 56 20.703125 56 \n",
       "C 30 56 35.09375 61.109375 38 64.0625 \n",
       "C 38 64.953125 38 65.5 37.40625 65.5 \n",
       "C 37.296875 65.5 37.09375 65.5 36.296875 65.046875 \n",
       "C 32.796875 63.40625 28.703125 62.078125 23.703125 62.078125 \n",
       "C 20.703125 62.078125 16.203125 62.484375 11.296875 64.671875 \n",
       "C 10.203125 65.171875 10 65.171875 9.90625 65.171875 \n",
       "C 9.40625 65.171875 9.296875 65.0625 9.296875 63.078125 \n",
       "L 9.296875 34.421875 \n",
       "C 9.296875 32.640625 9.296875 32.140625 10.296875 32.140625 \n",
       "C 10.796875 32.140625 11 32.34375 11.5 33.03125 \n",
       "C 14.703125 37.515625 19.09375 39.40625 24.09375 39.40625 \n",
       "C 27.59375 39.40625 35.09375 37.21875 35.09375 20.140625 \n",
       "C 35.09375 16.953125 35.09375 11.171875 32.09375 6.578125 \n",
       "C 29.59375 2.484375 25.703125 0.390625 21.40625 0.390625 \n",
       "C 14.796875 0.390625 8.09375 5 6.296875 12.71875 \n",
       "C 6.703125 12.609375 7.5 12.421875 7.90625 12.421875 \n",
       "C 9.203125 12.421875 11.703125 13.125 11.703125 16.21875 \n",
       "C 11.703125 18.90625 9.796875 20 7.90625 20 \n",
       "C 5.59375 20 4.09375 18.59375 4.09375 15.796875 \n",
       "C 4.09375 7.09375 11 -2 21.59375 -2 \n",
       "C 31.90625 -2 41.703125 6.875 41.703125 19.75 \n",
       "C 41.703125 31.71875 33.90625 41 24.203125 41 \n",
       "C 19.09375 41 14.796875 39.109375 11.40625 35.53125 \n",
       "z\n",
       "\" id=\"CMR17-53\"/>\n",
       "     <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "\" id=\"CMR17-54\"/>\n",
       "     <path d=\"M 45.09375 61.578125 \n",
       "L 45.09375 63.765625 \n",
       "L 21.59375 63.765625 \n",
       "C 9.90625 63.765625 9.703125 65.078125 9.296875 67 \n",
       "L 7.5 67 \n",
       "L 4.59375 48.34375 \n",
       "L 6.40625 48.34375 \n",
       "C 6.703125 50.234375 7.40625 55.3125 8.59375 57.203125 \n",
       "C 9.203125 58 16.59375 58 18.296875 58 \n",
       "L 40.296875 58 \n",
       "L 29.203125 41.578125 \n",
       "C 21.796875 30.53125 16.703125 15.609375 16.703125 2.578125 \n",
       "C 16.703125 1.390625 16.703125 -2 20.296875 -2 \n",
       "C 23.90625 -2 23.90625 1.390625 23.90625 2.671875 \n",
       "L 23.90625 7.265625 \n",
       "C 23.90625 23.5625 26.703125 34.3125 31.296875 41.1875 \n",
       "z\n",
       "\" id=\"CMR17-55\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(224.374913 132.71284)scale(0.1 -0.1)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "     <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "     <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "     <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "     <use transform=\"translate(182.761908 0)scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_3\">\n",
       "    <!-- 6510 -->\n",
       "    <defs>\n",
       "     <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "\" id=\"CMR17-48\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(134.611721 132.71284)scale(0.1 -0.1)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "     <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "     <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "     <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_4\">\n",
       "    <!-- Alzheimer's Disease KAM -->\n",
       "    <defs>\n",
       "     <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "     <path d=\"M 15.296875 68.6875 \n",
       "L 3.40625 67.59375 \n",
       "L 3.40625 65 \n",
       "C 9.296875 65 10.203125 64.40625 10.203125 59.640625 \n",
       "L 10.203125 6.65625 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.203125 0 12.703125 0 \n",
       "C 15.296875 0 19.703125 0 22.09375 -0.203125 \n",
       "L 22.09375 2.390625 \n",
       "C 15.703125 2.390625 15.296875 2.78125 15.296875 6.65625 \n",
       "z\n",
       "\" id=\"CMR17-108\"/>\n",
       "     <path d=\"M 36.09375 40.703125 \n",
       "C 36.90625 41.71875 36.90625 41.90625 36.90625 42.109375 \n",
       "C 36.90625 43 36.5 43 34.703125 43 \n",
       "L 4.296875 43 \n",
       "L 3.296875 27.5 \n",
       "L 5.09375 27.5 \n",
       "C 5.796875 37.203125 6.90625 41.40625 18.40625 41.40625 \n",
       "L 30.59375 41.40625 \n",
       "L 2 1.5 \n",
       "C 2 0 2.09375 0 4.203125 0 \n",
       "L 35.796875 0 \n",
       "L 37.203125 18 \n",
       "L 35.40625 18 \n",
       "C 34.59375 6.90625 33.296875 1.796875 21.09375 1.796875 \n",
       "L 8.296875 1.796875 \n",
       "z\n",
       "\" id=\"CMR17-122\"/>\n",
       "     <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.5 43.59375 16.5 37 15.296875 33.703125 \n",
       "L 15.203125 33.703125 \n",
       "L 15.203125 68.6875 \n",
       "L 3.296875 67.59375 \n",
       "L 3.296875 65 \n",
       "C 9.203125 65 10.09375 64.40625 10.09375 59.640625 \n",
       "L 10.09375 6.65625 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-104\"/>\n",
       "     <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "     <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "     <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "     <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "     <path d=\"M 16 62.890625 \n",
       "C 16.203125 61.796875 16.296875 61.390625 16.296875 59.6875 \n",
       "C 16.296875 53.359375 13.796875 47.53125 9.90625 43.40625 \n",
       "C 9.5 43 9.296875 42.703125 9.296875 42.296875 \n",
       "C 9.296875 41.890625 9.703125 41.5 10.09375 41.5 \n",
       "C 10.796875 41.5 17.90625 48.515625 17.90625 59.546875 \n",
       "C 17.90625 61.75 17.703125 69.15625 12.5 69.15625 \n",
       "C 10.296875 69.15625 8.40625 67.671875 8.40625 65.078125 \n",
       "C 8.40625 62.484375 10.296875 61 12.5 61 \n",
       "C 13.40625 61 15.09375 61.296875 16 62.890625 \n",
       "\" id=\"CMR17-39\"/>\n",
       "     <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "     <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 37 0 \n",
       "C 53.09375 0 66 14.734375 66 33.453125 \n",
       "C 66 52.28125 53.296875 68 37 68 \n",
       "z\n",
       "M 23.796875 2.59375 \n",
       "C 19.796875 2.59375 19.59375 3.078125 19.59375 6.46875 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 35 65.40625 \n",
       "C 43.296875 65.40625 58.59375 60.53125 58.59375 33.453125 \n",
       "C 58.59375 19.40625 54.5 13.734375 52.59375 11.046875 \n",
       "C 50.203125 7.96875 44.5 2.59375 35 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-68\"/>\n",
       "     <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "     <path d=\"M 36.90625 41.796875 \n",
       "L 54.203125 59.09375 \n",
       "C 57.796875 62.59375 61.59375 65.40625 67.203125 65.40625 \n",
       "L 67.203125 68 \n",
       "C 65.203125 68 62.703125 68 60.703125 68 \n",
       "C 58 68 53.296875 68 50.703125 68.09375 \n",
       "L 50.703125 65.5 \n",
       "C 53.296875 65.3125 53.703125 63.515625 53.703125 62.796875 \n",
       "C 53.703125 62.109375 53.296875 61.015625 52.90625 60.421875 \n",
       "L 19.703125 27.15625 \n",
       "L 19.703125 60.9375 \n",
       "C 19.703125 65.40625 20.796875 65.40625 27.703125 65.40625 \n",
       "L 27.703125 68 \n",
       "C 24.796875 68 19.5 68 16.40625 68 \n",
       "C 13.296875 68 8 68 5.09375 68 \n",
       "L 5.09375 65.40625 \n",
       "C 12 65.40625 13.09375 65.40625 13.09375 60.921875 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.296875 0 16.40625 0 \n",
       "C 19.5 0 24.796875 0 27.703125 0 \n",
       "L 27.703125 2.5 \n",
       "C 20.796875 2.5 19.703125 2.5 19.703125 6.984375 \n",
       "L 19.703125 24.671875 \n",
       "L 32.5 37.34375 \n",
       "L 51.40625 8.390625 \n",
       "C 52 7.390625 52.703125 6.09375 52.703125 5.09375 \n",
       "C 52.703125 2.5 49.90625 2.5 48.59375 2.5 \n",
       "L 48.59375 0 \n",
       "C 51.796875 0 56.40625 0 59.703125 0 \n",
       "C 62.09375 0 66.203125 0 68.5 -0.09375 \n",
       "L 68.5 2.5 \n",
       "C 63.703125 2.5 62.203125 3.09375 59.203125 7.703125 \n",
       "z\n",
       "\" id=\"CMR17-75\"/>\n",
       "     <path d=\"M 20.90625 66 \n",
       "C 20.296875 67.5 20.203125 67.5 18.09375 67.5 \n",
       "L 5.296875 67.5 \n",
       "L 5.296875 65 \n",
       "C 12.203125 65 13.296875 65 13.296875 60.546875 \n",
       "L 13.296875 9.53125 \n",
       "C 13.296875 7.046875 13.296875 2.390625 5.296875 2.390625 \n",
       "L 5.296875 -0.203125 \n",
       "C 7.59375 0 11.796875 0 14.296875 0 \n",
       "C 16.796875 0 21 0 23.296875 -0.203125 \n",
       "L 23.296875 2.390625 \n",
       "C 15.296875 2.390625 15.296875 7.046875 15.296875 9.53125 \n",
       "L 15.296875 64.609375 \n",
       "L 15.40625 64.609375 \n",
       "L 38.90625 1.390625 \n",
       "C 39.296875 0.390625 39.5 -0.203125 40.296875 -0.203125 \n",
       "C 41.09375 -0.203125 41.296875 0.390625 41.703125 1.390625 \n",
       "L 65.40625 65.09375 \n",
       "L 65.5 65.09375 \n",
       "L 65.5 6.84375 \n",
       "C 65.5 2.390625 64.40625 2.390625 57.5 2.390625 \n",
       "L 57.5 -0.203125 \n",
       "C 60.203125 0 65.59375 0 68.5 0 \n",
       "C 71.40625 0 76.90625 0 79.59375 -0.203125 \n",
       "L 79.59375 2.390625 \n",
       "C 72.703125 2.390625 71.59375 2.390625 71.59375 6.84375 \n",
       "L 71.59375 60.546875 \n",
       "C 71.59375 65 72.703125 65 79.59375 65 \n",
       "L 79.59375 67.5 \n",
       "L 66.90625 67.5 \n",
       "C 64.796875 67.5 64.703125 67.40625 64.09375 65.90625 \n",
       "L 42.5 7.84375 \n",
       "z\n",
       "\" id=\"CMR17-77\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(7.2 196.850177)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "     <use transform=\"translate(93.943488 0)scale(0.996264)\" xlink:href=\"#CMR17-122\"/>\n",
       "     <use transform=\"translate(134.429149 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "     <use transform=\"translate(185.324519 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(225.810179 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(250.681237 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(327.600827 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(368.086488 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(403.367256 0)scale(0.996264)\" xlink:href=\"#CMR17-39\"/>\n",
       "     <use transform=\"translate(428.238314 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(494.115618 0)scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "     <use transform=\"translate(564.509174 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(589.380232 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(625.18157 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(665.667231 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(711.357708 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(747.159045 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(817.720672 0)scale(0.996264)\" xlink:href=\"#CMR17-75\"/>\n",
       "     <use transform=\"translate(889.395449 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "     <use transform=\"translate(958.467879 0)scale(0.996264)\" xlink:href=\"#CMR17-77\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_5\">\n",
       "    <!-- E-GEOD-5281 -->\n",
       "    <defs>\n",
       "     <path d=\"M 60.703125 25.234375 \n",
       "L 58.90625 25.234375 \n",
       "C 56.703125 10.5625 55.40625 2.59375 37.796875 2.59375 \n",
       "L 23.90625 2.59375 \n",
       "C 19.90625 2.59375 19.703125 3.078125 19.703125 6.46875 \n",
       "L 19.703125 34 \n",
       "L 29.09375 34 \n",
       "C 38.5 34 39.40625 30.625 39.40625 22.34375 \n",
       "L 41.203125 22.34375 \n",
       "L 41.203125 48.140625 \n",
       "L 39.40625 48.140625 \n",
       "C 39.40625 39.953125 38.5 36.59375 29.09375 36.59375 \n",
       "L 19.703125 36.59375 \n",
       "L 19.703125 61.140625 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 37.59375 65 \n",
       "C 53.09375 65 54.90625 58.890625 56.296875 45.765625 \n",
       "L 58.09375 45.765625 \n",
       "L 55.703125 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 7.046875 \n",
       "C 13.09375 2.59375 12 2.59375 5.09375 2.59375 \n",
       "L 5.09375 0 \n",
       "L 57.09375 0 \n",
       "z\n",
       "\" id=\"CMR17-69\"/>\n",
       "     <path d=\"M 25.5 19 \n",
       "L 25.5 23.375 \n",
       "L 0.40625 23.375 \n",
       "L 0.40625 19 \n",
       "z\n",
       "\" id=\"CMR17-45\"/>\n",
       "     <path d=\"M 62.09375 19.265625 \n",
       "C 62.09375 23.015625 62.40625 23.515625 68.59375 23.515625 \n",
       "L 68.59375 26.09375 \n",
       "C 66.203125 26 60.796875 26 58.09375 26 \n",
       "C 55.296875 26 47.5 26 43.703125 26 \n",
       "L 43.703125 23.515625 \n",
       "L 46.703125 23.515625 \n",
       "C 55.296875 23.515625 55.5 22.40625 55.5 18.765625 \n",
       "L 55.5 12.09375 \n",
       "C 55.5 0.984375 42.5 0.984375 39.40625 0.984375 \n",
       "C 29.09375 0.984375 12.203125 7.984375 12.203125 33.75 \n",
       "C 12.203125 59.5 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.65625 59.90625 42.828125 \n",
       "C 60.09375 41.625 60.09375 41.421875 61 41.421875 \n",
       "C 62.09375 41.421875 62.09375 41.625 62.09375 43.421875 \n",
       "L 62.09375 67 \n",
       "C 62.09375 68.78125 62 68.890625 61.40625 68.890625 \n",
       "C 61 68.890625 60.90625 68.78125 60.203125 67.59375 \n",
       "L 55.796875 59.234375 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 13.203125 19.796875 -2 37.40625 -2 \n",
       "C 45.5 -2 53.40625 0.875 56.5 6.515625 \n",
       "C 58.203125 2.46875 61 -0.3125 61.5 -0.3125 \n",
       "C 62 -0.3125 62.09375 -0.203125 62.09375 1.5625 \n",
       "z\n",
       "\" id=\"CMR17-71\"/>\n",
       "     <path d=\"M 67.296875 33.359375 \n",
       "C 67.296875 53.703125 52.59375 69 36 69 \n",
       "C 19 69 4.59375 53.5 4.59375 33.359375 \n",
       "C 4.59375 13.203125 19.203125 -2 35.90625 -2 \n",
       "C 53 -2 67.296875 13.5 67.296875 33.359375 \n",
       "M 36 -0.203125 \n",
       "C 24.59375 -0.203125 12.203125 11.109375 12.203125 34.640625 \n",
       "C 12.203125 58.0625 25.796875 67.203125 35.90625 67.203125 \n",
       "C 46.5 67.203125 59.703125 57.78125 59.703125 34.640625 \n",
       "C 59.703125 10.71875 47 -0.203125 36 -0.203125 \n",
       "\" id=\"CMR17-79\"/>\n",
       "     <path d=\"M 27.203125 35.375 \n",
       "C 33.5 38.546875 39.90625 43.328125 39.90625 50.984375 \n",
       "C 39.90625 60.015625 31.09375 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 58.734375 5.90625 49.6875 \n",
       "C 5.90625 47.203125 6.5 42.921875 10.40625 39.140625 \n",
       "C 11.40625 38.15625 15.59375 35.171875 18.296875 33.28125 \n",
       "C 13.796875 30.984375 3.296875 25.53125 3.296875 14.59375 \n",
       "C 3.296875 4.359375 13.09375 -2 22.796875 -2 \n",
       "C 33.5 -2 42.5 5.65625 42.5 15.78125 \n",
       "C 42.5 24.84375 36.40625 29.015625 32.40625 31.703125 \n",
       "z\n",
       "M 14.09375 44.09375 \n",
       "C 13.296875 44.59375 9.296875 47.671875 9.296875 52.359375 \n",
       "C 9.296875 58.421875 15.59375 63 22.796875 63 \n",
       "C 30.703125 63 36.5 57.4375 36.5 50.96875 \n",
       "C 36.5 41.703125 26.09375 36.421875 25.59375 36.421875 \n",
       "C 25.5 36.421875 25.40625 36.421875 24.59375 37.03125 \n",
       "z\n",
       "M 32.5 23.734375 \n",
       "C 34 22.640625 38.796875 19.375 38.796875 13.296875 \n",
       "C 38.796875 5.953125 31.40625 0.390625 23 0.390625 \n",
       "C 13.90625 0.390625 7 6.84375 7 14.6875 \n",
       "C 7 22.546875 13.09375 29.09375 20 32.1875 \n",
       "z\n",
       "\" id=\"CMR17-56\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(189.39016 237.948681)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-69\"/>\n",
       "     <use transform=\"translate(62.586301 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(92.662267 0)scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "     <use transform=\"translate(165.017613 0)scale(0.996264)\" xlink:href=\"#CMR17-69\"/>\n",
       "     <use transform=\"translate(227.603913 0)scale(0.996264)\" xlink:href=\"#CMR17-79\"/>\n",
       "     <use transform=\"translate(299.318702 0)scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "     <use transform=\"translate(369.712258 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(399.788224 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "     <use transform=\"translate(445.478701 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "     <use transform=\"translate(491.169178 0)scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "     <use transform=\"translate(536.859655 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_6\">\n",
       "    <!-- HGNC Gene Names -->\n",
       "    <defs>\n",
       "     <path d=\"M 56.09375 61.015625 \n",
       "C 56.09375 65.5 57.203125 65.5 64.09375 65.5 \n",
       "L 64.09375 68 \n",
       "C 61.203125 68 55.90625 68 52.796875 68 \n",
       "C 49.703125 68 44.40625 68 41.5 68 \n",
       "L 41.5 65.5 \n",
       "C 48.40625 65.5 49.5 65.5 49.5 61 \n",
       "L 49.5 36.578125 \n",
       "L 19.703125 36.578125 \n",
       "L 19.703125 61 \n",
       "C 19.703125 65.5 20.796875 65.5 27.703125 65.5 \n",
       "L 27.703125 68 \n",
       "C 24.796875 68 19.5 68 16.40625 68 \n",
       "C 13.296875 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 6.984375 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.296875 0 16.40625 0 \n",
       "C 19.5 0 24.796875 0 27.703125 0 \n",
       "L 27.703125 2.5 \n",
       "C 20.796875 2.5 19.703125 2.5 19.703125 6.96875 \n",
       "L 19.703125 34 \n",
       "L 49.5 34 \n",
       "L 49.5 6.96875 \n",
       "C 49.5 2.5 48.40625 2.5 41.5 2.5 \n",
       "L 41.5 0 \n",
       "C 44.40625 0 49.703125 0 52.796875 0 \n",
       "C 55.90625 0 61.203125 0 64.09375 0 \n",
       "L 64.09375 2.5 \n",
       "C 57.203125 2.5 56.09375 2.5 56.09375 6.984375 \n",
       "z\n",
       "\" id=\"CMR17-72\"/>\n",
       "     <path d=\"M 20.5 66.40625 \n",
       "C 19.796875 67.5 19.703125 67.5 17.796875 67.5 \n",
       "L 5.09375 67.5 \n",
       "L 5.09375 65 \n",
       "L 7 65 \n",
       "C 11 65 12.796875 64.515625 13.09375 64.40625 \n",
       "L 13.09375 9.53125 \n",
       "C 13.09375 7.046875 13.09375 2.390625 5.09375 2.390625 \n",
       "L 5.09375 -0.203125 \n",
       "C 7.40625 0 11.59375 0 14.09375 0 \n",
       "C 16.59375 0 20.796875 0 23.09375 -0.203125 \n",
       "L 23.09375 2.390625 \n",
       "C 15.09375 2.390625 15.09375 7.046875 15.09375 9.53125 \n",
       "L 15.09375 63.015625 \n",
       "C 15.796875 62.421875 15.796875 62.21875 16.5 61.234375 \n",
       "L 53.59375 1.1875 \n",
       "C 54.296875 0 54.59375 0 55.09375 0 \n",
       "C 56.09375 0 56.09375 0.296875 56.09375 2.171875 \n",
       "L 56.09375 57.890625 \n",
       "C 56.09375 60.359375 56.09375 65 64.09375 65 \n",
       "L 64.09375 67.59375 \n",
       "C 61.796875 67.5 57.59375 67.5 55.09375 67.5 \n",
       "C 52.59375 67.5 48.40625 67.5 46.09375 67.59375 \n",
       "L 46.09375 65 \n",
       "C 54.09375 65 54.09375 60.34375 54.09375 57.859375 \n",
       "L 54.09375 11.921875 \n",
       "z\n",
       "\" id=\"CMR17-78\"/>\n",
       "     <path d=\"M 62.09375 67.015625 \n",
       "C 62.09375 68.796875 62 68.90625 61.40625 68.90625 \n",
       "C 61 68.90625 60.90625 68.796875 60.203125 67.609375 \n",
       "L 55.796875 59.25 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 12.90625 20.09375 -2 37.296875 -2 \n",
       "C 52.59375 -2 62.09375 11.40625 62.09375 22.828125 \n",
       "C 62.09375 23.8125 62.09375 24.21875 61.203125 24.21875 \n",
       "C 60.40625 24.21875 60.40625 23.921875 60.296875 23.03125 \n",
       "C 59.5 9.140625 49.5 0.59375 38.703125 0.59375 \n",
       "C 28.5 0.59375 12.203125 7.734375 12.203125 33.546875 \n",
       "C 12.203125 59.453125 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.6875 59.90625 42.890625 \n",
       "C 60.09375 41.6875 60.09375 41.484375 61 41.484375 \n",
       "C 62.09375 41.484375 62.09375 41.6875 62.09375 43.484375 \n",
       "z\n",
       "\" id=\"CMR17-67\"/>\n",
       "     <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(134.027508 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-72\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "     <use transform=\"translate(141.427776 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(210.500206 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "     <use transform=\"translate(307.086052 0)scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "     <use transform=\"translate(379.441398 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(419.927059 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(470.822429 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(541.384055 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(610.456485 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(656.146962 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(733.066552 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(773.552213 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"pbb38949aee\">\n",
       "   <rect height=\"217.44\" width=\"224.87453\" x=\"73.23563\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x1192dac50>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "hgnc_names = pbt.summary.get_names(graph, 'HGNC')\n",
    "df_names = set(df['Gene.symbol'])\n",
    "overlapping_hgnc_names = hgnc_names & df_names\n",
    "\n",
    "plt.title('HGNC Gene Names')\n",
    "venn2([hgnc_names, df_names], set_labels=[\"Alzheimer's Disease KAM\", 'E-GEOD-5281'])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "# Analysis"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "## Data Integration\n",
    "\n",
    "Finally, the differential gene expression data are ovelayed on the BEL graph with [pbt.integration.overlay_type_data](http://pybel-tools.readthedocs.io/en/latest/integration.html#pybel_tools.integration.overlay_type_data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "label = 'weight'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "pbt.integration.overlay_type_data(graph, data, label, GENE, 'HGNC', overwrite=False, impute=0)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "## Subgraph Generation\n",
    "\n",
    "A subgraph is induced around an example biological process, Inflammatory Response, by selecting all of the source nodes of upstream causal edges, where this process is the target node. These networks are then enriched by expandind around all of those nodes' upstream edges as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "inflamation_node = BIOPROCESS, 'GOBP', 'inflammatory response'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "### Iteration 1/2\n",
    "\n",
    "All of the nodes that have upstream causal relations to Inflammatory Response are added."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Name: \n",
      "Number of nodes: 32\n",
      "Number of edges: 53\n",
      "Network density: 0.05342741935483871\n",
      "Number weakly connected components: 1\n",
      "Average in-degree: 1.65625\n",
      "Average out-degree: 1.65625\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height) {\n",
       "    const function_colors = {\n",
       "        \"Abundance\": \"#AEC7E8\",\n",
       "        \"Gene\": \"#FFBB78\",\n",
       "        \"RNA\": \"#FF9896\",\n",
       "        \"miRNA\": \"#D62728\",\n",
       "        \"Protein\": \"#1F77B4\",\n",
       "        \"Complex\": \"#98DF8A\",\n",
       "        \"Composite\": \"#9467BD\",\n",
       "        \"BiologicalProcess\": \"#2CA02C\",\n",
       "        \"Pathology\": \"#FF7F0E\"\n",
       "    };\n",
       "\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#888\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"bxvqysqagfzpuome\";\n",
       "        var width = 900;\n",
       "        var height = 600;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"pybel_version\": \"0.4.1\", \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"leukotriene\", \"cname\": \"leukotriene\", \"id\": [\"Abundance\", \"CHEBI\", \"leukotriene\"]}, {\"function\": \"Abundance\", \"namespace\": \"ADO\", \"name\": \"neuroprotectin\", \"cname\": \"neuroprotectin\", \"id\": [\"Abundance\", \"ADO\", \"neuroprotectin\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"melatonin\", \"cname\": \"melatonin\", \"id\": [\"Abundance\", \"CHEBI\", \"melatonin\"]}, {\"function\": \"Abundance\", \"namespace\": \"MESHCS\", \"name\": \"Neurofibrillary Tangles\", \"cname\": \"Neurofibrillary Tangles\", \"id\": [\"Abundance\", \"MESHCS\", \"Neurofibrillary Tangles\"]}, {\"function\": \"Abundance\", \"namespace\": \"BRCO\", \"name\": \"Astrocyte\", \"cname\": \"Astrocyte\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"noradrenaline\", \"cname\": \"noradrenaline\", \"id\": [\"Abundance\", \"CHEBI\", \"noradrenaline\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CX3CR1\", \"weight\": -0.81994598, \"cname\": \"CX3CR1\", \"id\": [\"Gene\", \"HGNC\", \"CX3CR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TREM2\", \"weight\": -0.40134828, \"cname\": \"TREM2\", \"id\": [\"Gene\", \"HGNC\", \"TREM2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IRAK1\", \"weight\": 0.72985984, \"cname\": \"IRAK1\", \"id\": [\"Gene\", \"HGNC\", \"IRAK1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IRAK2\", \"weight\": -0.26455035, \"cname\": \"IRAK2\", \"id\": [\"Gene\", \"HGNC\", \"IRAK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PTX3\", \"weight\": -1.00547039, \"cname\": \"PTX3\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL23A\", \"weight\": -0.28486748, \"cname\": \"IL23A\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"weight\": 0.18225965, \"cname\": \"SCARF1\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"weight\": -1.61550531, \"cname\": \"IL12B\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR2\", \"weight\": 0.13520554, \"cname\": \"TLR2\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CD14\", \"weight\": -0.73001192, \"cname\": \"CD14\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR1\", \"weight\": -0.5889405, \"cname\": \"FPR1\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NOD2\", \"weight\": -0.62084437, \"cname\": \"NOD2\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TREM1\", \"weight\": 1.13272121, \"cname\": \"TREM1\", \"id\": [\"Gene\", \"HGNC\", \"TREM1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PLA2G7\", \"weight\": -0.52445663, \"cname\": \"PLA2G7\", \"id\": [\"Gene\", \"HGNC\", \"PLA2G7\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHI3L1\", \"weight\": 0.14672037, \"cname\": \"CHI3L1\", \"id\": [\"Gene\", \"HGNC\", \"CHI3L1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MYD88\", \"weight\": 0.981683, \"cname\": \"MYD88\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR2\", \"weight\": 5.3e-05, \"cname\": \"FPR2\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Obesity\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Diabetes Mellitus, Type 2\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Hypersensitivity\"}, \"source\": 2, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Stroke\"}, \"source\": 2, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Asthma\"}, \"source\": 2, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces A beta peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2011 Jan 5;6(1):e15816. doi: 10.1371/journal.pone.0015816.\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces A beta peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2011 Jan 5;6(1):e15816. doi: 10.1371/journal.pone.0015816.\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces Abeta42 peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2011\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces Abeta42 peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2011\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces Abeta42 peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2011\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"Recent studies showed that melatonin, an indoleamine secreted by the pineal gland, may play an important/ role in aging and AD as an antioxidant and neuroprotector. Melatonin decreases during aging and patients with AD have a/ more profound reduction in this hormone.Melatonin efficiently protects neuronal cells from Abeta-mediated toxicity via/ antioxidant and anti-amyloid properties: it not only inhibits Abeta generation, but also arrests the formation of amyloid/ fibrils by a structure-dependent interaction with Abeta. Our recent studies have demonstrated that melatonin efficiently/ attenuates Alzheimer-like tau hyperphosphorylation. Although the exact mechanism is still not fully understood, a direct/ regulatory influence of melatonin on the activities of protein kinases and protein phosphatases is proposed. Additionally,/ melatonin also plays a role in protecting cholinergic neurons and in anti-inflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Pharmacol Sin. 2006 Jan;27(1):41-9.\", \"reference\": \"16364209\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\"}, \"annotations\": {}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;20(4):1189-99. doi: 10.3233/JAD-2010-091336.\", \"reference\": \"20413894\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Experimental evidence suggests that cortical noradrenaline (NA) depletion due to degeneration of the locus/ ceruleus (LC) - a pathological hallmark of AD - plays a permissive role in the development of inflammation in AD. Our/ results indicate for the first time that PPARgamma expression can be modulated by the cAMP signalling pathway, and/ suggest that the anti-inflammatory effects of NA on brain cells may be partly mediated by increasing PPARgamma levels./ Conversely, decreased NA due to LC cell death in AD may reduce endogenous PPARgamma expression and therefore potentiate/ neuroinflammatory processes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2003 Aug;86(4):907-16.\", \"reference\": \"12887689\"}, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Experimental evidence suggests that cortical noradrenaline (NA) depletion due to degeneration of the locus/ ceruleus (LC) - a pathological hallmark of AD - plays a permissive role in the development of inflammation in AD. Our/ results indicate for the first time that PPARgamma expression can be modulated by the cAMP signalling pathway, and/ suggest that the anti-inflammatory effects of NA on brain cells may be partly mediated by increasing PPARgamma levels./ Conversely, decreased NA due to LC cell death in AD may reduce endogenous PPARgamma expression and therefore potentiate/ neuroinflammatory processes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2003 Aug;86(4):907-16.\", \"reference\": \"12887689\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" We found that bilateral microinjection of amyloid beta (Aβ)1-40 fibrils into the hippocampal CA1 area of resulted in significant upregulation of CX3CR1 messenger RNA (mRNA) and protein expression (via increasing histone H3 acetylation in the Cx3cr1 promoter region), synaptic dysfunction, and cognitive impairment,\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2013 Dec;34(12):2843-52. doi: 10.1016/j.neurobiolaging.2013.06.003. Epub 2013 Jul 12.\", \"reference\": \"23855980\"}, \"annotations\": {\"NervousSystem\": \"Hippocampus\", \"Encode_Feature_Types\": \"Promoter\"}, \"source\": 9, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Recently, a novel variant in the gene encoding the triggering receptor expressed on myeloid cells 2 (TREM2) has been identified that has refocused the spotlight back onto inflammation as a major contributing factor in AD. Variants in TREM2 triple one's risk of developing late-onset AD. TREM2 is expressed on microglial cells, the resident macrophages in the CNS, and functions to stimulate phagocytosis on one hand and to suppress cytokine production and inflammation on the other hand. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"N Engl J Med. 2013 Jan 10;368(2):117-27. doi: 10.1056/NEJMoa1211851. Epub 2012 Nov 14.\", \"reference\": \"23150934\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 11, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 12, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 14, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 17, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 18, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 19, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 19, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 20, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 21, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 21, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 22, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 22, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 23, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 24, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 25, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 26, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Recent Pat Antiinfect Drug Discov. 2009 Jan;4(1):51-6.\", \"reference\": \"19149696\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\", \"Cell\": \"monocyte\"}, \"source\": 27, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Recent Pat Antiinfect Drug Discov. 2009 Jan;4(1):51-6.\", \"reference\": \"19149696\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\", \"Cell\": \"neutrophil\"}, \"source\": 27, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Platelets are an important source of amyloid-ss (Ass) in the circulatory system and play an important pro-inflammatory role. Upon activation, they adhere to leukocytes and endothelial cells by means of adhesive proteins like P-selectin, platelet endothelial cell adhesion molecule-1 (PECAM) and intercellular adhesion molecule-1 and -2 (ICAM-1 and -2) and secrete inflammatory mediators (chemokines, interleukins). In addition, platelets contain important enzymes involved in inflammatory intermediary synthesis like phospholipase A(2) (PLA(2)) and cyclooxygenase-2 (COX-2), and recent reports demonstrated significant changes in platelet levels and activities in Alzheimer's disease. Thus, as platelets represent an important link between Ass deposition and inflammatory reactions especially at endothelial level, they can be considered a valuable cellular model to evaluate potential peripheral inflammatory biomarkers in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biogerontology. 2010 Oct;11(5):627-33. doi: 10.1007/s10522-010-9281-8. Epub 2010 May 9.\", \"reference\": \"20454929\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 28, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"There was a strong correlation between YKL-40 and markers of neurodegeneration (total tau and p-tau). In the whole sample, we found a negative correlation between YKL-40 and CTh in AD vulnerable areas in Aβ42+ subjects but not in Aβ42 participants. Our results suggest that YKL-40 could track the inflammatory processes associated to tau-related neurodegeneration in the presence of the AD pathophysiological process.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2015 Jun;36(6):2018-23. doi: 10.1016/j.neurobiolaging.2015.03.001\", \"reference\": \"25865441\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 29, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 30, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 31, \"target\": 1, \"key\": 0}]};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "subgraph = pbt.mutation.get_upstream_causal_subgraph(graph, inflamation_node)\n",
    "\n",
    "pbt.summary.print_summary(subgraph)\n",
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "### Iteration 2/2\n",
    "\n",
    "All of the nodes upstream of the nodes within the subgraph are added as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Name: \n",
      "Number of nodes: 278\n",
      "Number of edges: 1188\n",
      "Network density: 0.01542736929589902\n",
      "Number weakly connected components: 1\n",
      "Average in-degree: 4.273381294964029\n",
      "Average out-degree: 4.273381294964029\n"
     ]
    }
   ],
   "source": [
    "pbt.mutation.expand_upstream_causal_subgraph(graph, subgraph)\n",
    "pbt.summary.print_summary(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "### Post-Processing\n",
    "\n",
    "Inconsistient edges are thrown away, literature bias is removed, and nodes on the periphery of the subgraph that are not annotated with experimental data are also removed, since they cannot contribute to the propogation of scores."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "pbt.mutation.remove_inconsistent_edges(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "pbt.mutation.collapse_consistent_edges(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "pbt.generation.remove_unweighted_leaves(subgraph, label)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "pbt.generation.remove_unweighted_sources(subgraph, label)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Name: \n",
      "Number of nodes: 129\n",
      "Number of edges: 740\n",
      "Network density: 0.04481589147286822\n",
      "Number weakly connected components: 1\n",
      "Average in-degree: 5.736434108527132\n",
      "Average out-degree: 5.736434108527132\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height) {\n",
       "    const function_colors = {\n",
       "        \"Abundance\": \"#AEC7E8\",\n",
       "        \"Gene\": \"#FFBB78\",\n",
       "        \"RNA\": \"#FF9896\",\n",
       "        \"miRNA\": \"#D62728\",\n",
       "        \"Protein\": \"#1F77B4\",\n",
       "        \"Complex\": \"#98DF8A\",\n",
       "        \"Composite\": \"#9467BD\",\n",
       "        \"BiologicalProcess\": \"#2CA02C\",\n",
       "        \"Pathology\": \"#FF7F0E\"\n",
       "    };\n",
       "\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#888\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"fthumgyvqacsjedq\";\n",
       "        var width = 900;\n",
       "        var height = 600;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"pybel_version\": \"0.4.1\", \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"leukotriene\", \"cname\": \"leukotriene\", \"id\": [\"Abundance\", \"CHEBI\", \"leukotriene\"]}, {\"function\": \"Abundance\", \"namespace\": \"MESHCS\", \"name\": \"Neurofibrillary Tangles\", \"cname\": \"Neurofibrillary Tangles\", \"id\": [\"Abundance\", \"MESHCS\", \"Neurofibrillary Tangles\"]}, {\"function\": \"Abundance\", \"namespace\": \"BRCO\", \"name\": \"Astrocyte\", \"cname\": \"Astrocyte\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CX3CR1\", \"weight\": -0.81994598, \"cname\": \"CX3CR1\", \"id\": [\"Gene\", \"HGNC\", \"CX3CR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TREM2\", \"weight\": -0.40134828, \"cname\": \"TREM2\", \"id\": [\"Gene\", \"HGNC\", \"TREM2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IRAK1\", \"weight\": 0.72985984, \"cname\": \"IRAK1\", \"id\": [\"Gene\", \"HGNC\", \"IRAK1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IRAK2\", \"weight\": -0.26455035, \"cname\": \"IRAK2\", \"id\": [\"Gene\", \"HGNC\", \"IRAK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PTX3\", \"weight\": -1.00547039, \"cname\": \"PTX3\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL23A\", \"weight\": -0.28486748, \"cname\": \"IL23A\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"weight\": 0.18225965, \"cname\": \"SCARF1\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"weight\": -1.61550531, \"cname\": \"IL12B\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR2\", \"weight\": 0.13520554, \"cname\": \"TLR2\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CD14\", \"weight\": -0.73001192, \"cname\": \"CD14\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR1\", \"weight\": -0.5889405, \"cname\": \"FPR1\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NOD2\", \"weight\": -0.62084437, \"cname\": \"NOD2\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TREM1\", \"weight\": 1.13272121, \"cname\": \"TREM1\", \"id\": [\"Gene\", \"HGNC\", \"TREM1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PLA2G7\", \"weight\": -0.52445663, \"cname\": \"PLA2G7\", \"id\": [\"Gene\", \"HGNC\", \"PLA2G7\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHI3L1\", \"weight\": 0.14672037, \"cname\": \"CHI3L1\", \"id\": [\"Gene\", \"HGNC\", \"CHI3L1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MYD88\", \"weight\": 0.981683, \"cname\": \"MYD88\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR2\", \"weight\": 5.3e-05, \"cname\": \"FPR2\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"c.275341G>C\"}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, var(c.275341G>C))\", \"id\": [\"Gene\", \"HGNC\", \"APP\", [\"hgvs\", \"c.275341G>C\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"c.717G>C\"}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, var(c.717G>C))\", \"id\": [\"Gene\", \"HGNC\", \"APP\", [\"hgvs\", \"c.717G>C\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTSD\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"c.224C>T\"}], \"weight\": 0.88300839, \"cname\": \"g(HGNC:CTSD, var(c.224C>T))\", \"id\": [\"Gene\", \"HGNC\", \"CTSD\", [\"hgvs\", \"c.224C>T\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APOE\", \"weight\": 0.04406715, \"cname\": \"APOE\", \"id\": [\"Gene\", \"HGNC\", \"APOE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"weight\": -0.13701466, \"cname\": \"APP\", \"id\": [\"Gene\", \"HGNC\", \"APP\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSEN1\", \"weight\": 0.00266298, \"cname\": \"PSEN1\", \"id\": [\"Gene\", \"HGNC\", \"PSEN1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSEN2\", \"weight\": -0.02429789, \"cname\": \"PSEN2\", \"id\": [\"Gene\", \"HGNC\", \"PSEN2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PPARG\", \"weight\": -0.06808657, \"cname\": \"PPARG\", \"id\": [\"Gene\", \"HGNC\", \"PPARG\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTSD\", \"weight\": 0.88300839, \"cname\": \"CTSD\", \"id\": [\"Gene\", \"HGNC\", \"CTSD\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CR1\", \"weight\": 0.02399208, \"cname\": \"CR1\", \"id\": [\"Gene\", \"HGNC\", \"CR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PICALM\", \"weight\": -0.14693108, \"cname\": \"PICALM\", \"id\": [\"Gene\", \"HGNC\", \"PICALM\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SORL1\", \"weight\": -0.01427188, \"cname\": \"SORL1\", \"id\": [\"Gene\", \"HGNC\", \"SORL1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3A\", \"weight\": -0.45894849, \"cname\": \"GSK3A\", \"id\": [\"Gene\", \"HGNC\", \"GSK3A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ACHE\", \"weight\": 0.49236940000000007, \"cname\": \"ACHE\", \"id\": [\"Gene\", \"HGNC\", \"ACHE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"INSR\", \"weight\": 0.049953899999999996, \"cname\": \"INSR\", \"id\": [\"Gene\", \"HGNC\", \"INSR\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NOS3\", \"weight\": -0.1714975, \"cname\": \"NOS3\", \"id\": [\"Gene\", \"HGNC\", \"NOS3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"BACE1\", \"weight\": -0.00189721, \"cname\": \"BACE1\", \"id\": [\"Gene\", \"HGNC\", \"BACE1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"INS\", \"weight\": 0.99910812, \"cname\": \"INS\", \"id\": [\"Gene\", \"HGNC\", \"INS\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"STK11\", \"weight\": 0.17049337, \"cname\": \"STK11\", \"id\": [\"Gene\", \"HGNC\", \"STK11\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SPTLC1\", \"weight\": -0.0808145, \"cname\": \"SPTLC1\", \"id\": [\"Gene\", \"HGNC\", \"SPTLC1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APBA1\", \"weight\": -0.48809369999999996, \"cname\": \"APBA1\", \"id\": [\"Gene\", \"HGNC\", \"APBA1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SORCS1\", \"weight\": -1.5152121, \"cname\": \"SORCS1\", \"id\": [\"Gene\", \"HGNC\", \"SORCS1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSENEN\", \"weight\": 1.24788008, \"cname\": \"PSENEN\", \"id\": [\"Gene\", \"HGNC\", \"PSENEN\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CCR2\", \"weight\": 0.72408115, \"cname\": \"CCR2\", \"id\": [\"Gene\", \"HGNC\", \"CCR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"BACE2\", \"weight\": -0.34884808, \"cname\": \"BACE2\", \"id\": [\"Gene\", \"HGNC\", \"BACE2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IGF1\", \"weight\": 0.98026104, \"cname\": \"IGF1\", \"id\": [\"Gene\", \"HGNC\", \"IGF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"RELA\", \"weight\": 0.39927783, \"cname\": \"RELA\", \"id\": [\"Gene\", \"HGNC\", \"RELA\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CAST\", \"weight\": 0.03364871, \"cname\": \"CAST\", \"id\": [\"Gene\", \"HGNC\", \"CAST\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NFKB1\", \"weight\": -1.33219344, \"cname\": \"NFKB1\", \"id\": [\"Gene\", \"HGNC\", \"NFKB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MIR124-1\", \"weight\": 0, \"cname\": \"MIR124-1\", \"id\": [\"Gene\", \"HGNC\", \"MIR124-1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IDE\", \"weight\": 0.28059224, \"cname\": \"IDE\", \"id\": [\"Gene\", \"HGNC\", \"IDE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL1B\", \"weight\": -1.63771704, \"cname\": \"IL1B\", \"id\": [\"Gene\", \"HGNC\", \"IL1B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ATXN1\", \"weight\": 0.00954063, \"cname\": \"ATXN1\", \"id\": [\"Gene\", \"HGNC\", \"ATXN1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SPTLC3\", \"weight\": 0.19053622, \"cname\": \"SPTLC3\", \"id\": [\"Gene\", \"HGNC\", \"SPTLC3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ADAM10\", \"weight\": -0.051464199999999995, \"cname\": \"ADAM10\", \"id\": [\"Gene\", \"HGNC\", \"ADAM10\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FTL\", \"weight\": 0.70775495, \"cname\": \"FTL\", \"id\": [\"Gene\", \"HGNC\", \"FTL\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"DDIT3\", \"weight\": 1.40355466, \"cname\": \"DDIT3\", \"id\": [\"Gene\", \"HGNC\", \"DDIT3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CLSTN1\", \"weight\": 0.38076231, \"cname\": \"CLSTN1\", \"id\": [\"Gene\", \"HGNC\", \"CLSTN1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"KLC1\", \"weight\": -0.09664774, \"cname\": \"KLC1\", \"id\": [\"Gene\", \"HGNC\", \"KLC1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GPR3\", \"weight\": 0.14999666, \"cname\": \"GPR3\", \"id\": [\"Gene\", \"HGNC\", \"GPR3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APH1A\", \"weight\": -0.42715932, \"cname\": \"APH1A\", \"id\": [\"Gene\", \"HGNC\", \"APH1A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"A2M\", \"weight\": 1.08160069, \"cname\": \"A2M\", \"id\": [\"Gene\", \"HGNC\", \"A2M\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MTRNR2L2\", \"weight\": 0, \"cname\": \"MTRNR2L2\", \"id\": [\"Gene\", \"HGNC\", \"MTRNR2L2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APBB1\", \"weight\": 0.59661584, \"cname\": \"APBB1\", \"id\": [\"Gene\", \"HGNC\", \"APBB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APBA2\", \"weight\": -0.19241516, \"cname\": \"APBA2\", \"id\": [\"Gene\", \"HGNC\", \"APBA2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"LDLR\", \"weight\": -0.01435724, \"cname\": \"LDLR\", \"id\": [\"Gene\", \"HGNC\", \"LDLR\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PIN1\", \"weight\": -0.25496963, \"cname\": \"PIN1\", \"id\": [\"Gene\", \"HGNC\", \"PIN1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SPTLC2\", \"weight\": -0.06391096, \"cname\": \"SPTLC2\", \"id\": [\"Gene\", \"HGNC\", \"SPTLC2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NAT8B\", \"weight\": 0.20466742, \"cname\": \"NAT8B\", \"id\": [\"Gene\", \"HGNC\", \"NAT8B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GGA3\", \"weight\": -0.81325519, \"cname\": \"GGA3\", \"id\": [\"Gene\", \"HGNC\", \"GGA3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PRNP\", \"weight\": 0.08884805, \"cname\": \"PRNP\", \"id\": [\"Gene\", \"HGNC\", \"PRNP\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ABCA2\", \"weight\": -0.03926027, \"cname\": \"ABCA2\", \"id\": [\"Gene\", \"HGNC\", \"ABCA2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MARCKS\", \"weight\": 0.07542072, \"cname\": \"MARCKS\", \"id\": [\"Gene\", \"HGNC\", \"MARCKS\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL34\", \"weight\": -0.18066962, \"cname\": \"IL34\", \"id\": [\"Gene\", \"HGNC\", \"IL34\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ADAM9\", \"weight\": -0.16119139, \"cname\": \"ADAM9\", \"id\": [\"Gene\", \"HGNC\", \"ADAM9\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ADAM17\", \"weight\": 0.21966586, \"cname\": \"ADAM17\", \"id\": [\"Gene\", \"HGNC\", \"ADAM17\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTSB\", \"weight\": -0.04291486, \"cname\": \"CTSB\", \"id\": [\"Gene\", \"HGNC\", \"CTSB\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTSE\", \"weight\": -1.2097365, \"cname\": \"CTSE\", \"id\": [\"Gene\", \"HGNC\", \"CTSE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GCG\", \"weight\": -0.03157631, \"cname\": \"GCG\", \"id\": [\"Gene\", \"HGNC\", \"GCG\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ECE1\", \"weight\": -0.09027805, \"cname\": \"ECE1\", \"id\": [\"Gene\", \"HGNC\", \"ECE1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ACE\", \"weight\": 0.38262923, \"cname\": \"ACE\", \"id\": [\"Gene\", \"HGNC\", \"ACE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"DPP4\", \"weight\": 0.17409829999999998, \"cname\": \"DPP4\", \"id\": [\"Gene\", \"HGNC\", \"DPP4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"DHCR24\", \"weight\": -0.17838582, \"cname\": \"DHCR24\", \"id\": [\"Gene\", \"HGNC\", \"DHCR24\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"BLMH\", \"weight\": -0.85202484, \"cname\": \"BLMH\", \"id\": [\"Gene\", \"HGNC\", \"BLMH\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ALB\", \"weight\": 0.02354135, \"cname\": \"ALB\", \"id\": [\"Gene\", \"HGNC\", \"ALB\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"LEP\", \"weight\": 1.01085844, \"cname\": \"LEP\", \"id\": [\"Gene\", \"HGNC\", \"LEP\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MMP9\", \"weight\": -0.32509715, \"cname\": \"MMP9\", \"id\": [\"Gene\", \"HGNC\", \"MMP9\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MMP2\", \"weight\": -0.47336532, \"cname\": \"MMP2\", \"id\": [\"Gene\", \"HGNC\", \"MMP2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"BCL2L1\", \"weight\": -0.29881302, \"cname\": \"BCL2L1\", \"id\": [\"Gene\", \"HGNC\", \"BCL2L1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ABCA1\", \"weight\": 0.17888996, \"cname\": \"ABCA1\", \"id\": [\"Gene\", \"HGNC\", \"ABCA1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MMP7\", \"weight\": -1.19454136, \"cname\": \"MMP7\", \"id\": [\"Gene\", \"HGNC\", \"MMP7\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PLAT\", \"weight\": 1.04346523, \"cname\": \"PLAT\", \"id\": [\"Gene\", \"HGNC\", \"PLAT\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SERPINI1\", \"weight\": -1.09712643, \"cname\": \"SERPINI1\", \"id\": [\"Gene\", \"HGNC\", \"SERPINI1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PLAU\", \"weight\": -0.44174472, \"cname\": \"PLAU\", \"id\": [\"Gene\", \"HGNC\", \"PLAU\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TTR\", \"weight\": 4.07054127, \"cname\": \"TTR\", \"id\": [\"Gene\", \"HGNC\", \"TTR\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL4\", \"weight\": 0.91448898, \"cname\": \"IL4\", \"id\": [\"Gene\", \"HGNC\", \"IL4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APPBP2\", \"weight\": -0.01975785, \"cname\": \"APPBP2\", \"id\": [\"Gene\", \"HGNC\", \"APPBP2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PAWR\", \"weight\": -0.17425334, \"cname\": \"PAWR\", \"id\": [\"Gene\", \"HGNC\", \"PAWR\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CSNK1A1\", \"weight\": -0.09186504, \"cname\": \"CSNK1A1\", \"id\": [\"Gene\", \"HGNC\", \"CSNK1A1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"QPCT\", \"weight\": 0.51242099, \"cname\": \"QPCT\", \"id\": [\"Gene\", \"HGNC\", \"QPCT\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"UBE2I\", \"weight\": -0.15740177, \"cname\": \"UBE2I\", \"id\": [\"Gene\", \"HGNC\", \"UBE2I\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"RANBP9\", \"weight\": -0.20181786, \"cname\": \"RANBP9\", \"id\": [\"Gene\", \"HGNC\", \"RANBP9\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CAPN2\", \"weight\": -0.02208264, \"cname\": \"CAPN2\", \"id\": [\"Gene\", \"HGNC\", \"CAPN2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"LRRTM3\", \"weight\": 0, \"cname\": \"LRRTM3\", \"id\": [\"Gene\", \"HGNC\", \"LRRTM3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"RTN4R\", \"weight\": -1.01416883, \"cname\": \"RTN4R\", \"id\": [\"Gene\", \"HGNC\", \"RTN4R\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"HSPA1B\", \"weight\": 0, \"cname\": \"HSPA1B\", \"id\": [\"Gene\", \"HGNC\", \"HSPA1B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SP1\", \"weight\": -0.08678866, \"cname\": \"SP1\", \"id\": [\"Gene\", \"HGNC\", \"SP1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"S100A9\", \"weight\": -0.60784743, \"cname\": \"S100A9\", \"id\": [\"Gene\", \"HGNC\", \"S100A9\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"HTR4\", \"weight\": -0.15785455, \"cname\": \"HTR4\", \"id\": [\"Gene\", \"HGNC\", \"HTR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ALOX5\", \"weight\": -0.188697, \"cname\": \"ALOX5\", \"id\": [\"Gene\", \"HGNC\", \"ALOX5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"S100B\", \"weight\": 0.27537864, \"cname\": \"S100B\", \"id\": [\"Gene\", \"HGNC\", \"S100B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GRB2\", \"weight\": -0.02552084, \"cname\": \"GRB2\", \"id\": [\"Gene\", \"HGNC\", \"GRB2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTNNB1\", \"weight\": 0.03837192, \"cname\": \"CTNNB1\", \"id\": [\"Gene\", \"HGNC\", \"CTNNB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FRAT2\", \"weight\": -0.50393273, \"cname\": \"FRAT2\", \"id\": [\"Gene\", \"HGNC\", \"FRAT2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PIK3R5\", \"weight\": -0.23683106, \"cname\": \"PIK3R5\", \"id\": [\"Gene\", \"HGNC\", \"PIK3R5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MIR146A\", \"weight\": -0.53979604, \"cname\": \"MIR146A\", \"id\": [\"Gene\", \"HGNC\", \"MIR146A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"HMGB1\", \"weight\": -0.09432351, \"cname\": \"HMGB1\", \"id\": [\"Gene\", \"HGNC\", \"HMGB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CEBPD\", \"weight\": 0.47557886, \"cname\": \"CEBPD\", \"id\": [\"Gene\", \"HGNC\", \"CEBPD\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"STAT3\", \"weight\": 0.04651478, \"cname\": \"STAT3\", \"id\": [\"Gene\", \"HGNC\", \"STAT3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"OSM\", \"weight\": 0.64942734, \"cname\": \"OSM\", \"id\": [\"Gene\", \"HGNC\", \"OSM\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"ÃŸ-Amyloid (AÃŸ) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of AÃŸ, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3ÃŸ, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3ÃŸ after exposure to oligomeric AÃŸ in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3ÃŸ, even in the absence of AÃŸ, is sufficient to produce a phenocopy of AÃŸ-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3ÃŸ prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3ÃŸ inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3ÃŸ inhibition robustly decreased the oligomeric AÃŸ load in the mouse brain. All these findings support the idea that GSK3ÃŸ is aberrantly activated by the presence of AÃŸ, and contributes, at least in part, to the neuronal anatomical derangement associated with AÃŸ plaques in AD brains and to AÃŸ pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 0, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"ÃŸ-Amyloid (AÃŸ) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of AÃŸ, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3ÃŸ, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3ÃŸ after exposure to oligomeric AÃŸ in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3ÃŸ, even in the absence of AÃŸ, is sufficient to produce a phenocopy of AÃŸ-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3ÃŸ prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3ÃŸ inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3ÃŸ inhibition robustly decreased the oligomeric AÃŸ load in the mouse brain. All these findings support the idea that GSK3ÃŸ is aberrantly activated by the presence of AÃŸ, and contributes, at least in part, to the neuronal anatomical derangement associated with AÃŸ plaques in AD brains and to AÃŸ pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 5, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 0, \"target\": 5, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 0, \"target\": 5, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 5, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 0, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 0, \"target\": 10, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 0, \"target\": 10, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 16, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 0, \"target\": 20, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 20, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The serum IL-1β and TNF-Î± level were significantly increased, and the expressions of TLR4 and NF-ÎºB p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Aβ1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"NervousSystem\": \"Brain\", \"FluidAndSecretion\": \"Serum\"}, \"source\": 1, \"target\": 18, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Obesity\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Diabetes Mellitus, Type 2\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Hypersensitivity\"}, \"source\": 2, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Stroke\"}, \"source\": 2, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Asthma\"}, \"source\": 2, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\"}, \"annotations\": {}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;20(4):1189-99. doi: 10.3233/JAD-2010-091336.\", \"reference\": \"20413894\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Aβ) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Aβ in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Aβ production could make a significant contribution to Aβ burden in AD. Moreover, activated astrocytes may increase Aβ generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Aβ production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuroinflammation. 2011 Nov 2;8:150.\", \"reference\": \"22047170\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Aβ) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Aβ in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Aβ production could make a significant contribution to Aβ burden in AD. Moreover, activated astrocytes may increase Aβ generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Aβ production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuroinflammation. 2011 Nov 2;8:150.\", \"reference\": \"22047170\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Virol. 2009 Nov;83(21):11244-53. Epub 2009 Aug 26\", \"reference\": \"19710140\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 4, \"target\": 17, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Aβ) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 5, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Aβ) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 5, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" We found that bilateral microinjection of amyloid beta (Aβ)1-40 fibrils into the hippocampal CA1 area of resulted in significant upregulation of CX3CR1 messenger RNA (mRNA) and protein expression (via increasing histone H3 acetylation in the Cx3cr1 promoter region), synaptic dysfunction, and cognitive impairment,\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2013 Dec;34(12):2843-52. doi: 10.1016/j.neurobiolaging.2013.06.003. Epub 2013 Jul 12.\", \"reference\": \"23855980\"}, \"annotations\": {\"NervousSystem\": \"Hippocampus\", \"Encode_Feature_Types\": \"Promoter\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Recently, a novel variant in the gene encoding the triggering receptor expressed on myeloid cells 2 (TREM2) has been identified that has refocused the spotlight back onto inflammation as a major contributing factor in AD. Variants in TREM2 triple one's risk of developing late-onset AD. TREM2 is expressed on microglial cells, the resident macrophages in the CNS, and functions to stimulate phagocytosis on one hand and to suppress cytokine production and inflammation on the other hand. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"N Engl J Med. 2013 Jan 10;368(2):117-27. doi: 10.1056/NEJMoa1211851. Epub 2012 Nov 14.\", \"reference\": \"23150934\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 14, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 14, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 15, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 16, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 19, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 19, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 20, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 21, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 22, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 23, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Recent Pat Antiinfect Drug Discov. 2009 Jan;4(1):51-6.\", \"reference\": \"19149696\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\", \"Cell\": \"monocyte\"}, \"source\": 24, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Recent Pat Antiinfect Drug Discov. 2009 Jan;4(1):51-6.\", \"reference\": \"19149696\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\", \"Cell\": \"neutrophil\"}, \"source\": 24, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Platelets are an important source of amyloid-ss (Ass) in the circulatory system and play an important pro-inflammatory role. Upon activation, they adhere to leukocytes and endothelial cells by means of adhesive proteins like P-selectin, platelet endothelial cell adhesion molecule-1 (PECAM) and intercellular adhesion molecule-1 and -2 (ICAM-1 and -2) and secrete inflammatory mediators (chemokines, interleukins). In addition, platelets contain important enzymes involved in inflammatory intermediary synthesis like phospholipase A(2) (PLA(2)) and cyclooxygenase-2 (COX-2), and recent reports demonstrated significant changes in platelet levels and activities in Alzheimer's disease. Thus, as platelets represent an important link between Ass deposition and inflammatory reactions especially at endothelial level, they can be considered a valuable cellular model to evaluate potential peripheral inflammatory biomarkers in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biogerontology. 2010 Oct;11(5):627-33. doi: 10.1007/s10522-010-9281-8. Epub 2010 May 9.\", \"reference\": \"20454929\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 25, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"There was a strong correlation between YKL-40 and markers of neurodegeneration (total tau and p-tau). In the whole sample, we found a negative correlation between YKL-40 and CTh in AD vulnerable areas in Aβ42+ subjects but not in Aβ42 participants. Our results suggest that YKL-40 could track the inflammatory processes associated to tau-related neurodegeneration in the presence of the AD pathophysiological process.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2015 Jun;36(6):2018-23. doi: 10.1016/j.neurobiolaging.2015.03.001\", \"reference\": \"25865441\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 26, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 27, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 28, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 29, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 30, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cathepsin D, the most abundant lysosomal and endosomal aspartyl protease, shows beta and gamma secretase activity in vitro by cleaving the amyloid precursor protein (APP) into amyloid beta protein (AÃŸ). Polymorphism at position 224, C224T, on exon 2 of cathepsin D gene (CTSD) has been associated with an increased risk for Alzheimer's disease (AD) \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Neurol Scand. 2011 Jun;123(6):419-23. doi: 10.1111/j.1600-0404.2010.01400.x.\", \"reference\": \"20597865\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 31, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cathepsin D, the most abundant lysosomal and endosomal aspartyl protease, shows beta and gamma secretase activity in vitro by cleaving the amyloid precursor protein (APP) into amyloid beta protein (AÃŸ). Polymorphism at position 224, C224T, on exon 2 of cathepsin D gene (CTSD) has been associated with an increased risk for Alzheimer's disease (AD) \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Neurol Scand. 2011 Jun;123(6):419-23. doi: 10.1111/j.1600-0404.2010.01400.x.\", \"reference\": \"20597865\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 31, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of ÃŸ-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science. 2012 Feb 9. [Epub ahead of print]\", \"reference\": \"22323736\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"ApoE subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of ÃŸ-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science. 2012 Feb 9. [Epub ahead of print]\", \"reference\": \"22323736\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of ÃŸ-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science. 2012 Feb 9. [Epub ahead of print]\", \"reference\": \"22323736\"}, \"annotations\": {\"Subgraph\": \"ApoE subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of ÃŸ-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science. 2012 Feb 9. [Epub ahead of print]\", \"reference\": \"22323736\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" Recently, many amyloid PET-positive and cognitively normal subjects were found in PiB-PET studies. PiB-PET studies on healthy subjects have also shown that apolipoprotein (APO) E4 boosts the accumulation of amyloid-beta and may consequently accelerate the pathogenesis of AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain Nerve. 2010 Jul;62(7):757-67.\", \"reference\": \"20675880\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"ApoE subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \" Recently, many amyloid PET-positive and cognitively normal subjects were found in PiB-PET studies. PiB-PET studies on healthy subjects have also shown that apolipoprotein (APO) E4 boosts the accumulation of amyloid-beta and may consequently accelerate the pathogenesis of AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain Nerve. 2010 Jul;62(7):757-67.\", \"reference\": \"20675880\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Apolipoprotein E is the main lipid carrier in the brain and the best-established risk factor for late-onset Alzheimer's disease. Intracellular cholesterol levels influence the generation of amyloid-beta peptides, the toxic species thought to be a primary cause of Alzheimer's disease. Finally, compounds that modulate cholesterol metabolism affect amyloid-beta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Opin Lipidol. 2007 Jun;18(3):325-32.\", \"reference\": \"17495608\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"ApoE subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"Apolipoprotein E is the main lipid carrier in the brain and the best-established risk factor for late-onset Alzheimer's disease. Intracellular cholesterol levels influence the generation of amyloid-beta peptides, the toxic species thought to be a primary cause of Alzheimer's disease. Finally, compounds that modulate cholesterol metabolism affect amyloid-beta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Opin Lipidol. 2007 Jun;18(3):325-32.\", \"reference\": \"17495608\"}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \" ApoE was not protective, but was injurious, as deletion of ApoE delayed the neurodegeneration caused by alpha-synuclein and suppressed the accumulation of Abeta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Neurosci2008\", \"reference\": \"18297066\"}, \"annotations\": {\"Subgraph\": \"Synuclein subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \" ApoE was not protective, but was injurious, as deletion of ApoE delayed the neurodegeneration caused by alpha-synuclein and suppressed the accumulation of Abeta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Neurosci2008\", \"reference\": \"18297066\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 9}, {\"relation\": \"increases\", \"evidence\": \" ApoE was not protective, but was injurious, as deletion of ApoE delayed the neurodegeneration caused by alpha-synuclein and suppressed the accumulation of Abeta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Neurosci2008\", \"reference\": \"18297066\"}, \"annotations\": {\"Subgraph\": \"ApoE subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 10}, {\"relation\": \"decreases\", \"evidence\": \"We report that ApoE plays a role in facilitating the proteolytic clearance of soluble Abeta from the brain. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2008\", \"reference\": \"18549781\"}, \"annotations\": {\"Subgraph\": \"ApoE subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 11}, {\"relation\": \"decreases\", \"evidence\": \"We report that ApoE plays a role in facilitating the proteolytic clearance of soluble Abeta from the brain. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2008\", \"reference\": \"18549781\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 12}, {\"relation\": \"decreases\", \"evidence\": \" We further demonstrated that apolipoprotein E2 and E3 but not apolipoprotein E4 can decrease the fusogenic activity of Abeta(29-42) via a direct interaction. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1999\", \"reference\": \"10428074\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"ApoE subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 13}, {\"relation\": \"decreases\", \"evidence\": \" We further demonstrated that apolipoprotein E2 and E3 but not apolipoprotein E4 can decrease the fusogenic activity of Abeta(29-42) via a direct interaction. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1999\", \"reference\": \"10428074\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 14}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Upon binding, LRP8 transduces reelin signaling during neuronal development, and recent evidence has indicated that it interacts with the NR2A and NR2B subunits of NMDA receptor, being involved in neuronal functions such as maturation of NMDA receptor composition in the hippocampus, and the regulation of long-term potentiation. Subsequently, a novel interaction between reelin and AÃŸPP was discovered, leading to increase in the cell surface levels of AÃŸPP and affecting AÃŸPP processing and AÃŸ production. It was shown that reelin signaling in excitatory synapses can restore normal synaptic plasticity, which is impaired by oligomeric AÃŸ peptides at concentrations within the range detectable in the brains of AD patients. At high concentrations of AÃŸ peptides, reelin can no longer overcome the AÃŸ-induced functional suppression, and this condition coincides with a complete blockade of the reelin-dependent phosphorylation of NR2 subunits in NMDA receptors. This reversal requires the LRP receptor-dependent activation of tyrosine kinases of the Src family.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Toxicol. 2012;2012:187297. Epub 2012 Feb 8.\", \"reference\": \"22496686\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"GOCC\", \"name\": \"intracellular\"}, \"toLoc\": {\"namespace\": \"GOCC\", \"name\": \"extracellular space\"}}}, \"annotations\": {\"CellStructure\": \"Centrosome\", \"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Reelin subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Upon binding, LRP8 transduces reelin signaling during neuronal development, and recent evidence has indicated that it interacts with the NR2A and NR2B subunits of NMDA receptor, being involved in neuronal functions such as maturation of NMDA receptor composition in the hippocampus, and the regulation of long-term potentiation. Subsequently, a novel interaction between reelin and AÃŸPP was discovered, leading to increase in the cell surface levels of AÃŸPP and affecting AÃŸPP processing and AÃŸ production. It was shown that reelin signaling in excitatory synapses can restore normal synaptic plasticity, which is impaired by oligomeric AÃŸ peptides at concentrations within the range detectable in the brains of AD patients. At high concentrations of AÃŸ peptides, reelin can no longer overcome the AÃŸ-induced functional suppression, and this condition coincides with a complete blockade of the reelin-dependent phosphorylation of NR2 subunits in NMDA receptors. This reversal requires the LRP receptor-dependent activation of tyrosine kinases of the Src family.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Toxicol. 2012;2012:187297. Epub 2012 Feb 8.\", \"reference\": \"22496686\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"GOCC\", \"name\": \"intracellular\"}, \"toLoc\": {\"namespace\": \"GOCC\", \"name\": \"extracellular space\"}}}, \"annotations\": {\"CellStructure\": \"Centrosome\", \"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD), one of the major causes of disability and mortality in Western societies, is a progressive age-related neurodegenerative disorder. Increasing evidence suggests that the etiology of AD may involve disruptions of zinc (Zn) homeostasis. This review discusses current evidence supporting a potential role of Zn and zinc transporters (ZnTs) in processing of the amyloid beta protein precursor (APP) and amyloid beta (AÃŸ) peptide generation and aggregation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biofactors. 2012 Mar;38(2):98-106. doi: 10.1002/biof.199. Epub 2012 Mar 23.\", \"reference\": \"22447723\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"The low-density lipoprotein receptor (LDLR) has the highest affinity for apoE and plays an important role in brain cholesterol metabolism.These data suggest that increased APP expression and Abeta exposure alters microtubule function, leading to reduced transport of LDLR to the plasma membrane. Consequent deleterious effects on apoE uptake and function will have implications for AD pathogenesis and/or progression\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2010 Jan 1;5(1):e8556\", \"reference\": \"20049331\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"ApoE subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"The low-density lipoprotein receptor (LDLR) has the highest affinity for apoE and plays an important role in brain cholesterol metabolism.These data suggest that increased APP expression and Abeta exposure alters microtubule function, leading to reduced transport of LDLR to the plasma membrane. Consequent deleterious effects on apoE uptake and function will have implications for AD pathogenesis and/or progression\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2010 Jan 1;5(1):e8556\", \"reference\": \"20049331\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"The low-density lipoprotein receptor (LDLR) has the highest affinity for apoE and plays an important role in brain cholesterol metabolism.These data suggest that increased APP expression and Abeta exposure alters microtubule function, leading to reduced transport of LDLR to the plasma membrane. Consequent deleterious effects on apoE uptake and function will have implications for AD pathogenesis and/or progression\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2010 Jan 1;5(1):e8556\", \"reference\": \"20049331\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"DYRK1A phosphorylates APP at Thr688 and enhances APP amyloidogenic cleavage, resulting in increased level of Aβ40/42, formation of toxic Aβ oligomers and deposition of fibrillar amyloid in plaques. Brain amyloidosis contributes to loss of neuronal function and possibly also to neuronal loss. Moreover, elevated levels of Aβ may up-regulate DYRK1A expression and enhance the contribution of overexpressed DYRK1A to neurofibrillary degeneration and β-amyloidosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS J. 2011 Jan;278(2):236-45. doi: 10.1111/j.1742-4658.2010.07955.x.Epub 2010 Dec 13.\", \"reference\": \"21156028\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"DYRK1A subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"DYRK1A phosphorylates APP at Thr688 and enhances APP amyloidogenic cleavage, resulting in increased level of Aβ40/42, formation of toxic Aβ oligomers and deposition of fibrillar amyloid in plaques. Brain amyloidosis contributes to loss of neuronal function and possibly also to neuronal loss. Moreover, elevated levels of Aβ may up-regulate DYRK1A expression and enhance the contribution of overexpressed DYRK1A to neurofibrillary degeneration and β-amyloidosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS J. 2011 Jan;278(2):236-45. doi: 10.1111/j.1742-4658.2010.07955.x.Epub 2010 Dec 13.\", \"reference\": \"21156028\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \"DYRK1A phosphorylates APP at Thr688 and enhances APP amyloidogenic cleavage, resulting in increased level of Aβ40/42, formation of toxic Aβ oligomers and deposition of fibrillar amyloid in plaques. Brain amyloidosis contributes to loss of neuronal function and possibly also to neuronal loss. Moreover, elevated levels of Aβ may up-regulate DYRK1A expression and enhance the contribution of overexpressed DYRK1A to neurofibrillary degeneration and β-amyloidosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS J. 2011 Jan;278(2):236-45. doi: 10.1111/j.1742-4658.2010.07955.x.Epub 2010 Dec 13.\", \"reference\": \"21156028\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 9}, {\"relation\": \"increases\", \"evidence\": \"Depletion of Hrs and Tsg101, acting early in the multivesicular body pathway, retained APP in early endosomes and reduced Aβ(40) production. Conversely, depletion of CHMP6 and VPS4, acting late in the pathway, rerouted endosomal APP to the TGN for enhanced APP processing. We found that VPS35 (retromer)-mediated APP recycling to the TGN was required for efficient Aβ(40) production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A. 2012 Jul 24;109(30):E2077-82. Epub 2012 Jun 18\", \"reference\": \"22711829\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"trans-Golgi Network\"}}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 10}, {\"relation\": \"increases\", \"evidence\": \"Depletion of Hrs and Tsg101, acting early in the multivesicular body pathway, retained APP in early endosomes and reduced Aβ(40) production. Conversely, depletion of CHMP6 and VPS4, acting late in the pathway, rerouted endosomal APP to the TGN for enhanced APP processing. We found that VPS35 (retromer)-mediated APP recycling to the TGN was required for efficient Aβ(40) production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A. 2012 Jul 24;109(30):E2077-82. Epub 2012 Jun 18\", \"reference\": \"22711829\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"trans-Golgi Network\"}}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Endoplasmic reticulum-Golgi protein export\"}, \"source\": 33, \"target\": 0, \"key\": 11}, {\"relation\": \"increases\", \"evidence\": \" An interruption of the bidirectional trafficking of APP between the TGN and endosomes, particularly retromer-mediated retrieval of APP from early endosomes to the TGN, resulted in the accumulation of endocytosed APP in early endosomes with reduced APP processing. These data suggest that Aβ(40) is generated predominantly in the TGN, relying on an endocytosed pool of APP recycled from early endosomes to the TGN.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A. 2012 Jul 24;109(30):E2077-82. Epub 2012 Jun 18\", \"reference\": \"22711829\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"trans-Golgi Network\"}}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 12}, {\"relation\": \"increases\", \"evidence\": \" An interruption of the bidirectional trafficking of APP between the TGN and endosomes, particularly retromer-mediated retrieval of APP from early endosomes to the TGN, resulted in the accumulation of endocytosed APP in early endosomes with reduced APP processing. These data suggest that Aβ(40) is generated predominantly in the TGN, relying on an endocytosed pool of APP recycled from early endosomes to the TGN.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A. 2012 Jul 24;109(30):E2077-82. Epub 2012 Jun 18\", \"reference\": \"22711829\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"trans-Golgi Network\"}}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Endoplasmic reticulum-Golgi protein export\"}, \"source\": 33, \"target\": 0, \"key\": 13}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 14}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 15}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 16}, {\"relation\": \"increases\", \"evidence\": \"Cirrito et al. [26] also show that synaptic activity-induced increase in endocytosis drives more APP into the endocytic compartment, ultimately resulting in increased Aï¿½ production and release. Aï¿½ produced in the endocytic pathway is then brought to the cell surface where it is released into the extracellular fluid [70]. Inhibition of endocytosis reduces APP internalization and reduces Aï¿½ production and release in cell lines\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Membrane\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 17}, {\"relation\": \"increases\", \"evidence\": \"Cirrito et al. [26] also show that synaptic activity-induced increase in endocytosis drives more APP into the endocytic compartment, ultimately resulting in increased Aï¿½ production and release. Aï¿½ produced in the endocytic pathway is then brought to the cell surface where it is released into the extracellular fluid [70]. Inhibition of endocytosis reduces APP internalization and reduces Aï¿½ production and release in cell lines\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Membrane\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Synaptic vesicle endocytosis subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 18}, {\"relation\": \"increases\", \"evidence\": \"Expression of familial Alzheimer's disease (FAD) mutants of APP in primary neurons causes both intracellular accumulation of the C-terminal beta-secretase cleavage product of APP and increased secretion of Abeta, and eventually results in apoptotic death of the cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain Res Mol Brain Res2001\", \"reference\": \"11744168\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 19}, {\"relation\": \"increases\", \"evidence\": \"Expression of familial Alzheimer's disease (FAD) mutants of APP in primary neurons causes both intracellular accumulation of the C-terminal beta-secretase cleavage product of APP and increased secretion of Abeta, and eventually results in apoptotic death of the cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain Res Mol Brain Res2001\", \"reference\": \"11744168\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 20}, {\"relation\": \"increases\", \"evidence\": \"The formation of beta A4 amyloid in the brains of individuals with Alzheimer's disease requires the proteolytic cleavage of amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of biochemistry and cell biology1999\", \"reference\": \"10605825\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 21}, {\"relation\": \"increases\", \"evidence\": \"The formation of beta A4 amyloid in the brains of individuals with Alzheimer's disease requires the proteolytic cleavage of amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of biochemistry and cell biology1999\", \"reference\": \"10605825\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 22}, {\"relation\": \"increases\", \"evidence\": \"Several lines of evidence suggest that cathepsin D, the major lysosomal/endosomal aspartic protease, may be involved in this process. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of biochemistry and cell biology1999\", \"reference\": \"10605825\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 23}, {\"relation\": \"increases\", \"evidence\": \"Several lines of evidence suggest that cathepsin D, the major lysosomal/endosomal aspartic protease, may be involved in this process. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of biochemistry and cell biology1999\", \"reference\": \"10605825\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 24}, {\"relation\": \"decreases\", \"evidence\": \"SORL1 regulates and traps APP in the Golgi apparatus and reduces the production of the amyloid-beta peptide (Abeta), the main component of senile plaques in AD4. Cell biology experiments have indicated that SORL1 underexperssion leads to overproduction of Abeta3 , whereas SORL1 overexpression considerably reduces cellular APP and extracellular Abeta5. Although the underlying mechanisms of SORL1 in AD and neurodegenerative processes remain unclear, a recent comprehensive review on multiple functional domains of SORL1 suggested that SORL1 can assume various characters in AD pathogenesis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Rep. 2016 Mar 21;6:23362. doi: 10.1038/srep23362.\", \"reference\": \"26996954\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\", \"CellStructure\": \"Golgi Apparatus\"}, \"source\": 33, \"target\": 0, \"key\": 25}, {\"relation\": \"decreases\", \"evidence\": \"SORL1 regulates and traps APP in the Golgi apparatus and reduces the production of the amyloid-beta peptide (Abeta), the main component of senile plaques in AD4. Cell biology experiments have indicated that SORL1 underexperssion leads to overproduction of Abeta3 , whereas SORL1 overexpression considerably reduces cellular APP and extracellular Abeta5. Although the underlying mechanisms of SORL1 in AD and neurodegenerative processes remain unclear, a recent comprehensive review on multiple functional domains of SORL1 suggested that SORL1 can assume various characters in AD pathogenesis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Rep. 2016 Mar 21;6:23362. doi: 10.1038/srep23362.\", \"reference\": \"26996954\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 26}, {\"relation\": \"increases\", \"evidence\": \"Although the role of AICD as a gene transcription inducer is still controversial, several targets of AICD- or APP-mediated transcriptional activation are reportedly regulated by APP nuclear signaling, including APP itself, BACE, Tip60, GSK-3ÃƒÅ¸, p53, Mn-SOD, KAI1, Neprilysin, transgelin, a2actin, S100a9, and other genes\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMB Rep. 2010 Oct;43(10):656-63.\", \"reference\": \"21034527\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 33, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"17995932\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"17995932\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"This cleavage is similar to the PS-dependent gamma-secretase cleavage of the beta-amyloid precursor protein (betaAPP). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"EMBO journal2002\", \"reference\": \"12374741\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"This cleavage is similar to the PS-dependent gamma-secretase cleavage of the beta-amyloid precursor protein (betaAPP). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"EMBO journal2002\", \"reference\": \"12374741\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Presenilin-1 (PS1) is intimately involved in cleavage of amyloid precursor protein to form beta-amyloid peptides, certain forms of which aggregate in the brains of patients with Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett2002\", \"reference\": \"11814648\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"Presenilin-1 (PS1) is intimately involved in cleavage of amyloid precursor protein to form beta-amyloid peptides, certain forms of which aggregate in the brains of patients with Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett2002\", \"reference\": \"11814648\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"It was hypothesized that PS1 might directly cleave APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol2000\", \"reference\": \"10670705\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \"It was hypothesized that PS1 might directly cleave APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol2000\", \"reference\": \"10670705\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 9}, {\"relation\": \"increases\", \"evidence\": \"Therefore, our results show that PS1 binds to APP directly and suggest that the PS1 protein itself is involved in the metabolism of beta-amyloid peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Biophys Res Commun1997\", \"reference\": \"9344855\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 10}, {\"relation\": \"increases\", \"evidence\": \"Therefore, our results show that PS1 binds to APP directly and suggest that the PS1 protein itself is involved in the metabolism of beta-amyloid peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Biophys Res Commun1997\", \"reference\": \"9344855\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 11}, {\"relation\": \"increases\", \"evidence\": \"Our results indicate PS1 and APP can interact in the ER and Golgi, where PS1 is required for proper gamma-secretase processing of APP CTFs, and that PS1 mutations augment Abeta42 levels principally in Golgi-like vesicles.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry1998\", \"reference\": \"9843412\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 12}, {\"relation\": \"increases\", \"evidence\": \"Our results indicate PS1 and APP can interact in the ER and Golgi, where PS1 is required for proper gamma-secretase processing of APP CTFs, and that PS1 mutations augment Abeta42 levels principally in Golgi-like vesicles.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry1998\", \"reference\": \"9843412\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 13}, {\"relation\": \"increases\", \"evidence\": \"Missense substitutions in the presenilin 1 (PS1) and presenilin 2 (PS2) proteins are associated with early-onset familial Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1999\", \"reference\": \"10037471\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 14}, {\"relation\": \"increases\", \"evidence\": \"Missense substitutions in the presenilin 1 (PS1) and presenilin 2 (PS2) proteins are associated with early-onset familial Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1999\", \"reference\": \"10037471\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 15}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta) is generated by the consecutive cleavages of beta- and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2002\", \"reference\": \"12147673\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 16}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta) is generated by the consecutive cleavages of beta- and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2002\", \"reference\": \"12147673\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 17}, {\"relation\": \"increases\", \"evidence\": \"Abundant biochemical and genetic evidence suggests that presenilins are catalytic components of gamma-secretase, the protease responsible for generating the Alzheimer amyloid beta-protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2003\", \"reference\": \"12603837\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 18}, {\"relation\": \"increases\", \"evidence\": \"Abundant biochemical and genetic evidence suggests that presenilins are catalytic components of gamma-secretase, the protease responsible for generating the Alzheimer amyloid beta-protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2003\", \"reference\": \"12603837\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 34, \"target\": 0, \"key\": 19}, {\"relation\": \"increases\", \"evidence\": \"hypomethylation of the promoter of the presenilin 1 (PS1) gene, which will lead to overexpression of presenilin 1 and, consequently, to increased Abeta(1-42) (Abeta42) formation \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2005 Sep 30;386(2):69-71.\", \"reference\": \"16040194\"}, \"annotations\": {\"Encode_Feature_Types\": \"Promoter\"}, \"source\": 34, \"target\": 0, \"key\": 20}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 35, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 35, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Missense substitutions in the presenilin 1 (PS1) and presenilin 2 (PS2) proteins are associated with early-onset familial Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1999\", \"reference\": \"10037471\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 35, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Missense substitutions in the presenilin 1 (PS1) and presenilin 2 (PS2) proteins are associated with early-onset familial Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1999\", \"reference\": \"10037471\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 35, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta) is generated by the consecutive cleavages of beta- and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2002\", \"reference\": \"12147673\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 35, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta) is generated by the consecutive cleavages of beta- and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2002\", \"reference\": \"12147673\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 35, \"target\": 0, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Abundant biochemical and genetic evidence suggests that presenilins are catalytic components of gamma-secretase, the protease responsible for generating the Alzheimer amyloid beta-protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2003\", \"reference\": \"12603837\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 35, \"target\": 0, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"Abundant biochemical and genetic evidence suggests that presenilins are catalytic components of gamma-secretase, the protease responsible for generating the Alzheimer amyloid beta-protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2003\", \"reference\": \"12603837\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 35, \"target\": 0, \"key\": 7}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPARγ leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Aβ neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in β-catenin and inhibition of GSK-3β is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPARγ by a potent ligand leads to β-catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPARγ activation in neuroprotection, PPARγ agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPARγ activation . A recent report demonstrates that overexpression of PPARγ in cultured cells leads to a dramatic decrease in the production of Aβ, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Aβ secretion, PPARγ protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPARγ leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Aβ neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in β-catenin and inhibition of GSK-3β is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPARγ by a potent ligand leads to β-catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPARγ activation in neuroprotection, PPARγ agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPARγ activation . A recent report demonstrates that overexpression of PPARγ in cultured cells leads to a dramatic decrease in the production of Aβ, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Aβ secretion, PPARγ protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\"}, \"annotations\": {\"Subgraph\": \"Ubiquitin degradation subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPARγ leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Aβ neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in β-catenin and inhibition of GSK-3β is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPARγ by a potent ligand leads to β-catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPARγ activation in neuroprotection, PPARγ agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPARγ activation . A recent report demonstrates that overexpression of PPARγ in cultured cells leads to a dramatic decrease in the production of Aβ, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Aβ secretion, PPARγ protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPARγ leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Aβ neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in β-catenin and inhibition of GSK-3β is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPARγ by a potent ligand leads to β-catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPARγ activation in neuroprotection, PPARγ agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPARγ activation . A recent report demonstrates that overexpression of PPARγ in cultured cells leads to a dramatic decrease in the production of Aβ, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Aβ secretion, PPARγ protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\"}, \"annotations\": {\"UserdefinedCellLine\": \"App transgenic\", \"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPARγ leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Aβ neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in β-catenin and inhibition of GSK-3β is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPARγ by a potent ligand leads to β-catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPARγ activation in neuroprotection, PPARγ agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPARγ activation . A recent report demonstrates that overexpression of PPARγ in cultured cells leads to a dramatic decrease in the production of Aβ, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Aβ secretion, PPARγ protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\"}, \"annotations\": {\"UserdefinedCellLine\": \"App transgenic\", \"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Aβ) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2011 Mar;32(3):547.e1-6. Epub 2009 Aug 6.\", \"reference\": \"19660836\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Aβ) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2011 Mar;32(3):547.e1-6. Epub 2009 Aug 6.\", \"reference\": \"19660836\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical Hypotheses2006\", \"reference\": \"16828233\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 7}, {\"relation\": \"decreases\", \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical Hypotheses2006\", \"reference\": \"16828233\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 8}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Aβ) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2011 Mar;32(3):547.e1-6. Epub 2009 Aug 6.\", \"reference\": \"19660836\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 9}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Aβ) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2011 Mar;32(3):547.e1-6. Epub 2009 Aug 6.\", \"reference\": \"19660836\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 36, \"target\": 0, \"key\": 10}, {\"relation\": \"decreases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 36, \"target\": 12, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 36, \"target\": 17, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 36, \"target\": 19, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 36, \"target\": 21, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Lysosomal dysfunction may be the earliest histological change in AD. Amyloid plaques are full of active lysosomal hydrolases, implying that plaques may originate from lysosomal rupture. Cathepsins D and E (intracellular aspartyl proteases) are considered to influence Aβ peptides generation within the endosomalâ€“lysosomal pathway because they exhibit β- and γ-secretase like-activity [32]. For this reason, the endosomalâ€“lysosomal pathway is a site for cleavage of the APP into smaller β-amyloid-containing peptides, which are then degraded by cathepsins. Inhibition of cathepsins activity causes a rapid and pronounced build-up of potentially amyloidogenic protein fragments [33]. On the other hand, a failure to degrade aggregated Abeta_42 in late endosomes or secondary lysosomes was a mechanism that contributed to intracellular accumulation of Aβ in AD. The cysteine protease cathepsin B in lysosomes degrades A peptides, especially the aggregation-prone species Abeta_42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Biochim Biophys Sin (Shanghai). 2009 Jun;41(6):437-45.\", \"reference\": \"19499146\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 37, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Lysosomal dysfunction may be the earliest histological change in AD. Amyloid plaques are full of active lysosomal hydrolases, implying that plaques may originate from lysosomal rupture. Cathepsins D and E (intracellular aspartyl proteases) are considered to influence Aβ peptides generation within the endosomalâ€“lysosomal pathway because they exhibit β- and γ-secretase like-activity [32]. For this reason, the endosomalâ€“lysosomal pathway is a site for cleavage of the APP into smaller β-amyloid-containing peptides, which are then degraded by cathepsins. Inhibition of cathepsins activity causes a rapid and pronounced build-up of potentially amyloidogenic protein fragments [33]. On the other hand, a failure to degrade aggregated Abeta_42 in late endosomes or secondary lysosomes was a mechanism that contributed to intracellular accumulation of Aβ in AD. The cysteine protease cathepsin B in lysosomes degrades A peptides, especially the aggregation-prone species Abeta_42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Biochim Biophys Sin (Shanghai). 2009 Jun;41(6):437-45.\", \"reference\": \"19499146\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 37, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The CR1 variant rs6656401 influences risk and recurrence of CAA-ICH, as well as the severity of vascular amyloid deposition\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurology. 2012 Jan 31;78(5):334-41. Epub 2012 Jan 18\", \"reference\": \"22262751\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Complement system subgraph\"}, \"source\": 38, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The CR1 variant rs6656401 influences risk and recurrence of CAA-ICH, as well as the severity of vascular amyloid deposition\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurology. 2012 Jan 31;78(5):334-41. Epub 2012 Jan 18\", \"reference\": \"22262751\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 38, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"These data indicate that PICALM, an adaptor protein involved in clathrin-mediated endocytosis, regulates APP internalization and subsequent AÃŸ generation. PICALM contributes to amyloid plaque load in brain likely via its effect on AÃŸ metabolism.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Apr 26. [Epub ahead of print]\", \"reference\": \"22539346\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These data indicate that PICALM, an adaptor protein involved in clathrin-mediated endocytosis, regulates APP internalization and subsequent AÃŸ generation. PICALM contributes to amyloid plaque load in brain likely via its effect on AÃŸ metabolism.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Apr 26. [Epub ahead of print]\", \"reference\": \"22539346\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 39, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"SORL1 regulates and traps APP in the Golgi apparatus and reduces the production of the amyloid-beta peptide (Abeta), the main component of senile plaques in AD4. Cell biology experiments have indicated that SORL1 underexperssion leads to overproduction of Abeta3 , whereas SORL1 overexpression considerably reduces cellular APP and extracellular Abeta5. Although the underlying mechanisms of SORL1 in AD and neurodegenerative processes remain unclear, a recent comprehensive review on multiple functional domains of SORL1 suggested that SORL1 can assume various characters in AD pathogenesis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Rep. 2016 Mar 21;6:23362. doi: 10.1038/srep23362.\", \"reference\": \"26996954\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 40, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Akt substrates such as mammalian target of rapamycin (mTOR; Ser2448) and decreased levels of cell-cycle inhibitors (p27kip1) are found in AD temporal cortex when compared to controls . GSK-3a has been implicated in the production of Aï¿½ peptide while increased GSK-3ï¿½ activity has been implicated in tau hyperphosphorylation and neuronal cell death\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 41, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Akt substrates such as mammalian target of rapamycin (mTOR; Ser2448) and decreased levels of cell-cycle inhibitors (p27kip1) are found in AD temporal cortex when compared to controls . GSK-3a has been implicated in the production of Aï¿½ peptide while increased GSK-3ï¿½ activity has been implicated in tau hyperphosphorylation and neuronal cell death\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 41, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 41, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 41, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 41, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Acetylcholinesterase (AChE), an enzyme involved in the hydrolysis of the neurotransmitter acetylcholine, consistently colocalizes with the amyloid deposits characteristic of Alzheimer's disease and may contribute to the generation of amyloid proteins and/or physically affect fibril assembly.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Biol. 1997 Sep 26;272(3):348-61.\", \"reference\": \"9325095\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 42, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Acetylcholinesterase (AChE), an enzyme involved in the hydrolysis of the neurotransmitter acetylcholine, consistently colocalizes with the amyloid deposits characteristic of Alzheimer's disease and may contribute to the generation of amyloid proteins and/or physically affect fibril assembly.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Biol. 1997 Sep 26;272(3):348-61.\", \"reference\": \"9325095\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 42, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" The fact that acetylcholinesterase accelerates amyloid formation and the effect is sensitive to peripherical anionic site blockers of the enzyme, suggests that specific and new EC 3.1.1.7 (acetylcholinesterase) inhibitors may well provide an attractive possibility for treating Alzheimer's disease. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS J2008\", \"reference\": \"18205831\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 42, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" The fact that acetylcholinesterase accelerates amyloid formation and the effect is sensitive to peripherical anionic site blockers of the enzyme, suggests that specific and new EC 3.1.1.7 (acetylcholinesterase) inhibitors may well provide an attractive possibility for treating Alzheimer's disease. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS J2008\", \"reference\": \"18205831\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 42, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Acetylcholinesterase (AChE), an enzyme involved in the hydrolysis of the neurotransmitter acetylcholine, consistently colocalizes with the amyloid deposits characteristic of Alzheimer's disease and may contribute to the generation of amyloid proteins and/or physically affect fibril assembly.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Biol1997\", \"reference\": \"9325095\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 42, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Acetylcholinesterase (AChE), an enzyme involved in the hydrolysis of the neurotransmitter acetylcholine, consistently colocalizes with the amyloid deposits characteristic of Alzheimer's disease and may contribute to the generation of amyloid proteins and/or physically affect fibril assembly.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Biol1997\", \"reference\": \"9325095\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 42, \"target\": 0, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"As far as the metabolism of amyloid precursor protein (APP) in late-onset sporadic Alzheimer disease is concerned, neuronal insulin receptor dysfunction may result in the intracellular accumulation of Abeta and in subsequent cellular damage.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Pharmacol. 2004 Apr 19;490(1-3):115-25\", \"reference\": \"15094078\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 43, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"As far as the metabolism of amyloid precursor protein (APP) in late-onset sporadic Alzheimer disease is concerned, neuronal insulin receptor dysfunction may result in the intracellular accumulation of Abeta and in subsequent cellular damage.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Pharmacol. 2004 Apr 19;490(1-3):115-25\", \"reference\": \"15094078\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 43, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"First, Pin1 inhibits the production of Aβ, and enhances the activity of eNOS. Second, Aβ and eNOS form a mutual inhibition system. Third, the well-balanced feedback signaling loop avoids the development of AD, HTN, and CAA by inhibiting the frequent pathological characteristics of these diseases, including Aβ deposition in cerebral microvessels and cerebral microbleeds.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\"}, \"annotations\": {\"NervousSystem\": \"Cerebrum\", \"CardiovascularSystem\": \"Microvessels\", \"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 44, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"First, Pin1 inhibits the production of Aβ, and enhances the activity of eNOS. Second, Aβ and eNOS form a mutual inhibition system. Third, the well-balanced feedback signaling loop avoids the development of AD, HTN, and CAA by inhibiting the frequent pathological characteristics of these diseases, including Aβ deposition in cerebral microvessels and cerebral microbleeds.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\"}, \"annotations\": {\"NervousSystem\": \"Cerebrum\", \"CardiovascularSystem\": \"Microvessels\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 44, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"To explore the molecular mechanism underlying AD, HTN, and CAA, we hypothesize a feedback signaling loop consisted of Pin1, eNOS, and Aβ. Pin1 and eNOS mainly inhibit Aβ deposition in cerebral vessels, cerebral microbleeds, and elevation of blood pressure, preventing the development of AD, HTN, and CAA, however, Aβ plays an opposite role and aggravates these diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\"}, \"annotations\": {\"NervousSystem\": \"Cerebrum\", \"CardiovascularSystem\": \"Microvessels\", \"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 44, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"To explore the molecular mechanism underlying AD, HTN, and CAA, we hypothesize a feedback signaling loop consisted of Pin1, eNOS, and Aβ. Pin1 and eNOS mainly inhibit Aβ deposition in cerebral vessels, cerebral microbleeds, and elevation of blood pressure, preventing the development of AD, HTN, and CAA, however, Aβ plays an opposite role and aggravates these diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\"}, \"annotations\": {\"NervousSystem\": \"Cerebrum\", \"CardiovascularSystem\": \"Microvessels\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 44, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃŸ-secretase and, thus, an increase of AÃŸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃŸ-secretase and, thus, an increase of AÃŸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃŸ-secretase and, thus, an increase of AÃŸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Response to oxidative stress\"}, \"source\": 45, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃŸ-secretase and, thus, an increase of AÃŸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Ex vivo studies show that the levels of BACE1 are tightly regulated by the ATases. Specifically, up-regulation of ATase1 and ATase2 increases the levels of BACE1 and the generation of AÃŸ while siRNA-mediated down-regulation of either transferase achieves the opposite effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Mar 9;287(11):8424-33. Epub 2012 Jan 20\", \"reference\": \"22267734\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"NervousSystem\": \"Neurons\", \"Cell\": \"microglial cell\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Ex vivo studies show that the levels of BACE1 are tightly regulated by the ATases. Specifically, up-regulation of ATase1 and ATase2 increases the levels of BACE1 and the generation of AÃŸ while siRNA-mediated down-regulation of either transferase achieves the opposite effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Mar 9;287(11):8424-33. Epub 2012 Jan 20\", \"reference\": \"22267734\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"NervousSystem\": \"Neurons\", \"Cell\": \"microglial cell\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Interestingly, Fe65 acts as a functional linker between APP and LRP (another type I membrane protein containing two NPXY endocytosis motifs) in modulating endocytic APP trafficking and AÃŸ production. A conformational change introduced by phosphorylation at Thr668 (14 amino acids proximal to the YENPTY motif) interferes with Fe65 binding to APP and facilitates BACE1 and gamma-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2008 Oct 31;283(44):29615-9. Epub 2008 Jul 23.\", \"reference\": \"18650430\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"Interestingly, Fe65 acts as a functional linker between APP and LRP (another type I membrane protein containing two NPXY endocytosis motifs) in modulating endocytic APP trafficking and AÃŸ production. A conformational change introduced by phosphorylation at Thr668 (14 amino acids proximal to the YENPTY motif) interferes with Fe65 binding to APP and facilitates BACE1 and gamma-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2008 Oct 31;283(44):29615-9. Epub 2008 Jul 23.\", \"reference\": \"18650430\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"Interestingly, Fe65 acts as a functional linker between APP and LRP (another type I membrane protein containing two NPXY endocytosis motifs) in modulating endocytic APP trafficking and AÃŸ production. A conformational change introduced by phosphorylation at Thr668 (14 amino acids proximal to the YENPTY motif) interferes with Fe65 binding to APP and facilitates BACE1 and gamma-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2008 Oct 31;283(44):29615-9. Epub 2008 Jul 23.\", \"reference\": \"18650430\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 9}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 10}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 11}, {\"relation\": \"increases\", \"evidence\": \"The level of ÃŸ-site APP-cleaving enzyme 1 (BACE1) has been documented to increase in the brains of patients with Alzheimer's disease, which has resulted in elevation of ÃŸ-amyloid (AÃŸ) peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Pharmacol. 2012 Jan 30;675(1-3):15-21. Epub 2011 Dec 7.\", \"reference\": \"22166376\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 12}, {\"relation\": \"increases\", \"evidence\": \"The level of ÃŸ-site APP-cleaving enzyme 1 (BACE1) has been documented to increase in the brains of patients with Alzheimer's disease, which has resulted in elevation of ÃŸ-amyloid (AÃŸ) peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Pharmacol. 2012 Jan 30;675(1-3):15-21. Epub 2011 Dec 7.\", \"reference\": \"22166376\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 13}, {\"relation\": \"increases\", \"evidence\": \"CerebraiiR/IGF-1 signaling in APP  metabolism. Binding of insulin and IGF-1  to their receptors leads to autophosphorylation of the -subunit of the IR/IGF-IR and recruitment of IRS-1 /-2. IRS proteins activate mainly two pathways, the PI3 kinase pathway, and the RAS-RAF-MAP kinase cascade. Tau phosphorylation via IR/IGF-1 signaling is influenced by GSK-3and  the tau phosphatase  PP2A. InterÂ­ estingly, mTOR signaling couples the activity ofPP2A and GSK-3in a way that the activities of both enzymes change always in the same direction. suggesting that only a dysregulation of either PP2A or GSK-3induces tau hyperphosphorylation. The PI3K pathway as well as the RAS-RAF-MAPK  cascade regulate different transcription  factors involved in transcriptional  regulation of metabolism and clearance of  Â­ amyloid. APP is cleaved by a.-,  -secretase (BACE-1) and y-secretase (presenilin).  - and subsequent  y-cleavage of APP leads to generation of  -amyloid 1 -4o/ t-42â€¢  IGF-1 signaling promotes a switch lrom TrkA to p75NTR expression leading to increased -secretase activity due to an upregulation of BACE-1 expression. Recent data suggest  that not only BACE-1 is int1uenced  by IRIIGF- IR mediated signals but also a.Â­ secretase activity is stimulated by the PI3K pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 14}, {\"relation\": \"increases\", \"evidence\": \"CerebraiiR/IGF-1 signaling in APP  metabolism. Binding of insulin and IGF-1  to their receptors leads to autophosphorylation of the -subunit of the IR/IGF-IR and recruitment of IRS-1 /-2. IRS proteins activate mainly two pathways, the PI3 kinase pathway, and the RAS-RAF-MAP kinase cascade. Tau phosphorylation via IR/IGF-1 signaling is influenced by GSK-3and  the tau phosphatase  PP2A. InterÂ­ estingly, mTOR signaling couples the activity ofPP2A and GSK-3in a way that the activities of both enzymes change always in the same direction. suggesting that only a dysregulation of either PP2A or GSK-3induces tau hyperphosphorylation. The PI3K pathway as well as the RAS-RAF-MAPK  cascade regulate different transcription  factors involved in transcriptional  regulation of metabolism and clearance of  Â­ amyloid. APP is cleaved by a.-,  -secretase (BACE-1) and y-secretase (presenilin).  - and subsequent  y-cleavage of APP leads to generation of  -amyloid 1 -4o/ t-42â€¢  IGF-1 signaling promotes a switch lrom TrkA to p75NTR expression leading to increased -secretase activity due to an upregulation of BACE-1 expression. Recent data suggest  that not only BACE-1 is int1uenced  by IRIIGF- IR mediated signals but also a.Â­ secretase activity is stimulated by the PI3K pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 15}, {\"relation\": \"increases\", \"evidence\": \"CerebraiiR/IGF-1 signaling in APP  metabolism. Binding of insulin and IGF-1  to their receptors leads to autophosphorylation of the -subunit of the IR/IGF-IR and recruitment of IRS-1 /-2. IRS proteins activate mainly two pathways, the PI3 kinase pathway, and the RAS-RAF-MAP kinase cascade. Tau phosphorylation via IR/IGF-1 signaling is influenced by GSK-3and  the tau phosphatase  PP2A. InterÂ­ estingly, mTOR signaling couples the activity ofPP2A and GSK-3in a way that the activities of both enzymes change always in the same direction. suggesting that only a dysregulation of either PP2A or GSK-3induces tau hyperphosphorylation. The PI3K pathway as well as the RAS-RAF-MAPK  cascade regulate different transcription  factors involved in transcriptional  regulation of metabolism and clearance of  Â­ amyloid. APP is cleaved by a.-,  -secretase (BACE-1) and y-secretase (presenilin).  - and subsequent  y-cleavage of APP leads to generation of  -amyloid 1 -4o/ t-42â€¢  IGF-1 signaling promotes a switch lrom TrkA to p75NTR expression leading to increased -secretase activity due to an upregulation of BACE-1 expression. Recent data suggest  that not only BACE-1 is int1uenced  by IRIIGF- IR mediated signals but also a.Â­ secretase activity is stimulated by the PI3K pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Nerve growth factor subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 16}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 17}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 18}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\"}, \"annotations\": {\"Subgraph\": \"Response to oxidative stress\"}, \"source\": 45, \"target\": 0, \"key\": 19}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\"}, \"annotations\": {\"Subgraph\": \"Apoptosis signaling subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 20}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\"}, \"annotations\": {\"Subgraph\": \"MAPK-ERK subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 21}, {\"relation\": \"decreases\", \"evidence\": \"In this study, we aimed to investigate the possibility of P-gp as a potential therapeutic target for Alzheimer's disease by examining the impact of P-gp up-regulation on the clearance of Aβ, a neuropathological hallmark of Alzheimer's disease.Uptake studies for-radiolabelled Aβ Approximately 10-35% decrease in Aβ intracellular accumulation was observed in cells treated with rifampicin, dexamethasone, caffeine, verapamil, hyperforin, β-estradiol and pentylenetetrazole compared with control.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharm Pharmacol. 2011 Aug;63(8):1111-8\", \"reference\": \"21718295\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 22}, {\"relation\": \"decreases\", \"evidence\": \"In this study, we aimed to investigate the possibility of P-gp as a potential therapeutic target for Alzheimer's disease by examining the impact of P-gp up-regulation on the clearance of Aβ, a neuropathological hallmark of Alzheimer's disease.Uptake studies for-radiolabelled Aβ Approximately 10-35% decrease in Aβ intracellular accumulation was observed in cells treated with rifampicin, dexamethasone, caffeine, verapamil, hyperforin, β-estradiol and pentylenetetrazole compared with control.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharm Pharmacol. 2011 Aug;63(8):1111-8\", \"reference\": \"21718295\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 23}, {\"relation\": \"increases\", \"evidence\": \"RAGE stimulates functional BACE1 expression through NFAT1 activation, resulting in more Abeta production and deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J. 2009 Aug;23(8):2639-49. doi: 10.1096/fj.08-126383. Epub 2009 Mar 30.\", \"reference\": \"19332646\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 24}, {\"relation\": \"increases\", \"evidence\": \"RAGE stimulates functional BACE1 expression through NFAT1 activation, resulting in more Abeta production and deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J. 2009 Aug;23(8):2639-49. doi: 10.1096/fj.08-126383. Epub 2009 Mar 30.\", \"reference\": \"19332646\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 25}, {\"relation\": \"increases\", \"evidence\": \"RAGE stimulates functional BACE1 expression through NFAT1 activation, resulting in more Abeta production and deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J. 2009 Aug;23(8):2639-49. doi: 10.1096/fj.08-126383. Epub 2009 Mar 30.\", \"reference\": \"19332646\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 26}, {\"relation\": \"increases\", \"evidence\": \"RAGE stimulates functional BACE1 expression through NFAT1 activation, resulting in more Abeta production and deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J. 2009 Aug;23(8):2639-49. doi: 10.1096/fj.08-126383. Epub 2009 Mar 30.\", \"reference\": \"19332646\"}, \"annotations\": {\"Subgraph\": \"T cells signaling\"}, \"source\": 45, \"target\": 0, \"key\": 27}, {\"relation\": \"increases\", \"evidence\": \"Moreover, NPD1 suppresses Abeta42 peptide shedding by down-regulating beta-secretase-1 (BACE1) while activating the ä¼ª-secretase ADAM10 and up-regulating sAPPalpha, thus shifting the cleavage of betaAPP holoenzyme from an amyloidogenic into the non-amyloidogenic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2011\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 28}, {\"relation\": \"increases\", \"evidence\": \"Moreover, NPD1 suppresses Abeta42 peptide shedding by down-regulating beta-secretase-1 (BACE1) while activating the ä¼ª-secretase ADAM10 and up-regulating sAPPalpha, thus shifting the cleavage of betaAPP holoenzyme from an amyloidogenic into the non-amyloidogenic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2011\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 29}, {\"relation\": \"increases\", \"evidence\": \"Moreover, NPD1 suppresses Abeta42 peptide shedding by down-regulating beta-secretase-1 (BACE1) while activating the ä¼ª-secretase ADAM10 and up-regulating sAPPalpha, thus shifting the cleavage of betaAPP holoenzyme from an amyloidogenic into the non-amyloidogenic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2011\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 30}, {\"relation\": \"increases\", \"evidence\": \"Moreover, NPD1 suppresses Abeta42 peptide shedding by down-regulating beta-secretase-1 (BACE1) while activating the ä¼ª-secretase ADAM10 and up-regulating sAPPalpha, thus shifting the cleavage of betaAPP holoenzyme from an amyloidogenic into the non-amyloidogenic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2011\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 31}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the alphaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Neurobiol2011\", \"reference\": \"21431475\"}, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 32}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the alphaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Neurobiol2011\", \"reference\": \"21431475\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 33}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the alphaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Neurobiol2011\", \"reference\": \"21431475\"}, \"annotations\": {\"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 34}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the alphaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Neurobiol2011\", \"reference\": \"21431475\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 35}, {\"relation\": \"increases\", \"evidence\": \"A pathogenic mutation at codons 670/671 in APP (APP Swedish) leads to enhanced cleavage at the beta-secretase scissile bond and increased Abeta formation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A1999\", \"reference\": \"10500121\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 36}, {\"relation\": \"increases\", \"evidence\": \"A pathogenic mutation at codons 670/671 in APP (APP Swedish) leads to enhanced cleavage at the beta-secretase scissile bond and increased Abeta formation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A1999\", \"reference\": \"10500121\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 37}, {\"relation\": \"increases\", \"evidence\": \"The novel transmembrane aspartic protease BACE (for Beta-site APP Cleaving Enzyme) is the beta-secretase that cleaves amyloid precursor protein to initiate beta-amyloid formation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2000\", \"reference\": \"10956649\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 38}, {\"relation\": \"increases\", \"evidence\": \"The novel transmembrane aspartic protease BACE (for Beta-site APP Cleaving Enzyme) is the beta-secretase that cleaves amyloid precursor protein to initiate beta-amyloid formation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2000\", \"reference\": \"10956649\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 39}, {\"relation\": \"increases\", \"evidence\": \"A beta is generated by the sequential intracellular cleavage of APP by beta-secretase to generate the N-terminal end of A beta, and intramembranous cleavage by gamma-secretase to generate the C-terminal end.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Histol Histopathol2002\", \"reference\": \"11813874\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 40}, {\"relation\": \"increases\", \"evidence\": \"A beta is generated by the sequential intracellular cleavage of APP by beta-secretase to generate the N-terminal end of A beta, and intramembranous cleavage by gamma-secretase to generate the C-terminal end.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Histol Histopathol2002\", \"reference\": \"11813874\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 41}, {\"relation\": \"increases\", \"evidence\": \"beta-Secretase, a beta-site amyloid precursor protein (APP) cleaving enzyme (BACE), participates in the secretion of beta-amyloid peptides (Abeta), the major components of the toxic amyloid plaques found in the brains of patients with Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Biochim Pol2004\", \"reference\": \"15218540\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 42}, {\"relation\": \"increases\", \"evidence\": \"beta-Secretase, a beta-site amyloid precursor protein (APP) cleaving enzyme (BACE), participates in the secretion of beta-amyloid peptides (Abeta), the major components of the toxic amyloid plaques found in the brains of patients with Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Biochim Pol2004\", \"reference\": \"15218540\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 43}, {\"relation\": \"increases\", \"evidence\": \"Amyloid-beta (Abeta) the primary component of the senile plaques found in Alzheimer's disease (AD) is generated by the rate-limiting cleavage of amyloid precursor protein (APP) by beta-secretase followed by gamma-secretase cleavage.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2004\", \"reference\": \"15452128\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 44}, {\"relation\": \"increases\", \"evidence\": \"Amyloid-beta (Abeta) the primary component of the senile plaques found in Alzheimer's disease (AD) is generated by the rate-limiting cleavage of amyloid precursor protein (APP) by beta-secretase followed by gamma-secretase cleavage.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2004\", \"reference\": \"15452128\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 45}, {\"relation\": \"increases\", \"evidence\": \"The generation of beta-amyloid peptides requires the enzymatic activity of the beta-site APP-cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci2004\", \"reference\": \"15465276\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 46}, {\"relation\": \"increases\", \"evidence\": \"The generation of beta-amyloid peptides requires the enzymatic activity of the beta-site APP-cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci2004\", \"reference\": \"15465276\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 47}, {\"relation\": \"increases\", \"evidence\": \"The generation of beta-amyloid peptides by proteolytical processing of the amyloid precursor protein (APP) requires the enzymatic activity of the beta-site APP cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2005\", \"reference\": \"15663471\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 48}, {\"relation\": \"increases\", \"evidence\": \"The generation of beta-amyloid peptides by proteolytical processing of the amyloid precursor protein (APP) requires the enzymatic activity of the beta-site APP cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2005\", \"reference\": \"15663471\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 49}, {\"relation\": \"increases\", \"evidence\": \"Using an assay that incorporates full-length recombinant APP as a substrate for beta-secretase (BACE), we have identified a series of compounds that inhibit APP processing, but do not affect the cleavage of peptide substrates by BACE1.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2005\", \"reference\": \"15737955\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 50}, {\"relation\": \"increases\", \"evidence\": \"Sequential processing of amyloid precursor protein (APP) by membrane-bound proteases, BACE1 and gamma-secretase, plays a crucial role in the pathogenesis of Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2005\", \"reference\": \"15824102\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 51}, {\"relation\": \"increases\", \"evidence\": \"Sequential processing of amyloid precursor protein (APP) by membrane-bound proteases, BACE1 and gamma-secretase, plays a crucial role in the pathogenesis of Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2005\", \"reference\": \"15824102\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 52}, {\"relation\": \"increases\", \"evidence\": \"gamma-Secretase is a membrane protein complex that cleaves the beta-amyloid precursor protein (APP) within the transmembrane region, after prior processing by beta-secretase, producing amyloid beta peptides Abeta(40) and Abeta(42).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2005\", \"reference\": \"15890777\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 53}, {\"relation\": \"increases\", \"evidence\": \"Release of Abeta from the amyloid precursor protein (APP) requires proteolysis by the beta-site APP-cleaving enzyme (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2005\", \"reference\": \"16027115\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 54}, {\"relation\": \"increases\", \"evidence\": \"Release of Abeta from the amyloid precursor protein (APP) requires proteolysis by the beta-site APP-cleaving enzyme (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2005\", \"reference\": \"16027115\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 55}, {\"relation\": \"increases\", \"evidence\": \"Abeta is produced from beta-amyloid precursor protein (APP) by beta-secretase and gamma-secretase. beta-Secretase has been identified as beta-site APP cleaving enzyme1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Res2006\", \"reference\": \"16290302\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 56}, {\"relation\": \"increases\", \"evidence\": \"Abeta is produced from beta-amyloid precursor protein (APP) by beta-secretase and gamma-secretase. beta-Secretase has been identified as beta-site APP cleaving enzyme1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Res2006\", \"reference\": \"16290302\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 57}, {\"relation\": \"increases\", \"evidence\": \"The beta-amyloid peptide (Abeta) is a major component of the Alzheimer's disease (AD)-associated senile plaques and is generated by sequential cleavage of the beta-amyloid precursor protein (APP) by beta-secretase and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"RNA2006\", \"reference\": \"16888322\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 58}, {\"relation\": \"increases\", \"evidence\": \"The beta-amyloid peptide (Abeta) is a major component of the Alzheimer's disease (AD)-associated senile plaques and is generated by sequential cleavage of the beta-amyloid precursor protein (APP) by beta-secretase and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"RNA2006\", \"reference\": \"16888322\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 59}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta), which is a product of the proteolytic effect of beta-secretase (BACE) on an amyloid precursor protein, is closely associated with Alzheimer's disease (AD) pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Microbiol2006\", \"reference\": \"17205046\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 60}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta), which is a product of the proteolytic effect of beta-secretase (BACE) on an amyloid precursor protein, is closely associated with Alzheimer's disease (AD) pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Microbiol2006\", \"reference\": \"17205046\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 61}, {\"relation\": \"increases\", \"evidence\": \"TNF-alpha directly stimulated beta-site APP-cleaving enzyme (BACE1) expression and enhanced beta-processing of APP in astrocytes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol2007\", \"reference\": \"17255335\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 62}, {\"relation\": \"increases\", \"evidence\": \"TNF-alpha directly stimulated beta-site APP-cleaving enzyme (BACE1) expression and enhanced beta-processing of APP in astrocytes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol2007\", \"reference\": \"17255335\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 63}, {\"relation\": \"increases\", \"evidence\": \"The peptides not only inhibited the formation of APPbeta (a soluble N-terminal fragment of APP cleaved by beta-secretase), but also significantly reduced Abeta40 production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Peptides2007\", \"reference\": \"17293005\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 64}, {\"relation\": \"increases\", \"evidence\": \"The peptides not only inhibited the formation of APPbeta (a soluble N-terminal fragment of APP cleaved by beta-secretase), but also significantly reduced Abeta40 production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Peptides2007\", \"reference\": \"17293005\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 65}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is associated with accumulation of the neurotoxic peptide amyloid-beta (Abeta), which is produced by sequential cleavage of amyloid precursor protein (APP) by the aspartyl protease beta-secretase and the presenilin-dependent protease gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17360493\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 66}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is associated with accumulation of the neurotoxic peptide amyloid-beta (Abeta), which is produced by sequential cleavage of amyloid precursor protein (APP) by the aspartyl protease beta-secretase and the presenilin-dependent protease gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17360493\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 67}, {\"relation\": \"increases\", \"evidence\": \"Such an approach limits APP processing by beta-secretase, mainly through the endocytic pathway, and overcomes some of the limitations of BACE inhibition.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurodegener Dis2007\", \"reference\": \"17536186\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 68}, {\"relation\": \"increases\", \"evidence\": \"Such an approach limits APP processing by beta-secretase, mainly through the endocytic pathway, and overcomes some of the limitations of BACE inhibition.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurodegener Dis2007\", \"reference\": \"17536186\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 69}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic processing of the amyloid precursor protein (APP) by beta-secretase, beta-site APP cleaving enzyme (BACE1), is the initial step in the production of the amyloid beta (Abeta) peptide, which is involved in the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17573534\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 70}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic processing of the amyloid precursor protein (APP) by beta-secretase, beta-site APP cleaving enzyme (BACE1), is the initial step in the production of the amyloid beta (Abeta) peptide, which is involved in the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17573534\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 71}, {\"relation\": \"increases\", \"evidence\": \"To address this issue, temporal neocortex of 27 AD and 21 non-demented control brains was examined to assess mRNA levels of APP isoforms (total APP, APP containing the Kunitz protease inhibitor domain [APP-KPI] and APP770) and APP metabolic enzymatic partners (the APP cleaving enzymes beta-secretase [BACE] and presenilin-1 [PS-1], and putative clearance molecules, low-density lipoprotein receptor protein [LRP] and apolipoprotein E [apoE]).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain Res2007\", \"reference\": \"17586478\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 72}, {\"relation\": \"increases\", \"evidence\": \"To address this issue, temporal neocortex of 27 AD and 21 non-demented control brains was examined to assess mRNA levels of APP isoforms (total APP, APP containing the Kunitz protease inhibitor domain [APP-KPI] and APP770) and APP metabolic enzymatic partners (the APP cleaving enzymes beta-secretase [BACE] and presenilin-1 [PS-1], and putative clearance molecules, low-density lipoprotein receptor protein [LRP] and apolipoprotein E [apoE]).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain Res2007\", \"reference\": \"17586478\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 73}, {\"relation\": \"increases\", \"evidence\": \"Pathologically, AD is characterized by the deposition in the brain of amyloid-beta peptides derived from proteolysis of amyloid precursor protein (APP) by beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Clin Pract Neurol2007\", \"reference\": \"17611486\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 74}, {\"relation\": \"increases\", \"evidence\": \"Pathologically, AD is characterized by the deposition in the brain of amyloid-beta peptides derived from proteolysis of amyloid precursor protein (APP) by beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Clin Pract Neurol2007\", \"reference\": \"17611486\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 75}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated by proteolytic processing of amyloid precursor protein (APP) via beta and gamma-secretases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res2007\", \"reference\": \"17908048\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 76}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated by proteolytic processing of amyloid precursor protein (APP) via beta and gamma-secretases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res2007\", \"reference\": \"17908048\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 77}, {\"relation\": \"increases\", \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"17995932\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 78}, {\"relation\": \"increases\", \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"17995932\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 79}, {\"relation\": \"increases\", \"evidence\": \"Quetiapine also decreased brain Abeta peptides, beta-secretase activity and expression, and the level of C99 (an APP C-terminal fragment following cleavage by beta-secretase) in the transgenic mice.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging2009\", \"reference\": \"18079026\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 80}, {\"relation\": \"increases\", \"evidence\": \"Quetiapine also decreased brain Abeta peptides, beta-secretase activity and expression, and the level of C99 (an APP C-terminal fragment following cleavage by beta-secretase) in the transgenic mice.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging2009\", \"reference\": \"18079026\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 81}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic cleavage of amyloid precursor protein by beta-secretase (BACE-1) and gamma-secretase leads to formation of beta-amyloid (A beta) a key component of amyloid plaques, which are considered the hallmark of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bioorg Med Chem Lett2008\", \"reference\": \"18162398\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 82}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic cleavage of amyloid precursor protein by beta-secretase (BACE-1) and gamma-secretase leads to formation of beta-amyloid (A beta) a key component of amyloid plaques, which are considered the hallmark of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bioorg Med Chem Lett2008\", \"reference\": \"18162398\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 83}, {\"relation\": \"increases\", \"evidence\": \"Here we describe that the two enzymatic activities responsible for Abeta production, beta-secretase and gamma-secretase, are inhibited in parallel by cholesterol reduction.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2008\", \"reference\": \"18308724\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 84}, {\"relation\": \"increases\", \"evidence\": \"Here we describe that the two enzymatic activities responsible for Abeta production, beta-secretase and gamma-secretase, are inhibited in parallel by cholesterol reduction.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2008\", \"reference\": \"18308724\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 85}, {\"relation\": \"increases\", \"evidence\": \"There was also an induced expression of the key APP processing enzyme i.e. beta-site APP cleaving enzyme 1 in both high fat/cholesterol-fed C57BL/6 and LDLR-/- mice accompanied by an increased generation of C-terminal fragments of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"18410513\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 86}, {\"relation\": \"increases\", \"evidence\": \"There was also an induced expression of the key APP processing enzyme i.e. beta-site APP cleaving enzyme 1 in both high fat/cholesterol-fed C57BL/6 and LDLR-/- mice accompanied by an increased generation of C-terminal fragments of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"18410513\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 87}, {\"relation\": \"increases\", \"evidence\": \"The study provides new cell-based assays for the profiling of small molecule inhibitors of QC and points to conspicuous differences in processing of APP depending on sequence at the beta-secretase cleavage site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry2008\", \"reference\": \"18570439\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 88}, {\"relation\": \"increases\", \"evidence\": \"The study provides new cell-based assays for the profiling of small molecule inhibitors of QC and points to conspicuous differences in processing of APP depending on sequence at the beta-secretase cleavage site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry2008\", \"reference\": \"18570439\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 89}, {\"relation\": \"increases\", \"evidence\": \"The Abeta peptide results from cleavage of APP initially by BACE1 to produce the C99 fragment and releases soluble APPbeta (sAPPbeta); C99 is then further cleaved by gamma-secretase leading to the Abeta peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Scand J Clin Lab Invest2009\", \"reference\": \"18609117\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 90}, {\"relation\": \"increases\", \"evidence\": \"Here, we show that lysines 587 and 595 of APP, which are immediately adjacent to the site of beta-secretase cleavage, are covalently pmodified by SUMO proteins in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Biophys Res Commun2008\", \"reference\": \"18675254\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 91}, {\"relation\": \"increases\", \"evidence\": \"Here, we show that lysines 587 and 595 of APP, which are immediately adjacent to the site of beta-secretase cleavage, are covalently pmodified by SUMO proteins in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Biophys Res Commun2008\", \"reference\": \"18675254\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 92}, {\"relation\": \"increases\", \"evidence\": \"Specifically, hypoxia significantly increases beta-site APP cleaving enzyme (BACE1) gene transcription through the over-expression of hypoxia inducible factor 1alpha, resulting in increased BACE1 secretase activity and amyloid-beta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2009\", \"reference\": \"19196431\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 93}, {\"relation\": \"increases\", \"evidence\": \"Specifically, hypoxia significantly increases beta-site APP cleaving enzyme (BACE1) gene transcription through the over-expression of hypoxia inducible factor 1alpha, resulting in increased BACE1 secretase activity and amyloid-beta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2009\", \"reference\": \"19196431\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 94}, {\"relation\": \"increases\", \"evidence\": \"Specifically, hypoxia significantly increases beta-site APP cleaving enzyme (BACE1) gene transcription through the over-expression of hypoxia inducible factor 1alpha, resulting in increased BACE1 secretase activity and amyloid-beta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2009\", \"reference\": \"19196431\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Hypoxia response subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 95}, {\"relation\": \"increases\", \"evidence\": \"It is produced from amyloid precursor protein (APP) by proteolytic processing dependent on the beta-site APP-cleaving enzyme 1 (BACE1) and gamma-secretase, and is degraded by a broad range of proteases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Scand J Clin Lab Invest2009\", \"reference\": \"19199126\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 96}, {\"relation\": \"increases\", \"evidence\": \"It is produced from amyloid precursor protein (APP) by proteolytic processing dependent on the beta-site APP-cleaving enzyme 1 (BACE1) and gamma-secretase, and is degraded by a broad range of proteases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Scand J Clin Lab Invest2009\", \"reference\": \"19199126\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 97}, {\"relation\": \"increases\", \"evidence\": \"Three-dimensional quantitative structure-activity relationship (3D-QSAR) pmodels were developed based on comparative molecular field analysis (CoMFA) and comparative molecular similarity indices analysis (CoMSIA), on a series of 43 hydroxyethylamine derivatives, acting as potent inhibitors of beta-site amyloid precursor protein (APP) cleavage enzyme (BACE-1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Divers2010\", \"reference\": \"19330459\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 98}, {\"relation\": \"increases\", \"evidence\": \"Three-dimensional quantitative structure-activity relationship (3D-QSAR) pmodels were developed based on comparative molecular field analysis (CoMFA) and comparative molecular similarity indices analysis (CoMSIA), on a series of 43 hydroxyethylamine derivatives, acting as potent inhibitors of beta-site amyloid precursor protein (APP) cleavage enzyme (BACE-1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Divers2010\", \"reference\": \"19330459\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 99}, {\"relation\": \"increases\", \"evidence\": \"beta-Site amyloid precursor protein (APP) cleaving enzyme 1 (BACE1) is a membrane-bound protease that is essential for the production of beta-amyloid protein (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci Res2009\", \"reference\": \"19405102\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 100}, {\"relation\": \"increases\", \"evidence\": \"beta-Site amyloid precursor protein (APP) cleaving enzyme 1 (BACE1) is a membrane-bound protease that is essential for the production of beta-amyloid protein (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci Res2009\", \"reference\": \"19405102\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 101}, {\"relation\": \"increases\", \"evidence\": \"Expression levels of the amyloid precursor protein (APP) and beta-site amyloid (Abeta) cleaving enzyme 1 (BACE1) have been implicated in Alzheimer disease (AD) progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Hum Mutat2009\", \"reference\": \"19462468\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 102}, {\"relation\": \"increases\", \"evidence\": \"Expression levels of the amyloid precursor protein (APP) and beta-site amyloid (Abeta) cleaving enzyme 1 (BACE1) have been implicated in Alzheimer disease (AD) progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Hum Mutat2009\", \"reference\": \"19462468\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 103}, {\"relation\": \"increases\", \"evidence\": \" The effect of nicotine (2 mg/(kg day)) on Abeta-induced spatial learning and memory impairments was assessed by evaluation of performance in the radial arm water maze (RAWM), in vivo electrophysiological recordings of early-phase long-term potentiation (E-LTP) in urethane-anesthetized rats, and immunoblot analysis to determine changes in the levels of beta-site amyloid precursor protein (APP)-cleaving enzyme (BACE), Abeta and memory-related protei\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging2011\", \"reference\": \"19464074\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 104}, {\"relation\": \"increases\", \"evidence\": \" The effect of nicotine (2 mg/(kg day)) on Abeta-induced spatial learning and memory impairments was assessed by evaluation of performance in the radial arm water maze (RAWM), in vivo electrophysiological recordings of early-phase long-term potentiation (E-LTP) in urethane-anesthetized rats, and immunoblot analysis to determine changes in the levels of beta-site amyloid precursor protein (APP)-cleaving enzyme (BACE), Abeta and memory-related protei\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging2011\", \"reference\": \"19464074\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 105}, {\"relation\": \"increases\", \"evidence\": \"Beta-site amyloid precursor protein cleaving enzyme (BACE1) is the rate-limiting enzyme for production of beta-amyloid peptides (Abeta), which are proposed to drive the pathological changes found in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med2009\", \"reference\": \"19669607\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 106}, {\"relation\": \"increases\", \"evidence\": \"Beta-site amyloid precursor protein cleaving enzyme (BACE1) is the rate-limiting enzyme for production of beta-amyloid peptides (Abeta), which are proposed to drive the pathological changes found in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med2009\", \"reference\": \"19669607\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 107}, {\"relation\": \"increases\", \"evidence\": \"Core CSF biomarkers include secreted Abeta and amyloid precursor protein (APP) isoforms, Abeta oligomers and beta-site APP-cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Gerontol2010\", \"reference\": \"19698775\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 108}, {\"relation\": \"increases\", \"evidence\": \"Core CSF biomarkers include secreted Abeta and amyloid precursor protein (APP) isoforms, Abeta oligomers and beta-site APP-cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Gerontol2010\", \"reference\": \"19698775\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 109}, {\"relation\": \"increases\", \"evidence\": \" In the amyloidogenic pathway, a small proportion of APP is cleaved by beta- and gamma-secretases, known as beta-site APP-cleaving enzyme 1 (BACE1) and presenilin, respectively, leading to the secretion of Abeta peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J2010\", \"reference\": \"19729516\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 110}, {\"relation\": \"increases\", \"evidence\": \" In the amyloidogenic pathway, a small proportion of APP is cleaved by beta- and gamma-secretases, known as beta-site APP-cleaving enzyme 1 (BACE1) and presenilin, respectively, leading to the secretion of Abeta peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J2010\", \"reference\": \"19729516\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 111}, {\"relation\": \"increases\", \"evidence\": \"We have recently demonstrated that bis(7)-Cognitin, a promising multifunctional anti-Alzheimer's dimer, can remarkably reduce the generation of amyloid beta peptide (Abeta) by inhibiting beta-secretase (BACE-1) and activating alpha-secretase activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Pharmacol2009\", \"reference\": \"19765582\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 112}, {\"relation\": \"increases\", \"evidence\": \"We have recently demonstrated that bis(7)-Cognitin, a promising multifunctional anti-Alzheimer's dimer, can remarkably reduce the generation of amyloid beta peptide (Abeta) by inhibiting beta-secretase (BACE-1) and activating alpha-secretase activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Pharmacol2009\", \"reference\": \"19765582\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 113}, {\"relation\": \"increases\", \"evidence\": \"A beta is generated upon the sequential proteolytic cleavage of transmembrane amyloid precursor protein (APP) by two membrane-bound proteases, beta-secretase (BACE1) and the gamma-secretase complex comprising presenilin 1 (PS1), nicastrin, APH-1 and PEN-2.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci Res2010\", \"reference\": \"19885829\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 114}, {\"relation\": \"increases\", \"evidence\": \"A beta is generated upon the sequential proteolytic cleavage of transmembrane amyloid precursor protein (APP) by two membrane-bound proteases, beta-secretase (BACE1) and the gamma-secretase complex comprising presenilin 1 (PS1), nicastrin, APH-1 and PEN-2.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci Res2010\", \"reference\": \"19885829\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 115}, {\"relation\": \"increases\", \"evidence\": \"In Alzheimer disease, this feedback loop is disrupted, and the increased level of Abeta oligomers bind to PrP(C) and prevent it from regulating BACE1 activity. PrP(C) interacts with and inhibits the beta-secretase BACE1, the rate-limiting enzyme in the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prion2009\", \"reference\": \"19887909\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 116}, {\"relation\": \"increases\", \"evidence\": \"In Alzheimer disease, this feedback loop is disrupted, and the increased level of Abeta oligomers bind to PrP(C) and prevent it from regulating BACE1 activity. PrP(C) interacts with and inhibits the beta-secretase BACE1, the rate-limiting enzyme in the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prion2009\", \"reference\": \"19887909\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 117}, {\"relation\": \"increases\", \"evidence\": \"The beta-site APP cleaving enzyme (BACE1) is responsible for the first step in the production of the beta-amyloid protein of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2010\", \"reference\": \"20067575\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 118}, {\"relation\": \"increases\", \"evidence\": \"The beta-site APP cleaving enzyme (BACE1) is responsible for the first step in the production of the beta-amyloid protein of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2010\", \"reference\": \"20067575\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 119}, {\"relation\": \"increases\", \"evidence\": \"beta-Site amyloid precursor protein cleaving enzyme 1 (BACE1) initiates amyloid-beta (Abeta) generation that is central to the pathophysiology of Alzheimer's disease (AD). Therefore, lowering Abeta levels by BACE1 manipulations represents a key therapeutic strategy, but it remains unclear whether partial inhibition of BACE1, as expected for AD treatments, can improve memory deficits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2010\", \"reference\": \"20089133\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 120}, {\"relation\": \"increases\", \"evidence\": \"beta-Site amyloid precursor protein cleaving enzyme 1 (BACE1) initiates amyloid-beta (Abeta) generation that is central to the pathophysiology of Alzheimer's disease (AD). Therefore, lowering Abeta levels by BACE1 manipulations represents a key therapeutic strategy, but it remains unclear whether partial inhibition of BACE1, as expected for AD treatments, can improve memory deficits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2010\", \"reference\": \"20089133\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 121}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated from AbetaPP through an initial cleavage by the beta-secretase, BACE-1, which results in the generation of the soluble AbetaPPbeta fragment (sAbetaPPbeta) and the membrane bound C-terminal fragment beta (CTFbeta or C99). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2010\", \"reference\": \"20157255\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 122}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated from AbetaPP through an initial cleavage by the beta-secretase, BACE-1, which results in the generation of the soluble AbetaPPbeta fragment (sAbetaPPbeta) and the membrane bound C-terminal fragment beta (CTFbeta or C99). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2010\", \"reference\": \"20157255\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 123}, {\"relation\": \"increases\", \"evidence\": \"We previously indicated that amyloid beta (Abeta) augments protein levels of beta-site amyloid precursor protein cleaving enzyme-1 (BACE-1) through oxidative stress.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Biophys Res Commun2010\", \"reference\": \"20171164\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 124}, {\"relation\": \"increases\", \"evidence\": \"We previously indicated that amyloid beta (Abeta) augments protein levels of beta-site amyloid precursor protein cleaving enzyme-1 (BACE-1) through oxidative stress.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Biophys Res Commun2010\", \"reference\": \"20171164\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 125}, {\"relation\": \"increases\", \"evidence\": \"Familial Alzheimer's disease mutations in presenilin 1 do not alter levels of the secreted amyloid-beta protein precursor generated by beta-secretase cleavage.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res2010\", \"reference\": \"20205669\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 126}, {\"relation\": \"increases\", \"evidence\": \"Abeta is derived from a type I transmembrane protein, amyloid precursor protein (APP), by the sequential proteolytic events mediated by beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochim Biophys Acta2010\", \"reference\": \"20303415\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 127}, {\"relation\": \"increases\", \"evidence\": \"Abeta is derived from a type I transmembrane protein, amyloid precursor protein (APP), by the sequential proteolytic events mediated by beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochim Biophys Acta2010\", \"reference\": \"20303415\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 128}, {\"relation\": \"increases\", \"evidence\": \"BACKGROUND: A central feature of Alzheimer's disease is the cleavage of the amyloid precursor protein (APP) to form beta-amyloid peptide (Abeta) by the beta-secretase and gamma-secretase enzymes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Brain2010\", \"reference\": \"20409323\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 129}, {\"relation\": \"increases\", \"evidence\": \"BACKGROUND: A central feature of Alzheimer's disease is the cleavage of the amyloid precursor protein (APP) to form beta-amyloid peptide (Abeta) by the beta-secretase and gamma-secretase enzymes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Brain2010\", \"reference\": \"20409323\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 130}, {\"relation\": \"increases\", \"evidence\": \"Basic understanding of the activities of the amyloid beta peptide (Abeta) and associated proteins such as beta-site APP-cleaving enzyme 1 (BACE1) is necessary to develop effective medical responses to AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol2010\", \"reference\": \"20451519\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 131}, {\"relation\": \"increases\", \"evidence\": \"Basic understanding of the activities of the amyloid beta peptide (Abeta) and associated proteins such as beta-site APP-cleaving enzyme 1 (BACE1) is necessary to develop effective medical responses to AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol2010\", \"reference\": \"20451519\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 132}, {\"relation\": \"increases\", \"evidence\": \"Cleavage of APP by BACE1 is the first proteolytic step in the production of amyloid-beta (Abeta), which accumulates in senile plaques in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol2010\", \"reference\": \"20685197\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 133}, {\"relation\": \"increases\", \"evidence\": \"Cleavage of APP by BACE1 is the first proteolytic step in the production of amyloid-beta (Abeta), which accumulates in senile plaques in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol2010\", \"reference\": \"20685197\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 134}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated from the beta-amyloid precursor protein (APP) through the proteolysis of beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett2010\", \"reference\": \"20727383\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 135}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated from the beta-amyloid precursor protein (APP) through the proteolysis of beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett2010\", \"reference\": \"20727383\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 136}, {\"relation\": \"increases\", \"evidence\": \"The Swedish mutation (K595N/M596L) of amyloid precursor protein (APP-swe) has been known to increase abnormal cleavage of cellular APP by Beta-secretase (BACE), which causes tau protein hyperphosphorylation and early-onset Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMB Rep2010\", \"reference\": \"21034535\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 137}, {\"relation\": \"increases\", \"evidence\": \"The Swedish mutation (K595N/M596L) of amyloid precursor protein (APP-swe) has been known to increase abnormal cleavage of cellular APP by Beta-secretase (BACE), which causes tau protein hyperphosphorylation and early-onset Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMB Rep2010\", \"reference\": \"21034535\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 138}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the betaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Neurobiol2011\", \"reference\": \"21431475\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 139}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the betaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Neurobiol2011\", \"reference\": \"21431475\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 140}, {\"relation\": \"increases\", \"evidence\": \"Amyloid-beta peptide (Abeta) is generated by sequential cleavage of the amyloid precursor protein (APP) by beta-site amyloid precursor protein cleaving enzyme 1 (beta-secretase, or BACE1) and gamma-secretase. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci Res2011\", \"reference\": \"21433051\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 141}, {\"relation\": \"increases\", \"evidence\": \"Amyloid-beta peptide (Abeta) is generated by sequential cleavage of the amyloid precursor protein (APP) by beta-site amyloid precursor protein cleaving enzyme 1 (beta-secretase, or BACE1) and gamma-secretase. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci Res2011\", \"reference\": \"21433051\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 142}, {\"relation\": \"increases\", \"evidence\": \"To release the amyloidogenic peptide A beta from the Alzheimer amyloid precursor protein (APP), two secretases act sequentially: first, beta-secretase cleaves close to the membrane within the ectodomain and then gamma-secretase cuts within the transmembrane domain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Med1998\", \"reference\": \"9623986\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 143}, {\"relation\": \"increases\", \"evidence\": \"To release the amyloidogenic peptide A beta from the Alzheimer amyloid precursor protein (APP), two secretases act sequentially: first, beta-secretase cleaves close to the membrane within the ectodomain and then gamma-secretase cuts within the transmembrane domain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Med1998\", \"reference\": \"9623986\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 144}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic processing of APP by beta-secretase, on the other hand, exposes the N-terminus of beta-amyloid, which is liberated after gamma-secretase cleavage at the variable amyloid C-terminus.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol1998\", \"reference\": \"9775403\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 145}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic processing of APP by beta-secretase, on the other hand, exposes the N-terminus of beta-amyloid, which is liberated after gamma-secretase cleavage at the variable amyloid C-terminus.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol1998\", \"reference\": \"9775403\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 146}, {\"relation\": \"increases\", \"evidence\": \"Pulse-chase experiments revealed beta-secretase cleavage from immature full-length amyloid precursor protein harboring the Swedish mutation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Neurosci1998\", \"reference\": \"10096041\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 147}, {\"relation\": \"increases\", \"evidence\": \"Pulse-chase experiments revealed beta-secretase cleavage from immature full-length amyloid precursor protein harboring the Swedish mutation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Neurosci1998\", \"reference\": \"10096041\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 148}, {\"relation\": \"increases\", \"evidence\": \"Solubilized Asp2 protein cleaves a synthetic APP peptide substrate at the beta-secretase site, and the rate of cleavage is increased tenfold by a mutation associated with early-onset Alzheimer's disease in Sweden.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature1999\", \"reference\": \"10591213\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 149}, {\"relation\": \"increases\", \"evidence\": \"Solubilized Asp2 protein cleaves a synthetic APP peptide substrate at the beta-secretase site, and the rate of cleavage is increased tenfold by a mutation associated with early-onset Alzheimer's disease in Sweden.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature1999\", \"reference\": \"10591213\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 150}, {\"relation\": \"increases\", \"evidence\": \"Transient expression of Asp 2 in cells expressing APP causes an increase in the secretion of the N-terminal fragment of APP and an increase in the cell-associated C-terminal beta-secretase APP fragment.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Cell Neurosci1999\", \"reference\": \"10656250\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 151}, {\"relation\": \"increases\", \"evidence\": \"Transient expression of Asp 2 in cells expressing APP causes an increase in the secretion of the N-terminal fragment of APP and an increase in the cell-associated C-terminal beta-secretase APP fragment.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Cell Neurosci1999\", \"reference\": \"10656250\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 152}, {\"relation\": \"increases\", \"evidence\": \"Recombinant memapsin 2 specifically hydrolyzed peptides derived from the beta-secretase site of both the wild-type and Swedish mutant beta-amyloid precursor protein (APP) with over 60-fold increase of catalytic efficiency for the latter. Human aspartic protease memapsin 2 cleaves the beta-secretase site of beta-amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2000\", \"reference\": \"10677483\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 153}, {\"relation\": \"increases\", \"evidence\": \"Recombinant memapsin 2 specifically hydrolyzed peptides derived from the beta-secretase site of both the wild-type and Swedish mutant beta-amyloid precursor protein (APP) with over 60-fold increase of catalytic efficiency for the latter. Human aspartic protease memapsin 2 cleaves the beta-secretase site of beta-amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2000\", \"reference\": \"10677483\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 154}, {\"relation\": \"increases\", \"evidence\": \"IFNgamma in combination with TNFalpha or IL-1beta seems to trigger Abeta production by supporting beta-secretase cleavage of the immature APP molecule.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis2000\", \"reference\": \"11114266\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 155}, {\"relation\": \"increases\", \"evidence\": \"IFNgamma in combination with TNFalpha or IL-1beta seems to trigger Abeta production by supporting beta-secretase cleavage of the immature APP molecule.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis2000\", \"reference\": \"11114266\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 156}, {\"relation\": \"increases\", \"evidence\": \"Amyloid precursor protein/P-selectin, which is sorted from early to late endosomes, undergoes significantly less alpha-secretase cleavage, and more beta-secretase cleavage, than amyloid precursor protein/P-selectin768A, a mutant that recycles more efficiently to the cell surface.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Traffic2001\", \"reference\": \"11737828\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 157}, {\"relation\": \"increases\", \"evidence\": \"Amyloid precursor protein/P-selectin, which is sorted from early to late endosomes, undergoes significantly less alpha-secretase cleavage, and more beta-secretase cleavage, than amyloid precursor protein/P-selectin768A, a mutant that recycles more efficiently to the cell surface.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Traffic2001\", \"reference\": \"11737828\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 158}, {\"relation\": \"increases\", \"evidence\": \"Homeostatic APPsbetaswe levels with aging suggest that progressive amyloid deposition in brain results not from increased beta-secretase cleavage of APP but from impaired Abeta/amyloid clearance mechanisms.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol2002\", \"reference\": \"11839594\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 159}, {\"relation\": \"increases\", \"evidence\": \"Homeostatic APPsbetaswe levels with aging suggest that progressive amyloid deposition in brain results not from increased beta-secretase cleavage of APP but from impaired Abeta/amyloid clearance mechanisms.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol2002\", \"reference\": \"11839594\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 160}, {\"relation\": \"increases\", \"evidence\": \"ELISA analysis of beta-secretase cleavage of the Swedish amyloid precursor protein in the secretory and endocytic pathways.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2002\", \"reference\": \"11953452\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 161}, {\"relation\": \"increases\", \"evidence\": \"ELISA analysis of beta-secretase cleavage of the Swedish amyloid precursor protein in the secretory and endocytic pathways.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2002\", \"reference\": \"11953452\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 162}, {\"relation\": \"increases\", \"evidence\": \"beta-Site APP-cleaving enzyme (BACE) initiates the processing of the amyloid precursor protein (APP) leading to the generation of beta-amyloid, the main component of Alzheimer's disease senile plaques.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2002\", \"reference\": \"11953458\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 163}, {\"relation\": \"increases\", \"evidence\": \"beta-Site APP-cleaving enzyme (BACE) initiates the processing of the amyloid precursor protein (APP) leading to the generation of beta-amyloid, the main component of Alzheimer's disease senile plaques.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2002\", \"reference\": \"11953458\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 164}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase cleavage represents the first step in the generation of Abeta polypeptides and initiates the amyloid cascade that leads to neurodegeneration in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Ann Neurol2002\", \"reference\": \"12112088\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 165}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase cleavage represents the first step in the generation of Abeta polypeptides and initiates the amyloid cascade that leads to neurodegeneration in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Ann Neurol2002\", \"reference\": \"12112088\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 166}, {\"relation\": \"increases\", \"evidence\": \"We show that, in Sf9 cells, BACE performs the expected beta-secretase cleavage of APP, generating C99.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2002\", \"reference\": \"12423249\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 167}, {\"relation\": \"increases\", \"evidence\": \"We show that, in Sf9 cells, BACE performs the expected beta-secretase cleavage of APP, generating C99.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2002\", \"reference\": \"12423249\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 168}, {\"relation\": \"increases\", \"evidence\": \"Our recent knockout studies show that BACE1 is critical for Abeta generation, but the knockout mice show an otherwise normal phenotype, raising the possibility that therapeutic BACE1 inhibition could be accomplished without major mechanism based toxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging2002\", \"reference\": \"12470797\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 169}, {\"relation\": \"increases\", \"evidence\": \"Our recent knockout studies show that BACE1 is critical for Abeta generation, but the knockout mice show an otherwise normal phenotype, raising the possibility that therapeutic BACE1 inhibition could be accomplished without major mechanism based toxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging2002\", \"reference\": \"12470797\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 170}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartic protease that cleaves the amyloid precursor protein (APP) at the beta-secretase site, a critical step in the Alzheimer disease pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2003\", \"reference\": \"12473667\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 171}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartic protease that cleaves the amyloid precursor protein (APP) at the beta-secretase site, a critical step in the Alzheimer disease pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2003\", \"reference\": \"12473667\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 172}, {\"relation\": \"increases\", \"evidence\": \"APP is cleaved either by beta-secretase or by alpha-secretase to initiate amyloidogenic (release of A beta) or nonamyloidogenic processing of APP, respectively.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2003\", \"reference\": \"12515826\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 173}, {\"relation\": \"increases\", \"evidence\": \"APP is cleaved either by beta-secretase or by alpha-secretase to initiate amyloidogenic (release of A beta) or nonamyloidogenic processing of APP, respectively.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2003\", \"reference\": \"12515826\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 174}, {\"relation\": \"increases\", \"evidence\": \"Employing a superior BACE1 cleavage sequence to probe cellular APP processing.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2003\", \"reference\": \"12603825\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 175}, {\"relation\": \"increases\", \"evidence\": \"Employing a superior BACE1 cleavage sequence to probe cellular APP processing.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2003\", \"reference\": \"12603825\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 176}, {\"relation\": \"increases\", \"evidence\": \"An integral membrane aspartyl protease, BACE, is responsible for beta-secretase processing of the beta-amyloid precursor protein (APP) to the large secreted sAPPbeta and membrane-bound CTFbeta of 99 residues.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Neurosci2003\", \"reference\": \"14501002\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 177}, {\"relation\": \"increases\", \"evidence\": \"An integral membrane aspartyl protease, BACE, is responsible for beta-secretase processing of the beta-amyloid precursor protein (APP) to the large secreted sAPPbeta and membrane-bound CTFbeta of 99 residues.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Neurosci2003\", \"reference\": \"14501002\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 178}, {\"relation\": \"increases\", \"evidence\": \"Here we report evidence that heparan sulfate (HS) interacts with beta-site APP-cleaving enzyme (BACE) 1 and regulates its cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2003\", \"reference\": \"14530380\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 179}, {\"relation\": \"increases\", \"evidence\": \"Here we report evidence that heparan sulfate (HS) interacts with beta-site APP-cleaving enzyme (BACE) 1 and regulates its cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2003\", \"reference\": \"14530380\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 180}, {\"relation\": \"increases\", \"evidence\": \"Together, these results suggest that T668 phosphorylation may facilitate the BACE1 cleavage of APP to increase Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2003\", \"reference\": \"14557249\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 181}, {\"relation\": \"increases\", \"evidence\": \"Together, these results suggest that T668 phosphorylation may facilitate the BACE1 cleavage of APP to increase Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2003\", \"reference\": \"14557249\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 182}, {\"relation\": \"increases\", \"evidence\": \"Our results indicate that BACE1 siRNA specifically impacts on beta-cleavage of APP and may be a potential therapeutic approach for treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2004\", \"reference\": \"14600149\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 183}, {\"relation\": \"increases\", \"evidence\": \"Our results indicate that BACE1 siRNA specifically impacts on beta-cleavage of APP and may be a potential therapeutic approach for treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2004\", \"reference\": \"14600149\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 184}, {\"relation\": \"increases\", \"evidence\": \"The cerebral deposition of amyloid beta-peptide (Abeta) is a major factor in the etiology of Alzheimer's disease. beta-Secretase (BACE) initiates the generation of Abeta by cleaving the amyloid precursor protein at the beta-site and is therefore a prime target for therapeutic intervention.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Anal Biochem2003\", \"reference\": \"14622952\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 185}, {\"relation\": \"increases\", \"evidence\": \"The cerebral deposition of amyloid beta-peptide (Abeta) is a major factor in the etiology of Alzheimer's disease. beta-Secretase (BACE) initiates the generation of Abeta by cleaving the amyloid precursor protein at the beta-site and is therefore a prime target for therapeutic intervention.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Anal Biochem2003\", \"reference\": \"14622952\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 186}, {\"relation\": \"increases\", \"evidence\": \"BACE1, the major beta-secretase involved in cleaving APP, has been identified as a type 1 membrane-associated aspartyl protease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Cell Biol2004\", \"reference\": \"14701757\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 187}, {\"relation\": \"increases\", \"evidence\": \"beta-site APP cleaving enzyme 1 (BACE1) is the beta-secretase enzyme required for generating pathogenic beta-amyloid (Abeta) peptides in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2004\", \"reference\": \"14715132\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 188}, {\"relation\": \"increases\", \"evidence\": \"beta-site APP cleaving enzyme 1 (BACE1) is the beta-secretase enzyme required for generating pathogenic beta-amyloid (Abeta) peptides in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2004\", \"reference\": \"14715132\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 189}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's beta-secretase (BACE1) is a membrane-bound protease that cleaves the amyloid precursor protein (APP) in the trans-Golgi network, an initial step in the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Glycoconj J2004\", \"reference\": \"14973371\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 190}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's beta-secretase (BACE1) is a membrane-bound protease that cleaves the amyloid precursor protein (APP) in the trans-Golgi network, an initial step in the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Glycoconj J2004\", \"reference\": \"14973371\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 191}, {\"relation\": \"increases\", \"evidence\": \"The activity of BACE-1 as measured by the formation of the cleavage product of amyloid beta precursor protein, transiently increased up to 48 h after injury, but returned to basal level 7 days post injury.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neural Transm2004\", \"reference\": \"15057522\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 192}, {\"relation\": \"increases\", \"evidence\": \"The activity of BACE-1 as measured by the formation of the cleavage product of amyloid beta precursor protein, transiently increased up to 48 h after injury, but returned to basal level 7 days post injury.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neural Transm2004\", \"reference\": \"15057522\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 193}, {\"relation\": \"increases\", \"evidence\": \"BACE is an aspartyl protease that cleaves the amyloid precursor protein (APP) at the beta-secretase cleavage site and is involved in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Cell Neurosci2004\", \"reference\": \"15080893\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 194}, {\"relation\": \"increases\", \"evidence\": \"BACE is an aspartyl protease that cleaves the amyloid precursor protein (APP) at the beta-secretase cleavage site and is involved in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Cell Neurosci2004\", \"reference\": \"15080893\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 195}, {\"relation\": \"increases\", \"evidence\": \"The beta-site APP-cleaving enzyme (BACE1) has been identified as the key enzyme leading to beta-amyloid formation, and cholinergic mechanisms have been shown to control APP processing.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci Res2004\", \"reference\": \"15211591\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 196}, {\"relation\": \"increases\", \"evidence\": \"The beta-site APP-cleaving enzyme (BACE1) has been identified as the key enzyme leading to beta-amyloid formation, and cholinergic mechanisms have been shown to control APP processing.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci Res2004\", \"reference\": \"15211591\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 197}, {\"relation\": \"increases\", \"evidence\": \"Because alpha-secretase and BACE-1 cleave APP within the secretory pathway, it is likely that the two enzymes compete for the APP substrate.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Neurosci2004\", \"reference\": \"15314262\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 198}, {\"relation\": \"increases\", \"evidence\": \"Because alpha-secretase and BACE-1 cleave APP within the secretory pathway, it is likely that the two enzymes compete for the APP substrate.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Neurosci2004\", \"reference\": \"15314262\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 199}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's beta-secretase (BACE1) cleaves amyloid precursor protein to produce amyloid beta-peptide, which is a crucial initiation process of the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Glycoconj J2004\", \"reference\": \"15467394\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 200}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's beta-secretase (BACE1) cleaves amyloid precursor protein to produce amyloid beta-peptide, which is a crucial initiation process of the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Glycoconj J2004\", \"reference\": \"15467394\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 201}, {\"relation\": \"increases\", \"evidence\": \"Mapping to the Down syndrome critical region (chromosome 21) and identified as a homologue of BACE1, BACE2 also cleaves amyloid precursor protein at the beta-site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Proteome Res2004\", \"reference\": \"15473697\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 202}, {\"relation\": \"increases\", \"evidence\": \"Mapping to the Down syndrome critical region (chromosome 21) and identified as a homologue of BACE1, BACE2 also cleaves amyloid precursor protein at the beta-site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Proteome Res2004\", \"reference\": \"15473697\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 203}, {\"relation\": \"increases\", \"evidence\": \"Preventing Abeta42 production with an M596I mutation (beta-), which blocks beta-secretase cleavage of APP, or by treatment with a gamma-secretase inhibitor increased the resistance of APP(FAD)-expressing cells to apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2004\", \"reference\": \"15584903\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 204}, {\"relation\": \"increases\", \"evidence\": \"Preventing Abeta42 production with an M596I mutation (beta-), which blocks beta-secretase cleavage of APP, or by treatment with a gamma-secretase inhibitor increased the resistance of APP(FAD)-expressing cells to apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2004\", \"reference\": \"15584903\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 205}, {\"relation\": \"increases\", \"evidence\": \"Although pmodest overexpression enhanced amyloid deposition, high BACE overexpression inhibited amyloid formation despite increased beta-cleavage of APP. BACE overexpression alters the subcellular processing of APP and inhibits Abeta deposition in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2005\", \"reference\": \"15642747\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 206}, {\"relation\": \"increases\", \"evidence\": \"Although pmodest overexpression enhanced amyloid deposition, high BACE overexpression inhibited amyloid formation despite increased beta-cleavage of APP. BACE overexpression alters the subcellular processing of APP and inhibits Abeta deposition in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2005\", \"reference\": \"15642747\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 207}, {\"relation\": \"increases\", \"evidence\": \"Here, we established a novel approach to regulate production of Abeta based on intracellular expression of single chain antibodies (intrabodies) raised to an epitope adjacent to the beta-secretase cleavage site of human APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2005\", \"reference\": \"15767460\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 208}, {\"relation\": \"increases\", \"evidence\": \"Here, we established a novel approach to regulate production of Abeta based on intracellular expression of single chain antibodies (intrabodies) raised to an epitope adjacent to the beta-secretase cleavage site of human APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2005\", \"reference\": \"15767460\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 209}, {\"relation\": \"increases\", \"evidence\": \"BACE1 increased APP cleavage at the beta-site and Abeta production whereas BACE2 did not.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J2005\", \"reference\": \"15857888\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 210}, {\"relation\": \"increases\", \"evidence\": \"BACE1 increased APP cleavage at the beta-site and Abeta production whereas BACE2 did not.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J2005\", \"reference\": \"15857888\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 211}, {\"relation\": \"increases\", \"evidence\": \"The aspartic protease beta-secretase (BACE) cleaves the amyloid precursor protein into a 42 residue beta-peptide, which is the principal biochemical marker of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Structure2005\", \"reference\": \"16216580\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 212}, {\"relation\": \"increases\", \"evidence\": \"The aspartic protease beta-secretase (BACE) cleaves the amyloid precursor protein into a 42 residue beta-peptide, which is the principal biochemical marker of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Structure2005\", \"reference\": \"16216580\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 213}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase [beta-site amyloid precursor protein-cleaving enzyme 1 (BACE1)] is the key rate-limiting enzyme for the production of the beta-amyloid (Abeta) peptide involved in the pathogenesis of Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2005\", \"reference\": \"16306400\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 214}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase [beta-site amyloid precursor protein-cleaving enzyme 1 (BACE1)] is the key rate-limiting enzyme for the production of the beta-amyloid (Abeta) peptide involved in the pathogenesis of Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2005\", \"reference\": \"16306400\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 215}, {\"relation\": \"increases\", \"evidence\": \"This was associated with inefficient plasminogen binding and plasmin activation, the displacement of beta-secretase (BACE) from DRMs to APP-containing membrane fractions, increased beta-cleavage of APP and high levels of Abeta peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"EMBO J2006\", \"reference\": \"16407971\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 216}, {\"relation\": \"increases\", \"evidence\": \"Heparin can promote beta-secretase cleavage of APP in neuroblastoma cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry2006\", \"reference\": \"16716081\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 217}, {\"relation\": \"increases\", \"evidence\": \"Heparin can promote beta-secretase cleavage of APP in neuroblastoma cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry2006\", \"reference\": \"16716081\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 218}, {\"relation\": \"increases\", \"evidence\": \"In AD, the specific inhibition of beta- or beta-secretase activities would decrease the production of Abeta from its precursor, in such a way that its relative concentration could be low enough to avoid the formation of aggregates.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bull Mem Acad R Med Belg2005\", \"reference\": \"16768248\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 219}, {\"relation\": \"increases\", \"evidence\": \"In AD, the specific inhibition of beta- or beta-secretase activities would decrease the production of Abeta from its precursor, in such a way that its relative concentration could be low enough to avoid the formation of aggregates.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bull Mem Acad R Med Belg2005\", \"reference\": \"16768248\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 220}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartyl protease that specifically cleaves amyloid precursor protein (APP) at the beta-secretase site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Biol2006\", \"reference\": \"16979658\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 221}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartyl protease that specifically cleaves amyloid precursor protein (APP) at the beta-secretase site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Biol2006\", \"reference\": \"16979658\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 222}, {\"relation\": \"increases\", \"evidence\": \"Using high-throughput siRNA screening technology, we assessed 15,200 genes for their role in Abeta42 secretion and identified leucine-rich repeat transmembrane 3 (LRRTM3) as a neuronal gene that promotes APP processing by BACE1. siRNAs targeting LRRTM3 inhibit the secretion of Abeta40, Abeta42, and sAPPbeta, the N-terminal APP fragment produced by BACE1 cleavage, from cultured cells and primary neurons by up to 60%, whereas overexpression increases Abeta secretion.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17098871\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 223}, {\"relation\": \"increases\", \"evidence\": \"Using high-throughput siRNA screening technology, we assessed 15,200 genes for their role in Abeta42 secretion and identified leucine-rich repeat transmembrane 3 (LRRTM3) as a neuronal gene that promotes APP processing by BACE1. siRNAs targeting LRRTM3 inhibit the secretion of Abeta40, Abeta42, and sAPPbeta, the N-terminal APP fragment produced by BACE1 cleavage, from cultured cells and primary neurons by up to 60%, whereas overexpression increases Abeta secretion.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17098871\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 224}, {\"relation\": \"increases\", \"evidence\": \"Hypoxia up-regulated beta-secretase cleavage of APP and amyloid-beta protein (Abeta) production by increasing BACE1 gene transcription and expression both in vitro and in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17121991\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 225}, {\"relation\": \"increases\", \"evidence\": \"Hypoxia up-regulated beta-secretase cleavage of APP and amyloid-beta protein (Abeta) production by increasing BACE1 gene transcription and expression both in vitro and in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17121991\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 226}, {\"relation\": \"increases\", \"evidence\": \"Hypoxia up-regulated beta-secretase cleavage of APP and amyloid-beta protein (Abeta) production by increasing BACE1 gene transcription and expression both in vitro and in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17121991\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Hypoxia response subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 227}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach for the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2007\", \"reference\": \"17156133\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 228}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach for the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2007\", \"reference\": \"17156133\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 229}, {\"relation\": \"increases\", \"evidence\": \"In addition, we show that some NF-kappaB inhibitors decrease sAPPbeta and APP-CTFbeta suggesting that they reduce the beta-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett2007\", \"reference\": \"17223266\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 230}, {\"relation\": \"increases\", \"evidence\": \"In addition, we show that some NF-kappaB inhibitors decrease sAPPbeta and APP-CTFbeta suggesting that they reduce the beta-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett2007\", \"reference\": \"17223266\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 231}, {\"relation\": \"increases\", \"evidence\": \"Beta-site amyloid precursor protein cleaving enzyme 1 (BACE1) (beta-secretase) initiates generation of beta-amyloid (Abeta), which plays an early role in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2007\", \"reference\": \"17409228\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 232}, {\"relation\": \"increases\", \"evidence\": \"Beta-site amyloid precursor protein cleaving enzyme 1 (BACE1) (beta-secretase) initiates generation of beta-amyloid (Abeta), which plays an early role in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2007\", \"reference\": \"17409228\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 233}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase is a potential target for inhibitory drugs against Alzheimer's disease as it cleaves amyloid precursor protein (APP) to form insoluble amyloid plaques and vascular deposits in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Comput Biol Chem2007\", \"reference\": \"17500040\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 234}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase is a potential target for inhibitory drugs against Alzheimer's disease as it cleaves amyloid precursor protein (APP) to form insoluble amyloid plaques and vascular deposits in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Comput Biol Chem2007\", \"reference\": \"17500040\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 235}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach to the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Anal Bioanal Chem2007\", \"reference\": \"17541560\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 236}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach to the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Anal Bioanal Chem2007\", \"reference\": \"17541560\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 237}, {\"relation\": \"increases\", \"evidence\": \"Beta-site APP-cleaving enzyme (BACE) is required for production of the Alzheimer's disease (AD)-associated Abeta protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2007\", \"reference\": \"17553422\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 238}, {\"relation\": \"increases\", \"evidence\": \"Beta-site APP-cleaving enzyme (BACE) is required for production of the Alzheimer's disease (AD)-associated Abeta protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2007\", \"reference\": \"17553422\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 239}, {\"relation\": \"increases\", \"evidence\": \"The aspartyl protease beta-site amyloid precursor protein cleaving enzyme 1 (BACE1) initiates processing of amyloid precursor protein (APP) into amyloid beta (Abeta) peptide, the major component of Alzheimer disease (AD) plaques.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2007\", \"reference\": \"17616527\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 240}, {\"relation\": \"increases\", \"evidence\": \"The aspartyl protease beta-site amyloid precursor protein cleaving enzyme 1 (BACE1) initiates processing of amyloid precursor protein (APP) into amyloid beta (Abeta) peptide, the major component of Alzheimer disease (AD) plaques.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2007\", \"reference\": \"17616527\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 241}, {\"relation\": \"increases\", \"evidence\": \"The proteolytic enzyme beta-secretase (BACE-1) produces amyloid beta (Abeta) peptide, the primary constituent of neurofibrillary plaques, implicated in Alzheimer's disease, by cleavage of the amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bioorg Med Chem Lett2008\", \"reference\": \"18068983\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 242}, {\"relation\": \"increases\", \"evidence\": \"The proteolytic enzyme beta-secretase (BACE-1) produces amyloid beta (Abeta) peptide, the primary constituent of neurofibrillary plaques, implicated in Alzheimer's disease, by cleavage of the amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bioorg Med Chem Lett2008\", \"reference\": \"18068983\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 243}, {\"relation\": \"increases\", \"evidence\": \"After inhibitor treatment, the improved memory function was accompanied by reduced amyloid plaque load, decreased Abeta40 and Abeta42, and reduced C-terminal beta-secretase fragment derived from APP by beta-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2008\", \"reference\": \"18184658\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 244}, {\"relation\": \"increases\", \"evidence\": \"Beta-amyloid (Abeta) peptides that accumulate in Alzheimer disease are generated from the beta-amyloid precursor protein (betaAPP) by cleavages by beta-secretase BACE1 and by presenilin-dependent gamma-secretase activities.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2008\", \"reference\": \"18263584\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 245}, {\"relation\": \"increases\", \"evidence\": \"Beta-amyloid (Abeta) peptides that accumulate in Alzheimer disease are generated from the beta-amyloid precursor protein (betaAPP) by cleavages by beta-secretase BACE1 and by presenilin-dependent gamma-secretase activities.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2008\", \"reference\": \"18263584\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 246}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach for the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochim Biophys Acta2008\", \"reference\": \"18295609\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 247}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach for the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochim Biophys Acta2008\", \"reference\": \"18295609\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 248}, {\"relation\": \"increases\", \"evidence\": \"In our opinion, albeit based on limited available data, a future potential therapeutic strategy is to mimic the mechanism by which the normal cellular form of the prion protein inhibits the beta-secretase beta-site amyloid precursor protein cleaving enzyme-1 (BACE1), and hence the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Expert Opin Ther Targets2008\", \"reference\": \"18479216\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 249}, {\"relation\": \"increases\", \"evidence\": \"In our opinion, albeit based on limited available data, a future potential therapeutic strategy is to mimic the mechanism by which the normal cellular form of the prion protein inhibits the beta-secretase beta-site amyloid precursor protein cleaving enzyme-1 (BACE1), and hence the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Expert Opin Ther Targets2008\", \"reference\": \"18479216\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 250}, {\"relation\": \"increases\", \"evidence\": \"BACE-1 cleavage is limiting for the production of Abeta, making it a particularly good drug target for the generation of inhibitors that lower Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets2008\", \"reference\": \"18673212\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 251}, {\"relation\": \"increases\", \"evidence\": \"One group described the poor kinetics of BACE 1 for cleaving the wild-type (WT) beta-secretase site of APP found in most AD patients.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biol Chem2008\", \"reference\": \"18979625\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 252}, {\"relation\": \"increases\", \"evidence\": \"One group described the poor kinetics of BACE 1 for cleaving the wild-type (WT) beta-secretase site of APP found in most AD patients.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biol Chem2008\", \"reference\": \"18979625\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 253}, {\"relation\": \"increases\", \"evidence\": \"These results indicate that post-translational S-palmitoylation of BACE1 is not required for APP processing, and that BACE1 can efficiently cleave APP in both raft and non-raft microdomains.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2009\", \"reference\": \"19074428\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 254}, {\"relation\": \"increases\", \"evidence\": \"Because beta-site APP-cleaving enzyme 1 (BACE1), an essential protease for Abeta production, is up-regulated in cells overexpressing RAGE and in RAGE-injected brains of Tg2576 mice, the molecular mechanisms underlying RAGE, BACE1 expression, and Abeta production were examined.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J2009\", \"reference\": \"19332646\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 255}, {\"relation\": \"increases\", \"evidence\": \"Because beta-site APP-cleaving enzyme 1 (BACE1), an essential protease for Abeta production, is up-regulated in cells overexpressing RAGE and in RAGE-injected brains of Tg2576 mice, the molecular mechanisms underlying RAGE, BACE1 expression, and Abeta production were examined.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J2009\", \"reference\": \"19332646\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 256}, {\"relation\": \"increases\", \"evidence\": \"However, unlike BRI2, the binding of BRI3 to the beta-secretase cleaved APP C-terminal fragment is negligible and BRI3 does not cause the massive accumulation of this APP fragment, suggesting that, unlike BRI2, BRI3 is a poor gamma-cleavage inhibitor.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2009\", \"reference\": \"19366692\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 257}, {\"relation\": \"increases\", \"evidence\": \"However, unlike BRI2, the binding of BRI3 to the beta-secretase cleaved APP C-terminal fragment is negligible and BRI3 does not cause the massive accumulation of this APP fragment, suggesting that, unlike BRI2, BRI3 is a poor gamma-cleavage inhibitor.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2009\", \"reference\": \"19366692\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 258}, {\"relation\": \"increases\", \"evidence\": \"BACE 1 (beta-site APP-cleaving enzyme 1 or beta-secretase), the key enzyme required for generating Abeta from the beta-amyloid precursor protein (APP), is regarded as an ideal target for AD therapeutic drug design.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2009\", \"reference\": \"19442147\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 259}, {\"relation\": \"increases\", \"evidence\": \"BACE 1 (beta-site APP-cleaving enzyme 1 or beta-secretase), the key enzyme required for generating Abeta from the beta-amyloid precursor protein (APP), is regarded as an ideal target for AD therapeutic drug design.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2009\", \"reference\": \"19442147\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 260}, {\"relation\": \"increases\", \"evidence\": \"The beta-secretase, beta-site amyloid precursor protein cleaving enzyme (BACE1; also called Asp2, memapsin 2), is the enzyme responsible for initiating Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2009\", \"reference\": \"19828790\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 261}, {\"relation\": \"increases\", \"evidence\": \"The beta-secretase, beta-site amyloid precursor protein cleaving enzyme (BACE1; also called Asp2, memapsin 2), is the enzyme responsible for initiating Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2009\", \"reference\": \"19828790\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 262}, {\"relation\": \"increases\", \"evidence\": \"BACE1 initiates the amyloidogenic processing of APP; therefore, early or active amyloidogenic loci might exhibit site-specific BACE1 and Abeta elevation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurosci2009\", \"reference\": \"20092570\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 263}, {\"relation\": \"increases\", \"evidence\": \"BACE1 initiates the amyloidogenic processing of APP; therefore, early or active amyloidogenic loci might exhibit site-specific BACE1 and Abeta elevation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurosci2009\", \"reference\": \"20092570\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 264}, {\"relation\": \"increases\", \"evidence\": \"Taken together, ATXN1 functions as a genetic risk pmodifier that contributes to AD pathogenesis through a loss-of-function mechanism by regulating beta-secretase cleavage of APP and Abeta levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2010\", \"reference\": \"20097758\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 265}, {\"relation\": \"increases\", \"evidence\": \"Taken together, ATXN1 functions as a genetic risk pmodifier that contributes to AD pathogenesis through a loss-of-function mechanism by regulating beta-secretase cleavage of APP and Abeta levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2010\", \"reference\": \"20097758\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 266}, {\"relation\": \"increases\", \"evidence\": \"Three proteases that are involved in the processing of amyloid precursor protein-alpha-secretase, beta-secretase and gamma-secretase-are of particular interest as they are central to the generation and pmodulation of amyloid-beta peptide and can be targeted by small compounds in vitro and in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Rev Neurol2010\", \"reference\": \"20139999\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 267}, {\"relation\": \"increases\", \"evidence\": \"Three proteases that are involved in the processing of amyloid precursor protein-alpha-secretase, beta-secretase and gamma-secretase-are of particular interest as they are central to the generation and pmodulation of amyloid-beta peptide and can be targeted by small compounds in vitro and in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Rev Neurol2010\", \"reference\": \"20139999\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 268}, {\"relation\": \"increases\", \"evidence\": \"Although pmoderate decreases of either gamma-secretase or BACE1 are not associated with mechanism-based toxicities, they provide only pmodest benefits in reducing Abeta in the brains of APPswe/PS1DeltaE9 mice.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Transl Med2010\", \"reference\": \"20371462\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 269}, {\"relation\": \"increases\", \"evidence\": \"Although pmoderate decreases of either gamma-secretase or BACE1 are not associated with mechanism-based toxicities, they provide only pmodest benefits in reducing Abeta in the brains of APPswe/PS1DeltaE9 mice.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Transl Med2010\", \"reference\": \"20371462\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 270}, {\"relation\": \"increases\", \"evidence\": \"Reticulons are a group of membrane-bound proteins involved in diverse cellular functions, and are suggested to act as inhibitors of beta-secretase enzyme 1 (BACE1) activity that cleaves amyloid precursor protein. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropathology2010\", \"reference\": \"20374499\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 271}, {\"relation\": \"increases\", \"evidence\": \"Here we tested the possibility of targeting the cellular environment of beta-secretase cleavage instead of the beta-secretase enzyme itself. beta-Secretase has an acidic pH optimum and cleaves the amyloid precursor protein in the acidic endosomes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2010\", \"reference\": \"20592218\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 272}, {\"relation\": \"increases\", \"evidence\": \"ABCA2 expression promoted b-secretase (BACE1) cleavage of APP not at the common Asp1 amino acid site (beta-site) of Abeta in APP but at the Glu11 site (beta'-site) to increase C89 carboxyl-terminal fragment levels (beta'-CTF/C89). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res2010\", \"reference\": \"20704561\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 273}, {\"relation\": \"increases\", \"evidence\": \"We discovered a nonpeptidic compound, TAK-070, that inhibited BACE1, a rate-limiting protease for the generation of Abeta peptides that are considered causative for Alzheimer's disease (AD), in a noncompetitive manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2010\", \"reference\": \"20720123\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 274}, {\"relation\": \"increases\", \"evidence\": \"We discovered a nonpeptidic compound, TAK-070, that inhibited BACE1, a rate-limiting protease for the generation of Abeta peptides that are considered causative for Alzheimer's disease (AD), in a noncompetitive manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2010\", \"reference\": \"20720123\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 275}, {\"relation\": \"increases\", \"evidence\": \"Western blot analyses detected increased levels of BACE1 protein and beta-site-cleavage amyloid precursor protein C-terminal fragments in plaque-bearing human and monkey cortex relative to controls. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurosci2010\", \"reference\": \"20726888\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 276}, {\"relation\": \"increases\", \"evidence\": \"beta-Site APP-cleaving enzyme 1 (BACE1) initiates amyloid-beta (Abeta) generation and thus represents a prime therapeutic target in treating Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2010\", \"reference\": \"20886088\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 277}, {\"relation\": \"increases\", \"evidence\": \"Importantly, recent evidence reveals that expression and activity levels of the beta-site APP cleaving enzyme 1 (BACE1), which initiates amyloid-beta (Abeta) production, are elevated in AD brains.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Brain2010\", \"reference\": \"21059265\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 278}, {\"relation\": \"increases\", \"evidence\": \"Importantly, recent evidence reveals that expression and activity levels of the beta-site APP cleaving enzyme 1 (BACE1), which initiates amyloid-beta (Abeta) production, are elevated in AD brains.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Brain2010\", \"reference\": \"21059265\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 279}, {\"relation\": \"increases\", \"evidence\": \"BACE1 (beta-site beta-amyloid precursor protein (APP)-cleaving enzyme 1) mediates the first proteolytic cleavage of APP, leading to amyloid beta-peptide (Abeta) production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2011\", \"reference\": \"21245145\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 280}, {\"relation\": \"increases\", \"evidence\": \"Increased beta-secretase cleavage of APP after introduction of the Swedish double mutation causes apical missorting of about 20% of beta-secretase-cleaved APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem1995\", \"reference\": \"7876155\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 281}, {\"relation\": \"increases\", \"evidence\": \"Taken together, the data suggest that the processing pathway for betaPP is similar for both betaPP-wt and betaPP-sw cells and that increased Abeta production by betaPP-sw cells arises from enhanced cleavage of mutant betaPP by beta-secretase, the as-yet unidentified enzyme(s) that cleaves at the NH2 terminus of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem1996\", \"reference\": \"8621560\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 282}, {\"relation\": \"increases\", \"evidence\": \"Taken together, the data suggest that the processing pathway for betaPP is similar for both betaPP-wt and betaPP-sw cells and that increased Abeta production by betaPP-sw cells arises from enhanced cleavage of mutant betaPP by beta-secretase, the as-yet unidentified enzyme(s) that cleaves at the NH2 terminus of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem1996\", \"reference\": \"8621560\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 283}, {\"relation\": \"increases\", \"evidence\": \"Heparin promotes beta-secretase cleavage of the Alzheimer's amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurochem Int1997\", \"reference\": \"9152995\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 284}, {\"relation\": \"increases\", \"evidence\": \"Sequential proteolytic cleavage of APP by beta-secretase and gamma-secretase liberates Abeta from APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biol Chem2010\", \"reference\": \"20731541\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 285}, {\"relation\": \"increases\", \"evidence\": \"Sequential proteolytic cleavage of APP by beta-secretase and gamma-secretase liberates Abeta from APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biol Chem2010\", \"reference\": \"20731541\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 286}, {\"relation\": \"increases\", \"evidence\": \"Formation of stable complexes between two Alzheimer's disease gene products: presenilin-2 and beta-amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Med1997\", \"reference\": \"9055862\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 287}, {\"relation\": \"increases\", \"evidence\": \"Formation of stable complexes between two Alzheimer's disease gene products: presenilin-2 and beta-amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Med1997\", \"reference\": \"9055862\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 288}, {\"relation\": \"increases\", \"evidence\": \"The novel transmembrane aspartic protease BACE (for Beta-site APP Cleaving Enzyme) is the beta-secretase that cleaves amyloid precursor protein to initiate beta-amyloid formation. As such, BACE is a prime therapeutic target for the treatment of Alzheimer's disease. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2000\", \"reference\": \"10956649\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 45, \"target\": 0, \"key\": 289}, {\"relation\": \"increases\", \"evidence\": \"beta-Secretase (BACE) carries out the first of two proteolysis steps to generate the amyloid-beta peptides that accumulate in the senile plaques in Alzheimer's disease (AD). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Sci2004\", \"reference\": \"15466887\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 290}, {\"relation\": \"increases\", \"evidence\": \"beta-Secretase (BACE) carries out the first of two proteolysis steps to generate the amyloid-beta peptides that accumulate in the senile plaques in Alzheimer's disease (AD). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Sci2004\", \"reference\": \"15466887\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 291}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartic protease that cleaves the amyloid precursor protein (APP) at the beta-secretase site, a critical step in the Alzheimer disease pathogenesis. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2003\", \"reference\": \"12473667\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 292}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartic protease that cleaves the amyloid precursor protein (APP) at the beta-secretase site, a critical step in the Alzheimer disease pathogenesis. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2003\", \"reference\": \"12473667\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 293}, {\"relation\": \"increases\", \"evidence\": \"LRRTM3 promotes processing of amyloid-precursor protein by BACE1 \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17098871\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 294}, {\"relation\": \"increases\", \"evidence\": \"The complex mediates the intramembraneous proteolysis of beta-secretase cleaved beta-amyloid precursor protein (APP) leading to the secretion of the Alzheimer's disease-associated amyloid beta-peptide (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2004\", \"reference\": \"15189355\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 295}, {\"relation\": \"increases\", \"evidence\": \"The complex mediates the intramembraneous proteolysis of beta-secretase cleaved beta-amyloid precursor protein (APP) leading to the secretion of the Alzheimer's disease-associated amyloid beta-peptide (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2004\", \"reference\": \"15189355\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 296}, {\"relation\": \"increases\", \"evidence\": \"The familial Alzheimer's disease gene product amyloid beta precursor protein (APP) is sequentially processed by beta- and gamma-secretases to generate the Abeta peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2002\", \"reference\": \"11724784\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 297}, {\"relation\": \"increases\", \"evidence\": \"The familial Alzheimer's disease gene product amyloid beta precursor protein (APP) is sequentially processed by beta- and gamma-secretases to generate the Abeta peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2002\", \"reference\": \"11724784\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 298}, {\"relation\": \"increases\", \"evidence\": \"The level of ÃŸ-site APP-cleaving enzyme 1 (BACE1) has been documented to increase in the brains of patients with Alzheimer's disease, which has resulted in elevation of ÃŸ-amyloid (AÃŸ) peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Pharmacol. 2012 Jan 30;675(1-3):15-21. Epub 2011 Dec 7.\", \"reference\": \"22166376\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 299}, {\"relation\": \"increases\", \"evidence\": \"The level of ÃŸ-site APP-cleaving enzyme 1 (BACE1) has been documented to increase in the brains of patients with Alzheimer's disease, which has resulted in elevation of ÃŸ-amyloid (AÃŸ) peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Pharmacol. 2012 Jan 30;675(1-3):15-21. Epub 2011 Dec 7.\", \"reference\": \"22166376\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 0, \"key\": 300}, {\"relation\": \"increases\", \"evidence\": \"Down regulation of DNMT results in hypomethylation of BACE1 and APP which are involved in Abeta production and causes upregulation of their protein expression; in turn SP1 transcription factor increases which finally results in Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Free Radic Biol Med. 2009 May 1;46(9):1241-9. doi: 10.1016/j.freeradbiomed.2009.02.006. Epub 2009 Feb 23.\", \"reference\": \"19245828\"}, \"annotations\": {\"Species\": \"10090\"}, \"source\": 45, \"target\": 0, \"key\": 301}, {\"relation\": \"decreases\", \"evidence\": \"Insulin modulates metabolism of β-amyloid precursor protein (APP) in neurons, decreasing the intracellular accumulation of β-amyloid (Aβ) peptides, which are pivotal in AD pathogenesis. The present study investigates whether the widely prescribed insulin-sensitizing drug, metformin (GlucophageR), affects APP metabolism and Aβ generation in various cell models. We demonstrate that metformin, at doses that lead to activation of the AMP-activated protein kinase (AMPK), significantly increases the generation of both intracellular and extracellular Aβ species\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A. 2009 Mar 10;106(10):3907-12. doi: 10.1073/pnas.0807991106. Epub 2009 Feb 23.\", \"reference\": \"19237574\"}, \"annotations\": {\"NervousSystem\": \"Neurons\", \"Subgraph\": \"Insulin signal transduction\"}, \"source\": 46, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Insulin modulates metabolism of β-amyloid precursor protein (APP) in neurons, decreasing the intracellular accumulation of β-amyloid (Aβ) peptides, which are pivotal in AD pathogenesis. The present study investigates whether the widely prescribed insulin-sensitizing drug, metformin (GlucophageR), affects APP metabolism and Aβ generation in various cell models. We demonstrate that metformin, at doses that lead to activation of the AMP-activated protein kinase (AMPK), significantly increases the generation of both intracellular and extracellular Aβ species\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A. 2009 Mar 10;106(10):3907-12. doi: 10.1073/pnas.0807991106. Epub 2009 Feb 23.\", \"reference\": \"19237574\"}, \"annotations\": {\"NervousSystem\": \"Neurons\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 46, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Insulin and insulin receptors were shown to decrease in a normal brain with aging, but increase in AD brains [29]. Several basic science studies have explored and shown the relationship between the increased insulin and AD pathology in the aspects other than Aβ degradation alone. For example, insulin increases the secretion of Aβ into extracellular space [31], stimulates tau phosphorylation to form neurofibrillary tangles, and impairs insulin signal transduction [32] and [40] (reviewed by Gasparini and Hoyer). Insulin also affected APP processing in vivo, a critical molecular step in generating Aβ, to secrete sAPP [13], [16] and [81]. In addition, Aβ reduces insulin binding to insulin receptors [95].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2006 Feb;27(2):190-8. Epub 2005 Feb 17.\", \"reference\": \"16399206\"}, \"object\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"GOCC\", \"name\": \"intracellular\"}, \"toLoc\": {\"namespace\": \"GOCC\", \"name\": \"extracellular space\"}}}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Insulin signal transduction\"}, \"source\": 46, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Insulin and insulin receptors were shown to decrease in a normal brain with aging, but increase in AD brains [29]. Several basic science studies have explored and shown the relationship between the increased insulin and AD pathology in the aspects other than Aβ degradation alone. For example, insulin increases the secretion of Aβ into extracellular space [31], stimulates tau phosphorylation to form neurofibrillary tangles, and impairs insulin signal transduction [32] and [40] (reviewed by Gasparini and Hoyer). Insulin also affected APP processing in vivo, a critical molecular step in generating Aβ, to secrete sAPP [13], [16] and [81]. In addition, Aβ reduces insulin binding to insulin receptors [95].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2006 Feb;27(2):190-8. Epub 2005 Feb 17.\", \"reference\": \"16399206\"}, \"object\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"GOCC\", \"name\": \"intracellular\"}, \"toLoc\": {\"namespace\": \"GOCC\", \"name\": \"extracellular space\"}}}, \"annotations\": {\"NervousSystem\": \"Brain\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 46, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include ÃŸ-amyloid protein (AÃŸ) generation and tau phosphorylation. In particular, AMPK may regulate AÃŸ generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating AÃŸ generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of AÃŸ. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including AÃŸ generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Akt subgraph\"}, \"source\": 47, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include ÃŸ-amyloid protein (AÃŸ) generation and tau phosphorylation. In particular, AMPK may regulate AÃŸ generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating AÃŸ generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of AÃŸ. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including AÃŸ generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 47, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include ÃŸ-amyloid protein (AÃŸ) generation and tau phosphorylation. In particular, AMPK may regulate AÃŸ generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating AÃŸ generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of AÃŸ. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including AÃŸ generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 47, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by AÃŸ (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2011 Oct 12;31(41):14820-30.\", \"reference\": \"21994399\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 48, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by AÃŸ (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2011 Oct 12;31(41):14820-30.\", \"reference\": \"21994399\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 48, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The phosphotyrosine binding domain of the neuronal protein X11alpha/mint-1 binds to the C-terminus of amyloid precursor protein (APP) and inhibits catabolism to beta-amyloid (Abeta), but the mechanism of this effect is unclear.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2004\", \"reference\": \"14756819\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 49, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The phosphotyrosine binding domain of the neuronal protein X11alpha/mint-1 binds to the C-terminus of amyloid precursor protein (APP) and inhibits catabolism to beta-amyloid (Abeta), but the mechanism of this effect is unclear.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2004\", \"reference\": \"14756819\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 49, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The neuronal adaptor X11alpha interacts with the conserved -GYENPTY- sequence in the C-terminus of amyloid precursor protein (APP) or its Swedish mutation (APPswe) to inhibit Abeta40 and Abeta42 secretion.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience2003\", \"reference\": \"12849748\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 49, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Both in vitro and in vivo, overexpression of the sortilin-related VPS10 domain containing receptor 1 (SORCS1) protein lowers amyloid-β generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Psychogeriatrics. 2012 Dec;12(4):215-25. doi: 10.1111/j.1479-8301.2012.00405.x.\", \"reference\": \"23279143\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 50, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"PEN-2 is an integral membrane protein that is a necessary component of the gamma-secretase complex, which is central in the pathogenesis of Alzheimer's disease and is also required for Notch signaling. In the absence of PEN-2, Notch signaling fails to guide normal development in Caenorhabditis elegans, and amyloid beta peptide is not generated from the amyloid precursor protein\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2003 May 30;278(22):20117-23. Epub 2003 Mar 14.\", \"reference\": \"12639958\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"UserdefinedSpecies\": \"C elegans\", \"Subgraph\": \"Notch signaling subgraph\"}, \"source\": 51, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"PEN-2 is an integral membrane protein that is a necessary component of the gamma-secretase complex, which is central in the pathogenesis of Alzheimer's disease and is also required for Notch signaling. In the absence of PEN-2, Notch signaling fails to guide normal development in Caenorhabditis elegans, and amyloid beta peptide is not generated from the amyloid precursor protein\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2003 May 30;278(22):20117-23. Epub 2003 Mar 14.\", \"reference\": \"12639958\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"UserdefinedSpecies\": \"C elegans\", \"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 51, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"PEN-2 is an integral membrane protein that is a necessary component of the gamma-secretase complex, which is central in the pathogenesis of Alzheimer's disease and is also required for Notch signaling. In the absence of PEN-2, Notch signaling fails to guide normal development in Caenorhabditis elegans, and amyloid beta peptide is not generated from the amyloid precursor protein\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2003 May 30;278(22):20117-23. Epub 2003 Mar 14.\", \"reference\": \"12639958\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"UserdefinedSpecies\": \"C elegans\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 51, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 52, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 52, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 52, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 52, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 53, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 53, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\"}, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 53, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that BACE2 contributes to Abeta production in individuals bearing the Flemish mutation, and that selective inhibition of these highly similar proteases may be feasible and therapeutically advantageous.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2000\", \"reference\": \"10931940\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 53, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that BACE2 contributes to Abeta production in individuals bearing the Flemish mutation, and that selective inhibition of these highly similar proteases may be feasible and therapeutically advantageous.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2000\", \"reference\": \"10931940\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 53, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"BACE2 cleaves the amyloid precursor protein at the beta-secretase site and is thought to contribute to amyloid beta protein production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett2002\", \"reference\": \"12052539\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 53, \"target\": 0, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"BACE2 cleaves the amyloid precursor protein at the beta-secretase site and is thought to contribute to amyloid beta protein production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett2002\", \"reference\": \"12052539\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 53, \"target\": 0, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"Mapping to the Down syndrome critical region (chromosome 21) and identified as a homologue of BACE1, BACE2 also cleaves amyloid precursor protein at the beta-site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Proteome Res2004\", \"reference\": \"15473697\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 53, \"target\": 0, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"Mapping to the Down syndrome critical region (chromosome 21) and identified as a homologue of BACE1, BACE2 also cleaves amyloid precursor protein at the beta-site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Proteome Res2004\", \"reference\": \"15473697\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 53, \"target\": 0, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \"Insulin and IGF-I may pmodulate brain levels of insulin degrading enzyme, which would also lead to an accumulation of Abeta amyloid.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neural Plast2005\", \"reference\": \"16444902\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 54, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Insulin and IGF-I may pmodulate brain levels of insulin degrading enzyme, which would also lead to an accumulation of Abeta amyloid.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neural Plast2005\", \"reference\": \"16444902\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 54, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In addition, NF-ÎºB p65 expression leads to up-regulated β-secretase cleavage and Aβ production, while non-steroidal anti-inflammatory drugs (NSAIDs) inhibited BACE1 transcriptional activation induced by strong NF-ÎºB activator tumour necrosis factor-alpha (TNF-Î±).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Neuropsychopharmacol. 2012 Feb;15(1):77-90\", \"reference\": \"21329555\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In addition, NF-ÎºB p65 expression leads to up-regulated β-secretase cleavage and Aβ production, while non-steroidal anti-inflammatory drugs (NSAIDs) inhibited BACE1 transcriptional activation induced by strong NF-ÎºB activator tumour necrosis factor-alpha (TNF-Î±).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Neuropsychopharmacol. 2012 Feb;15(1):77-90\", \"reference\": \"21329555\"}, \"annotations\": {\"Subgraph\": \"Nuclear factor Kappa beta subgraph\"}, \"source\": 55, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that overexpression of endogenous calpain inhibitor calpastatin (CAST) under the control of the calcium/calpmodulin-dependent protein kinase II promoter in APP/PS1 mice caused a remarkable decrease of amyloid plaque burdens and prevented Tau phosphorylation and the loss of synapses.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2010\", \"reference\": \"20595388\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 56, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"We found that overexpression of endogenous calpain inhibitor calpastatin (CAST) under the control of the calcium/calpmodulin-dependent protein kinase II promoter in APP/PS1 mice caused a remarkable decrease of amyloid plaque burdens and prevented Tau phosphorylation and the loss of synapses.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2010\", \"reference\": \"20595388\"}, \"annotations\": {\"Subgraph\": \"Calpastatin-calpain subgraph\"}, \"source\": 56, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2010\", \"reference\": \"20595388\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 56, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2010\", \"reference\": \"20595388\"}, \"annotations\": {\"Subgraph\": \"Calpastatin-calpain subgraph\"}, \"source\": 56, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2010\", \"reference\": \"20595388\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 56, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"In addition, we show that some NF-kappaB inhibitors decrease sAPPbeta and APP-CTFbeta suggesting that they reduce the beta-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett2007\", \"reference\": \"17223266\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 57, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In addition, we show that some NF-kappaB inhibitors decrease sAPPbeta and APP-CTFbeta suggesting that they reduce the beta-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett2007\", \"reference\": \"17223266\"}, \"annotations\": {\"Subgraph\": \"Nuclear factor Kappa beta subgraph\"}, \"source\": 57, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The role of miR-124 on the expression of ÃŸ-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the ÃŸ-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by AÃŸ neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Toxicol Lett. 2012 Feb 25;209(1):94-105. Epub 2011 Dec 8.\", \"reference\": \"22178568\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"miRNA subgraph\"}, \"source\": 58, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The role of miR-124 on the expression of ÃŸ-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the ÃŸ-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by AÃŸ neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Toxicol Lett. 2012 Feb 25;209(1):94-105. Epub 2011 Dec 8.\", \"reference\": \"22178568\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 58, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The role of miR-124 on the expression of β-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the β-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by Aβ neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Toxicol Lett. 2012 Feb 25;209(1):94-105. Epub 2011 Dec 8.\", \"reference\": \"22178568\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 58, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The role of miR-124 on the expression of β-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the β-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by Aβ neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Toxicol Lett. 2012 Feb 25;209(1):94-105. Epub 2011 Dec 8.\", \"reference\": \"22178568\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 58, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"The role of miR-124 on the expression of β-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the β-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by Aβ neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Toxicol Lett. 2012 Feb 25;209(1):94-105. Epub 2011 Dec 8.\", \"reference\": \"22178568\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 58, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that  -amyloid accumulation is the  critical event  in development of disease (I 49,  I 50].  Lots  of research has been  done  on  the formation and accumulation of Al3, however, in the  last years  the mechanism' of  amyloid clearance came   into focus. For  Al3 clearance several mechanisms are  known (Fig.  2):  i) EnzyÂ­ matic  degradation by activated microglia or  by  insulin   deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and  angiotensin converting  enzyme (ACE); ii) Receptor-mediated transport across the blood  brain  barrier (BBB) by  binding to  the  low-density  lipoprotein  receptorÂ­ related protein (LRP) either  directly or after  binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M)  to be delivered  to  peripheral sites   of  degradation, e.g.,   liver  and kidney.  (Review in [151 ]).  Concerning insulin   resistance it has  been  shown  that   IDE expression  is stimulated  by  the IR/lGF-1 R  cascade  (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Endothelin subgraph\"}, \"source\": 59, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that  -amyloid accumulation is the  critical event  in development of disease (I 49,  I 50].  Lots  of research has been  done  on  the formation and accumulation of Al3, however, in the  last years  the mechanism' of  amyloid clearance came   into focus. For  Al3 clearance several mechanisms are  known (Fig.  2):  i) EnzyÂ­ matic  degradation by activated microglia or  by  insulin   deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and  angiotensin converting  enzyme (ACE); ii) Receptor-mediated transport across the blood  brain  barrier (BBB) by  binding to  the  low-density  lipoprotein  receptorÂ­ related protein (LRP) either  directly or after  binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M)  to be delivered  to  peripheral sites   of  degradation, e.g.,   liver  and kidney.  (Review in [151 ]).  Concerning insulin   resistance it has  been  shown  that   IDE expression  is stimulated  by  the IR/lGF-1 R  cascade  (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 59, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that  -amyloid accumulation is the  critical event  in development of disease (I 49,  I 50].  Lots  of research has been  done  on  the formation and accumulation of Al3, however, in the  last years  the mechanism' of  amyloid clearance came   into focus. For  Al3 clearance several mechanisms are  known (Fig.  2):  i) EnzyÂ­ matic  degradation by activated microglia or  by  insulin   deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and  angiotensin converting  enzyme (ACE); ii) Receptor-mediated transport across the blood  brain  barrier (BBB) by  binding to  the  low-density  lipoprotein  receptorÂ­ related protein (LRP) either  directly or after  binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M)  to be delivered  to  peripheral sites   of  degradation, e.g.,   liver  and kidney.  (Review in [151 ]).  Concerning insulin   resistance it has  been  shown  that   IDE expression  is stimulated  by  the IR/lGF-1 R  cascade  (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 59, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Since Aβ is secreted extracellularly, and deposits outside the neuronal cells in the AD brain, we took an approach to screen any secreted protease(s) in neuronal and non-neuronal cell culture media for the ability to degrade Aβ. Among all secreted proteases from the cells, only IDE degraded Aβ. We found that under physiological conditions IDE is secreted at high levels from the microglial cells, and degrades Aβ extracellularly [73]. Purified IDE from rat liver and brain was shown to degrade Aβ effectively. IDE is present in the soluble fractions from human brains, and binds and degrades Aβ specifically [57] and [71]. Primary cultured neurons were also shown to clear Aβ via extracellular IDE as well as IDE on the cell surface [90]. IDE from brain homogenates degrades different forms of Aβ: Aβ40, Aβ42 and an Aβ mutant in one type of AD (Dutch Variant 1-40 Q) [61] and [71]. Aβ42 is the longer form of Aβ and more abundant in the AD brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2006 Feb;27(2):190-8. Epub 2005 Feb 17.\", \"reference\": \"16399206\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 59, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Since Aβ is secreted extracellularly, and deposits outside the neuronal cells in the AD brain, we took an approach to screen any secreted protease(s) in neuronal and non-neuronal cell culture media for the ability to degrade Aβ. Among all secreted proteases from the cells, only IDE degraded Aβ. We found that under physiological conditions IDE is secreted at high levels from the microglial cells, and degrades Aβ extracellularly [73]. Purified IDE from rat liver and brain was shown to degrade Aβ effectively. IDE is present in the soluble fractions from human brains, and binds and degrades Aβ specifically [57] and [71]. Primary cultured neurons were also shown to clear Aβ via extracellular IDE as well as IDE on the cell surface [90]. IDE from brain homogenates degrades different forms of Aβ: Aβ40, Aβ42 and an Aβ mutant in one type of AD (Dutch Variant 1-40 Q) [61] and [71]. Aβ42 is the longer form of Aβ and more abundant in the AD brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2006 Feb;27(2):190-8. Epub 2005 Feb 17.\", \"reference\": \"16399206\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 59, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that NOS2 upregulation impairs amyloid β degradation through negative regulation of IDE activity and thus loss of NOS2 activity will positively influence amyloid β clearance.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuroimmune Pharmacol. 2012 Mar;7(1):165-72\", \"reference\": \"22227962\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 59, \"target\": 0, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that NOS2 upregulation impairs amyloid β degradation through negative regulation of IDE activity and thus loss of NOS2 activity will positively influence amyloid β clearance.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuroimmune Pharmacol. 2012 Mar;7(1):165-72\", \"reference\": \"22227962\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 59, \"target\": 0, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \" IL-1 levels are elevated in Alzheimer brain, and overexpression of IL-1 is associated with beta-amyloid plaque progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol2000\", \"reference\": \"10850859\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 60, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" IL-1 levels are elevated in Alzheimer brain, and overexpression of IL-1 is associated with beta-amyloid plaque progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol2000\", \"reference\": \"10850859\"}, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 60, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" IL-1 levels are elevated in Alzheimer brain, and overexpression of IL-1 is associated with beta-amyloid plaque progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2000 Jun;59(6):471-6.\", \"reference\": \"10850859\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 60, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" IL-1 levels are elevated in Alzheimer brain, and overexpression of IL-1 is associated with beta-amyloid plaque progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuropathol Exp Neurol. 2000 Jun;59(6):471-6.\", \"reference\": \"10850859\"}, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 60, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2008 Mar;104(5):1387-93. Epub 2007 Nov 16.\", \"reference\": \"18021299\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 60, \"target\": 0, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2008 Mar;104(5):1387-93. Epub 2007 Nov 16.\", \"reference\": \"18021299\"}, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 60, \"target\": 0, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2008 Mar;104(5):1387-93. Epub 2007 Nov 16.\", \"reference\": \"18021299\"}, \"annotations\": {\"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 60, \"target\": 0, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"18021299\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 60, \"target\": 0, \"key\": 7}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"18021299\"}, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 60, \"target\": 0, \"key\": 8}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"18021299\"}, \"annotations\": {\"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 60, \"target\": 0, \"key\": 9}, {\"relation\": \"decreases\", \"evidence\": \"We show that knock-down of ATXN1 significantly increases the levels of both Abeta40 and Abeta42. This effect could be rescued with concurrent overexpression of ATXN1. Moreover, overexpression of ATXN1 decreased Abeta levels. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Rev Neurol2010\", \"reference\": \"20139999\"}, \"annotations\": {\"Subgraph\": \"Akt subgraph\"}, \"source\": 61, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"We show that knock-down of ATXN1 significantly increases the levels of both Abeta40 and Abeta42. This effect could be rescued with concurrent overexpression of ATXN1. Moreover, overexpression of ATXN1 decreased Abeta levels. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Rev Neurol2010\", \"reference\": \"20139999\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 61, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by AÃŸ (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2011 Oct 12;31(41):14820-30.\", \"reference\": \"21994399\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 62, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by AÃŸ (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2011 Oct 12;31(41):14820-30.\", \"reference\": \"21994399\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 62, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 63, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 63, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Our laboratory has previously shown that EGCG can increase non-amyloidogenic processing of APP through promotion of the beta-secretase ADAM10, which consequently reduced Abeta deposition and improved cognition in AD mice\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2010\", \"reference\": \"20849853\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 63, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Our laboratory has previously shown that EGCG can increase non-amyloidogenic processing of APP through promotion of the beta-secretase ADAM10, which consequently reduced Abeta deposition and improved cognition in AD mice\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2010\", \"reference\": \"20849853\"}, \"annotations\": {\"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 63, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"The γ-secretase complex comprises presenilins (PS1 or PS2), nicastrin, APH-1, and PEN-2. Herein, we find that PEN-2 can interact with ferritin light chain (FTL), an important component of the iron storage protein ferritin. In addition, we show that overexpression of FTL increases the protein levels of PEN-2 and PS1 amino-terminal fragment (NTF) and promotes γ-secretase activity for more production of Aβ and notch intracellular domain (NICD). Furthermore, iron treatments increase the levels of FTL, PEN-2 and PS1 NTF and promote γ-secretase-mediated NICD production. Moreover, downregulation of FTL decreases the levels of PEN-2 and PS1 NTF. Together, our results suggest that iron can increase γ-secretase activity through promoting the level of FTL that interacts with and stabilizes PEN-2, providing a new molecular link between iron, PEN-2/γ-secretase and Aβ generation in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2013 Aug 26;548:90-4. doi: 10.1016/j.neulet.2013.05.018. Epub 2013 May 15.\", \"reference\": \"23685131\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 64, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The γ-secretase complex comprises presenilins (PS1 or PS2), nicastrin, APH-1, and PEN-2. Herein, we find that PEN-2 can interact with ferritin light chain (FTL), an important component of the iron storage protein ferritin. In addition, we show that overexpression of FTL increases the protein levels of PEN-2 and PS1 amino-terminal fragment (NTF) and promotes γ-secretase activity for more production of Aβ and notch intracellular domain (NICD). Furthermore, iron treatments increase the levels of FTL, PEN-2 and PS1 NTF and promote γ-secretase-mediated NICD production. Moreover, downregulation of FTL decreases the levels of PEN-2 and PS1 NTF. Together, our results suggest that iron can increase γ-secretase activity through promoting the level of FTL that interacts with and stabilizes PEN-2, providing a new molecular link between iron, PEN-2/γ-secretase and Aβ generation in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2013 Aug 26;548:90-4. doi: 10.1016/j.neulet.2013.05.018. Epub 2013 May 15.\", \"reference\": \"23685131\"}, \"annotations\": {\"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 64, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Activated gadd153 can generate oxidative damage and reactive oxygen species (ROS), increase β-amyloid (Aβ) levels, disturb iron homeostasis and induce inflammation as well as cell death, which are all pathological hallmarks of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2011;6(10):e26420\", \"reference\": \"22046282\"}, \"annotations\": {\"Subgraph\": \"Unfolded protein response subgraph\", \"MeSHDisease\": \"Inflammation\"}, \"source\": 65, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Activated gadd153 can generate oxidative damage and reactive oxygen species (ROS), increase β-amyloid (Aβ) levels, disturb iron homeostasis and induce inflammation as well as cell death, which are all pathological hallmarks of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2011;6(10):e26420\", \"reference\": \"22046282\"}, \"annotations\": {\"Subgraph\": \"Unfolded protein response subgraph\", \"MeSHDisease\": \"Alzheimer Disease\"}, \"source\": 65, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Activated gadd153 can generate oxidative damage and reactive oxygen species (ROS), increase β-amyloid (Aβ) levels, disturb iron homeostasis and induce inflammation as well as cell death, which are all pathological hallmarks of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2011;6(10):e26420\", \"reference\": \"22046282\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\", \"MeSHDisease\": \"Inflammation\"}, \"source\": 65, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Activated gadd153 can generate oxidative damage and reactive oxygen species (ROS), increase β-amyloid (Aβ) levels, disturb iron homeostasis and induce inflammation as well as cell death, which are all pathological hallmarks of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2011;6(10):e26420\", \"reference\": \"22046282\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\", \"MeSHDisease\": \"Alzheimer Disease\"}, \"source\": 65, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Here, we show that siRNA-mediated loss of calsyntenin-1 in cultured neurons alters APP processing to increase production of AÃŸ. We also show that calsyntenin-1 is reduced in Alzheimer's disease brains and that the extent of this reduction correlates with increased AÃŸ levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Hum Mol Genet. 2012 Mar 27. [Epub ahead of print]\", \"reference\": \"22434822\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Axonal transport subgraph\"}, \"source\": 66, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Here, we show that siRNA-mediated loss of calsyntenin-1 in cultured neurons alters APP processing to increase production of AÃŸ. We also show that calsyntenin-1 is reduced in Alzheimer's disease brains and that the extent of this reduction correlates with increased AÃŸ levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Hum Mol Genet. 2012 Mar 27. [Epub ahead of print]\", \"reference\": \"22434822\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 66, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Here, we show that siRNA-mediated loss of calsyntenin-1 in cultured neurons alters APP processing to increase production of AÃŸ. We also show that calsyntenin-1 is reduced in Alzheimer's disease brains and that the extent of this reduction correlates with increased AÃŸ levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Hum Mol Genet. 2012 Mar 27. [Epub ahead of print]\", \"reference\": \"22434822\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Calsyntenin subgraph\"}, \"source\": 66, \"target\": 0, \"key\": 2}, {\"relation\": \"directlyIncreases\", \"evidence\": \"Differentiated neural cultures derived from KLC1-suppressed hESC contained neural rosettes but further differentiation revealed obvious morphological changes along with reduced levels of microtubule-associated neural proteins, including Tau and less secreted AÃŸ, supporting the previously established connection between KLC1, Tau and AÃŸ\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2012;7(1):e29755. Epub 2012 Jan 17\", \"reference\": \"22272245\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Tau protein subgraph\"}, \"source\": 67, \"target\": 0, \"key\": 0}, {\"relation\": \"directlyIncreases\", \"evidence\": \"Differentiated neural cultures derived from KLC1-suppressed hESC contained neural rosettes but further differentiation revealed obvious morphological changes along with reduced levels of microtubule-associated neural proteins, including Tau and less secreted AÃŸ, supporting the previously established connection between KLC1, Tau and AÃŸ\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2012;7(1):e29755. Epub 2012 Jan 17\", \"reference\": \"22272245\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 67, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" In this issue of Nature Medicine, Thathiah et al.4 now provide provocative evidence that the adaptor protein β mediates the Aβ-altering effects of these GPCRs by promoting Aβ generation. This newly uncovered function of β-arrestin 2 suggests it could be targeted to decrease amyloid pathology in patients with Alzheimer's disease. Production of the amyloid-β peptide in Alzheimer's disease by the γ-secretase complex can be regulated by certain G proteinâ€“coupled receptors. This regulation seems to be mediated by β-arrestin-2, whose expression was found to be elevated in Alzheimer's disease brains.Recruitment of β-arrestin 2 to a GPCR leads to interaction with the γ-secretase complex via the Aph-1 subunit. Other members of the complex include presenilin-1 (PS-1), nicastrin (Nct) and Pen-2. The complex then moves laterally into lipid rafts, where γ-secretase activation is enhanced. Internalization may also occur to localize γ-secretase to late endosomes, where its activation is also increased. Cleavage of APP by β-secretase (BACE1) to release soluble APP (sAPPb) followed by γ-secretase produces Aβ and APP intracellular domain (AICD). Increased production and secretion of Aβ from cells can lead to extracellular Aβ aggregation in the form of plaques. Mutagenesis of GPR3 in regions of the protein that specifically interact with either G protein or β-arrestin 2 further showed that β-arrestin 2, not G protein, mediates the ability of GPR3 to increase Aβ levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Med. 2013 Jan;19(1):22-4. doi: 10.1038/nm.3053.\", \"reference\": \"23296004\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"G-protein-mediated signaling\"}, \"source\": 68, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" In this issue of Nature Medicine, Thathiah et al.4 now provide provocative evidence that the adaptor protein β mediates the Aβ-altering effects of these GPCRs by promoting Aβ generation. This newly uncovered function of β-arrestin 2 suggests it could be targeted to decrease amyloid pathology in patients with Alzheimer's disease. Production of the amyloid-β peptide in Alzheimer's disease by the γ-secretase complex can be regulated by certain G proteinâ€“coupled receptors. This regulation seems to be mediated by β-arrestin-2, whose expression was found to be elevated in Alzheimer's disease brains.Recruitment of β-arrestin 2 to a GPCR leads to interaction with the γ-secretase complex via the Aph-1 subunit. Other members of the complex include presenilin-1 (PS-1), nicastrin (Nct) and Pen-2. The complex then moves laterally into lipid rafts, where γ-secretase activation is enhanced. Internalization may also occur to localize γ-secretase to late endosomes, where its activation is also increased. Cleavage of APP by β-secretase (BACE1) to release soluble APP (sAPPb) followed by γ-secretase produces Aβ and APP intracellular domain (AICD). Increased production and secretion of Aβ from cells can lead to extracellular Aβ aggregation in the form of plaques. Mutagenesis of GPR3 in regions of the protein that specifically interact with either G protein or β-arrestin 2 further showed that β-arrestin 2, not G protein, mediates the ability of GPR3 to increase Aβ levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Med. 2013 Jan;19(1):22-4. doi: 10.1038/nm.3053.\", \"reference\": \"23296004\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 68, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" In this issue of Nature Medicine, Thathiah et al.4 now provide provocative evidence that the adaptor protein β mediates the Aβ-altering effects of these GPCRs by promoting Aβ generation. This newly uncovered function of β-arrestin 2 suggests it could be targeted to decrease amyloid pathology in patients with Alzheimer's disease. Production of the amyloid-β peptide in Alzheimer's disease by the γ-secretase complex can be regulated by certain G proteinâ€“coupled receptors. This regulation seems to be mediated by β-arrestin-2, whose expression was found to be elevated in Alzheimer's disease brains.Recruitment of β-arrestin 2 to a GPCR leads to interaction with the γ-secretase complex via the Aph-1 subunit. Other members of the complex include presenilin-1 (PS-1), nicastrin (Nct) and Pen-2. The complex then moves laterally into lipid rafts, where γ-secretase activation is enhanced. Internalization may also occur to localize γ-secretase to late endosomes, where its activation is also increased. Cleavage of APP by β-secretase (BACE1) to release soluble APP (sAPPb) followed by γ-secretase produces Aβ and APP intracellular domain (AICD). Increased production and secretion of Aβ from cells can lead to extracellular Aβ aggregation in the form of plaques. Mutagenesis of GPR3 in regions of the protein that specifically interact with either G protein or β-arrestin 2 further showed that β-arrestin 2, not G protein, mediates the ability of GPR3 to increase Aβ levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Med. 2013 Jan;19(1):22-4. doi: 10.1038/nm.3053.\", \"reference\": \"23296004\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 68, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Similar to the loss of presenilin or nicastrin, the inactivation of endogenous mAPH-1 using small interfering RNAs results in the decrease of presenilin levels, accumulation of gamma-secretase substrates (APP carboxyl-terminal fragments), and reduction of gamma-secretase products (amyloid-beta peptides and the intracellular domains of APP and Notch). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Biological Chemistry2002\", \"reference\": \"12297508\"}, \"annotations\": {\"Subgraph\": \"Notch signaling subgraph\"}, \"source\": 69, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Similar to the loss of presenilin or nicastrin, the inactivation of endogenous mAPH-1 using small interfering RNAs results in the decrease of presenilin levels, accumulation of gamma-secretase substrates (APP carboxyl-terminal fragments), and reduction of gamma-secretase products (amyloid-beta peptides and the intracellular domains of APP and Notch). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Biological Chemistry2002\", \"reference\": \"12297508\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 69, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Similar to the loss of presenilin or nicastrin, the inactivation of endogenous mAPH-1 using small interfering RNAs results in the decrease of presenilin levels, accumulation of gamma-secretase substrates (APP carboxyl-terminal fragments), and reduction of gamma-secretase products (amyloid-beta peptides and the intracellular domains of APP and Notch). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Biological Chemistry2002\", \"reference\": \"12297508\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 69, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"alpha(2)M) is an abundant plasma/extracellular space protein implicated in clearance of amyloid beta (Abeta)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis2003\", \"reference\": \"14678766\"}, \"annotations\": {\"Subgraph\": \"Alpha 2 macroglobulin subgraph\"}, \"source\": 70, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"alpha(2)M) is an abundant plasma/extracellular space protein implicated in clearance of amyloid beta (Abeta)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis2003\", \"reference\": \"14678766\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 70, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, we investigated whether alpha2-macroglobulin (alpha2M), a protein present in neuritic plaques and elevated in Alzheimer's disease brain, is a potential regulatory factor for A beta fibril formation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1998\", \"reference\": \"9489740\"}, \"annotations\": {\"Subgraph\": \"Alpha 2 macroglobulin subgraph\"}, \"source\": 70, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present study, we investigated whether alpha2-macroglobulin (alpha2M), a protein present in neuritic plaques and elevated in Alzheimer's disease brain, is a potential regulatory factor for A beta fibril formation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1998\", \"reference\": \"9489740\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 70, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"alpha(2)-Macroglobulin (alpha(2)M) is an abundant plasma/extracellular space protein implicated in clearance of amyloid beta (Abeta)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2003 Dec;14(3):504-12.\", \"reference\": \"14678766\"}, \"annotations\": {\"Subgraph\": \"Alpha 2 macroglobulin subgraph\"}, \"source\": 70, \"target\": 0, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"alpha(2)-Macroglobulin (alpha(2)M) is an abundant plasma/extracellular space protein implicated in clearance of amyloid beta (Abeta)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2003 Dec;14(3):504-12.\", \"reference\": \"14678766\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 70, \"target\": 0, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"Humanin (HN) is a short neuroprotective peptide abolishing Abeta neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Psychiatry. 2009 Feb;14(2):206-22. Epub 2008 Sep 23\", \"reference\": \"18813209\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 71, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"We report here that (i) a single amino acid mutation at the Thr-668 residue of APP695, located 14 amino acids toward the amino-terminal end from the (682)YENPTY(687) motif, reduced the interaction between members of the Fe65 family of proteins and APP, whereas interaction of APP with the phosphotyrosine interaction domain of other APP binders such as X11-like and mammalian disabled-1 was not influenced by this mutation; (ii) the phosphorylation of APP at Thr-668 diminished the interaction of APP with Fe65 by causing a conformational change in the cytoplasmic domain that contains the Fe65-binding motif, YENPTY; and (iii) the expression of Fe65 slightly suppressed maturation of APP and decreased production of beta-amyloid (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Biological Chemistry2001\", \"reference\": \"11517218\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 72, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \" Overexpression of neuronal adaptor protein X11β has been shown to decrease the production of amyloid-β\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroreport. 2012 Feb 15;23(3):146-51. doi: 10.1097/WNR.0b013e32834f1934.\", \"reference\": \"22222501\"}, \"annotations\": {}, \"source\": 73, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our in vivo studies using transgenic mice have shown that overexpression of LRP in central nervous system (CNS) neurons increases soluble brain Abeta and this increase correlates with deficits in memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Ann N Y Acad Sci2006\", \"reference\": \"17185504\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 74, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our in vivo studies using transgenic mice have shown that overexpression of LRP in central nervous system (CNS) neurons increases soluble brain Abeta and this increase correlates with deficits in memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Ann N Y Acad Sci2006\", \"reference\": \"17185504\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 74, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Furthermore, LDLR overexpression dramatically reduced A beta aggregation and enhanced A beta clearance from the brain extracellular space. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2009\", \"reference\": \"20005821\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 74, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Furthermore, LDLR overexpression dramatically reduced A beta aggregation and enhanced A beta clearance from the brain extracellular space. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2009\", \"reference\": \"20005821\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 74, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Furthermore, LDLR overexpression dramatically reduced A beta aggregation and enhanced A beta clearance from the brain extracellular space. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2009\", \"reference\": \"20005821\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 74, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Furthermore, LDLR overexpression dramatically reduced A beta aggregation and enhanced A beta clearance from the brain extracellular space. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2009\", \"reference\": \"20005821\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 74, \"target\": 0, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer disease (AD) is characterized by the presence of senile plaques of amyloid-ÃŸ (AÃŸ) peptides derived from amyloid precursor protein (APP) and neurofibrillary tangles made of hyperphosphorylated Tau. Increasing APP gene dosage or expression has been shown to cause familial early-onset AD. However, whether and how protein stability of APP is regulated is unclear. The prolyl isomerase Pin1 and glycogen synthase kinase-3ÃŸ (GSK3ÃŸ) have been shown to have the opposite effects on APP processing and Tau hyperphosphorylation, relevant to the pathogenesis of AD. However, nothing is known about their relationship. In this study, we found that Pin1 binds to the pT330-P motif in GSK3ÃŸ to inhibit its kinase activity. Furthermore, Pin1 promotes protein turnover of APP by inhibiting GSK3ÃŸ activity. A point mutation either at Thr-330, the Pin1-binding site in GSK3ÃŸ, or at Thr-668, the GSK3ÃŸ phosphorylation site in APP, abolished the regulation of GSK3ÃŸ activity, Thr-668 phosphorylation, and APP stability by Pin1, resulting in reduced non-amyloidogenic APP processing and increased APP levels. These results uncover a novel role of Pin1 in inhibiting GSK3ÃŸ kinase activity to reduce APP protein levels, providing a previously unrecognized mechanism by which Pin1 protects against Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Mar 2;287(10):6969-73. Epub 2011 Dec 19.\", \"reference\": \"22184106\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 75, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer disease (AD) is characterized by the presence of senile plaques of amyloid-ÃŸ (AÃŸ) peptides derived from amyloid precursor protein (APP) and neurofibrillary tangles made of hyperphosphorylated Tau. Increasing APP gene dosage or expression has been shown to cause familial early-onset AD. However, whether and how protein stability of APP is regulated is unclear. The prolyl isomerase Pin1 and glycogen synthase kinase-3ÃŸ (GSK3ÃŸ) have been shown to have the opposite effects on APP processing and Tau hyperphosphorylation, relevant to the pathogenesis of AD. However, nothing is known about their relationship. In this study, we found that Pin1 binds to the pT330-P motif in GSK3ÃŸ to inhibit its kinase activity. Furthermore, Pin1 promotes protein turnover of APP by inhibiting GSK3ÃŸ activity. A point mutation either at Thr-330, the Pin1-binding site in GSK3ÃŸ, or at Thr-668, the GSK3ÃŸ phosphorylation site in APP, abolished the regulation of GSK3ÃŸ activity, Thr-668 phosphorylation, and APP stability by Pin1, resulting in reduced non-amyloidogenic APP processing and increased APP levels. These results uncover a novel role of Pin1 in inhibiting GSK3ÃŸ kinase activity to reduce APP protein levels, providing a previously unrecognized mechanism by which Pin1 protects against Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Mar 2;287(10):6969-73. Epub 2011 Dec 19.\", \"reference\": \"22184106\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 75, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"First, Pin1 inhibits the production of Aβ, and enhances the activity of eNOS. Second, Aβ and eNOS form a mutual inhibition system. Third, the well-balanced feedback signaling loop avoids the development of AD, HTN, and CAA by inhibiting the frequent pathological characteristics of these diseases, including Aβ deposition in cerebral microvessels and cerebral microbleeds.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\"}, \"annotations\": {\"NervousSystem\": \"Cerebrum\", \"CardiovascularSystem\": \"Microvessels\", \"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 75, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"First, Pin1 inhibits the production of Aβ, and enhances the activity of eNOS. Second, Aβ and eNOS form a mutual inhibition system. Third, the well-balanced feedback signaling loop avoids the development of AD, HTN, and CAA by inhibiting the frequent pathological characteristics of these diseases, including Aβ deposition in cerebral microvessels and cerebral microbleeds.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\"}, \"annotations\": {\"NervousSystem\": \"Cerebrum\", \"CardiovascularSystem\": \"Microvessels\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 75, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"To explore the molecular mechanism underlying AD, HTN, and CAA, we hypothesize a feedback signaling loop consisted of Pin1, eNOS, and Aβ. Pin1 and eNOS mainly inhibit Aβ deposition in cerebral vessels, cerebral microbleeds, and elevation of blood pressure, preventing the development of AD, HTN, and CAA, however, Aβ plays an opposite role and aggravates these diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\"}, \"annotations\": {\"NervousSystem\": \"Cerebrum\", \"CardiovascularSystem\": \"Microvessels\", \"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 75, \"target\": 0, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"To explore the molecular mechanism underlying AD, HTN, and CAA, we hypothesize a feedback signaling loop consisted of Pin1, eNOS, and Aβ. Pin1 and eNOS mainly inhibit Aβ deposition in cerebral vessels, cerebral microbleeds, and elevation of blood pressure, preventing the development of AD, HTN, and CAA, however, Aβ plays an opposite role and aggravates these diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\"}, \"annotations\": {\"NervousSystem\": \"Cerebrum\", \"CardiovascularSystem\": \"Microvessels\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 75, \"target\": 0, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"Alzheimer disease (AD) is characterized by the presence of senile plaques of amyloid-ÃŸ (AÃŸ) peptides derived from amyloid precursor protein (APP) and neurofibrillary tangles made of hyperphosphorylated Tau. Increasing APP gene dosage or expression has been shown to cause familial early-onset AD. However, whether and how protein stability of APP is regulated is unclear. The prolyl isomerase Pin1 and glycogen synthase kinase-3ÃŸ (GSK3ÃŸ) have been shown to have the opposite effects on APP processing and Tau hyperphosphorylation, relevant to the pathogenesis of AD. However, nothing is known about their relationship. In this study, we found that Pin1 binds to the pT330-P motif in GSK3ÃŸ to inhibit its kinase activity. Furthermore, Pin1 promotes protein turnover of APP by inhibiting GSK3ÃŸ activity. A point mutation either at Thr-330, the Pin1-binding site in GSK3ÃŸ, or at Thr-668, the GSK3ÃŸ phosphorylation site in APP, abolished the regulation of GSK3ÃŸ activity, Thr-668 phosphorylation, and APP stability by Pin1, resulting in reduced non-amyloidogenic APP processing and increased APP levels. These results uncover a novel role of Pin1 in inhibiting GSK3ÃŸ kinase activity to reduce APP protein levels, providing a previously unrecognized mechanism by which Pin1 protects against Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Mar 2;287(10):6969-73. Epub 2011 Dec 19.\", \"reference\": \"22184106\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 75, \"target\": 5, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Alzheimer disease (AD) is characterized by the presence of senile plaques of amyloid-ÃŸ (AÃŸ) peptides derived from amyloid precursor protein (APP) and neurofibrillary tangles made of hyperphosphorylated Tau. Increasing APP gene dosage or expression has been shown to cause familial early-onset AD. However, whether and how protein stability of APP is regulated is unclear. The prolyl isomerase Pin1 and glycogen synthase kinase-3ÃŸ (GSK3ÃŸ) have been shown to have the opposite effects on APP processing and Tau hyperphosphorylation, relevant to the pathogenesis of AD. However, nothing is known about their relationship. In this study, we found that Pin1 binds to the pT330-P motif in GSK3ÃŸ to inhibit its kinase activity. Furthermore, Pin1 promotes protein turnover of APP by inhibiting GSK3ÃŸ activity. A point mutation either at Thr-330, the Pin1-binding site in GSK3ÃŸ, or at Thr-668, the GSK3ÃŸ phosphorylation site in APP, abolished the regulation of GSK3ÃŸ activity, Thr-668 phosphorylation, and APP stability by Pin1, resulting in reduced non-amyloidogenic APP processing and increased APP levels. These results uncover a novel role of Pin1 in inhibiting GSK3ÃŸ kinase activity to reduce APP protein levels, providing a previously unrecognized mechanism by which Pin1 protects against Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Mar 2;287(10):6969-73. Epub 2011 Dec 19.\", \"reference\": \"22184106\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 75, \"target\": 5, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by AÃŸ (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2011 Oct 12;31(41):14820-30.\", \"reference\": \"21994399\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 76, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by AÃŸ (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2011 Oct 12;31(41):14820-30.\", \"reference\": \"21994399\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 76, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Ex vivo studies show that the levels of BACE1 are tightly regulated by the ATases. Specifically, up-regulation of ATase1 and ATase2 increases the levels of BACE1 and the generation of AÃŸ while siRNA-mediated down-regulation of either transferase achieves the opposite effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Mar 9;287(11):8424-33. Epub 2012 Jan 20\", \"reference\": \"22267734\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"NervousSystem\": \"Neurons\", \"Cell\": \"microglial cell\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 77, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Ex vivo studies show that the levels of BACE1 are tightly regulated by the ATases. Specifically, up-regulation of ATase1 and ATase2 increases the levels of BACE1 and the generation of AÃŸ while siRNA-mediated down-regulation of either transferase achieves the opposite effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Mar 9;287(11):8424-33. Epub 2012 Jan 20\", \"reference\": \"22267734\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"NervousSystem\": \"Neurons\", \"Cell\": \"microglial cell\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 77, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"RNAi silencing of GGA3 also elevated levels of BACE and Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron2007\", \"reference\": \"17553422\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 78, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Cellular overexpression of PrP(C) inhibited the beta-secretase cleavage of APP and reduced Abeta formation. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17573534\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 79, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In our opinion, albeit based on limited available data, a future potential therapeutic strategy is to mimic the mechanism by which the normal cellular form of the prion protein inhibits the beta-secretase beta-site amyloid precursor protein cleaving enzyme-1 (BACE1), and hence the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Expert Opin Ther Targets2008\", \"reference\": \"18479216\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 79, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"In our opinion, albeit based on limited available data, a future potential therapeutic strategy is to mimic the mechanism by which the normal cellular form of the prion protein inhibits the beta-secretase beta-site amyloid precursor protein cleaving enzyme-1 (BACE1), and hence the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Expert Opin Ther Targets2008\", \"reference\": \"18479216\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 79, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Cellular prion protein regulates beta-secretase cleavage of the Alzheimer's amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17573534\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 79, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Cellular prion protein regulates beta-secretase cleavage of the Alzheimer's amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17573534\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 79, \"target\": 0, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"PrP(C) decreases amyloid-beta (Abeta) production, which is involved in AD pathogenesis, by inhibiting beta-secretase (BACE1) activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2013\", \"reference\": \"23577068\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 79, \"target\": 0, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"PrP(C) decreases amyloid-beta (Abeta) production, which is involved in AD pathogenesis, by inhibiting beta-secretase (BACE1) activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2013\", \"reference\": \"23577068\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 79, \"target\": 0, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \" This report identifies ABCA2 as a key regulator of endogenous APP expression and processing and suggests a possible biochemical mechanism linking ABCA2 expression, APP processing and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2010 Nov;7(7):566-77.\", \"reference\": \"20704561\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 80, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" This report identifies ABCA2 as a key regulator of endogenous APP expression and processing and suggests a possible biochemical mechanism linking ABCA2 expression, APP processing and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2010 Nov;7(7):566-77.\", \"reference\": \"20704561\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"ATP binding cassette transport subgraph\"}, \"source\": 80, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"It is believed that amyloid-beta peptide (Abeta) plays a central role in the pathogenesis of Alzheimer's disease (AD). Thus, the process of amyloid precursor protein (APP) cleavage is a key event and has raised much attention in the field of AD research. It is proposed that APP, beta- and gamma-secretases are all located on the lipid raft, and the meeting of them is an indispensable step for Abeta generation. Endocytosis can lead to clustering of APP, beta- and gamma-secretases from separate smaller lipid rafts into a larger one. On the other hand, for myristoylated alanine-rich C kinase substrate (MARCKS), phosphorylation by protein kinase C (PKC) or interaction with Ca(2+) can lead to its release from membrane into cytoplasm. This process induces the release of actins and phosphatidylinositol 4, 5-bisphosphate (PIP2), which are important factors for endocytosis. Thus, the present review proposes that MARCKS may be implicated in Abeta generation, by modulating free PIP2 level and actin movement, causing endocytosis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Bull. 2010 Aug;26(4):338-44.\", \"reference\": \"20651816\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Phosphatidylinositol 3 subgraph\"}, \"source\": 81, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \" We observed that the neuronal cells primarily produce IL-34 and that microglia express its receptor, colony-stimulating factor 1 receptor. IL-34 promoted microglial proliferation and clearance of soluble oligomeric amyloid-β (oAβ), which mediates synaptic dysfunction and neuronal damage in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2011 Oct;179(4):2016-27. Epub 2011 Aug 26.\", \"reference\": \"21872563\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 82, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \" We observed that the neuronal cells primarily produce IL-34 and that microglia express its receptor, colony-stimulating factor 1 receptor. IL-34 promoted microglial proliferation and clearance of soluble oligomeric amyloid-β (oAβ), which mediates synaptic dysfunction and neuronal damage in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2011 Oct;179(4):2016-27. Epub 2011 Aug 26.\", \"reference\": \"21872563\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 82, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 83, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 83, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 84, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 84, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Cathepsin B cleaves the wild-type beta-secretase site sequence in AbetaPP to produce Abeta, and cathepsin B inhibitors administered to animal pmodels expressing AbetaPP containing the wild-type beta-secretase site sequence reduce brain Abeta in a manner consistent with beta-secretase inhibition. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2011\", \"reference\": \"21613740\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 85, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Lysosomal dysfunction may be the earliest histological change in AD. Amyloid plaques are full of active lysosomal hydrolases, implying that plaques may originate from lysosomal rupture. Cathepsins D and E (intracellular aspartyl proteases) are considered to influence Aβ peptides generation within the endosomalâ€“lysosomal pathway because they exhibit β- and γ-secretase like-activity [32]. For this reason, the endosomalâ€“lysosomal pathway is a site for cleavage of the APP into smaller β-amyloid-containing peptides, which are then degraded by cathepsins. Inhibition of cathepsins activity causes a rapid and pronounced build-up of potentially amyloidogenic protein fragments [33]. On the other hand, a failure to degrade aggregated Abeta_42 in late endosomes or secondary lysosomes was a mechanism that contributed to intracellular accumulation of Aβ in AD. The cysteine protease cathepsin B in lysosomes degrades A peptides, especially the aggregation-prone species Abeta_42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Biochim Biophys Sin (Shanghai). 2009 Jun;41(6):437-45.\", \"reference\": \"19499146\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 86, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Lysosomal dysfunction may be the earliest histological change in AD. Amyloid plaques are full of active lysosomal hydrolases, implying that plaques may originate from lysosomal rupture. Cathepsins D and E (intracellular aspartyl proteases) are considered to influence Aβ peptides generation within the endosomalâ€“lysosomal pathway because they exhibit β- and γ-secretase like-activity [32]. For this reason, the endosomalâ€“lysosomal pathway is a site for cleavage of the APP into smaller β-amyloid-containing peptides, which are then degraded by cathepsins. Inhibition of cathepsins activity causes a rapid and pronounced build-up of potentially amyloidogenic protein fragments [33]. On the other hand, a failure to degrade aggregated Abeta_42 in late endosomes or secondary lysosomes was a mechanism that contributed to intracellular accumulation of Aβ in AD. The cysteine protease cathepsin B in lysosomes degrades A peptides, especially the aggregation-prone species Abeta_42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Biochim Biophys Sin (Shanghai). 2009 Jun;41(6):437-45.\", \"reference\": \"19499146\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 86, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Pharmacological agents, such as dipeptidyl peptidase-4 (DPP-4) inhibitors, which increase the level of glucagon-like peptide-1 (GLP-1) and ameliorate T2D, have become valuable candidates as disease modifying agents in the treatment of AD. In addition, endogenous GLP-1 levels decrease amyloid beta (Aβ) peptide and tau phosphorylation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropharmacology. 2013 Sep;72:291-300. doi: 10.1016/j.neuropharm.2013.04.008. Epub 2013 Apr 17.\", \"reference\": \"23603201\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Tau protein subgraph\"}, \"source\": 87, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Pharmacological agents, such as dipeptidyl peptidase-4 (DPP-4) inhibitors, which increase the level of glucagon-like peptide-1 (GLP-1) and ameliorate T2D, have become valuable candidates as disease modifying agents in the treatment of AD. In addition, endogenous GLP-1 levels decrease amyloid beta (Aβ) peptide and tau phosphorylation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropharmacology. 2013 Sep;72:291-300. doi: 10.1016/j.neuropharm.2013.04.008. Epub 2013 Apr 17.\", \"reference\": \"23603201\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Insulin signal transduction\"}, \"source\": 87, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Pharmacological agents, such as dipeptidyl peptidase-4 (DPP-4) inhibitors, which increase the level of glucagon-like peptide-1 (GLP-1) and ameliorate T2D, have become valuable candidates as disease modifying agents in the treatment of AD. In addition, endogenous GLP-1 levels decrease amyloid beta (Aβ) peptide and tau phosphorylation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropharmacology. 2013 Sep;72:291-300. doi: 10.1016/j.neuropharm.2013.04.008. Epub 2013 Apr 17.\", \"reference\": \"23603201\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 87, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Pharmacological agents, such as dipeptidyl peptidase-4 (DPP-4) inhibitors, which increase the level of glucagon-like peptide-1 (GLP-1) and ameliorate T2D, have become valuable candidates as disease modifying agents in the treatment of AD. In addition, endogenous GLP-1 levels decrease amyloid beta (Aβ) peptide and tau phosphorylation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropharmacology. 2013 Sep;72:291-300. doi: 10.1016/j.neuropharm.2013.04.008. Epub 2013 Apr 17.\", \"reference\": \"23603201\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Glucagon subgraph\"}, \"source\": 87, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that  -amyloid accumulation is the  critical event  in development of disease (I 49,  I 50].  Lots  of research has been  done  on  the formation and accumulation of Al3, however, in the  last years  the mechanism' of  amyloid clearance came   into focus. For  Al3 clearance several mechanisms are  known (Fig.  2):  i) EnzyÂ­ matic  degradation by activated microglia or  by  insulin   deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and  angiotensin converting  enzyme (ACE); ii) Receptor-mediated transport across the blood  brain  barrier (BBB) by  binding to  the  low-density  lipoprotein  receptorÂ­ related protein (LRP) either  directly or after  binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M)  to be delivered  to  peripheral sites   of  degradation, e.g.,   liver  and kidney.  (Review in [151 ]).  Concerning insulin   resistance it has  been  shown  that   IDE expression  is stimulated  by  the IR/lGF-1 R  cascade  (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Endothelin subgraph\"}, \"source\": 88, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that  -amyloid accumulation is the  critical event  in development of disease (I 49,  I 50].  Lots  of research has been  done  on  the formation and accumulation of Al3, however, in the  last years  the mechanism' of  amyloid clearance came   into focus. For  Al3 clearance several mechanisms are  known (Fig.  2):  i) EnzyÂ­ matic  degradation by activated microglia or  by  insulin   deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and  angiotensin converting  enzyme (ACE); ii) Receptor-mediated transport across the blood  brain  barrier (BBB) by  binding to  the  low-density  lipoprotein  receptorÂ­ related protein (LRP) either  directly or after  binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M)  to be delivered  to  peripheral sites   of  degradation, e.g.,   liver  and kidney.  (Review in [151 ]).  Concerning insulin   resistance it has  been  shown  that   IDE expression  is stimulated  by  the IR/lGF-1 R  cascade  (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 88, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that  -amyloid accumulation is the  critical event  in development of disease (I 49,  I 50].  Lots  of research has been  done  on  the formation and accumulation of Al3, however, in the  last years  the mechanism' of  amyloid clearance came   into focus. For  Al3 clearance several mechanisms are  known (Fig.  2):  i) EnzyÂ­ matic  degradation by activated microglia or  by  insulin   deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and  angiotensin converting  enzyme (ACE); ii) Receptor-mediated transport across the blood  brain  barrier (BBB) by  binding to  the  low-density  lipoprotein  receptorÂ­ related protein (LRP) either  directly or after  binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M)  to be delivered  to  peripheral sites   of  degradation, e.g.,   liver  and kidney.  (Review in [151 ]).  Concerning insulin   resistance it has  been  shown  that   IDE expression  is stimulated  by  the IR/lGF-1 R  cascade  (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 88, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that  -amyloid accumulation is the  critical event  in development of disease (I 49,  I 50].  Lots  of research has been  done  on  the formation and accumulation of Al3, however, in the  last years  the mechanism' of  amyloid clearance came   into focus. For  Al3 clearance several mechanisms are  known (Fig.  2):  i) EnzyÂ­ matic  degradation by activated microglia or  by  insulin   deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and  angiotensin converting  enzyme (ACE); ii) Receptor-mediated transport across the blood  brain  barrier (BBB) by  binding to  the  low-density  lipoprotein  receptorÂ­ related protein (LRP) either  directly or after  binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M)  to be delivered  to  peripheral sites   of  degradation, e.g.,   liver  and kidney.  (Review in [151 ]).  Concerning insulin   resistance it has  been  shown  that   IDE expression  is stimulated  by  the IR/lGF-1 R  cascade  (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Lower angiotensin-converting enzyme (ACE) activity could increase the risk of Alzheimer's disease (AD) as ACE functions to degrade amyloid-β (Aβ).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Alzheimers Res Ther. 2014 May 15;6(3):27\", \"reference\": \"24987467\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Lower angiotensin-converting enzyme (ACE) activity could increase the risk of Alzheimer's disease (AD) as ACE functions to degrade amyloid-β (Aβ).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Alzheimers Res Ther. 2014 May 15;6(3):27\", \"reference\": \"24987467\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Lower CSF ACE protein level, and to a lesser extent serum ACE protein level and CSF ACE activity, were associated with lower CSF Aβ, indicating more brain Aβ pathology; adjusted regression coefficients (B) (95% CI) per SD increase were 0.09 (0.04; 0.15), 0.06 (0.00; 0.12) and 0.05 (0.00; 0.11), respectively. Further, lower CSF ACE protein level was associated with lower CSF tau and ptau levels; adjusted B's (95% CI) per SD increase were 0.15 (0.06; 0.25) and 0.17 (0.10; 0.25), respectively.These results strengthen the hypothesis that ACE degrades Aβ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Alzheimers Res Ther. 2014 May 15;6(3):27\", \"reference\": \"24987467\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"FluidAndSecretion\": \"Serum\", \"NervousSystem\": \"Brain\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Lower CSF ACE protein level, and to a lesser extent serum ACE protein level and CSF ACE activity, were associated with lower CSF Aβ, indicating more brain Aβ pathology; adjusted regression coefficients (B) (95% CI) per SD increase were 0.09 (0.04; 0.15), 0.06 (0.00; 0.12) and 0.05 (0.00; 0.11), respectively. Further, lower CSF ACE protein level was associated with lower CSF tau and ptau levels; adjusted B's (95% CI) per SD increase were 0.15 (0.06; 0.25) and 0.17 (0.10; 0.25), respectively.These results strengthen the hypothesis that ACE degrades Aβ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Alzheimers Res Ther. 2014 May 15;6(3):27\", \"reference\": \"24987467\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"FluidAndSecretion\": \"Serum\", \"NervousSystem\": \"Brain\", \"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"ACE was found to significantly inhibit A beta aggregation in a dose response manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2001\", \"reference\": \"11604391\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"ACE was found to significantly inhibit A beta aggregation in a dose response manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2001\", \"reference\": \"11604391\"}, \"annotations\": {\"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \"These findings led to the hypothesis that ACE may affect susceptibility to AD by degrading A beta and preventing the accumulation of amyloid plaques in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2001\", \"reference\": \"11604391\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 7}, {\"relation\": \"decreases\", \"evidence\": \"These findings led to the hypothesis that ACE may affect susceptibility to AD by degrading A beta and preventing the accumulation of amyloid plaques in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2001\", \"reference\": \"11604391\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 8}, {\"relation\": \"decreases\", \"evidence\": \"The most striking fact was that ACE degraded A beta by cleaving A beta-(1-40) at the site Asp(7)-Ser(8).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2001\", \"reference\": \"11604391\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 9}, {\"relation\": \"decreases\", \"evidence\": \"The most striking fact was that ACE degraded A beta by cleaving A beta-(1-40) at the site Asp(7)-Ser(8).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2001\", \"reference\": \"11604391\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 10}, {\"relation\": \"decreases\", \"evidence\": \"ACE was found to significantly inhibit A beta aggregation in a dose response manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2001 Dec 21;276(51):47863-8. Epub 2001 Oct 16.\", \"reference\": \"11604391\"}, \"annotations\": {\"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 11}, {\"relation\": \"decreases\", \"evidence\": \"ACE was found to significantly inhibit A beta aggregation in a dose response manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2001 Dec 21;276(51):47863-8. Epub 2001 Oct 16.\", \"reference\": \"11604391\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 12}, {\"relation\": \"decreases\", \"evidence\": \"The most striking fact was that ACE degraded A beta by cleaving A beta-(1-40) at the site Asp(7)-Ser(8).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2001 Dec 21;276(51):47863-8. Epub 2001 Oct 16.\", \"reference\": \"11604391\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 13}, {\"relation\": \"decreases\", \"evidence\": \"The most striking fact was that ACE degraded A beta by cleaving A beta-(1-40) at the site Asp(7)-Ser(8).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2001 Dec 21;276(51):47863-8. Epub 2001 Oct 16.\", \"reference\": \"11604391\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 89, \"target\": 0, \"key\": 14}, {\"relation\": \"increases\", \"evidence\": \"for the first time it was shown that the Aβ40 and Aβ42 are substrates of DPPIV. DPPIV prohibits the fibrillation of peptides and promotes disaggregation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurochem Int. 2013 Jun;62(8):1048-54. doi: 10.1016/j.neuint.2013.03.017. Epub 2013 Apr 8.\", \"reference\": \"23579020\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 90, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Seladin-1 was considered a novel neuroprotective factor, because of its anti-apoptotic activity. Subsequently, it was demonstrated that seladin-1 has also enzymatic activity [3-β-hydroxysterol delta-24-reductase, (DHCR24)], which catalyzes the synthesis of cholesterol from desmosterol. The amount of membrane cholesterol may play an important role both in protecting neuronal cells against toxic insults and in inhibiting the production of β-amyloid. We demonstrated that seladin-1 overexpression increases the amount of membrane cholesterol and induces resistance against β-amyloid aggregates in neuroblastoma cells, whereas a specific inhibitor of DHCR24 increased cell vulnerability. We also hypothesized that seladin-1 might be a mediator of the neuroprotective effects of estrogens. We first demonstrated that, in human fetal neuroepithelial cells (FNC), 17β-estradiol, raloxifene, and tamoxifen exert protective effects against β-amyloid toxicity and oxidative stress. In addition, these molecules significantly increased the expression of seladin-1 and the amount of cell cholesterol. Then, we showed that, upon seladin-1 silencing, the protective effects of estrogens were abolished, thus indicating this factor as a fundamental mediator of estrogen-mediated neuroprotection, at least in FNC cells. Furthermore, we detected the presence of functionally active half-palindromic estrogen responsive elements upstream the coding region of the seladin-1 gene. Overall, our results indicate that seladin-1 may be viewed as a multi-faceted protein, which conjugates both the neuroprotective properties of estrogens and the important functions of cholesterol in maintaining brain homeostasis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2011 Sep 15;191:107-17. doi: 10.1016/j.neuroscience.2011.03.011. Epub 2011 Mar 22.\", \"reference\": \"21396986\"}, \"annotations\": {\"UserdefinedCellLine\": \"Neuroblastoma cell\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 91, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Seladin-1 was considered a novel neuroprotective factor, because of its anti-apoptotic activity. Subsequently, it was demonstrated that seladin-1 has also enzymatic activity [3-β-hydroxysterol delta-24-reductase, (DHCR24)], which catalyzes the synthesis of cholesterol from desmosterol. The amount of membrane cholesterol may play an important role both in protecting neuronal cells against toxic insults and in inhibiting the production of β-amyloid. We demonstrated that seladin-1 overexpression increases the amount of membrane cholesterol and induces resistance against β-amyloid aggregates in neuroblastoma cells, whereas a specific inhibitor of DHCR24 increased cell vulnerability. We also hypothesized that seladin-1 might be a mediator of the neuroprotective effects of estrogens. We first demonstrated that, in human fetal neuroepithelial cells (FNC), 17β-estradiol, raloxifene, and tamoxifen exert protective effects against β-amyloid toxicity and oxidative stress. In addition, these molecules significantly increased the expression of seladin-1 and the amount of cell cholesterol. Then, we showed that, upon seladin-1 silencing, the protective effects of estrogens were abolished, thus indicating this factor as a fundamental mediator of estrogen-mediated neuroprotection, at least in FNC cells. Furthermore, we detected the presence of functionally active half-palindromic estrogen responsive elements upstream the coding region of the seladin-1 gene. Overall, our results indicate that seladin-1 may be viewed as a multi-faceted protein, which conjugates both the neuroprotective properties of estrogens and the important functions of cholesterol in maintaining brain homeostasis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2011 Sep 15;191:107-17. doi: 10.1016/j.neuroscience.2011.03.011. Epub 2011 Mar 22.\", \"reference\": \"21396986\"}, \"annotations\": {\"UserdefinedCellLine\": \"Neuroblastoma cell\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 91, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Human bleomycin hydrolase (hBH) is a neutral cysteine protease genetically associated with increased risk for Alzheimer disease. We show here that ectopic expression of hBH in 293APPwt and CHOAPPsw cells altered the processing of amyloid precursor protein (APP) and increased significantly the release of its proteolytic fragment, beta amyloid (Abeta). We also found that hBH interacted and colocalized with APP as determined by subcellular fractionation, in vitro binding assay, and confocal immunolocalization. Metabolic labeling and pulse-chase experiments showed that ectopic hBH expression increased secretion of soluble APPalpha/beta products without changing the half-life of cellular APP. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J. 2000 Sep;14(12):1837-47.\", \"reference\": \"10973933\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\", \"UserdefinedCellLine\": \"293APPwt\"}, \"source\": 92, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Human bleomycin hydrolase (hBH) is a neutral cysteine protease genetically associated with increased risk for Alzheimer disease. We show here that ectopic expression of hBH in 293APPwt and CHOAPPsw cells altered the processing of amyloid precursor protein (APP) and increased significantly the release of its proteolytic fragment, beta amyloid (Abeta). We also found that hBH interacted and colocalized with APP as determined by subcellular fractionation, in vitro binding assay, and confocal immunolocalization. Metabolic labeling and pulse-chase experiments showed that ectopic hBH expression increased secretion of soluble APPalpha/beta products without changing the half-life of cellular APP. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB J. 2000 Sep;14(12):1837-47.\", \"reference\": \"10973933\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\", \"UserdefinedCellLine\": \"CHOAPPsw\"}, \"source\": 92, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Human serum albumin (HSA) binds 95% of Aβ in blood plasma and is thought to inhibit plaque formation in peripheral tissue. However, the role of albumin in binding Aβ in the cerebrospinal fluid has been largely overlooked. Here we investigate the effect of HSA on both Aβ(1-40) and Aβ(1-42) fibril growth. We show that at micromolar cerebrospinal fluid levels, HSA inhibits the kinetics of Aβ fibrillization, significantly increasing the lag time and decreasing the total amount of fibrils produced\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Aug 10;287(33):28163-8. doi: 10.1074/jbc.C112.360800. Epub 2012 Jun 20.\", \"reference\": \"22718756\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"FluidAndSecretion\": \"Cerebrospinal Fluid\", \"Subgraph\": \"Albumin subgraph\"}, \"source\": 93, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Human serum albumin (HSA) binds 95% of Aβ in blood plasma and is thought to inhibit plaque formation in peripheral tissue. However, the role of albumin in binding Aβ in the cerebrospinal fluid has been largely overlooked. Here we investigate the effect of HSA on both Aβ(1-40) and Aβ(1-42) fibril growth. We show that at micromolar cerebrospinal fluid levels, HSA inhibits the kinetics of Aβ fibrillization, significantly increasing the lag time and decreasing the total amount of fibrils produced\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2012 Aug 10;287(33):28163-8. doi: 10.1074/jbc.C112.360800. Epub 2012 Jun 20.\", \"reference\": \"22718756\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"FluidAndSecretion\": \"Cerebrospinal Fluid\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 93, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Human serum albumin (HSA) is a potent inhibitor of Aβ self-association and this novel, to our knowledge, function of HSA is of potential therapeutic interest for the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biophys J. 2013 Oct 1;105(7):1700-9\", \"reference\": \"24094411\"}, \"annotations\": {\"FluidAndSecretion\": \"Serum\", \"Species\": \"9606\", \"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Albumin subgraph\"}, \"source\": 93, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Human serum albumin (HSA) is a potent inhibitor of Aβ self-association and this novel, to our knowledge, function of HSA is of potential therapeutic interest for the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biophys J. 2013 Oct 1;105(7):1700-9\", \"reference\": \"24094411\"}, \"annotations\": {\"FluidAndSecretion\": \"Serum\", \"Species\": \"9606\", \"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 93, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2009 May 22;455(3):191-4. doi: 10.1016/j.neulet.2009.03.066. Epub 2009 Mar 25.\", \"reference\": \"19429119\"}, \"annotations\": {\"Subgraph\": \"Leptin subgraph\"}, \"source\": 94, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2009 May 22;455(3):191-4. doi: 10.1016/j.neulet.2009.03.066. Epub 2009 Mar 25.\", \"reference\": \"19429119\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 94, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Treatment with leptin reversed the 27-OHC-induced increase in Abeta and phosphorylated tau by decreasing the levels of BACE-1 and GSK-3beta respectively.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease2010\", \"reference\": \"20157255\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 94, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2009 May 22;455(3):191-4. doi: 10.1016/j.neulet.2009.03.066. Epub 2009 Mar 25.\", \"reference\": \"19429119\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 94, \"target\": 5, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2009 May 22;455(3):191-4. doi: 10.1016/j.neulet.2009.03.066. Epub 2009 Mar 25.\", \"reference\": \"19429119\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 94, \"target\": 5, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2009 May 22;455(3):191-4. doi: 10.1016/j.neulet.2009.03.066. Epub 2009 Mar 25.\", \"reference\": \"19429119\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Leptin subgraph\"}, \"source\": 94, \"target\": 5, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Treatment with leptin reversed the 27-OHC-induced increase in Abeta and phosphorylated tau by decreasing the levels of BACE-1 and GSK-3beta respectively.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease2010\", \"reference\": \"20157255\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 94, \"target\": 5, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Estrogen activates matrix metalloproteinases-2 and -9 to increase beta amyloid degradation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Cell Neurosci. 2012 Apr;49(4):423-9\", \"reference\": \"22402435\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 95, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Estrogen activates matrix metalloproteinases-2 and -9 to increase beta amyloid degradation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Cell Neurosci. 2012 Apr;49(4):423-9\", \"reference\": \"22402435\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 95, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"OBJECTIVE: Urinary-type plasminogen activator (uPA) binding to uPA receptor (uPAR) promotes the activation of matrix metalloproteinase-9 (MMP-9), which degrades amyloid beta protein (Abeta) in vitro.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Clin Neuropathol2001\", \"reference\": \"11327298\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 95, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"OBJECTIVE: Urinary-type plasminogen activator (uPA) binding to uPA receptor (uPAR) promotes the activation of matrix metalloproteinase-9 (MMP-9), which degrades amyloid beta protein (Abeta) in vitro.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Clin Neuropathol2001\", \"reference\": \"11327298\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 95, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that MMP-9 can degrade fAbeta and may contribute to ongoing clearance of plaques from amyloid-laden brains.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2006\", \"reference\": \"16787929\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {}, \"source\": 95, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Our aim was to test if this enzyme could also degrade the insoluble 40-42 residues long A beta peptides purified from Alzheimer Disease brain. Our results indicate that MMP2 hydrolyzes A beta 1-40 and A beta 1-42 peptides at Lys 16-Leu 17, at Leu 34-Met 35, and Met 35-Val 36 peptide bonds.These results suggest that MMP2 has the ability of degrading A beta of AD in vitro.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Biophys Res Commun. 1994 Dec 30;205(3):1755-61\", \"reference\": \"7811262\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"NervousSystem\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 96, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our aim was to test if this enzyme could also degrade the insoluble 40-42 residues long A beta peptides purified from Alzheimer Disease brain. Our results indicate that MMP2 hydrolyzes A beta 1-40 and A beta 1-42 peptides at Lys 16-Leu 17, at Leu 34-Met 35, and Met 35-Val 36 peptide bonds.These results suggest that MMP2 has the ability of degrading A beta of AD in vitro.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Biophys Res Commun. 1994 Dec 30;205(3):1755-61\", \"reference\": \"7811262\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"NervousSystem\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 96, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Estrogen activates matrix metalloproteinases-2 and -9 to increase beta amyloid degradation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Cell Neurosci. 2012 Apr;49(4):423-9\", \"reference\": \"22402435\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 96, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Estrogen activates matrix metalloproteinases-2 and -9 to increase beta amyloid degradation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Cell Neurosci. 2012 Apr;49(4):423-9\", \"reference\": \"22402435\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 96, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Cells overexpressing Bcl-xL were significantly protected from beta-amyloid neurotoxicity and staurosporine-induced apoptosis compared to vector-transfected controls.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2001;102(1):139-50\", \"reference\": \"11226677\"}, \"annotations\": {\"MeSHDisease\": \"Wounds and Injuries\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 97, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Cells overexpressing Bcl-xL were significantly protected from beta-amyloid neurotoxicity and staurosporine-induced apoptosis compared to vector-transfected controls.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2001;102(1):139-50\", \"reference\": \"11226677\"}, \"annotations\": {\"MeSHDisease\": \"Wounds and Injuries\", \"Subgraph\": \"Bcl-2 subgraph\"}, \"source\": 97, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Deletion of Abca1 increases Abeta deposition\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2005 Dec 30;280(52):43236-42. Epub 2005 Oct 5.\", \"reference\": \"16207708\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 98, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Deletion of Abca1 increases Abeta deposition\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2005 Dec 30;280(52):43236-42. Epub 2005 Oct 5.\", \"reference\": \"16207708\"}, \"annotations\": {\"Subgraph\": \"ATP binding cassette transport subgraph\"}, \"source\": 98, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"In contrast, enhanced FasL shedding, by recombinant MMP-7, completely protected neurons from Abeta neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Biol. 2002 Sep 17;12(18):1595-600\", \"reference\": \"12372252\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"NervousSystem\": \"Neurons\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 99, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In contrast, enhanced FasL shedding, by recombinant MMP-7, completely protected neurons from Abeta neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Biol. 2002 Sep 17;12(18):1595-600\", \"reference\": \"12372252\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"NervousSystem\": \"Neurons\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 99, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Thus, endogenous tPA, preferentially its aggregate form, could degrade Aβ molecules and maintain low levels of brain Aβ, resulting in the delay of AD pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2014 Mar;35(3):511-9\", \"reference\": \"24126163\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 100, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Thus, endogenous tPA, preferentially its aggregate form, could degrade Aβ molecules and maintain low levels of brain Aβ, resulting in the delay of AD pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2014 Mar;35(3):511-9\", \"reference\": \"24126163\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Plasminogen activator subgraph\"}, \"source\": 100, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Subsequent studies showed that it was plasmin, the product of tPA activation of plasminogen, that specifically cleaved A beta 1-40 in the amino terminal region between Arg5 and His6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry. 1999 Aug 31;38(35):11570-6\", \"reference\": \"10471309\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 100, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Subsequent studies showed that it was plasmin, the product of tPA activation of plasminogen, that specifically cleaved A beta 1-40 in the amino terminal region between Arg5 and His6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry. 1999 Aug 31;38(35):11570-6\", \"reference\": \"10471309\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Plasminogen activator subgraph\"}, \"source\": 100, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Additionally, plasmin (activated by tPA) attenuates Abeta neurotoxicity by degrading the peptide and rendering it inactive.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Neurosci2003\", \"reference\": \"14501010\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Plasminogen activator subgraph\"}, \"source\": 100, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Additionally, plasmin (activated by tPA) attenuates Abeta neurotoxicity by degrading the peptide and rendering it inactive.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Mol Neurosci2003\", \"reference\": \"14501010\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 100, \"target\": 0, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Thus, neuroserpin inhibition of tissue plasminogen activator activity leads to reduced plasmin and may be responsible for reduced clearance of amyloid-beta in the Alzheimer disease brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2009 Apr;109(2):303-15\", \"reference\": \"19222708\"}, \"annotations\": {\"Species\": \"9606\", \"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 101, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Thus, neuroserpin inhibition of tissue plasminogen activator activity leads to reduced plasmin and may be responsible for reduced clearance of amyloid-beta in the Alzheimer disease brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2009 Apr;109(2):303-15\", \"reference\": \"19222708\"}, \"annotations\": {\"Species\": \"9606\", \"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Plasminogen activator subgraph\"}, \"source\": 101, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"This suggests that increased expression of uPA may initially serve as a protective mechanism leading to localized degradation and clearance of the pathogenic stimulus A beta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2003 May 23;278(21):19054-61\", \"reference\": \"12754271\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 102, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"This suggests that increased expression of uPA may initially serve as a protective mechanism leading to localized degradation and clearance of the pathogenic stimulus A beta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2003 May 23;278(21):19054-61\", \"reference\": \"12754271\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Plasminogen activator subgraph\"}, \"source\": 102, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"An increase of the Abeta binding protein transthyretin suggests that increased clearance of Abeta underlies the reduction in plaques.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Biophys Res Commun2009\", \"reference\": \"19523444\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 103, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol2008\", \"reference\": \"18941241\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 104, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol2008\", \"reference\": \"18941241\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 104, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol2008\", \"reference\": \"18941241\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 104, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Finally, we show that PAT1a promotes APP/APLPs processing, resulting in increased secretion of beta-amyloid peptide\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2006\", \"reference\": \"17050537\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 105, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"PAR-4 (prostate apoptosis response-4) is a leucine zipper protein that was initially identified to be associated with neuronal degeneration and aberrant Abeta production in pmodels of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2005\", \"reference\": \"15671026\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 106, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"PAR-4 (prostate apoptosis response-4) is a leucine zipper protein that was initially identified to be associated with neuronal degeneration and aberrant Abeta production in pmodels of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2005\", \"reference\": \"15671026\"}, \"annotations\": {\"Subgraph\": \"Apoptosis signaling subgraph\"}, \"source\": 106, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" Overexpression of constitutively active CK1epsilon, one of the CK1 isoforms expressed in brain, leads to an increase in Abeta peptide production. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17360493\"}, \"annotations\": {\"Subgraph\": \"Wnt signaling subgraph\"}, \"source\": 107, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Here we show that glutaminyl cyclase (QC) catalyzes the formation of Abeta 3(pE)-40/42 after amyloidogenic processing of APP in two different cell lines, applying specific ELISAs and Western blotting based on urea-PAGE.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry2008\", \"reference\": \"18570439\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 108, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Here we show that glutaminyl cyclase (QC) catalyzes the formation of Abeta 3(pE)-40/42 after amyloidogenic processing of APP in two different cell lines, applying specific ELISAs and Western blotting based on urea-PAGE.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry2008\", \"reference\": \"18570439\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 108, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Here we show that glutaminyl cyclase (QC) catalyzes the formation of Abeta 3(pE)-40/42 after amyloidogenic processing of APP in two different cell lines, applying specific ELISAs and Western blotting based on urea-PAGE. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry2008\", \"reference\": \"18570439\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 108, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"urther, overexpression of the SUMO E2 enzyme ubc9 along with SUMO-1 results in decreased levels of Abeta aggregates in cells transfected with the familial Alzheimer's disease-associated V642F mutant APP, indicating the potential of up-regulating activity of the cellular sumoylation machinery as an approach against Alzheimer's disease. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Biophys Res Commun2008\", \"reference\": \"18675254\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 109, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \" Importantly, knockdown of endogenous RanBP9 significantly reduced Abeta generation in Chinese hamster ovary cells and in primary neurons, demonstrating its physiological role in BACE1 cleavage of APP. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2009\", \"reference\": \"19251705\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 110, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Consistently, overexpression of calpain in heterologous APP expressing cells up-regulated the level of BACE1 and increased Abeta production. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2010\", \"reference\": \"20595388\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 111, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Consistently, overexpression of calpain in heterologous APP expressing cells up-regulated the level of BACE1 and increased Abeta production. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem2010\", \"reference\": \"20595388\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 111, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Using high-throughput siRNA screening technology, we assessed 15,200 genes for their role in Abeta42 secretion and identified leucine-rich repeat transmembrane 3 (LRRTM3) as a neuronal gene that promotes APP processing by BACE1. siRNAs targeting LRRTM3 inhibit the secretion of Abeta40, Abeta42, and sAPPbeta, the N-terminal APP fragment produced by BACE1 cleavage, from cultured cells and primary neurons by up to 60%, whereas overexpression increases Abeta secretion.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17098871\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 112, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Using high-throughput siRNA screening technology, we assessed 15,200 genes for their role in Abeta42 secretion and identified leucine-rich repeat transmembrane 3 (LRRTM3) as a neuronal gene that promotes APP processing by BACE1. siRNAs targeting LRRTM3 inhibit the secretion of Abeta40, Abeta42, and sAPPbeta, the N-terminal APP fragment produced by BACE1 cleavage, from cultured cells and primary neurons by up to 60%, whereas overexpression increases Abeta secretion.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17098871\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 112, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Overexpression of NgR decreases Abeta production in neuroblastoma culture, and targeted disruption of NgR expression increases transgenic mouse brain Abeta levels, Abeta plaque deposition, and dystrophic neurites. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2006\", \"reference\": \"16452662\"}, \"annotations\": {\"Subgraph\": \"Axonal guidance subgraph\"}, \"source\": 113, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Overexpression of NgR decreases Abeta production in neuroblastoma culture, and targeted disruption of NgR expression increases transgenic mouse brain Abeta levels, Abeta plaque deposition, and dystrophic neurites. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2006\", \"reference\": \"16452662\"}, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 113, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Overexpression of NgR decreases Abeta production in neuroblastoma culture, and targeted disruption of NgR expression increases transgenic mouse brain Abeta levels, Abeta plaque deposition, and dystrophic neurites. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci2006\", \"reference\": \"16452662\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 113, \"target\": 0, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \" Deletion of NgR expression increases Abeta plaque deposition in transgenic mice, while excess soluble NgR treatment reduces Abeta plaque deposition in mice.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res2007\", \"reference\": \"18220524\"}, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 113, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol2008\", \"reference\": \"18941241\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 114, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol2008\", \"reference\": \"18941241\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 114, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol2008\", \"reference\": \"18941241\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 114, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Down regulation of DNMT results in hypomethylation of BACE1 and APP which are involved in Abeta production and causes upregulation of their protein expression; in turn SP1 transcription factor increases which finally results in Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Free Radic Biol Med. 2009 May 1;46(9):1241-9. doi: 10.1016/j.freeradbiomed.2009.02.006. Epub 2009 Feb 23.\", \"reference\": \"19245828\"}, \"annotations\": {\"Species\": \"10090\"}, \"source\": 115, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In CAA, and probably in AD, corticosteroid treatment suppresses the secondarily induced vascular inflammation (Chang et al., 2012, Kloppenborg et al., 2010 and Machida et al., 2012) and the activation of S100A9/Mrp14 (Gebhardt et al., 2002 and Kametani and Ikeda, 2013). Recently, it has been reported that anti-Aβ autoantibodies were progressively reduced in the cerebrospinal fluid of a patient with CAA-related inflammation, following intravenous steroid administration (Piazza et al., 2013). Reduction of S100A9/Mrp14 might induce suppression of amyloid fibril formation (Zhang et al., 2012) and increase phagocytosis of fibrillar Aβ in microglia cells (Kummer et al., 2012), resulting in reduced Aβ amyloid fibril deposition.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2014 Apr;35(4):e11-2\", \"reference\": \"24262203\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 116, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In CAA, and probably in AD, corticosteroid treatment suppresses the secondarily induced vascular inflammation (Chang et al., 2012, Kloppenborg et al., 2010 and Machida et al., 2012) and the activation of S100A9/Mrp14 (Gebhardt et al., 2002 and Kametani and Ikeda, 2013). Recently, it has been reported that anti-Aβ autoantibodies were progressively reduced in the cerebrospinal fluid of a patient with CAA-related inflammation, following intravenous steroid administration (Piazza et al., 2013). Reduction of S100A9/Mrp14 might induce suppression of amyloid fibril formation (Zhang et al., 2012) and increase phagocytosis of fibrillar Aβ in microglia cells (Kummer et al., 2012), resulting in reduced Aβ amyloid fibril deposition.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2014 Apr;35(4):e11-2\", \"reference\": \"24262203\"}, \"annotations\": {\"Subgraph\": \"Metabolism of steroid hormones subgraph\"}, \"source\": 116, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Pro-inflammatory S100A9 protein is increasingly recognized as an important contributor to inflammation-related neurodegeneration. Here, we provide insights into S100A9 specific mechanisms of action in Alzheimer's disease (AD). Due to its inherent amyloidogenicity S100A9 contributes to amyloid plaque formation together with Aβ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Neuropathol. 2014 Apr;127(4):507-22\", \"reference\": \"24240735\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\"}, \"source\": 116, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Pro-inflammatory S100A9 protein is increasingly recognized as an important contributor to inflammation-related neurodegeneration. Here, we provide insights into S100A9 specific mechanisms of action in Alzheimer's disease (AD). Due to its inherent amyloidogenicity S100A9 contributes to amyloid plaque formation together with Aβ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Neuropathol. 2014 Apr;127(4):507-22\", \"reference\": \"24240735\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 116, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"We therefore conclude that Mrp14 promotes APP processing and Aβ accumulation under neuroinflammatory conditions.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2012 Dec 5;32(49):17824-9.\", \"reference\": \"23223301\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\"}, \"source\": 116, \"target\": 0, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"We therefore conclude that Mrp14 promotes APP processing and Aβ accumulation under neuroinflammatory conditions.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2012 Dec 5;32(49):17824-9.\", \"reference\": \"23223301\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 116, \"target\": 0, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"The 5-HT4 receptor may play a role in memory and learning and 5-HT4 receptor activation has been suggested to modulate acetylcholine release and to reduce amyloid-β (Aβ) accumulation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2011;26(3):457-66. doi: 10.3233/JAD-2011-110056.\", \"reference\": \"21673407\"}, \"annotations\": {}, \"source\": 117, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The formation of endogenous inflammatory lipid mediators, leukotrienes, is initiated by 5-lipoxygenase (5-LOX), which is also expressed in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Med Hypotheses. 2000 Jan;54(1):75-6\", \"reference\": \"10790729\"}, \"annotations\": {\"NervousSystem\": \"Neurons\"}, \"source\": 118, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\"}, \"annotations\": {}, \"source\": 119, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {}, \"source\": 119, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with AÃƒÅ¸ is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane ÃƒÅ¸-subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the ÃƒÅ¸-subunit, and causes autophosphorylation of the ÃƒÅ¸-subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against AÃƒÅ¸ toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3ÃƒÅ¸ and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3ÃƒÅ¸ signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with AÃƒÅ¸ activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, AÃƒÅ¸ monomers may bind to IGF-1/insulin receptors, as already shown for AÃƒÅ¸ oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 120, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with AÃƒÅ¸ is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane ÃƒÅ¸-subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the ÃƒÅ¸-subunit, and causes autophosphorylation of the ÃƒÅ¸-subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against AÃƒÅ¸ toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3ÃƒÅ¸ and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3ÃƒÅ¸ signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with AÃƒÅ¸ activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, AÃƒÅ¸ monomers may bind to IGF-1/insulin receptors, as already shown for AÃƒÅ¸ oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Regulation of cytoskeleton subgraph\"}, \"source\": 120, \"target\": 5, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The PI3K/Akt signaling: PI3K, a membrane-associated second messenger protein, and its downstream kinase, Akt, are associated with neuronal survival [111] and plasticity [112] via activation of transcription pathways and protein synthesis. PI3K pathway, which required the activation of IGF-1/insulin receptors, is the most convincing prosurvival effect of AÃƒÅ¸42 monomers. The PI3K signaling pathway is important in the transmission of survival signals in many cell types including neurons [113, 114]. The PI3K-Akt signaling cascade, initiated by IRS, is phosphorylated by stimulated insulin- andIGF-receptor tyrosine kinases [115]. One of the kinases known to lie downstream of PI3K is Akt, which can be directly activated by products of PI3K [116] by promoting its phosphorylation at Ser473 and Thr308 [117]. Activated Akt, in turn, phosphorylates a wide range of substrates activating anti-apoptotic (survival) factors and inactivating pro-apoptotic factors [114, 117]. Certain proapoptotic mediators, such as the transcription factor forkhead (FOXO), the tau kinase GSK-3ÃƒÅ¸, and the Bcl2 antagonist BAD proteins, are inactivated by Akt [118, 119]. Akt downregulates the activities of GSK-3a and GSK-3ÃƒÅ¸ by phosphorylating the former at Ser21 and the latter at Ser9 [118]. Phosphorylation/inactivation of GSK-3ÃƒÅ¸, suppresses GSK-3ÃƒÅ¸-dependent phosphorylation of tau at residues overphosphorylated in AD and prevents apoptosis of confluent cells. Treatment of cortical neurons with AÃƒÅ¸42 monomers increased Ser9 phosphorylation (inhibition) of the Akt substrate, GSK- 3ÃƒÅ¸ [97]. Inhibition of GSK-3ÃƒÅ¸ promotes cell survival through a variety of mechanisms including a reduced degradation of ÃƒÅ¸-catenin, which then translocates into the nucleus and activates the transcription of protective genes [126].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 121, \"target\": 5, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The PI3K/Akt signaling: PI3K, a membrane-associated second messenger protein, and its downstream kinase, Akt, are associated with neuronal survival [111] and plasticity [112] via activation of transcription pathways and protein synthesis. PI3K pathway, which required the activation of IGF-1/insulin receptors, is the most convincing prosurvival effect of AÃƒÅ¸42 monomers. The PI3K signaling pathway is important in the transmission of survival signals in many cell types including neurons [113, 114]. The PI3K-Akt signaling cascade, initiated by IRS, is phosphorylated by stimulated insulin- andIGF-receptor tyrosine kinases [115]. One of the kinases known to lie downstream of PI3K is Akt, which can be directly activated by products of PI3K [116] by promoting its phosphorylation at Ser473 and Thr308 [117]. Activated Akt, in turn, phosphorylates a wide range of substrates activating anti-apoptotic (survival) factors and inactivating pro-apoptotic factors [114, 117]. Certain proapoptotic mediators, such as the transcription factor forkhead (FOXO), the tau kinase GSK-3ÃƒÅ¸, and the Bcl2 antagonist BAD proteins, are inactivated by Akt [118, 119]. Akt downregulates the activities of GSK-3a and GSK-3ÃƒÅ¸ by phosphorylating the former at Ser21 and the latter at Ser9 [118]. Phosphorylation/inactivation of GSK-3ÃƒÅ¸, suppresses GSK-3ÃƒÅ¸-dependent phosphorylation of tau at residues overphosphorylated in AD and prevents apoptosis of confluent cells. Treatment of cortical neurons with AÃƒÅ¸42 monomers increased Ser9 phosphorylation (inhibition) of the Akt substrate, GSK- 3ÃƒÅ¸ [97]. Inhibition of GSK-3ÃƒÅ¸ promotes cell survival through a variety of mechanisms including a reduced degradation of ÃƒÅ¸-catenin, which then translocates into the nucleus and activates the transcription of protective genes [126].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Beta-Catenin subgraph\"}, \"source\": 121, \"target\": 5, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In situ, FRAT-2 significantly increased GSK3 beta-mediated phosphorylation of tau at a primed epitope while not significantly affecting the phosphorylation of unprimed sites.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Biological Chemistry2005\", \"reference\": \"15522877\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 122, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In situ, FRAT-2 significantly increased GSK3 beta-mediated phosphorylation of tau at a primed epitope while not significantly affecting the phosphorylation of unprimed sites.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Biological Chemistry2005\", \"reference\": \"15522877\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 122, \"target\": 5, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with AÃƒÅ¸ is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane ÃƒÅ¸-subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the ÃƒÅ¸-subunit, and causes autophosphorylation of the ÃƒÅ¸-subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against AÃƒÅ¸ toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3ÃƒÅ¸ and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3ÃƒÅ¸ signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with AÃƒÅ¸ activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, AÃƒÅ¸ monomers may bind to IGF-1/insulin receptors, as already shown for AÃƒÅ¸ oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 123, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with AÃƒÅ¸ is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane ÃƒÅ¸-subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the ÃƒÅ¸-subunit, and causes autophosphorylation of the ÃƒÅ¸-subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against AÃƒÅ¸ toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3ÃƒÅ¸ and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3ÃƒÅ¸ signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with AÃƒÅ¸ activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, AÃƒÅ¸ monomers may bind to IGF-1/insulin receptors, as already shown for AÃƒÅ¸ oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Regulation of cytoskeleton subgraph\"}, \"source\": 123, \"target\": 5, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 124, \"target\": 8, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 124, \"target\": 8, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"MicroRNA-146a (miRNA-146a) is an inducible, 22 nucleotide, small RNA over-expressed in Alzheimer's disease (AD) brain. Up-regulated miRNA-146a targets several inflammation-related and membrane-associated messenger RNAs (mRNAs), including those encoding complement factor-H (CFH) and the interleukin-1 receptor associated kinase-1 (IRAK-1), resulting in significant decreases in their expression (p<0.05, ANOVA). In this study we assayed miRNA-146a, CFH, IRAK-1 and tetraspanin-12 (TSPAN12), abundances in primary human neuronal-glial (HNG) co-cultures, in human astroglial (HAG) and microglial (HMG) cells stressed with Aβ42 peptide and tumor necrosis factor alpha (TNFÃƒÅ½Ã‚Â±). The results indicate a consistent inverse relationship between miRNA-146a and CFH, IRAK-1 and TSPAN12 expression levels, and indicate that HNG, HAG and HMG cell types each respond differently to Aβ42-peptide+TNFÃƒÅ½Ã‚Â±-triggered stress. While the strongest miRNA-146a-IRAK-1 response was found in HAG cells, the largest miRNA-146a-TSPAN12 response was found in HNG cells, and the most significant miRNA-146a-CFH changes were found in HMG cells, the 'resident scavenging macrophages' of the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2011 Jul 20;499(2):109-13. doi: 10.1016/j.neulet.2011.05.044. Epub 2011 May 26.\", \"reference\": \"21640790\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 124, \"target\": 8, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"MicroRNA-146a (miRNA-146a) is an inducible, 22 nucleotide, small RNA over-expressed in Alzheimer's disease (AD) brain. Up-regulated miRNA-146a targets several inflammation-related and membrane-associated messenger RNAs (mRNAs), including those encoding complement factor-H (CFH) and the interleukin-1 receptor associated kinase-1 (IRAK-1), resulting in significant decreases in their expression (p<0.05, ANOVA). In this study we assayed miRNA-146a, CFH, IRAK-1 and tetraspanin-12 (TSPAN12), abundances in primary human neuronal-glial (HNG) co-cultures, in human astroglial (HAG) and microglial (HMG) cells stressed with Aβ42 peptide and tumor necrosis factor alpha (TNFÃƒÅ½Ã‚Â±). The results indicate a consistent inverse relationship between miRNA-146a and CFH, IRAK-1 and TSPAN12 expression levels, and indicate that HNG, HAG and HMG cell types each respond differently to Aβ42-peptide+TNFÃƒÅ½Ã‚Â±-triggered stress. While the strongest miRNA-146a-IRAK-1 response was found in HAG cells, the largest miRNA-146a-TSPAN12 response was found in HNG cells, and the most significant miRNA-146a-CFH changes were found in HMG cells, the 'resident scavenging macrophages' of the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2011 Jul 20;499(2):109-13. doi: 10.1016/j.neulet.2011.05.044. Epub 2011 May 26.\", \"reference\": \"21640790\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 124, \"target\": 8, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 124, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Aβ42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Aβ42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 124, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 125, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 125, \"target\": 10, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 125, \"target\": 10, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"CCAAT/enhancer binding protein delta (CEBPD) elevating PTX3 expression inhibits macrophage-mediated phagocytosis of dying neuron cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2012 Feb;33(2):422.e11-25. doi: 10.1016/j.neurobiolaging.2010.09.017. Epub 2010 Nov 26.\", \"reference\": \"21112127\"}, \"annotations\": {\"Subgraph\": \"Innate immune system subgraph\"}, \"source\": 126, \"target\": 11, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We propose that IL-1 and the IL-6 family of cytokines regulate YKL-40 expression during sterile inflammation via both STAT3 and RelB/p50 complexes\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 127, \"target\": 26, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We propose that IL-1 and the IL-6 family of cytokines regulate YKL-40 expression during sterile inflammation via both STAT3 and RelB/p50 complexes\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Subgraph\": \"JAK-STAT signaling subgraph\"}, \"source\": 127, \"target\": 26, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The secreted protein, YKL-40, has been proposed as a biomarker of a variety of human diseases characterized by ongoing inflammation, including chronic neurologic pathologies such as multiple sclerosis and Alzheimer's disease. However, inflammatory mediators and the molecular mechanism responsible for enhanced expression of YKL-40 remained elusive. Using several mouse models of inflammation, we now show that YKL-40 expression correlated with increased expression of both IL-1 and IL-6. Furthermore, IL-1 together with IL-6 or the IL-6 family cytokine, oncostatin M, synergistically upregulated YKL-40 expression in both primary human and mouse astrocytes in vitro. The robust cytokine-driven expression of YKL-40 in astrocytes required both STAT3 and NF-kB binding elements of the YKL-40 promoter. In addition, YKL-40 expression was enhanced by constitutively active STAT3 and inhibited by dominant-negative IkBalpha. Surprisingly, cytokine-driven expression of YKL-40 in astrocytes was independent of the p65 subunit of NF-kB and instead required subunits RelB and p50. Mechanistically, we show that IL-1-induced RelB/p50 complex formation was further promoted by oncostatin M and that these complexes directly bound to the YKL-40 promoter. Moreover, we found that expression of RelB was strongly upregulated during inflammation in vivo and by IL-1 in astrocytes in vitro. We propose that IL-1 and the IL-6 family of cytokines regulate YKL-40 expression during sterile inflammation via both STAT3 and RelB/p50 complexes. These results suggest that IL-1 may regulate the expression of specific anti-inflammatory genes in nonlymphoid tissues via the canonical activation of the RelB/p50 complexes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874.\", \"reference\": \"25681350\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 128, \"target\": 26, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The secreted protein, YKL-40, has been proposed as a biomarker of a variety of human diseases characterized by ongoing inflammation, including chronic neurologic pathologies such as multiple sclerosis and Alzheimer's disease. However, inflammatory mediators and the molecular mechanism responsible for enhanced expression of YKL-40 remained elusive. Using several mouse models of inflammation, we now show that YKL-40 expression correlated with increased expression of both IL-1 and IL-6. Furthermore, IL-1 together with IL-6 or the IL-6 family cytokine, oncostatin M, synergistically upregulated YKL-40 expression in both primary human and mouse astrocytes in vitro. The robust cytokine-driven expression of YKL-40 in astrocytes required both STAT3 and NF-kB binding elements of the YKL-40 promoter. In addition, YKL-40 expression was enhanced by constitutively active STAT3 and inhibited by dominant-negative IkBalpha. Surprisingly, cytokine-driven expression of YKL-40 in astrocytes was independent of the p65 subunit of NF-kB and instead required subunits RelB and p50. Mechanistically, we show that IL-1-induced RelB/p50 complex formation was further promoted by oncostatin M and that these complexes directly bound to the YKL-40 promoter. Moreover, we found that expression of RelB was strongly upregulated during inflammation in vivo and by IL-1 in astrocytes in vitro. We propose that IL-1 and the IL-6 family of cytokines regulate YKL-40 expression during sterile inflammation via both STAT3 and RelB/p50 complexes. These results suggest that IL-1 may regulate the expression of specific anti-inflammatory genes in nonlymphoid tissues via the canonical activation of the RelB/p50 complexes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874.\", \"reference\": \"25681350\"}, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 128, \"target\": 26, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In addition, IL-6 and OSM moderately upregulate YKL-40 expression in human astrocytes [...] demonstrate that YKL-40 expression correlates with the expression of both IL-1β and IL-6\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 128, \"target\": 26, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In addition, IL-6 and OSM moderately upregulate YKL-40 expression in human astrocytes [...] demonstrate that YKL-40 expression correlates with the expression of both IL-1β and IL-6\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 128, \"target\": 26, \"key\": 3}]};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pbt.summary.print_summary(subgraph)\n",
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "This entire procedure can be run automatically with [pbt.generation.generate_mechanism](http://pybel-tools.readthedocs.io/en/latest/generation.html#pybel_tools.generation.generate_mechanism)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "## Running"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "runner = npa.NpaRunner(subgraph, inflamation_node, label)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "While the algorithm can be immediately run with [pbt.analysis.npa.NpaRunner.run](http://pybel-tools.readthedocs.io/en/latest/npa.html#pybel_tools.analysis.npa.NpaRunner.run), the process is outlined to show the evolution of the graph throughouts its steps. For animation purposes, the functin [pbt.analysis.npa.NpaRunner.run_with_graph_transformation](http://pybel-tools.readthedocs.io/en/latest/npa.html#pybel_tools.analysis.npa.NpaRunner.run_with_graph_transformation) yields a new `BELGraph` at each step through the process as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{('Abundance', 'BRCO', 'Astrocyte'),\n",
       " ('Abundance', 'CHEBI', 'leukotriene'),\n",
       " ('Abundance', 'MESHCS', 'Neurofibrillary Tangles'),\n",
       " ('Gene', 'HGNC', 'CD14'),\n",
       " ('Gene', 'HGNC', 'CHI3L1'),\n",
       " ('Gene', 'HGNC', 'IL23A'),\n",
       " ('Gene', 'HGNC', 'IRAK1'),\n",
       " ('Gene', 'HGNC', 'IRAK2'),\n",
       " ('Gene', 'HGNC', 'PTX3'),\n",
       " ('Gene', 'HGNC', 'TLR2')}"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height) {\n",
       "    const function_colors = {\n",
       "        \"Abundance\": \"#AEC7E8\",\n",
       "        \"Gene\": \"#FFBB78\",\n",
       "        \"RNA\": \"#FF9896\",\n",
       "        \"miRNA\": \"#D62728\",\n",
       "        \"Protein\": \"#1F77B4\",\n",
       "        \"Complex\": \"#98DF8A\",\n",
       "        \"Composite\": \"#9467BD\",\n",
       "        \"BiologicalProcess\": \"#2CA02C\",\n",
       "        \"Pathology\": \"#FF7F0E\"\n",
       "    };\n",
       "\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#888\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"eqktpfygsabvdcho\";\n",
       "        var width = 900;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"pybel_version\": \"0.4.1\", \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"weight\": 0.18225965, \"cname\": \"SCARF1\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"weight\": -1.61550531, \"cname\": \"IL12B\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"ÃŸ-Amyloid (AÃŸ) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of AÃŸ, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3ÃŸ, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3ÃŸ after exposure to oligomeric AÃŸ in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3ÃŸ, even in the absence of AÃŸ, is sufficient to produce a phenocopy of AÃŸ-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3ÃŸ prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3ÃŸ inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3ÃŸ inhibition robustly decreased the oligomeric AÃŸ load in the mouse brain. All these findings support the idea that GSK3ÃŸ is aberrantly activated by the presence of AÃŸ, and contributes, at least in part, to the neuronal anatomical derangement associated with AÃŸ plaques in AD brains and to AÃŸ pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"ÃŸ-Amyloid (AÃŸ) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of AÃŸ, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3ÃŸ, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3ÃŸ after exposure to oligomeric AÃŸ in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3ÃŸ, even in the absence of AÃŸ, is sufficient to produce a phenocopy of AÃŸ-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3ÃŸ prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3ÃŸ inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3ÃŸ inhibition robustly decreased the oligomeric AÃŸ load in the mouse brain. All these findings support the idea that GSK3ÃŸ is aberrantly activated by the presence of AÃŸ, and contributes, at least in part, to the neuronal anatomical derangement associated with AÃŸ plaques in AD brains and to AÃŸ pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 0, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 10, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The serum IL-1β and TNF-Î± level were significantly increased, and the expressions of TLR4 and NF-ÎºB p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Aβ1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"NervousSystem\": \"Brain\", \"FluidAndSecretion\": \"Serum\"}, \"source\": 1, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Aβ) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Aβ) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 7, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 1, \"key\": 0}]};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{('Gene', 'HGNC', 'IL12B')}"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height) {\n",
       "    const function_colors = {\n",
       "        \"Abundance\": \"#AEC7E8\",\n",
       "        \"Gene\": \"#FFBB78\",\n",
       "        \"RNA\": \"#FF9896\",\n",
       "        \"miRNA\": \"#D62728\",\n",
       "        \"Protein\": \"#1F77B4\",\n",
       "        \"Complex\": \"#98DF8A\",\n",
       "        \"Composite\": \"#9467BD\",\n",
       "        \"BiologicalProcess\": \"#2CA02C\",\n",
       "        \"Pathology\": \"#FF7F0E\"\n",
       "    };\n",
       "\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#888\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"gefkpjuthdzvyrba\";\n",
       "        var width = 900;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"pybel_version\": \"0.4.1\", \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"weight\": 0.18225965, \"cname\": \"SCARF1\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"ÃŸ-Amyloid (AÃŸ) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of AÃŸ, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3ÃŸ, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3ÃŸ after exposure to oligomeric AÃŸ in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3ÃŸ, even in the absence of AÃŸ, is sufficient to produce a phenocopy of AÃŸ-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3ÃŸ prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3ÃŸ inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3ÃŸ inhibition robustly decreased the oligomeric AÃŸ load in the mouse brain. All these findings support the idea that GSK3ÃŸ is aberrantly activated by the presence of AÃŸ, and contributes, at least in part, to the neuronal anatomical derangement associated with AÃŸ plaques in AD brains and to AÃŸ pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"ÃŸ-Amyloid (AÃŸ) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of AÃŸ, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3ÃŸ, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3ÃŸ after exposure to oligomeric AÃŸ in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3ÃŸ, even in the absence of AÃŸ, is sufficient to produce a phenocopy of AÃŸ-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3ÃŸ prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3ÃŸ inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3ÃŸ inhibition robustly decreased the oligomeric AÃŸ load in the mouse brain. All these findings support the idea that GSK3ÃŸ is aberrantly activated by the presence of AÃŸ, and contributes, at least in part, to the neuronal anatomical derangement associated with AÃŸ plaques in AD brains and to AÃŸ pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 0, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The serum IL-1β and TNF-Î± level were significantly increased, and the expressions of TLR4 and NF-ÎºB p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Aβ1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"NervousSystem\": \"Brain\", \"FluidAndSecretion\": \"Serum\"}, \"source\": 1, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Aβ) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Aβ) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 7, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 0}]};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.npa:no leaves.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height) {\n",
       "    const function_colors = {\n",
       "        \"Abundance\": \"#AEC7E8\",\n",
       "        \"Gene\": \"#FFBB78\",\n",
       "        \"RNA\": \"#FF9896\",\n",
       "        \"miRNA\": \"#D62728\",\n",
       "        \"Protein\": \"#1F77B4\",\n",
       "        \"Complex\": \"#98DF8A\",\n",
       "        \"Composite\": \"#9467BD\",\n",
       "        \"BiologicalProcess\": \"#2CA02C\",\n",
       "        \"Pathology\": \"#FF7F0E\"\n",
       "    };\n",
       "\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#888\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"qoprbtxzgqjmehsd\";\n",
       "        var width = 900;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"pybel_version\": \"0.4.1\", \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"weight\": 0.18225965, \"cname\": \"SCARF1\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"ÃŸ-Amyloid (AÃŸ) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of AÃŸ, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3ÃŸ, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3ÃŸ after exposure to oligomeric AÃŸ in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3ÃŸ, even in the absence of AÃŸ, is sufficient to produce a phenocopy of AÃŸ-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3ÃŸ prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3ÃŸ inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3ÃŸ inhibition robustly decreased the oligomeric AÃŸ load in the mouse brain. All these findings support the idea that GSK3ÃŸ is aberrantly activated by the presence of AÃŸ, and contributes, at least in part, to the neuronal anatomical derangement associated with AÃŸ plaques in AD brains and to AÃŸ pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"ÃŸ-Amyloid (AÃŸ) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of AÃŸ, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3ÃŸ, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3ÃŸ after exposure to oligomeric AÃŸ in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3ÃŸ, even in the absence of AÃŸ, is sufficient to produce a phenocopy of AÃŸ-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3ÃŸ prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3ÃŸ inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3ÃŸ inhibition robustly decreased the oligomeric AÃŸ load in the mouse brain. All these findings support the idea that GSK3ÃŸ is aberrantly activated by the presence of AÃŸ, and contributes, at least in part, to the neuronal anatomical derangement associated with AÃŸ plaques in AD brains and to AÃŸ pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 0, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The serum IL-1β and TNF-Î± level were significantly increased, and the expressions of TLR4 and NF-ÎºB p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Aβ1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"NervousSystem\": \"Brain\", \"FluidAndSecretion\": \"Serum\"}, \"source\": 1, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Aβ) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Aβ) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 7, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 0}]};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.remove_random_edge()\n",
    "\n",
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{('Gene', 'HGNC', 'SCARB1')}"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.npa:no leaves.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.remove_random_edge()\n",
    "leaves = runner.score_leaves()\n",
    "len(leaves)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height) {\n",
       "    const function_colors = {\n",
       "        \"Abundance\": \"#AEC7E8\",\n",
       "        \"Gene\": \"#FFBB78\",\n",
       "        \"RNA\": \"#FF9896\",\n",
       "        \"miRNA\": \"#D62728\",\n",
       "        \"Protein\": \"#1F77B4\",\n",
       "        \"Complex\": \"#98DF8A\",\n",
       "        \"Composite\": \"#9467BD\",\n",
       "        \"BiologicalProcess\": \"#2CA02C\",\n",
       "        \"Pathology\": \"#FF7F0E\"\n",
       "    };\n",
       "\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#888\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"uehmbfvqztaoykrj\";\n",
       "        var width = 900;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"pybel_version\": \"0.4.1\", \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Aβ induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"NervousSystem\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"ÃŸ-Amyloid (AÃŸ) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of AÃŸ, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3ÃŸ, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3ÃŸ after exposure to oligomeric AÃŸ in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3ÃŸ, even in the absence of AÃŸ, is sufficient to produce a phenocopy of AÃŸ-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3ÃŸ prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3ÃŸ inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3ÃŸ inhibition robustly decreased the oligomeric AÃŸ load in the mouse brain. All these findings support the idea that GSK3ÃŸ is aberrantly activated by the presence of AÃŸ, and contributes, at least in part, to the neuronal anatomical derangement associated with AÃŸ plaques in AD brains and to AÃŸ pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"ÃŸ-Amyloid (AÃŸ) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of AÃŸ, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3ÃŸ, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3ÃŸ after exposure to oligomeric AÃŸ in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3ÃŸ, even in the absence of AÃŸ, is sufficient to produce a phenocopy of AÃŸ-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3ÃŸ prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3ÃŸ inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3ÃŸ inhibition robustly decreased the oligomeric AÃŸ load in the mouse brain. All these findings support the idea that GSK3ÃŸ is aberrantly activated by the presence of AÃŸ, and contributes, at least in part, to the neuronal anatomical derangement associated with AÃŸ plaques in AD brains and to AÃŸ pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 2, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 0, \"target\": 3, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 0, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 0, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The serum IL-1β and TNF-Î± level were significantly increased, and the expressions of TLR4 and NF-ÎºB p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Aβ1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"NervousSystem\": \"Brain\", \"FluidAndSecretion\": \"Serum\"}, \"source\": 1, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Aβ) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Aβ) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1ÃŸ, and show potentiation by AÃŸ. Potentiation of the inflammatory response of AÃŸ could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by AÃŸ.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 0}]};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "runner.run()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "74.162180860000021"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.get_final_score()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "## Robustness Analysis\n",
    "\n",
    "The algorithm is random, so it can be run multiple times to assess the robustness of the final scores using [pbt.analysis.npa.multirun](http://pybel-tools.readthedocs.io/en/latest/npa.html#pybel_tools.analysis.npa.multirun)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 22.1 s, sys: 281 ms, total: 22.4 s\n",
      "Wall time: 22.4 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "runners = list(npa.multirun(subgraph, inflamation_node, label, default_score=0, runs=500))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"274pt\" version=\"1.1\" viewBox=\"0 0 387 274\" width=\"387pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 274.853533 \n",
       "L 387.499682 274.853533 \n",
       "L 387.499682 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 41.999682 238.94218 \n",
       "L 376.799682 238.94218 \n",
       "L 376.799682 21.50218 \n",
       "L 41.999682 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#pc3340fef23)\" d=\"M 57.217864 238.94218 \n",
       "L 87.654228 238.94218 \n",
       "L 87.654228 31.856466 \n",
       "L 57.217864 31.856466 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#pc3340fef23)\" d=\"M 87.654228 238.94218 \n",
       "L 118.090592 238.94218 \n",
       "L 118.090592 172.236685 \n",
       "L 87.654228 172.236685 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#pc3340fef23)\" d=\"M 118.090592 238.94218 \n",
       "L 148.526955 238.94218 \n",
       "L 148.526955 146.350971 \n",
       "L 118.090592 146.350971 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#pc3340fef23)\" d=\"M 148.526955 238.94218 \n",
       "L 178.963319 238.94218 \n",
       "L 178.963319 197.126795 \n",
       "L 148.526955 197.126795 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#pc3340fef23)\" d=\"M 178.963319 238.94218 \n",
       "L 209.399682 238.94218 \n",
       "L 209.399682 203.100422 \n",
       "L 178.963319 203.100422 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#pc3340fef23)\" d=\"M 209.399682 238.94218 \n",
       "L 239.836046 238.94218 \n",
       "L 239.836046 213.056466 \n",
       "L 209.399682 213.056466 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#pc3340fef23)\" d=\"M 239.836046 238.94218 \n",
       "L 270.27241 238.94218 \n",
       "L 270.27241 224.008114 \n",
       "L 239.836046 224.008114 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#pc3340fef23)\" d=\"M 270.27241 238.94218 \n",
       "L 300.708773 238.94218 \n",
       "L 300.708773 230.977345 \n",
       "L 270.27241 230.977345 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#pc3340fef23)\" d=\"M 300.708773 238.94218 \n",
       "L 331.145137 238.94218 \n",
       "L 331.145137 236.950971 \n",
       "L 300.708773 236.950971 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#pc3340fef23)\" d=\"M 331.145137 238.94218 \n",
       "L 361.581501 238.94218 \n",
       "L 361.581501 235.955367 \n",
       "L 331.145137 235.955367 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"ma7fa76d6e0\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"64.414346\" xlink:href=\"#ma7fa76d6e0\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- $70$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 45.09375 61.578125 \n",
       "L 45.09375 63.765625 \n",
       "L 21.59375 63.765625 \n",
       "C 9.90625 63.765625 9.703125 65.078125 9.296875 67 \n",
       "L 7.5 67 \n",
       "L 4.59375 48.34375 \n",
       "L 6.40625 48.34375 \n",
       "C 6.703125 50.234375 7.40625 55.3125 8.59375 57.203125 \n",
       "C 9.203125 58 16.59375 58 18.296875 58 \n",
       "L 40.296875 58 \n",
       "L 29.203125 41.578125 \n",
       "C 21.796875 30.53125 16.703125 15.609375 16.703125 2.578125 \n",
       "C 16.703125 1.390625 16.703125 -2 20.296875 -2 \n",
       "C 23.90625 -2 23.90625 1.390625 23.90625 2.671875 \n",
       "L 23.90625 7.265625 \n",
       "C 23.90625 23.5625 26.703125 34.3125 31.296875 41.1875 \n",
       "z\n",
       "\" id=\"CMR17-55\"/>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "\" id=\"CMR17-48\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(59.433011 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"108.007189\" xlink:href=\"#ma7fa76d6e0\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- $80$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 27.203125 35.375 \n",
       "C 33.5 38.546875 39.90625 43.328125 39.90625 50.984375 \n",
       "C 39.90625 60.015625 31.09375 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 58.734375 5.90625 49.6875 \n",
       "C 5.90625 47.203125 6.5 42.921875 10.40625 39.140625 \n",
       "C 11.40625 38.15625 15.59375 35.171875 18.296875 33.28125 \n",
       "C 13.796875 30.984375 3.296875 25.53125 3.296875 14.59375 \n",
       "C 3.296875 4.359375 13.09375 -2 22.796875 -2 \n",
       "C 33.5 -2 42.5 5.65625 42.5 15.78125 \n",
       "C 42.5 24.84375 36.40625 29.015625 32.40625 31.703125 \n",
       "z\n",
       "M 14.09375 44.09375 \n",
       "C 13.296875 44.59375 9.296875 47.671875 9.296875 52.359375 \n",
       "C 9.296875 58.421875 15.59375 63 22.796875 63 \n",
       "C 30.703125 63 36.5 57.4375 36.5 50.96875 \n",
       "C 36.5 41.703125 26.09375 36.421875 25.59375 36.421875 \n",
       "C 25.5 36.421875 25.40625 36.421875 24.59375 37.03125 \n",
       "z\n",
       "M 32.5 23.734375 \n",
       "C 34 22.640625 38.796875 19.375 38.796875 13.296875 \n",
       "C 38.796875 5.953125 31.40625 0.390625 23 0.390625 \n",
       "C 13.90625 0.390625 7 6.84375 7 14.6875 \n",
       "C 7 22.546875 13.09375 29.09375 20 32.1875 \n",
       "z\n",
       "\" id=\"CMR17-56\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(103.025854 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"151.600032\" xlink:href=\"#ma7fa76d6e0\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- $90$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 35.203125 29.125 \n",
       "C 35.203125 7.25 26.09375 0.484375 18.59375 0.484375 \n",
       "C 16.296875 0.484375 10.703125 0.484375 8.40625 4.59375 \n",
       "C 11 4.1875 12.90625 5.578125 12.90625 8.09375 \n",
       "C 12.90625 10.8125 10.703125 11.703125 9.296875 11.703125 \n",
       "C 8.40625 11.703125 5.703125 11.296875 5.703125 7.90625 \n",
       "C 5.703125 1.109375 11.59375 -2 18.796875 -2 \n",
       "C 30.296875 -2 42 10.53125 42 32.390625 \n",
       "C 42 59.640625 30.796875 65.296875 23.203125 65.296875 \n",
       "C 13.296875 65.296875 3.796875 56.6875 3.796875 43.40625 \n",
       "C 3.796875 31.109375 11.90625 22 22.09375 22 \n",
       "C 30.5 22 34.203125 29.734375 35.203125 33 \n",
       "z\n",
       "M 22.296875 23.59375 \n",
       "C 19.59375 23.59375 15.796875 24.09375 12.703125 30.03125 \n",
       "C 10.59375 33.890625 10.59375 38.453125 10.59375 43.296875 \n",
       "C 10.59375 49.15625 10.59375 53.203125 13.40625 57.5625 \n",
       "C 14.796875 59.640625 17.40625 63 23.203125 63 \n",
       "C 35 63 35 45.078125 35 41.125 \n",
       "C 35 34.09375 31.796875 23.59375 22.296875 23.59375 \n",
       "\" id=\"CMR17-57\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(146.618696 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-57\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"195.192875\" xlink:href=\"#ma7fa76d6e0\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- $100$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(187.720872 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"238.785717\" xlink:href=\"#ma7fa76d6e0\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- $110$ -->\n",
       "      <g transform=\"translate(231.313715 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"282.37856\" xlink:href=\"#ma7fa76d6e0\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_6\">\n",
       "      <!-- $120$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(274.906557 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_7\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"325.971403\" xlink:href=\"#ma7fa76d6e0\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- $130$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 22.09375 33.703125 \n",
       "C 31 33.703125 34.90625 25.96875 34.90625 17.03125 \n",
       "C 34.90625 5 28.5 0.390625 22.703125 0.390625 \n",
       "C 17.40625 0.390625 8.796875 3.03125 6.09375 10.828125 \n",
       "C 6.59375 10.625 7.09375 10.625 7.59375 10.625 \n",
       "C 10 10.625 11.796875 12.21875 11.796875 14.8125 \n",
       "C 11.796875 17.703125 9.59375 19 7.59375 19 \n",
       "C 5.90625 19 3.296875 18.203125 3.296875 14.46875 \n",
       "C 3.296875 5.234375 12.296875 -2 22.90625 -2 \n",
       "C 34 -2 42.5 6.71875 42.5 16.953125 \n",
       "C 42.5 26.671875 34.5 33.703125 25 34.796875 \n",
       "C 32.59375 36.375 39.90625 43.0625 39.90625 52.015625 \n",
       "C 39.90625 59.6875 32 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 59.828125 5.90625 52.015625 \n",
       "C 5.90625 48.59375 8.5 48 9.796875 48 \n",
       "C 11.90625 48 13.703125 49.296875 13.703125 51.890625 \n",
       "C 13.703125 54.46875 11.90625 55.765625 9.796875 55.765625 \n",
       "C 9.40625 55.765625 8.90625 55.765625 8.5 55.578125 \n",
       "C 11.40625 61.859375 19.296875 63 22.796875 63 \n",
       "C 26.296875 63 32.90625 61.328125 32.90625 51.875 \n",
       "C 32.90625 49.109375 32.5 44.1875 29.09375 39.84375 \n",
       "C 26.09375 36 22.703125 36 19.40625 35.6875 \n",
       "C 18.90625 35.6875 16.59375 35.453125 16.203125 35.453125 \n",
       "C 15.5 35.359375 15.09375 35.265625 15.09375 34.5 \n",
       "C 15.09375 33.796875 15.203125 33.703125 17.203125 33.703125 \n",
       "z\n",
       "\" id=\"CMR17-51\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(318.4994 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-51\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_8\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"369.564246\" xlink:href=\"#ma7fa76d6e0\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- $140$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(362.092243 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_9\">\n",
       "     <!-- Score -->\n",
       "     <defs>\n",
       "      <path d=\"M 43.796875 67.03125 \n",
       "C 43.796875 68.8125 43.703125 68.921875 43.09375 68.921875 \n",
       "C 42.5 68.921875 42.296875 68.515625 41.90625 67.625 \n",
       "L 38.90625 61.046875 \n",
       "C 35.09375 66.515625 29.296875 69 23 69 \n",
       "C 12.796875 69 4.59375 60.953125 4.59375 50.828125 \n",
       "C 4.59375 47.75 5.296875 43.375 8.703125 39.21875 \n",
       "C 12.40625 34.65625 15.90625 33.859375 23.90625 31.765625 \n",
       "C 26.90625 30.96875 31.90625 29.671875 32.90625 29.28125 \n",
       "C 38.59375 26.90625 41.59375 20.953125 41.59375 15.390625 \n",
       "C 41.59375 8.234375 36.5 0.59375 27.796875 0.59375 \n",
       "C 17.09375 0.59375 7.09375 6.546875 6.40625 20.484375 \n",
       "C 6.296875 22.078125 6.203125 22.171875 5.5 22.171875 \n",
       "C 4.703125 22.171875 4.59375 22.078125 4.59375 20.078125 \n",
       "L 4.59375 -0.015625 \n",
       "C 4.59375 -1.796875 4.703125 -1.90625 5.296875 -1.90625 \n",
       "C 6 -1.90625 6.203125 -1.40625 7.59375 1.6875 \n",
       "C 7.703125 1.984375 7.703125 2.1875 9.59375 5.953125 \n",
       "C 14.59375 -0.609375 23 -2 27.796875 -2 \n",
       "C 38.703125 -2 46.40625 7.046875 46.40625 17.5625 \n",
       "C 46.40625 22.71875 44.5 26.6875 43.40625 28.484375 \n",
       "C 39.203125 34.84375 34.703125 36.03125 29.90625 37.21875 \n",
       "C 29.203125 37.53125 29 37.53125 23.40625 38.90625 \n",
       "C 18 40.296875 15.703125 41 13.203125 43.6875 \n",
       "C 10.40625 46.75 9.40625 49.9375 9.40625 53.109375 \n",
       "C 9.40625 59.5625 14.59375 66.609375 23.09375 66.609375 \n",
       "C 33.703125 66.609375 40.296875 59.15625 41.796875 46.765625 \n",
       "C 42.09375 44.96875 42.203125 44.875 42.90625 44.875 \n",
       "C 43.796875 44.875 43.796875 45.171875 43.796875 46.859375 \n",
       "z\n",
       "\" id=\"CMR17-83\"/>\n",
       "      <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "\" id=\"CMR17-99\"/>\n",
       "      <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "\" id=\"CMR17-111\"/>\n",
       "      <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "      <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(197.762748 265.716355)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(91.38103 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(137.071507 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(172.352275 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"m0730b7508d\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m0730b7508d\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(30.018347 242.401429)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m0730b7508d\" y=\"214.05207\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- $25$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 11.40625 57.6875 \n",
       "C 12.40625 57.28125 16.5 56 20.703125 56 \n",
       "C 30 56 35.09375 61.109375 38 64.0625 \n",
       "C 38 64.953125 38 65.5 37.40625 65.5 \n",
       "C 37.296875 65.5 37.09375 65.5 36.296875 65.046875 \n",
       "C 32.796875 63.40625 28.703125 62.078125 23.703125 62.078125 \n",
       "C 20.703125 62.078125 16.203125 62.484375 11.296875 64.671875 \n",
       "C 10.203125 65.171875 10 65.171875 9.90625 65.171875 \n",
       "C 9.40625 65.171875 9.296875 65.0625 9.296875 63.078125 \n",
       "L 9.296875 34.421875 \n",
       "C 9.296875 32.640625 9.296875 32.140625 10.296875 32.140625 \n",
       "C 10.796875 32.140625 11 32.34375 11.5 33.03125 \n",
       "C 14.703125 37.515625 19.09375 39.40625 24.09375 39.40625 \n",
       "C 27.59375 39.40625 35.09375 37.21875 35.09375 20.140625 \n",
       "C 35.09375 16.953125 35.09375 11.171875 32.09375 6.578125 \n",
       "C 29.59375 2.484375 25.703125 0.390625 21.40625 0.390625 \n",
       "C 14.796875 0.390625 8.09375 5 6.296875 12.71875 \n",
       "C 6.703125 12.609375 7.5 12.421875 7.90625 12.421875 \n",
       "C 9.203125 12.421875 11.703125 13.125 11.703125 16.21875 \n",
       "C 11.703125 18.90625 9.796875 20 7.90625 20 \n",
       "C 5.59375 20 4.09375 18.59375 4.09375 15.796875 \n",
       "C 4.09375 7.09375 11 -2 21.59375 -2 \n",
       "C 31.90625 -2 41.703125 6.875 41.703125 19.75 \n",
       "C 41.703125 31.71875 33.90625 41 24.203125 41 \n",
       "C 19.09375 41 14.796875 39.109375 11.40625 35.53125 \n",
       "z\n",
       "\" id=\"CMR17-53\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(25.037012 217.511319)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m0730b7508d\" y=\"189.16196\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_12\">\n",
       "      <!-- $50$ -->\n",
       "      <g transform=\"translate(25.037012 192.621209)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m0730b7508d\" y=\"164.27185\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_13\">\n",
       "      <!-- $75$ -->\n",
       "      <g transform=\"translate(25.037012 167.731099)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m0730b7508d\" y=\"139.38174\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_14\">\n",
       "      <!-- $100$ -->\n",
       "      <g transform=\"translate(20.055677 142.840989)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_14\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m0730b7508d\" y=\"114.49163\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- $125$ -->\n",
       "      <g transform=\"translate(20.055677 117.95088)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_7\">\n",
       "     <g id=\"line2d_15\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m0730b7508d\" y=\"89.60152\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- $150$ -->\n",
       "      <g transform=\"translate(20.055677 93.06077)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_8\">\n",
       "     <g id=\"line2d_16\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m0730b7508d\" y=\"64.711411\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- $175$ -->\n",
       "      <g transform=\"translate(20.055677 68.17066)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_9\">\n",
       "     <g id=\"line2d_17\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m0730b7508d\" y=\"39.821301\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_18\">\n",
       "      <!-- $200$ -->\n",
       "      <g transform=\"translate(20.055677 43.28055)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_19\">\n",
       "     <!-- Frequency -->\n",
       "     <defs>\n",
       "      <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      <path d=\"M 41.09375 43.78125 \n",
       "L 39.5 43.78125 \n",
       "L 36.5 33.9375 \n",
       "C 34.203125 39.203125 30 43.78125 23.296875 43.78125 \n",
       "C 13.09375 43.78125 3.296875 34.734375 3.296875 21.609375 \n",
       "C 3.296875 8.5625 12.5 -0.59375 22.5 -0.59375 \n",
       "C 30.203125 -0.59375 34.5 5.28125 35.796875 7.46875 \n",
       "L 35.796875 -12.328125 \n",
       "C 35.796875 -16.109375 35.40625 -16.609375 29 -16.609375 \n",
       "L 29 -19.1875 \n",
       "C 31.40625 -19 35.796875 -19 38.40625 -19 \n",
       "C 41 -19 45.5 -19 47.90625 -19.1875 \n",
       "L 47.90625 -16.609375 \n",
       "C 41.5 -16.609375 41.09375 -16.203125 41.09375 -12.328125 \n",
       "z\n",
       "M 36 13.84375 \n",
       "C 36 9.5625 32.40625 5.578125 32.09375 5.28125 \n",
       "C 29 1.796875 25.40625 1 23 1 \n",
       "C 15.5 1 9.703125 10.453125 9.703125 21.484375 \n",
       "C 9.703125 33.140625 16.296875 42 24 42 \n",
       "C 32 42 36 32.9375 36 27.765625 \n",
       "z\n",
       "\" id=\"CMR17-113\"/>\n",
       "      <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "      <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "      <path d=\"M 39 34.328125 \n",
       "C 41.5 40.40625 46 40.40625 47.40625 40.40625 \n",
       "L 47.40625 43 \n",
       "C 45.5 43 43 43 41.09375 43 \n",
       "C 39 43 35.703125 43 33.703125 43.09375 \n",
       "L 33.703125 40.5 \n",
       "C 37.5 40.203125 37.59375 37.40625 37.59375 36.609375 \n",
       "C 37.59375 35.625 37.40625 35.125 36.90625 33.921875 \n",
       "L 26 7.015625 \n",
       "L 14.09375 36.03125 \n",
       "C 13.59375 37.21875 13.59375 37.90625 13.59375 38.015625 \n",
       "C 13.59375 40.203125 15.90625 40.40625 18.296875 40.40625 \n",
       "L 18.296875 43.09375 \n",
       "C 15.90625 43 11.59375 43 9.09375 43 \n",
       "C 6.40625 43 3.203125 43 1 43.09375 \n",
       "L 1 40.40625 \n",
       "C 6.40625 40.40625 7 39.90625 8.296875 36.71875 \n",
       "L 23.203125 0.234375 \n",
       "C 18.703125 -11.53125 16.09375 -18.40625 9.796875 -18.40625 \n",
       "C 8.703125 -18.40625 6.203125 -18.109375 4.40625 -16.3125 \n",
       "C 6.703125 -16.109375 7.703125 -14.71875 7.703125 -13.015625 \n",
       "C 7.703125 -11.328125 6.5 -9.828125 4.5 -9.828125 \n",
       "C 2.296875 -9.828125 1.203125 -11.328125 1.203125 -13.125 \n",
       "C 1.203125 -17.109375 5.296875 -20 9.796875 -20 \n",
       "C 15.59375 -20 19.203125 -14.40625 21.203125 -9.4375 \n",
       "z\n",
       "\" id=\"CMR17-121\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(14.118498 152.444442)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(52.176591 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(87.457359 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(127.943019 0)scale(0.996264)\" xlink:href=\"#CMR17-113\"/>\n",
       "      <use transform=\"translate(176.235935 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(227.131305 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(267.616966 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      <use transform=\"translate(318.512335 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(358.997996 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 41.999682 238.94218 \n",
       "L 41.999682 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 376.799682 238.94218 \n",
       "L 376.799682 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 41.999682 238.94218 \n",
       "L 376.799682 238.94218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 41.999682 21.50218 \n",
       "L 376.799682 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_20\">\n",
       "    <!-- Histogram of NPA scores for inflammatory response -->\n",
       "    <defs>\n",
       "     <path d=\"M 56.09375 61.015625 \n",
       "C 56.09375 65.5 57.203125 65.5 64.09375 65.5 \n",
       "L 64.09375 68 \n",
       "C 61.203125 68 55.90625 68 52.796875 68 \n",
       "C 49.703125 68 44.40625 68 41.5 68 \n",
       "L 41.5 65.5 \n",
       "C 48.40625 65.5 49.5 65.5 49.5 61 \n",
       "L 49.5 36.578125 \n",
       "L 19.703125 36.578125 \n",
       "L 19.703125 61 \n",
       "C 19.703125 65.5 20.796875 65.5 27.703125 65.5 \n",
       "L 27.703125 68 \n",
       "C 24.796875 68 19.5 68 16.40625 68 \n",
       "C 13.296875 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 6.984375 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.296875 0 16.40625 0 \n",
       "C 19.5 0 24.796875 0 27.703125 0 \n",
       "L 27.703125 2.5 \n",
       "C 20.796875 2.5 19.703125 2.5 19.703125 6.96875 \n",
       "L 19.703125 34 \n",
       "L 49.5 34 \n",
       "L 49.5 6.96875 \n",
       "C 49.5 2.5 48.40625 2.5 41.5 2.5 \n",
       "L 41.5 0 \n",
       "C 44.40625 0 49.703125 0 52.796875 0 \n",
       "C 55.90625 0 61.203125 0 64.09375 0 \n",
       "L 64.09375 2.5 \n",
       "C 57.203125 2.5 56.09375 2.5 56.09375 6.984375 \n",
       "z\n",
       "\" id=\"CMR17-72\"/>\n",
       "     <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "     <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "     <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "     <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "\" id=\"CMR17-103\"/>\n",
       "     <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "     <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "     <path d=\"M 15.296875 40 \n",
       "L 26.703125 40 \n",
       "L 26.703125 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 54.59375 \n",
       "C 15.09375 62.84375 19.296875 68 23.59375 68 \n",
       "C 24.90625 68 26.40625 67.609375 27.40625 66.90625 \n",
       "C 26.59375 66.703125 24.5 66.015625 24.5 63.546875 \n",
       "C 24.5 60.953125 26.5 60.15625 27.90625 60.15625 \n",
       "C 29.296875 60.15625 31.296875 60.953125 31.296875 63.546875 \n",
       "C 31.296875 67.3125 27.703125 69.59375 23.703125 69.59375 \n",
       "C 18 69.59375 10.203125 64.9375 10.203125 54.390625 \n",
       "L 10.203125 42.578125 \n",
       "L 2.203125 42.578125 \n",
       "L 2.203125 40 \n",
       "L 10.203125 40 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.40625 0 13 0 \n",
       "C 15.90625 0 21 0 23.703125 -0.203125 \n",
       "L 23.703125 2.390625 \n",
       "C 16.5 2.390625 15.296875 2.390625 15.296875 6.859375 \n",
       "z\n",
       "\" id=\"CMR17-102\"/>\n",
       "     <path d=\"M 20.5 66.40625 \n",
       "C 19.796875 67.5 19.703125 67.5 17.796875 67.5 \n",
       "L 5.09375 67.5 \n",
       "L 5.09375 65 \n",
       "L 7 65 \n",
       "C 11 65 12.796875 64.515625 13.09375 64.40625 \n",
       "L 13.09375 9.53125 \n",
       "C 13.09375 7.046875 13.09375 2.390625 5.09375 2.390625 \n",
       "L 5.09375 -0.203125 \n",
       "C 7.40625 0 11.59375 0 14.09375 0 \n",
       "C 16.59375 0 20.796875 0 23.09375 -0.203125 \n",
       "L 23.09375 2.390625 \n",
       "C 15.09375 2.390625 15.09375 7.046875 15.09375 9.53125 \n",
       "L 15.09375 63.015625 \n",
       "C 15.796875 62.421875 15.796875 62.21875 16.5 61.234375 \n",
       "L 53.59375 1.1875 \n",
       "C 54.296875 0 54.59375 0 55.09375 0 \n",
       "C 56.09375 0 56.09375 0.296875 56.09375 2.171875 \n",
       "L 56.09375 57.890625 \n",
       "C 56.09375 60.359375 56.09375 65 64.09375 65 \n",
       "L 64.09375 67.59375 \n",
       "C 61.796875 67.5 57.59375 67.5 55.09375 67.5 \n",
       "C 52.59375 67.5 48.40625 67.5 46.09375 67.59375 \n",
       "L 46.09375 65 \n",
       "C 54.09375 65 54.09375 60.34375 54.09375 57.859375 \n",
       "L 54.09375 11.921875 \n",
       "z\n",
       "\" id=\"CMR17-78\"/>\n",
       "     <path d=\"M 19.796875 31.90625 \n",
       "L 35.796875 31.90625 \n",
       "C 48.703125 31.90625 58.09375 40.28125 58.09375 49.75 \n",
       "C 58.09375 59.234375 48.90625 68 35.796875 68 \n",
       "L 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.921875 \n",
       "L 13.203125 6.96875 \n",
       "C 13.203125 2.5 12.09375 2.5 5.203125 2.5 \n",
       "L 5.203125 0 \n",
       "C 8.09375 0 13.40625 0 16.5 0 \n",
       "C 19.59375 0 24.90625 0 27.796875 0 \n",
       "L 27.796875 2.5 \n",
       "C 20.90625 2.5 19.796875 2.5 19.796875 6.96875 \n",
       "z\n",
       "M 19.59375 34 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 33.90625 65.40625 \n",
       "C 46.796875 65.40625 50.5 58.03125 50.5 49.75 \n",
       "C 50.5 40.6875 46.09375 34 33.90625 34 \n",
       "z\n",
       "\" id=\"CMR17-80\"/>\n",
       "     <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "     <path d=\"M 39.90625 68.890625 \n",
       "C 38.796875 68.6875 37.90625 68.5 36.59375 68.203125 \n",
       "C 33.703125 69.6875 29.90625 69.6875 28.90625 69.6875 \n",
       "C 20.40625 69.6875 9.90625 64.90625 9.90625 54.25 \n",
       "L 9.90625 42.578125 \n",
       "L 1.90625 42.578125 \n",
       "L 1.90625 40 \n",
       "L 9.90625 40 \n",
       "L 9.90625 6.671875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 9.90625 0 12.40625 0 \n",
       "C 15 0 19.40625 0 21.796875 -0.203125 \n",
       "L 21.796875 2.390625 \n",
       "C 15.40625 2.390625 15 2.78125 15 6.671875 \n",
       "L 15 40 \n",
       "L 34.796875 40 \n",
       "L 34.796875 6.671875 \n",
       "C 34.796875 2.875 34.40625 2.390625 28 2.390625 \n",
       "L 28 -0.203125 \n",
       "C 30.40625 0 34.796875 0 37.296875 0 \n",
       "C 39.90625 0 44.296875 0 46.703125 -0.203125 \n",
       "L 46.703125 2.390625 \n",
       "C 40.296875 2.390625 39.90625 2.78125 39.90625 6.671875 \n",
       "z\n",
       "M 14.796875 42.578125 \n",
       "L 14.796875 54.203125 \n",
       "C 14.796875 62.9375 21.40625 68 28.703125 68 \n",
       "C 29.203125 68 34.703125 68 34.703125 66.90625 \n",
       "C 34.703125 66.796875 34.59375 66.703125 34.09375 66.203125 \n",
       "C 33.203125 65.609375 32.796875 64.609375 32.796875 63.609375 \n",
       "C 32.796875 62.203125 33.5 61.015625 34.796875 60.515625 \n",
       "L 34.796875 42.578125 \n",
       "z\n",
       "\" id=\"CMR17-13\"/>\n",
       "     <path d=\"M 22 -16.609375 \n",
       "C 15.59375 -16.609375 15.203125 -16.203125 15.203125 -12.328125 \n",
       "L 15.203125 6.28125 \n",
       "C 18.09375 2.09375 22.296875 -0.59375 27.59375 -0.59375 \n",
       "C 37.59375 -0.59375 47.703125 8.171875 47.703125 21.6875 \n",
       "C 47.703125 34.234375 39 43.78125 28.703125 43.78125 \n",
       "C 22.703125 43.78125 17.796875 40.5 15 36.53125 \n",
       "L 15 43.78125 \n",
       "L 3.09375 42.6875 \n",
       "L 3.09375 40.109375 \n",
       "C 9 40.109375 9.90625 39.5 9.90625 34.734375 \n",
       "L 9.90625 -12.328125 \n",
       "C 9.90625 -16.109375 9.5 -16.609375 3.09375 -16.609375 \n",
       "L 3.09375 -19.1875 \n",
       "C 5.5 -19 9.90625 -19 12.5 -19 \n",
       "C 15.09375 -19 19.59375 -19 22 -19.1875 \n",
       "z\n",
       "M 15.203125 31.4375 \n",
       "C 15.203125 32.84375 15.203125 34.921875 19.09375 38.609375 \n",
       "C 19.59375 39.015625 23 42 28 42 \n",
       "C 35.296875 42 41.296875 32.9375 41.296875 21.59375 \n",
       "C 41.296875 10.25 34.90625 1 27 1 \n",
       "C 23.40625 1 19.5 2.6875 16.5 7.375 \n",
       "C 15.203125 9.5625 15.203125 10.15625 15.203125 11.75 \n",
       "z\n",
       "\" id=\"CMR17-112\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(76.170122 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-72\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(93.943488 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(129.744826 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(165.025594 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(210.716071 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(256.406548 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(291.687316 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(337.377793 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(444.373349 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(490.063826 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(547.613305 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(616.685735 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "     <use transform=\"translate(671.46478 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "     <use transform=\"translate(770.613176 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(806.414514 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "     <use transform=\"translate(846.900174 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(892.590651 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(927.871419 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(968.35708 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(1034.234384 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(1061.707896 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(1107.398373 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(1172.755107 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(1197.626166 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(1248.521535 0)scale(0.996264)\" xlink:href=\"#CMR17-13\"/>\n",
       "     <use transform=\"translate(1298.263759 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(1343.954236 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(1420.873826 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(1497.793416 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(1543.483893 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(1578.764661 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(1624.455138 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(1659.735906 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "     <use transform=\"translate(1738.104788 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(1773.385556 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(1813.871216 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(1849.672554 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "     <use transform=\"translate(1903.170362 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(1948.860839 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(1999.756209 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(2035.557547 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"pc3340fef23\">\n",
       "   <rect height=\"217.44\" width=\"334.8\" x=\"41.999682\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x1198e5f98>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.title('Histogram of NPA scores for {}'.format(calculate_canonical_name(subgraph, inflamation_node)))\n",
    "plt.ylabel('Frequency')\n",
    "plt.xlabel('Score')\n",
    "plt.hist([runner.get_final_score() for runner in runners])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "## Full Workflow\n",
    "\n",
    "This workflow can all be run with [pbt.analysis.npa.workflow_all_average](http://pybel-tools.readthedocs.io/en/latest/npa.html#pybel_tools.analysis.npa.workflow_all_average) but is outlined here for a deeper understanding.\n",
    "\n",
    "### Generate Candidate Mechanisms\n",
    "\n",
    "Candidate mechanism are generated with the aforementioned function [pbt.generation.generate_mechanism](http://pybel-tools.readthedocs.io/en/latest/generation.html#pybel_tools.generation.generate_mechanism) to assess the upstream causal subgraphs of all biological processes."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "all_bioprocesses = pbt.selection.get_nodes_by_function(graph, BIOPROCESS)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 1.85 s, sys: 18.8 ms, total: 1.87 s\n",
      "Wall time: 1.87 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "candidate_mechanisms = {node: pbt.generation.generate_mechanism(graph, node, label) for node in all_bioprocesses}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Axonal Transport')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'mitochondrial calcium ion transport')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'neuronal signal transduction')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of ryanodine-sensitive calcium-release channel activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of cholesterol biosynthetic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'response to stress')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of energy homeostasis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of mitochondrial translation in response to stress')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'embryo development')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of cell fate commitment')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cell differentiation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cell-cell adhesion mediated by cadherin')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'chromosome segregation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cholesterol import')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'neural tube development')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'mitochondrial fission')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'negative regulation of synaptic growth at neuromuscular junction')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of protein neddylation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'lipid metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cholesterol metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'synapse maturation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'protein transport')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'actin cytoskeleton reorganization')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'calcium-mediated signaling')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'sequestering of calcium ion')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of cholesterol homeostasis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of natural killer cell apoptotic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of synapse assembly')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'dendrite development')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Wnt Signaling Pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'negative regulation of mitochondrial membrane potential')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'complement activation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'learning or memory')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PTS', 'Homeostasis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of insulin receptor signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Cell Survival')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'insulin-like growth factor receptor signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Neuronal Plasticity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'glucose transport')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'signal transduction')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Polymorphism, Single Nucleotide')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of cell death')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of mRNA stability')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of translation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'activation of protein kinase activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'exocytosis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of cell proliferation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'energy homeostasis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'JNK cascade')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'negative regulation of phosphatidylinositol 3-kinase activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Aging')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of insulin-like growth factor receptor signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'neurotrophin production')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'amyloid precursor protein catabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of amyloid precursor protein catabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'macrophage activation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'wound healing')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PTS', 'neuronal pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'ceramide biosynthetic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of angiogenesis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'axon guidance')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of peptidase activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cytoskeleton organization')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of insulin receptor signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of choline O-acetyltransferase activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'amyloid precursor protein metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'glucose homeostasis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'acetylcholine metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'type B pancreatic cell proliferation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'fatty acid catabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'insulin secretion')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of gluconeogenesis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'negative regulation of Notch signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'negative regulation of synaptic transmission, cholinergic')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of long-term synaptic potentiation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'negative regulation of long-term synaptic potentiation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'neurotransmitter secretion')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Membrane Potential, Mitochondrial')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'leptin-mediated signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of heart rate')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'adrenergic receptor signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'oxygen homeostasis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of neurotransmitter levels')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'adenosine metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'platelet aggregation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'platelet activation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'arachidonic acid secretion')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'activation of adenylate cyclase activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'calcium ion transport into cytosol')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'norepinephrine uptake')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'dopamine uptake')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'serotonin secretion, neurotransmission')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'iron ion transport')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'amyloid precursor protein biosynthetic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of protein phosphatase type 2A activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of prostaglandin-endoperoxide synthase activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'nerve growth factor production')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'protein kinase A signaling')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PTS', 'Ras family mediated signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of catalase activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PTS', 'nuclear factor kappa B signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'negative regulation of DNA replication')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of androgen receptor activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'aging')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'myelination')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'bone remodeling')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of nitric-oxide synthase activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'synaptic vesicle transport')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'natural killer cell mediated cytotoxicity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'glycoprotein transport')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Renin-Angiotensin System')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of immune response')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of tissue remodeling')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Stress, Physiological')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of cell cycle checkpoint')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'endothelial cell proliferation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'glucocorticoid receptor signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of beta-amyloid clearance')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'urokinase plasminogen activator signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'peptidyl-cysteine S-trans-nitrosylation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of blood pressure')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'phospholipase C-activating G-protein coupled acetylcholine receptor signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'activation of phospholipase C activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of mitochondrial membrane permeability involved in apoptotic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PTS', 'nerve growth factor signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'negative regulation of L-glutamate transport')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of viral process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'glutamate homeostasis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'negative regulation of myelination')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'response to heat')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'transcription, DNA-templated')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of cellular process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cell death in response to oxidative stress')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'mitochondrial toxicity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'memory impairment')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'water homeostasis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'CHT endocytosis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'B cell differentiation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PTS', 'chaperone signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'microtubule assembly')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'APP endoproteolysis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'NOTCH1 endoproteolysis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PTS', 'Wnt/beta_catenin pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'neurofibrillary tangle formation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'oxidative pmodification')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'glucose regulation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'glucose disposal')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'granulovacuolar degeneration')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PMIBP', 'Neuronal injury-induced glial apoE secretion')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of ERK1 and ERK2 cascade')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'gamma-aminobutyric acid signaling pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of voltage-gated potassium channel activity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'synaptic vesicle to endosome fusion')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'DNA synthesis involved in DNA repair')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'mitochondrial genome maintenance')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of calcium ion import')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of calcium-mediated signaling')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cholesterol transport')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'blood coagulation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'pyramidal neuron development')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cell-cell adhesion involved in synapse maturation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'neuron cell-cell adhesion')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of synapse assembly')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of long-term neuronal synaptic plasticity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'mammillary axonal complex development')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cell motility')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cytokine metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cytokine production involved in inflammatory response')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'CD8-positive, alpha-beta T cell activation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'beta-amyloid metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'homeostatic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Exercise')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'endoplasmic reticulum unfolded protein response')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'dendrite extension')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PTS', 'innate immune response pathway')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'negative regulation of phagocytosis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cytosolic calcium ion homeostasis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cellular lipid metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'PTS', 'glutamate signaling pathway via AMPA receptor')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'glutamate secretion')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of synaptic transmission')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of lipid metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'axonal defasciculation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of synaptic metaplasticity')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'sphingolipid metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of sphingomyelin catabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of gene expression, epigenetic')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of oxidative phosphorylation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'mitochondrial electron transport, cytochrome c to oxygen')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'MESHPP', 'Transfection')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'migration in host')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'stress fiber assembly')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of stress fiber assembly')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'macropinocytosis')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 21.4 s, sys: 252 ms, total: 21.6 s\n",
      "Wall time: 21.8 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "scores = {}\n",
    "\n",
    "for i, (node, subgraph) in enumerate(candidate_mechanisms.items()):\n",
    "    average_score = npa.multirun_average(subgraph, node, label, runs=40)\n",
    "    scores[node] = average_score"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(440, 239)"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(candidate_mechanisms), sum(1 for score in scores.values() if score)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "scores_cute_names = {calculate_canonical_name(graph, n): s for n, s in scores.items()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "scores_df = pd.DataFrame.from_dict(scores_cute_names, orient='index')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "The distribution of average NPA scores is shown to be mostly normal."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"261pt\" version=\"1.1\" viewBox=\"0 0 374 261\" width=\"374pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 261.997857 \n",
       "L 374.644006 261.997857 \n",
       "L 374.644006 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 29.144006 238.94218 \n",
       "L 363.944006 238.94218 \n",
       "L 363.944006 21.50218 \n",
       "L 29.144006 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#p55c358b09b)\" d=\"M 44.362188 238.94218 \n",
       "L 74.798551 238.94218 \n",
       "L 74.798551 235.142442 \n",
       "L 44.362188 235.142442 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#p55c358b09b)\" d=\"M 74.798551 238.94218 \n",
       "L 105.234915 238.94218 \n",
       "L 105.234915 233.242573 \n",
       "L 74.798551 233.242573 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#p55c358b09b)\" d=\"M 105.234915 238.94218 \n",
       "L 135.671278 238.94218 \n",
       "L 135.671278 233.242573 \n",
       "L 105.234915 233.242573 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#p55c358b09b)\" d=\"M 135.671278 238.94218 \n",
       "L 166.107642 238.94218 \n",
       "L 166.107642 231.342704 \n",
       "L 135.671278 231.342704 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#p55c358b09b)\" d=\"M 166.107642 238.94218 \n",
       "L 196.544006 238.94218 \n",
       "L 196.544006 208.544277 \n",
       "L 166.107642 208.544277 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#p55c358b09b)\" d=\"M 196.544006 238.94218 \n",
       "L 226.980369 238.94218 \n",
       "L 226.980369 31.856466 \n",
       "L 196.544006 31.856466 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#p55c358b09b)\" d=\"M 226.980369 238.94218 \n",
       "L 257.416733 238.94218 \n",
       "L 257.416733 85.052796 \n",
       "L 226.980369 85.052796 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#p55c358b09b)\" d=\"M 257.416733 238.94218 \n",
       "L 287.853097 238.94218 \n",
       "L 287.853097 208.544277 \n",
       "L 257.416733 208.544277 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#p55c358b09b)\" d=\"M 287.853097 238.94218 \n",
       "L 318.28946 238.94218 \n",
       "L 318.28946 223.743228 \n",
       "L 287.853097 223.743228 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#p55c358b09b)\" d=\"M 318.28946 238.94218 \n",
       "L 348.725824 238.94218 \n",
       "L 348.725824 231.342704 \n",
       "L 318.28946 231.342704 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 43.092714 238.94218 \n",
       "L 43.092714 21.50218 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_2\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"m0dd8df9a31\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.092714\" xlink:href=\"#m0dd8df9a31\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- $-15$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 65.90625 23 \n",
       "C 67.59375 23 69.40625 23 69.40625 25 \n",
       "C 69.40625 27 67.59375 27 65.90625 27 \n",
       "L 11.796875 27 \n",
       "C 10.09375 27 8.296875 27 8.296875 25 \n",
       "C 8.296875 23 10.09375 23 11.796875 23 \n",
       "z\n",
       "\" id=\"CMSY10-0\"/>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "       <path d=\"M 11.40625 57.6875 \n",
       "C 12.40625 57.28125 16.5 56 20.703125 56 \n",
       "C 30 56 35.09375 61.109375 38 64.0625 \n",
       "C 38 64.953125 38 65.5 37.40625 65.5 \n",
       "C 37.296875 65.5 37.09375 65.5 36.296875 65.046875 \n",
       "C 32.796875 63.40625 28.703125 62.078125 23.703125 62.078125 \n",
       "C 20.703125 62.078125 16.203125 62.484375 11.296875 64.671875 \n",
       "C 10.203125 65.171875 10 65.171875 9.90625 65.171875 \n",
       "C 9.40625 65.171875 9.296875 65.0625 9.296875 63.078125 \n",
       "L 9.296875 34.421875 \n",
       "C 9.296875 32.640625 9.296875 32.140625 10.296875 32.140625 \n",
       "C 10.796875 32.140625 11 32.34375 11.5 33.03125 \n",
       "C 14.703125 37.515625 19.09375 39.40625 24.09375 39.40625 \n",
       "C 27.59375 39.40625 35.09375 37.21875 35.09375 20.140625 \n",
       "C 35.09375 16.953125 35.09375 11.171875 32.09375 6.578125 \n",
       "C 29.59375 2.484375 25.703125 0.390625 21.40625 0.390625 \n",
       "C 14.796875 0.390625 8.09375 5 6.296875 12.71875 \n",
       "C 6.703125 12.609375 7.5 12.421875 7.90625 12.421875 \n",
       "C 9.203125 12.421875 11.703125 13.125 11.703125 16.21875 \n",
       "C 11.703125 18.90625 9.796875 20 7.90625 20 \n",
       "C 5.59375 20 4.09375 18.59375 4.09375 15.796875 \n",
       "C 4.09375 7.09375 11 -2 21.59375 -2 \n",
       "C 31.90625 -2 41.703125 6.875 41.703125 19.75 \n",
       "C 41.703125 31.71875 33.90625 41 24.203125 41 \n",
       "C 19.09375 41 14.796875 39.109375 11.40625 35.53125 \n",
       "z\n",
       "\" id=\"CMR17-53\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(34.237007 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(123.177945 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 104.513384 238.94218 \n",
       "L 104.513384 21.50218 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"104.513384\" xlink:href=\"#m0dd8df9a31\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- $-10$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "\" id=\"CMR17-48\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(95.657677 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(123.177945 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 165.934054 238.94218 \n",
       "L 165.934054 21.50218 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"165.934054\" xlink:href=\"#m0dd8df9a31\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- $-5$ -->\n",
       "      <g transform=\"translate(159.569014 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 227.354723 238.94218 \n",
       "L 227.354723 21.50218 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"227.354723\" xlink:href=\"#m0dd8df9a31\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(224.864056 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 288.775393 238.94218 \n",
       "L 288.775393 21.50218 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"288.775393\" xlink:href=\"#m0dd8df9a31\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- $5$ -->\n",
       "      <g transform=\"translate(286.284725 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 350.196063 238.94218 \n",
       "L 350.196063 21.50218 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"350.196063\" xlink:href=\"#m0dd8df9a31\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_6\">\n",
       "      <!-- $10$ -->\n",
       "      <g transform=\"translate(345.214727 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 29.144006 238.94218 \n",
       "L 363.944006 238.94218 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_14\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"ma81932f13f\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"29.144006\" xlink:href=\"#ma81932f13f\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(17.162671 242.401429)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_15\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 29.144006 200.944801 \n",
       "L 363.944006 200.944801 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_16\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"29.144006\" xlink:href=\"#ma81932f13f\" y=\"200.944801\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- $20$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(12.181335 204.40405)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_17\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 29.144006 162.947422 \n",
       "L 363.944006 162.947422 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_18\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"29.144006\" xlink:href=\"#ma81932f13f\" y=\"162.947422\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- $40$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(12.181335 166.406671)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_19\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 29.144006 124.950044 \n",
       "L 363.944006 124.950044 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_20\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"29.144006\" xlink:href=\"#ma81932f13f\" y=\"124.950044\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- $60$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "\" id=\"CMR17-54\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(12.181335 128.409293)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_21\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 29.144006 86.952665 \n",
       "L 363.944006 86.952665 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_22\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"29.144006\" xlink:href=\"#ma81932f13f\" y=\"86.952665\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- $80$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 27.203125 35.375 \n",
       "C 33.5 38.546875 39.90625 43.328125 39.90625 50.984375 \n",
       "C 39.90625 60.015625 31.09375 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 58.734375 5.90625 49.6875 \n",
       "C 5.90625 47.203125 6.5 42.921875 10.40625 39.140625 \n",
       "C 11.40625 38.15625 15.59375 35.171875 18.296875 33.28125 \n",
       "C 13.796875 30.984375 3.296875 25.53125 3.296875 14.59375 \n",
       "C 3.296875 4.359375 13.09375 -2 22.796875 -2 \n",
       "C 33.5 -2 42.5 5.65625 42.5 15.78125 \n",
       "C 42.5 24.84375 36.40625 29.015625 32.40625 31.703125 \n",
       "z\n",
       "M 14.09375 44.09375 \n",
       "C 13.296875 44.59375 9.296875 47.671875 9.296875 52.359375 \n",
       "C 9.296875 58.421875 15.59375 63 22.796875 63 \n",
       "C 30.703125 63 36.5 57.4375 36.5 50.96875 \n",
       "C 36.5 41.703125 26.09375 36.421875 25.59375 36.421875 \n",
       "C 25.5 36.421875 25.40625 36.421875 24.59375 37.03125 \n",
       "z\n",
       "M 32.5 23.734375 \n",
       "C 34 22.640625 38.796875 19.375 38.796875 13.296875 \n",
       "C 38.796875 5.953125 31.40625 0.390625 23 0.390625 \n",
       "C 13.90625 0.390625 7 6.84375 7 14.6875 \n",
       "C 7 22.546875 13.09375 29.09375 20 32.1875 \n",
       "z\n",
       "\" id=\"CMR17-56\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(12.181335 90.411914)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_23\">\n",
       "      <path clip-path=\"url(#p55c358b09b)\" d=\"M 29.144006 48.955286 \n",
       "L 363.944006 48.955286 \n",
       "\" style=\"fill:none;stroke:#b0b0b0;stroke-linecap:square;stroke-width:0.8;\"/>\n",
       "     </g>\n",
       "     <g id=\"line2d_24\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"29.144006\" xlink:href=\"#ma81932f13f\" y=\"48.955286\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_12\">\n",
       "      <!-- $100$ -->\n",
       "      <g transform=\"translate(7.2 52.414535)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 29.144006 238.94218 \n",
       "L 29.144006 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 363.944006 238.94218 \n",
       "L 363.944006 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 29.144006 238.94218 \n",
       "L 363.944006 238.94218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 29.144006 21.50218 \n",
       "L 363.944006 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_13\">\n",
       "    <!-- 0 -->\n",
       "    <g transform=\"translate(193.617511 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"p55c358b09b\">\n",
       "   <rect height=\"217.44\" width=\"334.8\" x=\"29.144006\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x1198f5080>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "scores_df.hist()\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>positive regulation of neuron apoptotic process</th>\n",
       "      <td>-14.896657</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>glial cell differentiation</th>\n",
       "      <td>-13.059167</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>production of molecular mediator involved in inflammatory response</th>\n",
       "      <td>-10.993091</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cell death</th>\n",
       "      <td>-10.718924</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>chronic inflammatory response</th>\n",
       "      <td>-10.368632</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>neuron apoptotic process</th>\n",
       "      <td>-9.245212</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>beta-amyloid formation</th>\n",
       "      <td>-7.770215</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cell-matrix adhesion</th>\n",
       "      <td>-7.657543</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>negative regulation of neuron projection regeneration</th>\n",
       "      <td>-6.735642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>vasoconstriction</th>\n",
       "      <td>-5.107942</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>reactive oxygen species metabolic process</th>\n",
       "      <td>-5.033316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>negative regulation of calcium-mediated signaling</th>\n",
       "      <td>-4.989171</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Lipid Peroxidation</th>\n",
       "      <td>-4.961603</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>response to oxidative stress</th>\n",
       "      <td>-4.576016</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>DNA biosynthetic process</th>\n",
       "      <td>-3.907494</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>phagocytosis</th>\n",
       "      <td>-3.826924</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>apoptotic process</th>\n",
       "      <td>-3.613041</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>gliogenesis</th>\n",
       "      <td>-3.318600</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>regulation of glucose metabolic process</th>\n",
       "      <td>-3.199888</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>beta-amyloid clearance</th>\n",
       "      <td>-2.650228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cell-cell signaling</th>\n",
       "      <td>-2.552514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>synapse assembly</th>\n",
       "      <td>-2.552514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cell adhesion</th>\n",
       "      <td>-2.552514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>regulation of growth</th>\n",
       "      <td>-2.552514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>growth</th>\n",
       "      <td>-2.552514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>synaptic transmission, glutamatergic</th>\n",
       "      <td>-2.552514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>RNA metabolic process</th>\n",
       "      <td>-2.552514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cell migration</th>\n",
       "      <td>-2.552514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>response to endoplasmic reticulum stress</th>\n",
       "      <td>-2.486175</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>response to reactive oxygen species</th>\n",
       "      <td>-2.480802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cAMP-mediated signaling</th>\n",
       "      <td>2.408039</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>leukocyte mediated immunity</th>\n",
       "      <td>2.408039</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mitochondrial transport</th>\n",
       "      <td>2.480802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>regulation of synaptic transmission, cholinergic</th>\n",
       "      <td>2.480802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>axon transport of mitochondrion</th>\n",
       "      <td>2.480802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sodium ion homeostasis</th>\n",
       "      <td>2.480802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>nitric oxide-cGMP-mediated signaling pathway</th>\n",
       "      <td>2.480802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>regulation of cellular response to oxidative stress</th>\n",
       "      <td>2.552514</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>regulation of neuronal synaptic plasticity</th>\n",
       "      <td>2.625366</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>reelin-mediated signaling pathway</th>\n",
       "      <td>3.116388</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Insulin Resistance</th>\n",
       "      <td>3.314902</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>learning</th>\n",
       "      <td>3.363660</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>regulation of synaptic plasticity</th>\n",
       "      <td>3.944863</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>negative regulation of microtubule depolymerization</th>\n",
       "      <td>3.968856</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>generation of neurons</th>\n",
       "      <td>3.968856</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>blood circulation</th>\n",
       "      <td>4.051542</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>calcium ion import</th>\n",
       "      <td>4.303770</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>positive regulation of Notch signaling pathway</th>\n",
       "      <td>4.398431</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>calcium ion homeostasis</th>\n",
       "      <td>4.995532</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>negative regulation of neuron apoptotic process</th>\n",
       "      <td>5.010068</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>protein import into mitochondrial matrix</th>\n",
       "      <td>5.033316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>electron transport chain</th>\n",
       "      <td>5.033316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mitochondrial calcium ion homeostasis</th>\n",
       "      <td>5.033316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>regulation of catalytic activity</th>\n",
       "      <td>5.056096</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>energy reserve metabolic process</th>\n",
       "      <td>6.602340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>glucose metabolic process</th>\n",
       "      <td>6.602340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mitochondria-nucleus signaling pathway</th>\n",
       "      <td>7.776194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cognition</th>\n",
       "      <td>8.760978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>mitochondrion organization</th>\n",
       "      <td>9.773005</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>memory</th>\n",
       "      <td>9.880314</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>246 rows × 1 columns</p>\n",
       "</div>"
      ],
      "text/latex": [
       "\\begin{center}{\\begin{tabular}{lr}\n",
       "\\toprule\n",
       "{} &          0 \\\\\n",
       "\\midrule\n",
       "positive regulation of neuron apoptotic process    & -14.896657 \\\\\n",
       "glial cell differentiation                         & -13.059167 \\\\\n",
       "production of molecular mediator involved in in... & -10.993091 \\\\\n",
       "cell death                                         & -10.718924 \\\\\n",
       "chronic inflammatory response                      & -10.368632 \\\\\n",
       "neuron apoptotic process                           &  -9.245212 \\\\\n",
       "beta-amyloid formation                             &  -7.770215 \\\\\n",
       "cell-matrix adhesion                               &  -7.657543 \\\\\n",
       "negative regulation of neuron projection regene... &  -6.735642 \\\\\n",
       "vasoconstriction                                   &  -5.107942 \\\\\n",
       "reactive oxygen species metabolic process          &  -5.033316 \\\\\n",
       "negative regulation of calcium-mediated signaling  &  -4.989171 \\\\\n",
       "Lipid Peroxidation                                 &  -4.961603 \\\\\n",
       "response to oxidative stress                       &  -4.576016 \\\\\n",
       "DNA biosynthetic process                           &  -3.907494 \\\\\n",
       "phagocytosis                                       &  -3.826924 \\\\\n",
       "apoptotic process                                  &  -3.613041 \\\\\n",
       "gliogenesis                                        &  -3.318600 \\\\\n",
       "regulation of glucose metabolic process            &  -3.199888 \\\\\n",
       "beta-amyloid clearance                             &  -2.650228 \\\\\n",
       "cell-cell signaling                                &  -2.552514 \\\\\n",
       "synapse assembly                                   &  -2.552514 \\\\\n",
       "cell adhesion                                      &  -2.552514 \\\\\n",
       "regulation of growth                               &  -2.552514 \\\\\n",
       "growth                                             &  -2.552514 \\\\\n",
       "synaptic transmission, glutamatergic               &  -2.552514 \\\\\n",
       "RNA metabolic process                              &  -2.552514 \\\\\n",
       "cell migration                                     &  -2.552514 \\\\\n",
       "response to endoplasmic reticulum stress           &  -2.486175 \\\\\n",
       "response to reactive oxygen species                &  -2.480802 \\\\\n",
       "Calcium dysfunction                                &  -2.480802 \\\\\n",
       "copper ion export                                  &  -2.480802 \\\\\n",
       "leukotriene production involved in inflammatory... &  -2.480802 \\\\\n",
       "microglial cell activation                         &  -2.456158 \\\\\n",
       "death                                              &  -2.204332 \\\\\n",
       "immune response                                    &  -1.995922 \\\\\n",
       "negative regulation of synapse assembly            &  -1.739891 \\\\\n",
       "microglial cell activation involved in immune r... &  -1.734151 \\\\\n",
       "lymphocyte activation                              &  -1.730321 \\\\\n",
       "natural killer cell activation                     &  -1.730321 \\\\\n",
       "inflammatory response pathway                      &  -1.697560 \\\\\n",
       "negative regulation of synaptic plasticity         &  -1.697560 \\\\\n",
       "Endoplasmic Reticulum Stress                       &  -1.684919 \\\\\n",
       "Mitochondrial Swelling                             &  -1.682256 \\\\\n",
       "synaptic transmission                              &  -1.569498 \\\\\n",
       "central nervous system development                 &  -1.516629 \\\\\n",
       "positive regulation of synaptic transmission, g... &  -1.477688 \\\\\n",
       "cell cycle process                                 &  -1.268407 \\\\\n",
       "positive regulation of clathrin-mediated endocy... &  -1.181924 \\\\\n",
       "cell cycle                                         &  -1.099180 \\\\\n",
       "positive regulation of CREB transcription facto... &  -1.014442 \\\\\n",
       "neuron death                                       &  -0.989465 \\\\\n",
       "stress response to copper ion                      &  -0.980261 \\\\\n",
       "mitochondrial depolarization                       &  -0.980261 \\\\\n",
       "cholesterol efflux                                 &  -0.977603 \\\\\n",
       "activation of MAPKK activity                       &  -0.974388 \\\\\n",
       "phagocytosis, engulfment                           &  -0.951158 \\\\\n",
       "neuron migration                                   &  -0.901970 \\\\\n",
       "apoptotic signaling pathway                        &  -0.887919 \\\\\n",
       "behavior                                           &  -0.874834 \\\\\n",
       "metabolic process                                  &  -0.874834 \\\\\n",
       "calcium ion transport                              &  -0.863812 \\\\\n",
       "regulation of dendritic cell antigen processing... &  -0.802697 \\\\\n",
       "regulation of short-term neuronal synaptic plas... &  -0.796136 \\\\\n",
       "regulation of synaptic activity                    &  -0.796136 \\\\\n",
       "response to unfolded protein                       &  -0.787240 \\\\\n",
       "Unfolded Protein Response                          &  -0.787240 \\\\\n",
       "xenobiotic metabolic process                       &  -0.736280 \\\\\n",
       "synaptic transmission, cholinergic                 &  -0.733048 \\\\\n",
       "axonogenesis                                       &  -0.724081 \\\\\n",
       "potassium ion import                               &  -0.707331 \\\\\n",
       "neuron remodeling                                  &  -0.693359 \\\\\n",
       "ERK1 and ERK2 cascade                              &  -0.629780 \\\\\n",
       "stress-activated protein kinase signaling cascade  &  -0.618517 \\\\\n",
       "protein catabolic process                          &  -0.578285 \\\\\n",
       "regulation of cell cycle                           &  -0.578285 \\\\\n",
       "positive regulation of protein neddylation         &  -0.578285 \\\\\n",
       "detection of misfolded protein                     &  -0.528864 \\\\\n",
       "receptor-mediated endocytosis                      &  -0.528864 \\\\\n",
       "response to misfolded protein                      &  -0.528864 \\\\\n",
       "retrograde axon cargo transport                    &  -0.507086 \\\\\n",
       "negative regulation of cell cycle                  &  -0.500330 \\\\\n",
       "positive regulation of cell cycle checkpoint       &  -0.500330 \\\\\n",
       "plasma membrane fusion                             &  -0.492646 \\\\\n",
       "regulation of microtubule cytoskeleton organiza... &  -0.488539 \\\\\n",
       "cell redox homeostasis                             &  -0.487726 \\\\\n",
       "actin crosslink formation                          &  -0.450985 \\\\\n",
       "insulin receptor signaling pathway                 &  -0.436550 \\\\\n",
       "neuron projection development                      &  -0.428702 \\\\\n",
       "iron ion homeostasis                               &  -0.428692 \\\\\n",
       "regulation of inflammatory response                &  -0.401348 \\\\\n",
       "regulation of phagocytosis                         &  -0.401348 \\\\\n",
       "immune response to tumor cell                      &  -0.401348 \\\\\n",
       "neuron differentiation                             &  -0.344869 \\\\\n",
       "regulation of cAMP-dependent protein kinase act... &  -0.328605 \\\\\n",
       "positive regulation of vasodilation                &  -0.319455 \\\\\n",
       "cellular response to nitrosative stress            &  -0.319455 \\\\\n",
       "cell communication                                 &  -0.319455 \\\\\n",
       "signaling                                          &  -0.275379 \\\\\n",
       "positive regulation of endocytosis                 &  -0.265133 \\\\\n",
       "positive regulation of lysosomal membrane perme... &  -0.246874 \\\\\n",
       "growth cone collapse response                      &  -0.241361 \\\\\n",
       "telencephalon microglial cell migration            &  -0.224749 \\\\\n",
       "negative regulation of insulin-like growth fact... &  -0.224749 \\\\\n",
       "negative regulation of insulin receptor signali... &  -0.224749 \\\\\n",
       "viral entry into host cell                         &  -0.219236 \\\\\n",
       "positive regulation of viral entry into host cell  &  -0.219236 \\\\\n",
       "Cell Transformation, Viral                         &  -0.219236 \\\\\n",
       "Virus Internalization                              &  -0.219236 \\\\\n",
       "regulation of cAMP biosynthetic process            &  -0.214147 \\\\\n",
       "T cell activation                                  &  -0.181812 \\\\\n",
       "insulin receptor signaling pathway via phosphat... &  -0.181724 \\\\\n",
       "cholesterol biosynthetic process via desmosterol   &  -0.178386 \\\\\n",
       "Neurogenesis                                       &  -0.161082 \\\\\n",
       "regulation of neuron migration                     &  -0.160568 \\\\\n",
       "clathrin-mediated endocytosis                      &  -0.146931 \\\\\n",
       "cell cycle arrest                                  &  -0.139991 \\\\\n",
       "positive regulation of oxidative phosphorylation   &  -0.139848 \\\\\n",
       "ATP biosynthetic process                           &  -0.139848 \\\\\n",
       "inflammatory response                              &  -0.134181 \\\\\n",
       "Longevity                                          &  -0.130448 \\\\\n",
       "cAMP catabolic process                             &  -0.129720 \\\\\n",
       "tricarboxylic acid cycle                           &  -0.129703 \\\\\n",
       "T cell tolerance induction                         &  -0.119921 \\\\\n",
       "ERK cascade                                        &  -0.112185 \\\\\n",
       "AKT signaling                                      &  -0.112185 \\\\\n",
       "regulation of long-term synaptic potentiation      &  -0.088134 \\\\\n",
       "lipid oxidation                                    &  -0.087136 \\\\\n",
       "Apoptosis                                          &  -0.086318 \\\\\n",
       "T cell proliferation                               &  -0.061891 \\\\\n",
       "cellular response to oxidative stress              &  -0.055395 \\\\\n",
       "positive regulation of cellular response to oxi... &  -0.047020 \\\\\n",
       "mitochondrion degradation                          &  -0.047020 \\\\\n",
       "astrocyte activation                               &  -0.043343 \\\\\n",
       "transmission of nerve impulse                      &  -0.038631 \\\\\n",
       "loss\\_of\\_synapses                                   &  -0.033649 \\\\\n",
       "insulin secretion involved in cellular response... &  -0.031576 \\\\\n",
       "Notch signaling pathway                            &  -0.002663 \\\\\n",
       "insulin sensitivity                                &  -0.001897 \\\\\n",
       "Endoplasmic Reticulum-Associated Degradation       &   0.000000 \\\\\n",
       "hydrogen peroxide-mediated programmed cell death   &   0.000000 \\\\\n",
       "Cell Cycle                                         &   0.000000 \\\\\n",
       "regulation of adenosine receptor signaling pathway &   0.000000 \\\\\n",
       "astrogliosis                                       &   0.000000 \\\\\n",
       "rhythmic excitation                                &   0.000000 \\\\\n",
       "necrotic cell death                                &   0.000000 \\\\\n",
       "MAPK cascade                                       &   0.001925 \\\\\n",
       "regulation of cell-cell adhesion                   &   0.002663 \\\\\n",
       "calcium-dependent cell-matrix adhesion             &   0.002663 \\\\\n",
       "pre-B cell receptor expression                     &   0.014272 \\\\\n",
       "long term synaptic depression                      &   0.023992 \\\\\n",
       "Jak\\_Stat signaling pathway                         &   0.028584 \\\\\n",
       "spatial memory                                     &   0.033649 \\\\\n",
       "spatial learning                                   &   0.033649 \\\\\n",
       "APP processing pathway                             &   0.035623 \\\\\n",
       "TOR signaling                                      &   0.055106 \\\\\n",
       "cell proliferation                                 &   0.063991 \\\\\n",
       "neuron development                                 &   0.099697 \\\\\n",
       "Cell Death                                         &   0.130448 \\\\\n",
       "regulation of mitochondrial membrane potential     &   0.137015 \\\\\n",
       "beta amyloid induced stress                        &   0.139848 \\\\\n",
       "cell growth                                        &   0.139991 \\\\\n",
       "methylglyoxal catabolic process to D-lactate       &   0.141563 \\\\\n",
       "negative regulation of cell cycle checkpoint       &   0.149404 \\\\\n",
       "actin polymerization or depolymerization           &   0.155939 \\\\\n",
       "angiogenesis                                       &   0.172131 \\\\\n",
       "regulation of MAPK cascade                         &   0.181055 \\\\\n",
       "autophagy                                          &   0.205420 \\\\\n",
       "negative regulation of insulin secretion involv... &   0.228197 \\\\\n",
       "acetylcholine secretion                            &   0.228197 \\\\\n",
       "amyloid pathway                                    &   0.236800 \\\\\n",
       "Energy Metabolism                                  &   0.257822 \\\\\n",
       "regulation of synaptic transmission, glutamatergic &   0.264256 \\\\\n",
       "regulation of protein kinase activity              &   0.267315 \\\\\n",
       "beta amyloid induced neuron death                  &   0.279695 \\\\\n",
       "positive regulation of mitochondrial membrane p... &   0.283572 \\\\\n",
       "activation of GTPase activity                      &   0.290838 \\\\\n",
       "lipid transport                                    &   0.310151 \\\\\n",
       "immune system process                              &   0.319455 \\\\\n",
       "dopamine receptor signaling pathway                &   0.328605 \\\\\n",
       "APP secretion                                      &   0.354627 \\\\\n",
       "positive regulation of gene expression             &   0.380815 \\\\\n",
       "Oxidative Stress                                   &   0.383921 \\\\\n",
       "monocyte aggregation                               &   0.401556 \\\\\n",
       "glucose import                                     &   0.435266 \\\\\n",
       "mitochondrial pathway                              &   0.436550 \\\\\n",
       "membrane lipid catabolic process                   &   0.492369 \\\\\n",
       "positive regulation of apoptotic process           &   0.557552 \\\\\n",
       "long-term synaptic potentiation                    &   0.580290 \\\\\n",
       "positive regulation of synaptic plasticity         &   0.646805 \\\\\n",
       "prolactin secretion                                &   0.649910 \\\\\n",
       "amyloidogenic pathway                              &   0.724081 \\\\\n",
       "lactic acid secretion                              &   0.724081 \\\\\n",
       "ST6GAL1 secretion                                  &   0.794680 \\\\\n",
       "endocytosis                                        &   0.871338 \\\\\n",
       "long-term memory                                   &   0.877871 \\\\\n",
       "synaptic vesicle budding from endosome             &   0.917142 \\\\\n",
       "pyroptosis                                         &   0.933887 \\\\\n",
       "regulation of N-methyl-D-aspartate selective gl... &   0.999108 \\\\\n",
       "Long-Term Potentiation                             &   0.999108 \\\\\n",
       "endoplasmic reticulum calcium ion homeostasis      &   1.026676 \\\\\n",
       "cholesterol homeostasis                            &   1.045690 \\\\\n",
       "neurogenesis                                       &   1.054179 \\\\\n",
       "cellular senescence                                &   1.175179 \\\\\n",
       "cytokine secretion                                 &   1.378606 \\\\\n",
       "cytokine production                                &   1.378606 \\\\\n",
       "amyloid fibril formation                           &   1.503534 \\\\\n",
       "vasodilation                                       &   1.608650 \\\\\n",
       "maintenance of blood-brain barrier                 &   1.697560 \\\\\n",
       "chemokine biosynthetic process                     &   1.701339 \\\\\n",
       "brain development                                  &   2.200780 \\\\\n",
       "positive regulation of neuron death                &   2.256916 \\\\\n",
       "regulation of dendritic cell dendrite assembly     &   2.373630 \\\\\n",
       "axon cargo transport                               &   2.373630 \\\\\n",
       "negative regulation of synaptic transmission, g... &   2.408039 \\\\\n",
       "zinc ion homeostasis                               &   2.408039 \\\\\n",
       "cAMP-mediated signaling                            &   2.408039 \\\\\n",
       "leukocyte mediated immunity                        &   2.408039 \\\\\n",
       "mitochondrial transport                            &   2.480802 \\\\\n",
       "regulation of synaptic transmission, cholinergic   &   2.480802 \\\\\n",
       "axon transport of mitochondrion                    &   2.480802 \\\\\n",
       "sodium ion homeostasis                             &   2.480802 \\\\\n",
       "nitric oxide-cGMP-mediated signaling pathway       &   2.480802 \\\\\n",
       "regulation of cellular response to oxidative st... &   2.552514 \\\\\n",
       "regulation of neuronal synaptic plasticity         &   2.625366 \\\\\n",
       "reelin-mediated signaling pathway                  &   3.116388 \\\\\n",
       "Insulin Resistance                                 &   3.314902 \\\\\n",
       "learning                                           &   3.363660 \\\\\n",
       "regulation of synaptic plasticity                  &   3.944863 \\\\\n",
       "negative regulation of microtubule depolymeriza... &   3.968856 \\\\\n",
       "generation of neurons                              &   3.968856 \\\\\n",
       "blood circulation                                  &   4.051542 \\\\\n",
       "calcium ion import                                 &   4.303770 \\\\\n",
       "positive regulation of Notch signaling pathway     &   4.398431 \\\\\n",
       "calcium ion homeostasis                            &   4.995532 \\\\\n",
       "negative regulation of neuron apoptotic process    &   5.010068 \\\\\n",
       "protein import into mitochondrial matrix           &   5.033316 \\\\\n",
       "electron transport chain                           &   5.033316 \\\\\n",
       "mitochondrial calcium ion homeostasis              &   5.033316 \\\\\n",
       "regulation of catalytic activity                   &   5.056096 \\\\\n",
       "energy reserve metabolic process                   &   6.602340 \\\\\n",
       "glucose metabolic process                          &   6.602340 \\\\\n",
       "mitochondria-nucleus signaling pathway             &   7.776194 \\\\\n",
       "cognition                                          &   8.760978 \\\\\n",
       "mitochondrion organization                         &   9.773005 \\\\\n",
       "memory                                             &   9.880314 \\\\\n",
       "\\bottomrule\n",
       "\\end{tabular}\n",
       "}\\end{center}"
      ],
      "text/plain": [
       "                                                            0\n",
       "positive regulation of neuron apoptotic process    -14.896657\n",
       "glial cell differentiation                         -13.059167\n",
       "production of molecular mediator involved in in... -10.993091\n",
       "cell death                                         -10.718924\n",
       "chronic inflammatory response                      -10.368632\n",
       "neuron apoptotic process                            -9.245212\n",
       "beta-amyloid formation                              -7.770215\n",
       "cell-matrix adhesion                                -7.657543\n",
       "negative regulation of neuron projection regene...  -6.735642\n",
       "vasoconstriction                                    -5.107942\n",
       "reactive oxygen species metabolic process           -5.033316\n",
       "negative regulation of calcium-mediated signaling   -4.989171\n",
       "Lipid Peroxidation                                  -4.961603\n",
       "response to oxidative stress                        -4.576016\n",
       "DNA biosynthetic process                            -3.907494\n",
       "phagocytosis                                        -3.826924\n",
       "apoptotic process                                   -3.613041\n",
       "gliogenesis                                         -3.318600\n",
       "regulation of glucose metabolic process             -3.199888\n",
       "beta-amyloid clearance                              -2.650228\n",
       "cell-cell signaling                                 -2.552514\n",
       "synapse assembly                                    -2.552514\n",
       "cell adhesion                                       -2.552514\n",
       "regulation of growth                                -2.552514\n",
       "growth                                              -2.552514\n",
       "synaptic transmission, glutamatergic                -2.552514\n",
       "RNA metabolic process                               -2.552514\n",
       "cell migration                                      -2.552514\n",
       "response to endoplasmic reticulum stress            -2.486175\n",
       "response to reactive oxygen species                 -2.480802\n",
       "...                                                       ...\n",
       "cAMP-mediated signaling                              2.408039\n",
       "leukocyte mediated immunity                          2.408039\n",
       "mitochondrial transport                              2.480802\n",
       "regulation of synaptic transmission, cholinergic     2.480802\n",
       "axon transport of mitochondrion                      2.480802\n",
       "sodium ion homeostasis                               2.480802\n",
       "nitric oxide-cGMP-mediated signaling pathway         2.480802\n",
       "regulation of cellular response to oxidative st...   2.552514\n",
       "regulation of neuronal synaptic plasticity           2.625366\n",
       "reelin-mediated signaling pathway                    3.116388\n",
       "Insulin Resistance                                   3.314902\n",
       "learning                                             3.363660\n",
       "regulation of synaptic plasticity                    3.944863\n",
       "negative regulation of microtubule depolymeriza...   3.968856\n",
       "generation of neurons                                3.968856\n",
       "blood circulation                                    4.051542\n",
       "calcium ion import                                   4.303770\n",
       "positive regulation of Notch signaling pathway       4.398431\n",
       "calcium ion homeostasis                              4.995532\n",
       "negative regulation of neuron apoptotic process      5.010068\n",
       "protein import into mitochondrial matrix             5.033316\n",
       "electron transport chain                             5.033316\n",
       "mitochondrial calcium ion homeostasis                5.033316\n",
       "regulation of catalytic activity                     5.056096\n",
       "energy reserve metabolic process                     6.602340\n",
       "glucose metabolic process                            6.602340\n",
       "mitochondria-nucleus signaling pathway               7.776194\n",
       "cognition                                            8.760978\n",
       "mitochondrion organization                           9.773005\n",
       "memory                                               9.880314\n",
       "\n",
       "[246 rows x 1 columns]"
      ]
     },
     "execution_count": 49,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "scores_df[scores_df[0].notnull()].sort_values(0)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "# Conclusions\n",
    "\n",
    "## Problems\n",
    "\n",
    "This algorithm overcomes deterministic problems with cyclic graphs by randomly removing edges. It can be run multiple times to assess the stability of the score with `pbt.analysis.npa.average_npa_run`. The effect of the default score can also be checked with a grid search.\n",
    "\n",
    "## Looking Forward\n",
    "\n",
    "While this new algorithm is generally applicable and overcomes the original algorithm, it must throw away information to do so. Further algorithmic development, such as heat diffusion approaches could allow for a more thorough investigation of the propogation of effects of the differential gene expression.\n",
    "\n",
    "This algorithm can be run over the mechanisms generated for each biological process. Further investigation into the upstream mechanism generation can yield bias-free candidate mechanisms. They can also be matched to canonical mechanisms in NeuroMMSigDB to identify concordance between the biological dogma and all possible mechanisms, then to identify previously unknown cross-talk between mechanisms."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
