{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Abstract\n",
    "\n",
    "**Author:** [Charles Tapley Hoyt](https://github.com/cthoyt)\n",
    "\n",
    "**Estimated Run Time:** 4 minutes\n",
    "\n",
    "This notebooks demonstrates the usage of a variant on the [Network Perturbation Amplitude](http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-6-54) algorithm to analyze an BEL knowledge assembly in the context of a differential gene expression experiment. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Notebook Imports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import logging\n",
    "import os\n",
    "import time\n",
    "\n",
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "from pandas.tools.plotting import scatter_matrix\n",
    "from matplotlib_venn import venn2\n",
    "\n",
    "import pybel\n",
    "import pybel_tools as pbt\n",
    "import pybel_tools.analysis.cmpa as cmpa\n",
    "from pybel.constants import *\n",
    "from pybel.canonicalize import calculate_canonical_name\n",
    "from pybel_tools.visualization import to_jupyter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%config InlineBackend.figure_format = 'svg'\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Notebook Provenance\n",
    "\n",
    "The time of execution, random number generator seed, and the versions of the software packages used are displayed explicitly."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Sun Aug 27 11:44:09 2017'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "time.asctime()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# seed the random number generator\n",
    "import random\n",
    "random.seed(127)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.7.3-dev'"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel.__version__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.2.2-dev'"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pbt.__version__"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Local Path Definitions\n",
    "\n",
    "To make this notebook interoperable across many machines, locations to the repositories that contain the data used in this notebook are referenced from the environment, set in `~/.bashrc` to point to the place where the repositories have been cloned. Assuming the repositories have been `git clone`'d into the `~/dev` folder, the entries in `~/.bashrc` should look like:\n",
    "\n",
    "```bash\n",
    "...\n",
    "export BMS_BASE=~/dev/bms\n",
    "...\n",
    "```\n",
    "\n",
    "#### BMS \n",
    "\n",
    "The biological model store (BMS) is the internal Fraunhofer SCAI repository for keeping BEL models under version control. It can be downloaded from https://tor-2.scai.fraunhofer.de/gf/project/bms/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "bms_base = os.environ['BMS_BASE']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### OwnCloud\n",
    "\n",
    "The differential gene expression data used in this notebook is currently not published, and is obfuscated with reference through our team's internal data storage system with [OwnCloud](https://owncloud.org/)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "owncloud_base = os.environ['OWNCLOUD_BASE']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Data\n",
    "\n",
    "## Alzheimer's Disease Knowledge Assembly\n",
    "\n",
    "The Alzheimer's Disease Knowledge Assembly has been precompiled with the following command line script, and will be loaded from this format for improved performance. In general, derived data, such as the gpickle representation of a BEL script, are not saved under version control to ensure that the most up-to-date data is always used.\n",
    "\n",
    "```sh\n",
    "pybel convert --path \"$BMS_BASE/aetionomy/alzheimers.bel\" --pickle \"$BMS_BASE/aetionomy/alzheimers.gpickle\"\n",
    "```\n",
    "\n",
    "The BEL script can also be compiled from inside this notebook with the following python code:\n",
    "\n",
    "```python\n",
    ">>> import os\n",
    ">>> import pybel\n",
    ">>> # Input from BEL script\n",
    ">>> bel_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.bel')\n",
    ">>> graph = pybel.from_path(bel_path)\n",
    ">>> # Output to gpickle for fast loading later\n",
    ">>> pickle_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.gpickle')\n",
    ">>> pybel.to_pickle(graph, pickle_path)\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pickle_path = os.path.join(bms_base, 'aetionomy', 'alzheimers', 'alzheimers.gpickle')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "graph = pybel.from_pickle(pickle_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'4.0.3'"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "graph.version"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "All orthologies are discared before analysis."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pbt.filters.remove_nodes_by_namespace(graph, 'MGI')\n",
    "pbt.filters.remove_nodes_by_namespace(graph, 'RGD')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To merge the differential gene expression annotations, the entire graph is collapsed to genes using [pbt.mutation.collapse_by_central_dogma_to_genes](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.collapse_by_central_dogma_to_genes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pbt.mutation.collapse_by_central_dogma_to_genes(graph)\n",
    "\n",
    "pbt.mutation.rewire_variants_to_genes(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nodes: 3701\n",
      "Edges: 19562\n",
      "Citations: 1760\n",
      "Authors: 9305\n",
      "Network density: 0.0014285401315933604\n",
      "Components: 70\n",
      "Average degree: 5.2855984868954335\n",
      "Compilation warnings: 2578\n"
     ]
    }
   ],
   "source": [
    "pbt.mutation.parse_authors(graph)\n",
    "pbt.summary.print_summary(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"280pt\" version=\"1.1\" viewBox=\"0 0 712 280\" width=\"712pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 280.8 \n",
       "L 712.791699 280.8 \n",
       "L 712.791699 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 89.335068 257.744323 \n",
       "L 330.860125 257.744323 \n",
       "L 330.860125 21.50218 \n",
       "L 89.335068 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#pa3d5c4fcf9)\" d=\"M -32488.326206 250.361756 \n",
       "L 100.31348 250.361756 \n",
       "L 100.31348 235.596622 \n",
       "L -32488.326206 235.596622 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#pa3d5c4fcf9)\" d=\"M -32488.326206 220.831488 \n",
       "L 162.420526 220.831488 \n",
       "L 162.420526 206.066354 \n",
       "L -32488.326206 206.066354 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#pa3d5c4fcf9)\" d=\"M -32488.326206 191.30122 \n",
       "L 196.15059 191.30122 \n",
       "L 196.15059 176.536086 \n",
       "L -32488.326206 176.536086 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#pa3d5c4fcf9)\" d=\"M -32488.326206 161.770953 \n",
       "L 237.573012 161.770953 \n",
       "L 237.573012 147.005819 \n",
       "L -32488.326206 147.005819 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#pa3d5c4fcf9)\" d=\"M -32488.326206 132.240685 \n",
       "L 247.002185 132.240685 \n",
       "L 247.002185 117.475551 \n",
       "L -32488.326206 117.475551 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#pa3d5c4fcf9)\" d=\"M -32488.326206 102.710417 \n",
       "L 251.680273 102.710417 \n",
       "L 251.680273 87.945283 \n",
       "L -32488.326206 87.945283 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#pa3d5c4fcf9)\" d=\"M -32488.326206 73.180149 \n",
       "L 279.765752 73.180149 \n",
       "L 279.765752 58.415015 \n",
       "L -32488.326206 58.415015 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#pa3d5c4fcf9)\" d=\"M -32488.326206 43.649881 \n",
       "L 319.881714 43.649881 \n",
       "L 319.881714 28.884747 \n",
       "L -32488.326206 28.884747 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"md1ad39f64b\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"186.423348\" xlink:href=\"#md1ad39f64b\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- $10^{2}$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "z\n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "z\n",
       "\" id=\"CMR17-48\"/>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(179.456394 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"294.617883\" xlink:href=\"#md1ad39f64b\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- $10^{3}$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 22.09375 33.703125 \n",
       "C 31 33.703125 34.90625 25.96875 34.90625 17.03125 \n",
       "C 34.90625 5 28.5 0.390625 22.703125 0.390625 \n",
       "C 17.40625 0.390625 8.796875 3.03125 6.09375 10.828125 \n",
       "C 6.59375 10.625 7.09375 10.625 7.59375 10.625 \n",
       "C 10 10.625 11.796875 12.21875 11.796875 14.8125 \n",
       "C 11.796875 17.703125 9.59375 19 7.59375 19 \n",
       "C 5.90625 19 3.296875 18.203125 3.296875 14.46875 \n",
       "C 3.296875 5.234375 12.296875 -2 22.90625 -2 \n",
       "C 34 -2 42.5 6.71875 42.5 16.953125 \n",
       "C 42.5 26.671875 34.5 33.703125 25 34.796875 \n",
       "C 32.59375 36.375 39.90625 43.0625 39.90625 52.015625 \n",
       "C 39.90625 59.6875 32 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 59.828125 5.90625 52.015625 \n",
       "C 5.90625 48.59375 8.5 48 9.796875 48 \n",
       "C 11.90625 48 13.703125 49.296875 13.703125 51.890625 \n",
       "C 13.703125 54.46875 11.90625 55.765625 9.796875 55.765625 \n",
       "C 9.40625 55.765625 8.90625 55.765625 8.5 55.578125 \n",
       "C 11.40625 61.859375 19.296875 63 22.796875 63 \n",
       "C 26.296875 63 32.90625 61.328125 32.90625 51.875 \n",
       "C 32.90625 49.109375 32.5 44.1875 29.09375 39.84375 \n",
       "C 26.09375 36 22.703125 36 19.40625 35.6875 \n",
       "C 18.90625 35.6875 16.59375 35.453125 16.203125 35.453125 \n",
       "C 15.5 35.359375 15.09375 35.265625 15.09375 34.5 \n",
       "C 15.09375 33.796875 15.203125 33.703125 17.203125 33.703125 \n",
       "z\n",
       "\" id=\"CMR17-51\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(287.650929 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-51\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 2 \n",
       "\" id=\"m8d2aa11a19\" style=\"stroke:#000000;stroke-width:0.6;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"110.798613\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"129.850725\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"143.368414\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"153.853548\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_7\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"162.420526\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_8\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"169.663802\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_9\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"175.938214\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_10\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"181.472637\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_11\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"218.993148\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_12\">\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"238.04526\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_13\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"251.562949\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_14\">\n",
       "     <g id=\"line2d_14\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"262.048082\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_15\">\n",
       "     <g id=\"line2d_15\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"270.615061\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_16\">\n",
       "     <g id=\"line2d_16\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"277.858337\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_17\">\n",
       "     <g id=\"line2d_17\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"284.132749\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_18\">\n",
       "     <g id=\"line2d_18\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"289.667172\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_19\">\n",
       "     <g id=\"line2d_19\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"327.187683\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_20\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"m5c1494a952\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#m5c1494a952\" y=\"242.979189\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- Reaction -->\n",
       "      <defs>\n",
       "       <path d=\"M 39.203125 33.53125 \n",
       "C 48.703125 35.4375 56.703125 41.8125 56.703125 49.78125 \n",
       "C 56.703125 59.140625 46.09375 67.59375 32.09375 67.59375 \n",
       "L 5.203125 67.59375 \n",
       "L 5.203125 65 \n",
       "C 12.09375 65 13.203125 65 13.203125 60.515625 \n",
       "L 13.203125 6.484375 \n",
       "C 13.203125 2 12.09375 2 5.203125 2 \n",
       "L 5.203125 -0.40625 \n",
       "C 8.09375 -0.40625 13.296875 -0.40625 16.40625 -0.40625 \n",
       "C 19.5 -0.40625 24.703125 -0.40625 27.59375 -0.40625 \n",
       "L 27.59375 2 \n",
       "C 20.703125 2 19.59375 2 19.59375 6.515625 \n",
       "L 19.59375 33 \n",
       "L 31.296875 33 \n",
       "C 37 33 40.203125 30.6875 41.59375 29.390625 \n",
       "C 45.59375 25.5 45.59375 22.796875 45.59375 16 \n",
       "C 45.59375 9.296875 45.59375 5.890625 48.59375 2.5 \n",
       "C 52.40625 -1.59375 57.5 -2 59.703125 -2 \n",
       "C 67.09375 -2 68.203125 6.1875 68.203125 7.859375 \n",
       "C 68.203125 8.4375 68.203125 9.140625 67.296875 9.140625 \n",
       "C 66.5 9.140625 66.40625 8.546875 66.40625 8.15625 \n",
       "C 65.90625 1.75 63 -0.40625 59.90625 -0.40625 \n",
       "C 54.40625 -0.40625 53.796875 5.859375 53.203125 11.9375 \n",
       "C 52.90625 14.03125 52.703125 15.515625 52.5 17.703125 \n",
       "C 51.90625 22.6875 51.09375 29.953125 39.203125 33.53125 \n",
       "z\n",
       "M 31 34.59375 \n",
       "L 19.59375 34.59375 \n",
       "L 19.59375 61.140625 \n",
       "C 19.59375 64.515625 19.796875 65 23.796875 65 \n",
       "L 31.09375 65 \n",
       "C 39.40625 65 49.09375 62.140625 49.09375 49.84375 \n",
       "C 49.09375 37.0625 38.5 34.59375 31 34.59375 \n",
       "z\n",
       "\" id=\"CMR17-82\"/>\n",
       "       <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "       <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "       <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "z\n",
       "\" id=\"CMR17-99\"/>\n",
       "       <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "       <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "z\n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "       <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "z\n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "z\n",
       "\" id=\"CMR17-111\"/>\n",
       "       <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(44.00651 246.438439)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-82\"/>\n",
       "       <use transform=\"translate(67.791102 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(108.276762 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(153.96724 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(194.4529 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(229.733668 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(254.604726 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(300.295203 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_21\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#m5c1494a952\" y=\"213.448921\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- Composite -->\n",
       "      <defs>\n",
       "       <path d=\"M 62.09375 67.015625 \n",
       "C 62.09375 68.796875 62 68.90625 61.40625 68.90625 \n",
       "C 61 68.90625 60.90625 68.796875 60.203125 67.609375 \n",
       "L 55.796875 59.25 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 12.90625 20.09375 -2 37.296875 -2 \n",
       "C 52.59375 -2 62.09375 11.40625 62.09375 22.828125 \n",
       "C 62.09375 23.8125 62.09375 24.21875 61.203125 24.21875 \n",
       "C 60.40625 24.21875 60.40625 23.921875 60.296875 23.03125 \n",
       "C 59.5 9.140625 49.5 0.59375 38.703125 0.59375 \n",
       "C 28.5 0.59375 12.203125 7.734375 12.203125 33.546875 \n",
       "C 12.203125 59.453125 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.6875 59.90625 42.890625 \n",
       "C 60.09375 41.6875 60.09375 41.484375 61 41.484375 \n",
       "C 62.09375 41.484375 62.09375 41.6875 62.09375 43.484375 \n",
       "z\n",
       "\" id=\"CMR17-67\"/>\n",
       "       <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "       <path d=\"M 22 -16.609375 \n",
       "C 15.59375 -16.609375 15.203125 -16.203125 15.203125 -12.328125 \n",
       "L 15.203125 6.28125 \n",
       "C 18.09375 2.09375 22.296875 -0.59375 27.59375 -0.59375 \n",
       "C 37.59375 -0.59375 47.703125 8.171875 47.703125 21.6875 \n",
       "C 47.703125 34.234375 39 43.78125 28.703125 43.78125 \n",
       "C 22.703125 43.78125 17.796875 40.5 15 36.53125 \n",
       "L 15 43.78125 \n",
       "L 3.09375 42.6875 \n",
       "L 3.09375 40.109375 \n",
       "C 9 40.109375 9.90625 39.5 9.90625 34.734375 \n",
       "L 9.90625 -12.328125 \n",
       "C 9.90625 -16.109375 9.5 -16.609375 3.09375 -16.609375 \n",
       "L 3.09375 -19.1875 \n",
       "C 5.5 -19 9.90625 -19 12.5 -19 \n",
       "C 15.09375 -19 19.59375 -19 22 -19.1875 \n",
       "z\n",
       "M 15.203125 31.4375 \n",
       "C 15.203125 32.84375 15.203125 34.921875 19.09375 38.609375 \n",
       "C 19.59375 39.015625 23 42 28 42 \n",
       "C 35.296875 42 41.296875 32.9375 41.296875 21.59375 \n",
       "C 41.296875 10.25 34.90625 1 27 1 \n",
       "C 23.40625 1 19.5 2.6875 16.5 7.375 \n",
       "C 15.203125 9.5625 15.203125 10.15625 15.203125 11.75 \n",
       "z\n",
       "\" id=\"CMR17-112\"/>\n",
       "       <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(36.064035 216.908171)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(66.509881 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(112.200358 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(189.119948 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(242.617756 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(288.308233 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(324.109571 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(348.980629 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(384.261397 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_22\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#m5c1494a952\" y=\"183.918653\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- Pathology -->\n",
       "      <defs>\n",
       "       <path d=\"M 19.796875 31.90625 \n",
       "L 35.796875 31.90625 \n",
       "C 48.703125 31.90625 58.09375 40.28125 58.09375 49.75 \n",
       "C 58.09375 59.234375 48.90625 68 35.796875 68 \n",
       "L 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.921875 \n",
       "L 13.203125 6.96875 \n",
       "C 13.203125 2.5 12.09375 2.5 5.203125 2.5 \n",
       "L 5.203125 0 \n",
       "C 8.09375 0 13.40625 0 16.5 0 \n",
       "C 19.59375 0 24.90625 0 27.796875 0 \n",
       "L 27.796875 2.5 \n",
       "C 20.90625 2.5 19.796875 2.5 19.796875 6.96875 \n",
       "z\n",
       "M 19.59375 34 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 33.90625 65.40625 \n",
       "C 46.796875 65.40625 50.5 58.03125 50.5 49.75 \n",
       "C 50.5 40.6875 46.09375 34 33.90625 34 \n",
       "z\n",
       "\" id=\"CMR17-80\"/>\n",
       "       <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.5 43.59375 16.5 37 15.296875 33.703125 \n",
       "L 15.203125 33.703125 \n",
       "L 15.203125 68.6875 \n",
       "L 3.296875 67.59375 \n",
       "L 3.296875 65 \n",
       "C 9.203125 65 10.09375 64.40625 10.09375 59.640625 \n",
       "L 10.09375 6.65625 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-104\"/>\n",
       "       <path d=\"M 15.296875 68.6875 \n",
       "L 3.40625 67.59375 \n",
       "L 3.40625 65 \n",
       "C 9.296875 65 10.203125 64.40625 10.203125 59.640625 \n",
       "L 10.203125 6.65625 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.203125 0 12.703125 0 \n",
       "C 15.296875 0 19.703125 0 22.09375 -0.203125 \n",
       "L 22.09375 2.390625 \n",
       "C 15.703125 2.390625 15.296875 2.78125 15.296875 6.65625 \n",
       "z\n",
       "\" id=\"CMR17-108\"/>\n",
       "       <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "z\n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "z\n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "z\n",
       "\" id=\"CMR17-103\"/>\n",
       "       <path d=\"M 39 34.328125 \n",
       "C 41.5 40.40625 46 40.40625 47.40625 40.40625 \n",
       "L 47.40625 43 \n",
       "C 45.5 43 43 43 41.09375 43 \n",
       "C 39 43 35.703125 43 33.703125 43.09375 \n",
       "L 33.703125 40.5 \n",
       "C 37.5 40.203125 37.59375 37.40625 37.59375 36.609375 \n",
       "C 37.59375 35.625 37.40625 35.125 36.90625 33.921875 \n",
       "L 26 7.015625 \n",
       "L 14.09375 36.03125 \n",
       "C 13.59375 37.21875 13.59375 37.90625 13.59375 38.015625 \n",
       "C 13.59375 40.203125 15.90625 40.40625 18.296875 40.40625 \n",
       "L 18.296875 43.09375 \n",
       "C 15.90625 43 11.59375 43 9.09375 43 \n",
       "C 6.40625 43 3.203125 43 1 43.09375 \n",
       "L 1 40.40625 \n",
       "C 6.40625 40.40625 7 39.90625 8.296875 36.71875 \n",
       "L 23.203125 0.234375 \n",
       "C 18.703125 -11.53125 16.09375 -18.40625 9.796875 -18.40625 \n",
       "C 8.703125 -18.40625 6.203125 -18.109375 4.40625 -16.3125 \n",
       "C 6.703125 -16.109375 7.703125 -14.71875 7.703125 -13.015625 \n",
       "C 7.703125 -11.328125 6.5 -9.828125 4.5 -9.828125 \n",
       "C 2.296875 -9.828125 1.203125 -11.328125 1.203125 -13.125 \n",
       "C 1.203125 -17.109375 5.296875 -20 9.796875 -20 \n",
       "C 15.59375 -20 19.203125 -14.40625 21.203125 -9.4375 \n",
       "z\n",
       "\" id=\"CMR17-121\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(38.471647 187.377903)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(59.983847 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(105.674324 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(140.955092 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(191.850461 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(237.540938 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(262.411997 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(308.102474 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(353.792951 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_23\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#m5c1494a952\" y=\"154.388386\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_6\">\n",
       "      <!-- Protein -->\n",
       "      <defs>\n",
       "       <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(50.067111 157.847635)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(62.586301 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(97.867068 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(143.557545 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(178.838313 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(219.323974 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(244.195032 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_24\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#m5c1494a952\" y=\"124.858118\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- Abundance -->\n",
       "      <defs>\n",
       "       <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "       <path d=\"M 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 -0.203125 \n",
       "L 11.703125 -0.203125 \n",
       "L 14.59375 6.890625 \n",
       "C 17.5 2.203125 21.796875 -1 27.59375 -1 \n",
       "C 37.59375 -1 47.703125 7.8125 47.703125 21.40625 \n",
       "C 47.703125 34.1875 38.703125 43.59375 28.59375 43.59375 \n",
       "C 22.40625 43.59375 18 40.296875 15 36.40625 \n",
       "z\n",
       "M 15.203125 31.3125 \n",
       "C 15.203125 33.09375 15.203125 33.40625 16.59375 35.5 \n",
       "C 20.5 41.515625 26.09375 42 28 42 \n",
       "C 31 42 41.296875 40.390625 41.296875 21.5 \n",
       "C 41.296875 1.6875 29.5 0.59375 27.09375 0.59375 \n",
       "C 24 0.59375 19.5 1.78125 16.296875 7.59375 \n",
       "C 15.203125 9.5 15.203125 9.703125 15.203125 11.484375 \n",
       "z\n",
       "\" id=\"CMR17-98\"/>\n",
       "       <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "       <path d=\"M 29.203125 67.59375 \n",
       "L 29.203125 65 \n",
       "C 35.09375 65 36 64.40625 36 59.6875 \n",
       "L 36 36.640625 \n",
       "C 35.59375 37.140625 31.5 43.59375 23.40625 43.59375 \n",
       "C 13.203125 43.59375 3.296875 34.5 3.296875 21.3125 \n",
       "C 3.296875 8.203125 12.59375 -1 22.40625 -1 \n",
       "C 30.90625 -1 35.296875 5.265625 35.796875 5.9375 \n",
       "L 35.796875 -0.984375 \n",
       "L 47.90625 -0.984375 \n",
       "L 47.90625 2 \n",
       "C 42 2 41.09375 2.59375 41.09375 7.390625 \n",
       "L 41.09375 68.6875 \n",
       "z\n",
       "M 35.796875 11.796875 \n",
       "C 35.796875 8.796875 34 6.09375 31.703125 4.09375 \n",
       "C 28.296875 1.09375 24.90625 0.59375 23 0.59375 \n",
       "C 20.09375 0.59375 9.703125 2.09375 9.703125 21.1875 \n",
       "C 9.703125 40.8125 21.296875 42 23.90625 42 \n",
       "C 28.5 42 32.203125 39.40625 34.5 35.796875 \n",
       "C 35.796875 33.703125 35.796875 33.40625 35.796875 31.609375 \n",
       "z\n",
       "\" id=\"CMR17-100\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(33.351976 128.317367)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "       <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "       <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "       <use transform=\"translate(170.863169 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(221.758539 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(272.653909 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(318.344386 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(369.239756 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(409.725416 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_25\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#m5c1494a952\" y=\"95.32785\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- BiologicalProcess -->\n",
       "      <defs>\n",
       "       <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 39.59375 0 \n",
       "C 52.40625 0 60.703125 9.109375 60.703125 18.1875 \n",
       "C 60.703125 27.203125 52.5 34.890625 41.796875 34.890625 \n",
       "C 51.40625 36.8125 58.09375 43.296875 58.09375 51 \n",
       "C 58.09375 59.40625 49.90625 68 37 68 \n",
       "z\n",
       "M 19.5 36 \n",
       "L 19.5 61.515625 \n",
       "C 19.5 64.921875 19.703125 65.40625 23.703125 65.40625 \n",
       "L 36.5 65.40625 \n",
       "C 46.40625 65.40625 50.796875 57.40625 50.796875 51.203125 \n",
       "C 50.796875 43.609375 44.703125 36 33.90625 36 \n",
       "z\n",
       "M 23.703125 2.59375 \n",
       "C 19.703125 2.59375 19.5 3.078125 19.5 6.453125 \n",
       "L 19.5 34.40625 \n",
       "L 38 34.40625 \n",
       "C 48.296875 34.40625 53.296875 25.390625 53.296875 18.15625 \n",
       "C 53.296875 10.921875 47.59375 2.59375 36.703125 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-66\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(7.2 98.787099)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-66\"/>\n",
       "       <use transform=\"translate(65.188754 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(90.059813 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(135.75029 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(160.621349 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(206.311826 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(252.002303 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(276.873361 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(317.359022 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(363.049499 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(387.920557 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(450.506858 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(485.787626 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(534.080541 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(574.566202 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(615.051862 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(650.8532 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_7\">\n",
       "     <g id=\"line2d_26\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#m5c1494a952\" y=\"65.797582\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- Complex -->\n",
       "      <defs>\n",
       "       <path d=\"M 26 23.078125 \n",
       "C 29 26.96875 32.40625 31.34375 35.203125 34.625 \n",
       "C 38.5 38.40625 42.296875 40 46.703125 40 \n",
       "L 46.703125 42.59375 \n",
       "C 45 42.59375 41.5 42.59375 39.703125 42.59375 \n",
       "C 37 42.59375 33.90625 42.59375 31.59375 42.6875 \n",
       "L 31.59375 40.09375 \n",
       "C 33 39.890625 33.796875 38.890625 33.796875 37.5 \n",
       "C 33.796875 35.703125 32.796875 34.609375 32.296875 33.921875 \n",
       "L 24.90625 24.578125 \n",
       "L 16.09375 36.03125 \n",
       "C 15.203125 37.125 15.203125 37.3125 15.203125 37.8125 \n",
       "C 15.203125 39.015625 16.203125 40 17.90625 40 \n",
       "L 17.90625 42.6875 \n",
       "C 15.5 42.59375 11.203125 42.59375 8.703125 42.59375 \n",
       "C 6 42.59375 2.796875 42.59375 0.59375 42.6875 \n",
       "L 0.59375 40 \n",
       "C 6 40 7.203125 39.59375 9.59375 36.609375 \n",
       "L 21.09375 21.796875 \n",
       "C 21.296875 21.59375 21.796875 21 21.796875 20.703125 \n",
       "C 21.796875 20.296875 13.09375 9.671875 12 8.28125 \n",
       "C 7.5 2.890625 2.796875 2.5 0.09375 2.5 \n",
       "L 0.09375 0 \n",
       "C 1.796875 0 5.296875 0 7.09375 0 \n",
       "C 9.09375 0 13.5 0 15.296875 -0.09375 \n",
       "L 15.296875 2.5 \n",
       "C 14.703125 2.59375 13.09375 2.796875 13.09375 5.078125 \n",
       "C 13.09375 6.875 13.90625 7.96875 14.796875 9.15625 \n",
       "L 23.09375 19.3125 \n",
       "L 31.796875 8 \n",
       "C 32.59375 6.90625 33.796875 5.40625 33.796875 4.703125 \n",
       "C 33.796875 3.296875 32.59375 2.5 31 2.5 \n",
       "L 31 -0.09375 \n",
       "C 33.40625 0 37.703125 0 40.203125 0 \n",
       "C 42.90625 0 46.09375 0 48.296875 -0.09375 \n",
       "L 48.296875 2.5 \n",
       "C 43.296875 2.5 41.796875 2.6875 39.40625 5.78125 \n",
       "z\n",
       "\" id=\"CMR17-120\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(43.868113 69.256831)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(66.509881 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(112.200358 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(189.119948 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(240.015317 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(264.886376 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(305.372036 0)scale(0.996264)\" xlink:href=\"#CMR17-120\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_8\">\n",
       "     <g id=\"line2d_27\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#m5c1494a952\" y=\"36.267314\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- Gene -->\n",
       "      <defs>\n",
       "       <path d=\"M 62.09375 19.265625 \n",
       "C 62.09375 23.015625 62.40625 23.515625 68.59375 23.515625 \n",
       "L 68.59375 26.09375 \n",
       "C 66.203125 26 60.796875 26 58.09375 26 \n",
       "C 55.296875 26 47.5 26 43.703125 26 \n",
       "L 43.703125 23.515625 \n",
       "L 46.703125 23.515625 \n",
       "C 55.296875 23.515625 55.5 22.40625 55.5 18.765625 \n",
       "L 55.5 12.09375 \n",
       "C 55.5 0.984375 42.5 0.984375 39.40625 0.984375 \n",
       "C 29.09375 0.984375 12.203125 7.984375 12.203125 33.75 \n",
       "C 12.203125 59.5 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.65625 59.90625 42.828125 \n",
       "C 60.09375 41.625 60.09375 41.421875 61 41.421875 \n",
       "C 62.09375 41.421875 62.09375 41.625 62.09375 43.421875 \n",
       "L 62.09375 67 \n",
       "C 62.09375 68.78125 62 68.890625 61.40625 68.890625 \n",
       "C 61 68.890625 60.90625 68.78125 60.203125 67.59375 \n",
       "L 55.796875 59.234375 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 13.203125 19.796875 -2 37.40625 -2 \n",
       "C 45.5 -2 53.40625 0.875 56.5 6.515625 \n",
       "C 58.203125 2.46875 61 -0.3125 61.5 -0.3125 \n",
       "C 62 -0.3125 62.09375 -0.203125 62.09375 1.5625 \n",
       "z\n",
       "\" id=\"CMR17-71\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(60.126662 39.726563)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "       <use transform=\"translate(72.355346 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(112.841006 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(163.736376 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path d=\"M 89.335068 257.744323 \n",
       "L 89.335068 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path d=\"M 330.860125 257.744323 \n",
       "L 330.860125 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 89.335068 257.744323 \n",
       "L 330.860125 257.744323 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 89.335068 21.50218 \n",
       "L 330.860125 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_11\">\n",
       "    <!-- Number of nodes: 3701 -->\n",
       "    <defs>\n",
       "     <path d=\"M 20.5 66.40625 \n",
       "C 19.796875 67.5 19.703125 67.5 17.796875 67.5 \n",
       "L 5.09375 67.5 \n",
       "L 5.09375 65 \n",
       "L 7 65 \n",
       "C 11 65 12.796875 64.515625 13.09375 64.40625 \n",
       "L 13.09375 9.53125 \n",
       "C 13.09375 7.046875 13.09375 2.390625 5.09375 2.390625 \n",
       "L 5.09375 -0.203125 \n",
       "C 7.40625 0 11.59375 0 14.09375 0 \n",
       "C 16.59375 0 20.796875 0 23.09375 -0.203125 \n",
       "L 23.09375 2.390625 \n",
       "C 15.09375 2.390625 15.09375 7.046875 15.09375 9.53125 \n",
       "L 15.09375 63.015625 \n",
       "C 15.796875 62.421875 15.796875 62.21875 16.5 61.234375 \n",
       "L 53.59375 1.1875 \n",
       "C 54.296875 0 54.59375 0 55.09375 0 \n",
       "C 56.09375 0 56.09375 0.296875 56.09375 2.171875 \n",
       "L 56.09375 57.890625 \n",
       "C 56.09375 60.359375 56.09375 65 64.09375 65 \n",
       "L 64.09375 67.59375 \n",
       "C 61.796875 67.5 57.59375 67.5 55.09375 67.5 \n",
       "C 52.59375 67.5 48.40625 67.5 46.09375 67.59375 \n",
       "L 46.09375 65 \n",
       "C 54.09375 65 54.09375 60.34375 54.09375 57.859375 \n",
       "L 54.09375 11.921875 \n",
       "z\n",
       "\" id=\"CMR17-78\"/>\n",
       "     <path d=\"M 15.296875 40 \n",
       "L 26.703125 40 \n",
       "L 26.703125 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 54.59375 \n",
       "C 15.09375 62.84375 19.296875 68 23.59375 68 \n",
       "C 24.90625 68 26.40625 67.609375 27.40625 66.90625 \n",
       "C 26.59375 66.703125 24.5 66.015625 24.5 63.546875 \n",
       "C 24.5 60.953125 26.5 60.15625 27.90625 60.15625 \n",
       "C 29.296875 60.15625 31.296875 60.953125 31.296875 63.546875 \n",
       "C 31.296875 67.3125 27.703125 69.59375 23.703125 69.59375 \n",
       "C 18 69.59375 10.203125 64.9375 10.203125 54.390625 \n",
       "L 10.203125 42.578125 \n",
       "L 2.203125 42.578125 \n",
       "L 2.203125 40 \n",
       "L 10.203125 40 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.40625 0 13 0 \n",
       "C 15.90625 0 21 0 23.703125 -0.203125 \n",
       "L 23.703125 2.390625 \n",
       "C 16.5 2.390625 15.296875 2.390625 15.296875 6.859375 \n",
       "z\n",
       "\" id=\"CMR17-102\"/>\n",
       "     <path d=\"M 16.59375 38.921875 \n",
       "C 16.59375 41.40625 14.59375 43 12.5 43 \n",
       "C 10.5 43 8.40625 41.40625 8.40625 38.921875 \n",
       "C 8.40625 36.4375 10.40625 34.84375 12.5 34.84375 \n",
       "C 14.5 34.84375 16.59375 36.4375 16.59375 38.921875 \n",
       "z\n",
       "M 16.59375 4.078125 \n",
       "C 16.59375 6.5625 14.59375 8.15625 12.5 8.15625 \n",
       "C 10.5 8.15625 8.40625 6.5625 8.40625 4.078125 \n",
       "C 8.40625 1.59375 10.40625 0 12.5 0 \n",
       "C 14.5 0 16.59375 1.59375 16.59375 4.078125 \n",
       "z\n",
       "\" id=\"CMR17-58\"/>\n",
       "     <path d=\"M 45.09375 61.578125 \n",
       "L 45.09375 63.765625 \n",
       "L 21.59375 63.765625 \n",
       "C 9.90625 63.765625 9.703125 65.078125 9.296875 67 \n",
       "L 7.5 67 \n",
       "L 4.59375 48.34375 \n",
       "L 6.40625 48.34375 \n",
       "C 6.703125 50.234375 7.40625 55.3125 8.59375 57.203125 \n",
       "C 9.203125 58 16.59375 58 18.296875 58 \n",
       "L 40.296875 58 \n",
       "L 29.203125 41.578125 \n",
       "C 21.796875 30.53125 16.703125 15.609375 16.703125 2.578125 \n",
       "C 16.703125 1.390625 16.703125 -2 20.296875 -2 \n",
       "C 23.90625 -2 23.90625 1.390625 23.90625 2.671875 \n",
       "L 23.90625 7.265625 \n",
       "C 23.90625 23.5625 26.703125 34.3125 31.296875 41.1875 \n",
       "z\n",
       "\" id=\"CMR17-55\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(150.399171 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "     <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(194.284936 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "     <use transform=\"translate(247.782744 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(288.268404 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(353.625139 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(399.315616 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(456.865095 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(507.760464 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(556.05338 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "     <use transform=\"translate(606.94875 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(647.43441 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(683.235748 0)scale(0.996264)\" xlink:href=\"#CMR17-58\"/>\n",
       "     <use transform=\"translate(748.592467 0)scale(0.996264)\" xlink:href=\"#CMR17-51\"/>\n",
       "     <use transform=\"translate(794.282944 0)scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "     <use transform=\"translate(839.973421 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     <use transform=\"translate(885.663898 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"legend_1\">\n",
       "    <g id=\"patch_15\">\n",
       "     <path d=\"M 286.87879 252.744323 \n",
       "L 323.860125 252.744323 \n",
       "Q 325.860125 252.744323 325.860125 250.744323 \n",
       "L 325.860125 237.807145 \n",
       "Q 325.860125 235.807145 323.860125 235.807145 \n",
       "L 286.87879 235.807145 \n",
       "Q 284.87879 235.807145 284.87879 237.807145 \n",
       "L 284.87879 250.744323 \n",
       "Q 284.87879 252.744323 286.87879 252.744323 \n",
       "z\n",
       "\" style=\"fill:#ffffff;opacity:0.8;stroke:#cccccc;stroke-linejoin:miter;\"/>\n",
       "    </g>\n",
       "    <g id=\"patch_16\">\n",
       "     <path d=\"M 288.87879 246.807145 \n",
       "L 308.87879 246.807145 \n",
       "L 308.87879 239.807145 \n",
       "L 288.87879 239.807145 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "    </g>\n",
       "    <g id=\"text_12\">\n",
       "     <!-- 0 -->\n",
       "     <g transform=\"translate(316.87879 246.807145)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_2\">\n",
       "   <g id=\"patch_17\">\n",
       "    <path d=\"M 456.934139 257.744323 \n",
       "L 698.459196 257.744323 \n",
       "L 698.459196 21.50218 \n",
       "L 456.934139 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_18\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 254.463182 \n",
       "L 467.91255 254.463182 \n",
       "L 467.91255 247.900901 \n",
       "L -16835.502011 247.900901 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_19\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 241.338619 \n",
       "L 467.91255 241.338619 \n",
       "L 467.91255 234.776337 \n",
       "L -16835.502011 234.776337 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_20\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 228.214055 \n",
       "L 525.590599 228.214055 \n",
       "L 525.590599 221.651774 \n",
       "L -16835.502011 221.651774 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_21\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 215.089492 \n",
       "L 541.668563 215.089492 \n",
       "L 541.668563 208.52721 \n",
       "L -16835.502011 208.52721 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_22\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 201.964928 \n",
       "L 547.520441 201.964928 \n",
       "L 547.520441 195.402647 \n",
       "L -16835.502011 195.402647 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_23\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 188.840365 \n",
       "L 553.931383 188.840365 \n",
       "L 553.931383 182.278083 \n",
       "L -16835.502011 182.278083 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_24\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 175.715801 \n",
       "L 572.860294 175.715801 \n",
       "L 572.860294 169.153519 \n",
       "L -16835.502011 169.153519 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_25\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 162.591238 \n",
       "L 573.608088 162.591238 \n",
       "L 573.608088 156.028956 \n",
       "L -16835.502011 156.028956 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_26\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 149.466674 \n",
       "L 583.764689 149.466674 \n",
       "L 583.764689 142.904392 \n",
       "L -16835.502011 142.904392 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_27\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 136.342111 \n",
       "L 586.550808 136.342111 \n",
       "L 586.550808 129.779829 \n",
       "L -16835.502011 129.779829 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_28\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 123.217547 \n",
       "L 601.127511 123.217547 \n",
       "L 601.127511 116.655265 \n",
       "L -16835.502011 116.655265 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_29\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 110.092984 \n",
       "L 606.120653 110.092984 \n",
       "L 606.120653 103.530702 \n",
       "L -16835.502011 103.530702 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_30\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 96.96842 \n",
       "L 637.971256 96.96842 \n",
       "L 637.971256 90.406138 \n",
       "L -16835.502011 90.406138 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_31\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 83.843857 \n",
       "L 641.195944 83.843857 \n",
       "L 641.195944 77.281575 \n",
       "L -16835.502011 77.281575 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_32\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 70.719293 \n",
       "L 652.922641 70.719293 \n",
       "L 652.922641 64.157011 \n",
       "L -16835.502011 64.157011 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_33\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 57.59473 \n",
       "L 663.280317 57.59473 \n",
       "L 663.280317 51.032448 \n",
       "L -16835.502011 51.032448 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_34\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 44.470166 \n",
       "L 687.106702 44.470166 \n",
       "L 687.106702 37.907884 \n",
       "L -16835.502011 37.907884 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_35\">\n",
       "    <path clip-path=\"url(#p3102247790)\" d=\"M -16835.502011 31.345602 \n",
       "L 687.480784 31.345602 \n",
       "L 687.480784 24.783321 \n",
       "L -16835.502011 24.783321 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_3\">\n",
       "    <g id=\"xtick_20\">\n",
       "     <g id=\"line2d_28\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"467.91255\" xlink:href=\"#md1ad39f64b\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_13\">\n",
       "      <!-- $10^{0}$ -->\n",
       "      <g transform=\"translate(460.945596 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_21\">\n",
       "     <g id=\"line2d_29\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"525.590599\" xlink:href=\"#md1ad39f64b\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_14\">\n",
       "      <!-- $10^{1}$ -->\n",
       "      <g transform=\"translate(518.623645 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-49\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_22\">\n",
       "     <g id=\"line2d_30\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"583.268647\" xlink:href=\"#md1ad39f64b\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- $10^{2}$ -->\n",
       "      <g transform=\"translate(576.301693 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_23\">\n",
       "     <g id=\"line2d_31\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"640.946696\" xlink:href=\"#md1ad39f64b\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- $10^{3}$ -->\n",
       "      <g transform=\"translate(633.979742 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-51\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_24\">\n",
       "     <g id=\"line2d_32\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"698.624744\" xlink:href=\"#md1ad39f64b\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- $10^{4}$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(691.65779 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_25\">\n",
       "     <g id=\"line2d_33\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"458.978108\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_26\">\n",
       "     <g id=\"line2d_34\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"462.32297\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_27\">\n",
       "     <g id=\"line2d_35\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"465.273348\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_28\">\n",
       "     <g id=\"line2d_36\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"485.275373\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_29\">\n",
       "     <g id=\"line2d_37\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"495.431973\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_30\">\n",
       "     <g id=\"line2d_38\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"502.638196\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_31\">\n",
       "     <g id=\"line2d_39\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"508.227776\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_32\">\n",
       "     <g id=\"line2d_40\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"512.794796\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_33\">\n",
       "     <g id=\"line2d_41\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"516.656156\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_34\">\n",
       "     <g id=\"line2d_42\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"520.001018\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_35\">\n",
       "     <g id=\"line2d_43\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"522.951396\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_36\">\n",
       "     <g id=\"line2d_44\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"542.953422\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_37\">\n",
       "     <g id=\"line2d_45\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"553.110022\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_38\">\n",
       "     <g id=\"line2d_46\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"560.316244\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_39\">\n",
       "     <g id=\"line2d_47\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"565.905825\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_40\">\n",
       "     <g id=\"line2d_48\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"570.472844\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_41\">\n",
       "     <g id=\"line2d_49\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"574.334205\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_42\">\n",
       "     <g id=\"line2d_50\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"577.679067\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_43\">\n",
       "     <g id=\"line2d_51\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"580.629445\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_44\">\n",
       "     <g id=\"line2d_52\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"600.63147\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_45\">\n",
       "     <g id=\"line2d_53\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"610.78807\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_46\">\n",
       "     <g id=\"line2d_54\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"617.994293\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_47\">\n",
       "     <g id=\"line2d_55\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"623.583873\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_48\">\n",
       "     <g id=\"line2d_56\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"628.150893\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_49\">\n",
       "     <g id=\"line2d_57\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"632.012253\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_50\">\n",
       "     <g id=\"line2d_58\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"635.357116\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_51\">\n",
       "     <g id=\"line2d_59\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"638.307493\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_52\">\n",
       "     <g id=\"line2d_60\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"658.309519\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_53\">\n",
       "     <g id=\"line2d_61\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"668.466119\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_54\">\n",
       "     <g id=\"line2d_62\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"675.672341\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_55\">\n",
       "     <g id=\"line2d_63\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"681.261922\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_56\">\n",
       "     <g id=\"line2d_64\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"685.828942\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_57\">\n",
       "     <g id=\"line2d_65\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"689.690302\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_58\">\n",
       "     <g id=\"line2d_66\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"693.035164\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_59\">\n",
       "     <g id=\"line2d_67\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"695.985542\" xlink:href=\"#m8d2aa11a19\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_4\">\n",
       "    <g id=\"ytick_9\">\n",
       "     <g id=\"line2d_68\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"251.182042\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_18\">\n",
       "      <!-- prognosticBiomarkerFor -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.09375 26.140625 \n",
       "C 26 26.34375 25.5 26.9375 25.5 27.234375 \n",
       "C 25.5 27.4375 32.59375 33.921875 33.5 34.71875 \n",
       "C 39.203125 39.5 42.796875 40 45.40625 40 \n",
       "L 45.40625 42.703125 \n",
       "C 44.796875 42.703125 44.59375 42.703125 44.5 42.5 \n",
       "L 39.59375 42.5 \n",
       "C 36.796875 42.5 32.5 42.5 29.796875 42.703125 \n",
       "L 29.796875 40.09375 \n",
       "C 31.203125 39.984375 32.203125 39.1875 32.203125 37.796875 \n",
       "C 32.203125 35.796875 30.203125 33.90625 30.09375 33.90625 \n",
       "L 15 20.140625 \n",
       "L 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 6.65625 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 9.90625 0 12.40625 0 \n",
       "C 14.90625 0 19.296875 0 21.703125 -0.203125 \n",
       "L 21.703125 2.390625 \n",
       "C 15.296875 2.390625 14.90625 2.78125 14.90625 6.6875 \n",
       "L 14.90625 17.65625 \n",
       "L 21.796875 23.9375 \n",
       "L 31.90625 8.96875 \n",
       "C 33.09375 7.171875 34.09375 5.765625 34.09375 4.46875 \n",
       "C 34.09375 2.6875 32.296875 2.390625 31.09375 2.390625 \n",
       "L 31.09375 -0.203125 \n",
       "C 33.5 0 37.796875 0 40.296875 0 \n",
       "C 42.09375 0 45.796875 0 47.5 0 \n",
       "L 47.5 2.390625 \n",
       "C 42.796875 2.390625 41.703125 3.28125 38.703125 7.671875 \n",
       "z\n",
       "\" id=\"CMR17-107\"/>\n",
       "       <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(345.160125 254.641291)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(86.176138 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(131.866615 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(177.557092 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(228.452462 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(274.142939 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(309.944276 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(345.225044 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(370.096103 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(410.581763 0)scale(0.996264)\" xlink:href=\"#CMR17-66\"/>\n",
       "       <use transform=\"translate(475.770518 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(500.641576 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(546.332053 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(623.251643 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(668.94212 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(704.222888 0)scale(0.996264)\" xlink:href=\"#CMR17-107\"/>\n",
       "       <use transform=\"translate(749.91335 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(790.39901 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(825.679778 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "       <use transform=\"translate(877.856369 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(923.546846 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_10\">\n",
       "     <g id=\"line2d_69\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"238.057478\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_19\">\n",
       "      <!-- rateLimitingStepOf -->\n",
       "      <defs>\n",
       "       <path d=\"M 54.203125 25.234375 \n",
       "L 52.40625 25.234375 \n",
       "C 51.59375 15.359375 50.296875 2.59375 33.203125 2.59375 \n",
       "L 23.90625 2.59375 \n",
       "C 19.90625 2.59375 19.703125 3.078125 19.703125 6.46875 \n",
       "L 19.703125 60.90625 \n",
       "C 19.703125 64.515625 19.90625 65.5 27.5 65.5 \n",
       "L 29.703125 65.5 \n",
       "L 29.703125 68 \n",
       "C 25.59375 68 20.796875 68 16.703125 68 \n",
       "C 13.59375 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 7.0625 \n",
       "C 13.09375 2.59375 12 2.59375 5.09375 2.59375 \n",
       "L 5.09375 0 \n",
       "L 51.796875 0 \n",
       "z\n",
       "\" id=\"CMR17-76\"/>\n",
       "       <path d=\"M 43.796875 67.03125 \n",
       "C 43.796875 68.8125 43.703125 68.921875 43.09375 68.921875 \n",
       "C 42.5 68.921875 42.296875 68.515625 41.90625 67.625 \n",
       "L 38.90625 61.046875 \n",
       "C 35.09375 66.515625 29.296875 69 23 69 \n",
       "C 12.796875 69 4.59375 60.953125 4.59375 50.828125 \n",
       "C 4.59375 47.75 5.296875 43.375 8.703125 39.21875 \n",
       "C 12.40625 34.65625 15.90625 33.859375 23.90625 31.765625 \n",
       "C 26.90625 30.96875 31.90625 29.671875 32.90625 29.28125 \n",
       "C 38.59375 26.90625 41.59375 20.953125 41.59375 15.390625 \n",
       "C 41.59375 8.234375 36.5 0.59375 27.796875 0.59375 \n",
       "C 17.09375 0.59375 7.09375 6.546875 6.40625 20.484375 \n",
       "C 6.296875 22.078125 6.203125 22.171875 5.5 22.171875 \n",
       "C 4.703125 22.171875 4.59375 22.078125 4.59375 20.078125 \n",
       "L 4.59375 -0.015625 \n",
       "C 4.59375 -1.796875 4.703125 -1.90625 5.296875 -1.90625 \n",
       "C 6 -1.90625 6.203125 -1.40625 7.59375 1.6875 \n",
       "C 7.703125 1.984375 7.703125 2.1875 9.59375 5.953125 \n",
       "C 14.59375 -0.609375 23 -2 27.796875 -2 \n",
       "C 38.703125 -2 46.40625 7.046875 46.40625 17.5625 \n",
       "C 46.40625 22.71875 44.5 26.6875 43.40625 28.484375 \n",
       "C 39.203125 34.84375 34.703125 36.03125 29.90625 37.21875 \n",
       "C 29.203125 37.53125 29 37.53125 23.40625 38.90625 \n",
       "C 18 40.296875 15.703125 41 13.203125 43.6875 \n",
       "C 10.40625 46.75 9.40625 49.9375 9.40625 53.109375 \n",
       "C 9.40625 59.5625 14.59375 66.609375 23.09375 66.609375 \n",
       "C 33.703125 66.609375 40.296875 59.15625 41.796875 46.765625 \n",
       "C 42.09375 44.96875 42.203125 44.875 42.90625 44.875 \n",
       "C 43.796875 44.875 43.796875 45.171875 43.796875 46.859375 \n",
       "z\n",
       "\" id=\"CMR17-83\"/>\n",
       "       <path d=\"M 67.296875 33.359375 \n",
       "C 67.296875 53.703125 52.59375 69 36 69 \n",
       "C 19 69 4.59375 53.5 4.59375 33.359375 \n",
       "C 4.59375 13.203125 19.203125 -2 35.90625 -2 \n",
       "C 53 -2 67.296875 13.5 67.296875 33.359375 \n",
       "z\n",
       "M 36 -0.203125 \n",
       "C 24.59375 -0.203125 12.203125 11.109375 12.203125 34.640625 \n",
       "C 12.203125 58.0625 25.796875 67.203125 35.90625 67.203125 \n",
       "C 46.5 67.203125 59.703125 57.78125 59.703125 34.640625 \n",
       "C 59.703125 10.71875 47 -0.203125 36 -0.203125 \n",
       "z\n",
       "\" id=\"CMR17-79\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(365.362169 241.516727)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(35.280768 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(80.971245 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(116.252013 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(156.737673 0)scale(0.996264)\" xlink:href=\"#CMR17-76\"/>\n",
       "       <use transform=\"translate(214.119081 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(238.99014 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(315.909729 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(340.780788 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(376.061556 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(400.932614 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(451.827984 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(497.518461 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "       <use transform=\"translate(548.413831 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(583.694599 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(624.180259 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(675.075629 0)scale(0.996264)\" xlink:href=\"#CMR17-79\"/>\n",
       "       <use transform=\"translate(746.790417 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_11\">\n",
       "     <g id=\"line2d_70\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"224.932914\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_20\">\n",
       "      <!-- regulates -->\n",
       "      <g transform=\"translate(411.107449 228.392164)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(35.280768 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(121.456905 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "       <use transform=\"translate(172.352275 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(197.223334 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(242.913811 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(278.194579 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(318.680239 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_12\">\n",
       "     <g id=\"line2d_71\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"211.808351\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_21\">\n",
       "      <!-- hasProduct -->\n",
       "      <g transform=\"translate(400.176166 215.2676)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(194.973485 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(230.254253 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(278.547169 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(329.442538 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "       <use transform=\"translate(380.337908 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(420.823569 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_13\">\n",
       "     <g id=\"line2d_72\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"198.683787\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_22\">\n",
       "      <!-- hasReactant -->\n",
       "      <g transform=\"translate(396.329507 202.143037)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-82\"/>\n",
       "       <use transform=\"translate(200.178287 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(240.663947 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(286.354424 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(326.840085 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(362.120852 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(407.811329 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(456.104245 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_14\">\n",
       "     <g id=\"line2d_73\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"185.559224\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_23\">\n",
       "      <!-- biomarkerFor -->\n",
       "      <g transform=\"translate(391.597208 189.018473)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(121.456905 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(198.376495 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(244.066972 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(279.34774 0)scale(0.996264)\" xlink:href=\"#CMR17-107\"/>\n",
       "       <use transform=\"translate(325.038202 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(365.523863 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(400.80463 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "       <use transform=\"translate(452.981222 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(498.671699 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_15\">\n",
       "     <g id=\"line2d_74\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"172.43466\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_24\">\n",
       "      <!-- isA -->\n",
       "      <g transform=\"translate(435.765029 175.89391)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(24.871058 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(60.672396 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_16\">\n",
       "     <g id=\"line2d_75\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"159.310097\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_25\">\n",
       "      <!-- hasMember -->\n",
       "      <defs>\n",
       "       <path d=\"M 20.90625 66 \n",
       "C 20.296875 67.5 20.203125 67.5 18.09375 67.5 \n",
       "L 5.296875 67.5 \n",
       "L 5.296875 65 \n",
       "C 12.203125 65 13.296875 65 13.296875 60.546875 \n",
       "L 13.296875 9.53125 \n",
       "C 13.296875 7.046875 13.296875 2.390625 5.296875 2.390625 \n",
       "L 5.296875 -0.203125 \n",
       "C 7.59375 0 11.796875 0 14.296875 0 \n",
       "C 16.796875 0 21 0 23.296875 -0.203125 \n",
       "L 23.296875 2.390625 \n",
       "C 15.296875 2.390625 15.296875 7.046875 15.296875 9.53125 \n",
       "L 15.296875 64.609375 \n",
       "L 15.40625 64.609375 \n",
       "L 38.90625 1.390625 \n",
       "C 39.296875 0.390625 39.5 -0.203125 40.296875 -0.203125 \n",
       "C 41.09375 -0.203125 41.296875 0.390625 41.703125 1.390625 \n",
       "L 65.40625 65.09375 \n",
       "L 65.5 65.09375 \n",
       "L 65.5 6.84375 \n",
       "C 65.5 2.390625 64.40625 2.390625 57.5 2.390625 \n",
       "L 57.5 -0.203125 \n",
       "C 60.203125 0 65.59375 0 68.5 0 \n",
       "C 71.40625 0 76.90625 0 79.59375 -0.203125 \n",
       "L 79.59375 2.390625 \n",
       "C 72.703125 2.390625 71.59375 2.390625 71.59375 6.84375 \n",
       "L 71.59375 60.546875 \n",
       "C 71.59375 65 72.703125 65 79.59375 65 \n",
       "L 79.59375 67.5 \n",
       "L 66.90625 67.5 \n",
       "C 64.796875 67.5 64.703125 67.40625 64.09375 65.90625 \n",
       "L 42.5 7.84375 \n",
       "z\n",
       "\" id=\"CMR17-77\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(399.761082 162.769346)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-77\"/>\n",
       "       <use transform=\"translate(217.074125 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(257.559786 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(331.876922 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "       <use transform=\"translate(385.37473 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(425.86039 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_17\">\n",
       "     <g id=\"line2d_76\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"146.185533\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_26\">\n",
       "      <!-- directlyDecreases -->\n",
       "      <defs>\n",
       "       <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 37 0 \n",
       "C 53.09375 0 66 14.734375 66 33.453125 \n",
       "C 66 52.28125 53.296875 68 37 68 \n",
       "z\n",
       "M 23.796875 2.59375 \n",
       "C 19.796875 2.59375 19.59375 3.078125 19.59375 6.46875 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 35 65.40625 \n",
       "C 43.296875 65.40625 58.59375 60.53125 58.59375 33.453125 \n",
       "C 58.59375 19.40625 54.5 13.734375 52.59375 11.046875 \n",
       "C 50.203125 7.96875 44.5 2.59375 35 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-68\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(374.909831 149.644783)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(111.047196 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(151.532857 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(192.018517 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(227.299285 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(252.170343 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "       <use transform=\"translate(300.463259 0)scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "       <use transform=\"translate(370.856815 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(411.342475 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(451.828136 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(487.108904 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(527.594564 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(573.285041 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(609.086379 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(649.572039 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_18\">\n",
       "     <g id=\"line2d_77\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"133.06097\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_27\">\n",
       "      <!-- causesNoChange -->\n",
       "      <g transform=\"translate(377.594177 136.520219)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(40.48566 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(86.176138 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "       <use transform=\"translate(137.071507 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(172.872845 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(213.358505 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(249.159843 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "       <use transform=\"translate(318.232273 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(363.92275 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(430.432631 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(481.328 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(527.018477 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(577.913847 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(623.604324 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_19\">\n",
       "     <g id=\"line2d_78\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"119.936406\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_28\">\n",
       "      <!-- directlyIncreases -->\n",
       "      <defs>\n",
       "       <path d=\"M 19.703125 61.015625 \n",
       "C 19.703125 65.5 20.90625 65.5 28.09375 65.5 \n",
       "L 28.09375 68 \n",
       "C 24.90625 68 19.796875 68 16.40625 68 \n",
       "C 13 68 7.90625 68 4.703125 68 \n",
       "L 4.703125 65.5 \n",
       "C 11.90625 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 6.984375 \n",
       "C 13.09375 2.5 11.90625 2.5 4.703125 2.5 \n",
       "L 4.703125 0 \n",
       "C 7.90625 0 13 0 16.40625 0 \n",
       "C 19.796875 0 24.90625 0 28.09375 0 \n",
       "L 28.09375 2.5 \n",
       "C 20.90625 2.5 19.703125 2.5 19.703125 6.984375 \n",
       "z\n",
       "\" id=\"CMR17-73\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(377.815591 123.395655)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(111.047196 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(151.532857 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(192.018517 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(227.299285 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(252.170343 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "       <use transform=\"translate(300.463259 0)scale(0.996264)\" xlink:href=\"#CMR17-73\"/>\n",
       "       <use transform=\"translate(333.101668 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(383.997038 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(424.482699 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(459.763466 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(500.249127 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(545.939604 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(581.740942 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(622.226602 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_20\">\n",
       "     <g id=\"line2d_79\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"106.811843\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_29\">\n",
       "      <!-- hasVariant -->\n",
       "      <defs>\n",
       "       <path d=\"M 59 58.515625 \n",
       "C 59.90625 60.71875 61.5 65 68.203125 65 \n",
       "L 68.203125 67.6875 \n",
       "C 65.90625 67.484375 63.296875 67.484375 60.796875 67.484375 \n",
       "C 58.40625 67.484375 54.296875 67.484375 52 67.6875 \n",
       "L 52 65 \n",
       "C 57 65 57.203125 61.5 57.203125 60.90625 \n",
       "C 57.203125 59.796875 56.90625 59.109375 56.59375 58.3125 \n",
       "L 36.90625 6.96875 \n",
       "L 16.296875 60.71875 \n",
       "C 16 61.421875 15.796875 61.921875 15.796875 62.40625 \n",
       "C 15.796875 65 19.796875 65 21.90625 65 \n",
       "L 21.90625 67.484375 \n",
       "C 18.703125 67.484375 14 67.484375 10.703125 67.484375 \n",
       "C 8.09375 67.484375 3.40625 67.484375 1 67.6875 \n",
       "L 1 65 \n",
       "C 6.796875 65 7.90625 64.609375 9.09375 61.421875 \n",
       "L 32.90625 -0.40625 \n",
       "C 33.40625 -1.796875 33.5 -2 34.59375 -2 \n",
       "C 35.703125 -2 35.796875 -1.796875 36.296875 -0.40625 \n",
       "z\n",
       "\" id=\"CMR17-86\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(403.081957 110.271092)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-86\"/>\n",
       "       <use transform=\"translate(193.652359 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(239.342836 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(274.623604 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(299.494662 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(345.185139 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(393.478055 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_21\">\n",
       "     <g id=\"line2d_80\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"93.687279\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_30\">\n",
       "      <!-- negativeCorrelation -->\n",
       "      <defs>\n",
       "       <path d=\"M 39.09375 33.796875 \n",
       "C 40 36 41.59375 40 47.40625 40 \n",
       "L 47.40625 42.59375 \n",
       "C 45.5 42.59375 43 42.59375 41.09375 42.59375 \n",
       "C 39 42.59375 35.703125 42.59375 33.703125 42.6875 \n",
       "L 33.703125 40.09375 \n",
       "C 36.203125 39.890625 37.59375 38.703125 37.59375 36.109375 \n",
       "C 37.59375 35.203125 37.59375 35 37 33.609375 \n",
       "L 26 5.890625 \n",
       "L 14.09375 35.71875 \n",
       "C 13.5 37.015625 13.5 37.203125 13.5 37.609375 \n",
       "C 13.5 40 16.796875 40 18.296875 40 \n",
       "L 18.296875 42.6875 \n",
       "C 15.90625 42.59375 11.5 42.59375 9 42.59375 \n",
       "C 6.296875 42.59375 3.203125 42.59375 1 42.6875 \n",
       "L 1 40 \n",
       "C 4.796875 40 6.90625 40 8.09375 36.90625 \n",
       "L 22.59375 0.6875 \n",
       "C 23.203125 -0.796875 23.296875 -1 24.203125 -1 \n",
       "C 25.09375 -1 25.203125 -0.796875 25.796875 0.6875 \n",
       "z\n",
       "\" id=\"CMR17-118\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(364.642684 97.146528)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(91.38103 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(137.071507 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(182.761984 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(218.042752 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(242.913811 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "       <use transform=\"translate(288.604273 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(329.089933 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(395.599814 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(441.290291 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(476.571059 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(511.851826 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(552.337487 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(577.208545 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(622.899022 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(658.17979 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(683.050849 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(728.741326 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_22\">\n",
       "     <g id=\"line2d_81\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"80.562716\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_31\">\n",
       "      <!-- positiveCorrelation -->\n",
       "      <g transform=\"translate(367.077994 84.021965)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(53.497809 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(99.188286 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(134.989623 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(159.860682 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(195.14145 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(220.012508 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "       <use transform=\"translate(265.70297 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(306.18863 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(372.698511 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(418.388988 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(453.669756 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(488.950524 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(529.436184 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(554.307243 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(599.99772 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(635.278488 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(660.149546 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(705.840023 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_23\">\n",
       "     <g id=\"line2d_82\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"67.438152\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_32\">\n",
       "      <!-- hasComponent -->\n",
       "      <g transform=\"translate(385.121476 70.897401)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(198.897065 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(244.587542 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(321.507132 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(375.004941 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(420.695418 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(471.590787 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(512.076448 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(560.369364 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_24\">\n",
       "     <g id=\"line2d_83\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"54.313589\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_33\">\n",
       "      <!-- decreases -->\n",
       "      <g transform=\"translate(409.945181 57.772838)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(91.38103 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(131.866691 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(167.147459 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(207.633119 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(253.323596 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(289.124934 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(329.610594 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_25\">\n",
       "     <g id=\"line2d_84\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"41.189025\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_34\">\n",
       "      <!-- association -->\n",
       "      <g transform=\"translate(402.500792 44.648274)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(81.491815 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(117.293152 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(165.586068 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(206.071729 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(230.942787 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(276.633264 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(311.914032 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(336.785091 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(382.475568 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_26\">\n",
       "     <g id=\"line2d_85\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"456.934139\" xlink:href=\"#m5c1494a952\" y=\"28.064462\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_35\">\n",
       "      <!-- increases -->\n",
       "      <g transform=\"translate(411.605611 31.523711)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(24.871058 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(116.252089 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(151.532857 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(192.018517 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(237.708994 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(273.510332 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(313.995992 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_36\">\n",
       "    <path d=\"M 456.934139 257.744323 \n",
       "L 456.934139 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_37\">\n",
       "    <path d=\"M 698.459196 257.744323 \n",
       "L 698.459196 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_38\">\n",
       "    <path d=\"M 456.934139 257.744323 \n",
       "L 698.459196 257.744323 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_39\">\n",
       "    <path d=\"M 456.934139 21.50218 \n",
       "L 698.459196 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_36\">\n",
       "    <!-- Number of edges: 19562 -->\n",
       "    <defs>\n",
       "     <path d=\"M 35.203125 29.125 \n",
       "C 35.203125 7.25 26.09375 0.484375 18.59375 0.484375 \n",
       "C 16.296875 0.484375 10.703125 0.484375 8.40625 4.59375 \n",
       "C 11 4.1875 12.90625 5.578125 12.90625 8.09375 \n",
       "C 12.90625 10.8125 10.703125 11.703125 9.296875 11.703125 \n",
       "C 8.40625 11.703125 5.703125 11.296875 5.703125 7.90625 \n",
       "C 5.703125 1.109375 11.59375 -2 18.796875 -2 \n",
       "C 30.296875 -2 42 10.53125 42 32.390625 \n",
       "C 42 59.640625 30.796875 65.296875 23.203125 65.296875 \n",
       "C 13.296875 65.296875 3.796875 56.6875 3.796875 43.40625 \n",
       "C 3.796875 31.109375 11.90625 22 22.09375 22 \n",
       "C 30.5 22 34.203125 29.734375 35.203125 33 \n",
       "z\n",
       "M 22.296875 23.59375 \n",
       "C 19.59375 23.59375 15.796875 24.09375 12.703125 30.03125 \n",
       "C 10.59375 33.890625 10.59375 38.453125 10.59375 43.296875 \n",
       "C 10.59375 49.15625 10.59375 53.203125 13.40625 57.5625 \n",
       "C 14.796875 59.640625 17.40625 63 23.203125 63 \n",
       "C 35 63 35 45.078125 35 41.125 \n",
       "C 35 34.09375 31.796875 23.59375 22.296875 23.59375 \n",
       "z\n",
       "\" id=\"CMR17-57\"/>\n",
       "     <path d=\"M 11.40625 57.6875 \n",
       "C 12.40625 57.28125 16.5 56 20.703125 56 \n",
       "C 30 56 35.09375 61.109375 38 64.0625 \n",
       "C 38 64.953125 38 65.5 37.40625 65.5 \n",
       "C 37.296875 65.5 37.09375 65.5 36.296875 65.046875 \n",
       "C 32.796875 63.40625 28.703125 62.078125 23.703125 62.078125 \n",
       "C 20.703125 62.078125 16.203125 62.484375 11.296875 64.671875 \n",
       "C 10.203125 65.171875 10 65.171875 9.90625 65.171875 \n",
       "C 9.40625 65.171875 9.296875 65.0625 9.296875 63.078125 \n",
       "L 9.296875 34.421875 \n",
       "C 9.296875 32.640625 9.296875 32.140625 10.296875 32.140625 \n",
       "C 10.796875 32.140625 11 32.34375 11.5 33.03125 \n",
       "C 14.703125 37.515625 19.09375 39.40625 24.09375 39.40625 \n",
       "C 27.59375 39.40625 35.09375 37.21875 35.09375 20.140625 \n",
       "C 35.09375 16.953125 35.09375 11.171875 32.09375 6.578125 \n",
       "C 29.59375 2.484375 25.703125 0.390625 21.40625 0.390625 \n",
       "C 14.796875 0.390625 8.09375 5 6.296875 12.71875 \n",
       "C 6.703125 12.609375 7.5 12.421875 7.90625 12.421875 \n",
       "C 9.203125 12.421875 11.703125 13.125 11.703125 16.21875 \n",
       "C 11.703125 18.90625 9.796875 20 7.90625 20 \n",
       "C 5.59375 20 4.09375 18.59375 4.09375 15.796875 \n",
       "C 4.09375 7.09375 11 -2 21.59375 -2 \n",
       "C 31.90625 -2 41.703125 6.875 41.703125 19.75 \n",
       "C 41.703125 31.71875 33.90625 41 24.203125 41 \n",
       "C 19.09375 41 14.796875 39.109375 11.40625 35.53125 \n",
       "z\n",
       "\" id=\"CMR17-53\"/>\n",
       "     <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "z\n",
       "\" id=\"CMR17-54\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(515.884662 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "     <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(194.284936 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "     <use transform=\"translate(247.782744 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(288.268404 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(353.625139 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(399.315616 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(456.865095 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(497.350755 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "     <use transform=\"translate(548.246125 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(593.936602 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(634.422262 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(670.2236 0)scale(0.996264)\" xlink:href=\"#CMR17-58\"/>\n",
       "     <use transform=\"translate(735.580319 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "     <use transform=\"translate(781.270796 0)scale(0.996264)\" xlink:href=\"#CMR17-57\"/>\n",
       "     <use transform=\"translate(826.961273 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "     <use transform=\"translate(872.65175 0)scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "     <use transform=\"translate(918.342227 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"legend_2\">\n",
       "    <g id=\"patch_40\">\n",
       "     <path d=\"M 654.477861 252.744323 \n",
       "L 691.459196 252.744323 \n",
       "Q 693.459196 252.744323 693.459196 250.744323 \n",
       "L 693.459196 237.807145 \n",
       "Q 693.459196 235.807145 691.459196 235.807145 \n",
       "L 654.477861 235.807145 \n",
       "Q 652.477861 235.807145 652.477861 237.807145 \n",
       "L 652.477861 250.744323 \n",
       "Q 652.477861 252.744323 654.477861 252.744323 \n",
       "z\n",
       "\" style=\"fill:#ffffff;opacity:0.8;stroke:#cccccc;stroke-linejoin:miter;\"/>\n",
       "    </g>\n",
       "    <g id=\"patch_41\">\n",
       "     <path d=\"M 656.477861 246.807145 \n",
       "L 676.477861 246.807145 \n",
       "L 676.477861 239.807145 \n",
       "L 656.477861 239.807145 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "    </g>\n",
       "    <g id=\"text_37\">\n",
       "     <!-- 0 -->\n",
       "     <g transform=\"translate(684.477861 246.807145)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"pa3d5c4fcf9\">\n",
       "   <rect height=\"236.242143\" width=\"241.525057\" x=\"89.335068\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p3102247790\">\n",
       "   <rect height=\"236.242143\" width=\"241.525057\" x=\"456.934139\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x1153be668>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pbt.summary.plot_summary(graph, plt, figsize=(10, 4))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Differential Gene Expression\n",
    "\n",
    "Differential gene expression data can be obtained from many sources, including ADNI and other large clinical studies. This analysis is concerned with the log-fold-changes on each gene, and not necessarily the p-value. This is better as a data-driven process becuase it does not require a model or multiple hypothesis testing on raw data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "data_path = os.path.join(owncloud_base, 'alzheimers', 'SevAD.csv')\n",
    "target_columns = ['Gene.symbol', 'logFC']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene.symbol</th>\n",
       "      <th>logFC</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ZNF616</td>\n",
       "      <td>-4.244691</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DEFB125</td>\n",
       "      <td>3.974393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>SNAP23</td>\n",
       "      <td>3.337636</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PHLDB2</td>\n",
       "      <td>3.192559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>LOC389895</td>\n",
       "      <td>-4.296850</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/latex": [
       "\\begin{center}{\\begin{tabular}{llr}\n",
       "\\toprule\n",
       "{} & Gene.symbol &     logFC \\\\\n",
       "\\midrule\n",
       "0 &      ZNF616 & -4.244691 \\\\\n",
       "1 &     DEFB125 &  3.974393 \\\\\n",
       "3 &      SNAP23 &  3.337636 \\\\\n",
       "4 &      PHLDB2 &  3.192559 \\\\\n",
       "5 &   LOC389895 & -4.296850 \\\\\n",
       "\\bottomrule\n",
       "\\end{tabular}\n",
       "}\\end{center}"
      ],
      "text/plain": [
       "  Gene.symbol     logFC\n",
       "0      ZNF616 -4.244691\n",
       "1     DEFB125  3.974393\n",
       "3      SNAP23  3.337636\n",
       "4      PHLDB2  3.192559\n",
       "5   LOC389895 -4.296850"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.read_csv(data_path)\n",
    "df = df.loc[df['Gene.symbol'].notnull(), target_columns]\n",
    "df.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Data Summary"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "data = {k: v for _, k, v in df.itertuples()}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "On the left, the number of shared elements in the knowledge assembly and differential gene data set are counted.\n",
    "\n",
    "On the right, a histogram of the log-fold-changes shows that the data are normally distributed, as expected for differential gene expression data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "hgnc_names = pbt.summary.get_names_by_namespace(graph, 'HGNC')\n",
    "df_names = set(df['Gene.symbol'])\n",
    "overlapping_hgnc_names = hgnc_names & df_names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"208pt\" version=\"1.1\" viewBox=\"0 0 713 208\" width=\"713pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 208.8 \n",
       "L 713.821707 208.8 \n",
       "L 713.821707 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path clip-path=\"url(#pdc54370fad)\" d=\"M 117.53042 108.459939 \n",
       "C 116.86855 104.741616 116.535682 100.972184 116.535682 97.195413 \n",
       "C 116.535682 93.418643 116.86855 89.64921 117.53042 85.930888 \n",
       "C 114.761346 89.028344 113.229346 93.040656 113.229346 97.195413 \n",
       "C 113.229346 101.350171 114.761346 105.362482 117.53042 108.459939 \n",
       "\" style=\"fill:#ff0000;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#pdc54370fad)\" d=\"M 117.53042 85.930888 \n",
       "C 118.664697 84.662098 119.984137 83.571878 121.444032 82.697168 \n",
       "C 122.903927 81.822458 124.487602 81.173249 126.141398 80.771538 \n",
       "C 127.795194 80.369827 129.500219 80.220203 131.198703 80.327736 \n",
       "C 132.897188 80.435269 134.569729 80.79873 136.159658 81.405804 \n",
       "C 137.749586 82.012878 139.23874 82.856631 140.576664 83.908473 \n",
       "C 141.914587 84.960316 143.085996 86.208234 144.051202 87.609944 \n",
       "C 145.016407 89.011654 145.764382 90.551145 146.269784 92.176254 \n",
       "C 146.775187 93.801364 147.032243 95.493528 147.032243 97.195413 \n",
       "C 147.032243 98.897298 146.775187 100.589463 146.269784 102.214572 \n",
       "C 145.764382 103.839682 145.016407 105.379172 144.051202 106.780882 \n",
       "C 143.085996 108.182592 141.914587 109.43051 140.576664 110.482353 \n",
       "C 139.23874 111.534196 137.749586 112.377948 136.159658 112.985023 \n",
       "C 134.569729 113.592097 132.897188 113.955558 131.198703 114.06309 \n",
       "C 129.500219 114.170623 127.795194 114.020999 126.141398 113.619288 \n",
       "C 124.487602 113.217578 122.903927 112.568369 121.444032 111.693659 \n",
       "C 119.984137 110.818948 118.664697 109.728729 117.53042 108.459939 \n",
       "C 118.926235 116.301488 121.767466 123.815726 125.90907 130.619055 \n",
       "C 130.050675 137.422385 135.420957 143.397035 141.745768 148.238001 \n",
       "C 148.070578 153.078967 155.240428 156.702449 162.889301 158.923469 \n",
       "C 170.538174 161.144489 178.533659 161.9246 186.467594 161.223976 \n",
       "C 194.401528 160.523353 202.136566 158.354125 209.277839 154.827029 \n",
       "C 216.419112 151.299933 222.842997 146.476027 228.221559 140.601569 \n",
       "C 233.600121 134.727111 237.840251 127.903794 240.725493 120.479943 \n",
       "C 243.610735 113.056092 245.091144 105.160222 245.091144 97.195413 \n",
       "C 245.091144 89.230604 243.610735 81.334734 240.725493 73.910884 \n",
       "C 237.840251 66.487033 233.600121 59.663715 228.221559 53.789257 \n",
       "C 222.842997 47.914799 216.419112 43.090893 209.277839 39.563798 \n",
       "C 202.136566 36.036702 194.401528 33.867473 186.467594 33.16685 \n",
       "C 178.533659 32.466227 170.538174 33.246337 162.889301 35.467357 \n",
       "C 155.240428 37.688377 148.070578 41.311859 141.745768 46.152825 \n",
       "C 135.420957 50.993792 130.050675 56.968441 125.90907 63.771771 \n",
       "C 121.767466 70.575101 118.926235 78.089339 117.53042 85.930888 \n",
       "\" style=\"fill:#008000;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#pdc54370fad)\" d=\"M 117.53042 108.459939 \n",
       "C 118.664697 109.728729 119.984137 110.818948 121.444032 111.693659 \n",
       "C 122.903927 112.568369 124.487602 113.217578 126.141398 113.619288 \n",
       "C 127.795194 114.020999 129.500219 114.170623 131.198703 114.06309 \n",
       "C 132.897188 113.955558 134.569729 113.592097 136.159658 112.985023 \n",
       "C 137.749586 112.377948 139.23874 111.534196 140.576664 110.482353 \n",
       "C 141.914587 109.43051 143.085996 108.182592 144.051202 106.780882 \n",
       "C 145.016407 105.379172 145.764382 103.839682 146.269784 102.214572 \n",
       "C 146.775187 100.589463 147.032243 98.897298 147.032243 97.195413 \n",
       "C 147.032243 95.493528 146.775187 93.801364 146.269784 92.176254 \n",
       "C 145.764382 90.551145 145.016407 89.011654 144.051202 87.609944 \n",
       "C 143.085996 86.208234 141.914587 84.960316 140.576664 83.908473 \n",
       "C 139.23874 82.856631 137.749586 82.012878 136.159658 81.405804 \n",
       "C 134.569729 80.79873 132.897188 80.435269 131.198703 80.327736 \n",
       "C 129.500219 80.220203 127.795194 80.369827 126.141398 80.771538 \n",
       "C 124.487602 81.173249 122.903927 81.822458 121.444032 82.697168 \n",
       "C 119.984137 83.571878 118.664697 84.662098 117.53042 85.930888 \n",
       "C 116.86855 89.64921 116.535682 93.418643 116.535682 97.195413 \n",
       "C 116.535682 100.972184 116.86855 104.741616 117.53042 108.459939 \n",
       "\" style=\"fill:#b25900;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_1\">\n",
       "    <!-- 88 -->\n",
       "    <defs>\n",
       "     <path d=\"M 27.203125 35.375 \n",
       "C 33.5 38.546875 39.90625 43.328125 39.90625 50.984375 \n",
       "C 39.90625 60.015625 31.09375 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 58.734375 5.90625 49.6875 \n",
       "C 5.90625 47.203125 6.5 42.921875 10.40625 39.140625 \n",
       "C 11.40625 38.15625 15.59375 35.171875 18.296875 33.28125 \n",
       "C 13.796875 30.984375 3.296875 25.53125 3.296875 14.59375 \n",
       "C 3.296875 4.359375 13.09375 -2 22.796875 -2 \n",
       "C 33.5 -2 42.5 5.65625 42.5 15.78125 \n",
       "C 42.5 24.84375 36.40625 29.015625 32.40625 31.703125 \n",
       "z\n",
       "M 14.09375 44.09375 \n",
       "C 13.296875 44.59375 9.296875 47.671875 9.296875 52.359375 \n",
       "C 9.296875 58.421875 15.59375 63 22.796875 63 \n",
       "C 30.703125 63 36.5 57.4375 36.5 50.96875 \n",
       "C 36.5 41.703125 26.09375 36.421875 25.59375 36.421875 \n",
       "C 25.5 36.421875 25.40625 36.421875 24.59375 37.03125 \n",
       "z\n",
       "M 32.5 23.734375 \n",
       "C 34 22.640625 38.796875 19.375 38.796875 13.296875 \n",
       "C 38.796875 5.953125 31.40625 0.390625 23 0.390625 \n",
       "C 13.90625 0.390625 7 6.84375 7 14.6875 \n",
       "C 7 22.546875 13.09375 29.09375 20 32.1875 \n",
       "z\n",
       "\" id=\"CMR17-56\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(109.901179 99.686073)scale(0.1 -0.1)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "     <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_2\">\n",
       "    <!-- 20741 -->\n",
       "    <defs>\n",
       "     <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "     <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "z\n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "z\n",
       "\" id=\"CMR17-48\"/>\n",
       "     <path d=\"M 45.09375 61.578125 \n",
       "L 45.09375 63.765625 \n",
       "L 21.59375 63.765625 \n",
       "C 9.90625 63.765625 9.703125 65.078125 9.296875 67 \n",
       "L 7.5 67 \n",
       "L 4.59375 48.34375 \n",
       "L 6.40625 48.34375 \n",
       "C 6.703125 50.234375 7.40625 55.3125 8.59375 57.203125 \n",
       "C 9.203125 58 16.59375 58 18.296875 58 \n",
       "L 40.296875 58 \n",
       "L 29.203125 41.578125 \n",
       "C 21.796875 30.53125 16.703125 15.609375 16.703125 2.578125 \n",
       "C 16.703125 1.390625 16.703125 -2 20.296875 -2 \n",
       "C 23.90625 -2 23.90625 1.390625 23.90625 2.671875 \n",
       "L 23.90625 7.265625 \n",
       "C 23.90625 23.5625 26.703125 34.3125 31.296875 41.1875 \n",
       "z\n",
       "\" id=\"CMR17-55\"/>\n",
       "     <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "     <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(183.608355 99.686073)scale(0.1 -0.1)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "     <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "     <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "     <use transform=\"translate(182.761908 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_3\">\n",
       "    <!-- 1446 -->\n",
       "    <defs>\n",
       "     <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "z\n",
       "\" id=\"CMR17-54\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(121.821292 99.686073)scale(0.1 -0.1)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "     <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "     <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "     <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_4\">\n",
       "    <!-- Alzheimer's Disease KA -->\n",
       "    <defs>\n",
       "     <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "     <path d=\"M 15.296875 68.6875 \n",
       "L 3.40625 67.59375 \n",
       "L 3.40625 65 \n",
       "C 9.296875 65 10.203125 64.40625 10.203125 59.640625 \n",
       "L 10.203125 6.65625 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.203125 0 12.703125 0 \n",
       "C 15.296875 0 19.703125 0 22.09375 -0.203125 \n",
       "L 22.09375 2.390625 \n",
       "C 15.703125 2.390625 15.296875 2.78125 15.296875 6.65625 \n",
       "z\n",
       "\" id=\"CMR17-108\"/>\n",
       "     <path d=\"M 36.09375 40.703125 \n",
       "C 36.90625 41.71875 36.90625 41.90625 36.90625 42.109375 \n",
       "C 36.90625 43 36.5 43 34.703125 43 \n",
       "L 4.296875 43 \n",
       "L 3.296875 27.5 \n",
       "L 5.09375 27.5 \n",
       "C 5.796875 37.203125 6.90625 41.40625 18.40625 41.40625 \n",
       "L 30.59375 41.40625 \n",
       "L 2 1.5 \n",
       "C 2 0 2.09375 0 4.203125 0 \n",
       "L 35.796875 0 \n",
       "L 37.203125 18 \n",
       "L 35.40625 18 \n",
       "C 34.59375 6.90625 33.296875 1.796875 21.09375 1.796875 \n",
       "L 8.296875 1.796875 \n",
       "z\n",
       "\" id=\"CMR17-122\"/>\n",
       "     <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.5 43.59375 16.5 37 15.296875 33.703125 \n",
       "L 15.203125 33.703125 \n",
       "L 15.203125 68.6875 \n",
       "L 3.296875 67.59375 \n",
       "L 3.296875 65 \n",
       "C 9.203125 65 10.09375 64.40625 10.09375 59.640625 \n",
       "L 10.09375 6.65625 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-104\"/>\n",
       "     <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "     <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "z\n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "     <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "     <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "     <path d=\"M 16 62.890625 \n",
       "C 16.203125 61.796875 16.296875 61.390625 16.296875 59.6875 \n",
       "C 16.296875 53.359375 13.796875 47.53125 9.90625 43.40625 \n",
       "C 9.5 43 9.296875 42.703125 9.296875 42.296875 \n",
       "C 9.296875 41.890625 9.703125 41.5 10.09375 41.5 \n",
       "C 10.796875 41.5 17.90625 48.515625 17.90625 59.546875 \n",
       "C 17.90625 61.75 17.703125 69.15625 12.5 69.15625 \n",
       "C 10.296875 69.15625 8.40625 67.671875 8.40625 65.078125 \n",
       "C 8.40625 62.484375 10.296875 61 12.5 61 \n",
       "C 13.40625 61 15.09375 61.296875 16 62.890625 \n",
       "z\n",
       "\" id=\"CMR17-39\"/>\n",
       "     <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "     <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 37 0 \n",
       "C 53.09375 0 66 14.734375 66 33.453125 \n",
       "C 66 52.28125 53.296875 68 37 68 \n",
       "z\n",
       "M 23.796875 2.59375 \n",
       "C 19.796875 2.59375 19.59375 3.078125 19.59375 6.46875 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 35 65.40625 \n",
       "C 43.296875 65.40625 58.59375 60.53125 58.59375 33.453125 \n",
       "C 58.59375 19.40625 54.5 13.734375 52.59375 11.046875 \n",
       "C 50.203125 7.96875 44.5 2.59375 35 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-68\"/>\n",
       "     <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "     <path d=\"M 36.90625 41.796875 \n",
       "L 54.203125 59.09375 \n",
       "C 57.796875 62.59375 61.59375 65.40625 67.203125 65.40625 \n",
       "L 67.203125 68 \n",
       "C 65.203125 68 62.703125 68 60.703125 68 \n",
       "C 58 68 53.296875 68 50.703125 68.09375 \n",
       "L 50.703125 65.5 \n",
       "C 53.296875 65.3125 53.703125 63.515625 53.703125 62.796875 \n",
       "C 53.703125 62.109375 53.296875 61.015625 52.90625 60.421875 \n",
       "L 19.703125 27.15625 \n",
       "L 19.703125 60.9375 \n",
       "C 19.703125 65.40625 20.796875 65.40625 27.703125 65.40625 \n",
       "L 27.703125 68 \n",
       "C 24.796875 68 19.5 68 16.40625 68 \n",
       "C 13.296875 68 8 68 5.09375 68 \n",
       "L 5.09375 65.40625 \n",
       "C 12 65.40625 13.09375 65.40625 13.09375 60.921875 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.296875 0 16.40625 0 \n",
       "C 19.5 0 24.796875 0 27.703125 0 \n",
       "L 27.703125 2.5 \n",
       "C 20.796875 2.5 19.703125 2.5 19.703125 6.984375 \n",
       "L 19.703125 24.671875 \n",
       "L 32.5 37.34375 \n",
       "L 51.40625 8.390625 \n",
       "C 52 7.390625 52.703125 6.09375 52.703125 5.09375 \n",
       "C 52.703125 2.5 49.90625 2.5 48.59375 2.5 \n",
       "L 48.59375 0 \n",
       "C 51.796875 0 56.40625 0 59.703125 0 \n",
       "C 62.09375 0 66.203125 0 68.5 -0.09375 \n",
       "L 68.5 2.5 \n",
       "C 63.703125 2.5 62.203125 3.09375 59.203125 7.703125 \n",
       "z\n",
       "\" id=\"CMR17-75\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(7.2 126.458)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "     <use transform=\"translate(93.943488 0)scale(0.996264)\" xlink:href=\"#CMR17-122\"/>\n",
       "     <use transform=\"translate(134.429149 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "     <use transform=\"translate(185.324519 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(225.810179 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(250.681237 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(327.600827 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(368.086488 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(403.367256 0)scale(0.996264)\" xlink:href=\"#CMR17-39\"/>\n",
       "     <use transform=\"translate(428.238314 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(494.115618 0)scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "     <use transform=\"translate(564.509174 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(589.380232 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(625.18157 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(665.667231 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(711.357708 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(747.159045 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(817.720672 0)scale(0.996264)\" xlink:href=\"#CMR17-75\"/>\n",
       "     <use transform=\"translate(889.395449 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_5\">\n",
       "    <!-- E-GEOD-5281 -->\n",
       "    <defs>\n",
       "     <path d=\"M 60.703125 25.234375 \n",
       "L 58.90625 25.234375 \n",
       "C 56.703125 10.5625 55.40625 2.59375 37.796875 2.59375 \n",
       "L 23.90625 2.59375 \n",
       "C 19.90625 2.59375 19.703125 3.078125 19.703125 6.46875 \n",
       "L 19.703125 34 \n",
       "L 29.09375 34 \n",
       "C 38.5 34 39.40625 30.625 39.40625 22.34375 \n",
       "L 41.203125 22.34375 \n",
       "L 41.203125 48.140625 \n",
       "L 39.40625 48.140625 \n",
       "C 39.40625 39.953125 38.5 36.59375 29.09375 36.59375 \n",
       "L 19.703125 36.59375 \n",
       "L 19.703125 61.140625 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 37.59375 65 \n",
       "C 53.09375 65 54.90625 58.890625 56.296875 45.765625 \n",
       "L 58.09375 45.765625 \n",
       "L 55.703125 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 7.046875 \n",
       "C 13.09375 2.59375 12 2.59375 5.09375 2.59375 \n",
       "L 5.09375 0 \n",
       "L 57.09375 0 \n",
       "z\n",
       "\" id=\"CMR17-69\"/>\n",
       "     <path d=\"M 25.5 19 \n",
       "L 25.5 23.375 \n",
       "L 0.40625 23.375 \n",
       "L 0.40625 19 \n",
       "z\n",
       "\" id=\"CMR17-45\"/>\n",
       "     <path d=\"M 62.09375 19.265625 \n",
       "C 62.09375 23.015625 62.40625 23.515625 68.59375 23.515625 \n",
       "L 68.59375 26.09375 \n",
       "C 66.203125 26 60.796875 26 58.09375 26 \n",
       "C 55.296875 26 47.5 26 43.703125 26 \n",
       "L 43.703125 23.515625 \n",
       "L 46.703125 23.515625 \n",
       "C 55.296875 23.515625 55.5 22.40625 55.5 18.765625 \n",
       "L 55.5 12.09375 \n",
       "C 55.5 0.984375 42.5 0.984375 39.40625 0.984375 \n",
       "C 29.09375 0.984375 12.203125 7.984375 12.203125 33.75 \n",
       "C 12.203125 59.5 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.65625 59.90625 42.828125 \n",
       "C 60.09375 41.625 60.09375 41.421875 61 41.421875 \n",
       "C 62.09375 41.421875 62.09375 41.625 62.09375 43.421875 \n",
       "L 62.09375 67 \n",
       "C 62.09375 68.78125 62 68.890625 61.40625 68.890625 \n",
       "C 61 68.890625 60.90625 68.78125 60.203125 67.59375 \n",
       "L 55.796875 59.234375 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 13.203125 19.796875 -2 37.40625 -2 \n",
       "C 45.5 -2 53.40625 0.875 56.5 6.515625 \n",
       "C 58.203125 2.46875 61 -0.3125 61.5 -0.3125 \n",
       "C 62 -0.3125 62.09375 -0.203125 62.09375 1.5625 \n",
       "z\n",
       "\" id=\"CMR17-71\"/>\n",
       "     <path d=\"M 67.296875 33.359375 \n",
       "C 67.296875 53.703125 52.59375 69 36 69 \n",
       "C 19 69 4.59375 53.5 4.59375 33.359375 \n",
       "C 4.59375 13.203125 19.203125 -2 35.90625 -2 \n",
       "C 53 -2 67.296875 13.5 67.296875 33.359375 \n",
       "z\n",
       "M 36 -0.203125 \n",
       "C 24.59375 -0.203125 12.203125 11.109375 12.203125 34.640625 \n",
       "C 12.203125 58.0625 25.796875 67.203125 35.90625 67.203125 \n",
       "C 46.5 67.203125 59.703125 57.78125 59.703125 34.640625 \n",
       "C 59.703125 10.71875 47 -0.203125 36 -0.203125 \n",
       "z\n",
       "\" id=\"CMR17-79\"/>\n",
       "     <path d=\"M 11.40625 57.6875 \n",
       "C 12.40625 57.28125 16.5 56 20.703125 56 \n",
       "C 30 56 35.09375 61.109375 38 64.0625 \n",
       "C 38 64.953125 38 65.5 37.40625 65.5 \n",
       "C 37.296875 65.5 37.09375 65.5 36.296875 65.046875 \n",
       "C 32.796875 63.40625 28.703125 62.078125 23.703125 62.078125 \n",
       "C 20.703125 62.078125 16.203125 62.484375 11.296875 64.671875 \n",
       "C 10.203125 65.171875 10 65.171875 9.90625 65.171875 \n",
       "C 9.40625 65.171875 9.296875 65.0625 9.296875 63.078125 \n",
       "L 9.296875 34.421875 \n",
       "C 9.296875 32.640625 9.296875 32.140625 10.296875 32.140625 \n",
       "C 10.796875 32.140625 11 32.34375 11.5 33.03125 \n",
       "C 14.703125 37.515625 19.09375 39.40625 24.09375 39.40625 \n",
       "C 27.59375 39.40625 35.09375 37.21875 35.09375 20.140625 \n",
       "C 35.09375 16.953125 35.09375 11.171875 32.09375 6.578125 \n",
       "C 29.59375 2.484375 25.703125 0.390625 21.40625 0.390625 \n",
       "C 14.796875 0.390625 8.09375 5 6.296875 12.71875 \n",
       "C 6.703125 12.609375 7.5 12.421875 7.90625 12.421875 \n",
       "C 9.203125 12.421875 11.703125 13.125 11.703125 16.21875 \n",
       "C 11.703125 18.90625 9.796875 20 7.90625 20 \n",
       "C 5.59375 20 4.09375 18.59375 4.09375 15.796875 \n",
       "C 4.09375 7.09375 11 -2 21.59375 -2 \n",
       "C 31.90625 -2 41.703125 6.875 41.703125 19.75 \n",
       "C 41.703125 31.71875 33.90625 41 24.203125 41 \n",
       "C 19.09375 41 14.796875 39.109375 11.40625 35.53125 \n",
       "z\n",
       "\" id=\"CMR17-53\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(180.813413 173.834283)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-69\"/>\n",
       "     <use transform=\"translate(62.586301 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(92.662267 0)scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "     <use transform=\"translate(165.017613 0)scale(0.996264)\" xlink:href=\"#CMR17-69\"/>\n",
       "     <use transform=\"translate(227.603913 0)scale(0.996264)\" xlink:href=\"#CMR17-79\"/>\n",
       "     <use transform=\"translate(299.318702 0)scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "     <use transform=\"translate(369.712258 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(399.788224 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "     <use transform=\"translate(445.478701 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "     <use transform=\"translate(491.169178 0)scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "     <use transform=\"translate(536.859655 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_6\">\n",
       "    <!-- Gene Overlap -->\n",
       "    <defs>\n",
       "     <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "     <path d=\"M 39.09375 33.796875 \n",
       "C 40 36 41.59375 40 47.40625 40 \n",
       "L 47.40625 42.59375 \n",
       "C 45.5 42.59375 43 42.59375 41.09375 42.59375 \n",
       "C 39 42.59375 35.703125 42.59375 33.703125 42.6875 \n",
       "L 33.703125 40.09375 \n",
       "C 36.203125 39.890625 37.59375 38.703125 37.59375 36.109375 \n",
       "C 37.59375 35.203125 37.59375 35 37 33.609375 \n",
       "L 26 5.890625 \n",
       "L 14.09375 35.71875 \n",
       "C 13.5 37.015625 13.5 37.203125 13.5 37.609375 \n",
       "C 13.5 40 16.796875 40 18.296875 40 \n",
       "L 18.296875 42.6875 \n",
       "C 15.90625 42.59375 11.5 42.59375 9 42.59375 \n",
       "C 6.296875 42.59375 3.203125 42.59375 1 42.6875 \n",
       "L 1 40 \n",
       "C 4.796875 40 6.90625 40 8.09375 36.90625 \n",
       "L 22.59375 0.6875 \n",
       "C 23.203125 -0.796875 23.296875 -1 24.203125 -1 \n",
       "C 25.09375 -1 25.203125 -0.796875 25.796875 0.6875 \n",
       "z\n",
       "\" id=\"CMR17-118\"/>\n",
       "     <path d=\"M 22 -16.609375 \n",
       "C 15.59375 -16.609375 15.203125 -16.203125 15.203125 -12.328125 \n",
       "L 15.203125 6.28125 \n",
       "C 18.09375 2.09375 22.296875 -0.59375 27.59375 -0.59375 \n",
       "C 37.59375 -0.59375 47.703125 8.171875 47.703125 21.6875 \n",
       "C 47.703125 34.234375 39 43.78125 28.703125 43.78125 \n",
       "C 22.703125 43.78125 17.796875 40.5 15 36.53125 \n",
       "L 15 43.78125 \n",
       "L 3.09375 42.6875 \n",
       "L 3.09375 40.109375 \n",
       "C 9 40.109375 9.90625 39.5 9.90625 34.734375 \n",
       "L 9.90625 -12.328125 \n",
       "C 9.90625 -16.109375 9.5 -16.609375 3.09375 -16.609375 \n",
       "L 3.09375 -19.1875 \n",
       "C 5.5 -19 9.90625 -19 12.5 -19 \n",
       "C 15.09375 -19 19.59375 -19 22 -19.1875 \n",
       "z\n",
       "M 15.203125 31.4375 \n",
       "C 15.203125 32.84375 15.203125 34.921875 19.09375 38.609375 \n",
       "C 19.59375 39.015625 23 42 28 42 \n",
       "C 35.296875 42 41.296875 32.9375 41.296875 21.59375 \n",
       "C 41.296875 10.25 34.90625 1 27 1 \n",
       "C 23.40625 1 19.5 2.6875 16.5 7.375 \n",
       "C 15.203125 9.5625 15.203125 10.15625 15.203125 11.75 \n",
       "z\n",
       "\" id=\"CMR17-112\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(144.00217 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "     <use transform=\"translate(72.355346 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(112.841006 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(163.736376 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(234.298003 0)scale(0.996264)\" xlink:href=\"#CMR17-79\"/>\n",
       "     <use transform=\"translate(306.012791 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "     <use transform=\"translate(351.703253 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(392.188914 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(427.469682 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "     <use transform=\"translate(452.34074 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(498.031217 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_2\">\n",
       "   <g id=\"patch_5\">\n",
       "    <path d=\"M 364.190755 172.888647 \n",
       "L 703.121707 172.888647 \n",
       "L 703.121707 21.50218 \n",
       "L 364.190755 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#pb6a07e4947)\" d=\"M 379.596708 172.888647 \n",
       "L 410.408612 172.888647 \n",
       "L 410.408612 172.862226 \n",
       "L 379.596708 172.862226 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#pb6a07e4947)\" d=\"M 410.408612 172.888647 \n",
       "L 441.220517 172.888647 \n",
       "L 441.220517 172.690492 \n",
       "L 410.408612 172.690492 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#pb6a07e4947)\" d=\"M 441.220517 172.888647 \n",
       "L 472.032422 172.888647 \n",
       "L 472.032422 169.533222 \n",
       "L 441.220517 169.533222 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#pb6a07e4947)\" d=\"M 472.032422 172.888647 \n",
       "L 502.844326 172.888647 \n",
       "L 502.844326 154.407389 \n",
       "L 472.032422 154.407389 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#pb6a07e4947)\" d=\"M 502.844326 172.888647 \n",
       "L 533.656231 172.888647 \n",
       "L 533.656231 76.862727 \n",
       "L 502.844326 76.862727 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#pb6a07e4947)\" d=\"M 533.656231 172.888647 \n",
       "L 564.468136 172.888647 \n",
       "L 564.468136 28.711059 \n",
       "L 533.656231 28.711059 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#pb6a07e4947)\" d=\"M 564.468136 172.888647 \n",
       "L 595.28004 172.888647 \n",
       "L 595.28004 146.732185 \n",
       "L 564.468136 146.732185 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path clip-path=\"url(#pb6a07e4947)\" d=\"M 595.28004 172.888647 \n",
       "L 626.091945 172.888647 \n",
       "L 626.091945 168.793443 \n",
       "L 595.28004 168.793443 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path clip-path=\"url(#pb6a07e4947)\" d=\"M 626.091945 172.888647 \n",
       "L 656.90385 172.888647 \n",
       "L 656.90385 172.386654 \n",
       "L 626.091945 172.386654 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path clip-path=\"url(#pb6a07e4947)\" d=\"M 656.90385 172.888647 \n",
       "L 687.715754 172.888647 \n",
       "L 687.715754 172.809385 \n",
       "L 656.90385 172.809385 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"me4aa1079df\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"390.527839\" xlink:href=\"#me4aa1079df\" y=\"172.888647\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- $-4$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 65.90625 23 \n",
       "C 67.59375 23 69.40625 23 69.40625 25 \n",
       "C 69.40625 27 67.59375 27 65.90625 27 \n",
       "L 11.796875 27 \n",
       "C 10.09375 27 8.296875 27 8.296875 25 \n",
       "C 8.296875 23 10.09375 23 11.796875 23 \n",
       "z\n",
       "\" id=\"CMSY10-0\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(384.162799 186.807145)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"464.175418\" xlink:href=\"#me4aa1079df\" y=\"172.888647\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- $-2$ -->\n",
       "      <g transform=\"translate(457.810379 186.807145)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"537.822998\" xlink:href=\"#me4aa1079df\" y=\"172.888647\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(535.33233 186.807145)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"611.470578\" xlink:href=\"#me4aa1079df\" y=\"172.888647\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- $2$ -->\n",
       "      <g transform=\"translate(608.97991 186.807145)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"685.118157\" xlink:href=\"#me4aa1079df\" y=\"172.888647\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- $4$ -->\n",
       "      <g transform=\"translate(682.62749 186.807145)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_12\">\n",
       "     <!-- Log-Fold-Change -->\n",
       "     <defs>\n",
       "      <path d=\"M 54.203125 25.234375 \n",
       "L 52.40625 25.234375 \n",
       "C 51.59375 15.359375 50.296875 2.59375 33.203125 2.59375 \n",
       "L 23.90625 2.59375 \n",
       "C 19.90625 2.59375 19.703125 3.078125 19.703125 6.46875 \n",
       "L 19.703125 60.90625 \n",
       "C 19.703125 64.515625 19.90625 65.5 27.5 65.5 \n",
       "L 29.703125 65.5 \n",
       "L 29.703125 68 \n",
       "C 25.59375 68 20.796875 68 16.703125 68 \n",
       "C 13.59375 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 7.0625 \n",
       "C 13.09375 2.59375 12 2.59375 5.09375 2.59375 \n",
       "L 5.09375 0 \n",
       "L 51.796875 0 \n",
       "z\n",
       "\" id=\"CMR17-76\"/>\n",
       "      <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "z\n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "z\n",
       "\" id=\"CMR17-111\"/>\n",
       "      <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "z\n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "z\n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "z\n",
       "\" id=\"CMR17-103\"/>\n",
       "      <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      <path d=\"M 29.203125 67.59375 \n",
       "L 29.203125 65 \n",
       "C 35.09375 65 36 64.40625 36 59.6875 \n",
       "L 36 36.640625 \n",
       "C 35.59375 37.140625 31.5 43.59375 23.40625 43.59375 \n",
       "C 13.203125 43.59375 3.296875 34.5 3.296875 21.3125 \n",
       "C 3.296875 8.203125 12.59375 -1 22.40625 -1 \n",
       "C 30.90625 -1 35.296875 5.265625 35.796875 5.9375 \n",
       "L 35.796875 -0.984375 \n",
       "L 47.90625 -0.984375 \n",
       "L 47.90625 2 \n",
       "C 42 2 41.09375 2.59375 41.09375 7.390625 \n",
       "L 41.09375 68.6875 \n",
       "z\n",
       "M 35.796875 11.796875 \n",
       "C 35.796875 8.796875 34 6.09375 31.703125 4.09375 \n",
       "C 28.296875 1.09375 24.90625 0.59375 23 0.59375 \n",
       "C 20.09375 0.59375 9.703125 2.09375 9.703125 21.1875 \n",
       "C 9.703125 40.8125 21.296875 42 23.90625 42 \n",
       "C 28.5 42 32.203125 39.40625 34.5 35.796875 \n",
       "C 35.796875 33.703125 35.796875 33.40625 35.796875 31.609375 \n",
       "z\n",
       "\" id=\"CMR17-100\"/>\n",
       "      <path d=\"M 62.09375 67.015625 \n",
       "C 62.09375 68.796875 62 68.90625 61.40625 68.90625 \n",
       "C 61 68.90625 60.90625 68.796875 60.203125 67.609375 \n",
       "L 55.796875 59.25 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 12.90625 20.09375 -2 37.296875 -2 \n",
       "C 52.59375 -2 62.09375 11.40625 62.09375 22.828125 \n",
       "C 62.09375 23.8125 62.09375 24.21875 61.203125 24.21875 \n",
       "C 60.40625 24.21875 60.40625 23.921875 60.296875 23.03125 \n",
       "C 59.5 9.140625 49.5 0.59375 38.703125 0.59375 \n",
       "C 28.5 0.59375 12.203125 7.734375 12.203125 33.546875 \n",
       "C 12.203125 59.453125 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.6875 59.90625 42.890625 \n",
       "C 60.09375 41.6875 60.09375 41.484375 61 41.484375 \n",
       "C 62.09375 41.484375 62.09375 41.6875 62.09375 43.484375 \n",
       "z\n",
       "\" id=\"CMR17-67\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(496.434621 199.662822)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-76\"/>\n",
       "      <use transform=\"translate(57.381408 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(103.071885 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(148.762362 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "      <use transform=\"translate(178.838328 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(231.01492 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(276.705397 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "      <use transform=\"translate(301.576455 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "      <use transform=\"translate(352.471825 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "      <use transform=\"translate(382.547791 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "      <use transform=\"translate(449.057672 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(499.953042 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(545.643519 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      <use transform=\"translate(596.538889 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(642.229366 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"mf45fcd6021\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"364.190755\" xlink:href=\"#mf45fcd6021\" y=\"172.888647\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_13\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(352.20942 176.347896)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"364.190755\" xlink:href=\"#mf45fcd6021\" y=\"146.467978\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_14\">\n",
       "      <!-- $2000$ -->\n",
       "      <g transform=\"translate(337.265414 149.927227)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"364.190755\" xlink:href=\"#mf45fcd6021\" y=\"120.04731\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- $4000$ -->\n",
       "      <g transform=\"translate(337.265414 123.506559)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"364.190755\" xlink:href=\"#mf45fcd6021\" y=\"93.626641\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- $6000$ -->\n",
       "      <g transform=\"translate(337.265414 97.085891)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"364.190755\" xlink:href=\"#mf45fcd6021\" y=\"67.205973\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- $8000$ -->\n",
       "      <g transform=\"translate(337.265414 70.665222)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"364.190755\" xlink:href=\"#mf45fcd6021\" y=\"40.785305\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_18\">\n",
       "      <!-- $10000$ -->\n",
       "      <g transform=\"translate(332.284079 44.244554)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(182.761908 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_19\">\n",
       "     <!-- Frequency -->\n",
       "     <defs>\n",
       "      <path d=\"M 41.09375 43.78125 \n",
       "L 39.5 43.78125 \n",
       "L 36.5 33.9375 \n",
       "C 34.203125 39.203125 30 43.78125 23.296875 43.78125 \n",
       "C 13.09375 43.78125 3.296875 34.734375 3.296875 21.609375 \n",
       "C 3.296875 8.5625 12.5 -0.59375 22.5 -0.59375 \n",
       "C 30.203125 -0.59375 34.5 5.28125 35.796875 7.46875 \n",
       "L 35.796875 -12.328125 \n",
       "C 35.796875 -16.109375 35.40625 -16.609375 29 -16.609375 \n",
       "L 29 -19.1875 \n",
       "C 31.40625 -19 35.796875 -19 38.40625 -19 \n",
       "C 41 -19 45.5 -19 47.90625 -19.1875 \n",
       "L 47.90625 -16.609375 \n",
       "C 41.5 -16.609375 41.09375 -16.203125 41.09375 -12.328125 \n",
       "z\n",
       "M 36 13.84375 \n",
       "C 36 9.5625 32.40625 5.578125 32.09375 5.28125 \n",
       "C 29 1.796875 25.40625 1 23 1 \n",
       "C 15.5 1 9.703125 10.453125 9.703125 21.484375 \n",
       "C 9.703125 33.140625 16.296875 42 24 42 \n",
       "C 32 42 36 32.9375 36 27.765625 \n",
       "z\n",
       "\" id=\"CMR17-113\"/>\n",
       "      <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "      <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "z\n",
       "\" id=\"CMR17-99\"/>\n",
       "      <path d=\"M 39 34.328125 \n",
       "C 41.5 40.40625 46 40.40625 47.40625 40.40625 \n",
       "L 47.40625 43 \n",
       "C 45.5 43 43 43 41.09375 43 \n",
       "C 39 43 35.703125 43 33.703125 43.09375 \n",
       "L 33.703125 40.5 \n",
       "C 37.5 40.203125 37.59375 37.40625 37.59375 36.609375 \n",
       "C 37.59375 35.625 37.40625 35.125 36.90625 33.921875 \n",
       "L 26 7.015625 \n",
       "L 14.09375 36.03125 \n",
       "C 13.59375 37.21875 13.59375 37.90625 13.59375 38.015625 \n",
       "C 13.59375 40.203125 15.90625 40.40625 18.296875 40.40625 \n",
       "L 18.296875 43.09375 \n",
       "C 15.90625 43 11.59375 43 9.09375 43 \n",
       "C 6.40625 43 3.203125 43 1 43.09375 \n",
       "L 1 40.40625 \n",
       "C 6.40625 40.40625 7 39.90625 8.296875 36.71875 \n",
       "L 23.203125 0.234375 \n",
       "C 18.703125 -11.53125 16.09375 -18.40625 9.796875 -18.40625 \n",
       "C 8.703125 -18.40625 6.203125 -18.109375 4.40625 -16.3125 \n",
       "C 6.703125 -16.109375 7.703125 -14.71875 7.703125 -13.015625 \n",
       "C 7.703125 -11.328125 6.5 -9.828125 4.5 -9.828125 \n",
       "C 2.296875 -9.828125 1.203125 -11.328125 1.203125 -13.125 \n",
       "C 1.203125 -17.109375 5.296875 -20 9.796875 -20 \n",
       "C 15.59375 -20 19.203125 -14.40625 21.203125 -9.4375 \n",
       "z\n",
       "\" id=\"CMR17-121\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(326.346901 119.417675)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(52.176591 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(87.457359 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(127.943019 0)scale(0.996264)\" xlink:href=\"#CMR17-113\"/>\n",
       "      <use transform=\"translate(176.235935 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(227.131305 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(267.616966 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      <use transform=\"translate(318.512335 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(358.997996 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 364.190755 172.888647 \n",
       "L 364.190755 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_17\">\n",
       "    <path d=\"M 703.121707 172.888647 \n",
       "L 703.121707 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_18\">\n",
       "    <path d=\"M 364.190755 172.888647 \n",
       "L 703.121707 172.888647 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_19\">\n",
       "    <path d=\"M 364.190755 21.50218 \n",
       "L 703.121707 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_20\">\n",
       "    <!-- Distribution of Log-Fold-Changes in E-GEOD-5281 -->\n",
       "    <defs>\n",
       "     <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "     <path d=\"M 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 -0.203125 \n",
       "L 11.703125 -0.203125 \n",
       "L 14.59375 6.890625 \n",
       "C 17.5 2.203125 21.796875 -1 27.59375 -1 \n",
       "C 37.59375 -1 47.703125 7.8125 47.703125 21.40625 \n",
       "C 47.703125 34.1875 38.703125 43.59375 28.59375 43.59375 \n",
       "C 22.40625 43.59375 18 40.296875 15 36.40625 \n",
       "z\n",
       "M 15.203125 31.3125 \n",
       "C 15.203125 33.09375 15.203125 33.40625 16.59375 35.5 \n",
       "C 20.5 41.515625 26.09375 42 28 42 \n",
       "C 31 42 41.296875 40.390625 41.296875 21.5 \n",
       "C 41.296875 1.6875 29.5 0.59375 27.09375 0.59375 \n",
       "C 24 0.59375 19.5 1.78125 16.296875 7.59375 \n",
       "C 15.203125 9.5 15.203125 9.703125 15.203125 11.484375 \n",
       "z\n",
       "\" id=\"CMR17-98\"/>\n",
       "     <path d=\"M 15.296875 40 \n",
       "L 26.703125 40 \n",
       "L 26.703125 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 54.59375 \n",
       "C 15.09375 62.84375 19.296875 68 23.59375 68 \n",
       "C 24.90625 68 26.40625 67.609375 27.40625 66.90625 \n",
       "C 26.59375 66.703125 24.5 66.015625 24.5 63.546875 \n",
       "C 24.5 60.953125 26.5 60.15625 27.90625 60.15625 \n",
       "C 29.296875 60.15625 31.296875 60.953125 31.296875 63.546875 \n",
       "C 31.296875 67.3125 27.703125 69.59375 23.703125 69.59375 \n",
       "C 18 69.59375 10.203125 64.9375 10.203125 54.390625 \n",
       "L 10.203125 42.578125 \n",
       "L 2.203125 42.578125 \n",
       "L 2.203125 40 \n",
       "L 10.203125 40 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.40625 0 13 0 \n",
       "C 15.90625 0 21 0 23.703125 -0.203125 \n",
       "L 23.703125 2.390625 \n",
       "C 16.5 2.390625 15.296875 2.390625 15.296875 6.859375 \n",
       "z\n",
       "\" id=\"CMR17-102\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(401.864987 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "     <use transform=\"translate(70.393556 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(95.264614 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(131.065952 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(166.34672 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(201.627488 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(226.498546 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "     <use transform=\"translate(277.393916 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "     <use transform=\"translate(328.289286 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(363.570053 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(388.441112 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(434.131589 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(515.102925 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(560.793402 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(618.342881 0)scale(0.996264)\" xlink:href=\"#CMR17-76\"/>\n",
       "     <use transform=\"translate(675.724289 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(721.414766 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(767.105243 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(797.18121 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "     <use transform=\"translate(849.357801 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(895.048278 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "     <use transform=\"translate(919.919336 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "     <use transform=\"translate(970.814706 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(1000.890672 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "     <use transform=\"translate(1067.400553 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "     <use transform=\"translate(1118.295923 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(1163.9864 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(1214.88177 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(1260.572247 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(1301.057907 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(1366.935211 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(1391.80627 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(1472.777606 0)scale(0.996264)\" xlink:href=\"#CMR17-69\"/>\n",
       "     <use transform=\"translate(1535.363906 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(1565.439873 0)scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "     <use transform=\"translate(1637.795219 0)scale(0.996264)\" xlink:href=\"#CMR17-69\"/>\n",
       "     <use transform=\"translate(1700.381519 0)scale(0.996264)\" xlink:href=\"#CMR17-79\"/>\n",
       "     <use transform=\"translate(1772.096308 0)scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "     <use transform=\"translate(1842.489863 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(1872.56583 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "     <use transform=\"translate(1918.256307 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "     <use transform=\"translate(1963.946784 0)scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "     <use transform=\"translate(2009.637261 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"pdc54370fad\">\n",
       "   <rect height=\"151.386467\" width=\"154.692803\" x=\"101.813843\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pb6a07e4947\">\n",
       "   <rect height=\"151.386467\" width=\"338.930951\" x=\"364.190755\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x11adcf7b8>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "fix, (rax, lax) = plt.subplots(1, 2, figsize=(10, 3))\n",
    "\n",
    "lax.set_title('Distribution of Log-Fold-Changes in E-GEOD-5281')\n",
    "lax.set_xlabel('Log-Fold-Change')\n",
    "lax.set_ylabel('Frequency')\n",
    "lax.hist(list(data.values()))\n",
    "\n",
    "rax.set_title('Gene Overlap')\n",
    "venn2([hgnc_names, df_names], set_labels=[\"Alzheimer's Disease KA\", 'E-GEOD-5281'], ax=rax)\n",
    "plt.tight_layout()\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Analysis"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Data Integration\n",
    "\n",
    "Finally, the differential gene expression data are ovelayed on the BEL graph with [pbt.integration.overlay_type_data](http://pybel-tools.readthedocs.io/en/latest/integration.html#pybel_tools.integration.overlay_type_data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "label = 'weight'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pbt.integration.overlay_type_data(graph, data, label, GENE, 'HGNC', overwrite=False, impute=0)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Subgraph Generation\n",
    "\n",
    "A subgraph is induced around an example biological process, Inflammatory Response, by selecting all of the source nodes of upstream causal edges, where this process is the target node. These networks are then enriched by expandind around all of those nodes' upstream edges as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "inflamation_node = BIOPROCESS, 'GOBP', 'inflammatory response'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Iteration 1/2\n",
    "\n",
    "All of the nodes that have upstream causal relations to Inflammatory Response are added."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nodes: 40\n",
      "Edges: 65\n",
      "Citations: 24\n",
      "Authors: 164\n",
      "Network density: 0.041666666666666664\n",
      "Components: 1\n",
      "Average degree: 1.625\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"bqvxdaujygremoqt\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.7.3-dev\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Composite\", \"cname\": \"composite()\", \"id\": [\"Composite\", [\"Protein\", \"HGNC\", \"CASP1\"], [\"Protein\", \"HGNC\", \"SCAF11\"]]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"description\": \"This gene encodes a cell surface receptor and transmembrane precursor protein that is cleaved by secretases to form a number of peptides. Some of these peptides are secreted and can bind to the acetyltransferase complex APBB1/TIP60 to promote transcriptional activation, while others form the protein basis of the amyloid plaques found in the brains of patients with Alzheimer disease. In addition, two of the peptides are antimicrobial peptides, having been shown to have bacteriocidal and antifungal activities. Mutations in this gene have been implicated in autosomal dominant Alzheimer disease and cerebroarterial amyloidosis (cerebral amyloid angiopathy). Multiple transcript variants encoding several different isoforms have been found for this gene. [provided by RefSeq, Aug 2014]\", \"label\": \"amyloid beta precursor protein\", \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"leukotriene\", \"cname\": \"leukotriene\", \"id\": [\"Abundance\", \"CHEBI\", \"leukotriene\"]}, {\"function\": \"Abundance\", \"namespace\": \"ADO\", \"name\": \"neuroprotectin\", \"cname\": \"neuroprotectin\", \"id\": [\"Abundance\", \"ADO\", \"neuroprotectin\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Gliosis\", \"cname\": \"Gliosis\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"]}, {\"function\": \"Protein\", \"namespace\": \"ADO\", \"name\": \"Amyloid Beta\", \"cname\": \"Amyloid Beta\", \"id\": [\"Protein\", \"ADO\", \"Amyloid Beta\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"melatonin\", \"cname\": \"melatonin\", \"id\": [\"Abundance\", \"CHEBI\", \"melatonin\"]}, {\"function\": \"Abundance\", \"namespace\": \"MESHCS\", \"name\": \"Neurofibrillary Tangles\", \"cname\": \"Neurofibrillary Tangles\", \"id\": [\"Abundance\", \"MESHCS\", \"Neurofibrillary Tangles\"]}, {\"function\": \"Protein\", \"namespace\": \"SFAM\", \"name\": \"MMP Family\", \"cname\": \"MMP Family\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"]}, {\"function\": \"Complex\", \"cname\": \"complex()\", \"id\": [\"Complex\", [\"Protein\", \"ADO\", \"Amyloid Beta\"], [\"Protein\", \"MGI\", \"Tlr2\"]]}, {\"function\": \"Abundance\", \"namespace\": \"BRCO\", \"name\": \"Astrocyte\", \"cname\": \"Astrocyte\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"noradrenaline\", \"cname\": \"noradrenaline\", \"id\": [\"Abundance\", \"CHEBI\", \"noradrenaline\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"description\": \"The protein encoded by this gene is a serine-threonine kinase belonging to the glycogen synthase kinase subfamily. It is a negative regulator of glucose homeostasis and is involved in energy metabolism, inflammation, ER-stress, mitochondrial dysfunction, and apoptotic pathways. Defects in this gene have been associated with Parkinson disease and Alzheimer disease. [provided by RefSeq, Aug 2017]\", \"label\": \"glycogen synthase kinase 3 beta\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CX3CR1\", \"description\": \"Fractalkine is a transmembrane protein and chemokine involved in the adhesion and migration of leukocytes. The protein encoded by this gene is a receptor for fractalkine. The encoded protein also is a coreceptor for HIV-1, and some variations in this gene lead to increased susceptibility to HIV-1 infection and rapid progression to AIDS. Four transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Jan 2010]\", \"label\": \"C-X3-C motif chemokine receptor 1\", \"weight\": -0.81994598, \"cname\": \"CX3CR1\", \"id\": [\"Gene\", \"HGNC\", \"CX3CR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TREM2\", \"description\": \"This gene encodes a membrane protein that forms a receptor signaling complex with the TYRO protein tyrosine kinase binding protein. The encoded protein functions in immune response and may be involved in chronic inflammation by triggering the production of constitutive inflammatory cytokines. Defects in this gene are a cause of polycystic lipomembranous osteodysplasia with sclerosing leukoencephalopathy (PLOSL). Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Nov 2012]\", \"label\": \"triggering receptor expressed on myeloid cells 2\", \"weight\": -0.40134828, \"cname\": \"TREM2\", \"id\": [\"Gene\", \"HGNC\", \"TREM2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CFH\", \"description\": \"This gene is a member of the Regulator of Complement Activation (RCA) gene cluster and encodes a protein with twenty short consensus repeat (SCR) domains. This protein is secreted into the bloodstream and has an essential role in the regulation of complement activation, restricting this innate defense mechanism to microbial infections. Mutations in this gene have been associated with hemolytic-uremic syndrome (HUS) and chronic hypocomplementemic nephropathy. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Oct 2011]\", \"label\": \"complement factor H\", \"weight\": 1.86557178, \"cname\": \"CFH\", \"id\": [\"Gene\", \"HGNC\", \"CFH\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PTGS2\", \"description\": \"Prostaglandin-endoperoxide synthase (PTGS), also known as cyclooxygenase, is the key enzyme in prostaglandin biosynthesis, and acts both as a dioxygenase and as a peroxidase. There are two isozymes of PTGS: a constitutive PTGS1 and an inducible PTGS2, which differ in their regulation of expression and tissue distribution. This gene encodes the inducible isozyme. It is regulated by specific stimulatory events, suggesting that it is responsible for the prostanoid biosynthesis involved in inflammation and mitogenesis. [provided by RefSeq, Feb 2009]\", \"label\": \"prostaglandin-endoperoxide synthase 2\", \"weight\": -1.07023648, \"cname\": \"PTGS2\", \"id\": [\"Gene\", \"HGNC\", \"PTGS2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IRAK1\", \"description\": \"This gene encodes the interleukin-1 receptor-associated kinase 1, one of two putative serine/threonine kinases that become associated with the interleukin-1 receptor (IL1R) upon stimulation. This gene is partially responsible for IL1-induced upregulation of the transcription factor NF-kappa B. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 1 receptor associated kinase 1\", \"weight\": 0.72985984, \"cname\": \"IRAK1\", \"id\": [\"Gene\", \"HGNC\", \"IRAK1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IRAK2\", \"description\": \"IRAK2 encodes the interleukin-1 receptor-associated kinase 2, one of two putative serine/threonine kinases that become associated with the interleukin-1 receptor (IL1R) upon stimulation. IRAK2 is reported to participate in the IL1-induced upregulation of NF-kappaB. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 1 receptor associated kinase 2\", \"weight\": -0.26455035, \"cname\": \"IRAK2\", \"id\": [\"Gene\", \"HGNC\", \"IRAK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"description\": \"The advanced glycosylation end product (AGE) receptor encoded by this gene is a member of the immunoglobulin superfamily of cell surface receptors. It is a multiligand receptor, and besides AGE, interacts with other molecules implicated in homeostasis, development, and inflammation, and certain diseases, such as diabetes and Alzheimer's disease. Many alternatively spliced transcript variants encoding different isoforms, as well as non-protein-coding variants, have been described for this gene (PMID:18089847). [provided by RefSeq, May 2011]\", \"label\": \"advanced glycosylation end-product specific receptor\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PTX3\", \"description\": \"This gene encodes a member of the pentraxin protein family. The expression of this protein is induced by inflammatory cytokines in response to inflammatory stimuli in several mesenchymal and epithelial cell types, particularly endothelial cells and mononuclear phagocytes. The protein promotes fibrocyte differentiation and is involved in regulating inflammation and complement activation. It also plays a role in angiogenesis and tissue remodeling. The protein serves as a biomarker for several inflammatory conditions. [provided by RefSeq, Jun 2016]\", \"label\": \"pentraxin 3\", \"weight\": -1.00547039, \"cname\": \"PTX3\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL23A\", \"description\": \"This gene encodes a subunit of the heterodimeric cytokine interleukin 23 (IL23). IL23 is composed of this protein and the p40 subunit of interleukin 12 (IL12B). The receptor of IL23 is formed by the beta 1 subunit of IL12 (IL12RB1) and an IL23 specific subunit, IL23R. Both IL23 and IL12 can activate the transcription activator STAT4, and stimulate the production of interferon-gamma (IFNG). In contrast to IL12, which acts mainly on naive CD4(+) T cells, IL23 preferentially acts on memory CD4(+) T cells. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 23 subunit alpha\", \"weight\": -0.28486748, \"cname\": \"IL23A\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"description\": null, \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"description\": \"The protein encoded by this gene is a scavenger receptor that is expressed in endothelial cells. It regulates the uptake of chemically modified low density lipoproteins, including acetylated low density lipoprotein (Ac-LDL), and it may be involved in atherogenesis. This gene is regulated by the transcription factors ZNF444/EZF-2 and SP1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2013]\", \"label\": \"scavenger receptor class F member 1\", \"weight\": 0.18225965, \"cname\": \"SCARF1\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"description\": \"The protein encoded by this gene is a plasma membrane receptor for high density lipoprotein cholesterol (HDL). The encoded protein mediates cholesterol transfer to and from HDL. In addition, this protein is a receptor for hepatitis C virus glycoprotein E2. Two transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Mar 2011]\", \"label\": \"scavenger receptor class B member 1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"description\": null, \"label\": \"scavenger receptor class A member 5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"description\": \"This gene encodes a subunit of interleukin 12, a cytokine that acts on T and natural killer cells, and has a broad array of biological activities. Interleukin 12 is a disulfide-linked heterodimer composed of the 40 kD cytokine receptor like subunit encoded by this gene, and a 35 kD subunit encoded by IL12A. This cytokine is expressed by activated macrophages that serve as an essential inducer of Th1 cells development. This cytokine has been found to be important for sustaining a sufficient number of memory/effector Th1 cells to mediate long-term protection to an intracellular pathogen. Overexpression of this gene was observed in the central nervous system of patients with multiple sclerosis (MS), suggesting a role of this cytokine in the pathogenesis of the disease. The promoter polymorphism of this gene has been reported to be associated with the severity of atopic and non-atopic asthma in children. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 12B\", \"weight\": -1.61550531, \"cname\": \"IL12B\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. They recognize pathogen-associated molecular patterns that are expressed on infectious agents, and mediate the production of cytokines necessary for the development of effective immunity. The various TLRs exhibit different patterns of expression. This receptor has been implicated in signal transduction events induced by lipopolysaccharide (LPS) found in most gram-negative bacteria. Mutations in this gene have been associated with differences in LPS responsiveness. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012]\", \"label\": \"toll like receptor 4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR2\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. This protein is a cell-surface protein that can form heterodimers with other TLR family members to recognize conserved molecules derived from microorganisms known as pathogen-associated molecular patterns (PAMPs). Activation of TLRs by PAMPs leads to an up-regulation of signaling pathways to modulate the host's inflammatory response. This gene is also thought to promote apoptosis in response to bacterial lipoproteins. This gene has been implicated in the pathogenesis of several autoimmune diseases. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]\", \"label\": \"toll like receptor 2\", \"weight\": 0.13520554, \"cname\": \"TLR2\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PLA2G4A\", \"description\": \"This gene encodes a member of the cytosolic phospholipase A2 group IV family. The enzyme catalyzes the hydrolysis of membrane phospholipids to release arachidonic acid which is subsequently metabolized into eicosanoids. Eicosanoids, including prostaglandins and leukotrienes, are lipid-based cellular hormones that regulate hemodynamics, inflammatory responses, and other intracellular pathways. The hydrolysis reaction also produces lysophospholipids that are converted into platelet-activating factor. The enzyme is activated by increased intracellular Ca(2+) levels and phosphorylation, resulting in its translocation from the cytosol and nucleus to perinuclear membrane vesicles. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2015]\", \"label\": \"phospholipase A2 group IVA\", \"weight\": 0.52335304, \"cname\": \"PLA2G4A\", \"id\": [\"Gene\", \"HGNC\", \"PLA2G4A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"description\": \"The protein encoded by this gene is a serine/threonine protein kinase that is activated by autophosphorylation after binding to dsRNA. The activated form of the encoded protein can phosphorylate translation initiation factor EIF2S1, which in turn inhibits protein synthesis. This protein is also activated by manganese ions and heparin. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Oct 2011]\", \"label\": \"eukaryotic translation initiation factor 2 alpha kinase 2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CD14\", \"description\": \"The protein encoded by this gene is a surface antigen that is preferentially expressed on monocytes/macrophages. It cooperates with other proteins to mediate the innate immune response to bacterial lipopolysaccharide. Alternative splicing results in multiple transcript variants encoding the same protein. [provided by RefSeq, Mar 2010]\", \"label\": \"CD14 molecule\", \"weight\": -0.73001192, \"cname\": \"CD14\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR1\", \"description\": \"This gene encodes a G protein-coupled receptor of mammalian phagocytic cells that is a member of the G-protein coupled receptor 1 family. The protein mediates the response of phagocytic cells to invasion of the host by microorganisms and is important in host defense and inflammation.[provided by RefSeq, Jul 2010]\", \"label\": \"formyl peptide receptor 1\", \"weight\": -0.5889405, \"cname\": \"FPR1\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NOD2\", \"description\": \"This gene is a member of the Nod1/Apaf-1 family and encodes a protein with two caspase recruitment (CARD) domains and six leucine-rich repeats (LRRs). The protein is primarily expressed in the peripheral blood leukocytes. It plays a role in the immune response to intracellular bacterial lipopolysaccharides (LPS) by recognizing the muramyl dipeptide (MDP) derived from them and activating the NFKB protein. Mutations in this gene have been associated with Crohn disease and Blau syndrome. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Jun 2014]\", \"label\": \"nucleotide binding oligomerization domain containing 2\", \"weight\": -0.62084437, \"cname\": \"NOD2\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR2\", \"description\": null, \"label\": \"formyl peptide receptor 2\", \"weight\": 5.3e-05, \"cname\": \"FPR2\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PLA2G7\", \"description\": \"The protein encoded by this gene is a secreted enzyme that catalyzes the degradation of platelet-activating factor to biologically inactive products. Defects in this gene are a cause of platelet-activating factor acetylhydrolase deficiency. Two transcript variants encoding the same protein have been found for this gene.[provided by RefSeq, Dec 2009]\", \"label\": \"phospholipase A2 group VII\", \"weight\": -0.52445663, \"cname\": \"PLA2G7\", \"id\": [\"Gene\", \"HGNC\", \"PLA2G7\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MYD88\", \"description\": \"This gene encodes a cytosolic adapter protein that plays a central role in the innate and adaptive immune response. This protein functions as an essential signal transducer in the interleukin-1 and Toll-like receptor signaling pathways. These pathways regulate that activation of numerous proinflammatory genes. The encoded protein consists of an N-terminal death domain and a C-terminal Toll-interleukin1 receptor domain. Patients with defects in this gene have an increased susceptibility to pyogenic bacterial infections. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010]\", \"label\": \"myeloid differentiation primary response 88\", \"weight\": 0.981683, \"cname\": \"MYD88\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHI3L1\", \"description\": \"Chitinases catalyze the hydrolysis of chitin, which is an abundant glycopolymer found in insect exoskeletons and fungal cell walls. The glycoside hydrolase 18 family of chitinases includes eight human family members. This gene encodes a glycoprotein member of the glycosyl hydrolase 18 family. The protein lacks chitinase activity and is secreted by activated macrophages, chondrocytes, neutrophils and synovial cells. The protein is thought to play a role in the process of inflammation and tissue remodeling. [provided by RefSeq, Sep 2009]\", \"label\": \"chitinase 3 like 1\", \"weight\": 0.14672037, \"cname\": \"CHI3L1\", \"id\": [\"Gene\", \"HGNC\", \"CHI3L1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TREM1\", \"description\": \"This gene encodes a receptor belonging to the Ig superfamily that is expressed on myeloid cells. This protein amplifies neutrophil and monocyte-mediated inflammatory responses triggered by bacterial and fungal infections by stimulating release of pro-inflammatory chemokines and cytokines, as well as increased surface expression of cell activation markers. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.[provided by RefSeq, Jun 2011]\", \"label\": \"triggering receptor expressed on myeloid cells 1\", \"weight\": 1.13272121, \"cname\": \"TREM1\", \"id\": [\"Gene\", \"HGNC\", \"TREM1\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Results show that p53 has two binding sites located at the cathepsin D promoter gene and that cathepsin D participates in p53-dependent apoptosis. Cathepsin D showed augmented activity soon after it was released and that was accompanied by increased levels of p53 protein, a cathepsin D transcription factor [16]. Therefore, the mechanism responsible for increase in cathepsin D activity might be an effect of increased synthesis regulated by p53. Cathepsin B has also been implicated in the activation of the pro-inflammatory caspases-1 and -11, and the cleavage of Bcl-2 family member Bid which may lead to cytochrome c release from the mitochondria and subsequent caspase activation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta biochimica et biophysica Sinica\", \"reference\": \"19499146\", \"authors\": [\"Zhang L\", \"Sheng R\", \"Qin Z\"], \"date\": \"2009-06-01\"}, \"line\": 6732, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Results show that p53 has two binding sites located at the cathepsin D promoter gene and that cathepsin D participates in p53-dependent apoptosis. Cathepsin D showed augmented activity soon after it was released and that was accompanied by increased levels of p53 protein, a cathepsin D transcription factor [16]. Therefore, the mechanism responsible for increase in cathepsin D activity might be an effect of increased synthesis regulated by p53. Cathepsin B has also been implicated in the activation of the pro-inflammatory caspases-1 and -11, and the cleavage of Bcl-2 family member Bid which may lead to cytochrome c release from the mitochondria and subsequent caspase activation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta biochimica et biophysica Sinica\", \"reference\": \"19499146\", \"authors\": [\"Zhang L\", \"Sheng R\", \"Qin Z\"], \"date\": \"2009-06-01\"}, \"line\": 6732, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Caspase subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS &amp; neurological disorders drug targets\", \"reference\": \"24059322\", \"authors\": [\"Rashid S\", \"Kamal MA\", \"ul Ain Q\", \"Nawaz MS\", \"Greig NH\"], \"date\": \"2014-03-01\"}, \"line\": 21087, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces A beta peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"21246057\", \"authors\": [\"Zhao Y\", \"Calon F\", \"Julien C\", \"Winkler JW\", \"Petasis NA\", \"Lukiw WJ\", \"Bazan NG\"], \"date\": \"2011-01-05\"}, \"line\": 23560, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces A beta peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"21246057\", \"authors\": [\"Zhao Y\", \"Calon F\", \"Julien C\", \"Winkler JW\", \"Petasis NA\", \"Lukiw WJ\", \"Bazan NG\"], \"date\": \"2011-01-05\"}, \"line\": 23560, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces Abeta42 peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"21246057\", \"authors\": [\"Zhao Y\", \"Calon F\", \"Julien C\", \"Winkler JW\", \"Petasis NA\", \"Lukiw WJ\", \"Bazan NG\"], \"date\": \"2011-01-05\"}, \"line\": 24066, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces Abeta42 peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"21246057\", \"authors\": [\"Zhao Y\", \"Calon F\", \"Julien C\", \"Winkler JW\", \"Petasis NA\", \"Lukiw WJ\", \"Bazan NG\"], \"date\": \"2011-01-05\"}, \"line\": 24066, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces Abeta42 peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"21246057\", \"authors\": [\"Zhao Y\", \"Calon F\", \"Julien C\", \"Winkler JW\", \"Petasis NA\", \"Lukiw WJ\", \"Bazan NG\"], \"date\": \"2011-01-05\"}, \"line\": 24066, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37424, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1β could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptosis in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGFβ1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGFβ2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37434, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Recent studies showed that melatonin, an indoleamine secreted by the pineal gland, may play an important/ role in aging and AD as an antioxidant and neuroprotector. Melatonin decreases during aging and patients with AD have a/ more profound reduction in this hormone.Melatonin efficiently protects neuronal cells from Abeta-mediated toxicity via/ antioxidant and anti-amyloid properties: it not only inhibits Abeta generation, but also arrests the formation of amyloid/ fibrils by a structure-dependent interaction with Abeta. Our recent studies have demonstrated that melatonin efficiently/ attenuates Alzheimer-like tau hyperphosphorylation. Although the exact mechanism is still not fully understood, a direct/ regulatory influence of melatonin on the activities of protein kinases and protein phosphatases is proposed. Additionally,/ melatonin also plays a role in protecting cholinergic neurons and in anti-inflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta pharmacologica Sinica\", \"reference\": \"16364209\", \"authors\": [\"Wang JZ\", \"Wang ZF\"], \"date\": \"2006-01-01\"}, \"line\": 37504, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of cellular and molecular medicine\", \"reference\": \"18494933\", \"authors\": [\"Esposito G\", \"Scuderi C\", \"Lu J\", \"Savani C\", \"De Filippis D\", \"Iuvone T\", \"Steardo L Jr\", \"Sheen V\", \"Steardo L\"], \"date\": \"2008-06-01\"}, \"line\": 37629, \"annotations\": {}, \"source\": 8, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"By coexpressing TLR1 or TLR6 in TLR2-transgenic HEK293 cells or silencing tlrs genes in RAW264.7 macrophages, we observed that TLR2-mediated Abeta42-triggered inflammatory activation was enhanced by TLR1 and suppressed by TLR6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"22198949\", \"authors\": [\"Fassbender K\", \"Liu S\", \"Walter J\", \"Liu Y\", \"Heneka MT\", \"Kiliaan AJ\", \"Wolf L\", \"Penke B\", \"Menger MD\", \"Rübe CE\", \"Hao W\", \"Hartmann T\"], \"date\": \"2012-02-01\"}, \"line\": 39434, \"annotations\": {\"MeSHAnatomy\": \"Macrophages\", \"Subgraph\": \"Toll like receptor subgraph\"}, \"source\": 10, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"By coexpressing TLR1 or TLR6 in TLR2-transgenic HEK293 cells or silencing tlrs genes in RAW264.7 macrophages, we observed that TLR2-mediated Abeta42-triggered inflammatory activation was enhanced by TLR1 and suppressed by TLR6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"22198949\", \"authors\": [\"Fassbender K\", \"Liu S\", \"Walter J\", \"Liu Y\", \"Heneka MT\", \"Kiliaan AJ\", \"Wolf L\", \"Penke B\", \"Menger MD\", \"Rübe CE\", \"Hao W\", \"Hartmann T\"], \"date\": \"2012-02-01\"}, \"line\": 39434, \"annotations\": {\"MeSHAnatomy\": \"Macrophages\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our study demonstrated that TLR2 is a primary receptor for Abeta to trigger neuroinflammatory activation and suggested that inhibition of TLR2 in microglia could be beneficial in Alzheimer's disease pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"22198949\", \"authors\": [\"Fassbender K\", \"Liu S\", \"Walter J\", \"Liu Y\", \"Heneka MT\", \"Kiliaan AJ\", \"Wolf L\", \"Penke B\", \"Menger MD\", \"Rübe CE\", \"Hao W\", \"Hartmann T\"], \"date\": \"2012-02-01\"}, \"line\": 39447, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Toll like receptor subgraph\"}, \"source\": 10, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our study demonstrated that TLR2 is a primary receptor for Abeta to trigger neuroinflammatory activation and suggested that inhibition of TLR2 in microglia could be beneficial in Alzheimer's disease pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"22198949\", \"authors\": [\"Fassbender K\", \"Liu S\", \"Walter J\", \"Liu Y\", \"Heneka MT\", \"Kiliaan AJ\", \"Wolf L\", \"Penke B\", \"Menger MD\", \"Rübe CE\", \"Hao W\", \"Hartmann T\"], \"date\": \"2012-02-01\"}, \"line\": 39447, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20413894\", \"authors\": [\"Wang HM\", \"Zhao YX\", \"Zhang S\", \"Liu GD\", \"Kang WY\", \"Tang HD\", \"Ding JQ\", \"Chen SD\"], \"date\": \"2010-01-01\"}, \"line\": 41354, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Experimental evidence suggests that cortical noradrenaline (NA) depletion due to degeneration of the locus/ ceruleus (LC) - a pathological hallmark of AD - plays a permissive role in the development of inflammation in AD. Our/ results indicate for the first time that PPARgamma expression can be modulated by the cAMP signalling pathway, and/ suggest that the anti-inflammatory effects of NA on brain cells may be partly mediated by increasing PPARgamma levels./ Conversely, decreased NA due to LC cell death in AD may reduce endogenous PPARgamma expression and therefore potentiate/ neuroinflammatory processes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"12887689\", \"authors\": [\"Klotz L\", \"Sastre M\", \"Kreutz A\", \"Gavrilyuk V\", \"Klockgether T\", \"Feinstein DL\", \"Heneka MT\"], \"date\": \"2003-08-01\"}, \"line\": 41400, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 12, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Experimental evidence suggests that cortical noradrenaline (NA) depletion due to degeneration of the locus/ ceruleus (LC) - a pathological hallmark of AD - plays a permissive role in the development of inflammation in AD. Our/ results indicate for the first time that PPARgamma expression can be modulated by the cAMP signalling pathway, and/ suggest that the anti-inflammatory effects of NA on brain cells may be partly mediated by increasing PPARgamma levels./ Conversely, decreased NA due to LC cell death in AD may reduce endogenous PPARgamma expression and therefore potentiate/ neuroinflammatory processes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"12887689\", \"authors\": [\"Klotz L\", \"Sastre M\", \"Kreutz A\", \"Gavrilyuk V\", \"Klockgether T\", \"Feinstein DL\", \"Heneka MT\"], \"date\": \"2003-08-01\"}, \"line\": 41400, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 12, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 13, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 13, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" We found that bilateral microinjection of amyloid beta (Abeta)1-40 fibrils into the hippocampal CA1 area of resulted in significant upregulation of CX3CR1 messenger RNA (mRNA) and protein expression (via increasing histone H3 acetylation in the Cx3cr1 promoter region), synaptic dysfunction, and cognitive impairment,\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"23855980\", \"authors\": [\"Naguib M\", \"Bie B\", \"Xu JJ\", \"Brown DL\", \"Yang H\", \"Wu J\"], \"date\": \"2013-12-01\"}, \"line\": 44274, \"annotations\": {\"MeSHAnatomy\": \"Hippocampus\", \"Encode_Feature_Types\": \"Promoter\"}, \"source\": 14, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Recently, a novel variant in the gene encoding the triggering receptor expressed on myeloid cells 2 (TREM2) has been identified that has refocused the spotlight back onto inflammation as a major contributing factor in AD. Variants in TREM2 triple one's risk of developing late-onset AD. TREM2 is expressed on microglial cells, the resident macrophages in the CNS, and functions to stimulate phagocytosis on one hand and to suppress cytokine production and inflammation on the other hand. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The New England journal of medicine\", \"reference\": \"23150934\", \"authors\": [\"Rademakers R\", \"Cruchaga C\", \"Guerreiro R\", \"Williams J\", \"Pocock J\", \"Lashley T\", \"Hardy J\", \"Hazrati L\", \"Amouyel P\", \"Kauwe JS\", \"Younkin S\", \"Goate A\", \"Powell J\", \"Singleton A\", \"Carrasquillo M\", \"Bras J\", \"Wojtas A\", \"Majounie E\", \"Sassi C\", \"Morgan K\", \"Collinge J\", \"Rogaeva E\", \"St George-Hyslop P\", \"Lambert JC\", \"Alzheimer Genetic Analysis Group.\"], \"date\": \"2013-01-10\"}, \"line\": 44670, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \" Here we provide evidence in AD brains of a specific up-regulation of an NF-kappaB-sensitive miRNA-146a highly complementary to the 3'-untranslated region of complement factor H (CFH), an important repressor of the inflammatory response of the brain.Transfection of HN cells using an NF-kappaB-containing pre-miRNA-146a promoter-luciferase reporter construct in stressed HN cells showed significant up-regulation of luciferase activity that paralleled decreases in CFH gene expression. Up-regulation of miRNA-146a coupled to down-regulation of CFH was observed in AD brain and in interleukin-1beta, Abeta42, and/or oxidatively stressed human neural (HN) cells in primary culture.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18801740\", \"authors\": [\"Lukiw WJ\", \"Zhao Y\", \"Cui JG\"], \"date\": \"2008-11-14\"}, \"line\": 8977, \"annotations\": {\"Subgraph\": \"Complement system subgraph\"}, \"source\": 16, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38534, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Prostaglandin subgraph\"}, \"source\": 17, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38534, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9367, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9367, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 18, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9369, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 19, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9369, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 19, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37596, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 20, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37602, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 21, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37701, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 22, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37598, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 23, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1203, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 24, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1203, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 24, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 25, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 25, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37600, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 25, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1205, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 26, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1205, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 26, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37699, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 27, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37586, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 28, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37590, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 28, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37584, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 29, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37712, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 29, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38536, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 30, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science signaling\", \"reference\": \"23092889\", \"authors\": [\"Tang D\", \"Kang R\"], \"date\": \"2012-10-23\"}, \"line\": 37518, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 31, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37708, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 32, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37592, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 33, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37588, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 34, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37594, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 35, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Platelets are an important source of amyloid-ss (Ass) in the circulatory system and play an important pro-inflammatory role. Upon activation, they adhere to leukocytes and endothelial cells by means of adhesive proteins like P-selectin, platelet endothelial cell adhesion molecule-1 (PECAM) and intercellular adhesion molecule-1 and -2 (ICAM-1 and -2) and secrete inflammatory mediators (chemokines, interleukins). In addition, platelets contain important enzymes involved in inflammatory intermediary synthesis like phospholipase A(2) (PLA(2)) and cyclooxygenase-2 (COX-2), and recent reports demonstrated significant changes in platelet levels and activities in Alzheimer's disease. Thus, as platelets represent an important link between Ass deposition and inflammatory reactions especially at endothelial level, they can be considered a valuable cellular model to evaluate potential peripheral inflammatory biomarkers in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biogerontology\", \"reference\": \"20454929\", \"authors\": [\"Casoli T\", \"Di Stefano G\", \"Balietti M\", \"Solazzi M\", \"Giorgetti B\", \"Fattoretti P\"], \"date\": \"2010-10-01\"}, \"line\": 37683, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 36, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37710, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 37, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"There was a strong correlation between YKL-40 and markers of neurodegeneration (total tau and p-tau). In the whole sample, we found a negative correlation between YKL-40 and CTh in AD vulnerable areas in Abeta42+ subjects but not in Abeta42 participants. Our results suggest that YKL-40 could track the inflammatory processes associated to tau-related neurodegeneration in the presence of the AD pathophysiological process.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"25865441\", \"authors\": [\"Vidal-Piñeiro D\", \"Pegueroles J\", \"González-Suárez A\", \"González-Ortiz S\", \"Medrano S\", \"Montal V\", \"Anton-Aguirre S\", \"Carmona-Iragui M\", \"Sala I\", \"Fortea J\", \"Blesa R\", \"Morenas-Rodríguez E\", \"Sánchez-Saudinós M\", \"Sampedro F\", \"Sánchez-Juan P\", \"Rodríguez-Rodríguez E\", \"Pozueta A\", \"Bartrés-Faz D\", \"Vilaplana E\", \"Lleó A\", \"Alcolea D\", \"Clarimón J\"], \"date\": \"2015-06-01\"}, \"line\": 44310, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Recent patents on anti-infective drug discovery\", \"reference\": \"19149696\", \"authors\": [\"Molloy EJ\"], \"date\": \"2009-01-01\"}, \"line\": 45458, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\", \"Cell\": \"neutrophil\"}, \"source\": 39, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Recent patents on anti-infective drug discovery\", \"reference\": \"19149696\", \"authors\": [\"Molloy EJ\"], \"date\": \"2009-01-01\"}, \"line\": 45458, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\", \"Cell\": \"monocyte\"}, \"source\": 39, \"target\": 1, \"key\": 1}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "subgraph = pbt.mutation.get_upstream_causal_subgraph(graph, inflamation_node)\n",
    "\n",
    "pbt.summary.print_summary(subgraph)\n",
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Iteration 2/2\n",
    "\n",
    "All of the nodes upstream of the nodes within the subgraph are added as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nodes: 323\n",
      "Edges: 1357\n",
      "Citations: 413\n",
      "Authors: 2132\n",
      "Network density: 0.013047324192835028\n",
      "Components: 1\n",
      "Average degree: 4.201238390092879\n"
     ]
    }
   ],
   "source": [
    "pbt.mutation.expand_upstream_causal_subgraph(graph, subgraph)\n",
    "pbt.summary.print_summary(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Post-Processing\n",
    "\n",
    "Inconsistient edges are thrown away, literature bias is removed, and nodes on the periphery of the subgraph that are not annotated with experimental data are also removed, since they cannot contribute to the propogation of scores."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pbt.mutation.remove_inconsistent_edges(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pbt.mutation.collapse_consistent_edges(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pbt.generation.remove_unweighted_leaves(subgraph, label)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pbt.generation.remove_unweighted_sources(subgraph, label)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nodes: 176\n",
      "Edges: 948\n",
      "Citations: 323\n",
      "Authors: 1696\n",
      "Network density: 0.030779220779220778\n",
      "Components: 1\n",
      "Average degree: 5.386363636363637\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"mkudpqcofahtyrge\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.7.3-dev\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"description\": \"This gene encodes a cell surface receptor and transmembrane precursor protein that is cleaved by secretases to form a number of peptides. Some of these peptides are secreted and can bind to the acetyltransferase complex APBB1/TIP60 to promote transcriptional activation, while others form the protein basis of the amyloid plaques found in the brains of patients with Alzheimer disease. In addition, two of the peptides are antimicrobial peptides, having been shown to have bacteriocidal and antifungal activities. Mutations in this gene have been implicated in autosomal dominant Alzheimer disease and cerebroarterial amyloidosis (cerebral amyloid angiopathy). Multiple transcript variants encoding several different isoforms have been found for this gene. [provided by RefSeq, Aug 2014]\", \"label\": \"amyloid beta precursor protein\", \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"leukotriene\", \"cname\": \"leukotriene\", \"id\": [\"Abundance\", \"CHEBI\", \"leukotriene\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Gliosis\", \"cname\": \"Gliosis\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"]}, {\"function\": \"Protein\", \"namespace\": \"ADO\", \"name\": \"Amyloid Beta\", \"cname\": \"Amyloid Beta\", \"id\": [\"Protein\", \"ADO\", \"Amyloid Beta\"]}, {\"function\": \"Abundance\", \"namespace\": \"MESHCS\", \"name\": \"Neurofibrillary Tangles\", \"cname\": \"Neurofibrillary Tangles\", \"id\": [\"Abundance\", \"MESHCS\", \"Neurofibrillary Tangles\"]}, {\"function\": \"Protein\", \"namespace\": \"SFAM\", \"name\": \"MMP Family\", \"cname\": \"MMP Family\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"]}, {\"function\": \"Abundance\", \"namespace\": \"BRCO\", \"name\": \"Astrocyte\", \"cname\": \"Astrocyte\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"description\": \"The protein encoded by this gene is a serine-threonine kinase belonging to the glycogen synthase kinase subfamily. It is a negative regulator of glucose homeostasis and is involved in energy metabolism, inflammation, ER-stress, mitochondrial dysfunction, and apoptotic pathways. Defects in this gene have been associated with Parkinson disease and Alzheimer disease. [provided by RefSeq, Aug 2017]\", \"label\": \"glycogen synthase kinase 3 beta\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CX3CR1\", \"description\": \"Fractalkine is a transmembrane protein and chemokine involved in the adhesion and migration of leukocytes. The protein encoded by this gene is a receptor for fractalkine. The encoded protein also is a coreceptor for HIV-1, and some variations in this gene lead to increased susceptibility to HIV-1 infection and rapid progression to AIDS. Four transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Jan 2010]\", \"label\": \"C-X3-C motif chemokine receptor 1\", \"weight\": -0.81994598, \"cname\": \"CX3CR1\", \"id\": [\"Gene\", \"HGNC\", \"CX3CR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TREM2\", \"description\": \"This gene encodes a membrane protein that forms a receptor signaling complex with the TYRO protein tyrosine kinase binding protein. The encoded protein functions in immune response and may be involved in chronic inflammation by triggering the production of constitutive inflammatory cytokines. Defects in this gene are a cause of polycystic lipomembranous osteodysplasia with sclerosing leukoencephalopathy (PLOSL). Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Nov 2012]\", \"label\": \"triggering receptor expressed on myeloid cells 2\", \"weight\": -0.40134828, \"cname\": \"TREM2\", \"id\": [\"Gene\", \"HGNC\", \"TREM2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CFH\", \"description\": \"This gene is a member of the Regulator of Complement Activation (RCA) gene cluster and encodes a protein with twenty short consensus repeat (SCR) domains. This protein is secreted into the bloodstream and has an essential role in the regulation of complement activation, restricting this innate defense mechanism to microbial infections. Mutations in this gene have been associated with hemolytic-uremic syndrome (HUS) and chronic hypocomplementemic nephropathy. Alternate transcriptional splice variants, encoding different isoforms, have been characterized. [provided by RefSeq, Oct 2011]\", \"label\": \"complement factor H\", \"weight\": 1.86557178, \"cname\": \"CFH\", \"id\": [\"Gene\", \"HGNC\", \"CFH\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PTGS2\", \"description\": \"Prostaglandin-endoperoxide synthase (PTGS), also known as cyclooxygenase, is the key enzyme in prostaglandin biosynthesis, and acts both as a dioxygenase and as a peroxidase. There are two isozymes of PTGS: a constitutive PTGS1 and an inducible PTGS2, which differ in their regulation of expression and tissue distribution. This gene encodes the inducible isozyme. It is regulated by specific stimulatory events, suggesting that it is responsible for the prostanoid biosynthesis involved in inflammation and mitogenesis. [provided by RefSeq, Feb 2009]\", \"label\": \"prostaglandin-endoperoxide synthase 2\", \"weight\": -1.07023648, \"cname\": \"PTGS2\", \"id\": [\"Gene\", \"HGNC\", \"PTGS2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IRAK1\", \"description\": \"This gene encodes the interleukin-1 receptor-associated kinase 1, one of two putative serine/threonine kinases that become associated with the interleukin-1 receptor (IL1R) upon stimulation. This gene is partially responsible for IL1-induced upregulation of the transcription factor NF-kappa B. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 1 receptor associated kinase 1\", \"weight\": 0.72985984, \"cname\": \"IRAK1\", \"id\": [\"Gene\", \"HGNC\", \"IRAK1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IRAK2\", \"description\": \"IRAK2 encodes the interleukin-1 receptor-associated kinase 2, one of two putative serine/threonine kinases that become associated with the interleukin-1 receptor (IL1R) upon stimulation. IRAK2 is reported to participate in the IL1-induced upregulation of NF-kappaB. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 1 receptor associated kinase 2\", \"weight\": -0.26455035, \"cname\": \"IRAK2\", \"id\": [\"Gene\", \"HGNC\", \"IRAK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"description\": \"The advanced glycosylation end product (AGE) receptor encoded by this gene is a member of the immunoglobulin superfamily of cell surface receptors. It is a multiligand receptor, and besides AGE, interacts with other molecules implicated in homeostasis, development, and inflammation, and certain diseases, such as diabetes and Alzheimer's disease. Many alternatively spliced transcript variants encoding different isoforms, as well as non-protein-coding variants, have been described for this gene (PMID:18089847). [provided by RefSeq, May 2011]\", \"label\": \"advanced glycosylation end-product specific receptor\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PTX3\", \"description\": \"This gene encodes a member of the pentraxin protein family. The expression of this protein is induced by inflammatory cytokines in response to inflammatory stimuli in several mesenchymal and epithelial cell types, particularly endothelial cells and mononuclear phagocytes. The protein promotes fibrocyte differentiation and is involved in regulating inflammation and complement activation. It also plays a role in angiogenesis and tissue remodeling. The protein serves as a biomarker for several inflammatory conditions. [provided by RefSeq, Jun 2016]\", \"label\": \"pentraxin 3\", \"weight\": -1.00547039, \"cname\": \"PTX3\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL23A\", \"description\": \"This gene encodes a subunit of the heterodimeric cytokine interleukin 23 (IL23). IL23 is composed of this protein and the p40 subunit of interleukin 12 (IL12B). The receptor of IL23 is formed by the beta 1 subunit of IL12 (IL12RB1) and an IL23 specific subunit, IL23R. Both IL23 and IL12 can activate the transcription activator STAT4, and stimulate the production of interferon-gamma (IFNG). In contrast to IL12, which acts mainly on naive CD4(+) T cells, IL23 preferentially acts on memory CD4(+) T cells. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 23 subunit alpha\", \"weight\": -0.28486748, \"cname\": \"IL23A\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"description\": null, \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"description\": \"The protein encoded by this gene is a scavenger receptor that is expressed in endothelial cells. It regulates the uptake of chemically modified low density lipoproteins, including acetylated low density lipoprotein (Ac-LDL), and it may be involved in atherogenesis. This gene is regulated by the transcription factors ZNF444/EZF-2 and SP1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2013]\", \"label\": \"scavenger receptor class F member 1\", \"weight\": 0.18225965, \"cname\": \"SCARF1\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"description\": \"The protein encoded by this gene is a plasma membrane receptor for high density lipoprotein cholesterol (HDL). The encoded protein mediates cholesterol transfer to and from HDL. In addition, this protein is a receptor for hepatitis C virus glycoprotein E2. Two transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Mar 2011]\", \"label\": \"scavenger receptor class B member 1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"description\": null, \"label\": \"scavenger receptor class A member 5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"description\": \"This gene encodes a subunit of interleukin 12, a cytokine that acts on T and natural killer cells, and has a broad array of biological activities. Interleukin 12 is a disulfide-linked heterodimer composed of the 40 kD cytokine receptor like subunit encoded by this gene, and a 35 kD subunit encoded by IL12A. This cytokine is expressed by activated macrophages that serve as an essential inducer of Th1 cells development. This cytokine has been found to be important for sustaining a sufficient number of memory/effector Th1 cells to mediate long-term protection to an intracellular pathogen. Overexpression of this gene was observed in the central nervous system of patients with multiple sclerosis (MS), suggesting a role of this cytokine in the pathogenesis of the disease. The promoter polymorphism of this gene has been reported to be associated with the severity of atopic and non-atopic asthma in children. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 12B\", \"weight\": -1.61550531, \"cname\": \"IL12B\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. They recognize pathogen-associated molecular patterns that are expressed on infectious agents, and mediate the production of cytokines necessary for the development of effective immunity. The various TLRs exhibit different patterns of expression. This receptor has been implicated in signal transduction events induced by lipopolysaccharide (LPS) found in most gram-negative bacteria. Mutations in this gene have been associated with differences in LPS responsiveness. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012]\", \"label\": \"toll like receptor 4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR2\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. This protein is a cell-surface protein that can form heterodimers with other TLR family members to recognize conserved molecules derived from microorganisms known as pathogen-associated molecular patterns (PAMPs). Activation of TLRs by PAMPs leads to an up-regulation of signaling pathways to modulate the host's inflammatory response. This gene is also thought to promote apoptosis in response to bacterial lipoproteins. This gene has been implicated in the pathogenesis of several autoimmune diseases. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]\", \"label\": \"toll like receptor 2\", \"weight\": 0.13520554, \"cname\": \"TLR2\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PLA2G4A\", \"description\": \"This gene encodes a member of the cytosolic phospholipase A2 group IV family. The enzyme catalyzes the hydrolysis of membrane phospholipids to release arachidonic acid which is subsequently metabolized into eicosanoids. Eicosanoids, including prostaglandins and leukotrienes, are lipid-based cellular hormones that regulate hemodynamics, inflammatory responses, and other intracellular pathways. The hydrolysis reaction also produces lysophospholipids that are converted into platelet-activating factor. The enzyme is activated by increased intracellular Ca(2+) levels and phosphorylation, resulting in its translocation from the cytosol and nucleus to perinuclear membrane vesicles. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2015]\", \"label\": \"phospholipase A2 group IVA\", \"weight\": 0.52335304, \"cname\": \"PLA2G4A\", \"id\": [\"Gene\", \"HGNC\", \"PLA2G4A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"description\": \"The protein encoded by this gene is a serine/threonine protein kinase that is activated by autophosphorylation after binding to dsRNA. The activated form of the encoded protein can phosphorylate translation initiation factor EIF2S1, which in turn inhibits protein synthesis. This protein is also activated by manganese ions and heparin. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Oct 2011]\", \"label\": \"eukaryotic translation initiation factor 2 alpha kinase 2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CD14\", \"description\": \"The protein encoded by this gene is a surface antigen that is preferentially expressed on monocytes/macrophages. It cooperates with other proteins to mediate the innate immune response to bacterial lipopolysaccharide. Alternative splicing results in multiple transcript variants encoding the same protein. [provided by RefSeq, Mar 2010]\", \"label\": \"CD14 molecule\", \"weight\": -0.73001192, \"cname\": \"CD14\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR1\", \"description\": \"This gene encodes a G protein-coupled receptor of mammalian phagocytic cells that is a member of the G-protein coupled receptor 1 family. The protein mediates the response of phagocytic cells to invasion of the host by microorganisms and is important in host defense and inflammation.[provided by RefSeq, Jul 2010]\", \"label\": \"formyl peptide receptor 1\", \"weight\": -0.5889405, \"cname\": \"FPR1\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NOD2\", \"description\": \"This gene is a member of the Nod1/Apaf-1 family and encodes a protein with two caspase recruitment (CARD) domains and six leucine-rich repeats (LRRs). The protein is primarily expressed in the peripheral blood leukocytes. It plays a role in the immune response to intracellular bacterial lipopolysaccharides (LPS) by recognizing the muramyl dipeptide (MDP) derived from them and activating the NFKB protein. Mutations in this gene have been associated with Crohn disease and Blau syndrome. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Jun 2014]\", \"label\": \"nucleotide binding oligomerization domain containing 2\", \"weight\": -0.62084437, \"cname\": \"NOD2\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR2\", \"description\": null, \"label\": \"formyl peptide receptor 2\", \"weight\": 5.3e-05, \"cname\": \"FPR2\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PLA2G7\", \"description\": \"The protein encoded by this gene is a secreted enzyme that catalyzes the degradation of platelet-activating factor to biologically inactive products. Defects in this gene are a cause of platelet-activating factor acetylhydrolase deficiency. Two transcript variants encoding the same protein have been found for this gene.[provided by RefSeq, Dec 2009]\", \"label\": \"phospholipase A2 group VII\", \"weight\": -0.52445663, \"cname\": \"PLA2G7\", \"id\": [\"Gene\", \"HGNC\", \"PLA2G7\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MYD88\", \"description\": \"This gene encodes a cytosolic adapter protein that plays a central role in the innate and adaptive immune response. This protein functions as an essential signal transducer in the interleukin-1 and Toll-like receptor signaling pathways. These pathways regulate that activation of numerous proinflammatory genes. The encoded protein consists of an N-terminal death domain and a C-terminal Toll-interleukin1 receptor domain. Patients with defects in this gene have an increased susceptibility to pyogenic bacterial infections. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010]\", \"label\": \"myeloid differentiation primary response 88\", \"weight\": 0.981683, \"cname\": \"MYD88\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHI3L1\", \"description\": \"Chitinases catalyze the hydrolysis of chitin, which is an abundant glycopolymer found in insect exoskeletons and fungal cell walls. The glycoside hydrolase 18 family of chitinases includes eight human family members. This gene encodes a glycoprotein member of the glycosyl hydrolase 18 family. The protein lacks chitinase activity and is secreted by activated macrophages, chondrocytes, neutrophils and synovial cells. The protein is thought to play a role in the process of inflammation and tissue remodeling. [provided by RefSeq, Sep 2009]\", \"label\": \"chitinase 3 like 1\", \"weight\": 0.14672037, \"cname\": \"CHI3L1\", \"id\": [\"Gene\", \"HGNC\", \"CHI3L1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TREM1\", \"description\": \"This gene encodes a receptor belonging to the Ig superfamily that is expressed on myeloid cells. This protein amplifies neutrophil and monocyte-mediated inflammatory responses triggered by bacterial and fungal infections by stimulating release of pro-inflammatory chemokines and cytokines, as well as increased surface expression of cell activation markers. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.[provided by RefSeq, Jun 2011]\", \"label\": \"triggering receptor expressed on myeloid cells 1\", \"weight\": 1.13272121, \"cname\": \"TREM1\", \"id\": [\"Gene\", \"HGNC\", \"TREM1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"c.275341G>C\"}], \"description\": \"This gene encodes a cell surface receptor and transmembrane precursor protein that is cleaved by secretases to form a number of peptides. Some of these peptides are secreted and can bind to the acetyltransferase complex APBB1/TIP60 to promote transcriptional activation, while others form the protein basis of the amyloid plaques found in the brains of patients with Alzheimer disease. In addition, two of the peptides are antimicrobial peptides, having been shown to have bacteriocidal and antifungal activities. Mutations in this gene have been implicated in autosomal dominant Alzheimer disease and cerebroarterial amyloidosis (cerebral amyloid angiopathy). Multiple transcript variants encoding several different isoforms have been found for this gene. [provided by RefSeq, Aug 2014]\", \"label\": \"amyloid beta precursor protein\", \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, var(c.275341G>C))\", \"id\": [\"Gene\", \"HGNC\", \"APP\", [\"hgvs\", \"c.275341G>C\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"c.717G>C\"}], \"description\": \"This gene encodes a cell surface receptor and transmembrane precursor protein that is cleaved by secretases to form a number of peptides. Some of these peptides are secreted and can bind to the acetyltransferase complex APBB1/TIP60 to promote transcriptional activation, while others form the protein basis of the amyloid plaques found in the brains of patients with Alzheimer disease. In addition, two of the peptides are antimicrobial peptides, having been shown to have bacteriocidal and antifungal activities. Mutations in this gene have been implicated in autosomal dominant Alzheimer disease and cerebroarterial amyloidosis (cerebral amyloid angiopathy). Multiple transcript variants encoding several different isoforms have been found for this gene. [provided by RefSeq, Aug 2014]\", \"label\": \"amyloid beta precursor protein\", \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, var(c.717G>C))\", \"id\": [\"Gene\", \"HGNC\", \"APP\", [\"hgvs\", \"c.717G>C\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTSD\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"c.224C>T\"}], \"description\": \"This gene encodes a member of the A1 family of peptidases. The encoded preproprotein is proteolytically processed to generate multiple protein products. These products include the cathepsin D light and heavy chains, which heterodimerize to form the mature enzyme. This enzyme exhibits pepsin-like activity and plays a role in protein turnover and in the proteolytic activation of hormones and growth factors. Mutations in this gene play a causal role in neuronal ceroid lipofuscinosis-10 and may be involved in the pathogenesis of several other diseases, including breast cancer and possibly Alzheimer's disease. [provided by RefSeq, Nov 2015]\", \"label\": \"cathepsin D\", \"weight\": 0.88300839, \"cname\": \"g(HGNC:CTSD, var(c.224C>T))\", \"id\": [\"Gene\", \"HGNC\", \"CTSD\", [\"hgvs\", \"c.224C>T\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APOE\", \"description\": \"The protein encoded by this gene is a major apoprotein of the chylomicron. It binds to a specific liver and peripheral cell receptor, and is essential for the normal catabolism of triglyceride-rich lipoprotein constituents. This gene maps to chromosome 19 in a cluster with the related apolipoprotein C1 and C2 genes. Mutations in this gene result in familial dysbetalipoproteinemia, or type III hyperlipoproteinemia (HLP III), in which increased plasma cholesterol and triglycerides are the consequence of impaired clearance of chylomicron and VLDL remnants. [provided by RefSeq, Jun 2016]\", \"label\": \"apolipoprotein E\", \"weight\": 0.04406715, \"cname\": \"APOE\", \"id\": [\"Gene\", \"HGNC\", \"APOE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"description\": \"This gene encodes a cell surface receptor and transmembrane precursor protein that is cleaved by secretases to form a number of peptides. Some of these peptides are secreted and can bind to the acetyltransferase complex APBB1/TIP60 to promote transcriptional activation, while others form the protein basis of the amyloid plaques found in the brains of patients with Alzheimer disease. In addition, two of the peptides are antimicrobial peptides, having been shown to have bacteriocidal and antifungal activities. Mutations in this gene have been implicated in autosomal dominant Alzheimer disease and cerebroarterial amyloidosis (cerebral amyloid angiopathy). Multiple transcript variants encoding several different isoforms have been found for this gene. [provided by RefSeq, Aug 2014]\", \"label\": \"amyloid beta precursor protein\", \"weight\": -0.13701466, \"cname\": \"APP\", \"id\": [\"Gene\", \"HGNC\", \"APP\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSEN1\", \"description\": \"Alzheimer's disease (AD) patients with an inherited form of the disease carry mutations in the presenilin proteins (PSEN1; PSEN2) or in the amyloid precursor protein (APP). These disease-linked mutations result in increased production of the longer form of amyloid-beta (main component of amyloid deposits found in AD brains). Presenilins are postulated to regulate APP processing through their effects on gamma-secretase, an enzyme that cleaves APP. Also, it is thought that the presenilins are involved in the cleavage of the Notch receptor, such that they either directly regulate gamma-secretase activity or themselves are protease enzymes. Several alternatively spliced transcript variants encoding different isoforms have been identified for this gene, the full-length nature of only some have been determined. [provided by RefSeq, Aug 2008]\", \"label\": \"presenilin 1\", \"weight\": 0.00266298, \"cname\": \"PSEN1\", \"id\": [\"Gene\", \"HGNC\", \"PSEN1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSEN2\", \"description\": \"Alzheimer's disease (AD) patients with an inherited form of the disease carry mutations in the presenilin proteins (PSEN1 or PSEN2) or the amyloid precursor protein (APP). These disease-linked mutations result in increased production of the longer form of amyloid-beta (main component of amyloid deposits found in AD brains). Presenilins are postulated to regulate APP processing through their effects on gamma-secretase, an enzyme that cleaves APP. Also, it is thought that the presenilins are involved in the cleavage of the Notch receptor such that, they either directly regulate gamma-secretase activity, or themselves act are protease enzymes. Two alternatively spliced transcript variants encoding different isoforms of PSEN2 have been identified. [provided by RefSeq, Jul 2008]\", \"label\": \"presenilin 2\", \"weight\": -0.02429789, \"cname\": \"PSEN2\", \"id\": [\"Gene\", \"HGNC\", \"PSEN2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PPARG\", \"description\": \"This gene encodes a member of the peroxisome proliferator-activated receptor (PPAR) subfamily of nuclear receptors. PPARs form heterodimers with retinoid X receptors (RXRs) and these heterodimers regulate transcription of various genes. Three subtypes of PPARs are known: PPAR-alpha, PPAR-delta, and PPAR-gamma. The protein encoded by this gene is PPAR-gamma and is a regulator of adipocyte differentiation. Additionally, PPAR-gamma has been implicated in the pathology of numerous diseases including obesity, diabetes, atherosclerosis and cancer. Alternatively spliced transcript variants that encode different isoforms have been described. [provided by RefSeq, Jul 2008]\", \"label\": \"peroxisome proliferator activated receptor gamma\", \"weight\": -0.06808657, \"cname\": \"PPARG\", \"id\": [\"Gene\", \"HGNC\", \"PPARG\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EGR1\", \"description\": \"The protein encoded by this gene belongs to the EGR family of C2H2-type zinc-finger proteins. It is a nuclear protein and functions as a transcriptional regulator. The products of target genes it activates are required for differentitation and mitogenesis. Studies suggest this is a cancer suppressor gene. [provided by RefSeq, Dec 2014]\", \"label\": \"early growth response 1\", \"weight\": -0.31091638, \"cname\": \"EGR1\", \"id\": [\"Gene\", \"HGNC\", \"EGR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NAB2\", \"description\": \"This gene encodes a member of the family of NGFI-A binding (NAB) proteins, which function in the nucleus to repress transcription induced by some members of the EGR (early growth response) family of transactivators. NAB proteins can homo- or hetero-multimerize with other EGR or NAB proteins through a conserved N-terminal domain, and repress transcription through two partially redundant C-terminal domains. Transcriptional repression by the encoded protein is mediated in part by interactions with the nucleosome remodeling and deactylase (NuRD) complex. Alternatively spliced transcript variants have been described, but their biological validity has not been determined. [provided by RefSeq, Jul 2008]\", \"label\": \"NGFI-A binding protein 2\", \"weight\": 0.33700701, \"cname\": \"NAB2\", \"id\": [\"Gene\", \"HGNC\", \"NAB2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"KLF1\", \"description\": \"This gene encodes a hematopoietic-specific transcription factor that induces high-level expression of adult beta-globin and other erythroid genes. The zinc-finger protein binds to the DNA sequence CCACACCCT found in the beta hemoglobin promoter. Heterozygous loss-of-function mutations in this gene result in the dominant In(Lu) blood phenotype. [provided by RefSeq, Oct 2009]\", \"label\": \"Kruppel like factor 1\", \"weight\": -0.77844709, \"cname\": \"KLF1\", \"id\": [\"Gene\", \"HGNC\", \"KLF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTSD\", \"description\": \"This gene encodes a member of the A1 family of peptidases. The encoded preproprotein is proteolytically processed to generate multiple protein products. These products include the cathepsin D light and heavy chains, which heterodimerize to form the mature enzyme. This enzyme exhibits pepsin-like activity and plays a role in protein turnover and in the proteolytic activation of hormones and growth factors. Mutations in this gene play a causal role in neuronal ceroid lipofuscinosis-10 and may be involved in the pathogenesis of several other diseases, including breast cancer and possibly Alzheimer's disease. [provided by RefSeq, Nov 2015]\", \"label\": \"cathepsin D\", \"weight\": 0.88300839, \"cname\": \"CTSD\", \"id\": [\"Gene\", \"HGNC\", \"CTSD\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CR1\", \"description\": \"This gene is a member of the receptors of complement activation (RCA) family and is located in the 'cluster RCA' region of chromosome 1. The gene encodes a monomeric single-pass type I membrane glycoprotein found on erythrocytes, leukocytes, glomerular podocytes, and splenic follicular dendritic cells. The Knops blood group system is a system of antigens located on this protein. The protein mediates cellular binding to particles and immune complexes that have activated complement. Decreases in expression of this protein and/or mutations in its gene have been associated with gallbladder carcinomas, mesangiocapillary glomerulonephritis, systemic lupus erythematosus and sarcoidosis. Mutations in this gene have also been associated with a reduction in Plasmodium falciparum rosetting, conferring protection against severe malaria. Alternate allele-specific splice variants, encoding different isoforms, have been characterized. Additional allele specific isoforms, including a secreted form, have been described but have not been fully characterized. [provided by RefSeq, Jul 2008]\", \"label\": \"complement C3b/C4b receptor 1 (Knops blood group)\", \"weight\": 0.02399208, \"cname\": \"CR1\", \"id\": [\"Gene\", \"HGNC\", \"CR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CLU\", \"description\": \"The protein encoded by this gene is a secreted chaperone that can under some stress conditions also be found in the cell cytosol. It has been suggested to be involved in several basic biological events such as cell death, tumor progression, and neurodegenerative disorders. Alternate splicing results in both coding and non-coding variants.[provided by RefSeq, May 2011]\", \"label\": \"clusterin\", \"weight\": 0.4376103, \"cname\": \"CLU\", \"id\": [\"Gene\", \"HGNC\", \"CLU\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PICALM\", \"description\": \"This gene encodes a clathrin assembly protein, which recruits clathrin and adaptor protein complex 2 (AP2) to cell membranes at sites of coated-pit formation and clathrin-vesicle assembly. The protein may be required to determine the amount of membrane to be recycled, possibly by regulating the size of the clathrin cage. The protein is involved in AP2-dependent clathrin-mediated endocytosis at the neuromuscular junction. A chromosomal translocation t(10;11)(p13;q14) leading to the fusion of this gene and the MLLT10 gene is found in acute lymphoblastic leukemia, acute myeloid leukemia and malignant lymphomas. The polymorphisms of this gene are associated with the risk of Alzheimer disease. Multiple alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, May 2011]\", \"label\": \"phosphatidylinositol binding clathrin assembly protein\", \"weight\": -0.14693108, \"cname\": \"PICALM\", \"id\": [\"Gene\", \"HGNC\", \"PICALM\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SORL1\", \"description\": \"This gene encodes a mosaic protein that belongs to at least two families: the vacuolar protein sorting 10 (VPS10) domain-containing receptor family, and the low density lipoprotein receptor (LDLR) family. The encoded protein also contains fibronectin type III repeats and an epidermal growth factor repeat. The encoded preproprotein is proteolytically processed to generate the mature receptor, which likely plays roles in endocytosis and sorting. Mutations in this gene may be associated with Alzheimer's disease. [provided by RefSeq, Feb 2016]\", \"label\": \"sortilin related receptor 1\", \"weight\": -0.01427188, \"cname\": \"SORL1\", \"id\": [\"Gene\", \"HGNC\", \"SORL1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3A\", \"description\": \"This gene encodes a multifunctional Ser/Thr protein kinase that is implicated in the control of several regulatory proteins including glycogen synthase, and transcription factors, such as JUN. It also plays a role in the WNT and PI3K signaling pathways, as well as regulates the production of beta-amyloid peptides associated with Alzheimer's disease. [provided by RefSeq, Oct 2011]\", \"label\": \"glycogen synthase kinase 3 alpha\", \"weight\": -0.45894849, \"cname\": \"GSK3A\", \"id\": [\"Gene\", \"HGNC\", \"GSK3A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ACHE\", \"description\": \"Acetylcholinesterase hydrolyzes the neurotransmitter, acetylcholine at neuromuscular junctions and brain cholinergic synapses, and thus terminates signal transmission. It is also found on the red blood cell membranes, where it constitutes the Yt blood group antigen. Acetylcholinesterase exists in multiple molecular forms which possess similar catalytic properties, but differ in their oligomeric assembly and mode of cell attachment to the cell surface. It is encoded by the single ACHE gene, and the structural diversity in the gene products arises from alternative mRNA splicing, and post-translational associations of catalytic and structural subunits. The major form of acetylcholinesterase found in brain, muscle and other tissues is the hydrophilic species, which forms disulfide-linked oligomers with collagenous, or lipid-containing structural subunits. The other, alternatively spliced form, expressed primarily in the erythroid tissues, differs at the C-terminal end, and contains a cleavable hydrophobic peptide with a GPI-anchor site. It associates with the membranes through the phosphoinositide (PI) moieties added post-translationally. [provided by RefSeq, Jul 2008]\", \"label\": \"acetylcholinesterase (Cartwright blood group)\", \"weight\": 0.49236940000000007, \"cname\": \"ACHE\", \"id\": [\"Gene\", \"HGNC\", \"ACHE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"INSR\", \"description\": \"This gene encodes a member of the receptor tyrosine kinase family of proteins. The encoded preproprotein is proteolytically processed to generate alpha and beta subunits that form a heterotetrameric receptor. Binding of insulin or other ligands to this receptor activates the insulin signaling pathway, which regulates glucose uptake and release, as well as the synthesis and storage of carbohydrates, lipids and protein. Mutations in this gene underlie the inherited severe insulin resistance syndromes including type A insulin resistance syndrome, Donohue syndrome and Rabson-Mendenhall syndrome. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Oct 2015]\", \"label\": \"insulin receptor\", \"weight\": 0.049953899999999996, \"cname\": \"INSR\", \"id\": [\"Gene\", \"HGNC\", \"INSR\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NOS3\", \"description\": \"Nitric oxide is a reactive free radical which acts as a biologic mediator in several processes, including neurotransmission and antimicrobial and antitumoral activities. Nitric oxide is synthesized from L-arginine by nitric oxide synthases. Variations in this gene are associated with susceptibility to coronary spasm. Alternative splicing and the use of alternative promoters results in multiple transcript variants. [provided by RefSeq, Oct 2016]\", \"label\": \"nitric oxide synthase 3\", \"weight\": -0.1714975, \"cname\": \"NOS3\", \"id\": [\"Gene\", \"HGNC\", \"NOS3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"BACE1\", \"description\": \"This gene encodes a member of the peptidase A1 family of aspartic proteases. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate the mature protease. This transmembrane protease catalyzes the first step in the formation of amyloid beta peptide from amyloid precursor protein. Amyloid beta peptides are the main constituent of amyloid beta plaques, which accumulate in the brains of human Alzheimer's disease patients. [provided by RefSeq, Nov 2015]\", \"label\": \"beta-secretase 1\", \"weight\": -0.00189721, \"cname\": \"BACE1\", \"id\": [\"Gene\", \"HGNC\", \"BACE1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"INS\", \"description\": \"After removal of the precursor signal peptide, proinsulin is post-translationally cleaved into three peptides: the B chain and A chain peptides, which are covalently linked via two disulfide bonds to form insulin, and C-peptide. Binding of insulin to the insulin receptor (INSR) stimulates glucose uptake. A multitude of mutant alleles with phenotypic effects have been identified. There is a read-through gene, INS-IGF2, which overlaps with this gene at the 5' region and with the IGF2 gene at the 3' region. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2010]\", \"label\": \"insulin\", \"weight\": 0.99910812, \"cname\": \"INS\", \"id\": [\"Gene\", \"HGNC\", \"INS\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"STK11\", \"description\": \"This gene, which encodes a member of the serine/threonine kinase family, regulates cell polarity and functions as a tumor suppressor. Mutations in this gene have been associated with Peutz-Jeghers syndrome, an autosomal dominant disorder characterized by the growth of polyps in the gastrointestinal tract, pigmented macules on the skin and mouth, and other neoplasms. Alternate transcriptional splice variants of this gene have been observed but have not been thoroughly characterized. [provided by RefSeq, Jul 2008]\", \"label\": \"serine/threonine kinase 11\", \"weight\": 0.17049337, \"cname\": \"STK11\", \"id\": [\"Gene\", \"HGNC\", \"STK11\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SPTLC1\", \"description\": \"This gene encodes a member of the class-II pyridoxal-phosphate-dependent aminotransferase family. The encoded protein is the long chain base subunit 1 of serine palmitoyltransferase. Serine palmitoyltransferase converts L-serine and palmitoyl-CoA to 3-oxosphinganine with pyridoxal 5'-phosphate and is the key enzyme in sphingolipid biosynthesis. Mutations in this gene were identified in patients with hereditary sensory neuropathy type 1. Alternatively spliced variants encoding different isoforms have been identified. Pseudogenes of this gene have been defined on chromosomes 1, 6, 10, and 13. [provided by RefSeq, Jul 2013]\", \"label\": \"serine palmitoyltransferase long chain base subunit 1\", \"weight\": -0.0808145, \"cname\": \"SPTLC1\", \"id\": [\"Gene\", \"HGNC\", \"SPTLC1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APBA1\", \"description\": \"The protein encoded by this gene is a member of the X11 protein family. It is a neuronal adapter protein that interacts with the Alzheimer's disease amyloid precursor protein (APP). It stabilizes APP and inhibits production of proteolytic APP fragments including the A beta peptide that is deposited in the brains of Alzheimer's disease patients. This gene product is believed to be involved in signal transduction processes. It is also regarded as a putative vesicular trafficking protein in the brain that can form a complex with the potential to couple synaptic vesicle exocytosis to neuronal cell adhesion. [provided by RefSeq, Jul 2008]\", \"label\": \"amyloid beta precursor protein binding family A member 1\", \"weight\": -0.48809369999999996, \"cname\": \"APBA1\", \"id\": [\"Gene\", \"HGNC\", \"APBA1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SORCS1\", \"description\": \"This gene encodes one family member of vacuolar protein sorting 10 (VPS10) domain-containing receptor proteins. The VPS10 domain name comes from the yeast carboxypeptidase Y sorting receptor Vps10 protein. Members of this gene family are large with many exons but the CDS lengths are usually less than 3700 nt. Very large introns typically separate the exons encoding the VPS10 domain; the remaining exons are separated by much smaller-sized introns. These genes are strongly expressed in the central nervous system. Two of the five family members (sortilin and sortilin-related receptor) are synthesized as preproproteins; it is not yet known if this encoded protein is also a preproprotein. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]\", \"label\": \"sortilin related VPS10 domain containing receptor 1\", \"weight\": -1.5152121, \"cname\": \"SORCS1\", \"id\": [\"Gene\", \"HGNC\", \"SORCS1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSENEN\", \"description\": \"Presenilins, which are components of the gamma-secretase protein complex, are required for intramembranous processing of some type I transmembrane proteins, such as the Notch proteins and the beta-amyloid precursor protein. Signaling by Notch receptors mediates a wide range of developmental cell fates. Processing of the beta-amyloid precursor protein generates neurotoxic amyloid beta peptides, the major component of senile plaques associated with Alzheimer's disease. This gene encodes a protein that is required for Notch pathway signaling, and for the activity and accumulation of gamma-secretase. Mutations resulting in haploinsufficiency for this gene cause familial acne inversa-2 (ACNINV2). Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]\", \"label\": \"presenilin enhancer gamma-secretase subunit\", \"weight\": 1.24788008, \"cname\": \"PSENEN\", \"id\": [\"Gene\", \"HGNC\", \"PSENEN\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IDE\", \"description\": \"This gene encodes a zinc metallopeptidase that degrades intracellular insulin, and thereby terminates insulins activity, as well as participating in intercellular peptide signalling by degrading diverse peptides such as glucagon, amylin, bradykinin, and kallidin. The preferential affinity of this enzyme for insulin results in insulin-mediated inhibition of the degradation of other peptides such as beta-amyloid. Deficiencies in this protein's function are associated with Alzheimer's disease and type 2 diabetes mellitus but mutations in this gene have not been shown to be causitive for these diseases. This protein localizes primarily to the cytoplasm but in some cell types localizes to the extracellular space, cell membrane, peroxisome, and mitochondrion. Alternative splicing results in multiple transcript variants encoding distinct isoforms. Additional transcript variants have been described but have not been experimentally verified.[provided by RefSeq, Sep 2009]\", \"label\": \"insulin degrading enzyme\", \"weight\": 0.28059224, \"cname\": \"IDE\", \"id\": [\"Gene\", \"HGNC\", \"IDE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CCR2\", \"description\": \"This gene encodes two isoforms of a receptor for monocyte chemoattractant protein-1, a chemokine which specifically mediates monocyte chemotaxis. Monocyte chemoattractant protein-1 is involved in monocyte infiltration in inflammatory diseases such as rheumatoid arthritis as well as in the inflammatory response against tumors. The receptors encoded by this gene mediate agonist-dependent calcium mobilization and inhibition of adenylyl cyclase. This gene is located in the chemokine receptor gene cluster region. Two alternatively spliced transcript variants are expressed by the gene. [provided by RefSeq, Mar 2009]\", \"label\": \"C-C motif chemokine receptor 2\", \"weight\": 0.72408115, \"cname\": \"CCR2\", \"id\": [\"Gene\", \"HGNC\", \"CCR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"BACE2\", \"description\": \"This gene encodes an integral membrane glycoprotein that functions as an aspartic protease. The encoded protein cleaves amyloid precursor protein into amyloid beta peptide, which is a critical step in the etiology of Alzheimer's disease and Down syndrome. The protein precursor is further processed into an active mature peptide. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jul 2013]\", \"label\": \"beta-site APP-cleaving enzyme 2\", \"weight\": -0.34884808, \"cname\": \"BACE2\", \"id\": [\"Gene\", \"HGNC\", \"BACE2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IGF1\", \"description\": \"The protein encoded by this gene is similar to insulin in function and structure and is a member of a family of proteins involved in mediating growth and development. The encoded protein is processed from a precursor, bound by a specific receptor, and secreted. Defects in this gene are a cause of insulin-like growth factor I deficiency. Alternative splicing results in multiple transcript variants encoding different isoforms that may undergo similar processing to generate mature protein. [provided by RefSeq, Sep 2015]\", \"label\": \"insulin like growth factor 1\", \"weight\": 0.98026104, \"cname\": \"IGF1\", \"id\": [\"Gene\", \"HGNC\", \"IGF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"RELA\", \"description\": \"NF-kappa-B is a ubiquitous transcription factor involved in several biological processes. It is held in the cytoplasm in an inactive state by specific inhibitors. Upon degradation of the inhibitor, NF-kappa-B moves to the nucleus and activates transcription of specific genes. NF-kappa-B is composed of NFKB1 or NFKB2 bound to either REL, RELA, or RELB. The most abundant form of NF-kappa-B is NFKB1 complexed with the product of this gene, RELA. Four transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Sep 2011]\", \"label\": \"RELA proto-oncogene, NF-kB subunit\", \"weight\": 0.39927783, \"cname\": \"RELA\", \"id\": [\"Gene\", \"HGNC\", \"RELA\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CAST\", \"description\": \"The protein encoded by this gene is an endogenous calpain (calcium-dependent cysteine protease) inhibitor. It consists of an N-terminal domain L and four repetitive calpain-inhibition domains (domains 1-4), and it is involved in the proteolysis of amyloid precursor protein. The calpain/calpastatin system is involved in numerous membrane fusion events, such as neural vesicle exocytosis and platelet and red-cell aggregation. The encoded protein is also thought to affect the expression levels of genes encoding structural or regulatory proteins. Alternatively spliced transcript variants encoding different isoforms have been described. [provided by RefSeq, Jun 2010]\", \"label\": \"calpastatin\", \"weight\": 0.03364871, \"cname\": \"CAST\", \"id\": [\"Gene\", \"HGNC\", \"CAST\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NFKB1\", \"description\": \"This gene encodes a 105 kD protein which can undergo cotranslational processing by the 26S proteasome to produce a 50 kD protein. The 105 kD protein is a Rel protein-specific transcription inhibitor and the 50 kD protein is a DNA binding subunit of the NF-kappa-B (NFKB) protein complex. NFKB is a transcription regulator that is activated by various intra- and extra-cellular stimuli such as cytokines, oxidant-free radicals, ultraviolet irradiation, and bacterial or viral products. Activated NFKB translocates into the nucleus and stimulates the expression of genes involved in a wide variety of biological functions. Inappropriate activation of NFKB has been associated with a number of inflammatory diseases while persistent inhibition of NFKB leads to inappropriate immune cell development or delayed cell growth. Alternative splicing results in multiple transcript variants encoding different isoforms, at least one of which is proteolytically processed. [provided by RefSeq, Feb 2016]\", \"label\": \"nuclear factor kappa B subunit 1\", \"weight\": -1.33219344, \"cname\": \"NFKB1\", \"id\": [\"Gene\", \"HGNC\", \"NFKB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"DKK1\", \"description\": \"This gene encodes a protein that is a member of the dickkopf family. It is a secreted protein with two cysteine rich regions and is involved in embryonic development through its inhibition of the WNT signaling pathway. Elevated levels of DKK1 in bone marrow plasma and peripheral blood is associated with the presence of osteolytic bone lesions in patients with multiple myeloma. [provided by RefSeq, Jul 2008]\", \"label\": \"dickkopf WNT signaling pathway inhibitor 1\", \"weight\": -1.58173755, \"cname\": \"DKK1\", \"id\": [\"Gene\", \"HGNC\", \"DKK1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"DHCR24\", \"description\": \"This gene encodes a flavin adenine dinucleotide (FAD)-dependent oxidoreductase which catalyzes the reduction of the delta-24 double bond of sterol intermediates during cholesterol biosynthesis. The protein contains a leader sequence that directs it to the endoplasmic reticulum membrane. Missense mutations in this gene have been associated with desmosterolosis. Also, reduced expression of the gene occurs in the temporal cortex of Alzheimer disease patients and overexpression has been observed in adrenal gland cancer cells. [provided by RefSeq, Jul 2008]\", \"label\": \"24-dehydrocholesterol reductase\", \"weight\": -0.17838582, \"cname\": \"DHCR24\", \"id\": [\"Gene\", \"HGNC\", \"DHCR24\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MIR124-1\", \"description\": \"microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]\", \"label\": \"microRNA 124-1\", \"weight\": 0, \"cname\": \"MIR124-1\", \"id\": [\"Gene\", \"HGNC\", \"MIR124-1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL1B\", \"description\": \"The protein encoded by this gene is a member of the interleukin 1 cytokine family. This cytokine is produced by activated macrophages as a proprotein, which is proteolytically processed to its active form by caspase 1 (CASP1/ICE). This cytokine is an important mediator of the inflammatory response, and is involved in a variety of cellular activities, including cell proliferation, differentiation, and apoptosis. The induction of cyclooxygenase-2 (PTGS2/COX2) by this cytokine in the central nervous system (CNS) is found to contribute to inflammatory pain hypersensitivity. This gene and eight other interleukin 1 family genes form a cytokine gene cluster on chromosome 2. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 1 beta\", \"weight\": -1.63771704, \"cname\": \"IL1B\", \"id\": [\"Gene\", \"HGNC\", \"IL1B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ATXN1\", \"description\": \"The autosomal dominant cerebellar ataxias (ADCA) are a heterogeneous group of neurodegenerative disorders characterized by progressive degeneration of the cerebellum, brain stem and spinal cord. Clinically, ADCA has been divided into three groups: ADCA types I-III. ADCAI is genetically heterogeneous, with five genetic loci, designated spinocerebellar ataxia (SCA) 1, 2, 3, 4 and 6, being assigned to five different chromosomes. ADCAII, which always presents with retinal degeneration (SCA7), and ADCAIII often referred to as the `pure' cerebellar syndrome (SCA5), are most likely homogeneous disorders. Several SCA genes have been cloned and shown to contain CAG repeats in their coding regions. ADCA is caused by the expansion of the CAG repeats, producing an elongated polyglutamine tract in the corresponding protein. The expanded repeats are variable in size and unstable, usually increasing in size when transmitted to successive generations. The function of the ataxins is not known. This locus has been mapped to chromosome 6, and it has been determined that the diseased allele contains 40-83 CAG repeats, compared to 6-39 in the normal allele, and is associated with spinocerebellar ataxia type 1 (SCA1). At least two transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2016]\", \"label\": \"ataxin 1\", \"weight\": 0.00954063, \"cname\": \"ATXN1\", \"id\": [\"Gene\", \"HGNC\", \"ATXN1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SPTLC3\", \"description\": \"This gene encodes a subunit of the serine palmitoyltransferase complex which catalyzes the rate-limiting step in sphingolipid biosynthesis. This subunit metabolizes lauroyl- and myristoyl-CoA and generates C14 and C16-sphingoid bases. [provided by RefSeq, Mar 2017]\", \"label\": \"serine palmitoyltransferase long chain base subunit 3\", \"weight\": 0.19053622, \"cname\": \"SPTLC3\", \"id\": [\"Gene\", \"HGNC\", \"SPTLC3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ADAM10\", \"description\": \"Members of the ADAM family are cell surface proteins with a unique structure possessing both potential adhesion and protease domains. This gene encodes and ADAM family member that cleaves many proteins including TNF-alpha and E-cadherin. Alternate splicing results in multiple transcript variants encoding different proteins that may undergo similar processing. [provided by RefSeq, Feb 2016]\", \"label\": \"ADAM metallopeptidase domain 10\", \"weight\": -0.051464199999999995, \"cname\": \"ADAM10\", \"id\": [\"Gene\", \"HGNC\", \"ADAM10\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FTL\", \"description\": \"This gene encodes the light subunit of the ferritin protein. Ferritin is the major intracellular iron storage protein in prokaryotes and eukaryotes. It is composed of 24 subunits of the heavy and light ferritin chains. Variation in ferritin subunit composition may affect the rates of iron uptake and release in different tissues. A major function of ferritin is the storage of iron in a soluble and nontoxic state. Defects in this light chain ferritin gene are associated with several neurodegenerative diseases and hyperferritinemia-cataract syndrome. This gene has multiple pseudogenes. [provided by RefSeq, Jul 2008]\", \"label\": \"ferritin light chain\", \"weight\": 0.70775495, \"cname\": \"FTL\", \"id\": [\"Gene\", \"HGNC\", \"FTL\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"DDIT3\", \"description\": \"This gene encodes a member of the CCAAT/enhancer-binding protein (C/EBP) family of transcription factors. The protein functions as a dominant-negative inhibitor by forming heterodimers with other C/EBP members, such as C/EBP and LAP (liver activator protein), and preventing their DNA binding activity. The protein is implicated in adipogenesis and erythropoiesis, is activated by endoplasmic reticulum stress, and promotes apoptosis. Fusion of this gene and FUS on chromosome 16 or EWSR1 on chromosome 22 induced by translocation generates chimeric proteins in myxoid liposarcomas or Ewing sarcoma. Multiple alternatively spliced transcript variants encoding two isoforms with different length have been identified. [provided by RefSeq, Aug 2010]\", \"label\": \"DNA damage inducible transcript 3\", \"weight\": 1.40355466, \"cname\": \"DDIT3\", \"id\": [\"Gene\", \"HGNC\", \"DDIT3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CLSTN1\", \"description\": \"This gene is a member of the calsyntenin family, a subset of the cadherin superfamily. The encoded transmembrane protein, also known as alcadein-alpha, is thought to bind to kinesin-1 motors to mediate the axonal anterograde transport of certain types of vesicle. Amyloid precursor protein (APP) is trafficked via these vesicles and so this protein is being investigated to see how it might contribute to the mechanisms underlying Alzheimer's disease. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2014]\", \"label\": \"calsyntenin 1\", \"weight\": 0.38076231, \"cname\": \"CLSTN1\", \"id\": [\"Gene\", \"HGNC\", \"CLSTN1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"KLC1\", \"description\": \"Conventional kinesin is a tetrameric molecule composed of two heavy chains and two light chains, and transports various cargos along microtubules toward their plus ends. The heavy chains provide the motor activity, while the light chains bind to various cargos. This gene encodes a member of the kinesin light chain family. It associates with kinesin heavy chain through an N-terminal domain, and six tetratricopeptide repeat (TPR) motifs are thought to be involved in binding of cargos such as vesicles, mitochondria, and the Golgi complex. Thus, kinesin light chains function as adapter molecules and not motors per se. Although previously named \\\"kinesin 2\\\", this gene is not a member of the kinesin-2 / kinesin heavy chain subfamily of kinesin motor proteins. Extensive alternative splicing produces isoforms with different C-termini that are proposed to bind to different cargos; however, the full-length nature and/or biological validity of most of these variants have not been determined. [provided by RefSeq, Jul 2008]\", \"label\": \"kinesin light chain 1\", \"weight\": -0.09664774, \"cname\": \"KLC1\", \"id\": [\"Gene\", \"HGNC\", \"KLC1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APH1A\", \"description\": \"This gene encodes a component of the gamma secretase complex that cleaves integral membrane proteins such as Notch receptors and beta-amyloid precursor protein. The gamma secretase complex contains this gene product, or the paralogous anterior pharynx defective 1 homolog B (APH1B), along with the presenilin, nicastrin, and presenilin enhancer-2 proteins. The precise function of this seven-transmembrane-domain protein is unknown though it is suspected of facilitating the association of nicastrin and presenilin in the gamma secretase complex as well as interacting with substrates of the gamma secretase complex prior to their proteolytic processing. Polymorphisms in a promoter region of this gene have been associated with an increased risk for developing sporadic Alzheimer's disease. Alternative splicing results in multiple protein-coding and non-protein-coding transcript variants. [provided by RefSeq, Aug 2011]\", \"label\": \"aph-1 homolog A, gamma-secretase subunit\", \"weight\": -0.42715932, \"cname\": \"APH1A\", \"id\": [\"Gene\", \"HGNC\", \"APH1A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GPR3\", \"description\": \"This gene is a member of the G protein-coupled receptor family and is found in the cell membrane. G protein-coupled receptors, characterized by a seven transmembrane domain motif, are involved in translating outside signals into G protein mediated intracellular effects. The encoded protein activates adenylate cyclase and modulates amyloid-beta production in a mouse model, suggesting that it may play a role in Alzheimer's disease. [provided by RefSeq, Oct 2012]\", \"label\": \"G protein-coupled receptor 3\", \"weight\": 0.14999666, \"cname\": \"GPR3\", \"id\": [\"Gene\", \"HGNC\", \"GPR3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"A2M\", \"description\": \"The protein encoded by this gene is a protease inhibitor and cytokine transporter. It uses a bait-and-trap mechanism to inhibit a broad spectrum of proteases, including trypsin, thrombin and collagenase. It can also inhibit inflammatory cytokines, and it thus disrupts inflammatory cascades. Mutations in this gene are a cause of alpha-2-macroglobulin deficiency. This gene is implicated in Alzheimer's disease (AD) due to its ability to mediate the clearance and degradation of A-beta, the major component of beta-amyloid deposits. A related pseudogene, which is also located on the p arm of chromosome 12, has been identified. [provided by RefSeq, Nov 2016]\", \"label\": \"alpha-2-macroglobulin\", \"weight\": 1.08160069, \"cname\": \"A2M\", \"id\": [\"Gene\", \"HGNC\", \"A2M\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MTRNR2L2\", \"description\": null, \"label\": \"MT-RNR2-like 2\", \"weight\": 0, \"cname\": \"MTRNR2L2\", \"id\": [\"Gene\", \"HGNC\", \"MTRNR2L2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APBB1\", \"description\": \"The protein encoded by this gene is a member of the Fe65 protein family. It is an adaptor protein localized in the nucleus. It interacts with the Alzheimer's disease amyloid precursor protein (APP), transcription factor CP2/LSF/LBP1 and the low-density lipoprotein receptor-related protein. APP functions as a cytosolic anchoring site that can prevent the gene product's nuclear translocation. This encoded protein could play an important role in the pathogenesis of Alzheimer's disease. It is thought to regulate transcription. Also it is observed to block cell cycle progression by downregulating thymidylate synthase expression. Multiple alternatively spliced transcript variants encoding different isoforms have been described for this gene. [provided by RefSeq, Mar 2012]\", \"label\": \"amyloid beta precursor protein binding family B member 1\", \"weight\": 0.59661584, \"cname\": \"APBB1\", \"id\": [\"Gene\", \"HGNC\", \"APBB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APBA2\", \"description\": \"The protein encoded by this gene is a member of the X11 protein family. It is a neuronal adapter protein that interacts with the Alzheimer's disease amyloid precursor protein (APP). It stabilizes APP and inhibits production of proteolytic APP fragments including the A beta peptide that is deposited in the brains of Alzheimer's disease patients. This gene product is believed to be involved in signal transduction processes. It is also regarded as a putative vesicular trafficking protein in the brain that can form a complex with the potential to couple synaptic vesicle exocytosis to neuronal cell adhesion. [provided by RefSeq, Jul 2017]\", \"label\": \"amyloid beta precursor protein binding family A member 2\", \"weight\": -0.19241516, \"cname\": \"APBA2\", \"id\": [\"Gene\", \"HGNC\", \"APBA2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"LDLR\", \"description\": \"The low density lipoprotein receptor (LDLR) gene family consists of cell surface proteins involved in receptor-mediated endocytosis of specific ligands. Low density lipoprotein (LDL) is normally bound at the cell membrane and taken into the cell ending up in lysosomes where the protein is degraded and the cholesterol is made available for repression of microsomal enzyme 3-hydroxy-3-methylglutaryl coenzyme A (HMG CoA) reductase, the rate-limiting step in cholesterol synthesis. At the same time, a reciprocal stimulation of cholesterol ester synthesis takes place. Mutations in this gene cause the autosomal dominant disorder, familial hypercholesterolemia. Alternate splicing results in multiple transcript variants.[provided by RefSeq, Sep 2010]\", \"label\": \"low density lipoprotein receptor\", \"weight\": -0.01435724, \"cname\": \"LDLR\", \"id\": [\"Gene\", \"HGNC\", \"LDLR\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PIN1\", \"description\": \"Peptidyl-prolyl cis/trans isomerases (PPIases) catalyze the cis/trans isomerization of peptidyl-prolyl peptide bonds. This gene encodes one of the PPIases, which specifically binds to phosphorylated ser/thr-pro motifs to catalytically regulate the post-phosphorylation conformation of its substrates. The conformational regulation catalyzed by this PPIase has a profound impact on key proteins involved in the regulation of cell growth, genotoxic and other stress responses, the immune response, induction and maintenance of pluripotency, germ cell development, neuronal differentiation, and survival. This enzyme also plays a key role in the pathogenesis of Alzheimer's disease and many cancers. Multiple alternatively spliced transcript variants have been found for this gene.[provided by RefSeq, Jun 2011]\", \"label\": \"peptidylprolyl cis/trans isomerase, NIMA-interacting 1\", \"weight\": -0.25496963, \"cname\": \"PIN1\", \"id\": [\"Gene\", \"HGNC\", \"PIN1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SPTLC2\", \"description\": \"This gene encodes a long chain base subunit of serine palmitoyltransferase. Serine palmitoyltransferase, which consists of two different subunits, is the key enzyme in sphingolipid biosynthesis. It catalyzes the pyridoxal-5-prime-phosphate-dependent condensation of L-serine and palmitoyl-CoA to 3-oxosphinganine. Mutations in this gene were identified in patients with hereditary sensory neuropathy type I. [provided by RefSeq, Mar 2011]\", \"label\": \"serine palmitoyltransferase long chain base subunit 2\", \"weight\": -0.06391096, \"cname\": \"SPTLC2\", \"id\": [\"Gene\", \"HGNC\", \"SPTLC2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NAT8B\", \"description\": \"The protein encoded by this gene is highly similar to the N-acetyltransferase 8 (NAT8) gene product, which is a kidney and liver protein with homology to bacterial acetyltransferases involved in drug resistance. This gene is localized on chromosome 2 in the vicinity of the NAT8 gene and may represent a pseudogene of NAT8. This gene contains two polymorphic nonsense mutations that disrupt the active site of the protein. The full-length product of this gene contains a complete acetyltransferase domain and is identical in length to NAT8. [provided by RefSeq, Jul 2008]\", \"label\": \"N-acetyltransferase 8B (putative, gene/pseudogene)\", \"weight\": 0.20466742, \"cname\": \"NAT8B\", \"id\": [\"Gene\", \"HGNC\", \"NAT8B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GGA3\", \"description\": \"This gene encodes a member of the Golgi-localized, gamma adaptin ear-containing, ARF-binding (GGA) family. This family includes ubiquitous coat proteins that regulate the trafficking of proteins between the trans-Golgi network and the lysosome. These proteins share an amino-terminal VHS domain which mediates sorting of the mannose 6-phosphate receptors at the trans-Golgi network. They also contain a carboxy-terminal region with homology to the ear domain of gamma-adaptins. Multiple alternatively spliced transcript variants have been identified in this gene. [provided by RefSeq, Feb 2010]\", \"label\": \"golgi associated, gamma adaptin ear containing, ARF binding protein 3\", \"weight\": -0.81325519, \"cname\": \"GGA3\", \"id\": [\"Gene\", \"HGNC\", \"GGA3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PRNP\", \"description\": \"The protein encoded by this gene is a membrane glycosylphosphatidylinositol-anchored glycoprotein that tends to aggregate into rod-like structures. The encoded protein contains a highly unstable region of five tandem octapeptide repeats. This gene is found on chromosome 20, approximately 20 kbp upstream of a gene which encodes a biochemically and structurally similar protein to the one encoded by this gene. Mutations in the repeat region as well as elsewhere in this gene have been associated with Creutzfeldt-Jakob disease, fatal familial insomnia, Gerstmann-Straussler disease, Huntington disease-like 1, and kuru. An overlapping open reading frame has been found for this gene that encodes a smaller, structurally unrelated protein, AltPrp. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Nov 2014]\", \"label\": \"prion protein\", \"weight\": 0.08884805, \"cname\": \"PRNP\", \"id\": [\"Gene\", \"HGNC\", \"PRNP\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ABCA2\", \"description\": \"The membrane-associated protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intracellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This protein is a member of the ABC1 subfamily. Members of the ABC1 subfamily comprise the only major ABC subfamily found exclusively in multicellular eukaryotes. This protein is highly expressed in brain tissue and may play a role in macrophage lipid metabolism and neural development. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]\", \"label\": \"ATP binding cassette subfamily A member 2\", \"weight\": -0.03926027, \"cname\": \"ABCA2\", \"id\": [\"Gene\", \"HGNC\", \"ABCA2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MARCKS\", \"description\": \"The protein encoded by this gene is a substrate for protein kinase C. It is localized to the plasma membrane and is an actin filament crosslinking protein. Phosphorylation by protein kinase C or binding to calcium-calmodulin inhibits its association with actin and with the plasma membrane, leading to its presence in the cytoplasm. The protein is thought to be involved in cell motility, phagocytosis, membrane trafficking and mitogenesis. [provided by RefSeq, Jul 2008]\", \"label\": \"myristoylated alanine rich protein kinase C substrate\", \"weight\": 0.07542072, \"cname\": \"MARCKS\", \"id\": [\"Gene\", \"HGNC\", \"MARCKS\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL34\", \"description\": \"Interleukin-34 is a cytokine that promotes the differentiation and viability of monocytes and macrophages through the colony-stimulating factor-1 receptor (CSF1R; MIM 164770) (Lin et al., 2008 [PubMed 18467591]).[supplied by OMIM, May 2008]\", \"label\": \"interleukin 34\", \"weight\": -0.18066962, \"cname\": \"IL34\", \"id\": [\"Gene\", \"HGNC\", \"IL34\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ADAM9\", \"description\": \"This gene encodes a member of the ADAM (a disintegrin and metalloprotease domain) family. Members of this family are membrane-anchored proteins structurally related to snake venom disintegrins, and have been implicated in a variety of biological processes involving cell-cell and cell-matrix interactions, including fertilization, muscle development, and neurogenesis. The protein encoded by this gene interacts with SH3 domain-containing proteins, binds mitotic arrest deficient 2 beta protein, and is also involved in TPA-induced ectodomain shedding of membrane-anchored heparin-binding EGF-like growth factor. Several alternatively spliced transcript variants have been identified for this gene. [provided by RefSeq, Jul 2010]\", \"label\": \"ADAM metallopeptidase domain 9\", \"weight\": -0.16119139, \"cname\": \"ADAM9\", \"id\": [\"Gene\", \"HGNC\", \"ADAM9\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ADAM17\", \"description\": \"This gene encodes a member of the ADAM (a disintegrin and metalloprotease domain) family. Members of this family are membrane-anchored proteins structurally related to snake venom disintegrins, and have been implicated in a variety of biologic processes involving cell-cell and cell-matrix interactions, including fertilization, muscle development, and neurogenesis. The encoded preproprotein is proteolytically processed to generate the mature protease. The encoded protease functions in the ectodomain shedding of tumor necrosis factor-alpha, in which soluble tumor necrosis factor-alpha is released from the membrane-bound precursor. This protease also functions in the processing of numerous other substrates, including cell adhesion proteins, cytokine and growth factor receptors and epidermal growth factor (EGF) receptor ligands. The encoded protein also plays a prominent role in the activation of the Notch signaling pathway. Elevated expression of this gene has been observed in specific cell types derived from psoriasis, rheumatoid arthritis, multiple sclerosis and Crohn's disease patients, suggesting that the encoded protein may play a role in autoimmune disease. [provided by RefSeq, Feb 2016]\", \"label\": \"ADAM metallopeptidase domain 17\", \"weight\": 0.21966586, \"cname\": \"ADAM17\", \"id\": [\"Gene\", \"HGNC\", \"ADAM17\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTSB\", \"description\": \"This gene encodes a member of the C1 family of peptidases. Alternative splicing of this gene results in multiple transcript variants. At least one of these variants encodes a preproprotein that is proteolytically processed to generate multiple protein products. These products include the cathepsin B light and heavy chains, which can dimerize to form the double chain form of the enzyme. This enzyme is a lysosomal cysteine protease with both endopeptidase and exopeptidase activity that may play a role in protein turnover. It is also known as amyloid precursor protein secretase and is involved in the proteolytic processing of amyloid precursor protein (APP). Incomplete proteolytic processing of APP has been suggested to be a causative factor in Alzheimer's disease, the most common cause of dementia. Overexpression of the encoded protein has been associated with esophageal adenocarcinoma and other tumors. Multiple pseudogenes of this gene have been identified. [provided by RefSeq, Nov 2015]\", \"label\": \"cathepsin B\", \"weight\": -0.04291486, \"cname\": \"CTSB\", \"id\": [\"Gene\", \"HGNC\", \"CTSB\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTSE\", \"description\": \"This gene encodes a member of the A1 family of peptidases. Alternative splicing of this gene results in multiple transcript variants. At least one of these variants encodes a preproprotein that is proteolytically processed to generate the mature enzyme. This enzyme, an aspartic endopeptidase, may be involved in antigen processing and the maturation of secretory proteins. Elevated expression of this gene has been observed in neurodegeneration. [provided by RefSeq, Nov 2015]\", \"label\": \"cathepsin E\", \"weight\": -1.2097365, \"cname\": \"CTSE\", \"id\": [\"Gene\", \"HGNC\", \"CTSE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GCG\", \"description\": \"The protein encoded by this gene is actually a preproprotein that is cleaved into four distinct mature peptides. One of these, glucagon, is a pancreatic hormone that counteracts the glucose-lowering action of insulin by stimulating glycogenolysis and gluconeogenesis. Glucagon is a ligand for a specific G-protein linked receptor whose signalling pathway controls cell proliferation. Two of the other peptides are secreted from gut endocrine cells and promote nutrient absorption through distinct mechanisms. Finally, the fourth peptide is similar to glicentin, an active enteroglucagon. [provided by RefSeq, Jul 2008]\", \"label\": \"glucagon\", \"weight\": -0.03157631, \"cname\": \"GCG\", \"id\": [\"Gene\", \"HGNC\", \"GCG\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ECE1\", \"description\": \"The protein encoded by this gene is involved in proteolytic processing of endothelin precursors to biologically active peptides. Mutations in this gene are associated with Hirschsprung disease, cardiac defects and autonomic dysfunction. Alternatively spliced transcript variants encoding different isoforms have been noted for this gene.[provided by RefSeq, Sep 2009]\", \"label\": \"endothelin converting enzyme 1\", \"weight\": -0.09027805, \"cname\": \"ECE1\", \"id\": [\"Gene\", \"HGNC\", \"ECE1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ACE\", \"description\": \"This gene encodes an enzyme involved in catalyzing the conversion of angiotensin I into a physiologically active peptide angiotensin II. Angiotensin II is a potent vasopressor and aldosterone-stimulating peptide that controls blood pressure and fluid-electrolyte balance. This enzyme plays a key role in the renin-angiotensin system. Many studies have associated the presence or absence of a 287 bp Alu repeat element in this gene with the levels of circulating enzyme or cardiovascular pathophysiologies. Multiple alternatively spliced transcript variants encoding different isoforms have been identified, and two most abundant spliced variants encode the somatic form and the testicular form, respectively, that are equally active. [provided by RefSeq, May 2010]\", \"label\": \"angiotensin I converting enzyme\", \"weight\": 0.38262923, \"cname\": \"ACE\", \"id\": [\"Gene\", \"HGNC\", \"ACE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"DPP4\", \"description\": \"The protein encoded by this gene is identical to adenosine deaminase complexing protein-2, and to the T-cell activation antigen CD26. It is an intrinsic membrane glycoprotein and a serine exopeptidase that cleaves X-proline dipeptides from the N-terminus of polypeptides. [provided by RefSeq, Jul 2008]\", \"label\": \"dipeptidyl peptidase 4\", \"weight\": 0.17409829999999998, \"cname\": \"DPP4\", \"id\": [\"Gene\", \"HGNC\", \"DPP4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"BLMH\", \"description\": \"Bleomycin hydrolase (BMH) is a cytoplasmic cysteine peptidase that is highly conserved through evolution; however, the only known activity of the enzyme is metabolic inactivation of the glycopeptide bleomycin (BLM), an essential component of combination chemotherapy regimens for cancer. The protein contains the signature active site residues of the cysteine protease papain superfamily. [provided by RefSeq, Jul 2008]\", \"label\": \"bleomycin hydrolase\", \"weight\": -0.85202484, \"cname\": \"BLMH\", \"id\": [\"Gene\", \"HGNC\", \"BLMH\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ALB\", \"description\": \"This gene encodes the most abundant protein in human blood. This protein functions in the regulation of blood plasma colloid osmotic pressure and acts as a carrier protein for a wide range of endogenous molecules including hormones, fatty acids, and metabolites, as well as exogenous drugs. Additionally, this protein exhibits an esterase-like activity with broad substrate specificity. The encoded preproprotein is proteolytically processed to generate the mature protein. A peptide derived from this protein, EPI-X4, is an endogenous inhibitor of the CXCR4 chemokine receptor. [provided by RefSeq, Jul 2016]\", \"label\": \"albumin\", \"weight\": 0.02354135, \"cname\": \"ALB\", \"id\": [\"Gene\", \"HGNC\", \"ALB\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"LEP\", \"description\": \"This gene encodes a protein that is secreted by white adipocytes, and which plays a major role in the regulation of body weight. This protein, which acts through the leptin receptor, functions as part of a signaling pathway that can inhibit food intake and/or regulate energy expenditure to maintain constancy of the adipose mass. This protein also has several endocrine functions, and is involved in the regulation of immune and inflammatory responses, hematopoiesis, angiogenesis and wound healing. Mutations in this gene and/or its regulatory regions cause severe obesity, and morbid obesity with hypogonadism. This gene has also been linked to type 2 diabetes mellitus development. [provided by RefSeq, Jul 2008]\", \"label\": \"leptin\", \"weight\": 1.01085844, \"cname\": \"LEP\", \"id\": [\"Gene\", \"HGNC\", \"LEP\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MMP9\", \"description\": \"Proteins of the matrix metalloproteinase (MMP) family are involved in the breakdown of extracellular matrix in normal physiological processes, such as embryonic development, reproduction, and tissue remodeling, as well as in disease processes, such as arthritis and metastasis. Most MMP's are secreted as inactive proproteins which are activated when cleaved by extracellular proteinases. The enzyme encoded by this gene degrades type IV and V collagens. Studies in rhesus monkeys suggest that the enzyme is involved in IL-8-induced mobilization of hematopoietic progenitor cells from bone marrow, and murine studies suggest a role in tumor-associated tissue remodeling. [provided by RefSeq, Jul 2008]\", \"label\": \"matrix metallopeptidase 9\", \"weight\": -0.32509715, \"cname\": \"MMP9\", \"id\": [\"Gene\", \"HGNC\", \"MMP9\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MMP2\", \"description\": \"This gene is a member of the matrix metalloproteinase (MMP) gene family, that are zinc-dependent enzymes capable of cleaving components of the extracellular matrix and molecules involved in signal transduction. The protein encoded by this gene is a gelatinase A, type IV collagenase, that contains three fibronectin type II repeats in its catalytic site that allow binding of denatured type IV and V collagen and elastin. Unlike most MMP family members, activation of this protein can occur on the cell membrane. This enzyme can be activated extracellularly by proteases, or, intracellulary by its S-glutathiolation with no requirement for proteolytical removal of the pro-domain. This protein is thought to be involved in multiple pathways including roles in the nervous system, endometrial menstrual breakdown, regulation of vascularization, and metastasis. Mutations in this gene have been associated with Winchester syndrome and Nodulosis-Arthropathy-Osteolysis (NAO) syndrome. Alternative splicing results in multiple transcript variants encoding different isoforms. [provided by RefSeq, Oct 2014]\", \"label\": \"matrix metallopeptidase 2\", \"weight\": -0.47336532, \"cname\": \"MMP2\", \"id\": [\"Gene\", \"HGNC\", \"MMP2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"BCL2L1\", \"description\": \"The protein encoded by this gene belongs to the BCL-2 protein family. BCL-2 family members form hetero- or homodimers and act as anti- or pro-apoptotic regulators that are involved in a wide variety of cellular activities. The proteins encoded by this gene are located at the outer mitochondrial membrane, and have been shown to regulate outer mitochondrial membrane channel (VDAC) opening. VDAC regulates mitochondrial membrane potential, and thus controls the production of reactive oxygen species and release of cytochrome C by mitochondria, both of which are the potent inducers of cell apoptosis. Alternative splicing results in multiple transcript variants encoding two different isoforms. The longer isoform acts as an apoptotic inhibitor and the shorter isoform acts as an apoptotic activator. [provided by RefSeq, Dec 2015]\", \"label\": \"BCL2 like 1\", \"weight\": -0.29881302, \"cname\": \"BCL2L1\", \"id\": [\"Gene\", \"HGNC\", \"BCL2L1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ABCA1\", \"description\": \"The membrane-associated protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters.  ABC proteins transport various molecules across extra- and intracellular membranes.  ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White).  This protein is a member of the ABC1 subfamily.  Members of the ABC1 subfamily comprise the only major ABC subfamily found exclusively in multicellular eukaryotes.  With cholesterol as its substrate, this protein functions as a cholesteral efflux pump in the cellular lipid removal pathway.  Mutations in this gene have been associated with Tangier's disease and familial high-density lipoprotein deficiency. [provided by RefSeq, Jul 2008]\", \"label\": \"ATP binding cassette subfamily A member 1\", \"weight\": 0.17888996, \"cname\": \"ABCA1\", \"id\": [\"Gene\", \"HGNC\", \"ABCA1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MMP7\", \"description\": \"This gene encodes a member of the peptidase M10 family of matrix metalloproteinases (MMPs). Proteins in this family are involved in the breakdown of extracellular matrix in normal physiological processes, such as embryonic development, reproduction, and tissue remodeling, as well as in disease processes, such as arthritis and metastasis. The encoded preproprotein is proteolytically processed to generate the mature protease. This secreted protease breaks down proteoglycans, fibronectin, elastin and casein and differs from most MMP family members in that it lacks a conserved C-terminal hemopexin domain. The enzyme is involved in wound healing, and studies in mice suggest that it regulates the activity of defensins in intestinal mucosa. The gene is part of a cluster of MMP genes on chromosome 11. This gene exhibits elevated expression levels in multiple human cancers. [provided by RefSeq, Jan 2016]\", \"label\": \"matrix metallopeptidase 7\", \"weight\": -1.19454136, \"cname\": \"MMP7\", \"id\": [\"Gene\", \"HGNC\", \"MMP7\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PLAT\", \"description\": \"This gene encodes tissue-type plasminogen activator, a secreted serine protease that converts the proenzyme plasminogen to plasmin, a fibrinolytic enzyme. The encoded preproprotein is proteolytically processed by plasmin or trypsin to generate heavy and light chains. These chains associate via disulfide linkages to form the heterodimeric enzyme. This enzyme plays a role in cell migration and tissue remodeling. Increased enzymatic activity causes hyperfibrinolysis, which manifests as excessive bleeding, while decreased activity leads to hypofibrinolysis, which can result in thrombosis or embolism. Alternative splicing of this gene results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Jan 2016]\", \"label\": \"plasminogen activator, tissue type\", \"weight\": 1.04346523, \"cname\": \"PLAT\", \"id\": [\"Gene\", \"HGNC\", \"PLAT\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SERPINI1\", \"description\": \"This gene encodes a member of the serpin superfamily of serine proteinase inhibitors. The protein is primarily secreted by axons in the brain, and preferentially reacts with and inhibits tissue-type plasminogen activator. It is thought to play a role in the regulation of axonal growth and the development of synaptic plasticity. Mutations in this gene result in familial encephalopathy with neuroserpin inclusion bodies (FENIB), which is a dominantly inherited form of familial encephalopathy and epilepsy characterized by the accumulation of mutant neuroserpin polymers. Multiple alternatively spliced variants, encoding the same protein, have been identified. [provided by RefSeq, Jul 2008]\", \"label\": \"serpin family I member 1\", \"weight\": -1.09712643, \"cname\": \"SERPINI1\", \"id\": [\"Gene\", \"HGNC\", \"SERPINI1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PLAU\", \"description\": \"This gene encodes a secreted serine protease that converts plasminogen to plasmin. The encoded preproprotein is proteolytically processed to generate A and B polypeptide chains. These chains associate via a single disulfide bond to form the catalytically inactive high molecular weight urokinase-type plasminogen activator (HMW-uPA). HMW-uPA can be further processed into the catalytically active low molecular weight urokinase-type plasminogen activator (LMW-uPA). This low molecular weight form does not bind to the urokinase-type plasminogen activator receptor. Mutations in this gene may be associated with Quebec platelet disorder and late-onset Alzheimer's disease. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Jan 2016]\", \"label\": \"plasminogen activator, urokinase\", \"weight\": -0.44174472, \"cname\": \"PLAU\", \"id\": [\"Gene\", \"HGNC\", \"PLAU\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TTR\", \"description\": \"This gene encodes transthyretin, one of the three prealbumins including alpha-1-antitrypsin, transthyretin and orosomucoid. Transthyretin is a carrier protein; it transports thyroid hormones in the plasma and cerebrospinal fluid, and also transports retinol (vitamin A) in the plasma. The protein consists of a tetramer of identical subunits. More than 80 different mutations in this gene have been reported; most mutations are related to amyloid deposition, affecting predominantly peripheral nerve and/or the heart, and a small portion of the gene mutations is non-amyloidogenic. The diseases caused by mutations include amyloidotic polyneuropathy, euthyroid hyperthyroxinaemia, amyloidotic vitreous opacities, cardiomyopathy, oculoleptomeningeal amyloidosis, meningocerebrovascular amyloidosis, carpal tunnel syndrome, etc. [provided by RefSeq, Jan 2009]\", \"label\": \"transthyretin\", \"weight\": 4.07054127, \"cname\": \"TTR\", \"id\": [\"Gene\", \"HGNC\", \"TTR\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL4\", \"description\": \"The protein encoded by this gene is a pleiotropic cytokine produced by activated T cells. This cytokine is a ligand for interleukin 4 receptor. The interleukin 4 receptor also binds to IL13, which may contribute to many overlapping functions of this cytokine and IL13. STAT6, a signal transducer and activator of transcription, has been shown to play a central role in mediating the immune regulatory signal of this cytokine. This gene, IL3, IL5, IL13, and CSF2 form a cytokine gene cluster on chromosome 5q, with this gene particularly close to IL13. This gene, IL13 and IL5 are found to be regulated coordinately by several long-range regulatory elements in an over 120 kilobase range on the chromosome. Two alternatively spliced transcript variants of this gene encoding distinct isoforms have been reported. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 4\", \"weight\": 0.91448898, \"cname\": \"IL4\", \"id\": [\"Gene\", \"HGNC\", \"IL4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APPBP2\", \"description\": \"The protein encoded by this gene interacts with microtubules and is functionally associated with beta-amyloid precursor protein transport and/or processing. The beta-amyloid precursor protein is a cell surface protein with signal-transducing properties, and it is thought to play a role in the pathogenesis of Alzheimer's disease. The encoded protein may be involved in regulating cell death. This gene has been found to be highly expressed in breast cancer. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Sep 2013]\", \"label\": \"amyloid beta precursor protein binding protein 2\", \"weight\": -0.01975785, \"cname\": \"APPBP2\", \"id\": [\"Gene\", \"HGNC\", \"APPBP2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PAWR\", \"description\": \"The tumor suppressor WT1 represses and activates transcription. The protein encoded by this gene is a WT1-interacting protein that itself functions as a transcriptional repressor. It contains a putative leucine zipper domain which interacts with the zinc finger DNA binding domain of WT1. This protein is specifically upregulated during apoptosis of prostate cells. [provided by RefSeq, Jul 2008]\", \"label\": \"pro-apoptotic WT1 regulator\", \"weight\": -0.17425334, \"cname\": \"PAWR\", \"id\": [\"Gene\", \"HGNC\", \"PAWR\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CSNK1A1\", \"description\": null, \"label\": \"casein kinase 1 alpha 1\", \"weight\": -0.09186504, \"cname\": \"CSNK1A1\", \"id\": [\"Gene\", \"HGNC\", \"CSNK1A1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"QPCT\", \"description\": \"This gene encodes human pituitary glutaminyl cyclase, which is responsible for the presence of pyroglutamyl residues in many neuroendocrine peptides. The amino acid sequence of this enzyme is 86% identical to that of bovine glutaminyl cyclase. [provided by RefSeq, Jul 2008]\", \"label\": \"glutaminyl-peptide cyclotransferase\", \"weight\": 0.51242099, \"cname\": \"QPCT\", \"id\": [\"Gene\", \"HGNC\", \"QPCT\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"UBE2I\", \"description\": \"The modification of proteins with ubiquitin is an important cellular mechanism for targeting abnormal or short-lived proteins for degradation. Ubiquitination involves at least three classes of enzymes: ubiquitin-activating enzymes, or E1s, ubiquitin-conjugating enzymes, or E2s, and ubiquitin-protein ligases, or E3s. This gene encodes a member of the E2 ubiquitin-conjugating enzyme family. Four alternatively spliced transcript variants encoding the same protein have been found for this gene. [provided by RefSeq, Jul 2008]\", \"label\": \"ubiquitin conjugating enzyme E2 I\", \"weight\": -0.15740177, \"cname\": \"UBE2I\", \"id\": [\"Gene\", \"HGNC\", \"UBE2I\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"RANBP9\", \"description\": \"This gene encodes a protein that binds RAN, a small GTP binding protein belonging to the RAS superfamily that is essential for the translocation of RNA and proteins through the nuclear pore complex. The protein encoded by this gene has also been shown to interact with several other proteins, including met proto-oncogene, homeodomain interacting protein kinase 2, androgen receptor, and cyclin-dependent kinase 11. [provided by RefSeq, Jul 2008]\", \"label\": \"RAN binding protein 9\", \"weight\": -0.20181786, \"cname\": \"RANBP9\", \"id\": [\"Gene\", \"HGNC\", \"RANBP9\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CAPN2\", \"description\": \"The calpains, calcium-activated neutral proteases, are nonlysosomal, intracellular cysteine proteases. The mammalian calpains include ubiquitous, stomach-specific, and muscle-specific proteins. The ubiquitous enzymes consist of heterodimers with distinct large, catalytic subunits associated with a common small, regulatory subunit. This gene encodes the large subunit of the ubiquitous enzyme, calpain 2. Multiple heterogeneous transcriptional start sites in the 5' UTR have been reported. Two transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Mar 2009]\", \"label\": \"calpain 2\", \"weight\": -0.02208264, \"cname\": \"CAPN2\", \"id\": [\"Gene\", \"HGNC\", \"CAPN2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"LRRTM3\", \"description\": null, \"label\": \"leucine rich repeat transmembrane neuronal 3\", \"weight\": 0, \"cname\": \"LRRTM3\", \"id\": [\"Gene\", \"HGNC\", \"LRRTM3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"RTN4R\", \"description\": \"This gene encodes the receptor for reticulon 4, oligodendrocyte myelin glycoprotein and myelin-associated glycoprotein. This receptor mediates axonal growth inhibition and may play a role in regulating axonal regeneration and plasticity in the adult central nervous system. [provided by RefSeq, Jul 2008]\", \"label\": \"reticulon 4 receptor\", \"weight\": -1.01416883, \"cname\": \"RTN4R\", \"id\": [\"Gene\", \"HGNC\", \"RTN4R\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"HSPA1B\", \"description\": \"This intronless gene encodes a 70kDa heat shock protein which is a member of the heat shock protein 70 family. In conjuction with other heat shock proteins, this protein stabilizes existing proteins against aggregation and mediates the folding of newly translated proteins in the cytosol and in organelles. It is also involved in the ubiquitin-proteasome pathway through interaction with the AU-rich element RNA-binding protein 1. The gene is located in the major histocompatibility complex class III region, in a cluster with two closely related genes which encode similar proteins. [provided by RefSeq, Jul 2008]\", \"label\": \"heat shock protein family A (Hsp70) member 1B\", \"weight\": 0, \"cname\": \"HSPA1B\", \"id\": [\"Gene\", \"HGNC\", \"HSPA1B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SP1\", \"description\": \"The protein encoded by this gene is a zinc finger transcription factor that binds to GC-rich motifs of many promoters. The encoded protein is involved in many cellular processes, including cell differentiation, cell growth, apoptosis, immune responses, response to DNA damage, and chromatin remodeling. Post-translational modifications such as phosphorylation, acetylation, glycosylation, and proteolytic processing significantly affect the activity of this protein, which can be an activator or a repressor. Three transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Nov 2014]\", \"label\": \"Sp1 transcription factor\", \"weight\": -0.08678866, \"cname\": \"SP1\", \"id\": [\"Gene\", \"HGNC\", \"SP1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"S100A9\", \"description\": \"The protein encoded by this gene is a member of the S100 family of proteins containing 2 EF-hand calcium-binding motifs. S100 proteins are localized in the cytoplasm and/or nucleus of a wide range of cells, and involved in the regulation of a number of cellular processes such as cell cycle progression and differentiation. S100 genes include at least 13 members which are located as a cluster on chromosome 1q21. This protein may function in the inhibition of casein kinase and altered expression of this protein is associated with the disease cystic fibrosis. This antimicrobial protein exhibits antifungal and antibacterial activity. [provided by RefSeq, Nov 2014]\", \"label\": \"S100 calcium binding protein A9\", \"weight\": -0.60784743, \"cname\": \"S100A9\", \"id\": [\"Gene\", \"HGNC\", \"S100A9\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"HTR4\", \"description\": \"This gene is a member of the family of serotonin receptors, which are G protein coupled receptors that stimulate cAMP production in response to serotonin (5-hydroxytryptamine). The gene product is a glycosylated transmembrane protein that functions in both the peripheral and central nervous system to modulate the release of various neurotransmitters. Multiple transcript variants encoding proteins with distinct C-terminal sequences have been described. [provided by RefSeq, May 2010]\", \"label\": \"5-hydroxytryptamine receptor 4\", \"weight\": -0.15785455, \"cname\": \"HTR4\", \"id\": [\"Gene\", \"HGNC\", \"HTR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ALOX5\", \"description\": \"This gene encodes a member of the lipoxygenase gene family and plays a dual role in the synthesis of leukotrienes from arachidonic acid. The encoded protein, which is expressed specifically in bone marrow-derived cells, catalyzes the conversion of arachidonic acid to 5(S)-hydroperoxy-6-trans-8,11,14-cis-eicosatetraenoic acid, and further to the allylic epoxide 5(S)-trans-7,9-trans-11,14-cis-eicosatetrenoic acid (leukotriene A4). Leukotrienes are important mediators of a number of inflammatory and allergic conditions. Mutations in the promoter region of this gene lead to a diminished response to antileukotriene drugs used in the treatment of asthma and may also be associated with atherosclerosis and several cancers. Alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012]\", \"label\": \"arachidonate 5-lipoxygenase\", \"weight\": -0.188697, \"cname\": \"ALOX5\", \"id\": [\"Gene\", \"HGNC\", \"ALOX5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MIR125B1\", \"description\": \"microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]\", \"label\": \"microRNA 125b-1\", \"weight\": 0, \"cname\": \"MIR125B1\", \"id\": [\"Gene\", \"HGNC\", \"MIR125B1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MIR125B2\", \"description\": \"microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]\", \"label\": \"microRNA 125b-2\", \"weight\": 0, \"cname\": \"MIR125B2\", \"id\": [\"Gene\", \"HGNC\", \"MIR125B2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"RTN4\", \"description\": \"This gene belongs to the family of reticulon encoding genes. Reticulons are associated with the endoplasmic reticulum, and are involved in neuroendocrine secretion or in membrane trafficking in neuroendocrine cells. The product of this gene is a potent neurite outgrowth inhibitor which may also help block the regeneration of the central nervous system in higher vertebrates. Alternatively spliced transcript variants derived both from differential splicing and differential promoter usage and encoding different isoforms have been identified. [provided by RefSeq, Jul 2008]\", \"label\": \"reticulon 4\", \"weight\": 0.11441, \"cname\": \"RTN4\", \"id\": [\"Gene\", \"HGNC\", \"RTN4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MME\", \"description\": \"The protein encoded by this gene is a type II transmembrane glycoprotein and a common acute lymphocytic leukemia antigen that is an important cell surface marker in the diagnosis of human acute lymphocytic leukemia (ALL). The encoded protein is present on leukemic cells of pre-B phenotype, which represent 85% of cases of ALL. This protein is not restricted to leukemic cells, however, and is found on a variety of normal tissues. The protein is a neutral endopeptidase that cleaves peptides at the amino side of hydrophobic residues and inactivates several peptide hormones including glucagon, enkephalins, substance P, neurotensin, oxytocin, and bradykinin. [provided by RefSeq, Aug 2017]\", \"label\": \"membrane metalloendopeptidase\", \"weight\": 0.036129400000000006, \"cname\": \"MME\", \"id\": [\"Gene\", \"HGNC\", \"MME\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"BDNF\", \"description\": \"This gene encodes a member of the nerve growth factor family of proteins. Alternative splicing results in multiple transcript variants, at least one of which encodes a preproprotein that is proteolytically processed to generate the mature protein. Binding of this protein to its cognate receptor promotes neuronal survival in the adult brain. Expression of this gene is reduced in Alzheimer's, Parkinson's, and Huntington's disease patients. This gene may play a role in the regulation of the stress response and in the biology of mood disorders. [provided by RefSeq, Nov 2015]\", \"label\": \"brain derived neurotrophic factor\", \"weight\": -0.39074377, \"cname\": \"BDNF\", \"id\": [\"Gene\", \"HGNC\", \"BDNF\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CST3\", \"description\": \"The cystatin superfamily encompasses proteins that contain multiple cystatin-like sequences. Some of the members are active cysteine protease inhibitors, while others have lost or perhaps never acquired this inhibitory activity. There are three inhibitory families in the superfamily, including the type 1 cystatins (stefins), type 2 cystatins and the kininogens. The type 2 cystatin proteins are a class of cysteine proteinase inhibitors found in a variety of human fluids and secretions, where they appear to provide protective functions. The cystatin locus on chromosome 20 contains the majority of the type 2 cystatin genes and pseudogenes. This gene is located in the cystatin locus and encodes the most abundant extracellular inhibitor of cysteine proteases, which is found in high concentrations in biological fluids and is expressed in virtually all organs of the body. A mutation in this gene has been associated with amyloid angiopathy. Expression of this protein in vascular wall smooth muscle cells is severely reduced in both atherosclerotic and aneurysmal aortic lesions, establishing its role in vascular disease. In addition, this protein has been shown to have an antimicrobial function, inhibiting the replication of herpes simplex virus. Alternative splicing results in multiple transcript variants encoding a single protein. [provided by RefSeq, Nov 2014]\", \"label\": \"cystatin C\", \"weight\": -0.03303922, \"cname\": \"CST3\", \"id\": [\"Gene\", \"HGNC\", \"CST3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ABCA7\", \"description\": \"The protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This protein is a member of the ABC1 subfamily. Members of the ABC1 subfamily comprise the only major ABC subfamily found exclusively in multicellular eukaryotes. This full transporter has been detected predominantly in myelo-lymphatic tissues with the highest expression in peripheral leukocytes, thymus, spleen, and bone marrow. The function of this protein is not yet known; however, the expression pattern suggests a role in lipid homeostasis in cells of the immune system. [provided by RefSeq, Jul 2008]\", \"label\": \"ATP binding cassette subfamily A member 7\", \"weight\": -0.98234365, \"cname\": \"ABCA7\", \"id\": [\"Gene\", \"HGNC\", \"ABCA7\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"LRP2\", \"description\": \"The protein encoded by this gene, low density lipoprotein-related protein 2 (LRP2) or megalin, is a multi-ligand endocytic receptor that is expressed in many different tissues but primarily in absorptive epithilial tissues such as the kidney. This glycoprotein has a large amino-terminal extracellular domain, a single transmembrane domain, and a short carboxy-terminal cytoplasmic tail. The extracellular ligand-binding-domains bind diverse macromolecules including albumin, apolipoproteins B and E, and lipoprotein lipase. The LRP2 protein is critical for the reuptake of numerous ligands, including lipoproteins, sterols, vitamin-binding proteins, and hormones. This protein also has a role in cell-signaling; extracellular ligands include parathyroid horomones and the morphogen sonic hedgehog while cytosolic ligands include MAP kinase scaffold proteins and JNK interacting proteins. Recycling of this membrane receptor is regulated by phosphorylation of its cytoplasmic domain. Mutations in this gene cause Donnai-Barrow syndrome (DBS) and facio-oculoacoustico-renal syndrome (FOAR).[provided by RefSeq, Aug 2009]\", \"label\": \"LDL receptor related protein 2\", \"weight\": -0.13257198, \"cname\": \"LRP2\", \"id\": [\"Gene\", \"HGNC\", \"LRP2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GRIN3A\", \"description\": \"This gene encodes a subunit of the N-methyl-D-aspartate (NMDA) receptors, which belong to the superfamily of glutamate-regulated ion channels, and function in physiological and pathological processes in the central nervous system. This subunit shows greater than 90% identity to the corresponding subunit in rat. Studies in the knockout mouse deficient in this subunit suggest that this gene may be involved in the development of synaptic elements by modulating NMDA receptor activity. [provided by RefSeq, Jul 2008]\", \"label\": \"glutamate ionotropic receptor NMDA type subunit 3A\", \"weight\": -0.01551801, \"cname\": \"GRIN3A\", \"id\": [\"Gene\", \"HGNC\", \"GRIN3A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"LRP1\", \"description\": \"This gene encodes a member of the low-density lipoprotein receptor family of proteins. The encoded preproprotein is proteolytically processed by furin to generate 515 kDa and 85 kDa subunits that form the mature receptor (PMID: 8546712). This receptor is involved in several cellular processes, including intracellular signaling, lipid homeostasis, and clearance of apoptotic cells. In addition, the encoded protein is necessary for the alpha 2-macroglobulin-mediated clearance of secreted amyloid precursor protein and beta-amyloid, the main component of amyloid plaques found in Alzheimer patients. Expression of this gene decreases with age and has been found to be lower than controls in brain tissue from Alzheimer's disease patients. [provided by RefSeq, Oct 2015]\", \"label\": \"LDL receptor related protein 1\", \"weight\": 0.39313774, \"cname\": \"LRP1\", \"id\": [\"Gene\", \"HGNC\", \"LRP1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MIR195\", \"description\": \"microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2009]\", \"label\": \"microRNA 195\", \"weight\": 0, \"cname\": \"MIR195\", \"id\": [\"Gene\", \"HGNC\", \"MIR195\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CASP8\", \"description\": \"This gene encodes a member of the cysteine-aspartic acid protease (caspase) family. Sequential activation of caspases plays a central role in the execution-phase of cell apoptosis. Caspases exist as inactive proenzymes composed of a prodomain, a large protease subunit, and a small protease subunit. Activation of caspases requires proteolytic processing at conserved internal aspartic residues to generate a heterodimeric enzyme consisting of the large and small subunits. This protein is involved in the programmed cell death induced by Fas and various apoptotic stimuli. The N-terminal FADD-like death effector domain of this protein suggests that it may interact with Fas-interacting protein FADD. This protein was detected in the insoluble fraction of the affected brain region from Huntington disease patients but not in those from normal controls, which implicated the role in neurodegenerative diseases. Many alternatively spliced transcript variants encoding different isoforms have been described, although not all variants have had their full-length sequences determined. [provided by RefSeq, Jul 2008]\", \"label\": \"caspase 8\", \"weight\": -0.15029163, \"cname\": \"CASP8\", \"id\": [\"Gene\", \"HGNC\", \"CASP8\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CRH\", \"description\": \"This gene encodes a member of the corticotropin-releasing factor family. The encoded preproprotein is proteolytically processed to generate the mature neuropeptide hormone. In response to stress, this hormone is secreted by the paraventricular nucleus (PVN) of the hypothalamus, binds to corticotropin releasing hormone receptors and stimulates the release of adrenocorticotropic hormone from the pituitary gland. Marked reduction in this protein has been observed in association with Alzheimer's disease. Autosomal recessive hypothalamic corticotropin deficiency has multiple and potentially fatal metabolic consequences including hypoglycemia and hepatitis. In addition to production in the hypothalamus, this protein is also synthesized in peripheral tissues, such as T lymphocytes, and is highly expressed in the placenta. In the placenta it is a marker that determines the length of gestation and the timing of parturition and delivery. A rapid increase in circulating levels of the hormone occurs at the onset of parturition, suggesting that, in addition to its metabolic functions, this protein may act as a trigger for parturition. [provided by RefSeq, Nov 2015]\", \"label\": \"corticotropin releasing hormone\", \"weight\": -0.2141471, \"cname\": \"CRH\", \"id\": [\"Gene\", \"HGNC\", \"CRH\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"HSPB8\", \"description\": \"The protein encoded by this gene belongs to the superfamily of small heat-shock proteins containing a conservative alpha-crystallin domain at the C-terminal part of the molecule. The expression of this gene in induced by estrogen in estrogen receptor-positive breast cancer cells, and this protein also functions as a chaperone in association with Bag3, a stimulator of macroautophagy. Thus, this gene appears to be involved in regulation of cell proliferation, apoptosis, and carcinogenesis, and mutations in this gene have been associated with different neuromuscular diseases, including Charcot-Marie-Tooth disease. [provided by RefSeq, Jul 2008]\", \"label\": \"heat shock protein family B (small) member 8\", \"weight\": 0.39864903, \"cname\": \"HSPB8\", \"id\": [\"Gene\", \"HGNC\", \"HSPB8\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APBA3\", \"description\": \"The protein encoded by this gene is a member of the X11 protein family. It is an adapter protein that interacts with the Alzheimer's disease amyloid precursor protein. This gene product is believed to be involved in signal transduction processes. This gene is a candidate gene for Alzheimer's disease. [provided by RefSeq, Jul 2008]\", \"label\": \"amyloid beta precursor protein binding family A member 3\", \"weight\": -1.34229521, \"cname\": \"APBA3\", \"id\": [\"Gene\", \"HGNC\", \"APBA3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"description\": \"This gene encodes a member of the N-methyl-D-aspartate (NMDA) receptor family within the ionotropic glutamate receptor superfamily. The encoded protein is a subunit of the NMDA receptor ion channel which acts as an agonist binding site for glutamate. The NMDA receptors mediate a slow calcium-permeable component of excitatory synaptic transmission in the central nervous system. The NMDA receptors are heterotetramers of seven genetically encoded, differentially expressed subunits including NR1 (GRIN1), NR2 (GRIN2A, GRIN2B, GRIN2C, or GRIN2D) and NR3 (GRIN3A or GRIN3B). The early expression of this gene in development suggests a role in brain development, circuit formation, synaptic plasticity, and cellular migration and differentiation. Naturally occurring mutations within this gene are associated with neurodevelopmental disorders including autism spectrum disorder, attention deficit hyperactivity disorder, epilepsy, and schizophrenia. [provided by RefSeq, Aug 2017]\", \"label\": \"glutamate ionotropic receptor NMDA type subunit 2B\", \"weight\": -0.10430164, \"cname\": \"GRIN2B\", \"id\": [\"Gene\", \"HGNC\", \"GRIN2B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GRIN2A\", \"description\": null, \"weight\": -0.29373059999999995, \"cname\": \"GRIN2A\", \"id\": [\"Gene\", \"HGNC\", \"GRIN2A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA4\", \"description\": \"This gene encodes a nicotinic acetylcholine receptor, which belongs to a superfamily of ligand-gated ion channels that play a role in fast signal transmission at synapses. These pentameric receptors can bind acetylcholine, which causes an extensive change in conformation that leads to the opening of an ion-conducting channel across the plasma membrane. This protein is an integral membrane receptor subunit that can interact with either nAChR beta-2 or nAChR beta-4 to form a functional receptor. Mutations in this gene cause nocturnal frontal lobe epilepsy type 1. Polymorphisms in this gene that provide protection against nicotine addiction have been described. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2012]\", \"label\": \"cholinergic receptor nicotinic alpha 4 subunit\", \"weight\": 0.57478705, \"cname\": \"CHRNA4\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TGFB2\", \"description\": \"This gene encodes a secreted ligand of the TGF-beta (transforming growth factor-beta) superfamily of proteins. Ligands of this family bind various TGF-beta receptors leading to recruitment and activation of SMAD family transcription factors that regulate gene expression. The encoded preproprotein is proteolytically processed to generate a latency-associated peptide (LAP) and a mature peptide, and is found in either a latent form composed of a mature peptide homodimer, a LAP homodimer, and a latent TGF-beta binding protein, or in an active form consisting solely of the mature peptide homodimer. The mature peptide may also form heterodimers with other TGF-beta family members. Disruption of the TGF-beta/SMAD pathway has been implicated in a variety of human cancers. A chromosomal translocation that includes this gene is associated with Peters' anomaly, a congenital defect of the anterior chamber of the eye. Mutations in this gene may be associated with Loeys-Dietz syndrome. This gene encodes multiple isoforms that may undergo similar proteolytic processing. [provided by RefSeq, Aug 2016]\", \"label\": \"transforming growth factor beta 2\", \"weight\": 0.26765361, \"cname\": \"TGFB2\", \"id\": [\"Gene\", \"HGNC\", \"TGFB2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GRIN2C\", \"description\": \"This gene encodes a subunit of the N-methyl-D-aspartate (NMDA) receptor, which is a subtype of ionotropic glutamate receptor. NMDA receptors are found in the central nervous system, are permeable to cations and have an important role in physiological processes such as learning, memory, and synaptic development. The receptor is a tetramer of different subunits (typically heterodimer of subunit 1 with one or more of subunits 2A-D), forming a channel that is permeable to calcium, potassium, and sodium, and whose properties are determined by subunit composition. Alterations in the subunit composition of the receptor are associated with pathophysiological conditions such as Parkinson's disease, Alzheimer's disease, depression, and schizophrenia. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jun 2013]\", \"label\": \"glutamate ionotropic receptor NMDA type subunit 2C\", \"weight\": 0.15747839, \"cname\": \"GRIN2C\", \"id\": [\"Gene\", \"HGNC\", \"GRIN2C\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GRIN2D\", \"description\": \"N-methyl-D-aspartate (NMDA) receptors are a class of ionotropic glutamate receptors. NMDA channel has been shown to be involved in long-term potentiation, an activity-dependent increase in the efficiency of synaptic transmission thought to underlie certain kinds of memory and learning. NMDA receptor channels are heteromers composed of the key receptor subunit NMDAR1 (GRIN1) and 1 or more of the 4 NMDAR2 subunits: NMDAR2A (GRIN2A), NMDAR2B (GRIN2B), NMDAR2C (GRIN2C), and NMDAR2D (GRIN2D). [provided by RefSeq, Mar 2010]\", \"label\": \"glutamate ionotropic receptor NMDA type subunit 2D\", \"weight\": -0.44105144, \"cname\": \"GRIN2D\", \"id\": [\"Gene\", \"HGNC\", \"GRIN2D\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GRIN3B\", \"description\": \"The protein encoded by this gene is a subunit of an N-methyl-D-aspartate (NMDA) receptor. The encoded protein is found primarily in motor neurons, where it forms a heterotetramer with GRIN1 to create an excitatory glycine receptor. Variations in this gene have been proposed to be linked to schizophrenia. [provided by RefSeq, Nov 2015]\", \"label\": \"glutamate ionotropic receptor NMDA type subunit 3B\", \"weight\": 0.55994342, \"cname\": \"GRIN3B\", \"id\": [\"Gene\", \"HGNC\", \"GRIN3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SOD1\", \"description\": \"The protein encoded by this gene binds copper and zinc ions and is one of two isozymes responsible for destroying free superoxide radicals in the body. The encoded isozyme is a soluble cytoplasmic protein, acting as a homodimer to convert naturally-occuring but harmful superoxide radicals to molecular oxygen and hydrogen peroxide. The other isozyme is a mitochondrial protein. Mutations in this gene have been implicated as causes of familial amyotrophic lateral sclerosis. Rare transcript variants have been reported for this gene. [provided by RefSeq, Jul 2008]\", \"label\": \"superoxide dismutase 1\", \"weight\": -0.05539543, \"cname\": \"SOD1\", \"id\": [\"Gene\", \"HGNC\", \"SOD1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA2\", \"description\": \"Nicotinic acetylcholine receptors (nAChRs) are ligand-gated ion channels formed by a pentameric arrangement of alpha and beta subunits to create distinct muscle and neuronal receptors. Neuronal receptors are found throughout the peripheral and central nervous system where they are involved in fast synaptic transmission. This gene encodes an alpha subunit that is widely expressed in the brain. The proposed structure for nAChR subunits is a conserved N-terminal extracellular domain followed by three conserved transmembrane domains, a variable cytoplasmic loop, a fourth conserved transmembrane domain, and a short C-terminal extracellular region. Mutations in this gene cause autosomal dominant nocturnal frontal lobe epilepsy type 4. Single nucleotide polymorphisms (SNPs) in this gene have been associated with nicotine dependence. [provided by RefSeq, Nov 2009]\", \"label\": \"cholinergic receptor nicotinic alpha 2 subunit\", \"weight\": -0.15115052, \"cname\": \"CHRNA2\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA3\", \"description\": \"This locus encodes a member of the nicotinic acetylcholine receptor family of proteins. Members of this family of proteins form pentameric complexes comprised of both alpha and beta subunits. This locus encodes an alpha-type subunit, as it contains characteristic adjacent cysteine residues. The encoded protein is a ligand-gated ion channel that likely plays a role in neurotransmission. Polymorphisms in this gene have been associated with an increased risk of smoking initiation and an increased susceptibility to lung cancer. Alternatively spliced transcript variants have been described. [provided by RefSeq, Nov 2009]\", \"label\": \"cholinergic receptor nicotinic alpha 3 subunit\", \"weight\": -0.6279239999999999, \"cname\": \"CHRNA3\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA5\", \"description\": \"The protein encoded by this gene is a nicotinic acetylcholine receptor subunit and a member of a superfamily of ligand-gated ion channels that mediate fast signal transmission at synapses. These receptors are thought to be heteropentamers composed of separate but similar subunits. Defects in this gene have been linked to susceptibility to lung cancer type 2 (LNCR2).[provided by RefSeq, Jun 2010]\", \"label\": \"cholinergic receptor nicotinic alpha 5 subunit\", \"weight\": 0.4580624, \"cname\": \"CHRNA5\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA6\", \"description\": \"This gene encodes an alpha subunit of neuronal nicotinic acetylcholine receptors. These receptors consist of five subunits and function as ion channels involved in neurotransmission. The encoded protein is a subunit of neuronal nicotinic acetylcholine receptors that mediate dopaminergic neurotransmission and are activated by acetylcholine and exogenous nicotine. Alternatively spliced transcript variants have been observed for this gene. Single nucleotide polymorphisms in this gene have been associated with both nicotine and alcohol dependence. [provided by RefSeq, Dec 2010]\", \"label\": \"cholinergic receptor nicotinic alpha 6 subunit\", \"weight\": -1.13569155, \"cname\": \"CHRNA6\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA6\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA7\", \"description\": \"The nicotinic acetylcholine receptors (nAChRs) are members of a superfamily of ligand-gated ion channels that mediate fast signal transmission at synapses. The nAChRs are thought to be hetero-pentamers composed of homologous subunits. The proposed structure for each subunit is a conserved N-terminal extracellular domain followed by three conserved transmembrane domains, a variable cytoplasmic loop, a fourth conserved transmembrane domain, and a short C-terminal extracellular region. The protein encoded by this gene forms a homo-oligomeric channel, displays marked permeability to calcium ions and is a major component of brain nicotinic receptors that are blocked by, and highly sensitive to, alpha-bungarotoxin. Once this receptor binds acetylcholine, it undergoes an extensive change in conformation that affects all subunits and leads to opening of an ion-conducting channel across the plasma membrane. This gene is located in a region identified as a major susceptibility locus for juvenile myoclonic epilepsy and a chromosomal location involved in the genetic transmission of schizophrenia. An evolutionarily recent partial duplication event in this region results in a hybrid containing sequence from this gene and a novel FAM7A gene. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Feb 2012]\", \"label\": \"cholinergic receptor nicotinic alpha 7 subunit\", \"weight\": 0, \"cname\": \"CHRNA7\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA7\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA9\", \"description\": \"This gene is a member of the ligand-gated ionic channel family and nicotinic acetylcholine receptor gene superfamily. It encodes a plasma membrane protein that forms homo- or hetero-oligomeric divalent cation channels. This protein is involved in cochlea hair cell development and is also expressed in the outer hair cells (OHCs) of the adult cochlea. [provided by RefSeq, Feb 2012]\", \"label\": \"cholinergic receptor nicotinic alpha 9 subunit\", \"weight\": 0.53800658, \"cname\": \"CHRNA9\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA9\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA10\", \"description\": null, \"label\": \"cholinergic receptor nicotinic alpha 10 subunit\", \"weight\": -0.44528092, \"cname\": \"CHRNA10\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA10\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA1\", \"description\": \"The muscle acetylcholine receptor consiststs of 5 subunits of 4 different types: 2 alpha subunits and 1 each of the beta, gamma, and delta subunits. This gene encodes an alpha subunit that plays a role in acetlycholine binding/channel gating. Alternatively spliced transcript variants encoding different isoforms have been identified. [provided by RefSeq, Nov 2012]\", \"label\": \"cholinergic receptor nicotinic alpha 1 subunit\", \"weight\": 0.56461989, \"cname\": \"CHRNA1\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MAPK8IP2\", \"description\": \"This gene encodes a scaffold protein that is thought to be involved in the regulation of the c-Jun amino-terminal kinase signaling pathway. This protein has been shown to interact with and regulate the activity of MAPK8/JNK1 and MAP2K7/MKK7 kinases. [provided by RefSeq, Jun 2017]\", \"label\": \"mitogen-activated protein kinase 8 interacting protein 2\", \"weight\": -0.07187538, \"cname\": \"MAPK8IP2\", \"id\": [\"Gene\", \"HGNC\", \"MAPK8IP2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ABCG1\", \"description\": \"The protein encoded by this gene is a member of the superfamily of ATP-binding cassette (ABC) transporters. ABC proteins transport various molecules across extra- and intra-cellular membranes. ABC genes are divided into seven distinct subfamilies (ABC1, MDR/TAP, MRP, ALD, OABP, GCN20, White). This protein is a member of the White subfamily. It is involved in macrophage cholesterol and phospholipids transport, and may regulate cellular lipid homeostasis in other cell types. Six alternative splice variants have been identified. [provided by RefSeq, Jul 2008]\", \"label\": \"ATP binding cassette subfamily G member 1\", \"weight\": -0.17414933, \"cname\": \"ABCG1\", \"id\": [\"Gene\", \"HGNC\", \"ABCG1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CPQ\", \"description\": \"This gene encodes a metallopeptidase that belongs to the peptidase M28 family. The encoded protein may catalyze the cleavage of dipeptides with unsubstituted terminals into amino acids. [provided by RefSeq, Jul 2013]\", \"label\": \"carboxypeptidase Q\", \"weight\": 0.45456024, \"cname\": \"CPQ\", \"id\": [\"Gene\", \"HGNC\", \"CPQ\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"S100B\", \"description\": \"The protein encoded by this gene is a member of the S100 family of proteins containing 2 EF-hand calcium-binding motifs. S100 proteins are localized in the cytoplasm and/or nucleus of a wide range of cells, and involved in the regulation of a number of cellular processes such as cell cycle progression and differentiation. S100 genes include at least 13 members which are located as a cluster on chromosome 1q21; however, this gene is located at 21q22.3. This protein may function in Neurite extension, proliferation of melanoma cells, stimulation of Ca2+ fluxes, inhibition of PKC-mediated phosphorylation, astrocytosis and axonal proliferation, and inhibition of microtubule assembly. Chromosomal rearrangements and altered expression of this gene have been implicated in several neurological, neoplastic, and other types of diseases, including Alzheimer's disease, Down's syndrome, epilepsy, amyotrophic lateral sclerosis, melanoma, and type I diabetes. [provided by RefSeq, Jul 2008]\", \"label\": \"S100 calcium binding protein B\", \"weight\": 0.27537864, \"cname\": \"S100B\", \"id\": [\"Gene\", \"HGNC\", \"S100B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MMP3\", \"description\": \"Proteins of the matrix metalloproteinase (MMP) family are involved in the breakdown of extracellular matrix in normal physiological processes, such as embryonic development, reproduction, and tissue remodeling, as well as in disease processes, such as arthritis and metastasis. Most MMP's are secreted as inactive proproteins which are activated when cleaved by extracellular proteinases. This gene encodes an enzyme which degrades fibronectin, laminin, collagens III, IV, IX, and X, and cartilage proteoglycans. The enzyme is thought to be involved in wound repair, progression of atherosclerosis, and tumor initiation. The gene is part of a cluster of MMP genes which localize to chromosome 11q22.3. [provided by RefSeq, Jul 2008]\", \"label\": \"matrix metallopeptidase 3\", \"weight\": 0.78577944, \"cname\": \"MMP3\", \"id\": [\"Gene\", \"HGNC\", \"MMP3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GRB2\", \"description\": \"The protein encoded by this gene binds the epidermal growth factor receptor and contains one SH2 domain and two SH3 domains. Its two SH3 domains direct complex formation with proline-rich regions of other proteins, and its SH2 domain binds tyrosine phosphorylated sequences. This gene is similar to the Sem5 gene of C.elegans, which is involved in the signal transduction pathway. Two alternatively spliced transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jul 2008]\", \"label\": \"growth factor receptor bound protein 2\", \"weight\": -0.02552084, \"cname\": \"GRB2\", \"id\": [\"Gene\", \"HGNC\", \"GRB2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTNNB1\", \"description\": \"The protein encoded by this gene is part of a complex of proteins that constitute adherens junctions (AJs). AJs are necessary for the creation and maintenance of epithelial cell layers by regulating cell growth and adhesion between cells. The encoded protein also anchors the actin cytoskeleton and may be responsible for transmitting the contact inhibition signal that causes cells to stop dividing once the epithelial sheet is complete. Finally, this protein binds to the product of the APC gene, which is mutated in adenomatous polyposis of the colon. Mutations in this gene are a cause of colorectal cancer (CRC), pilomatrixoma (PTR), medulloblastoma (MDB), and ovarian cancer. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Aug 2016]\", \"label\": \"catenin beta 1\", \"weight\": 0.03837192, \"cname\": \"CTNNB1\", \"id\": [\"Gene\", \"HGNC\", \"CTNNB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FRAT2\", \"description\": \"The protein encoded by this intronless gene belongs to the GSK-3-binding protein family. Studies show that this protein plays a role as a positive regulator of the WNT signaling pathway. It may be upregulated in tumor progression. [provided by RefSeq, Jul 2008]\", \"label\": \"FRAT2, WNT signaling pathway regulator\", \"weight\": -0.50393273, \"cname\": \"FRAT2\", \"id\": [\"Gene\", \"HGNC\", \"FRAT2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PIK3R5\", \"description\": \"Phosphatidylinositol 3-kinases (PI3Ks) phosphorylate the inositol ring of phosphatidylinositol at the 3-prime position, and play important roles in cell growth, proliferation, differentiation, motility, survival and intracellular trafficking. The PI3Ks are divided into three classes: I, II and III, and only the class I PI3Ks are involved in oncogenesis. This gene encodes the 101 kD regulatory subunit of the class I PI3K gamma complex, which is a dimeric enzyme, consisting of a 110 kD catalytic subunit gamma and a regulatory subunit of either 55, 87 or 101 kD. This protein recruits the catalytic subunit from the cytosol to the plasma membrane through high-affinity interaction with G-beta-gamma proteins. Multiple alternatively spliced transcript variants encoding two distinct isoforms have been found. [provided by RefSeq, Oct 2011]\", \"label\": \"phosphoinositide-3-kinase regulatory subunit 5\", \"weight\": -0.23683106, \"cname\": \"PIK3R5\", \"id\": [\"Gene\", \"HGNC\", \"PIK3R5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MIR146A\", \"description\": \"microRNAs (miRNAs) are short (20-24 nt) non-coding RNAs that are involved in post-transcriptional regulation of gene expression in multicellular organisms by affecting both the stability and translation of mRNAs. miRNAs are transcribed by RNA polymerase II as part of capped and polyadenylated primary transcripts (pri-miRNAs) that can be either protein-coding or non-coding. The primary transcript is cleaved by the Drosha ribonuclease III enzyme to produce an approximately 70-nt stem-loop precursor miRNA (pre-miRNA), which is further cleaved by the cytoplasmic Dicer ribonuclease to generate the mature miRNA and antisense miRNA star (miRNA*) products. The mature miRNA is incorporated into a RNA-induced silencing complex (RISC), which recognizes target mRNAs through imperfect base pairing with the miRNA and most commonly results in translational inhibition or destabilization of the target mRNA. Some of the targets of the encoded miRNA are the transcripts for tumor necrosis factor, interleukin 1 receptor-associated kinase 1, interleukin 1-beta, TNF receptor-associated factor 6, and complement factor H. The RefSeq represents the predicted microRNA stem-loop. [provided by RefSeq, Sep 2015]\", \"label\": \"microRNA 146a\", \"weight\": -0.53979604, \"cname\": \"MIR146A\", \"id\": [\"Gene\", \"HGNC\", \"MIR146A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NFKBIA\", \"description\": \"This gene encodes a member of the NF-kappa-B inhibitor family, which contain multiple ankrin repeat domains. The encoded protein interacts with REL dimers to inhibit NF-kappa-B/REL complexes which are involved in inflammatory responses. The encoded protein moves between the cytoplasm and the nucleus via a nuclear localization signal and CRM1-mediated nuclear export. Mutations in this gene have been found in ectodermal dysplasia anhidrotic with T-cell immunodeficiency autosomal dominant disease. [provided by RefSeq, Aug 2011]\", \"label\": \"NFKB inhibitor alpha\", \"weight\": 0.36591919, \"cname\": \"NFKBIA\", \"id\": [\"Gene\", \"HGNC\", \"NFKBIA\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"HMGB1\", \"description\": \"This gene encodes a protein that belongs to the High Mobility Group-box superfamily. The encoded non-histone, nuclear DNA-binding protein regulates transcription, and is involved in organization of DNA. This protein plays a role in several cellular processes, including inflammation, cell differentiation and tumor cell migration. Multiple pseudogenes of this gene have been identified. Alternative splicing results in multiple transcript variants that encode the same protein. [provided by RefSeq, Sep 2015]\", \"label\": \"high mobility group box 1\", \"weight\": -0.09432351, \"cname\": \"HMGB1\", \"id\": [\"Gene\", \"HGNC\", \"HMGB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CEBPD\", \"description\": \"The protein encoded by this intronless gene is a bZIP transcription factor which can bind as a homodimer to certain DNA regulatory regions. It can also form heterodimers with the related protein CEBP-alpha. The encoded protein is important in the regulation of genes involved in immune and inflammatory responses, and may be involved in the regulation of genes associated with activation and/or differentiation of macrophages. The cytogenetic location of this locus has been reported as both 8p11 and 8q11. [provided by RefSeq, Sep 2010]\", \"label\": \"CCAAT/enhancer binding protein delta\", \"weight\": 0.47557886, \"cname\": \"CEBPD\", \"id\": [\"Gene\", \"HGNC\", \"CEBPD\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"STAT3\", \"description\": \"The protein encoded by this gene is a member of the STAT protein family. In response to cytokines and growth factors, STAT family members are phosphorylated by the receptor associated kinases, and then form homo- or heterodimers that translocate to the cell nucleus where they act as transcription activators. This protein is activated through phosphorylation in response to various cytokines and growth factors including IFNs, EGF, IL5, IL6, HGF, LIF and BMP2. This protein mediates the expression of a variety of genes in response to cell stimuli, and thus plays a key role in many cellular processes such as cell growth and apoptosis. The small GTPase Rac1 has been shown to bind and regulate the activity of this protein. PIAS3 protein is a specific inhibitor of this protein. Mutations in this gene are associated with infantile-onset multisystem autoimmune disease and hyper-immunoglobulin E syndrome. Alternative splicing results in multiple transcript variants encoding distinct isoforms. [provided by RefSeq, Sep 2015]\", \"label\": \"signal transducer and activator of transcription 3\", \"weight\": 0.04651478, \"cname\": \"STAT3\", \"id\": [\"Gene\", \"HGNC\", \"STAT3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"OSM\", \"description\": \"This gene encodes a member of the leukemia inhibitory factor/oncostatin-M (LIF/OSM) family of proteins. The encoded preproprotein is proteolytically processed to generate the mature protein. This protein is a secreted cytokine and growth regulator that inhibits the proliferation of a number of tumor cell lines. This protein also regulates the production of other cytokines, including interleukin 6, granulocyte-colony stimulating factor and granulocyte-macrophage colony stimulating factor in endothelial cells. This gene and the related gene, leukemia inhibitory factor, also present on chromosome 22, may have resulted from the duplication of a common ancestral gene. Alternative splicing results in multiple transcript variants, at least one of which encodes an isoform that is proteolytically processed. [provided by RefSeq, Jan 2016]\", \"label\": \"oncostatin M\", \"weight\": 0.64942734, \"cname\": \"OSM\", \"id\": [\"Gene\", \"HGNC\", \"OSM\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"The serum IL-1β and TNF-Î± level were significantly increased, and the expressions of TLR4 and NF-ÎºB p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of developmental neuroscience : the official journal of the International Society for Developmental Neuroscience\", \"reference\": \"21515354\", \"authors\": [\"Ma WW\", \"Xiao R\", \"Zhou X\", \"He LL\", \"Ding BJ\", \"Feng JF\", \"Yu HL\", \"Yuan LH\"], \"date\": \"2011-08-01\"}, \"line\": 15007, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"MeSHAnatomy\": \"Serum\"}, \"source\": 0, \"target\": 23, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2746, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2746, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2749, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 8, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2749, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 8, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 8, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 1, \"target\": 8, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 8, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 23756, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 1, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24348, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 1, \"target\": 15, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24348, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 1, \"target\": 15, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1192, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 18, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1216, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 19, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1217, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 20, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1218, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 21, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis : an international journal on programmed cell death\", \"reference\": \"16532272\", \"authors\": [\"Yu MS\", \"Suen KC\", \"Kwok NS\", \"So KF\", \"Hugon J\", \"Chang RC\"], \"date\": \"2006-05-01\"}, \"line\": 20045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 26, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis : an international journal on programmed cell death\", \"reference\": \"16532272\", \"authors\": [\"Yu MS\", \"Suen KC\", \"Kwok NS\", \"So KF\", \"Hugon J\", \"Chang RC\"], \"date\": \"2006-05-01\"}, \"line\": 20045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 1, \"target\": 26, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS &amp; neurological disorders drug targets\", \"reference\": \"24059322\", \"authors\": [\"Rashid S\", \"Kamal MA\", \"ul Ain Q\", \"Nawaz MS\", \"Greig NH\"], \"date\": \"2014-03-01\"}, \"line\": 21087, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37424, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1β could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptosis in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGFβ1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGFβ2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37434, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroimmunology\", \"reference\": \"18353443\", \"authors\": [\"Choi K\", \"Zhuang H\", \"Crain B\", \"Doré S\"], \"date\": \"2008-03-01\"}, \"line\": 37319, \"annotations\": {\"Subgraph\": \"Prostaglandin subgraph\"}, \"source\": 4, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41420, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41420, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 4, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"8287928\", \"authors\": [\"Perry G\", \"Wujek JR\", \"DeWitt DA\", \"McKeon RJ\", \"Silver J\", \"Canning DR\", \"Frederickson RC\"], \"date\": \"1993-12-01\"}, \"line\": 41576, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 7, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"8287928\", \"authors\": [\"Perry G\", \"Wujek JR\", \"DeWitt DA\", \"McKeon RJ\", \"Silver J\", \"Canning DR\", \"Frederickson RC\"], \"date\": \"1993-12-01\"}, \"line\": 41576, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 4, \"target\": 7, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 33907, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 33913, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37595, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37601, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 16, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37700, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 17, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37597, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 18, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37599, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 20, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37698, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 22, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37585, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 23, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37589, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 23, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37583, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 24, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37711, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 24, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37707, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 27, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37591, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 28, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37587, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 29, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37593, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 30, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37709, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 32, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of cellular and molecular medicine\", \"reference\": \"18494933\", \"authors\": [\"Esposito G\", \"Scuderi C\", \"Lu J\", \"Savani C\", \"De Filippis D\", \"Iuvone T\", \"Steardo L Jr\", \"Sheen V\", \"Steardo L\"], \"date\": \"2008-06-01\"}, \"line\": 37629, \"annotations\": {}, \"source\": 5, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"16822591\", \"authors\": [\"Helbecque N\", \"Cottel D\", \"Hermant X\", \"Amouyel P\"], \"date\": \"2007-08-01\"}, \"line\": 18450, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20413894\", \"authors\": [\"Wang HM\", \"Zhao YX\", \"Zhang S\", \"Liu GD\", \"Kang WY\", \"Tang HD\", \"Ding JQ\", \"Chen SD\"], \"date\": \"2010-01-01\"}, \"line\": 41354, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3529, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3529, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 6121, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 6121, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34666, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 7, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34666, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"APOE subgraph\"}, \"source\": 7, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In addition, primary human astrocytes stimulated with the RIG-1 ligand 5'ppp RNA showed increased expression of amyloid precursor protein (APP) and amyloid-β (Abeta), supporting the idea that RIG-1 is involved in the pathology of MCI associated with early progression to AD.These findings suggest that RIG-1 may play a critical role in incipient AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"24694234\", \"authors\": [\"Brand FJ 3rd\", \"Keane RW\", \"de Rivero Vaccari JP\", \"Dietrich WD\", \"Mash DC\", \"Sedaghat C\"], \"date\": \"2014-04-02\"}, \"line\": 38742, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\", \"MeSHAnatomy\": \"Astrocytes\"}, \"source\": 7, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41421, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 7, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41421, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 7, \"target\": 4, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of virology\", \"reference\": \"19710140\", \"authors\": [\"Tribouillard-Tanvier D\", \"Striebel JF\", \"Peterson KE\", \"Chesebro B\"], \"date\": \"2009-11-01\"}, \"line\": 4834, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 7, \"target\": 22, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 8, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 8, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reviews in the neurosciences\", \"reference\": \"22718609\", \"authors\": [\"Liu Y\", \"Gao C\", \"Hölscher C\", \"Li L\"], \"date\": \"2011-12-21\"}, \"line\": 11905, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reviews in the neurosciences\", \"reference\": \"22718609\", \"authors\": [\"Liu Y\", \"Gao C\", \"Hölscher C\", \"Li L\"], \"date\": \"2011-12-21\"}, \"line\": 11905, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" We found that bilateral microinjection of amyloid beta (Abeta)1-40 fibrils into the hippocampal CA1 area of resulted in significant upregulation of CX3CR1 messenger RNA (mRNA) and protein expression (via increasing histone H3 acetylation in the Cx3cr1 promoter region), synaptic dysfunction, and cognitive impairment,\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"23855980\", \"authors\": [\"Naguib M\", \"Bie B\", \"Xu JJ\", \"Brown DL\", \"Yang H\", \"Wu J\"], \"date\": \"2013-12-01\"}, \"line\": 44274, \"annotations\": {\"MeSHAnatomy\": \"Hippocampus\", \"Encode_Feature_Types\": \"Promoter\"}, \"source\": 9, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Recently, a novel variant in the gene encoding the triggering receptor expressed on myeloid cells 2 (TREM2) has been identified that has refocused the spotlight back onto inflammation as a major contributing factor in AD. Variants in TREM2 triple one's risk of developing late-onset AD. TREM2 is expressed on microglial cells, the resident macrophages in the CNS, and functions to stimulate phagocytosis on one hand and to suppress cytokine production and inflammation on the other hand. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The New England journal of medicine\", \"reference\": \"23150934\", \"authors\": [\"Rademakers R\", \"Cruchaga C\", \"Guerreiro R\", \"Williams J\", \"Pocock J\", \"Lashley T\", \"Hardy J\", \"Hazrati L\", \"Amouyel P\", \"Kauwe JS\", \"Younkin S\", \"Goate A\", \"Powell J\", \"Singleton A\", \"Carrasquillo M\", \"Bras J\", \"Wojtas A\", \"Majounie E\", \"Sassi C\", \"Morgan K\", \"Collinge J\", \"Rogaeva E\", \"St George-Hyslop P\", \"Lambert JC\", \"Alzheimer Genetic Analysis Group.\"], \"date\": \"2013-01-10\"}, \"line\": 44670, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \" Here we provide evidence in AD brains of a specific up-regulation of an NF-kappaB-sensitive miRNA-146a highly complementary to the 3'-untranslated region of complement factor H (CFH), an important repressor of the inflammatory response of the brain.Transfection of HN cells using an NF-kappaB-containing pre-miRNA-146a promoter-luciferase reporter construct in stressed HN cells showed significant up-regulation of luciferase activity that paralleled decreases in CFH gene expression. Up-regulation of miRNA-146a coupled to down-regulation of CFH was observed in AD brain and in interleukin-1beta, Abeta42, and/or oxidatively stressed human neural (HN) cells in primary culture.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18801740\", \"authors\": [\"Lukiw WJ\", \"Zhao Y\", \"Cui JG\"], \"date\": \"2008-11-14\"}, \"line\": 8977, \"annotations\": {\"Subgraph\": \"Complement system subgraph\"}, \"source\": 11, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38534, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Prostaglandin subgraph\"}, \"source\": 12, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38534, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9367, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 0, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9367, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 13, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9369, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 14, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9369, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 14, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37596, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37602, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37701, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37598, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1203, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 19, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1203, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 19, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 20, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 20, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37600, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 20, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1205, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 21, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1205, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 21, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37699, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 22, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37586, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 23, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37590, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 23, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37584, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 24, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37712, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 24, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38536, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 25, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science signaling\", \"reference\": \"23092889\", \"authors\": [\"Tang D\", \"Kang R\"], \"date\": \"2012-10-23\"}, \"line\": 37518, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 26, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37708, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 27, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37592, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 28, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37588, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 29, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37594, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 30, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Platelets are an important source of amyloid-ss (Ass) in the circulatory system and play an important pro-inflammatory role. Upon activation, they adhere to leukocytes and endothelial cells by means of adhesive proteins like P-selectin, platelet endothelial cell adhesion molecule-1 (PECAM) and intercellular adhesion molecule-1 and -2 (ICAM-1 and -2) and secrete inflammatory mediators (chemokines, interleukins). In addition, platelets contain important enzymes involved in inflammatory intermediary synthesis like phospholipase A(2) (PLA(2)) and cyclooxygenase-2 (COX-2), and recent reports demonstrated significant changes in platelet levels and activities in Alzheimer's disease. Thus, as platelets represent an important link between Ass deposition and inflammatory reactions especially at endothelial level, they can be considered a valuable cellular model to evaluate potential peripheral inflammatory biomarkers in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biogerontology\", \"reference\": \"20454929\", \"authors\": [\"Casoli T\", \"Di Stefano G\", \"Balietti M\", \"Solazzi M\", \"Giorgetti B\", \"Fattoretti P\"], \"date\": \"2010-10-01\"}, \"line\": 37683, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 31, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37710, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 32, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"There was a strong correlation between YKL-40 and markers of neurodegeneration (total tau and p-tau). In the whole sample, we found a negative correlation between YKL-40 and CTh in AD vulnerable areas in Abeta42+ subjects but not in Abeta42 participants. Our results suggest that YKL-40 could track the inflammatory processes associated to tau-related neurodegeneration in the presence of the AD pathophysiological process.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"25865441\", \"authors\": [\"Vidal-Piñeiro D\", \"Pegueroles J\", \"González-Suárez A\", \"González-Ortiz S\", \"Medrano S\", \"Montal V\", \"Anton-Aguirre S\", \"Carmona-Iragui M\", \"Sala I\", \"Fortea J\", \"Blesa R\", \"Morenas-Rodríguez E\", \"Sánchez-Saudinós M\", \"Sampedro F\", \"Sánchez-Juan P\", \"Rodríguez-Rodríguez E\", \"Pozueta A\", \"Bartrés-Faz D\", \"Vilaplana E\", \"Lleó A\", \"Alcolea D\", \"Clarimón J\"], \"date\": \"2015-06-01\"}, \"line\": 44310, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 33, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Recent patents on anti-infective drug discovery\", \"reference\": \"19149696\", \"authors\": [\"Molloy EJ\"], \"date\": \"2009-01-01\"}, \"line\": 45458, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\", \"Cell\": \"neutrophil\"}, \"source\": 34, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Recent patents on anti-infective drug discovery\", \"reference\": \"19149696\", \"authors\": [\"Molloy EJ\"], \"date\": \"2009-01-01\"}, \"line\": 45458, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\", \"Cell\": \"monocyte\"}, \"source\": 34, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of clinical neuroscience : official journal of the Neurosurgical Society of Australasia\", \"reference\": \"15177383\", \"authors\": [\"Parihar MS\", \"Hemnani T\"], \"date\": \"2004-06-01\"}, \"line\": 356, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 35, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of clinical neuroscience : official journal of the Neurosurgical Society of Australasia\", \"reference\": \"15177383\", \"authors\": [\"Parihar MS\", \"Hemnani T\"], \"date\": \"2004-06-01\"}, \"line\": 357, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 36, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cathepsin D, the most abundant lysosomal and endosomal aspartyl protease, shows beta and gamma secretase activity in vitro by cleaving the amyloid precursor protein (APP) into amyloid beta protein (Abeta). Polymorphism at position 224, C224T, on exon 2 of cathepsin D gene (CTSD) has been associated with an increased risk for Alzheimer's disease (AD) \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta neurologica Scandinavica\", \"reference\": \"20597865\", \"authors\": [\"Jeong BH\", \"Lee KH\", \"Lee YJ\", \"Yun J\", \"Park YJ\", \"Kim YH\", \"Cho YS\", \"Choi EK\", \"Carp RI\", \"Kim YS\"], \"date\": \"2011-06-01\"}, \"line\": 1409, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 37, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cathepsin D, the most abundant lysosomal and endosomal aspartyl protease, shows beta and gamma secretase activity in vitro by cleaving the amyloid precursor protein (APP) into amyloid beta protein (Abeta). Polymorphism at position 224, C224T, on exon 2 of cathepsin D gene (CTSD) has been associated with an increased risk for Alzheimer's disease (AD) \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta neurologica Scandinavica\", \"reference\": \"20597865\", \"authors\": [\"Jeong BH\", \"Lee KH\", \"Lee YJ\", \"Yun J\", \"Park YJ\", \"Kim YH\", \"Cho YS\", \"Choi EK\", \"Carp RI\", \"Kim YS\"], \"date\": \"2011-06-01\"}, \"line\": 1409, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 37, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of Abeta-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science (New York, N.Y.)\", \"reference\": \"22323736\", \"authors\": [\"Karlo JC\", \"Wesson DW\", \"Cramer PE\", \"Brunden KR\", \"Wilson DA\", \"Cirrito JR\", \"Goebel WD\", \"Landreth GE\", \"Zinn AE\", \"Casali BT\", \"Restivo JL\", \"James MJ\", \"Lee CY\"], \"date\": \"2012-03-23\"}, \"line\": 5396, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of Abeta-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science (New York, N.Y.)\", \"reference\": \"22323736\", \"authors\": [\"Karlo JC\", \"Wesson DW\", \"Cramer PE\", \"Brunden KR\", \"Wilson DA\", \"Cirrito JR\", \"Goebel WD\", \"Landreth GE\", \"Zinn AE\", \"Casali BT\", \"Restivo JL\", \"James MJ\", \"Lee CY\"], \"date\": \"2012-03-23\"}, \"line\": 5396, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"APOE subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of Abeta-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science (New York, N.Y.)\", \"reference\": \"22323736\", \"authors\": [\"Karlo JC\", \"Wesson DW\", \"Cramer PE\", \"Brunden KR\", \"Wilson DA\", \"Cirrito JR\", \"Goebel WD\", \"Landreth GE\", \"Zinn AE\", \"Casali BT\", \"Restivo JL\", \"James MJ\", \"Lee CY\"], \"date\": \"2012-03-23\"}, \"line\": 41982, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of Abeta-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science (New York, N.Y.)\", \"reference\": \"22323736\", \"authors\": [\"Karlo JC\", \"Wesson DW\", \"Cramer PE\", \"Brunden KR\", \"Wilson DA\", \"Cirrito JR\", \"Goebel WD\", \"Landreth GE\", \"Zinn AE\", \"Casali BT\", \"Restivo JL\", \"James MJ\", \"Lee CY\"], \"date\": \"2012-03-23\"}, \"line\": 41982, \"annotations\": {\"Subgraph\": \"APOE subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" Recently, many amyloid PET-positive and cognitively normal subjects were found in PiB-PET studies. PiB-PET studies on healthy subjects have also shown that apolipoprotein (APO) E4 boosts the accumulation of amyloid-beta and may consequently accelerate the pathogenesis of AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain and nerve = Shinkei kenkyu no shinpo\", \"reference\": \"20675880\", \"authors\": [\"Ishii K\"], \"date\": \"2010-07-01\"}, \"line\": 4107, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \" Recently, many amyloid PET-positive and cognitively normal subjects were found in PiB-PET studies. PiB-PET studies on healthy subjects have also shown that apolipoprotein (APO) E4 boosts the accumulation of amyloid-beta and may consequently accelerate the pathogenesis of AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain and nerve = Shinkei kenkyu no shinpo\", \"reference\": \"20675880\", \"authors\": [\"Ishii K\"], \"date\": \"2010-07-01\"}, \"line\": 4107, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"APOE subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Apolipoprotein E is the main lipid carrier in the brain and the best-established risk factor for late-onset Alzheimer's disease. Intracellular cholesterol levels influence the generation of amyloid-beta peptides, the toxic species thought to be a primary cause of Alzheimer's disease. Finally, compounds that modulate cholesterol metabolism affect amyloid-beta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current opinion in lipidology\", \"reference\": \"17495608\", \"authors\": [\"Hirsch-Reinshagen V\", \"Wellington CL\"], \"date\": \"2007-06-01\"}, \"line\": 4756, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"Apolipoprotein E is the main lipid carrier in the brain and the best-established risk factor for late-onset Alzheimer's disease. Intracellular cholesterol levels influence the generation of amyloid-beta peptides, the toxic species thought to be a primary cause of Alzheimer's disease. Finally, compounds that modulate cholesterol metabolism affect amyloid-beta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current opinion in lipidology\", \"reference\": \"17495608\", \"authors\": [\"Hirsch-Reinshagen V\", \"Wellington CL\"], \"date\": \"2007-06-01\"}, \"line\": 4756, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"APOE subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \" ApoE was not protective, but was injurious, as deletion of ApoE delayed the neurodegeneration caused by alpha-synuclein and suppressed the accumulation of Abeta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature neuroscience\", \"reference\": \"18297066\", \"authors\": [\"Gallardo G\", \"Schlüter OM\", \"Südhof TC\"], \"date\": \"2008-03-01\"}, \"line\": 24748, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \" ApoE was not protective, but was injurious, as deletion of ApoE delayed the neurodegeneration caused by alpha-synuclein and suppressed the accumulation of Abeta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature neuroscience\", \"reference\": \"18297066\", \"authors\": [\"Gallardo G\", \"Schlüter OM\", \"Südhof TC\"], \"date\": \"2008-03-01\"}, \"line\": 24748, \"annotations\": {\"Subgraph\": \"APOE subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 9}, {\"relation\": \"increases\", \"evidence\": \" ApoE was not protective, but was injurious, as deletion of ApoE delayed the neurodegeneration caused by alpha-synuclein and suppressed the accumulation of Abeta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature neuroscience\", \"reference\": \"18297066\", \"authors\": [\"Gallardo G\", \"Schlüter OM\", \"Südhof TC\"], \"date\": \"2008-03-01\"}, \"line\": 24748, \"annotations\": {\"Subgraph\": \"Synuclein subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 10}, {\"relation\": \"decreases\", \"evidence\": \"We report that ApoE plays a role in facilitating the proteolytic clearance of soluble Abeta from the brain. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"18549781\", \"authors\": [\"Jiang Q\", \"Lee CY\", \"Mandrekar S\", \"Wilkinson B\", \"Cramer P\", \"Zelcer N\", \"Mann K\", \"Lamb B\", \"Willson TM\", \"Collins JL\", \"Richardson JC\", \"Smith JD\", \"Comery TA\", \"Riddell D\", \"Holtzman DM\", \"Tontonoz P\", \"Landreth GE\"], \"date\": \"2008-06-12\"}, \"line\": 24759, \"annotations\": {\"Subgraph\": \"APOE subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 11}, {\"relation\": \"decreases\", \"evidence\": \"We report that ApoE plays a role in facilitating the proteolytic clearance of soluble Abeta from the brain. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"18549781\", \"authors\": [\"Jiang Q\", \"Lee CY\", \"Mandrekar S\", \"Wilkinson B\", \"Cramer P\", \"Zelcer N\", \"Mann K\", \"Lamb B\", \"Willson TM\", \"Collins JL\", \"Richardson JC\", \"Smith JD\", \"Comery TA\", \"Riddell D\", \"Holtzman DM\", \"Tontonoz P\", \"Landreth GE\"], \"date\": \"2008-06-12\"}, \"line\": 24759, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 12}, {\"relation\": \"decreases\", \"evidence\": \" We further demonstrated that apolipoprotein E2 and E3 but not apolipoprotein E4 can decrease the fusogenic activity of Abeta(29-42) via a direct interaction. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"10428074\", \"authors\": [\"Rosseneu M\", \"Brasseur R\", \"Pillot T\", \"Thomas-Soumarmon A\", \"Vandekerchkhove J\", \"Lins L\"], \"date\": \"1999-08-01\"}, \"line\": 24944, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"APOE subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 13}, {\"relation\": \"decreases\", \"evidence\": \" We further demonstrated that apolipoprotein E2 and E3 but not apolipoprotein E4 can decrease the fusogenic activity of Abeta(29-42) via a direct interaction. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"10428074\", \"authors\": [\"Rosseneu M\", \"Brasseur R\", \"Pillot T\", \"Thomas-Soumarmon A\", \"Vandekerchkhove J\", \"Lins L\"], \"date\": \"1999-08-01\"}, \"line\": 24944, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 38, \"target\": 1, \"key\": 14}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"line\": 7308, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Upon binding, LRP8 transduces reelin signaling during neuronal development, and recent evidence has indicated that it interacts with the NR2A and NR2B subunits of NMDA receptor, being involved in neuronal functions such as maturation of NMDA receptor composition in the hippocampus, and the regulation of long-term potentiation. Subsequently, a novel interaction between reelin and AbetaPP was discovered, leading to increase in the cell surface levels of AbetaPP and affecting AbetaPP processing and Abeta production. It was shown that reelin signaling in excitatory synapses can restore normal synaptic plasticity, which is impaired by oligomeric Abeta peptides at concentrations within the range detectable in the brains of AD patients. At high concentrations of Abeta peptides, reelin can no longer overcome the Abeta-induced functional suppression, and this condition coincides with a complete blockade of the reelin-dependent phosphorylation of NR2 subunits in NMDA receptors. This reversal requires the LRP receptor-dependent activation of tyrosine kinases of the Src family.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 2208, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"GOCC\", \"name\": \"intracellular\"}, \"toLoc\": {\"namespace\": \"GOCC\", \"name\": \"extracellular space\"}}}, \"annotations\": {\"CellStructure\": \"Centrosome\", \"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Upon binding, LRP8 transduces reelin signaling during neuronal development, and recent evidence has indicated that it interacts with the NR2A and NR2B subunits of NMDA receptor, being involved in neuronal functions such as maturation of NMDA receptor composition in the hippocampus, and the regulation of long-term potentiation. Subsequently, a novel interaction between reelin and AbetaPP was discovered, leading to increase in the cell surface levels of AbetaPP and affecting AbetaPP processing and Abeta production. It was shown that reelin signaling in excitatory synapses can restore normal synaptic plasticity, which is impaired by oligomeric Abeta peptides at concentrations within the range detectable in the brains of AD patients. At high concentrations of Abeta peptides, reelin can no longer overcome the Abeta-induced functional suppression, and this condition coincides with a complete blockade of the reelin-dependent phosphorylation of NR2 subunits in NMDA receptors. This reversal requires the LRP receptor-dependent activation of tyrosine kinases of the Src family.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 2208, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"GOCC\", \"name\": \"intracellular\"}, \"toLoc\": {\"namespace\": \"GOCC\", \"name\": \"extracellular space\"}}}, \"annotations\": {\"CellStructure\": \"Centrosome\", \"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Reelin signaling subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD), one of the major causes of disability and mortality in Western societies, is a progressive age-related neurodegenerative disorder. Increasing evidence suggests that the etiology of AD may involve disruptions of zinc (Zn) homeostasis. This review discusses current evidence supporting a potential role of Zn and zinc transporters (ZnTs) in processing of the amyloid beta protein precursor (APP) and amyloid beta (Abeta) peptide generation and aggregation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioFactors (Oxford, England)\", \"reference\": \"22447723\", \"authors\": [\"Lyubartseva G\", \"Lovell MA\"], \"date\": \"2012-03-01\"}, \"line\": 2334, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"The low-density lipoprotein receptor (LDLR) has the highest affinity for apoE and plays an important role in brain cholesterol metabolism.These data suggest that increased APP expression and Abeta exposure alters microtubule function, leading to reduced transport of LDLR to the plasma membrane. Consequent deleterious effects on apoE uptake and function will have implications for AD pathogenesis and/or progression\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"20049331\", \"authors\": [\"Abisambra JF\", \"Fiorelli T\", \"Padmanabhan J\", \"Neame P\", \"Wefes I\", \"Potter H\"], \"date\": \"2010-01-01\"}, \"line\": 3474, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"The low-density lipoprotein receptor (LDLR) has the highest affinity for apoE and plays an important role in brain cholesterol metabolism.These data suggest that increased APP expression and Abeta exposure alters microtubule function, leading to reduced transport of LDLR to the plasma membrane. Consequent deleterious effects on apoE uptake and function will have implications for AD pathogenesis and/or progression\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"20049331\", \"authors\": [\"Abisambra JF\", \"Fiorelli T\", \"Padmanabhan J\", \"Neame P\", \"Wefes I\", \"Potter H\"], \"date\": \"2010-01-01\"}, \"line\": 3474, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"The low-density lipoprotein receptor (LDLR) has the highest affinity for apoE and plays an important role in brain cholesterol metabolism.These data suggest that increased APP expression and Abeta exposure alters microtubule function, leading to reduced transport of LDLR to the plasma membrane. Consequent deleterious effects on apoE uptake and function will have implications for AD pathogenesis and/or progression\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"20049331\", \"authors\": [\"Abisambra JF\", \"Fiorelli T\", \"Padmanabhan J\", \"Neame P\", \"Wefes I\", \"Potter H\"], \"date\": \"2010-01-01\"}, \"line\": 3474, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"APOE subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"DYRK1A phosphorylates APP at Thr688 and enhances APP amyloidogenic cleavage, resulting in increased level of Abeta40/42, formation of toxic Abeta oligomers and deposition of fibrillar amyloid in plaques. Brain amyloidosis contributes to loss of neuronal function and possibly also to neuronal loss. Moreover, elevated levels of Abeta may up-regulate DYRK1A expression and enhance the contribution of overexpressed DYRK1A to neurofibrillary degeneration and β-amyloidosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The FEBS journal\", \"reference\": \"21156028\", \"authors\": [\"Wegiel J\", \"Gong CX\", \"Hwang YW\"], \"date\": \"2011-01-01\"}, \"line\": 4041, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"DYRK1A phosphorylates APP at Thr688 and enhances APP amyloidogenic cleavage, resulting in increased level of Abeta40/42, formation of toxic Abeta oligomers and deposition of fibrillar amyloid in plaques. Brain amyloidosis contributes to loss of neuronal function and possibly also to neuronal loss. Moreover, elevated levels of Abeta may up-regulate DYRK1A expression and enhance the contribution of overexpressed DYRK1A to neurofibrillary degeneration and β-amyloidosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The FEBS journal\", \"reference\": \"21156028\", \"authors\": [\"Wegiel J\", \"Gong CX\", \"Hwang YW\"], \"date\": \"2011-01-01\"}, \"line\": 4041, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \"DYRK1A phosphorylates APP at Thr688 and enhances APP amyloidogenic cleavage, resulting in increased level of Abeta40/42, formation of toxic Abeta oligomers and deposition of fibrillar amyloid in plaques. Brain amyloidosis contributes to loss of neuronal function and possibly also to neuronal loss. Moreover, elevated levels of Abeta may up-regulate DYRK1A expression and enhance the contribution of overexpressed DYRK1A to neurofibrillary degeneration and β-amyloidosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The FEBS journal\", \"reference\": \"21156028\", \"authors\": [\"Wegiel J\", \"Gong CX\", \"Hwang YW\"], \"date\": \"2011-01-01\"}, \"line\": 4041, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"DYRK1A subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 9}, {\"relation\": \"increases\", \"evidence\": \"Depletion of Hrs and Tsg101, acting early in the multivesicular body pathway, retained APP in early endosomes and reduced Abeta(40) production. Conversely, depletion of CHMP6 and VPS4, acting late in the pathway, rerouted endosomal APP to the TGN for enhanced APP processing. We found that VPS35 (retromer)-mediated APP recycling to the TGN was required for efficient Abeta(40) production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"22711829\", \"authors\": [\"Choy RW\", \"Cheng Z\", \"Schekman R\"], \"date\": \"2012-07-24\"}, \"line\": 4216, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"trans-Golgi Network\"}}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 10}, {\"relation\": \"increases\", \"evidence\": \"Depletion of Hrs and Tsg101, acting early in the multivesicular body pathway, retained APP in early endosomes and reduced Abeta(40) production. Conversely, depletion of CHMP6 and VPS4, acting late in the pathway, rerouted endosomal APP to the TGN for enhanced APP processing. We found that VPS35 (retromer)-mediated APP recycling to the TGN was required for efficient Abeta(40) production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"22711829\", \"authors\": [\"Choy RW\", \"Cheng Z\", \"Schekman R\"], \"date\": \"2012-07-24\"}, \"line\": 4216, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"trans-Golgi Network\"}}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Endoplasmic reticulum-Golgi protein export\"}, \"source\": 39, \"target\": 1, \"key\": 11}, {\"relation\": \"increases\", \"evidence\": \" An interruption of the bidirectional trafficking of APP between the TGN and endosomes, particularly retromer-mediated retrieval of APP from early endosomes to the TGN, resulted in the accumulation of endocytosed APP in early endosomes with reduced APP processing. These data suggest that Abeta(40) is generated predominantly in the TGN, relying on an endocytosed pool of APP recycled from early endosomes to the TGN.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"22711829\", \"authors\": [\"Choy RW\", \"Cheng Z\", \"Schekman R\"], \"date\": \"2012-07-24\"}, \"line\": 4224, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"trans-Golgi Network\"}}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 12}, {\"relation\": \"increases\", \"evidence\": \" An interruption of the bidirectional trafficking of APP between the TGN and endosomes, particularly retromer-mediated retrieval of APP from early endosomes to the TGN, resulted in the accumulation of endocytosed APP in early endosomes with reduced APP processing. These data suggest that Abeta(40) is generated predominantly in the TGN, relying on an endocytosed pool of APP recycled from early endosomes to the TGN.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"22711829\", \"authors\": [\"Choy RW\", \"Cheng Z\", \"Schekman R\"], \"date\": \"2012-07-24\"}, \"line\": 4224, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"trans-Golgi Network\"}}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Endoplasmic reticulum-Golgi protein export\"}, \"source\": 39, \"target\": 1, \"key\": 13}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17428983\", \"authors\": [\"He X\", \"Cooley K\", \"Chung CH\", \"Dashti N\", \"Tang J\"], \"date\": \"2007-04-11\"}, \"line\": 4778, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 14}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17428983\", \"authors\": [\"He X\", \"Cooley K\", \"Chung CH\", \"Dashti N\", \"Tang J\"], \"date\": \"2007-04-11\"}, \"line\": 4778, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 15}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17428983\", \"authors\": [\"He X\", \"Cooley K\", \"Chung CH\", \"Dashti N\", \"Tang J\"], \"date\": \"2007-04-11\"}, \"line\": 4778, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 16}, {\"relation\": \"increases\", \"evidence\": \"Cirrito et al. [26] also show that synaptic activity-induced increase in endocytosis drives more APP into the endocytic compartment, ultimately resulting in increased Aï¿½ production and release. Aï¿½ produced in the endocytic pathway is then brought to the cell surface where it is released into the extracellular fluid [70]. Inhibition of endocytosis reduces APP internalization and reduces Aï¿½ production and release in cell lines\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 6404, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Membrane\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 17}, {\"relation\": \"increases\", \"evidence\": \"Cirrito et al. [26] also show that synaptic activity-induced increase in endocytosis drives more APP into the endocytic compartment, ultimately resulting in increased Aï¿½ production and release. Aï¿½ produced in the endocytic pathway is then brought to the cell surface where it is released into the extracellular fluid [70]. Inhibition of endocytosis reduces APP internalization and reduces Aï¿½ production and release in cell lines\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 6404, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Membrane\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Synaptic vesicle endocytosis subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 18}, {\"relation\": \"increases\", \"evidence\": \"Expression of familial Alzheimer's disease (FAD) mutants of APP in primary neurons causes both intracellular accumulation of the C-terminal beta-secretase cleavage product of APP and increased secretion of Abeta, and eventually results in apoptotic death of the cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain research. Molecular brain research\", \"reference\": \"11744168\", \"authors\": [\"Golde T\", \"Eckman CB\", \"McPhie DL\", \"Brant JB\", \"Yager D\", \"Neve RL\"], \"date\": \"2001-12-16\"}, \"line\": 25208, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 19}, {\"relation\": \"increases\", \"evidence\": \"Expression of familial Alzheimer's disease (FAD) mutants of APP in primary neurons causes both intracellular accumulation of the C-terminal beta-secretase cleavage product of APP and increased secretion of Abeta, and eventually results in apoptotic death of the cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain research. Molecular brain research\", \"reference\": \"11744168\", \"authors\": [\"Golde T\", \"Eckman CB\", \"McPhie DL\", \"Brant JB\", \"Yager D\", \"Neve RL\"], \"date\": \"2001-12-16\"}, \"line\": 25208, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 20}, {\"relation\": \"increases\", \"evidence\": \"The formation of beta A4 amyloid in the brains of individuals with Alzheimer's disease requires the proteolytic cleavage of amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The international journal of biochemistry &amp; cell biology\", \"reference\": \"10605825\", \"authors\": [\"Sadik G\", \"Yamagata F\", \"Kaji H\", \"Takeda K\", \"Hashimoto K\", \"Kameoka Y\", \"Miyanaga K\", \"Shinoda T\"], \"date\": \"1999-11-01\"}, \"line\": 32630, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 21}, {\"relation\": \"increases\", \"evidence\": \"The formation of beta A4 amyloid in the brains of individuals with Alzheimer's disease requires the proteolytic cleavage of amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The international journal of biochemistry &amp; cell biology\", \"reference\": \"10605825\", \"authors\": [\"Sadik G\", \"Yamagata F\", \"Kaji H\", \"Takeda K\", \"Hashimoto K\", \"Kameoka Y\", \"Miyanaga K\", \"Shinoda T\"], \"date\": \"1999-11-01\"}, \"line\": 32630, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 22}, {\"relation\": \"increases\", \"evidence\": \"Several lines of evidence suggest that cathepsin D, the major lysosomal/endosomal aspartic protease, may be involved in this process. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The international journal of biochemistry &amp; cell biology\", \"reference\": \"10605825\", \"authors\": [\"Sadik G\", \"Yamagata F\", \"Kaji H\", \"Takeda K\", \"Hashimoto K\", \"Kameoka Y\", \"Miyanaga K\", \"Shinoda T\"], \"date\": \"1999-11-01\"}, \"line\": 32634, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 23}, {\"relation\": \"increases\", \"evidence\": \"Several lines of evidence suggest that cathepsin D, the major lysosomal/endosomal aspartic protease, may be involved in this process. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The international journal of biochemistry &amp; cell biology\", \"reference\": \"10605825\", \"authors\": [\"Sadik G\", \"Yamagata F\", \"Kaji H\", \"Takeda K\", \"Hashimoto K\", \"Kameoka Y\", \"Miyanaga K\", \"Shinoda T\"], \"date\": \"1999-11-01\"}, \"line\": 32634, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 24}, {\"relation\": \"decreases\", \"evidence\": \"SORL1 regulates and traps APP in the Golgi apparatus and reduces the production of the amyloid-beta peptide (Abeta), the main component of senile plaques in AD4. Cell biology experiments have indicated that SORL1 underexperssion leads to overproduction of Abeta3 , whereas SORL1 overexpression considerably reduces cellular APP and extracellular Abeta5. Although the underlying mechanisms of SORL1 in AD and neurodegenerative processes remain unclear, a recent comprehensive review on multiple functional domains of SORL1 suggested that SORL1 can assume various characters in AD pathogenesis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Scientific reports\", \"reference\": \"26996954\", \"authors\": [\"Tsai SJ\", \"Liu ME\", \"Chen TR\", \"Huang CC\", \"Chou KH\", \"Kao HW\", \"Yang AC\", \"Lin CP\", \"Wang YH\"], \"date\": \"2016-03-21\"}, \"line\": 53584, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\", \"CellStructure\": \"Golgi Apparatus\"}, \"source\": 39, \"target\": 1, \"key\": 25}, {\"relation\": \"decreases\", \"evidence\": \"SORL1 regulates and traps APP in the Golgi apparatus and reduces the production of the amyloid-beta peptide (Abeta), the main component of senile plaques in AD4. Cell biology experiments have indicated that SORL1 underexperssion leads to overproduction of Abeta3 , whereas SORL1 overexpression considerably reduces cellular APP and extracellular Abeta5. Although the underlying mechanisms of SORL1 in AD and neurodegenerative processes remain unclear, a recent comprehensive review on multiple functional domains of SORL1 suggested that SORL1 can assume various characters in AD pathogenesis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Scientific reports\", \"reference\": \"26996954\", \"authors\": [\"Tsai SJ\", \"Liu ME\", \"Chen TR\", \"Huang CC\", \"Chou KH\", \"Kao HW\", \"Yang AC\", \"Lin CP\", \"Wang YH\"], \"date\": \"2016-03-21\"}, \"line\": 53590, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 39, \"target\": 1, \"key\": 26}, {\"relation\": \"increases\", \"evidence\": \"The death of cholinergic neurons in the cerebral cortex and certain subcortical regions is linked to irreversible dementia relevant to AD (Alzheimer's disease). Although multiple studies have shown that expression of a FAD (familial AD)-linked APP (amyloid Abeta precursor protein) or a PS (presenilin) mutant, but not that of wild-type APP or PS, induced neuronal death by activating intracellular death signals, it remains to be addressed how these signals are interrelated and what the key molecule involved in this process is. In the present study, we show that the PS1-mediated (or possibly the PS2-mediated) signal is essential for the APP-mediated death in a gamma-secretase-independent manner and vice versa. MOCA (modifier of cell adhesion), which was originally identified as being a PS- and Rac1-binding protein, is a common downstream constituent of these neuronal death signals. Detailed molecular analysis indicates that MOCA is a key molecule of the AD-relevant neuronal death signals that links the PS-mediated death signal with the APP-mediated death signal at a point between Rac1 [or Cdc42 (cell division cycle 42)] and ASK1 (apoptosis signal-regulating kinase 1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Biochemical journal\", \"reference\": \"22115042\", \"authors\": [\"Matsuoka M\", \"Tachi N\", \"Hashimoto Y\"], \"date\": \"2012-03-01\"}, \"line\": 34052, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 39, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Defects in axonal transport are an early pathological feature in Alzheimer's disease and defective APP transport is known to increase ABeta¸ production. We show that calsyntenin-1 and APP are co-transported through axons and that siRNA-induced loss of calsyntenin-1 markedly disrupts axonal transport of APP. Thus, perturbation to axonal transport of APP on calsyntenin-1 containing carriers induces alterations to APP processing that increase production of ABeta¸. Together, our findings suggest that disruption of calsyntenin-1-associated axonal transport of APP is a pathogenic mechanism in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human molecular genetics\", \"reference\": \"22434822\", \"authors\": [\"Francis PT\", \"Perkinton MS\", \"Gray EH\", \"Vagnoni A\", \"Noble W\", \"Miller CC\"], \"date\": \"2012-07-01\"}, \"line\": 34460, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Axons\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Membrane\"}}}, \"annotations\": {\"MeSHAnatomy\": \"Axons\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Defects in axonal transport are an early pathological feature in Alzheimer's disease and defective APP transport is known to increase ABeta¸ production. We show that calsyntenin-1 and APP are co-transported through axons and that siRNA-induced loss of calsyntenin-1 markedly disrupts axonal transport of APP. Thus, perturbation to axonal transport of APP on calsyntenin-1 containing carriers induces alterations to APP processing that increase production of ABeta¸. Together, our findings suggest that disruption of calsyntenin-1-associated axonal transport of APP is a pathogenic mechanism in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human molecular genetics\", \"reference\": \"22434822\", \"authors\": [\"Francis PT\", \"Perkinton MS\", \"Gray EH\", \"Vagnoni A\", \"Noble W\", \"Miller CC\"], \"date\": \"2012-07-01\"}, \"line\": 34460, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Axons\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Membrane\"}}}, \"annotations\": {\"MeSHAnatomy\": \"Axons\", \"Subgraph\": \"Axonal transport subgraph\"}, \"source\": 39, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Iron was further demonstrated to modulate expression of the Alzheimer's amyloid precursor holo-protein (APP) by a mechanism similar to that of regulation of ferritin-L and -H mRNA translation through an iron-responsive element (IRE) in their 5' untranslated regions (UTRs). Here, we discuss two aspects of the link between iron and AD, in relation to the recently discovered IRE in the 5'UTR of APP mRNA. The first is the physiological aspect: a compensatory neuroprotective response of amyloid-ÃƒÅ¸ protein (ABeta¸) in reducing iron-induced neurotoxicity. Thus, given that ABeta¸ possesses iron chelation sites, it is hypothesized that OS-induced intracellular iron may stimulate APP holo-protein translation (via the APP 5'UTR) and subsequently the generation of its cleavage product, ABeta¸, as a compensatory response that eventually reduces OS.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMC neuroscience\", \"reference\": \"19090990\", \"authors\": [\"Avramovich-Tirosh Y\", \"Amit T\", \"Bar-Am O\", \"Weinreb O\", \"Youdim MB\"], \"date\": \"2008-12-03\"}, \"line\": 36216, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In addition, based on timing of changes in cortical thinning and Abeta deposition such as Abeta deposition after cortical thinning; cortical thinning after Abeta deposition, or concurrent Abeta deposition and cortical thinning, we identified three types of relationships between cortical thinning and Abeta deposition: (1) Abeta-associated cortical thinning; (2) Abeta-independent cortical thinning; and (3) Abeta deposition only without cortical thinning. Taken together, these findings suggest that Abeta-weighted cortical thickness values can be used as an objective biomarker of structural changes caused by amyloid pathology in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"26027813\", \"authors\": [\"Koh JY\", \"Roh JH\", \"Lee JH\", \"Alzheimer’s Disease Neuroimaging Initiative (ADNI).\", \"Hwang J\", \"Kim CM\", \"Lee JM\"], \"date\": \"2015-01-01\"}, \"line\": 44382, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 4, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Although the role of AICD as a gene transcription inducer is still controversial, several targets of AICD- or APP-mediated transcriptional activation are reportedly regulated by APP nuclear signaling, including APP itself, BACE, Tip60, GSK-3ÃƒÅ¸, p53, Mn-SOD, KAI1, Neprilysin, transgelin, a2actin, S100a9, and other genes\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMB reports\", \"reference\": \"21034527\", \"authors\": [\"Chang KA\", \"Suh YH\"], \"date\": \"2010-10-01\"}, \"line\": 36289, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 39, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"line\": 7310, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"line\": 7310, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"17995932\", \"authors\": [\"Karlstrom H\", \"Brooks WS\", \"Kwok JB\", \"Broe GA\", \"Kril JJ\", \"McCann H\", \"Halliday GM\", \"Schofield PR\"], \"date\": \"2008-02-01\"}, \"line\": 25544, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"17995932\", \"authors\": [\"Karlstrom H\", \"Brooks WS\", \"Kwok JB\", \"Broe GA\", \"Kril JJ\", \"McCann H\", \"Halliday GM\", \"Schofield PR\"], \"date\": \"2008-02-01\"}, \"line\": 25544, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"This cleavage is similar to the PS-dependent gamma-secretase cleavage of the beta-amyloid precursor protein (betaAPP). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The EMBO journal\", \"reference\": \"12374741\", \"authors\": [\"Okochi M\", \"Tanaka T\", \"Haass C\", \"Fukumori A\", \"Tanii H\", \"Takeda M\", \"Steiner H\", \"Tomita T\", \"Iwatsubo T\", \"Kudo T\"], \"date\": \"2002-10-15\"}, \"line\": 30220, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"This cleavage is similar to the PS-dependent gamma-secretase cleavage of the beta-amyloid precursor protein (betaAPP). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The EMBO journal\", \"reference\": \"12374741\", \"authors\": [\"Okochi M\", \"Tanaka T\", \"Haass C\", \"Fukumori A\", \"Tanii H\", \"Takeda M\", \"Steiner H\", \"Tomita T\", \"Iwatsubo T\", \"Kudo T\"], \"date\": \"2002-10-15\"}, \"line\": 30220, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Presenilin-1 (PS1) is intimately involved in cleavage of amyloid precursor protein to form beta-amyloid peptides, certain forms of which aggregate in the brains of patients with Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"11814648\", \"authors\": [\"Collingridge GL\", \"Seabrook GR\", \"Rosahl TW\", \"Fitzjohn SM\", \"Morton RA\", \"Kuenzi FM\", \"Shearman M\", \"Smith D\", \"Zheng H\"], \"date\": \"2002-02-08\"}, \"line\": 30358, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"Presenilin-1 (PS1) is intimately involved in cleavage of amyloid precursor protein to form beta-amyloid peptides, certain forms of which aggregate in the brains of patients with Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"11814648\", \"authors\": [\"Collingridge GL\", \"Seabrook GR\", \"Rosahl TW\", \"Fitzjohn SM\", \"Morton RA\", \"Kuenzi FM\", \"Shearman M\", \"Smith D\", \"Zheng H\"], \"date\": \"2002-02-08\"}, \"line\": 30358, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"It was hypothesized that PS1 might directly cleave APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"10670705\", \"authors\": [\"Pradier L\", \"Tremp G\", \"Czech C\"], \"date\": \"2000-03-01\"}, \"line\": 30398, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \"It was hypothesized that PS1 might directly cleave APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"10670705\", \"authors\": [\"Pradier L\", \"Tremp G\", \"Czech C\"], \"date\": \"2000-03-01\"}, \"line\": 30398, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 9}, {\"relation\": \"increases\", \"evidence\": \"Therefore, our results show that PS1 binds to APP directly and suggest that the PS1 protein itself is involved in the metabolism of beta-amyloid peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemical and biophysical research communications\", \"reference\": \"9344855\", \"authors\": [\"Waragai M\", \"Kanazawa I\", \"Imafuku I\", \"Takeuchi S\", \"Udagawa Y\", \"Kawabata M\", \"Okazawa H\", \"Oyama F\"], \"date\": \"1997-10-20\"}, \"line\": 30408, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 10}, {\"relation\": \"increases\", \"evidence\": \"Therefore, our results show that PS1 binds to APP directly and suggest that the PS1 protein itself is involved in the metabolism of beta-amyloid peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemical and biophysical research communications\", \"reference\": \"9344855\", \"authors\": [\"Waragai M\", \"Kanazawa I\", \"Imafuku I\", \"Takeuchi S\", \"Udagawa Y\", \"Kawabata M\", \"Okazawa H\", \"Oyama F\"], \"date\": \"1997-10-20\"}, \"line\": 30408, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 11}, {\"relation\": \"increases\", \"evidence\": \"Our results indicate PS1 and APP can interact in the ER and Golgi, where PS1 is required for proper gamma-secretase processing of APP CTFs, and that PS1 mutations augment Abeta42 levels principally in Golgi-like vesicles.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"9843412\", \"authors\": [\"Koo EH\", \"Selkoe DJ\", \"Zhang J\", \"Xia W\", \"Shen J\", \"Ostaszewski BL\", \"Kimberly WT\", \"Seubert P\"], \"date\": \"1998-11-24\"}, \"line\": 30419, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 12}, {\"relation\": \"increases\", \"evidence\": \"Our results indicate PS1 and APP can interact in the ER and Golgi, where PS1 is required for proper gamma-secretase processing of APP CTFs, and that PS1 mutations augment Abeta42 levels principally in Golgi-like vesicles.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"9843412\", \"authors\": [\"Koo EH\", \"Selkoe DJ\", \"Zhang J\", \"Xia W\", \"Shen J\", \"Ostaszewski BL\", \"Kimberly WT\", \"Seubert P\"], \"date\": \"1998-11-24\"}, \"line\": 30419, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 13}, {\"relation\": \"increases\", \"evidence\": \"Missense substitutions in the presenilin 1 (PS1) and presenilin 2 (PS2) proteins are associated with early-onset familial Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"10037471\", \"authors\": [\"Duthie M\", \"Nishimura M\", \"Murgolo N\", \"Wang L\", \"VanderVere P\", \"Bayne ML\", \"Strader CD\", \"Xu D\", \"Zhang DM\", \"Levesque L\", \"Yu H\", \"Yu G\", \"Liang Y\", \"Rogaeva E\", \"St George-Hyslop P\", \"Levesque G\", \"Rommens JM\", \"Ikeda M\", \"Fraser PE\"], \"date\": \"1999-03-01\"}, \"line\": 30612, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 14}, {\"relation\": \"increases\", \"evidence\": \"Missense substitutions in the presenilin 1 (PS1) and presenilin 2 (PS2) proteins are associated with early-onset familial Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"10037471\", \"authors\": [\"Duthie M\", \"Nishimura M\", \"Murgolo N\", \"Wang L\", \"VanderVere P\", \"Bayne ML\", \"Strader CD\", \"Xu D\", \"Zhang DM\", \"Levesque L\", \"Yu H\", \"Yu G\", \"Liang Y\", \"Rogaeva E\", \"St George-Hyslop P\", \"Levesque G\", \"Rommens JM\", \"Ikeda M\", \"Fraser PE\"], \"date\": \"1999-03-01\"}, \"line\": 30612, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 15}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta) is generated by the consecutive cleavages of beta- and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"12147673\", \"authors\": [\"Haass C\", \"Lammich S\", \"Kaether C\", \"Edbauer D\", \"Ertl M\", \"Rietdorf J\", \"Capell A\", \"Steiner H\"], \"date\": \"2002-08-05\"}, \"line\": 30732, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 16}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta) is generated by the consecutive cleavages of beta- and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"12147673\", \"authors\": [\"Haass C\", \"Lammich S\", \"Kaether C\", \"Edbauer D\", \"Ertl M\", \"Rietdorf J\", \"Capell A\", \"Steiner H\"], \"date\": \"2002-08-05\"}, \"line\": 30732, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 17}, {\"relation\": \"increases\", \"evidence\": \"Abundant biochemical and genetic evidence suggests that presenilins are catalytic components of gamma-secretase, the protease responsible for generating the Alzheimer amyloid beta-protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"12603837\", \"authors\": [\"Siman R\", \"Velji J\"], \"date\": \"2003-03-01\"}, \"line\": 30757, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 18}, {\"relation\": \"increases\", \"evidence\": \"Abundant biochemical and genetic evidence suggests that presenilins are catalytic components of gamma-secretase, the protease responsible for generating the Alzheimer amyloid beta-protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"12603837\", \"authors\": [\"Siman R\", \"Velji J\"], \"date\": \"2003-03-01\"}, \"line\": 30757, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 40, \"target\": 1, \"key\": 19}, {\"relation\": \"increases\", \"evidence\": \"hypomethylation of the promoter of the presenilin 1 (PS1) gene, which will lead to overexpression of presenilin 1 and, consequently, to increased Abeta(1-42) (Abeta42) formation \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"16040194\", \"authors\": [\"Mulder C\", \"Schoonenboom NS\", \"Jansen EE\", \"Verhoeven NM\", \"van Kamp GJ\", \"Jakobs C\", \"Scheltens P\"], \"date\": \"2005-09-30\"}, \"line\": 43192, \"annotations\": {\"Encode_Feature_Types\": \"Promoter\"}, \"source\": 40, \"target\": 1, \"key\": 20}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"line\": 7311, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 41, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"line\": 7311, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 41, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Missense substitutions in the presenilin 1 (PS1) and presenilin 2 (PS2) proteins are associated with early-onset familial Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"10037471\", \"authors\": [\"Duthie M\", \"Nishimura M\", \"Murgolo N\", \"Wang L\", \"VanderVere P\", \"Bayne ML\", \"Strader CD\", \"Xu D\", \"Zhang DM\", \"Levesque L\", \"Yu H\", \"Yu G\", \"Liang Y\", \"Rogaeva E\", \"St George-Hyslop P\", \"Levesque G\", \"Rommens JM\", \"Ikeda M\", \"Fraser PE\"], \"date\": \"1999-03-01\"}, \"line\": 30613, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 41, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Missense substitutions in the presenilin 1 (PS1) and presenilin 2 (PS2) proteins are associated with early-onset familial Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"10037471\", \"authors\": [\"Duthie M\", \"Nishimura M\", \"Murgolo N\", \"Wang L\", \"VanderVere P\", \"Bayne ML\", \"Strader CD\", \"Xu D\", \"Zhang DM\", \"Levesque L\", \"Yu H\", \"Yu G\", \"Liang Y\", \"Rogaeva E\", \"St George-Hyslop P\", \"Levesque G\", \"Rommens JM\", \"Ikeda M\", \"Fraser PE\"], \"date\": \"1999-03-01\"}, \"line\": 30613, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 41, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta) is generated by the consecutive cleavages of beta- and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"12147673\", \"authors\": [\"Haass C\", \"Lammich S\", \"Kaether C\", \"Edbauer D\", \"Ertl M\", \"Rietdorf J\", \"Capell A\", \"Steiner H\"], \"date\": \"2002-08-05\"}, \"line\": 30733, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 41, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta) is generated by the consecutive cleavages of beta- and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"12147673\", \"authors\": [\"Haass C\", \"Lammich S\", \"Kaether C\", \"Edbauer D\", \"Ertl M\", \"Rietdorf J\", \"Capell A\", \"Steiner H\"], \"date\": \"2002-08-05\"}, \"line\": 30733, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 41, \"target\": 1, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Abundant biochemical and genetic evidence suggests that presenilins are catalytic components of gamma-secretase, the protease responsible for generating the Alzheimer amyloid beta-protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"12603837\", \"authors\": [\"Siman R\", \"Velji J\"], \"date\": \"2003-03-01\"}, \"line\": 30758, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 41, \"target\": 1, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"Abundant biochemical and genetic evidence suggests that presenilins are catalytic components of gamma-secretase, the protease responsible for generating the Alzheimer amyloid beta-protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"12603837\", \"authors\": [\"Siman R\", \"Velji J\"], \"date\": \"2003-03-01\"}, \"line\": 30758, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 41, \"target\": 1, \"key\": 7}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPARγ leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in β-catenin and inhibition of GSK-3β is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPARγ by a potent ligand leads to β-catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPARγ activation in neuroprotection, PPARγ agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPARγ activation . A recent report demonstrates that overexpression of PPARγ in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPARγ protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of diabetes science and technology\", \"reference\": \"19885299\", \"authors\": [\"de la Monte SM\", \"Wands JR\"], \"date\": \"2008-11-01\"}, \"line\": 7524, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPARγ leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in β-catenin and inhibition of GSK-3β is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPARγ by a potent ligand leads to β-catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPARγ activation in neuroprotection, PPARγ agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPARγ activation . A recent report demonstrates that overexpression of PPARγ in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPARγ protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of diabetes science and technology\", \"reference\": \"19885299\", \"authors\": [\"de la Monte SM\", \"Wands JR\"], \"date\": \"2008-11-01\"}, \"line\": 7524, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPARγ leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in β-catenin and inhibition of GSK-3β is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPARγ by a potent ligand leads to β-catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPARγ activation in neuroprotection, PPARγ agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPARγ activation . A recent report demonstrates that overexpression of PPARγ in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPARγ protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of diabetes science and technology\", \"reference\": \"19885299\", \"authors\": [\"de la Monte SM\", \"Wands JR\"], \"date\": \"2008-11-01\"}, \"line\": 7524, \"annotations\": {\"Subgraph\": \"Ubiquitin degradation subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPARγ leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in β-catenin and inhibition of GSK-3β is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPARγ by a potent ligand leads to β-catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPARγ activation in neuroprotection, PPARγ agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPARγ activation . A recent report demonstrates that overexpression of PPARγ in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPARγ protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of diabetes science and technology\", \"reference\": \"19885299\", \"authors\": [\"de la Monte SM\", \"Wands JR\"], \"date\": \"2008-11-01\"}, \"line\": 42065, \"annotations\": {\"UserdefinedCellLine\": \"App transgenic\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPARγ leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in β-catenin and inhibition of GSK-3β is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPARγ by a potent ligand leads to β-catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPARγ activation in neuroprotection, PPARγ agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPARγ activation . A recent report demonstrates that overexpression of PPARγ in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPARγ protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of diabetes science and technology\", \"reference\": \"19885299\", \"authors\": [\"de la Monte SM\", \"Wands JR\"], \"date\": \"2008-11-01\"}, \"line\": 42065, \"annotations\": {\"UserdefinedCellLine\": \"App transgenic\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Abeta) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"19660836\", \"authors\": [\"Combarros O\", \"Rodríguez-Rodríguez E\", \"Mateo I\", \"Vázquez-Higuera JL\", \"Infante J\", \"Berciano J\", \"Sánchez-Juan P\"], \"date\": \"2011-03-01\"}, \"line\": 5014, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Abeta) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"19660836\", \"authors\": [\"Combarros O\", \"Rodríguez-Rodríguez E\", \"Mateo I\", \"Vázquez-Higuera JL\", \"Infante J\", \"Berciano J\", \"Sánchez-Juan P\"], \"date\": \"2011-03-01\"}, \"line\": 5014, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"16828233\", \"authors\": [\"McCarty MF\"], \"date\": \"2006-01-01\"}, \"line\": 31947, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 7}, {\"relation\": \"decreases\", \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"16828233\", \"authors\": [\"McCarty MF\"], \"date\": \"2006-01-01\"}, \"line\": 31947, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 8}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Abeta) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"19660836\", \"authors\": [\"Combarros O\", \"Rodríguez-Rodríguez E\", \"Mateo I\", \"Vázquez-Higuera JL\", \"Infante J\", \"Berciano J\", \"Sánchez-Juan P\"], \"date\": \"2011-03-01\"}, \"line\": 41933, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 9}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Abeta) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"19660836\", \"authors\": [\"Combarros O\", \"Rodríguez-Rodríguez E\", \"Mateo I\", \"Vázquez-Higuera JL\", \"Infante J\", \"Berciano J\", \"Sánchez-Juan P\"], \"date\": \"2011-03-01\"}, \"line\": 41933, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 10}, {\"relation\": \"decreases\", \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"16828233\", \"authors\": [\"McCarty MF\"], \"date\": \"2006-01-01\"}, \"line\": 42152, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 11}, {\"relation\": \"decreases\", \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"16828233\", \"authors\": [\"McCarty MF\"], \"date\": \"2006-01-01\"}, \"line\": 42152, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 42, \"target\": 1, \"key\": 12}, {\"relation\": \"decreases\", \"evidence\": \"these findings suggest the existence of a down-regulation of PPARγ under inflammatory conditions, which would result in an increase in BACE1 transcription and Abeta generation. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"18564425\", \"authors\": [\"Sastre M\", \"Walter J\", \"Gentleman SM\"], \"date\": \"2008-06-18\"}, \"line\": 36707, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"these findings suggest the existence of a down-regulation of PPARγ under inflammatory conditions, which would result in an increase in BACE1 transcription and Abeta generation. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"18564425\", \"authors\": [\"Sastre M\", \"Walter J\", \"Gentleman SM\"], \"date\": \"2008-06-18\"}, \"line\": 36707, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 42, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"these findings suggest the existence of a down-regulation of PPARγ under inflammatory conditions, which would result in an increase in BACE1 transcription and Abeta generation. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"18564425\", \"authors\": [\"Sastre M\", \"Walter J\", \"Gentleman SM\"], \"date\": \"2008-06-18\"}, \"line\": 42243, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 4, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"these findings suggest the existence of a down-regulation of PPARγ under inflammatory conditions, which would result in an increase in BACE1 transcription and Abeta generation. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"18564425\", \"authors\": [\"Sastre M\", \"Walter J\", \"Gentleman SM\"], \"date\": \"2008-06-18\"}, \"line\": 42243, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 42, \"target\": 4, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37705, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 17, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37704, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 22, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37716, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 24, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37714, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 42, \"target\": 27, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"To further elucidate how the clusterin-dependent induction of Dkk1 by Aβ mediates neurotoxicity, we measured the effects of Aβ and Dkk1 protein on whole-genome expression in primary neurons, finding a common pathway suggestive of activation of wnt-planar cell polarity (PCP)-c-Jun N-terminal kinase (JNK) signalling leading to the induction of genes including EGR1 (early growth response-1), NAB2 (Ngfi-A-binding protein-2) and KLF10 (Krüppel-like factor-10) that, when individually silenced, protected against Aβ neurotoxicity and/or tau phosphorylation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"23164821\", \"authors\": [\"To AW\", \"Brion JP\", \"Reynolds CH\", \"Leroy K\", \"Noble W\", \"Robinson M\", \"Williamson R\", \"Hooper C\", \"Causevic M\", \"Breen G\", \"Lin K\", \"Ribe EM\", \"Wroe R\", \"Malik B\", \"Kellie S\", \"Lovestone S\", \"Usardi A\", \"Stephenson J\", \"Killick R\", \"Hodges A\", \"Lunnon K\", \"Bazenet C\", \"Simons JP\", \"Furney S\", \"Nolan PM\", \"Fernandes C\", \"Dobson R\", \"Lourdusamy A\", \"Al-Shawi R\"], \"date\": \"2014-01-01\"}, \"line\": 55957, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 43, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"To further elucidate how the clusterin-dependent induction of Dkk1 by Aβ mediates neurotoxicity, we measured the effects of Aβ and Dkk1 protein on whole-genome expression in primary neurons, finding a common pathway suggestive of activation of wnt-planar cell polarity (PCP)-c-Jun N-terminal kinase (JNK) signalling leading to the induction of genes including EGR1 (early growth response-1), NAB2 (Ngfi-A-binding protein-2) and KLF10 (Krüppel-like factor-10) that, when individually silenced, protected against Aβ neurotoxicity and/or tau phosphorylation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"23164821\", \"authors\": [\"To AW\", \"Brion JP\", \"Reynolds CH\", \"Leroy K\", \"Noble W\", \"Robinson M\", \"Williamson R\", \"Hooper C\", \"Causevic M\", \"Breen G\", \"Lin K\", \"Ribe EM\", \"Wroe R\", \"Malik B\", \"Kellie S\", \"Lovestone S\", \"Usardi A\", \"Stephenson J\", \"Killick R\", \"Hodges A\", \"Lunnon K\", \"Bazenet C\", \"Simons JP\", \"Furney S\", \"Nolan PM\", \"Fernandes C\", \"Dobson R\", \"Lourdusamy A\", \"Al-Shawi R\"], \"date\": \"2014-01-01\"}, \"line\": 55958, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 44, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"To further elucidate how the clusterin-dependent induction of Dkk1 by Aβ mediates neurotoxicity, we measured the effects of Aβ and Dkk1 protein on whole-genome expression in primary neurons, finding a common pathway suggestive of activation of wnt-planar cell polarity (PCP)-c-Jun N-terminal kinase (JNK) signalling leading to the induction of genes including EGR1 (early growth response-1), NAB2 (Ngfi-A-binding protein-2) and KLF10 (Krüppel-like factor-10) that, when individually silenced, protected against Aβ neurotoxicity and/or tau phosphorylation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"23164821\", \"authors\": [\"To AW\", \"Brion JP\", \"Reynolds CH\", \"Leroy K\", \"Noble W\", \"Robinson M\", \"Williamson R\", \"Hooper C\", \"Causevic M\", \"Breen G\", \"Lin K\", \"Ribe EM\", \"Wroe R\", \"Malik B\", \"Kellie S\", \"Lovestone S\", \"Usardi A\", \"Stephenson J\", \"Killick R\", \"Hodges A\", \"Lunnon K\", \"Bazenet C\", \"Simons JP\", \"Furney S\", \"Nolan PM\", \"Fernandes C\", \"Dobson R\", \"Lourdusamy A\", \"Al-Shawi R\"], \"date\": \"2014-01-01\"}, \"line\": 55959, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 45, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Lysosomal dysfunction may be the earliest histological change in AD. Amyloid plaques are full of active lysosomal hydrolases, implying that plaques may originate from lysosomal rupture. Cathepsins D and E (intracellular aspartyl proteases) are considered to influence Abeta peptides generation within the endosomalâ€“lysosomal pathway because they exhibit β- and γ-secretase like-activity [32]. For this reason, the endosomalâ€“lysosomal pathway is a site for cleavage of the APP into smaller β-amyloid-containing peptides, which are then degraded by cathepsins. Inhibition of cathepsins activity causes a rapid and pronounced build-up of potentially amyloidogenic protein fragments [33]. On the other hand, a failure to degrade aggregated Abeta_42 in late endosomes or secondary lysosomes was a mechanism that contributed to intracellular accumulation of Abeta in AD. The cysteine protease cathepsin B in lysosomes degrades A peptides, especially the aggregation-prone species Abeta_42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta biochimica et biophysica Sinica\", \"reference\": \"19499146\", \"authors\": [\"Zhang L\", \"Sheng R\", \"Qin Z\"], \"date\": \"2009-06-01\"}, \"line\": 6795, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 46, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Lysosomal dysfunction may be the earliest histological change in AD. Amyloid plaques are full of active lysosomal hydrolases, implying that plaques may originate from lysosomal rupture. Cathepsins D and E (intracellular aspartyl proteases) are considered to influence Abeta peptides generation within the endosomalâ€“lysosomal pathway because they exhibit β- and γ-secretase like-activity [32]. For this reason, the endosomalâ€“lysosomal pathway is a site for cleavage of the APP into smaller β-amyloid-containing peptides, which are then degraded by cathepsins. Inhibition of cathepsins activity causes a rapid and pronounced build-up of potentially amyloidogenic protein fragments [33]. On the other hand, a failure to degrade aggregated Abeta_42 in late endosomes or secondary lysosomes was a mechanism that contributed to intracellular accumulation of Abeta in AD. The cysteine protease cathepsin B in lysosomes degrades A peptides, especially the aggregation-prone species Abeta_42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta biochimica et biophysica Sinica\", \"reference\": \"19499146\", \"authors\": [\"Zhang L\", \"Sheng R\", \"Qin Z\"], \"date\": \"2009-06-01\"}, \"line\": 6795, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 46, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The CR1 variant rs6656401 influences risk and recurrence of CAA-ICH, as well as the severity of vascular amyloid deposition\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurology\", \"reference\": \"22262751\", \"authors\": [\"Meschia JF\", \"De Jager PL\", \"Bennett DA\", \"Brown DL\", \"Rosand J\", \"Slowik A\", \"Schwab K\", \"Greenberg SM\", \"Silliman SL\", \"Shulman JM\", \"Selim M\", \"Worrall BB\", \"Schneider JA\", \"Ayres AM\", \"Cortellini L\", \"Jagiella JM\", \"Biffi A\"], \"date\": \"2012-01-31\"}, \"line\": 3424, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 47, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The CR1 variant rs6656401 influences risk and recurrence of CAA-ICH, as well as the severity of vascular amyloid deposition\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurology\", \"reference\": \"22262751\", \"authors\": [\"Meschia JF\", \"De Jager PL\", \"Bennett DA\", \"Brown DL\", \"Rosand J\", \"Slowik A\", \"Schwab K\", \"Greenberg SM\", \"Silliman SL\", \"Shulman JM\", \"Selim M\", \"Worrall BB\", \"Schneider JA\", \"Ayres AM\", \"Cortellini L\", \"Jagiella JM\", \"Biffi A\"], \"date\": \"2012-01-31\"}, \"line\": 3424, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Complement system subgraph\"}, \"source\": 47, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Knockdown of clusterin in primary neurons reduced Aβ toxicity and DKK1 upregulation and, conversely, Aβ increased intracellular clusterin and decreased clusterin protein secretion, resulting in the p53-dependent induction of DKK1. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"23164821\", \"authors\": [\"To AW\", \"Brion JP\", \"Reynolds CH\", \"Leroy K\", \"Noble W\", \"Robinson M\", \"Williamson R\", \"Hooper C\", \"Causevic M\", \"Breen G\", \"Lin K\", \"Ribe EM\", \"Wroe R\", \"Malik B\", \"Kellie S\", \"Lovestone S\", \"Usardi A\", \"Stephenson J\", \"Killick R\", \"Hodges A\", \"Lunnon K\", \"Bazenet C\", \"Simons JP\", \"Furney S\", \"Nolan PM\", \"Fernandes C\", \"Dobson R\", \"Lourdusamy A\", \"Al-Shawi R\"], \"date\": \"2014-01-01\"}, \"line\": 54942, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"DKK1 subgraph\"}, \"source\": 48, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Knockdown of clusterin in primary neurons reduced Aβ toxicity and DKK1 upregulation and, conversely, Aβ increased intracellular clusterin and decreased clusterin protein secretion, resulting in the p53-dependent induction of DKK1. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"23164821\", \"authors\": [\"To AW\", \"Brion JP\", \"Reynolds CH\", \"Leroy K\", \"Noble W\", \"Robinson M\", \"Williamson R\", \"Hooper C\", \"Causevic M\", \"Breen G\", \"Lin K\", \"Ribe EM\", \"Wroe R\", \"Malik B\", \"Kellie S\", \"Lovestone S\", \"Usardi A\", \"Stephenson J\", \"Killick R\", \"Hodges A\", \"Lunnon K\", \"Bazenet C\", \"Simons JP\", \"Furney S\", \"Nolan PM\", \"Fernandes C\", \"Dobson R\", \"Lourdusamy A\", \"Al-Shawi R\"], \"date\": \"2014-01-01\"}, \"line\": 54942, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 48, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Clusterin inhibits the aggregation of A beta, which might be neuroprotective according to the aggregation-toxicity hypothesis of A beta. However, clusterin enhanced the oxidative stress of A beta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"8892344\", \"authors\": [\"Choi-Miura NH\", \"Oda T\"], \"date\": \"1996-09-01\"}, \"line\": 34615, \"annotations\": {\"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 48, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Clusterin inhibits the aggregation of A beta, which might be neuroprotective according to the aggregation-toxicity hypothesis of A beta. However, clusterin enhanced the oxidative stress of A beta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"8892344\", \"authors\": [\"Choi-Miura NH\", \"Oda T\"], \"date\": \"1996-09-01\"}, \"line\": 34615, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 48, \"target\": 4, \"key\": 1}, {\"relation\": \"directlyDecreases\", \"evidence\": \"We have previously proposed that CLU-client complexes serve as vehicles to dispose of damaged misfolded extracellular proteins in vivo via receptor-mediated endocytosis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"19996109\", \"authors\": [\"Wyatt AR\", \"Wilson MR\"], \"date\": \"2010-02-05\"}, \"line\": 34645, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"chap\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 48, \"target\": 4, \"key\": 2}, {\"relation\": \"directlyDecreases\", \"evidence\": \"We have previously proposed that CLU-client complexes serve as vehicles to dispose of damaged misfolded extracellular proteins in vivo via receptor-mediated endocytosis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"19996109\", \"authors\": [\"Wyatt AR\", \"Wilson MR\"], \"date\": \"2010-02-05\"}, \"line\": 34645, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"chap\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 48, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These data indicate that PICALM, an adaptor protein involved in clathrin-mediated endocytosis, regulates APP internalization and subsequent Abeta generation. PICALM contributes to amyloid plaque load in brain likely via its effect on Abeta metabolism.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22539346\", \"authors\": [\"Han S\", \"Lee JM\", \"Xiao Q\", \"Cirrito JR\", \"Gonzales E\", \"Gil SC\", \"Perez R\", \"Wang Y\", \"Yan P\"], \"date\": \"2012-06-15\"}, \"line\": 1574, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 49, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These data indicate that PICALM, an adaptor protein involved in clathrin-mediated endocytosis, regulates APP internalization and subsequent Abeta generation. PICALM contributes to amyloid plaque load in brain likely via its effect on Abeta metabolism.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22539346\", \"authors\": [\"Han S\", \"Lee JM\", \"Xiao Q\", \"Cirrito JR\", \"Gonzales E\", \"Gil SC\", \"Perez R\", \"Wang Y\", \"Yan P\"], \"date\": \"2012-06-15\"}, \"line\": 1574, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 49, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"SORL1 regulates and traps APP in the Golgi apparatus and reduces the production of the amyloid-beta peptide (Abeta), the main component of senile plaques in AD4. Cell biology experiments have indicated that SORL1 underexperssion leads to overproduction of Abeta3 , whereas SORL1 overexpression considerably reduces cellular APP and extracellular Abeta5. Although the underlying mechanisms of SORL1 in AD and neurodegenerative processes remain unclear, a recent comprehensive review on multiple functional domains of SORL1 suggested that SORL1 can assume various characters in AD pathogenesis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Scientific reports\", \"reference\": \"26996954\", \"authors\": [\"Tsai SJ\", \"Liu ME\", \"Chen TR\", \"Huang CC\", \"Chou KH\", \"Kao HW\", \"Yang AC\", \"Lin CP\", \"Wang YH\"], \"date\": \"2016-03-21\"}, \"line\": 53588, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 50, \"target\": 1, \"key\": 0}, {\"relation\": \"directlyDecreases\", \"evidence\": \"Using screening approaches in primary neurons, we identified brain-derived neurotrophic factor (BDNF) as a major inducer of Sorla that activates receptor gene transcription through the ERK (extracellular regulated kinase) pathway.These findings demonstrate that the beneficial effects ascribed to BDNF in APP metabolism act through induction of Sorla that encodes a negative regulator of neuronal APP processing\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"20007471\", \"authors\": [\"Rohe M\", \"Synowitz M\", \"Glass R\", \"Paul SM\", \"Nykjaer A\", \"Willnow TE\"], \"date\": \"2009-12-09\"}, \"line\": 36564, \"annotations\": {\"Subgraph\": \"MAPK-ERK subgraph\"}, \"source\": 50, \"target\": 4, \"key\": 0}, {\"relation\": \"directlyDecreases\", \"evidence\": \"Using screening approaches in primary neurons, we identified brain-derived neurotrophic factor (BDNF) as a major inducer of Sorla that activates receptor gene transcription through the ERK (extracellular regulated kinase) pathway.These findings demonstrate that the beneficial effects ascribed to BDNF in APP metabolism act through induction of Sorla that encodes a negative regulator of neuronal APP processing\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"20007471\", \"authors\": [\"Rohe M\", \"Synowitz M\", \"Glass R\", \"Paul SM\", \"Nykjaer A\", \"Willnow TE\"], \"date\": \"2009-12-09\"}, \"line\": 36564, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 50, \"target\": 4, \"key\": 1}, {\"relation\": \"directlyDecreases\", \"evidence\": \"SORLA is proposed to act as a retention factor for APP in the TGN, preventing the release of precursor molecules into the processing pathways. Consequently, over-expression of SORLA in neurons prevents the targeting of APP from TGN to the cell surface and to endosomes and reduces the production of Abeta peptides [9-11]. The importance of SORLA for AD is further supported by low levels of receptor expression seen in patients suffering from the disease\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMC systems biology\", \"reference\": \"22727043\", \"authors\": [\"Lao A\", \"Schmidt V\", \"Schmitz Y\", \"Willnow TE\", \"Wolkenhauer O\"], \"date\": \"2012-06-22\"}, \"line\": 34495, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 50, \"target\": 4, \"key\": 2}, {\"relation\": \"directlyDecreases\", \"evidence\": \"SORLA is proposed to act as a retention factor for APP in the TGN, preventing the release of precursor molecules into the processing pathways. Consequently, over-expression of SORLA in neurons prevents the targeting of APP from TGN to the cell surface and to endosomes and reduces the production of Abeta peptides [9-11]. The importance of SORLA for AD is further supported by low levels of receptor expression seen in patients suffering from the disease\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMC systems biology\", \"reference\": \"22727043\", \"authors\": [\"Lao A\", \"Schmidt V\", \"Schmitz Y\", \"Willnow TE\", \"Wolkenhauer O\"], \"date\": \"2012-06-22\"}, \"line\": 34495, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 50, \"target\": 4, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"SORL1 encodes the Sortilin-related receptor LR11/SorLA, a protein involved in the control of amyloid beta peptide production\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"22472873\", \"authors\": [\"Dartigues JF\", \"Campion D\", \"Berr C\", \"PHRC GMAJ Collaborators.\", \"Frebourg T\", \"Boutoleau-Bretonnière C\", \"Thauvin C\", \"Bombois S\", \"Hannequin D\", \"Dauchel H\", \"Etcharry-Bouyx F\", \"Croisile B\", \"Michon A\", \"Thomas-Anterion C\", \"Pariente J\", \"Paquet C\", \"Rovelet-Lecrux A\", \"Amouyel P\", \"Legallic S\", \"Rousseau S\", \"Wallon D\", \"Coutant S\", \"Pottier C\", \"Lambert JC\"], \"date\": \"2012-09-01\"}, \"line\": 34525, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 50, \"target\": 4, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"SORL1 encodes the Sortilin-related receptor LR11/SorLA, a protein involved in the control of amyloid beta peptide production\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"22472873\", \"authors\": [\"Dartigues JF\", \"Campion D\", \"Berr C\", \"PHRC GMAJ Collaborators.\", \"Frebourg T\", \"Boutoleau-Bretonnière C\", \"Thauvin C\", \"Bombois S\", \"Hannequin D\", \"Dauchel H\", \"Etcharry-Bouyx F\", \"Croisile B\", \"Michon A\", \"Thomas-Anterion C\", \"Pariente J\", \"Paquet C\", \"Rovelet-Lecrux A\", \"Amouyel P\", \"Legallic S\", \"Rousseau S\", \"Wallon D\", \"Coutant S\", \"Pottier C\", \"Lambert JC\"], \"date\": \"2012-09-01\"}, \"line\": 34525, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 50, \"target\": 4, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Akt substrates such as mammalian target of rapamycin (mTOR; Ser2448) and decreased levels of cell-cycle inhibitors (p27kip1) are found in AD temporal cortex when compared to controls . GSK-3a has been implicated in the production of Aï¿½ peptide while increased GSK-3ï¿½ activity has been implicated in tau hyperphosphorylation and neuronal cell death\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 6418, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 51, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Akt substrates such as mammalian target of rapamycin (mTOR; Ser2448) and decreased levels of cell-cycle inhibitors (p27kip1) are found in AD temporal cortex when compared to controls . GSK-3a has been implicated in the production of Aï¿½ peptide while increased GSK-3ï¿½ activity has been implicated in tau hyperphosphorylation and neuronal cell death\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 6418, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 51, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31387, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 51, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31387, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 51, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31387, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 51, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Acetylcholinesterase (AChE), an enzyme involved in the hydrolysis of the neurotransmitter acetylcholine, consistently colocalizes with the amyloid deposits characteristic of Alzheimer's disease and may contribute to the generation of amyloid proteins and/or physically affect fibril assembly.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular biology\", \"reference\": \"9325095\", \"authors\": [\"Garrido J\", \"Alvarez A\", \"Opazo C\", \"Alarcón R\", \"Inestrosa NC\"], \"date\": \"1997-09-26\"}, \"line\": 23535, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 52, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Acetylcholinesterase (AChE), an enzyme involved in the hydrolysis of the neurotransmitter acetylcholine, consistently colocalizes with the amyloid deposits characteristic of Alzheimer's disease and may contribute to the generation of amyloid proteins and/or physically affect fibril assembly.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular biology\", \"reference\": \"9325095\", \"authors\": [\"Garrido J\", \"Alvarez A\", \"Opazo C\", \"Alarcón R\", \"Inestrosa NC\"], \"date\": \"1997-09-26\"}, \"line\": 23535, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 52, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" The fact that acetylcholinesterase accelerates amyloid formation and the effect is sensitive to peripherical anionic site blockers of the enzyme, suggests that specific and new EC 3.1.1.7 (acetylcholinesterase) inhibitors may well provide an attractive possibility for treating Alzheimer's disease. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The FEBS journal\", \"reference\": \"18205831\", \"authors\": [\"Inestrosa NC\", \"Dinamarca MC\", \"Alvarez A\"], \"date\": \"2008-02-01\"}, \"line\": 23965, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 52, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" The fact that acetylcholinesterase accelerates amyloid formation and the effect is sensitive to peripherical anionic site blockers of the enzyme, suggests that specific and new EC 3.1.1.7 (acetylcholinesterase) inhibitors may well provide an attractive possibility for treating Alzheimer's disease. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The FEBS journal\", \"reference\": \"18205831\", \"authors\": [\"Inestrosa NC\", \"Dinamarca MC\", \"Alvarez A\"], \"date\": \"2008-02-01\"}, \"line\": 23965, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 52, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Acetylcholinesterase (AChE), an enzyme involved in the hydrolysis of the neurotransmitter acetylcholine, consistently colocalizes with the amyloid deposits characteristic of Alzheimer's disease and may contribute to the generation of amyloid proteins and/or physically affect fibril assembly.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular biology\", \"reference\": \"9325095\", \"authors\": [\"Garrido J\", \"Alvarez A\", \"Opazo C\", \"Alarcón R\", \"Inestrosa NC\"], \"date\": \"1997-09-26\"}, \"line\": 23991, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 52, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Acetylcholinesterase (AChE), an enzyme involved in the hydrolysis of the neurotransmitter acetylcholine, consistently colocalizes with the amyloid deposits characteristic of Alzheimer's disease and may contribute to the generation of amyloid proteins and/or physically affect fibril assembly.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular biology\", \"reference\": \"9325095\", \"authors\": [\"Garrido J\", \"Alvarez A\", \"Opazo C\", \"Alarcón R\", \"Inestrosa NC\"], \"date\": \"1997-09-26\"}, \"line\": 23991, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 52, \"target\": 1, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"As far as the metabolism of amyloid precursor protein (APP) in late-onset sporadic Alzheimer disease is concerned, neuronal insulin receptor dysfunction may result in the intracellular accumulation of Abeta and in subsequent cellular damage.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"European journal of pharmacology\", \"reference\": \"15094078\", \"authors\": [\"Hoyer S\"], \"date\": \"2004-04-19\"}, \"line\": 19931, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 53, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"As far as the metabolism of amyloid precursor protein (APP) in late-onset sporadic Alzheimer disease is concerned, neuronal insulin receptor dysfunction may result in the intracellular accumulation of Abeta and in subsequent cellular damage.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"European journal of pharmacology\", \"reference\": \"15094078\", \"authors\": [\"Hoyer S\"], \"date\": \"2004-04-19\"}, \"line\": 19931, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Insulin signal transduction\"}, \"source\": 53, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"First, Pin1 inhibits the production of Abeta, and enhances the activity of eNOS. Second, Abeta and eNOS form a mutual inhibition system. Third, the well-balanced feedback signaling loop avoids the development of AD, HTN, and CAA by inhibiting the frequent pathological characteristics of these diseases, including Abeta deposition in cerebral microvessels and cerebral microbleeds.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"24332564\", \"authors\": [\"Xu ZX\", \"Du WT\", \"Wang JZ\", \"Sun XW\", \"Zhu WD\", \"Zhang HY\", \"Wang XH\"], \"date\": \"2014-02-01\"}, \"line\": 16993, \"annotations\": {\"Anatomy\": \"cerebral blood vessel\", \"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 54, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"First, Pin1 inhibits the production of Abeta, and enhances the activity of eNOS. Second, Abeta and eNOS form a mutual inhibition system. Third, the well-balanced feedback signaling loop avoids the development of AD, HTN, and CAA by inhibiting the frequent pathological characteristics of these diseases, including Abeta deposition in cerebral microvessels and cerebral microbleeds.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"24332564\", \"authors\": [\"Xu ZX\", \"Du WT\", \"Wang JZ\", \"Sun XW\", \"Zhu WD\", \"Zhang HY\", \"Wang XH\"], \"date\": \"2014-02-01\"}, \"line\": 16993, \"annotations\": {\"Anatomy\": \"cerebral blood vessel\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 54, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"To explore the molecular mechanism underlying AD, HTN, and CAA, we hypothesize a feedback signaling loop consisted of Pin1, eNOS, and Abeta. Pin1 and eNOS mainly inhibit Abeta deposition in cerebral vessels, cerebral microbleeds, and elevation of blood pressure, preventing the development of AD, HTN, and CAA, however, Abeta plays an opposite role and aggravates these diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"24332564\", \"authors\": [\"Xu ZX\", \"Du WT\", \"Wang JZ\", \"Sun XW\", \"Zhu WD\", \"Zhang HY\", \"Wang XH\"], \"date\": \"2014-02-01\"}, \"line\": 17026, \"annotations\": {\"Anatomy\": \"cerebral blood vessel\", \"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 54, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"To explore the molecular mechanism underlying AD, HTN, and CAA, we hypothesize a feedback signaling loop consisted of Pin1, eNOS, and Abeta. Pin1 and eNOS mainly inhibit Abeta deposition in cerebral vessels, cerebral microbleeds, and elevation of blood pressure, preventing the development of AD, HTN, and CAA, however, Abeta plays an opposite role and aggravates these diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"24332564\", \"authors\": [\"Xu ZX\", \"Du WT\", \"Wang JZ\", \"Sun XW\", \"Zhu WD\", \"Zhang HY\", \"Wang XH\"], \"date\": \"2014-02-01\"}, \"line\": 17026, \"annotations\": {\"Anatomy\": \"cerebral blood vessel\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 54, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in Abeta-secretase and, thus, an increase of Abeta generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 1140, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in Abeta-secretase and, thus, an increase of Abeta generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 1140, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in Abeta-secretase and, thus, an increase of Abeta generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 1147, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in Abeta-secretase and, thus, an increase of Abeta generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 1147, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Response to oxidative stress\"}, \"source\": 55, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Ex vivo studies show that the levels of BACE1 are tightly regulated by the ATases. Specifically, up-regulation of ATase1 and ATase2 increases the levels of BACE1 and the generation of Abeta while siRNA-mediated down-regulation of either transferase achieves the opposite effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22267734\", \"authors\": [\"Ding Y\", \"Pehar M\", \"Ko MH\", \"Luo Y\", \"Puglielli L\", \"Kotch F\", \"Peters NR\", \"Salamat SM\"], \"date\": \"2012-03-09\"}, \"line\": 3045, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"MeSHAnatomy\": \"Neurons\", \"Cell\": \"microglial cell\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Ex vivo studies show that the levels of BACE1 are tightly regulated by the ATases. Specifically, up-regulation of ATase1 and ATase2 increases the levels of BACE1 and the generation of Abeta while siRNA-mediated down-regulation of either transferase achieves the opposite effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22267734\", \"authors\": [\"Ding Y\", \"Pehar M\", \"Ko MH\", \"Luo Y\", \"Puglielli L\", \"Kotch F\", \"Peters NR\", \"Salamat SM\"], \"date\": \"2012-03-09\"}, \"line\": 3045, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"MeSHAnatomy\": \"Neurons\", \"Cell\": \"microglial cell\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Interestingly, Fe65 acts as a functional linker between APP and LRP (another type I membrane protein containing two NPXY endocytosis motifs) in modulating endocytic APP trafficking and Abeta production. A conformational change introduced by phosphorylation at Thr668 (14 amino acids proximal to the YENPTY motif) interferes with Fe65 binding to APP and facilitates BACE1 and gamma-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18650430\", \"authors\": [\"Thinakaran G\", \"Koo EH\"], \"date\": \"2008-10-31\"}, \"line\": 3232, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"Interestingly, Fe65 acts as a functional linker between APP and LRP (another type I membrane protein containing two NPXY endocytosis motifs) in modulating endocytic APP trafficking and Abeta production. A conformational change introduced by phosphorylation at Thr668 (14 amino acids proximal to the YENPTY motif) interferes with Fe65 binding to APP and facilitates BACE1 and gamma-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18650430\", \"authors\": [\"Thinakaran G\", \"Koo EH\"], \"date\": \"2008-10-31\"}, \"line\": 3232, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"Interestingly, Fe65 acts as a functional linker between APP and LRP (another type I membrane protein containing two NPXY endocytosis motifs) in modulating endocytic APP trafficking and Abeta production. A conformational change introduced by phosphorylation at Thr668 (14 amino acids proximal to the YENPTY motif) interferes with Fe65 binding to APP and facilitates BACE1 and gamma-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18650430\", \"authors\": [\"Thinakaran G\", \"Koo EH\"], \"date\": \"2008-10-31\"}, \"line\": 3232, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17428983\", \"authors\": [\"He X\", \"Cooley K\", \"Chung CH\", \"Dashti N\", \"Tang J\"], \"date\": \"2007-04-11\"}, \"line\": 4779, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 9}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17428983\", \"authors\": [\"He X\", \"Cooley K\", \"Chung CH\", \"Dashti N\", \"Tang J\"], \"date\": \"2007-04-11\"}, \"line\": 4779, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 10}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17428983\", \"authors\": [\"He X\", \"Cooley K\", \"Chung CH\", \"Dashti N\", \"Tang J\"], \"date\": \"2007-04-11\"}, \"line\": 4779, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 11}, {\"relation\": \"increases\", \"evidence\": \"The level of Abeta-site APP-cleaving enzyme 1 (BACE1) has been documented to increase in the brains of patients with Alzheimer's disease, which has resulted in elevation of Abeta-amyloid (Abeta) peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"European journal of pharmacology\", \"reference\": \"22166376\", \"authors\": [\"Zhang J\", \"Chen XC\", \"Pan XD\", \"Lin N\", \"Zhu YG\", \"Lin ZY\", \"Chen LM\"], \"date\": \"2012-01-30\"}, \"line\": 5048, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 12}, {\"relation\": \"increases\", \"evidence\": \"The level of Abeta-site APP-cleaving enzyme 1 (BACE1) has been documented to increase in the brains of patients with Alzheimer's disease, which has resulted in elevation of Abeta-amyloid (Abeta) peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"European journal of pharmacology\", \"reference\": \"22166376\", \"authors\": [\"Zhang J\", \"Chen XC\", \"Pan XD\", \"Lin N\", \"Zhu YG\", \"Lin ZY\", \"Chen LM\"], \"date\": \"2012-01-30\"}, \"line\": 5048, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 13}, {\"relation\": \"increases\", \"evidence\": \"CerebraiiR/IGF-1 signaling in APP metabolism. Binding of insulin and IGF-1 to their receptors leads to autophosphorylation of the -subunit of the IR/IGF-IR and recruitment of IRS-1 /-2. IRS proteins activate mainly two pathways, the PI3 kinase pathway, and the RAS-RAF-MAP kinase cascade. Tau phosphorylation via IR/IGF-1 signaling is influenced by GSK-3and the tau phosphatase PP2A. InterÂ­ estingly, mTOR signaling couples the activity ofPP2A and GSK-3in a way that the activities of both enzymes change always in the same direction. suggesting that only a dysregulation of either PP2A or GSK-3induces tau hyperphosphorylation. The PI3K pathway as well as the RAS-RAF-MAPK cascade regulate different transcription factors involved in transcriptional regulation of metabolism and clearance of Â­ amyloid. APP is cleaved by a.-, -secretase (BACE-1) and y-secretase (presenilin). - and subsequent y-cleavage of APP leads to generation of -amyloid 1 -4o/ t-42â€¢ IGF-1 signaling promotes a switch lrom TrkA to p75NTR expression leading to increased -secretase activity due to an upregulation of BACE-1 expression. Recent data suggest that not only BACE-1 is int1uenced by IRIIGF- IR mediated signals but also a.Â­ secretase activity is stimulated by the PI3K pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19519303\", \"authors\": [\"Freude S\", \"Schilbach K\", \"Schubert M\"], \"date\": \"2009-06-01\"}, \"line\": 8272, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 14}, {\"relation\": \"increases\", \"evidence\": \"CerebraiiR/IGF-1 signaling in APP metabolism. Binding of insulin and IGF-1 to their receptors leads to autophosphorylation of the -subunit of the IR/IGF-IR and recruitment of IRS-1 /-2. IRS proteins activate mainly two pathways, the PI3 kinase pathway, and the RAS-RAF-MAP kinase cascade. Tau phosphorylation via IR/IGF-1 signaling is influenced by GSK-3and the tau phosphatase PP2A. InterÂ­ estingly, mTOR signaling couples the activity ofPP2A and GSK-3in a way that the activities of both enzymes change always in the same direction. suggesting that only a dysregulation of either PP2A or GSK-3induces tau hyperphosphorylation. The PI3K pathway as well as the RAS-RAF-MAPK cascade regulate different transcription factors involved in transcriptional regulation of metabolism and clearance of Â­ amyloid. APP is cleaved by a.-, -secretase (BACE-1) and y-secretase (presenilin). - and subsequent y-cleavage of APP leads to generation of -amyloid 1 -4o/ t-42â€¢ IGF-1 signaling promotes a switch lrom TrkA to p75NTR expression leading to increased -secretase activity due to an upregulation of BACE-1 expression. Recent data suggest that not only BACE-1 is int1uenced by IRIIGF- IR mediated signals but also a.Â­ secretase activity is stimulated by the PI3K pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19519303\", \"authors\": [\"Freude S\", \"Schilbach K\", \"Schubert M\"], \"date\": \"2009-06-01\"}, \"line\": 8272, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 15}, {\"relation\": \"increases\", \"evidence\": \"CerebraiiR/IGF-1 signaling in APP metabolism. Binding of insulin and IGF-1 to their receptors leads to autophosphorylation of the -subunit of the IR/IGF-IR and recruitment of IRS-1 /-2. IRS proteins activate mainly two pathways, the PI3 kinase pathway, and the RAS-RAF-MAP kinase cascade. Tau phosphorylation via IR/IGF-1 signaling is influenced by GSK-3and the tau phosphatase PP2A. InterÂ­ estingly, mTOR signaling couples the activity ofPP2A and GSK-3in a way that the activities of both enzymes change always in the same direction. suggesting that only a dysregulation of either PP2A or GSK-3induces tau hyperphosphorylation. The PI3K pathway as well as the RAS-RAF-MAPK cascade regulate different transcription factors involved in transcriptional regulation of metabolism and clearance of Â­ amyloid. APP is cleaved by a.-, -secretase (BACE-1) and y-secretase (presenilin). - and subsequent y-cleavage of APP leads to generation of -amyloid 1 -4o/ t-42â€¢ IGF-1 signaling promotes a switch lrom TrkA to p75NTR expression leading to increased -secretase activity due to an upregulation of BACE-1 expression. Recent data suggest that not only BACE-1 is int1uenced by IRIIGF- IR mediated signals but also a.Â­ secretase activity is stimulated by the PI3K pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19519303\", \"authors\": [\"Freude S\", \"Schilbach K\", \"Schubert M\"], \"date\": \"2009-06-01\"}, \"line\": 8272, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Nerve growth factor subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 16}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 10174, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 17}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 10174, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 18}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 10174, \"annotations\": {\"Subgraph\": \"Response to oxidative stress\"}, \"source\": 55, \"target\": 1, \"key\": 19}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 10174, \"annotations\": {\"Subgraph\": \"Apoptosis signaling subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 20}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 10178, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 21}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 10178, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 22}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 10178, \"annotations\": {\"Subgraph\": \"MAPK-ERK subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 23}, {\"relation\": \"increases\", \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in ÃƒÆ’Ã…Â¸-secretase and, thus, an increase of AÃƒÆ’Ã…Â¸ generation in neuronal cells expressing mutant PS2\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"22249458\", \"authors\": [\"Park MH\", \"Han SB\", \"Hwang DY\", \"Hong JT\", \"Jin HW\", \"Choi DY\", \"Oh KW\", \"Yoo HS\", \"Han JY\"], \"date\": \"2012-02-01\"}, \"line\": 10178, \"annotations\": {\"Subgraph\": \"Response to oxidative stress\"}, \"source\": 55, \"target\": 1, \"key\": 24}, {\"relation\": \"decreases\", \"evidence\": \"In this study, we aimed to investigate the possibility of P-gp as a potential therapeutic target for Alzheimer's disease by examining the impact of P-gp up-regulation on the clearance of Abeta, a neuropathological hallmark of Alzheimer's disease.Uptake studies for-radiolabelled Abeta Approximately 10-35% decrease in Abeta intracellular accumulation was observed in cells treated with rifampicin, dexamethasone, caffeine, verapamil, hyperforin, β-estradiol and pentylenetetrazole compared with control.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of pharmacy and pharmacology\", \"reference\": \"21718295\", \"authors\": [\"Abuznait AH\", \"Cain C\", \"Ingram D\", \"Kaddoumi A\", \"Burk D\"], \"date\": \"2011-08-01\"}, \"line\": 17566, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 25}, {\"relation\": \"decreases\", \"evidence\": \"In this study, we aimed to investigate the possibility of P-gp as a potential therapeutic target for Alzheimer's disease by examining the impact of P-gp up-regulation on the clearance of Abeta, a neuropathological hallmark of Alzheimer's disease.Uptake studies for-radiolabelled Abeta Approximately 10-35% decrease in Abeta intracellular accumulation was observed in cells treated with rifampicin, dexamethasone, caffeine, verapamil, hyperforin, β-estradiol and pentylenetetrazole compared with control.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of pharmacy and pharmacology\", \"reference\": \"21718295\", \"authors\": [\"Abuznait AH\", \"Cain C\", \"Ingram D\", \"Kaddoumi A\", \"Burk D\"], \"date\": \"2011-08-01\"}, \"line\": 17566, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 26}, {\"relation\": \"increases\", \"evidence\": \"RAGE stimulates functional BACE1 expression through NFAT1 activation, resulting in more Abeta production and deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"19332646\", \"authors\": [\"Cho HJ\", \"Son SM\", \"Jin SM\", \"Hong HS\", \"Shin DH\", \"Kim SJ\", \"Huh K\", \"Mook-Jung I\"], \"date\": \"2009-08-01\"}, \"line\": 23771, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 27}, {\"relation\": \"increases\", \"evidence\": \"RAGE stimulates functional BACE1 expression through NFAT1 activation, resulting in more Abeta production and deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"19332646\", \"authors\": [\"Cho HJ\", \"Son SM\", \"Jin SM\", \"Hong HS\", \"Shin DH\", \"Kim SJ\", \"Huh K\", \"Mook-Jung I\"], \"date\": \"2009-08-01\"}, \"line\": 23771, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 28}, {\"relation\": \"increases\", \"evidence\": \"RAGE stimulates functional BACE1 expression through NFAT1 activation, resulting in more Abeta production and deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"19332646\", \"authors\": [\"Cho HJ\", \"Son SM\", \"Jin SM\", \"Hong HS\", \"Shin DH\", \"Kim SJ\", \"Huh K\", \"Mook-Jung I\"], \"date\": \"2009-08-01\"}, \"line\": 23771, \"annotations\": {\"Subgraph\": \"T cells signaling\"}, \"source\": 55, \"target\": 1, \"key\": 29}, {\"relation\": \"increases\", \"evidence\": \"RAGE stimulates functional BACE1 expression through NFAT1 activation, resulting in more Abeta production and deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"19332646\", \"authors\": [\"Cho HJ\", \"Son SM\", \"Jin SM\", \"Hong HS\", \"Shin DH\", \"Kim SJ\", \"Huh K\", \"Mook-Jung I\"], \"date\": \"2009-08-01\"}, \"line\": 23771, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 30}, {\"relation\": \"increases\", \"evidence\": \"Moreover, NPD1 suppresses Abeta42 peptide shedding by down-regulating beta-secretase-1 (BACE1) while activating the β-secretase ADAM10 and up-regulating sAPPalpha, thus shifting the cleavage of betaAPP holoenzyme from an amyloidogenic into the non-amyloidogenic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"21246057\", \"authors\": [\"Zhao Y\", \"Calon F\", \"Julien C\", \"Winkler JW\", \"Petasis NA\", \"Lukiw WJ\", \"Bazan NG\"], \"date\": \"2011-01-05\"}, \"line\": 24053, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 31}, {\"relation\": \"increases\", \"evidence\": \"Moreover, NPD1 suppresses Abeta42 peptide shedding by down-regulating beta-secretase-1 (BACE1) while activating the β-secretase ADAM10 and up-regulating sAPPalpha, thus shifting the cleavage of betaAPP holoenzyme from an amyloidogenic into the non-amyloidogenic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"21246057\", \"authors\": [\"Zhao Y\", \"Calon F\", \"Julien C\", \"Winkler JW\", \"Petasis NA\", \"Lukiw WJ\", \"Bazan NG\"], \"date\": \"2011-01-05\"}, \"line\": 24053, \"annotations\": {\"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 32}, {\"relation\": \"increases\", \"evidence\": \"Moreover, NPD1 suppresses Abeta42 peptide shedding by down-regulating beta-secretase-1 (BACE1) while activating the β-secretase ADAM10 and up-regulating sAPPalpha, thus shifting the cleavage of betaAPP holoenzyme from an amyloidogenic into the non-amyloidogenic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"21246057\", \"authors\": [\"Zhao Y\", \"Calon F\", \"Julien C\", \"Winkler JW\", \"Petasis NA\", \"Lukiw WJ\", \"Bazan NG\"], \"date\": \"2011-01-05\"}, \"line\": 24053, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 33}, {\"relation\": \"increases\", \"evidence\": \"Moreover, NPD1 suppresses Abeta42 peptide shedding by down-regulating beta-secretase-1 (BACE1) while activating the β-secretase ADAM10 and up-regulating sAPPalpha, thus shifting the cleavage of betaAPP holoenzyme from an amyloidogenic into the non-amyloidogenic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"21246057\", \"authors\": [\"Zhao Y\", \"Calon F\", \"Julien C\", \"Winkler JW\", \"Petasis NA\", \"Lukiw WJ\", \"Bazan NG\"], \"date\": \"2011-01-05\"}, \"line\": 24053, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 34}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the alphaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular neurobiology\", \"reference\": \"21431475\", \"authors\": [\"Stark DT\", \"Bazan NG\"], \"date\": \"2011-04-01\"}, \"line\": 24095, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 35}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the alphaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular neurobiology\", \"reference\": \"21431475\", \"authors\": [\"Stark DT\", \"Bazan NG\"], \"date\": \"2011-04-01\"}, \"line\": 24095, \"annotations\": {\"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 36}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the alphaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular neurobiology\", \"reference\": \"21431475\", \"authors\": [\"Stark DT\", \"Bazan NG\"], \"date\": \"2011-04-01\"}, \"line\": 24095, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 37}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the alphaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular neurobiology\", \"reference\": \"21431475\", \"authors\": [\"Stark DT\", \"Bazan NG\"], \"date\": \"2011-04-01\"}, \"line\": 24095, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 38}, {\"relation\": \"increases\", \"evidence\": \"A pathogenic mutation at codons 670/671 in APP (APP Swedish) leads to enhanced cleavage at the beta-secretase scissile bond and increased Abeta formation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"10500121\", \"authors\": [\"Sinha S\", \"Lieberburg I\"], \"date\": \"1999-09-28\"}, \"line\": 25152, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 39}, {\"relation\": \"increases\", \"evidence\": \"A pathogenic mutation at codons 670/671 in APP (APP Swedish) leads to enhanced cleavage at the beta-secretase scissile bond and increased Abeta formation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"10500121\", \"authors\": [\"Sinha S\", \"Lieberburg I\"], \"date\": \"1999-09-28\"}, \"line\": 25152, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 40}, {\"relation\": \"increases\", \"evidence\": \"The novel transmembrane aspartic protease BACE (for Beta-site APP Cleaving Enzyme) is the beta-secretase that cleaves amyloid precursor protein to initiate beta-amyloid formation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"10956649\", \"authors\": [\"Haniu M\", \"Citron M\", \"Vassar R\", \"Louis JC\", \"Bennett BD\", \"Denis P\", \"Teplow DB\", \"Kahn S\"], \"date\": \"2000-12-01\"}, \"line\": 25183, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 41}, {\"relation\": \"increases\", \"evidence\": \"The novel transmembrane aspartic protease BACE (for Beta-site APP Cleaving Enzyme) is the beta-secretase that cleaves amyloid precursor protein to initiate beta-amyloid formation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"10956649\", \"authors\": [\"Haniu M\", \"Citron M\", \"Vassar R\", \"Louis JC\", \"Bennett BD\", \"Denis P\", \"Teplow DB\", \"Kahn S\"], \"date\": \"2000-12-01\"}, \"line\": 25183, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 42}, {\"relation\": \"increases\", \"evidence\": \"A beta is generated by the sequential intracellular cleavage of APP by beta-secretase to generate the N-terminal end of A beta, and intramembranous cleavage by gamma-secretase to generate the C-terminal end.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Histology and histopathology\", \"reference\": \"11813874\", \"authors\": [\"Takahashi RH\", \"Nam EE\", \"Edgar M\", \"Gouras GK\"], \"date\": \"2002-01-01\"}, \"line\": 25226, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 43}, {\"relation\": \"increases\", \"evidence\": \"A beta is generated by the sequential intracellular cleavage of APP by beta-secretase to generate the N-terminal end of A beta, and intramembranous cleavage by gamma-secretase to generate the C-terminal end.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Histology and histopathology\", \"reference\": \"11813874\", \"authors\": [\"Takahashi RH\", \"Nam EE\", \"Edgar M\", \"Gouras GK\"], \"date\": \"2002-01-01\"}, \"line\": 25226, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 44}, {\"relation\": \"increases\", \"evidence\": \"beta-Secretase, a beta-site amyloid precursor protein (APP) cleaving enzyme (BACE), participates in the secretion of beta-amyloid peptides (Abeta), the major components of the toxic amyloid plaques found in the brains of patients with Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta biochimica Polonica\", \"reference\": \"15218540\", \"authors\": [\"Nawrot B\"], \"date\": \"2004-01-01\"}, \"line\": 25257, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 45}, {\"relation\": \"increases\", \"evidence\": \"beta-Secretase, a beta-site amyloid precursor protein (APP) cleaving enzyme (BACE), participates in the secretion of beta-amyloid peptides (Abeta), the major components of the toxic amyloid plaques found in the brains of patients with Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta biochimica Polonica\", \"reference\": \"15218540\", \"authors\": [\"Nawrot B\"], \"date\": \"2004-01-01\"}, \"line\": 25257, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 46}, {\"relation\": \"increases\", \"evidence\": \"Amyloid-beta (Abeta) the primary component of the senile plaques found in Alzheimer's disease (AD) is generated by the rate-limiting cleavage of amyloid precursor protein (APP) by beta-secretase followed by gamma-secretase cleavage.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"15452128\", \"authors\": [\"Chiocco MJ\", \"Kulnane LS\", \"Younkin L\", \"Younkin S\", \"Evin G\", \"Lamb BT\"], \"date\": \"2004-12-10\"}, \"line\": 25284, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 47}, {\"relation\": \"increases\", \"evidence\": \"Amyloid-beta (Abeta) the primary component of the senile plaques found in Alzheimer's disease (AD) is generated by the rate-limiting cleavage of amyloid precursor protein (APP) by beta-secretase followed by gamma-secretase cleavage.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"15452128\", \"authors\": [\"Chiocco MJ\", \"Kulnane LS\", \"Younkin L\", \"Younkin S\", \"Evin G\", \"Lamb BT\"], \"date\": \"2004-12-10\"}, \"line\": 25284, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 48}, {\"relation\": \"increases\", \"evidence\": \"The generation of beta-amyloid peptides requires the enzymatic activity of the beta-site APP-cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of developmental neuroscience : the official journal of the International Society for Developmental Neuroscience\", \"reference\": \"15465276\", \"authors\": [\"Rossner S\"], \"date\": \"2004-11-01\"}, \"line\": 25297, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 49}, {\"relation\": \"increases\", \"evidence\": \"The generation of beta-amyloid peptides requires the enzymatic activity of the beta-site APP-cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of developmental neuroscience : the official journal of the International Society for Developmental Neuroscience\", \"reference\": \"15465276\", \"authors\": [\"Rossner S\"], \"date\": \"2004-11-01\"}, \"line\": 25297, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 50}, {\"relation\": \"increases\", \"evidence\": \"The generation of beta-amyloid peptides by proteolytical processing of the amyloid precursor protein (APP) requires the enzymatic activity of the beta-site APP cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"15663471\", \"authors\": [\"Rossner S\", \"Lange-Dohna C\", \"Zeitschel U\", \"Perez-Polo JR\"], \"date\": \"2005-01-01\"}, \"line\": 25312, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 51}, {\"relation\": \"increases\", \"evidence\": \"The generation of beta-amyloid peptides by proteolytical processing of the amyloid precursor protein (APP) requires the enzymatic activity of the beta-site APP cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"15663471\", \"authors\": [\"Rossner S\", \"Lange-Dohna C\", \"Zeitschel U\", \"Perez-Polo JR\"], \"date\": \"2005-01-01\"}, \"line\": 25312, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 52}, {\"relation\": \"increases\", \"evidence\": \"Using an assay that incorporates full-length recombinant APP as a substrate for beta-secretase (BACE), we have identified a series of compounds that inhibit APP processing, but do not affect the cleavage of peptide substrates by BACE1.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"15737955\", \"authors\": [\"Espeseth AS\", \"Xu M\", \"Huang Q\", \"Coburn CA\", \"Jones KL\", \"Ferrer M\", \"Zuck PD\", \"Strulovici B\", \"Price EA\", \"Wu G\", \"Wolfe AL\", \"Lineberger JE\", \"Sardana M\", \"Tugusheva K\", \"Pietrak BL\", \"Crouthamel MC\", \"Lai MT\", \"Dodson EC\", \"Bazzo R\", \"Shi XP\", \"Simon AJ\", \"Li Y\", \"Hazuda DJ\"], \"date\": \"2005-05-06\"}, \"line\": 25344, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 53}, {\"relation\": \"increases\", \"evidence\": \"Sequential processing of amyloid precursor protein (APP) by membrane-bound proteases, BACE1 and gamma-secretase, plays a crucial role in the pathogenesis of Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"15824102\", \"authors\": [\"Wong HK\", \"Sakurai T\", \"Oyama F\", \"Kaneko K\", \"Wada K\", \"Miyazaki H\", \"Kurosawa M\", \"De Strooper B\", \"Saftig P\", \"Nukina N\"], \"date\": \"2005-06-17\"}, \"line\": 25354, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 54}, {\"relation\": \"increases\", \"evidence\": \"Sequential processing of amyloid precursor protein (APP) by membrane-bound proteases, BACE1 and gamma-secretase, plays a crucial role in the pathogenesis of Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"15824102\", \"authors\": [\"Wong HK\", \"Sakurai T\", \"Oyama F\", \"Kaneko K\", \"Wada K\", \"Miyazaki H\", \"Kurosawa M\", \"De Strooper B\", \"Saftig P\", \"Nukina N\"], \"date\": \"2005-06-17\"}, \"line\": 25354, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 55}, {\"relation\": \"increases\", \"evidence\": \"gamma-Secretase is a membrane protein complex that cleaves the beta-amyloid precursor protein (APP) within the transmembrane region, after prior processing by beta-secretase, producing amyloid beta peptides Abeta(40) and Abeta(42).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"15890777\", \"authors\": [\"Zhou S\", \"Zhou H\", \"Walian PJ\", \"Jap BK\"], \"date\": \"2005-05-24\"}, \"line\": 25362, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 56}, {\"relation\": \"increases\", \"evidence\": \"Release of Abeta from the amyloid precursor protein (APP) requires proteolysis by the beta-site APP-cleaving enzyme (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"16027115\", \"authors\": [\"Rockenstein E\", \"Mante M\", \"Alford M\", \"Adame A\", \"Crews L\", \"Hashimoto M\", \"Esposito L\", \"Mucke L\", \"Masliah E\"], \"date\": \"2005-09-23\"}, \"line\": 25371, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 57}, {\"relation\": \"increases\", \"evidence\": \"Release of Abeta from the amyloid precursor protein (APP) requires proteolysis by the beta-site APP-cleaving enzyme (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"16027115\", \"authors\": [\"Rockenstein E\", \"Mante M\", \"Alford M\", \"Adame A\", \"Crews L\", \"Hashimoto M\", \"Esposito L\", \"Mucke L\", \"Masliah E\"], \"date\": \"2005-09-23\"}, \"line\": 25371, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 58}, {\"relation\": \"increases\", \"evidence\": \"Abeta is produced from beta-amyloid precursor protein (APP) by beta-secretase and gamma-secretase. beta-Secretase has been identified as beta-site APP cleaving enzyme1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience research\", \"reference\": \"16290302\", \"authors\": [\"Harada H\", \"Tamaoka A\", \"Ishii K\", \"Shoji S\", \"Kametaka S\", \"Kametani F\", \"Saito Y\", \"Murayama S\"], \"date\": \"2006-01-01\"}, \"line\": 25406, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 59}, {\"relation\": \"increases\", \"evidence\": \"Abeta is produced from beta-amyloid precursor protein (APP) by beta-secretase and gamma-secretase. beta-Secretase has been identified as beta-site APP cleaving enzyme1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience research\", \"reference\": \"16290302\", \"authors\": [\"Harada H\", \"Tamaoka A\", \"Ishii K\", \"Shoji S\", \"Kametaka S\", \"Kametani F\", \"Saito Y\", \"Murayama S\"], \"date\": \"2006-01-01\"}, \"line\": 25406, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 60}, {\"relation\": \"increases\", \"evidence\": \"The beta-amyloid peptide (Abeta) is a major component of the Alzheimer's disease (AD)-associated senile plaques and is generated by sequential cleavage of the beta-amyloid precursor protein (APP) by beta-secretase and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"RNA (New York, N.Y.)\", \"reference\": \"16888322\", \"authors\": [\"Rentmeister A\", \"Bill A\", \"Wahle T\", \"Walter J\", \"Famulok M\"], \"date\": \"2006-09-01\"}, \"line\": 25422, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 61}, {\"relation\": \"increases\", \"evidence\": \"The beta-amyloid peptide (Abeta) is a major component of the Alzheimer's disease (AD)-associated senile plaques and is generated by sequential cleavage of the beta-amyloid precursor protein (APP) by beta-secretase and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"RNA (New York, N.Y.)\", \"reference\": \"16888322\", \"authors\": [\"Rentmeister A\", \"Bill A\", \"Wahle T\", \"Walter J\", \"Famulok M\"], \"date\": \"2006-09-01\"}, \"line\": 25422, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 62}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta), which is a product of the proteolytic effect of beta-secretase (BACE) on an amyloid precursor protein, is closely associated with Alzheimer's disease (AD) pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of microbiology (Seoul, Korea)\", \"reference\": \"17205046\", \"authors\": [\"Lee TH\", \"Park YI\", \"Han YH\"], \"date\": \"2006-12-01\"}, \"line\": 25439, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 63}, {\"relation\": \"increases\", \"evidence\": \"Amyloid beta-peptide (Abeta), which is a product of the proteolytic effect of beta-secretase (BACE) on an amyloid precursor protein, is closely associated with Alzheimer's disease (AD) pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of microbiology (Seoul, Korea)\", \"reference\": \"17205046\", \"authors\": [\"Lee TH\", \"Park YI\", \"Han YH\"], \"date\": \"2006-12-01\"}, \"line\": 25439, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 64}, {\"relation\": \"increases\", \"evidence\": \"TNF-alpha directly stimulated beta-site APP-cleaving enzyme (BACE1) expression and enhanced beta-processing of APP in astrocytes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"17255335\", \"authors\": [\"Yamamoto M\", \"Kiyota T\", \"Horiba M\", \"Buescher JL\", \"Walsh SM\", \"Gendelman HE\", \"Ikezu T\"], \"date\": \"2007-02-01\"}, \"line\": 25449, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 65}, {\"relation\": \"increases\", \"evidence\": \"TNF-alpha directly stimulated beta-site APP-cleaving enzyme (BACE1) expression and enhanced beta-processing of APP in astrocytes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"17255335\", \"authors\": [\"Yamamoto M\", \"Kiyota T\", \"Horiba M\", \"Buescher JL\", \"Walsh SM\", \"Gendelman HE\", \"Ikezu T\"], \"date\": \"2007-02-01\"}, \"line\": 25449, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 66}, {\"relation\": \"increases\", \"evidence\": \"The peptides not only inhibited the formation of APPbeta (a soluble N-terminal fragment of APP cleaved by beta-secretase), but also significantly reduced Abeta40 production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Peptides\", \"reference\": \"17293005\", \"authors\": [\"Yeon SW\", \"Jeon YJ\", \"Hwang EM\", \"Kim TY\"], \"date\": \"2007-04-01\"}, \"line\": 25465, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 67}, {\"relation\": \"increases\", \"evidence\": \"The peptides not only inhibited the formation of APPbeta (a soluble N-terminal fragment of APP cleaved by beta-secretase), but also significantly reduced Abeta40 production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Peptides\", \"reference\": \"17293005\", \"authors\": [\"Yeon SW\", \"Jeon YJ\", \"Hwang EM\", \"Kim TY\"], \"date\": \"2007-04-01\"}, \"line\": 25465, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 68}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is associated with accumulation of the neurotoxic peptide amyloid-beta (Abeta), which is produced by sequential cleavage of amyloid precursor protein (APP) by the aspartyl protease beta-secretase and the presenilin-dependent protease gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17360493\", \"authors\": [\"Flajolet M\", \"He G\", \"Heiman M\", \"Lin A\", \"Nairn AC\", \"Greengard P\"], \"date\": \"2007-03-06\"}, \"line\": 25474, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 69}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is associated with accumulation of the neurotoxic peptide amyloid-beta (Abeta), which is produced by sequential cleavage of amyloid precursor protein (APP) by the aspartyl protease beta-secretase and the presenilin-dependent protease gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17360493\", \"authors\": [\"Flajolet M\", \"He G\", \"Heiman M\", \"Lin A\", \"Nairn AC\", \"Greengard P\"], \"date\": \"2007-03-06\"}, \"line\": 25474, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 70}, {\"relation\": \"increases\", \"evidence\": \"Such an approach limits APP processing by beta-secretase, mainly through the endocytic pathway, and overcomes some of the limitations of BACE inhibition.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuro-degenerative diseases\", \"reference\": \"17536186\", \"authors\": [\"Rakover I\", \"Arbel M\", \"Solomon B\"], \"date\": \"2007-01-01\"}, \"line\": 25488, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 71}, {\"relation\": \"increases\", \"evidence\": \"Such an approach limits APP processing by beta-secretase, mainly through the endocytic pathway, and overcomes some of the limitations of BACE inhibition.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuro-degenerative diseases\", \"reference\": \"17536186\", \"authors\": [\"Rakover I\", \"Arbel M\", \"Solomon B\"], \"date\": \"2007-01-01\"}, \"line\": 25488, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 72}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic processing of the amyloid precursor protein (APP) by beta-secretase, beta-site APP cleaving enzyme (BACE1), is the initial step in the production of the amyloid beta (Abeta) peptide, which is involved in the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17573534\", \"authors\": [\"Parkin ET\", \"Watt NT\", \"Hussain I\", \"Eckman EA\", \"Eckman CB\", \"Manson JC\", \"Baybutt HN\", \"Turner AJ\", \"Hooper NM\"], \"date\": \"2007-06-26\"}, \"line\": 25497, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 73}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic processing of the amyloid precursor protein (APP) by beta-secretase, beta-site APP cleaving enzyme (BACE1), is the initial step in the production of the amyloid beta (Abeta) peptide, which is involved in the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17573534\", \"authors\": [\"Parkin ET\", \"Watt NT\", \"Hussain I\", \"Eckman EA\", \"Eckman CB\", \"Manson JC\", \"Baybutt HN\", \"Turner AJ\", \"Hooper NM\"], \"date\": \"2007-06-26\"}, \"line\": 25497, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 74}, {\"relation\": \"increases\", \"evidence\": \"To address this issue, temporal neocortex of 27 AD and 21 non-demented control brains was examined to assess mRNA levels of APP isoforms (total APP, APP containing the Kunitz protease inhibitor domain [APP-KPI] and APP770) and APP metabolic enzymatic partners (the APP cleaving enzymes beta-secretase [BACE] and presenilin-1 [PS-1], and putative clearance molecules, low-density lipoprotein receptor protein [LRP] and apolipoprotein E [apoE]).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain research\", \"reference\": \"17586478\", \"authors\": [\"Matsui T\", \"Ingelsson M\", \"Fukumoto H\", \"Ramasamy K\", \"Kowa H\", \"Frosch MP\", \"Irizarry MC\", \"Hyman BT\"], \"date\": \"2007-08-03\"}, \"line\": 25516, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 75}, {\"relation\": \"increases\", \"evidence\": \"To address this issue, temporal neocortex of 27 AD and 21 non-demented control brains was examined to assess mRNA levels of APP isoforms (total APP, APP containing the Kunitz protease inhibitor domain [APP-KPI] and APP770) and APP metabolic enzymatic partners (the APP cleaving enzymes beta-secretase [BACE] and presenilin-1 [PS-1], and putative clearance molecules, low-density lipoprotein receptor protein [LRP] and apolipoprotein E [apoE]).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain research\", \"reference\": \"17586478\", \"authors\": [\"Matsui T\", \"Ingelsson M\", \"Fukumoto H\", \"Ramasamy K\", \"Kowa H\", \"Frosch MP\", \"Irizarry MC\", \"Hyman BT\"], \"date\": \"2007-08-03\"}, \"line\": 25516, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 76}, {\"relation\": \"increases\", \"evidence\": \"Pathologically, AD is characterized by the deposition in the brain of amyloid-beta peptides derived from proteolysis of amyloid precursor protein (APP) by beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature clinical practice. Neurology\", \"reference\": \"17611486\", \"authors\": [\"Cheng H\", \"Vetrivel KS\", \"Gong P\", \"Meckler X\", \"Parent A\", \"Thinakaran G\"], \"date\": \"2007-07-01\"}, \"line\": 25525, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 77}, {\"relation\": \"increases\", \"evidence\": \"Pathologically, AD is characterized by the deposition in the brain of amyloid-beta peptides derived from proteolysis of amyloid precursor protein (APP) by beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature clinical practice. Neurology\", \"reference\": \"17611486\", \"authors\": [\"Cheng H\", \"Vetrivel KS\", \"Gong P\", \"Meckler X\", \"Parent A\", \"Thinakaran G\"], \"date\": \"2007-07-01\"}, \"line\": 25525, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 78}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated by proteolytic processing of amyloid precursor protein (APP) via beta and gamma-secretases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"17908048\", \"authors\": [\"Arbel M\", \"Solomon B\"], \"date\": \"2007-09-01\"}, \"line\": 25534, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 79}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated by proteolytic processing of amyloid precursor protein (APP) via beta and gamma-secretases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"17908048\", \"authors\": [\"Arbel M\", \"Solomon B\"], \"date\": \"2007-09-01\"}, \"line\": 25534, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 80}, {\"relation\": \"increases\", \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"17995932\", \"authors\": [\"Karlstrom H\", \"Brooks WS\", \"Kwok JB\", \"Broe GA\", \"Kril JJ\", \"McCann H\", \"Halliday GM\", \"Schofield PR\"], \"date\": \"2008-02-01\"}, \"line\": 25543, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 81}, {\"relation\": \"increases\", \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"17995932\", \"authors\": [\"Karlstrom H\", \"Brooks WS\", \"Kwok JB\", \"Broe GA\", \"Kril JJ\", \"McCann H\", \"Halliday GM\", \"Schofield PR\"], \"date\": \"2008-02-01\"}, \"line\": 25543, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 82}, {\"relation\": \"increases\", \"evidence\": \"Quetiapine also decreased brain Abeta peptides, beta-secretase activity and expression, and the level of C99 (an APP C-terminal fragment following cleavage by beta-secretase) in the transgenic mice.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"18079026\", \"authors\": [\"He J\", \"Luo H\", \"Yan B\", \"Yu Y\", \"Wang H\", \"Wei Z\", \"Zhang Y\", \"Xu H\", \"Tempier A\", \"Li X\", \"Li XM\"], \"date\": \"2009-08-01\"}, \"line\": 25553, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 83}, {\"relation\": \"increases\", \"evidence\": \"Quetiapine also decreased brain Abeta peptides, beta-secretase activity and expression, and the level of C99 (an APP C-terminal fragment following cleavage by beta-secretase) in the transgenic mice.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"18079026\", \"authors\": [\"He J\", \"Luo H\", \"Yan B\", \"Yu Y\", \"Wang H\", \"Wei Z\", \"Zhang Y\", \"Xu H\", \"Tempier A\", \"Li X\", \"Li XM\"], \"date\": \"2009-08-01\"}, \"line\": 25553, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 84}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic cleavage of amyloid precursor protein by beta-secretase (BACE-1) and gamma-secretase leads to formation of beta-amyloid (A beta) a key component of amyloid plaques, which are considered the hallmark of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bioorganic &amp; medicinal chemistry letters\", \"reference\": \"18162398\", \"authors\": [\"Cole DC\", \"Stock JR\", \"Chopra R\", \"Cowling R\", \"Ellingboe JW\", \"Fan KY\", \"Harrison BL\", \"Hu Y\", \"Jacobsen S\", \"Jennings LD\", \"Jin G\", \"Lohse PA\", \"Malamas MS\", \"Manas ES\", \"Moore WJ\", \"O'Donnell MM\", \"Olland AM\", \"Robichaud AJ\", \"Svenson K\", \"Wu J\", \"Wagner E\", \"Bard J\"], \"date\": \"2008-02-01\"}, \"line\": 25570, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 85}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic cleavage of amyloid precursor protein by beta-secretase (BACE-1) and gamma-secretase leads to formation of beta-amyloid (A beta) a key component of amyloid plaques, which are considered the hallmark of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bioorganic &amp; medicinal chemistry letters\", \"reference\": \"18162398\", \"authors\": [\"Cole DC\", \"Stock JR\", \"Chopra R\", \"Cowling R\", \"Ellingboe JW\", \"Fan KY\", \"Harrison BL\", \"Hu Y\", \"Jacobsen S\", \"Jennings LD\", \"Jin G\", \"Lohse PA\", \"Malamas MS\", \"Manas ES\", \"Moore WJ\", \"O'Donnell MM\", \"Olland AM\", \"Robichaud AJ\", \"Svenson K\", \"Wu J\", \"Wagner E\", \"Bard J\"], \"date\": \"2008-02-01\"}, \"line\": 25570, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 86}, {\"relation\": \"increases\", \"evidence\": \"Here we describe that the two enzymatic activities responsible for Abeta production, beta-secretase and gamma-secretase, are inhibited in parallel by cholesterol reduction.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18308724\", \"authors\": [\"Grimm MO\", \"Grimm HS\", \"Tomic I\", \"Beyreuther K\", \"Hartmann T\", \"Bergmann C\"], \"date\": \"2008-04-25\"}, \"line\": 25579, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 87}, {\"relation\": \"increases\", \"evidence\": \"Here we describe that the two enzymatic activities responsible for Abeta production, beta-secretase and gamma-secretase, are inhibited in parallel by cholesterol reduction.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18308724\", \"authors\": [\"Grimm MO\", \"Grimm HS\", \"Tomic I\", \"Beyreuther K\", \"Hartmann T\", \"Bergmann C\"], \"date\": \"2008-04-25\"}, \"line\": 25579, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 88}, {\"relation\": \"increases\", \"evidence\": \"There was also an induced expression of the key APP processing enzyme i.e. beta-site APP cleaving enzyme 1 in both high fat/cholesterol-fed C57BL/6 and LDLR-/- mice accompanied by an increased generation of C-terminal fragments of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18410513\", \"authors\": [\"Thirumangalakudi L\", \"Prakasam A\", \"Zhang R\", \"Bimonte-Nelson H\", \"Sambamurti K\", \"Kindy MS\", \"Bhat NR\"], \"date\": \"2008-07-01\"}, \"line\": 25594, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 89}, {\"relation\": \"increases\", \"evidence\": \"There was also an induced expression of the key APP processing enzyme i.e. beta-site APP cleaving enzyme 1 in both high fat/cholesterol-fed C57BL/6 and LDLR-/- mice accompanied by an increased generation of C-terminal fragments of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18410513\", \"authors\": [\"Thirumangalakudi L\", \"Prakasam A\", \"Zhang R\", \"Bimonte-Nelson H\", \"Sambamurti K\", \"Kindy MS\", \"Bhat NR\"], \"date\": \"2008-07-01\"}, \"line\": 25594, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 90}, {\"relation\": \"increases\", \"evidence\": \"The study provides new cell-based assays for the profiling of small molecule inhibitors of QC and points to conspicuous differences in processing of APP depending on sequence at the beta-secretase cleavage site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"18570439\", \"authors\": [\"Cynis H\", \"Scheel E\", \"Saido TC\", \"Schilling S\", \"Demuth HU\"], \"date\": \"2008-07-15\"}, \"line\": 25606, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 91}, {\"relation\": \"increases\", \"evidence\": \"The study provides new cell-based assays for the profiling of small molecule inhibitors of QC and points to conspicuous differences in processing of APP depending on sequence at the beta-secretase cleavage site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"18570439\", \"authors\": [\"Cynis H\", \"Scheel E\", \"Saido TC\", \"Schilling S\", \"Demuth HU\"], \"date\": \"2008-07-15\"}, \"line\": 25606, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 92}, {\"relation\": \"increases\", \"evidence\": \"The Abeta peptide results from cleavage of APP initially by BACE1 to produce the C99 fragment and releases soluble APPbeta (sAPPbeta); C99 is then further cleaved by gamma-secretase leading to the Abeta peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Scandinavian journal of clinical and laboratory investigation\", \"reference\": \"18609117\", \"authors\": [\"Hampel H\", \"Shen Y\"], \"date\": \"2009-01-01\"}, \"line\": 25617, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 93}, {\"relation\": \"increases\", \"evidence\": \"Here, we show that lysines 587 and 595 of APP, which are immediately adjacent to the site of beta-secretase cleavage, are covalently pmodified by SUMO proteins in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemical and biophysical research communications\", \"reference\": \"18675254\", \"authors\": [\"Zhang YQ\", \"Sarge KD\"], \"date\": \"2008-10-03\"}, \"line\": 25626, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 94}, {\"relation\": \"increases\", \"evidence\": \"Here, we show that lysines 587 and 595 of APP, which are immediately adjacent to the site of beta-secretase cleavage, are covalently pmodified by SUMO proteins in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemical and biophysical research communications\", \"reference\": \"18675254\", \"authors\": [\"Zhang YQ\", \"Sarge KD\"], \"date\": \"2008-10-03\"}, \"line\": 25626, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 95}, {\"relation\": \"increases\", \"evidence\": \"Specifically, hypoxia significantly increases beta-site APP cleaving enzyme (BACE1) gene transcription through the over-expression of hypoxia inducible factor 1alpha, resulting in increased BACE1 secretase activity and amyloid-beta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"19196431\", \"authors\": [\"Guglielmotto M\", \"Aragno M\", \"Autelli R\", \"Giliberto L\", \"Novo E\", \"Colombatto S\", \"Danni O\", \"Parola M\", \"Smith MA\", \"Perry G\", \"Tamagno E\", \"Tabaton M\"], \"date\": \"2009-02-01\"}, \"line\": 25640, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 96}, {\"relation\": \"increases\", \"evidence\": \"Specifically, hypoxia significantly increases beta-site APP cleaving enzyme (BACE1) gene transcription through the over-expression of hypoxia inducible factor 1alpha, resulting in increased BACE1 secretase activity and amyloid-beta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"19196431\", \"authors\": [\"Guglielmotto M\", \"Aragno M\", \"Autelli R\", \"Giliberto L\", \"Novo E\", \"Colombatto S\", \"Danni O\", \"Parola M\", \"Smith MA\", \"Perry G\", \"Tamagno E\", \"Tabaton M\"], \"date\": \"2009-02-01\"}, \"line\": 25640, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 97}, {\"relation\": \"increases\", \"evidence\": \"Specifically, hypoxia significantly increases beta-site APP cleaving enzyme (BACE1) gene transcription through the over-expression of hypoxia inducible factor 1alpha, resulting in increased BACE1 secretase activity and amyloid-beta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"19196431\", \"authors\": [\"Guglielmotto M\", \"Aragno M\", \"Autelli R\", \"Giliberto L\", \"Novo E\", \"Colombatto S\", \"Danni O\", \"Parola M\", \"Smith MA\", \"Perry G\", \"Tamagno E\", \"Tabaton M\"], \"date\": \"2009-02-01\"}, \"line\": 25640, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Hypoxia response subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 98}, {\"relation\": \"increases\", \"evidence\": \"It is produced from amyloid precursor protein (APP) by proteolytic processing dependent on the beta-site APP-cleaving enzyme 1 (BACE1) and gamma-secretase, and is degraded by a broad range of proteases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Scandinavian journal of clinical and laboratory investigation\", \"reference\": \"19199126\", \"authors\": [\"Zetterberg H\"], \"date\": \"2009-01-01\"}, \"line\": 25650, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 99}, {\"relation\": \"increases\", \"evidence\": \"It is produced from amyloid precursor protein (APP) by proteolytic processing dependent on the beta-site APP-cleaving enzyme 1 (BACE1) and gamma-secretase, and is degraded by a broad range of proteases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Scandinavian journal of clinical and laboratory investigation\", \"reference\": \"19199126\", \"authors\": [\"Zetterberg H\"], \"date\": \"2009-01-01\"}, \"line\": 25650, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 100}, {\"relation\": \"increases\", \"evidence\": \"Three-dimensional quantitative structure-activity relationship (3D-QSAR) pmodels were developed based on comparative molecular field analysis (CoMFA) and comparative molecular similarity indices analysis (CoMSIA), on a series of 43 hydroxyethylamine derivatives, acting as potent inhibitors of beta-site amyloid precursor protein (APP) cleavage enzyme (BACE-1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular diversity\", \"reference\": \"19330459\", \"authors\": [\"Pandey A\", \"Mungalpara J\", \"Mohan CG\"], \"date\": \"2010-02-01\"}, \"line\": 25683, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 101}, {\"relation\": \"increases\", \"evidence\": \"Three-dimensional quantitative structure-activity relationship (3D-QSAR) pmodels were developed based on comparative molecular field analysis (CoMFA) and comparative molecular similarity indices analysis (CoMSIA), on a series of 43 hydroxyethylamine derivatives, acting as potent inhibitors of beta-site amyloid precursor protein (APP) cleavage enzyme (BACE-1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular diversity\", \"reference\": \"19330459\", \"authors\": [\"Pandey A\", \"Mungalpara J\", \"Mohan CG\"], \"date\": \"2010-02-01\"}, \"line\": 25683, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 102}, {\"relation\": \"increases\", \"evidence\": \"beta-Site amyloid precursor protein (APP) cleaving enzyme 1 (BACE1) is a membrane-bound protease that is essential for the production of beta-amyloid protein (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroscience research\", \"reference\": \"19405102\", \"authors\": [\"Kume H\", \"Murayama KS\", \"Araki W\"], \"date\": \"2009-10-01\"}, \"line\": 25704, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 103}, {\"relation\": \"increases\", \"evidence\": \"beta-Site amyloid precursor protein (APP) cleaving enzyme 1 (BACE1) is a membrane-bound protease that is essential for the production of beta-amyloid protein (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroscience research\", \"reference\": \"19405102\", \"authors\": [\"Kume H\", \"Murayama KS\", \"Araki W\"], \"date\": \"2009-10-01\"}, \"line\": 25704, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 104}, {\"relation\": \"increases\", \"evidence\": \"Expression levels of the amyloid precursor protein (APP) and beta-site amyloid (Abeta) cleaving enzyme 1 (BACE1) have been implicated in Alzheimer disease (AD) progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human mutation\", \"reference\": \"19462468\", \"authors\": [\"Bettens K\", \"Brouwers N\", \"Engelborghs S\", \"Van Miegroet H\", \"De Deyn PP\", \"Theuns J\", \"Sleegers K\", \"Van Broeckhoven C\"], \"date\": \"2009-08-01\"}, \"line\": 25720, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 105}, {\"relation\": \"increases\", \"evidence\": \"Expression levels of the amyloid precursor protein (APP) and beta-site amyloid (Abeta) cleaving enzyme 1 (BACE1) have been implicated in Alzheimer disease (AD) progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human mutation\", \"reference\": \"19462468\", \"authors\": [\"Bettens K\", \"Brouwers N\", \"Engelborghs S\", \"Van Miegroet H\", \"De Deyn PP\", \"Theuns J\", \"Sleegers K\", \"Van Broeckhoven C\"], \"date\": \"2009-08-01\"}, \"line\": 25720, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 106}, {\"relation\": \"increases\", \"evidence\": \" The effect of nicotine (2 mg/(kg day)) on Abeta-induced spatial learning and memory impairments was assessed by evaluation of performance in the radial arm water maze (RAWM), in vivo electrophysiological recordings of early-phase long-term potentiation (E-LTP) in urethane-anesthetized rats, and immunoblot analysis to determine changes in the levels of beta-site amyloid precursor protein (APP)-cleaving enzyme (BACE), Abeta and memory-related protei\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"19464074\", \"authors\": [\"Srivareerat M\", \"Tran TT\", \"Salim S\", \"Aleisa AM\", \"Alkadhi KA\"], \"date\": \"2011-05-01\"}, \"line\": 25732, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 107}, {\"relation\": \"increases\", \"evidence\": \" The effect of nicotine (2 mg/(kg day)) on Abeta-induced spatial learning and memory impairments was assessed by evaluation of performance in the radial arm water maze (RAWM), in vivo electrophysiological recordings of early-phase long-term potentiation (E-LTP) in urethane-anesthetized rats, and immunoblot analysis to determine changes in the levels of beta-site amyloid precursor protein (APP)-cleaving enzyme (BACE), Abeta and memory-related protei\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"19464074\", \"authors\": [\"Srivareerat M\", \"Tran TT\", \"Salim S\", \"Aleisa AM\", \"Alkadhi KA\"], \"date\": \"2011-05-01\"}, \"line\": 25732, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 108}, {\"relation\": \"increases\", \"evidence\": \"Beta-site amyloid precursor protein cleaving enzyme (BACE1) is the rate-limiting enzyme for production of beta-amyloid peptides (Abeta), which are proposed to drive the pathological changes found in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular medicine\", \"reference\": \"19669607\", \"authors\": [\"Carson R\", \"McKnight AJ\", \"Todd S\", \"Liu WW\", \"Heggarty S\", \"Craig D\", \"McGuinness B\", \"Irvine GB\", \"Passmore AP\", \"Johnston JA\"], \"date\": \"2009-01-01\"}, \"line\": 25741, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 109}, {\"relation\": \"increases\", \"evidence\": \"Beta-site amyloid precursor protein cleaving enzyme (BACE1) is the rate-limiting enzyme for production of beta-amyloid peptides (Abeta), which are proposed to drive the pathological changes found in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular medicine\", \"reference\": \"19669607\", \"authors\": [\"Carson R\", \"McKnight AJ\", \"Todd S\", \"Liu WW\", \"Heggarty S\", \"Craig D\", \"McGuinness B\", \"Irvine GB\", \"Passmore AP\", \"Johnston JA\"], \"date\": \"2009-01-01\"}, \"line\": 25741, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 110}, {\"relation\": \"increases\", \"evidence\": \"Core CSF biomarkers include secreted Abeta and amyloid precursor protein (APP) isoforms, Abeta oligomers and beta-site APP-cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental gerontology\", \"reference\": \"19698775\", \"authors\": [\"Zetterberg H\", \"Blennow K\", \"Hanse E\"], \"date\": \"2010-01-01\"}, \"line\": 25750, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 111}, {\"relation\": \"increases\", \"evidence\": \"Core CSF biomarkers include secreted Abeta and amyloid precursor protein (APP) isoforms, Abeta oligomers and beta-site APP-cleaving enzyme 1 (BACE1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental gerontology\", \"reference\": \"19698775\", \"authors\": [\"Zetterberg H\", \"Blennow K\", \"Hanse E\"], \"date\": \"2010-01-01\"}, \"line\": 25750, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 112}, {\"relation\": \"increases\", \"evidence\": \" In the amyloidogenic pathway, a small proportion of APP is cleaved by beta- and gamma-secretases, known as beta-site APP-cleaving enzyme 1 (BACE1) and presenilin, respectively, leading to the secretion of Abeta peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"19729516\", \"authors\": [\"Lakshmana MK\", \"Chung JY\", \"Wickramarachchi S\", \"Tak E\", \"Bianchi E\", \"Koo EH\", \"Kang DE\"], \"date\": \"2010-01-01\"}, \"line\": 25763, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 113}, {\"relation\": \"increases\", \"evidence\": \" In the amyloidogenic pathway, a small proportion of APP is cleaved by beta- and gamma-secretases, known as beta-site APP-cleaving enzyme 1 (BACE1) and presenilin, respectively, leading to the secretion of Abeta peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"19729516\", \"authors\": [\"Lakshmana MK\", \"Chung JY\", \"Wickramarachchi S\", \"Tak E\", \"Bianchi E\", \"Koo EH\", \"Kang DE\"], \"date\": \"2010-01-01\"}, \"line\": 25763, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 114}, {\"relation\": \"increases\", \"evidence\": \"A beta is generated upon the sequential proteolytic cleavage of transmembrane amyloid precursor protein (APP) by two membrane-bound proteases, beta-secretase (BACE1) and the gamma-secretase complex comprising presenilin 1 (PS1), nicastrin, APH-1 and PEN-2.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroscience research\", \"reference\": \"19885829\", \"authors\": [\"Oda A\", \"Tamaoka A\", \"Araki W\"], \"date\": \"2010-04-01\"}, \"line\": 25786, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 115}, {\"relation\": \"increases\", \"evidence\": \"A beta is generated upon the sequential proteolytic cleavage of transmembrane amyloid precursor protein (APP) by two membrane-bound proteases, beta-secretase (BACE1) and the gamma-secretase complex comprising presenilin 1 (PS1), nicastrin, APH-1 and PEN-2.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroscience research\", \"reference\": \"19885829\", \"authors\": [\"Oda A\", \"Tamaoka A\", \"Araki W\"], \"date\": \"2010-04-01\"}, \"line\": 25786, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 116}, {\"relation\": \"increases\", \"evidence\": \"In Alzheimer disease, this feedback loop is disrupted, and the increased level of Abeta oligomers bind to PrP(C) and prevent it from regulating BACE1 activity. PrP(C) interacts with and inhibits the beta-secretase BACE1, the rate-limiting enzyme in the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prion\", \"reference\": \"19887909\", \"authors\": [\"Kellett KA\", \"Hooper NM\"], \"date\": \"2009-10-01\"}, \"line\": 25795, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 117}, {\"relation\": \"increases\", \"evidence\": \"In Alzheimer disease, this feedback loop is disrupted, and the increased level of Abeta oligomers bind to PrP(C) and prevent it from regulating BACE1 activity. PrP(C) interacts with and inhibits the beta-secretase BACE1, the rate-limiting enzyme in the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prion\", \"reference\": \"19887909\", \"authors\": [\"Kellett KA\", \"Hooper NM\"], \"date\": \"2009-10-01\"}, \"line\": 25795, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 118}, {\"relation\": \"increases\", \"evidence\": \"The beta-site APP cleaving enzyme (BACE1) is responsible for the first step in the production of the beta-amyloid protein of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"20067575\", \"authors\": [\"Klaver DW\", \"Wilce MC\", \"Gasperini R\", \"Freeman C\", \"Juliano JP\", \"Parish C\", \"Foa L\", \"Aguilar MI\", \"Small DH\"], \"date\": \"2010-03-01\"}, \"line\": 25808, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 119}, {\"relation\": \"increases\", \"evidence\": \"The beta-site APP cleaving enzyme (BACE1) is responsible for the first step in the production of the beta-amyloid protein of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"20067575\", \"authors\": [\"Klaver DW\", \"Wilce MC\", \"Gasperini R\", \"Freeman C\", \"Juliano JP\", \"Parish C\", \"Foa L\", \"Aguilar MI\", \"Small DH\"], \"date\": \"2010-03-01\"}, \"line\": 25808, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 120}, {\"relation\": \"increases\", \"evidence\": \"beta-Site amyloid precursor protein cleaving enzyme 1 (BACE1) initiates amyloid-beta (Abeta) generation that is central to the pathophysiology of Alzheimer's disease (AD). Therefore, lowering Abeta levels by BACE1 manipulations represents a key therapeutic strategy, but it remains unclear whether partial inhibition of BACE1, as expected for AD treatments, can improve memory deficits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"20089133\", \"authors\": [\"Kimura R\", \"Devi L\", \"Ohno M\"], \"date\": \"2010-04-01\"}, \"line\": 25820, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 121}, {\"relation\": \"increases\", \"evidence\": \"beta-Site amyloid precursor protein cleaving enzyme 1 (BACE1) initiates amyloid-beta (Abeta) generation that is central to the pathophysiology of Alzheimer's disease (AD). Therefore, lowering Abeta levels by BACE1 manipulations represents a key therapeutic strategy, but it remains unclear whether partial inhibition of BACE1, as expected for AD treatments, can improve memory deficits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"20089133\", \"authors\": [\"Kimura R\", \"Devi L\", \"Ohno M\"], \"date\": \"2010-04-01\"}, \"line\": 25820, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 122}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated from AbetaPP through an initial cleavage by the beta-secretase, BACE-1, which results in the generation of the soluble AbetaPPbeta fragment (sAbetaPPbeta) and the membrane bound C-terminal fragment beta (CTFbeta or C99). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20157255\", \"authors\": [\"Marwarha G\", \"Dasari B\", \"Prasanthi JR\", \"Schommer J\", \"Ghribi O\"], \"date\": \"2010-01-01\"}, \"line\": 25884, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 123}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated from AbetaPP through an initial cleavage by the beta-secretase, BACE-1, which results in the generation of the soluble AbetaPPbeta fragment (sAbetaPPbeta) and the membrane bound C-terminal fragment beta (CTFbeta or C99). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20157255\", \"authors\": [\"Marwarha G\", \"Dasari B\", \"Prasanthi JR\", \"Schommer J\", \"Ghribi O\"], \"date\": \"2010-01-01\"}, \"line\": 25884, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 124}, {\"relation\": \"increases\", \"evidence\": \"We previously indicated that amyloid beta (Abeta) augments protein levels of beta-site amyloid precursor protein cleaving enzyme-1 (BACE-1) through oxidative stress.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemical and biophysical research communications\", \"reference\": \"20171164\", \"authors\": [\"Kihara T\", \"Shimmyo Y\", \"Akaike A\", \"Niidome T\", \"Sugimoto H\"], \"date\": \"2010-03-19\"}, \"line\": 25893, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 125}, {\"relation\": \"increases\", \"evidence\": \"We previously indicated that amyloid beta (Abeta) augments protein levels of beta-site amyloid precursor protein cleaving enzyme-1 (BACE-1) through oxidative stress.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemical and biophysical research communications\", \"reference\": \"20171164\", \"authors\": [\"Kihara T\", \"Shimmyo Y\", \"Akaike A\", \"Niidome T\", \"Sugimoto H\"], \"date\": \"2010-03-19\"}, \"line\": 25893, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 126}, {\"relation\": \"increases\", \"evidence\": \"Familial Alzheimer's disease mutations in presenilin 1 do not alter levels of the secreted amyloid-beta protein precursor generated by beta-secretase cleavage.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"20205669\", \"authors\": [\"Zhang C\", \"Browne A\", \"Kim DY\", \"Tanzi RE\"], \"date\": \"2010-02-01\"}, \"line\": 25900, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 127}, {\"relation\": \"increases\", \"evidence\": \"Abeta is derived from a type I transmembrane protein, amyloid precursor protein (APP), by the sequential proteolytic events mediated by beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochimica et biophysica acta\", \"reference\": \"20303415\", \"authors\": [\"Vetrivel KS\", \"Thinakaran G\"], \"date\": \"2010-08-01\"}, \"line\": 25910, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 128}, {\"relation\": \"increases\", \"evidence\": \"Abeta is derived from a type I transmembrane protein, amyloid precursor protein (APP), by the sequential proteolytic events mediated by beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochimica et biophysica acta\", \"reference\": \"20303415\", \"authors\": [\"Vetrivel KS\", \"Thinakaran G\"], \"date\": \"2010-08-01\"}, \"line\": 25910, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 129}, {\"relation\": \"increases\", \"evidence\": \"BACKGROUND: A central feature of Alzheimer's disease is the cleavage of the amyloid precursor protein (APP) to form beta-amyloid peptide (Abeta) by the beta-secretase and gamma-secretase enzymes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular brain\", \"reference\": \"20409323\", \"authors\": [\"Lorenzen A\", \"Samosh J\", \"Vandewark K\", \"Anborgh PH\", \"Seah C\", \"Magalhaes AC\", \"Cregan SP\", \"Ferguson SS\", \"Pasternak SH\"], \"date\": \"2010-04-21\"}, \"line\": 25920, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 130}, {\"relation\": \"increases\", \"evidence\": \"BACKGROUND: A central feature of Alzheimer's disease is the cleavage of the amyloid precursor protein (APP) to form beta-amyloid peptide (Abeta) by the beta-secretase and gamma-secretase enzymes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular brain\", \"reference\": \"20409323\", \"authors\": [\"Lorenzen A\", \"Samosh J\", \"Vandewark K\", \"Anborgh PH\", \"Seah C\", \"Magalhaes AC\", \"Cregan SP\", \"Ferguson SS\", \"Pasternak SH\"], \"date\": \"2010-04-21\"}, \"line\": 25920, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 131}, {\"relation\": \"increases\", \"evidence\": \"Basic understanding of the activities of the amyloid beta peptide (Abeta) and associated proteins such as beta-site APP-cleaving enzyme 1 (BACE1) is necessary to develop effective medical responses to AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"20451519\", \"authors\": [\"Lahiri DK\", \"Maloney B\"], \"date\": \"2010-09-01\"}, \"line\": 25936, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 132}, {\"relation\": \"increases\", \"evidence\": \"Basic understanding of the activities of the amyloid beta peptide (Abeta) and associated proteins such as beta-site APP-cleaving enzyme 1 (BACE1) is necessary to develop effective medical responses to AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"20451519\", \"authors\": [\"Lahiri DK\", \"Maloney B\"], \"date\": \"2010-09-01\"}, \"line\": 25936, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 133}, {\"relation\": \"increases\", \"evidence\": \"Cleavage of APP by BACE1 is the first proteolytic step in the production of amyloid-beta (Abeta), which accumulates in senile plaques in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"20685197\", \"authors\": [\"von Einem B\", \"Schwanzar D\", \"Rehn F\", \"Beyer AS\", \"Weber P\", \"Wagner M\", \"Schneckenburger H\", \"von Arnim CA\"], \"date\": \"2010-09-01\"}, \"line\": 26004, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 134}, {\"relation\": \"increases\", \"evidence\": \"Cleavage of APP by BACE1 is the first proteolytic step in the production of amyloid-beta (Abeta), which accumulates in senile plaques in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"20685197\", \"authors\": [\"von Einem B\", \"Schwanzar D\", \"Rehn F\", \"Beyer AS\", \"Weber P\", \"Wagner M\", \"Schneckenburger H\", \"von Arnim CA\"], \"date\": \"2010-09-01\"}, \"line\": 26004, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 135}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated from the beta-amyloid precursor protein (APP) through the proteolysis of beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"20727383\", \"authors\": [\"Liu H\", \"Li Z\", \"Qiu D\", \"Gu Q\", \"Lei Q\", \"Mao L\"], \"date\": \"2010-11-19\"}, \"line\": 26014, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 136}, {\"relation\": \"increases\", \"evidence\": \"Abeta is generated from the beta-amyloid precursor protein (APP) through the proteolysis of beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"20727383\", \"authors\": [\"Liu H\", \"Li Z\", \"Qiu D\", \"Gu Q\", \"Lei Q\", \"Mao L\"], \"date\": \"2010-11-19\"}, \"line\": 26014, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 137}, {\"relation\": \"increases\", \"evidence\": \"The Swedish mutation (K595N/M596L) of amyloid precursor protein (APP-swe) has been known to increase abnormal cleavage of cellular APP by Beta-secretase (BACE), which causes tau protein hyperphosphorylation and early-onset Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMB reports\", \"reference\": \"21034535\", \"authors\": [\"Shin J\", \"Yu SB\", \"Yu UY\", \"Jo SA\", \"Ahn JH\"], \"date\": \"2010-10-01\"}, \"line\": 26026, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 138}, {\"relation\": \"increases\", \"evidence\": \"The Swedish mutation (K595N/M596L) of amyloid precursor protein (APP-swe) has been known to increase abnormal cleavage of cellular APP by Beta-secretase (BACE), which causes tau protein hyperphosphorylation and early-onset Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMB reports\", \"reference\": \"21034535\", \"authors\": [\"Shin J\", \"Yu SB\", \"Yu UY\", \"Jo SA\", \"Ahn JH\"], \"date\": \"2010-10-01\"}, \"line\": 26026, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 139}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the betaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular neurobiology\", \"reference\": \"21431475\", \"authors\": [\"Stark DT\", \"Bazan NG\"], \"date\": \"2011-04-01\"}, \"line\": 26038, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 140}, {\"relation\": \"increases\", \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the betaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular neurobiology\", \"reference\": \"21431475\", \"authors\": [\"Stark DT\", \"Bazan NG\"], \"date\": \"2011-04-01\"}, \"line\": 26038, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 141}, {\"relation\": \"increases\", \"evidence\": \"Amyloid-beta peptide (Abeta) is generated by sequential cleavage of the amyloid precursor protein (APP) by beta-site amyloid precursor protein cleaving enzyme 1 (beta-secretase, or BACE1) and gamma-secretase. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroscience research\", \"reference\": \"21433051\", \"authors\": [\"Wu G\", \"Sankaranarayanan S\", \"Hsieh SH\", \"Simon AJ\", \"Savage MJ\"], \"date\": \"2011-06-01\"}, \"line\": 26048, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 142}, {\"relation\": \"increases\", \"evidence\": \"Amyloid-beta peptide (Abeta) is generated by sequential cleavage of the amyloid precursor protein (APP) by beta-site amyloid precursor protein cleaving enzyme 1 (beta-secretase, or BACE1) and gamma-secretase. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroscience research\", \"reference\": \"21433051\", \"authors\": [\"Wu G\", \"Sankaranarayanan S\", \"Hsieh SH\", \"Simon AJ\", \"Savage MJ\"], \"date\": \"2011-06-01\"}, \"line\": 26048, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 143}, {\"relation\": \"increases\", \"evidence\": \"To release the amyloidogenic peptide A beta from the Alzheimer amyloid precursor protein (APP), two secretases act sequentially: first, beta-secretase cleaves close to the membrane within the ectodomain and then gamma-secretase cuts within the transmembrane domain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature medicine\", \"reference\": \"9623986\", \"authors\": [\"Lee SJ\", \"Xia W\", \"Kosik KS\", \"Liyanage U\", \"Lansbury PT Jr\", \"Bickel PE\"], \"date\": \"1998-06-01\"}, \"line\": 26059, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 144}, {\"relation\": \"increases\", \"evidence\": \"To release the amyloidogenic peptide A beta from the Alzheimer amyloid precursor protein (APP), two secretases act sequentially: first, beta-secretase cleaves close to the membrane within the ectodomain and then gamma-secretase cuts within the transmembrane domain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature medicine\", \"reference\": \"9623986\", \"authors\": [\"Lee SJ\", \"Xia W\", \"Kosik KS\", \"Liyanage U\", \"Lansbury PT Jr\", \"Bickel PE\"], \"date\": \"1998-06-01\"}, \"line\": 26059, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 145}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic processing of APP by beta-secretase, on the other hand, exposes the N-terminus of beta-amyloid, which is liberated after gamma-secretase cleavage at the variable amyloid C-terminus.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"9775403\", \"authors\": [\"Perez-Polo JR\", \"Rossner S\", \"Schliebs R\", \"Ueberham U\", \"Bigl V\"], \"date\": \"1998-12-01\"}, \"line\": 26069, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 146}, {\"relation\": \"increases\", \"evidence\": \"Proteolytic processing of APP by beta-secretase, on the other hand, exposes the N-terminus of beta-amyloid, which is liberated after gamma-secretase cleavage at the variable amyloid C-terminus.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"9775403\", \"authors\": [\"Perez-Polo JR\", \"Rossner S\", \"Schliebs R\", \"Ueberham U\", \"Bigl V\"], \"date\": \"1998-12-01\"}, \"line\": 26069, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 147}, {\"relation\": \"increases\", \"evidence\": \"Pulse-chase experiments revealed beta-secretase cleavage from immature full-length amyloid precursor protein harboring the Swedish mutation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular neuroscience : MN\", \"reference\": \"10096041\", \"authors\": [\"Dyrks T\", \"Turner J\", \"Urmoneit B\"], \"date\": \"1998-10-01\"}, \"line\": 26086, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 148}, {\"relation\": \"increases\", \"evidence\": \"Pulse-chase experiments revealed beta-secretase cleavage from immature full-length amyloid precursor protein harboring the Swedish mutation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular neuroscience : MN\", \"reference\": \"10096041\", \"authors\": [\"Dyrks T\", \"Turner J\", \"Urmoneit B\"], \"date\": \"1998-10-01\"}, \"line\": 26086, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 149}, {\"relation\": \"increases\", \"evidence\": \"Solubilized Asp2 protein cleaves a synthetic APP peptide substrate at the beta-secretase site, and the rate of cleavage is increased tenfold by a mutation associated with early-onset Alzheimer's disease in Sweden.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature\", \"reference\": \"10591213\", \"authors\": [\"Stratman NC\", \"Mathews WR\", \"Brashier JR\", \"Shuck ME\", \"Buhl AE\", \"Carter DB\", \"Tory MC\", \"Pauley AM\", \"Tomasselli AG\", \"Parodi LA\", \"Heinrikson RL\", \"Gurney ME\", \"Yan R\", \"Miao H\", \"Bienkowski MJ\"], \"date\": \"1999-12-02\"}, \"line\": 26096, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 150}, {\"relation\": \"increases\", \"evidence\": \"Solubilized Asp2 protein cleaves a synthetic APP peptide substrate at the beta-secretase site, and the rate of cleavage is increased tenfold by a mutation associated with early-onset Alzheimer's disease in Sweden.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature\", \"reference\": \"10591213\", \"authors\": [\"Stratman NC\", \"Mathews WR\", \"Brashier JR\", \"Shuck ME\", \"Buhl AE\", \"Carter DB\", \"Tory MC\", \"Pauley AM\", \"Tomasselli AG\", \"Parodi LA\", \"Heinrikson RL\", \"Gurney ME\", \"Yan R\", \"Miao H\", \"Bienkowski MJ\"], \"date\": \"1999-12-02\"}, \"line\": 26096, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 151}, {\"relation\": \"increases\", \"evidence\": \"Transient expression of Asp 2 in cells expressing APP causes an increase in the secretion of the N-terminal fragment of APP and an increase in the cell-associated C-terminal beta-secretase APP fragment.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular neurosciences\", \"reference\": \"10656250\", \"authors\": [\"Chapman C\", \"Meek TD\", \"Tew DG\", \"Howlett DR\", \"Powell D\", \"Hussain I\", \"Gloger IS\", \"Murphy KE\", \"Southan CD\", \"Ryan DM\", \"Smith TS\", \"Simmons DL\", \"Walsh FS\", \"Dingwall C\", \"Christie G\"], \"date\": \"1999-12-01\"}, \"line\": 26106, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 152}, {\"relation\": \"increases\", \"evidence\": \"Transient expression of Asp 2 in cells expressing APP causes an increase in the secretion of the N-terminal fragment of APP and an increase in the cell-associated C-terminal beta-secretase APP fragment.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular neurosciences\", \"reference\": \"10656250\", \"authors\": [\"Chapman C\", \"Meek TD\", \"Tew DG\", \"Howlett DR\", \"Powell D\", \"Hussain I\", \"Gloger IS\", \"Murphy KE\", \"Southan CD\", \"Ryan DM\", \"Smith TS\", \"Simmons DL\", \"Walsh FS\", \"Dingwall C\", \"Christie G\"], \"date\": \"1999-12-01\"}, \"line\": 26106, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 153}, {\"relation\": \"increases\", \"evidence\": \"Recombinant memapsin 2 specifically hydrolyzed peptides derived from the beta-secretase site of both the wild-type and Swedish mutant beta-amyloid precursor protein (APP) with over 60-fold increase of catalytic efficiency for the latter. Human aspartic protease memapsin 2 cleaves the beta-secretase site of beta-amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"10677483\", \"authors\": [\"Koelsch G\", \"Lin X\", \"Wu S\", \"Downs D\", \"Dashti A\", \"Tang J\"], \"date\": \"2000-02-15\"}, \"line\": 26117, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 154}, {\"relation\": \"increases\", \"evidence\": \"Recombinant memapsin 2 specifically hydrolyzed peptides derived from the beta-secretase site of both the wild-type and Swedish mutant beta-amyloid precursor protein (APP) with over 60-fold increase of catalytic efficiency for the latter. Human aspartic protease memapsin 2 cleaves the beta-secretase site of beta-amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"10677483\", \"authors\": [\"Koelsch G\", \"Lin X\", \"Wu S\", \"Downs D\", \"Dashti A\", \"Tang J\"], \"date\": \"2000-02-15\"}, \"line\": 26117, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 155}, {\"relation\": \"increases\", \"evidence\": \"IFNgamma in combination with TNFalpha or IL-1beta seems to trigger Abeta production by supporting beta-secretase cleavage of the immature APP molecule.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"11114266\", \"authors\": [\"Blasko I\", \"Saurwein-Teissl M\", \"Eikelenboom P\", \"Veerhuis R\", \"Stampfer-Kountchev M\", \"Grubeck-Loebenstein B\"], \"date\": \"2000-12-01\"}, \"line\": 26127, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 156}, {\"relation\": \"increases\", \"evidence\": \"IFNgamma in combination with TNFalpha or IL-1beta seems to trigger Abeta production by supporting beta-secretase cleavage of the immature APP molecule.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"11114266\", \"authors\": [\"Blasko I\", \"Saurwein-Teissl M\", \"Eikelenboom P\", \"Veerhuis R\", \"Stampfer-Kountchev M\", \"Grubeck-Loebenstein B\"], \"date\": \"2000-12-01\"}, \"line\": 26127, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 157}, {\"relation\": \"increases\", \"evidence\": \"Amyloid precursor protein/P-selectin, which is sorted from early to late endosomes, undergoes significantly less alpha-secretase cleavage, and more beta-secretase cleavage, than amyloid precursor protein/P-selectin768A, a mutant that recycles more efficiently to the cell surface.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Traffic (Copenhagen, Denmark)\", \"reference\": \"11737828\", \"authors\": [\"Green SA\", \"Daugherty BL\"], \"date\": \"2001-12-01\"}, \"line\": 26142, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 158}, {\"relation\": \"increases\", \"evidence\": \"Amyloid precursor protein/P-selectin, which is sorted from early to late endosomes, undergoes significantly less alpha-secretase cleavage, and more beta-secretase cleavage, than amyloid precursor protein/P-selectin768A, a mutant that recycles more efficiently to the cell surface.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Traffic (Copenhagen, Denmark)\", \"reference\": \"11737828\", \"authors\": [\"Green SA\", \"Daugherty BL\"], \"date\": \"2001-12-01\"}, \"line\": 26142, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 159}, {\"relation\": \"increases\", \"evidence\": \"Homeostatic APPsbetaswe levels with aging suggest that progressive amyloid deposition in brain results not from increased beta-secretase cleavage of APP but from impaired Abeta/amyloid clearance mechanisms.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"11839594\", \"authors\": [\"D'Amato CJ\", \"Gaut JR\", \"Steinhilb ML\", \"Kao TC\", \"Gau JT\", \"Frey KA\", \"Turner RS\"], \"date\": \"2002-02-01\"}, \"line\": 26152, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 160}, {\"relation\": \"increases\", \"evidence\": \"Homeostatic APPsbetaswe levels with aging suggest that progressive amyloid deposition in brain results not from increased beta-secretase cleavage of APP but from impaired Abeta/amyloid clearance mechanisms.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"11839594\", \"authors\": [\"D'Amato CJ\", \"Gaut JR\", \"Steinhilb ML\", \"Kao TC\", \"Gau JT\", \"Frey KA\", \"Turner RS\"], \"date\": \"2002-02-01\"}, \"line\": 26152, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 161}, {\"relation\": \"increases\", \"evidence\": \"ELISA analysis of beta-secretase cleavage of the Swedish amyloid precursor protein in the secretory and endocytic pathways.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"11953452\", \"authors\": [\"Gaut JR\", \"Steinhilb ML\", \"Turner RS\"], \"date\": \"2002-03-01\"}, \"line\": 26161, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 162}, {\"relation\": \"increases\", \"evidence\": \"ELISA analysis of beta-secretase cleavage of the Swedish amyloid precursor protein in the secretory and endocytic pathways.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"11953452\", \"authors\": [\"Gaut JR\", \"Steinhilb ML\", \"Turner RS\"], \"date\": \"2002-03-01\"}, \"line\": 26161, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 163}, {\"relation\": \"increases\", \"evidence\": \"beta-Site APP-cleaving enzyme (BACE) initiates the processing of the amyloid precursor protein (APP) leading to the generation of beta-amyloid, the main component of Alzheimer's disease senile plaques.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"11953458\", \"authors\": [\"Bodendorf U\", \"Molinari M\", \"Paganetti P\", \"Fischer F\"], \"date\": \"2002-03-01\"}, \"line\": 26171, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 164}, {\"relation\": \"increases\", \"evidence\": \"beta-Site APP-cleaving enzyme (BACE) initiates the processing of the amyloid precursor protein (APP) leading to the generation of beta-amyloid, the main component of Alzheimer's disease senile plaques.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"11953458\", \"authors\": [\"Bodendorf U\", \"Molinari M\", \"Paganetti P\", \"Fischer F\"], \"date\": \"2002-03-01\"}, \"line\": 26171, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 165}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase cleavage represents the first step in the generation of Abeta polypeptides and initiates the amyloid cascade that leads to neurodegeneration in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Annals of neurology\", \"reference\": \"12112088\", \"authors\": [\"Evin G\", \"Masters CL\", \"Holsinger RM\", \"Beyreuther K\", \"McLean CA\"], \"date\": \"2002-06-01\"}, \"line\": 26181, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 166}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase cleavage represents the first step in the generation of Abeta polypeptides and initiates the amyloid cascade that leads to neurodegeneration in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Annals of neurology\", \"reference\": \"12112088\", \"authors\": [\"Evin G\", \"Masters CL\", \"Holsinger RM\", \"Beyreuther K\", \"McLean CA\"], \"date\": \"2002-06-01\"}, \"line\": 26181, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 167}, {\"relation\": \"increases\", \"evidence\": \"We show that, in Sf9 cells, BACE performs the expected beta-secretase cleavage of APP, generating C99.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"12423249\", \"authors\": [\"Pitsi D\", \"Kienlen-Campard P\", \"Octave JN\"], \"date\": \"2002-10-01\"}, \"line\": 26190, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 168}, {\"relation\": \"increases\", \"evidence\": \"We show that, in Sf9 cells, BACE performs the expected beta-secretase cleavage of APP, generating C99.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"12423249\", \"authors\": [\"Pitsi D\", \"Kienlen-Campard P\", \"Octave JN\"], \"date\": \"2002-10-01\"}, \"line\": 26190, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 169}, {\"relation\": \"increases\", \"evidence\": \"Our recent knockout studies show that BACE1 is critical for Abeta generation, but the knockout mice show an otherwise normal phenotype, raising the possibility that therapeutic BACE1 inhibition could be accomplished without major mechanism based toxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"12470797\", \"authors\": [\"Citron M\"], \"date\": \"2002-11-01\"}, \"line\": 26204, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 170}, {\"relation\": \"increases\", \"evidence\": \"Our recent knockout studies show that BACE1 is critical for Abeta generation, but the knockout mice show an otherwise normal phenotype, raising the possibility that therapeutic BACE1 inhibition could be accomplished without major mechanism based toxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"12470797\", \"authors\": [\"Citron M\"], \"date\": \"2002-11-01\"}, \"line\": 26204, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 171}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartic protease that cleaves the amyloid precursor protein (APP) at the beta-secretase site, a critical step in the Alzheimer disease pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12473667\", \"authors\": [\"Kitazume S\", \"Tachida Y\", \"Oka R\", \"Kotani N\", \"Ogawa K\", \"Suzuki M\", \"Dohmae N\", \"Takio K\", \"Saido TC\", \"Hashimoto Y\"], \"date\": \"2003-04-25\"}, \"line\": 26214, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 172}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartic protease that cleaves the amyloid precursor protein (APP) at the beta-secretase site, a critical step in the Alzheimer disease pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12473667\", \"authors\": [\"Kitazume S\", \"Tachida Y\", \"Oka R\", \"Kotani N\", \"Ogawa K\", \"Suzuki M\", \"Dohmae N\", \"Takio K\", \"Saido TC\", \"Hashimoto Y\"], \"date\": \"2003-04-25\"}, \"line\": 26214, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 173}, {\"relation\": \"increases\", \"evidence\": \"APP is cleaved either by beta-secretase or by alpha-secretase to initiate amyloidogenic (release of A beta) or nonamyloidogenic processing of APP, respectively.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"12515826\", \"authors\": [\"Ehehalt R\", \"Keller P\", \"Haass C\", \"Thiele C\", \"Simons K\"], \"date\": \"2003-01-06\"}, \"line\": 26225, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 174}, {\"relation\": \"increases\", \"evidence\": \"APP is cleaved either by beta-secretase or by alpha-secretase to initiate amyloidogenic (release of A beta) or nonamyloidogenic processing of APP, respectively.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"12515826\", \"authors\": [\"Ehehalt R\", \"Keller P\", \"Haass C\", \"Thiele C\", \"Simons K\"], \"date\": \"2003-01-06\"}, \"line\": 26225, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 175}, {\"relation\": \"increases\", \"evidence\": \"Employing a superior BACE1 cleavage sequence to probe cellular APP processing.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"12603825\", \"authors\": [\"Tomasselli AG\", \"Qahwash I\", \"Emmons TL\", \"Lu Y\", \"Leone JW\", \"Lull JM\", \"Fok KF\", \"Bannow CA\", \"Smith CW\", \"Bienkowski MJ\", \"Heinrikson RL\", \"Yan R\"], \"date\": \"2003-03-01\"}, \"line\": 26234, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 176}, {\"relation\": \"increases\", \"evidence\": \"Employing a superior BACE1 cleavage sequence to probe cellular APP processing.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"12603825\", \"authors\": [\"Tomasselli AG\", \"Qahwash I\", \"Emmons TL\", \"Lu Y\", \"Leone JW\", \"Lull JM\", \"Fok KF\", \"Bannow CA\", \"Smith CW\", \"Bienkowski MJ\", \"Heinrikson RL\", \"Yan R\"], \"date\": \"2003-03-01\"}, \"line\": 26234, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 177}, {\"relation\": \"increases\", \"evidence\": \"An integral membrane aspartyl protease, BACE, is responsible for beta-secretase processing of the beta-amyloid precursor protein (APP) to the large secreted sAPPbeta and membrane-bound CTFbeta of 99 residues.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular neuroscience : MN\", \"reference\": \"14501002\", \"authors\": [\"Marlow L\", \"Cain M\", \"Pappolla MA\", \"Sambamurti K\"], \"date\": \"2003-01-01\"}, \"line\": 26244, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 178}, {\"relation\": \"increases\", \"evidence\": \"An integral membrane aspartyl protease, BACE, is responsible for beta-secretase processing of the beta-amyloid precursor protein (APP) to the large secreted sAPPbeta and membrane-bound CTFbeta of 99 residues.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular neuroscience : MN\", \"reference\": \"14501002\", \"authors\": [\"Marlow L\", \"Cain M\", \"Pappolla MA\", \"Sambamurti K\"], \"date\": \"2003-01-01\"}, \"line\": 26244, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 179}, {\"relation\": \"increases\", \"evidence\": \"Here we report evidence that heparan sulfate (HS) interacts with beta-site APP-cleaving enzyme (BACE) 1 and regulates its cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"14530380\", \"authors\": [\"Scholefield Z\", \"Yates EA\", \"Wayne G\", \"Amour A\", \"McDowell W\", \"Turnbull JE\"], \"date\": \"2003-10-13\"}, \"line\": 26253, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 180}, {\"relation\": \"increases\", \"evidence\": \"Here we report evidence that heparan sulfate (HS) interacts with beta-site APP-cleaving enzyme (BACE) 1 and regulates its cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"14530380\", \"authors\": [\"Scholefield Z\", \"Yates EA\", \"Wayne G\", \"Amour A\", \"McDowell W\", \"Turnbull JE\"], \"date\": \"2003-10-13\"}, \"line\": 26253, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 181}, {\"relation\": \"increases\", \"evidence\": \"Together, these results suggest that T668 phosphorylation may facilitate the BACE1 cleavage of APP to increase Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"14557249\", \"authors\": [\"Lee MS\", \"Kao SC\", \"Lemere CA\", \"Xia W\", \"Tseng HC\", \"Zhou Y\", \"Neve R\", \"Ahlijanian MK\", \"Tsai LH\"], \"date\": \"2003-10-13\"}, \"line\": 26264, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 182}, {\"relation\": \"increases\", \"evidence\": \"Together, these results suggest that T668 phosphorylation may facilitate the BACE1 cleavage of APP to increase Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"14557249\", \"authors\": [\"Lee MS\", \"Kao SC\", \"Lemere CA\", \"Xia W\", \"Tseng HC\", \"Zhou Y\", \"Neve R\", \"Ahlijanian MK\", \"Tsai LH\"], \"date\": \"2003-10-13\"}, \"line\": 26264, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 183}, {\"relation\": \"increases\", \"evidence\": \"Our results indicate that BACE1 siRNA specifically impacts on beta-cleavage of APP and may be a potential therapeutic approach for treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"14600149\", \"authors\": [\"Kao SC\", \"Krichevsky AM\", \"Kosik KS\", \"Tsai LH\"], \"date\": \"2004-01-16\"}, \"line\": 26275, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 184}, {\"relation\": \"increases\", \"evidence\": \"Our results indicate that BACE1 siRNA specifically impacts on beta-cleavage of APP and may be a potential therapeutic approach for treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"14600149\", \"authors\": [\"Kao SC\", \"Krichevsky AM\", \"Kosik KS\", \"Tsai LH\"], \"date\": \"2004-01-16\"}, \"line\": 26275, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 185}, {\"relation\": \"increases\", \"evidence\": \"The cerebral deposition of amyloid beta-peptide (Abeta) is a major factor in the etiology of Alzheimer's disease. beta-Secretase (BACE) initiates the generation of Abeta by cleaving the amyloid precursor protein at the beta-site and is therefore a prime target for therapeutic intervention.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Analytical biochemistry\", \"reference\": \"14622952\", \"authors\": [\"Oh M\", \"Kim SY\", \"Oh YS\", \"Choi DY\", \"Sin HJ\", \"Jung IM\", \"Park WJ\"], \"date\": \"2003-12-01\"}, \"line\": 26286, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 186}, {\"relation\": \"increases\", \"evidence\": \"The cerebral deposition of amyloid beta-peptide (Abeta) is a major factor in the etiology of Alzheimer's disease. beta-Secretase (BACE) initiates the generation of Abeta by cleaving the amyloid precursor protein at the beta-site and is therefore a prime target for therapeutic intervention.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Analytical biochemistry\", \"reference\": \"14622952\", \"authors\": [\"Oh M\", \"Kim SY\", \"Oh YS\", \"Choi DY\", \"Sin HJ\", \"Jung IM\", \"Park WJ\"], \"date\": \"2003-12-01\"}, \"line\": 26286, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 187}, {\"relation\": \"increases\", \"evidence\": \"BACE1, the major beta-secretase involved in cleaving APP, has been identified as a type 1 membrane-associated aspartyl protease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular biology\", \"reference\": \"14701757\", \"authors\": [\"Christensen MA\", \"Zhou W\", \"Qing H\", \"Lehman A\", \"Philipsen S\", \"Song W\"], \"date\": \"2004-01-01\"}, \"line\": 26294, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 188}, {\"relation\": \"increases\", \"evidence\": \"beta-site APP cleaving enzyme 1 (BACE1) is the beta-secretase enzyme required for generating pathogenic beta-amyloid (Abeta) peptides in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"14715132\", \"authors\": [\"Ohno M\", \"Sametsky EA\", \"Younkin LH\", \"Oakley H\", \"Younkin SG\", \"Citron M\", \"Vassar R\", \"Disterhoft JF\"], \"date\": \"2004-01-08\"}, \"line\": 26304, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 189}, {\"relation\": \"increases\", \"evidence\": \"beta-site APP cleaving enzyme 1 (BACE1) is the beta-secretase enzyme required for generating pathogenic beta-amyloid (Abeta) peptides in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"14715132\", \"authors\": [\"Ohno M\", \"Sametsky EA\", \"Younkin LH\", \"Oakley H\", \"Younkin SG\", \"Citron M\", \"Vassar R\", \"Disterhoft JF\"], \"date\": \"2004-01-08\"}, \"line\": 26304, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 190}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's beta-secretase (BACE1) is a membrane-bound protease that cleaves the amyloid precursor protein (APP) in the trans-Golgi network, an initial step in the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Glycoconjugate journal\", \"reference\": \"14973371\", \"authors\": [\"Kitazume S\", \"Saido TC\", \"Hashimoto Y\"], \"date\": \"2004-01-01\"}, \"line\": 26314, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 191}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's beta-secretase (BACE1) is a membrane-bound protease that cleaves the amyloid precursor protein (APP) in the trans-Golgi network, an initial step in the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Glycoconjugate journal\", \"reference\": \"14973371\", \"authors\": [\"Kitazume S\", \"Saido TC\", \"Hashimoto Y\"], \"date\": \"2004-01-01\"}, \"line\": 26314, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 192}, {\"relation\": \"increases\", \"evidence\": \"The activity of BACE-1 as measured by the formation of the cleavage product of amyloid beta precursor protein, transiently increased up to 48 h after injury, but returned to basal level 7 days post injury.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neural transmission (Vienna, Austria : 1996)\", \"reference\": \"15057522\", \"authors\": [\"Blasko I\", \"Beer R\", \"Bigl M\", \"Apelt J\", \"Franz G\", \"Rudzki D\", \"Ransmayr G\", \"Kampfl A\", \"Schliebs R\"], \"date\": \"2004-04-01\"}, \"line\": 26330, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 193}, {\"relation\": \"increases\", \"evidence\": \"The activity of BACE-1 as measured by the formation of the cleavage product of amyloid beta precursor protein, transiently increased up to 48 h after injury, but returned to basal level 7 days post injury.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neural transmission (Vienna, Austria : 1996)\", \"reference\": \"15057522\", \"authors\": [\"Blasko I\", \"Beer R\", \"Bigl M\", \"Apelt J\", \"Franz G\", \"Rudzki D\", \"Ransmayr G\", \"Kampfl A\", \"Schliebs R\"], \"date\": \"2004-04-01\"}, \"line\": 26330, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 194}, {\"relation\": \"increases\", \"evidence\": \"BACE is an aspartyl protease that cleaves the amyloid precursor protein (APP) at the beta-secretase cleavage site and is involved in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular neurosciences\", \"reference\": \"15080893\", \"authors\": [\"Pastorino L\", \"Ikin AF\", \"Lamprianou S\", \"Vacaresse N\", \"Revelli JP\", \"Platt K\", \"Paganetti P\", \"Mathews PM\", \"Harroch S\", \"Buxbaum JD\"], \"date\": \"2004-04-01\"}, \"line\": 26340, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 195}, {\"relation\": \"increases\", \"evidence\": \"BACE is an aspartyl protease that cleaves the amyloid precursor protein (APP) at the beta-secretase cleavage site and is involved in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular neurosciences\", \"reference\": \"15080893\", \"authors\": [\"Pastorino L\", \"Ikin AF\", \"Lamprianou S\", \"Vacaresse N\", \"Revelli JP\", \"Platt K\", \"Paganetti P\", \"Mathews PM\", \"Harroch S\", \"Buxbaum JD\"], \"date\": \"2004-04-01\"}, \"line\": 26340, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 196}, {\"relation\": \"increases\", \"evidence\": \"The beta-site APP-cleaving enzyme (BACE1) has been identified as the key enzyme leading to beta-amyloid formation, and cholinergic mechanisms have been shown to control APP processing.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroscience research\", \"reference\": \"15211591\", \"authors\": [\"Züchner T\", \"Perez-Polo JR\", \"Schliebs R\"], \"date\": \"2004-07-15\"}, \"line\": 26351, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 197}, {\"relation\": \"increases\", \"evidence\": \"The beta-site APP-cleaving enzyme (BACE1) has been identified as the key enzyme leading to beta-amyloid formation, and cholinergic mechanisms have been shown to control APP processing.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroscience research\", \"reference\": \"15211591\", \"authors\": [\"Züchner T\", \"Perez-Polo JR\", \"Schliebs R\"], \"date\": \"2004-07-15\"}, \"line\": 26351, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 198}, {\"relation\": \"increases\", \"evidence\": \"Because alpha-secretase and BACE-1 cleave APP within the secretory pathway, it is likely that the two enzymes compete for the APP substrate.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular neuroscience : MN\", \"reference\": \"15314262\", \"authors\": [\"Gandhi S\", \"Refolo LM\", \"Sambamurti K\"], \"date\": \"2004-01-01\"}, \"line\": 26370, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 199}, {\"relation\": \"increases\", \"evidence\": \"Because alpha-secretase and BACE-1 cleave APP within the secretory pathway, it is likely that the two enzymes compete for the APP substrate.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular neuroscience : MN\", \"reference\": \"15314262\", \"authors\": [\"Gandhi S\", \"Refolo LM\", \"Sambamurti K\"], \"date\": \"2004-01-01\"}, \"line\": 26370, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 200}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's beta-secretase (BACE1) cleaves amyloid precursor protein to produce amyloid beta-peptide, which is a crucial initiation process of the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Glycoconjugate journal\", \"reference\": \"15467394\", \"authors\": [\"Kitazume S\", \"Suzuki M\", \"Saido TC\", \"Hashimoto Y\"], \"date\": \"2004-01-01\"}, \"line\": 26380, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 201}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's beta-secretase (BACE1) cleaves amyloid precursor protein to produce amyloid beta-peptide, which is a crucial initiation process of the pathogenesis of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Glycoconjugate journal\", \"reference\": \"15467394\", \"authors\": [\"Kitazume S\", \"Suzuki M\", \"Saido TC\", \"Hashimoto Y\"], \"date\": \"2004-01-01\"}, \"line\": 26380, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 202}, {\"relation\": \"increases\", \"evidence\": \"Mapping to the Down syndrome critical region (chromosome 21) and identified as a homologue of BACE1, BACE2 also cleaves amyloid precursor protein at the beta-site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of proteome research\", \"reference\": \"15473697\", \"authors\": [\"Chou KC\"], \"date\": \"2004-09-01\"}, \"line\": 26390, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 203}, {\"relation\": \"increases\", \"evidence\": \"Mapping to the Down syndrome critical region (chromosome 21) and identified as a homologue of BACE1, BACE2 also cleaves amyloid precursor protein at the beta-site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of proteome research\", \"reference\": \"15473697\", \"authors\": [\"Chou KC\"], \"date\": \"2004-09-01\"}, \"line\": 26390, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 204}, {\"relation\": \"increases\", \"evidence\": \"Preventing Abeta42 production with an M596I mutation (beta-), which blocks beta-secretase cleavage of APP, or by treatment with a gamma-secretase inhibitor increased the resistance of APP(FAD)-expressing cells to apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"15584903\", \"authors\": [\"Esposito L\", \"Gan L\", \"Yu GQ\", \"Essrich C\", \"Mucke L\"], \"date\": \"2004-12-01\"}, \"line\": 26402, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 205}, {\"relation\": \"increases\", \"evidence\": \"Preventing Abeta42 production with an M596I mutation (beta-), which blocks beta-secretase cleavage of APP, or by treatment with a gamma-secretase inhibitor increased the resistance of APP(FAD)-expressing cells to apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"15584903\", \"authors\": [\"Esposito L\", \"Gan L\", \"Yu GQ\", \"Essrich C\", \"Mucke L\"], \"date\": \"2004-12-01\"}, \"line\": 26402, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 206}, {\"relation\": \"increases\", \"evidence\": \"Although pmodest overexpression enhanced amyloid deposition, high BACE overexpression inhibited amyloid formation despite increased beta-cleavage of APP. BACE overexpression alters the subcellular processing of APP and inhibits Abeta deposition in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"15642747\", \"authors\": [\"Lee EB\", \"Zhang B\", \"Liu K\", \"Greenbaum EA\", \"Doms RW\", \"Trojanowski JQ\", \"Lee VM\"], \"date\": \"2005-01-17\"}, \"line\": 26412, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 207}, {\"relation\": \"increases\", \"evidence\": \"Although pmodest overexpression enhanced amyloid deposition, high BACE overexpression inhibited amyloid formation despite increased beta-cleavage of APP. BACE overexpression alters the subcellular processing of APP and inhibits Abeta deposition in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"15642747\", \"authors\": [\"Lee EB\", \"Zhang B\", \"Liu K\", \"Greenbaum EA\", \"Doms RW\", \"Trojanowski JQ\", \"Lee VM\"], \"date\": \"2005-01-17\"}, \"line\": 26412, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 208}, {\"relation\": \"increases\", \"evidence\": \"Here, we established a novel approach to regulate production of Abeta based on intracellular expression of single chain antibodies (intrabodies) raised to an epitope adjacent to the beta-secretase cleavage site of human APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"15767460\", \"authors\": [\"Paganetti P\", \"Calanca V\", \"Galli C\", \"Stefani M\", \"Molinari M\"], \"date\": \"2005-03-14\"}, \"line\": 26422, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 209}, {\"relation\": \"increases\", \"evidence\": \"Here, we established a novel approach to regulate production of Abeta based on intracellular expression of single chain antibodies (intrabodies) raised to an epitope adjacent to the beta-secretase cleavage site of human APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of cell biology\", \"reference\": \"15767460\", \"authors\": [\"Paganetti P\", \"Calanca V\", \"Galli C\", \"Stefani M\", \"Molinari M\"], \"date\": \"2005-03-14\"}, \"line\": 26422, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 210}, {\"relation\": \"increases\", \"evidence\": \"BACE1 increased APP cleavage at the beta-site and Abeta production whereas BACE2 did not.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"15857888\", \"authors\": [\"Sun X\", \"Wang Y\", \"Qing H\", \"Christensen MA\", \"Liu Y\", \"Zhou W\", \"Tong Y\", \"Xiao C\", \"Huang Y\", \"Zhang S\", \"Liu X\", \"Song W\"], \"date\": \"2005-05-01\"}, \"line\": 26431, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 211}, {\"relation\": \"increases\", \"evidence\": \"BACE1 increased APP cleavage at the beta-site and Abeta production whereas BACE2 did not.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"15857888\", \"authors\": [\"Sun X\", \"Wang Y\", \"Qing H\", \"Christensen MA\", \"Liu Y\", \"Zhou W\", \"Tong Y\", \"Xiao C\", \"Huang Y\", \"Zhang S\", \"Liu X\", \"Song W\"], \"date\": \"2005-05-01\"}, \"line\": 26431, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 212}, {\"relation\": \"increases\", \"evidence\": \"The aspartic protease beta-secretase (BACE) cleaves the amyloid precursor protein into a 42 residue beta-peptide, which is the principal biochemical marker of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Structure (London, England : 1993)\", \"reference\": \"16216580\", \"authors\": [\"Gorfe AA\", \"Caflisch A\"], \"date\": \"2005-10-01\"}, \"line\": 26441, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 213}, {\"relation\": \"increases\", \"evidence\": \"The aspartic protease beta-secretase (BACE) cleaves the amyloid precursor protein into a 42 residue beta-peptide, which is the principal biochemical marker of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Structure (London, England : 1993)\", \"reference\": \"16216580\", \"authors\": [\"Gorfe AA\", \"Caflisch A\"], \"date\": \"2005-10-01\"}, \"line\": 26441, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 214}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase [beta-site amyloid precursor protein-cleaving enzyme 1 (BACE1)] is the key rate-limiting enzyme for the production of the beta-amyloid (Abeta) peptide involved in the pathogenesis of Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"16306400\", \"authors\": [\"Velliquette RA\", \"O'Connor T\", \"Vassar R\"], \"date\": \"2005-11-23\"}, \"line\": 26452, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 215}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase [beta-site amyloid precursor protein-cleaving enzyme 1 (BACE1)] is the key rate-limiting enzyme for the production of the beta-amyloid (Abeta) peptide involved in the pathogenesis of Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"16306400\", \"authors\": [\"Velliquette RA\", \"O'Connor T\", \"Vassar R\"], \"date\": \"2005-11-23\"}, \"line\": 26452, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 216}, {\"relation\": \"increases\", \"evidence\": \"This was associated with inefficient plasminogen binding and plasmin activation, the displacement of beta-secretase (BACE) from DRMs to APP-containing membrane fractions, increased beta-cleavage of APP and high levels of Abeta peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The EMBO journal\", \"reference\": \"16407971\", \"authors\": [\"Crameri A\", \"Biondi E\", \"Kuehnle K\", \"Lütjohann D\", \"Thelen KM\", \"Perga S\", \"Dotti CG\", \"Nitsch RM\", \"Ledesma MD\", \"Mohajeri MH\"], \"date\": \"2006-01-25\"}, \"line\": 26461, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 217}, {\"relation\": \"increases\", \"evidence\": \"Heparin can promote beta-secretase cleavage of APP in neuroblastoma cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"16716081\", \"authors\": [\"Beckman M\", \"Holsinger RM\", \"Small DH\"], \"date\": \"2006-05-30\"}, \"line\": 26470, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 218}, {\"relation\": \"increases\", \"evidence\": \"Heparin can promote beta-secretase cleavage of APP in neuroblastoma cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"16716081\", \"authors\": [\"Beckman M\", \"Holsinger RM\", \"Small DH\"], \"date\": \"2006-05-30\"}, \"line\": 26470, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 219}, {\"relation\": \"increases\", \"evidence\": \"In AD, the specific inhibition of beta- or beta-secretase activities would decrease the production of Abeta from its precursor, in such a way that its relative concentration could be low enough to avoid the formation of aggregates.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bulletin et memoires de l'Academie royale de medecine de Belgique\", \"reference\": \"16768248\", \"authors\": [\"Octave JN\"], \"date\": \"2005-01-01\"}, \"line\": 26481, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 220}, {\"relation\": \"increases\", \"evidence\": \"In AD, the specific inhibition of beta- or beta-secretase activities would decrease the production of Abeta from its precursor, in such a way that its relative concentration could be low enough to avoid the formation of aggregates.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bulletin et memoires de l'Academie royale de medecine de Belgique\", \"reference\": \"16768248\", \"authors\": [\"Octave JN\"], \"date\": \"2005-01-01\"}, \"line\": 26481, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 221}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartyl protease that specifically cleaves amyloid precursor protein (APP) at the beta-secretase site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular biology\", \"reference\": \"16979658\", \"authors\": [\"He W\", \"Hu X\", \"Shi Q\", \"Zhou X\", \"Lu Y\", \"Fisher C\", \"Yan R\"], \"date\": \"2006-10-27\"}, \"line\": 26524, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 222}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartyl protease that specifically cleaves amyloid precursor protein (APP) at the beta-secretase site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular biology\", \"reference\": \"16979658\", \"authors\": [\"He W\", \"Hu X\", \"Shi Q\", \"Zhou X\", \"Lu Y\", \"Fisher C\", \"Yan R\"], \"date\": \"2006-10-27\"}, \"line\": 26524, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 223}, {\"relation\": \"increases\", \"evidence\": \"Using high-throughput siRNA screening technology, we assessed 15,200 genes for their role in Abeta42 secretion and identified leucine-rich repeat transmembrane 3 (LRRTM3) as a neuronal gene that promotes APP processing by BACE1. siRNAs targeting LRRTM3 inhibit the secretion of Abeta40, Abeta42, and sAPPbeta, the N-terminal APP fragment produced by BACE1 cleavage, from cultured cells and primary neurons by up to 60%, whereas overexpression increases Abeta secretion.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17098871\", \"authors\": [\"Majercak J\", \"Ray WJ\", \"Espeseth A\", \"Simon A\", \"Shi XP\", \"Wolffe C\", \"Getty K\", \"Marine S\", \"Stec E\", \"Ferrer M\", \"Strulovici B\", \"Bartz S\", \"Gates A\", \"Xu M\", \"Huang Q\", \"Ma L\", \"Shughrue P\", \"Burchard J\", \"Colussi D\", \"Pietrak B\", \"Kahana J\", \"Beher D\", \"Rosahl T\", \"Shearman M\", \"Hazuda D\", \"Sachs AB\", \"Koblan KS\", \"Seabrook GR\", \"Stone DJ\"], \"date\": \"2006-11-21\"}, \"line\": 26536, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 224}, {\"relation\": \"increases\", \"evidence\": \"Using high-throughput siRNA screening technology, we assessed 15,200 genes for their role in Abeta42 secretion and identified leucine-rich repeat transmembrane 3 (LRRTM3) as a neuronal gene that promotes APP processing by BACE1. siRNAs targeting LRRTM3 inhibit the secretion of Abeta40, Abeta42, and sAPPbeta, the N-terminal APP fragment produced by BACE1 cleavage, from cultured cells and primary neurons by up to 60%, whereas overexpression increases Abeta secretion.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17098871\", \"authors\": [\"Majercak J\", \"Ray WJ\", \"Espeseth A\", \"Simon A\", \"Shi XP\", \"Wolffe C\", \"Getty K\", \"Marine S\", \"Stec E\", \"Ferrer M\", \"Strulovici B\", \"Bartz S\", \"Gates A\", \"Xu M\", \"Huang Q\", \"Ma L\", \"Shughrue P\", \"Burchard J\", \"Colussi D\", \"Pietrak B\", \"Kahana J\", \"Beher D\", \"Rosahl T\", \"Shearman M\", \"Hazuda D\", \"Sachs AB\", \"Koblan KS\", \"Seabrook GR\", \"Stone DJ\"], \"date\": \"2006-11-21\"}, \"line\": 26536, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 225}, {\"relation\": \"increases\", \"evidence\": \"Hypoxia up-regulated beta-secretase cleavage of APP and amyloid-beta protein (Abeta) production by increasing BACE1 gene transcription and expression both in vitro and in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17121991\", \"authors\": [\"Sun X\", \"He G\", \"Qing H\", \"Zhou W\", \"Dobie F\", \"Cai F\", \"Staufenbiel M\", \"Huang LE\", \"Song W\"], \"date\": \"2006-12-05\"}, \"line\": 26553, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 226}, {\"relation\": \"increases\", \"evidence\": \"Hypoxia up-regulated beta-secretase cleavage of APP and amyloid-beta protein (Abeta) production by increasing BACE1 gene transcription and expression both in vitro and in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17121991\", \"authors\": [\"Sun X\", \"He G\", \"Qing H\", \"Zhou W\", \"Dobie F\", \"Cai F\", \"Staufenbiel M\", \"Huang LE\", \"Song W\"], \"date\": \"2006-12-05\"}, \"line\": 26553, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 227}, {\"relation\": \"increases\", \"evidence\": \"Hypoxia up-regulated beta-secretase cleavage of APP and amyloid-beta protein (Abeta) production by increasing BACE1 gene transcription and expression both in vitro and in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17121991\", \"authors\": [\"Sun X\", \"He G\", \"Qing H\", \"Zhou W\", \"Dobie F\", \"Cai F\", \"Staufenbiel M\", \"Huang LE\", \"Song W\"], \"date\": \"2006-12-05\"}, \"line\": 26553, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Hypoxia response subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 228}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach for the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"17156133\", \"authors\": [\"Hussain I\", \"Hawkins J\", \"Harrison D\", \"Hille C\", \"Wayne G\", \"Cutler L\", \"Buck T\", \"Walter D\", \"Demont E\", \"Howes C\", \"Naylor A\", \"Jeffrey P\", \"Gonzalez MI\", \"Dingwall C\", \"Michel A\", \"Redshaw S\", \"Davis JB\"], \"date\": \"2007-02-01\"}, \"line\": 26563, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 229}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach for the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"17156133\", \"authors\": [\"Hussain I\", \"Hawkins J\", \"Harrison D\", \"Hille C\", \"Wayne G\", \"Cutler L\", \"Buck T\", \"Walter D\", \"Demont E\", \"Howes C\", \"Naylor A\", \"Jeffrey P\", \"Gonzalez MI\", \"Dingwall C\", \"Michel A\", \"Redshaw S\", \"Davis JB\"], \"date\": \"2007-02-01\"}, \"line\": 26563, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 230}, {\"relation\": \"increases\", \"evidence\": \"In addition, we show that some NF-kappaB inhibitors decrease sAPPbeta and APP-CTFbeta suggesting that they reduce the beta-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"17223266\", \"authors\": [\"Paris D\", \"Patel N\", \"Quadros A\", \"Linan M\", \"Bakshi P\", \"Ait-Ghezala G\", \"Mullan M\"], \"date\": \"2007-03-19\"}, \"line\": 26572, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 231}, {\"relation\": \"increases\", \"evidence\": \"In addition, we show that some NF-kappaB inhibitors decrease sAPPbeta and APP-CTFbeta suggesting that they reduce the beta-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"17223266\", \"authors\": [\"Paris D\", \"Patel N\", \"Quadros A\", \"Linan M\", \"Bakshi P\", \"Ait-Ghezala G\", \"Mullan M\"], \"date\": \"2007-03-19\"}, \"line\": 26572, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 232}, {\"relation\": \"increases\", \"evidence\": \"Beta-site amyloid precursor protein cleaving enzyme 1 (BACE1) (beta-secretase) initiates generation of beta-amyloid (Abeta), which plays an early role in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17409228\", \"authors\": [\"Zhao J\", \"Fu Y\", \"Yasvoina M\", \"Shao P\", \"Hitt B\", \"O'Connor T\", \"Logan S\", \"Maus E\", \"Citron M\", \"Berry R\", \"Binder L\", \"Vassar R\"], \"date\": \"2007-04-04\"}, \"line\": 26584, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 233}, {\"relation\": \"increases\", \"evidence\": \"Beta-site amyloid precursor protein cleaving enzyme 1 (BACE1) (beta-secretase) initiates generation of beta-amyloid (Abeta), which plays an early role in Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17409228\", \"authors\": [\"Zhao J\", \"Fu Y\", \"Yasvoina M\", \"Shao P\", \"Hitt B\", \"O'Connor T\", \"Logan S\", \"Maus E\", \"Citron M\", \"Berry R\", \"Binder L\", \"Vassar R\"], \"date\": \"2007-04-04\"}, \"line\": 26584, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 234}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase is a potential target for inhibitory drugs against Alzheimer's disease as it cleaves amyloid precursor protein (APP) to form insoluble amyloid plaques and vascular deposits in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Computational biology and chemistry\", \"reference\": \"17500040\", \"authors\": [\"Zuo Z\", \"Gang C\", \"Zou H\", \"Mok PC\", \"Zhu W\", \"Chen K\", \"Jiang H\"], \"date\": \"2007-06-01\"}, \"line\": 26594, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 235}, {\"relation\": \"increases\", \"evidence\": \"Beta-secretase is a potential target for inhibitory drugs against Alzheimer's disease as it cleaves amyloid precursor protein (APP) to form insoluble amyloid plaques and vascular deposits in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Computational biology and chemistry\", \"reference\": \"17500040\", \"authors\": [\"Zuo Z\", \"Gang C\", \"Zou H\", \"Mok PC\", \"Zhu W\", \"Chen K\", \"Jiang H\"], \"date\": \"2007-06-01\"}, \"line\": 26594, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 236}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach to the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Analytical and bioanalytical chemistry\", \"reference\": \"17541560\", \"authors\": [\"Mancini F\", \"Naldi M\", \"Cavrini V\", \"Andrisano V\"], \"date\": \"2007-07-01\"}, \"line\": 26609, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 237}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach to the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Analytical and bioanalytical chemistry\", \"reference\": \"17541560\", \"authors\": [\"Mancini F\", \"Naldi M\", \"Cavrini V\", \"Andrisano V\"], \"date\": \"2007-07-01\"}, \"line\": 26609, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 238}, {\"relation\": \"increases\", \"evidence\": \"Beta-site APP-cleaving enzyme (BACE) is required for production of the Alzheimer's disease (AD)-associated Abeta protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"17553422\", \"authors\": [\"Tesco G\", \"Koh YH\", \"Kang EL\", \"Cameron AN\", \"Das S\", \"Sena-Esteves M\", \"Hiltunen M\", \"Yang SH\", \"Zhong Z\", \"Shen Y\", \"Simpkins JW\", \"Tanzi RE\"], \"date\": \"2007-06-07\"}, \"line\": 26624, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 239}, {\"relation\": \"increases\", \"evidence\": \"Beta-site APP-cleaving enzyme (BACE) is required for production of the Alzheimer's disease (AD)-associated Abeta protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"17553422\", \"authors\": [\"Tesco G\", \"Koh YH\", \"Kang EL\", \"Cameron AN\", \"Das S\", \"Sena-Esteves M\", \"Hiltunen M\", \"Yang SH\", \"Zhong Z\", \"Shen Y\", \"Simpkins JW\", \"Tanzi RE\"], \"date\": \"2007-06-07\"}, \"line\": 26624, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 240}, {\"relation\": \"increases\", \"evidence\": \"The aspartyl protease beta-site amyloid precursor protein cleaving enzyme 1 (BACE1) initiates processing of amyloid precursor protein (APP) into amyloid beta (Abeta) peptide, the major component of Alzheimer disease (AD) plaques.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"17616527\", \"authors\": [\"McConlogue L\", \"Buttini M\", \"Anderson JP\", \"Brigham EF\", \"Chen KS\", \"Freedman SB\", \"Games D\", \"Johnson-Wood K\", \"Lee M\", \"Zeller M\", \"Liu W\", \"Motter R\", \"Sinha S\"], \"date\": \"2007-09-07\"}, \"line\": 26634, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 241}, {\"relation\": \"increases\", \"evidence\": \"The aspartyl protease beta-site amyloid precursor protein cleaving enzyme 1 (BACE1) initiates processing of amyloid precursor protein (APP) into amyloid beta (Abeta) peptide, the major component of Alzheimer disease (AD) plaques.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"17616527\", \"authors\": [\"McConlogue L\", \"Buttini M\", \"Anderson JP\", \"Brigham EF\", \"Chen KS\", \"Freedman SB\", \"Games D\", \"Johnson-Wood K\", \"Lee M\", \"Zeller M\", \"Liu W\", \"Motter R\", \"Sinha S\"], \"date\": \"2007-09-07\"}, \"line\": 26634, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 242}, {\"relation\": \"increases\", \"evidence\": \"The proteolytic enzyme beta-secretase (BACE-1) produces amyloid beta (Abeta) peptide, the primary constituent of neurofibrillary plaques, implicated in Alzheimer's disease, by cleavage of the amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bioorganic &amp; medicinal chemistry letters\", \"reference\": \"18068983\", \"authors\": [\"Jennings LD\", \"Cole DC\", \"Stock JR\", \"Sukhdeo MN\", \"Ellingboe JW\", \"Cowling R\", \"Jin G\", \"Manas ES\", \"Fan KY\", \"Malamas MS\", \"Harrison BL\", \"Jacobsen S\", \"Chopra R\", \"Lohse PA\", \"Moore WJ\", \"O'Donnell MM\", \"Hu Y\", \"Robichaud AJ\", \"Turner MJ\", \"Wagner E\", \"Bard J\"], \"date\": \"2008-01-15\"}, \"line\": 26644, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 243}, {\"relation\": \"increases\", \"evidence\": \"The proteolytic enzyme beta-secretase (BACE-1) produces amyloid beta (Abeta) peptide, the primary constituent of neurofibrillary plaques, implicated in Alzheimer's disease, by cleavage of the amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Bioorganic &amp; medicinal chemistry letters\", \"reference\": \"18068983\", \"authors\": [\"Jennings LD\", \"Cole DC\", \"Stock JR\", \"Sukhdeo MN\", \"Ellingboe JW\", \"Cowling R\", \"Jin G\", \"Manas ES\", \"Fan KY\", \"Malamas MS\", \"Harrison BL\", \"Jacobsen S\", \"Chopra R\", \"Lohse PA\", \"Moore WJ\", \"O'Donnell MM\", \"Hu Y\", \"Robichaud AJ\", \"Turner MJ\", \"Wagner E\", \"Bard J\"], \"date\": \"2008-01-15\"}, \"line\": 26644, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 244}, {\"relation\": \"increases\", \"evidence\": \"After inhibitor treatment, the improved memory function was accompanied by reduced amyloid plaque load, decreased Abeta40 and Abeta42, and reduced C-terminal beta-secretase fragment derived from APP by beta-secretase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18184658\", \"authors\": [\"Hook VY\", \"Kindy M\", \"Hook G\"], \"date\": \"2008-03-21\"}, \"line\": 26652, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 245}, {\"relation\": \"increases\", \"evidence\": \"Beta-amyloid (Abeta) peptides that accumulate in Alzheimer disease are generated from the beta-amyloid precursor protein (betaAPP) by cleavages by beta-secretase BACE1 and by presenilin-dependent gamma-secretase activities.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18263584\", \"authors\": [\"Buggia-Prevot V\", \"Sevalle J\", \"Rossner S\", \"Checler F\"], \"date\": \"2008-04-11\"}, \"line\": 26662, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 246}, {\"relation\": \"increases\", \"evidence\": \"Beta-amyloid (Abeta) peptides that accumulate in Alzheimer disease are generated from the beta-amyloid precursor protein (betaAPP) by cleavages by beta-secretase BACE1 and by presenilin-dependent gamma-secretase activities.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18263584\", \"authors\": [\"Buggia-Prevot V\", \"Sevalle J\", \"Rossner S\", \"Checler F\"], \"date\": \"2008-04-11\"}, \"line\": 26662, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 247}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach for the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochimica et biophysica acta\", \"reference\": \"18295609\", \"authors\": [\"Shimmyo Y\", \"Kihara T\", \"Akaike A\", \"Niidome T\", \"Sugimoto H\"], \"date\": \"2008-05-01\"}, \"line\": 26672, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 248}, {\"relation\": \"increases\", \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach for the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochimica et biophysica acta\", \"reference\": \"18295609\", \"authors\": [\"Shimmyo Y\", \"Kihara T\", \"Akaike A\", \"Niidome T\", \"Sugimoto H\"], \"date\": \"2008-05-01\"}, \"line\": 26672, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 249}, {\"relation\": \"increases\", \"evidence\": \"In our opinion, albeit based on limited available data, a future potential therapeutic strategy is to mimic the mechanism by which the normal cellular form of the prion protein inhibits the beta-secretase beta-site amyloid precursor protein cleaving enzyme-1 (BACE1), and hence the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Expert opinion on therapeutic targets\", \"reference\": \"18479216\", \"authors\": [\"Griffiths HH\", \"Morten IJ\", \"Hooper NM\"], \"date\": \"2008-06-01\"}, \"line\": 26687, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 250}, {\"relation\": \"increases\", \"evidence\": \"In our opinion, albeit based on limited available data, a future potential therapeutic strategy is to mimic the mechanism by which the normal cellular form of the prion protein inhibits the beta-secretase beta-site amyloid precursor protein cleaving enzyme-1 (BACE1), and hence the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Expert opinion on therapeutic targets\", \"reference\": \"18479216\", \"authors\": [\"Griffiths HH\", \"Morten IJ\", \"Hooper NM\"], \"date\": \"2008-06-01\"}, \"line\": 26687, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 251}, {\"relation\": \"increases\", \"evidence\": \"BACE-1 cleavage is limiting for the production of Abeta, making it a particularly good drug target for the generation of inhibitors that lower Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS &amp; neurological disorders drug targets\", \"reference\": \"18673212\", \"authors\": [\"Venugopal C\", \"Demos CM\", \"Rao KS\", \"Pappolla MA\", \"Sambamurti K\"], \"date\": \"2008-06-01\"}, \"line\": 26697, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 252}, {\"relation\": \"increases\", \"evidence\": \"One group described the poor kinetics of BACE 1 for cleaving the wild-type (WT) beta-secretase site of APP found in most AD patients.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biological chemistry\", \"reference\": \"18979625\", \"authors\": [\"Hook V\", \"Schechter I\", \"Demuth HU\", \"Hook G\"], \"date\": \"2008-08-01\"}, \"line\": 26706, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 253}, {\"relation\": \"increases\", \"evidence\": \"One group described the poor kinetics of BACE 1 for cleaving the wild-type (WT) beta-secretase site of APP found in most AD patients.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biological chemistry\", \"reference\": \"18979625\", \"authors\": [\"Hook V\", \"Schechter I\", \"Demuth HU\", \"Hook G\"], \"date\": \"2008-08-01\"}, \"line\": 26706, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 254}, {\"relation\": \"increases\", \"evidence\": \"These results indicate that post-translational S-palmitoylation of BACE1 is not required for APP processing, and that BACE1 can efficiently cleave APP in both raft and non-raft microdomains.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"19074428\", \"authors\": [\"Vetrivel KS\", \"Meckler X\", \"Chen Y\", \"Nguyen PD\", \"Seidah NG\", \"Vassar R\", \"Wong PC\", \"Fukata M\", \"Kounnas MZ\", \"Thinakaran G\"], \"date\": \"2009-02-06\"}, \"line\": 26714, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 255}, {\"relation\": \"increases\", \"evidence\": \"Because beta-site APP-cleaving enzyme 1 (BACE1), an essential protease for Abeta production, is up-regulated in cells overexpressing RAGE and in RAGE-injected brains of Tg2576 mice, the molecular mechanisms underlying RAGE, BACE1 expression, and Abeta production were examined.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"19332646\", \"authors\": [\"Cho HJ\", \"Son SM\", \"Jin SM\", \"Hong HS\", \"Shin DH\", \"Kim SJ\", \"Huh K\", \"Mook-Jung I\"], \"date\": \"2009-08-01\"}, \"line\": 26725, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 256}, {\"relation\": \"increases\", \"evidence\": \"Because beta-site APP-cleaving enzyme 1 (BACE1), an essential protease for Abeta production, is up-regulated in cells overexpressing RAGE and in RAGE-injected brains of Tg2576 mice, the molecular mechanisms underlying RAGE, BACE1 expression, and Abeta production were examined.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"19332646\", \"authors\": [\"Cho HJ\", \"Son SM\", \"Jin SM\", \"Hong HS\", \"Shin DH\", \"Kim SJ\", \"Huh K\", \"Mook-Jung I\"], \"date\": \"2009-08-01\"}, \"line\": 26725, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 257}, {\"relation\": \"increases\", \"evidence\": \"However, unlike BRI2, the binding of BRI3 to the beta-secretase cleaved APP C-terminal fragment is negligible and BRI3 does not cause the massive accumulation of this APP fragment, suggesting that, unlike BRI2, BRI3 is a poor gamma-cleavage inhibitor.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"19366692\", \"authors\": [\"Matsuda S\", \"Matsuda Y\", \"D'Adamio L\"], \"date\": \"2009-06-05\"}, \"line\": 26737, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 258}, {\"relation\": \"increases\", \"evidence\": \"However, unlike BRI2, the binding of BRI3 to the beta-secretase cleaved APP C-terminal fragment is negligible and BRI3 does not cause the massive accumulation of this APP fragment, suggesting that, unlike BRI2, BRI3 is a poor gamma-cleavage inhibitor.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"19366692\", \"authors\": [\"Matsuda S\", \"Matsuda Y\", \"D'Adamio L\"], \"date\": \"2009-06-05\"}, \"line\": 26737, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 259}, {\"relation\": \"increases\", \"evidence\": \"BACE 1 (beta-site APP-cleaving enzyme 1 or beta-secretase), the key enzyme required for generating Abeta from the beta-amyloid precursor protein (APP), is regarded as an ideal target for AD therapeutic drug design.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"19442147\", \"authors\": [\"Huang WH\", \"Sheng R\", \"Hu YZ\"], \"date\": \"2009-01-01\"}, \"line\": 26747, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 260}, {\"relation\": \"increases\", \"evidence\": \"BACE 1 (beta-site APP-cleaving enzyme 1 or beta-secretase), the key enzyme required for generating Abeta from the beta-amyloid precursor protein (APP), is regarded as an ideal target for AD therapeutic drug design.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"19442147\", \"authors\": [\"Huang WH\", \"Sheng R\", \"Hu YZ\"], \"date\": \"2009-01-01\"}, \"line\": 26747, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 261}, {\"relation\": \"increases\", \"evidence\": \"The beta-secretase, beta-site amyloid precursor protein cleaving enzyme (BACE1; also called Asp2, memapsin 2), is the enzyme responsible for initiating Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"19828790\", \"authors\": [\"Vassar R\", \"Kovacs DM\", \"Yan R\", \"Wong PC\"], \"date\": \"2009-10-14\"}, \"line\": 26757, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 262}, {\"relation\": \"increases\", \"evidence\": \"The beta-secretase, beta-site amyloid precursor protein cleaving enzyme (BACE1; also called Asp2, memapsin 2), is the enzyme responsible for initiating Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"19828790\", \"authors\": [\"Vassar R\", \"Kovacs DM\", \"Yan R\", \"Wong PC\"], \"date\": \"2009-10-14\"}, \"line\": 26757, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 263}, {\"relation\": \"increases\", \"evidence\": \"BACE1 initiates the amyloidogenic processing of APP; therefore, early or active amyloidogenic loci might exhibit site-specific BACE1 and Abeta elevation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The European journal of neuroscience\", \"reference\": \"20092570\", \"authors\": [\"Zhang XM\", \"Cai Y\", \"Xiong K\", \"Cai H\", \"Luo XG\", \"Feng JC\", \"Clough RW\", \"Struble RG\", \"Patrylo PR\", \"Yan XX\"], \"date\": \"2009-12-01\"}, \"line\": 26767, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 264}, {\"relation\": \"increases\", \"evidence\": \"BACE1 initiates the amyloidogenic processing of APP; therefore, early or active amyloidogenic loci might exhibit site-specific BACE1 and Abeta elevation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The European journal of neuroscience\", \"reference\": \"20092570\", \"authors\": [\"Zhang XM\", \"Cai Y\", \"Xiong K\", \"Cai H\", \"Luo XG\", \"Feng JC\", \"Clough RW\", \"Struble RG\", \"Patrylo PR\", \"Yan XX\"], \"date\": \"2009-12-01\"}, \"line\": 26767, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 265}, {\"relation\": \"increases\", \"evidence\": \"Taken together, ATXN1 functions as a genetic risk pmodifier that contributes to AD pathogenesis through a loss-of-function mechanism by regulating beta-secretase cleavage of APP and Abeta levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20097758\", \"authors\": [\"Zhang C\", \"Browne A\", \"Child D\", \"Divito JR\", \"Stevenson JA\", \"Tanzi RE\"], \"date\": \"2010-03-19\"}, \"line\": 26777, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 266}, {\"relation\": \"increases\", \"evidence\": \"Taken together, ATXN1 functions as a genetic risk pmodifier that contributes to AD pathogenesis through a loss-of-function mechanism by regulating beta-secretase cleavage of APP and Abeta levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20097758\", \"authors\": [\"Zhang C\", \"Browne A\", \"Child D\", \"Divito JR\", \"Stevenson JA\", \"Tanzi RE\"], \"date\": \"2010-03-19\"}, \"line\": 26777, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 267}, {\"relation\": \"increases\", \"evidence\": \"Three proteases that are involved in the processing of amyloid precursor protein-alpha-secretase, beta-secretase and gamma-secretase-are of particular interest as they are central to the generation and pmodulation of amyloid-beta peptide and can be targeted by small compounds in vitro and in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature reviews. Neurology\", \"reference\": \"20139999\", \"authors\": [\"De Strooper B\", \"Vassar R\", \"Golde T\"], \"date\": \"2010-02-01\"}, \"line\": 26797, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 268}, {\"relation\": \"increases\", \"evidence\": \"Three proteases that are involved in the processing of amyloid precursor protein-alpha-secretase, beta-secretase and gamma-secretase-are of particular interest as they are central to the generation and pmodulation of amyloid-beta peptide and can be targeted by small compounds in vitro and in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature reviews. Neurology\", \"reference\": \"20139999\", \"authors\": [\"De Strooper B\", \"Vassar R\", \"Golde T\"], \"date\": \"2010-02-01\"}, \"line\": 26797, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 269}, {\"relation\": \"increases\", \"evidence\": \"Although pmoderate decreases of either gamma-secretase or BACE1 are not associated with mechanism-based toxicities, they provide only pmodest benefits in reducing Abeta in the brains of APPswe/PS1DeltaE9 mice.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science translational medicine\", \"reference\": \"20371462\", \"authors\": [\"Chow VW\", \"Savonenko AV\", \"Melnikova T\", \"Kim H\", \"Price DL\", \"Li T\", \"Wong PC\"], \"date\": \"2010-01-06\"}, \"line\": 26825, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 270}, {\"relation\": \"increases\", \"evidence\": \"Although pmoderate decreases of either gamma-secretase or BACE1 are not associated with mechanism-based toxicities, they provide only pmodest benefits in reducing Abeta in the brains of APPswe/PS1DeltaE9 mice.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science translational medicine\", \"reference\": \"20371462\", \"authors\": [\"Chow VW\", \"Savonenko AV\", \"Melnikova T\", \"Kim H\", \"Price DL\", \"Li T\", \"Wong PC\"], \"date\": \"2010-01-06\"}, \"line\": 26825, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 271}, {\"relation\": \"increases\", \"evidence\": \"Reticulons are a group of membrane-bound proteins involved in diverse cellular functions, and are suggested to act as inhibitors of beta-secretase enzyme 1 (BACE1) activity that cleaves amyloid precursor protein. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropathology : official journal of the Japanese Society of Neuropathology\", \"reference\": \"20374499\", \"authors\": [\"Heath JE\", \"Siedlak SL\", \"Zhu X\", \"Lee HG\", \"Thakur A\", \"Yan R\", \"Perry G\", \"Smith MA\", \"Castellani RJ\"], \"date\": \"2010-12-01\"}, \"line\": 26832, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 272}, {\"relation\": \"increases\", \"evidence\": \"Here we tested the possibility of targeting the cellular environment of beta-secretase cleavage instead of the beta-secretase enzyme itself. beta-Secretase has an acidic pH optimum and cleaves the amyloid precursor protein in the acidic endosomes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"20592218\", \"authors\": [\"Mitterreiter S\", \"Page RM\", \"Kamp F\", \"Hopson J\", \"Winkler E\", \"Ha HR\", \"Hamid R\", \"Herms J\", \"Mayer TU\", \"Nelson DJ\", \"Steiner H\", \"Stahl T\", \"Zeitschel U\", \"Rossner S\", \"Haass C\", \"Lichtenthaler SF\"], \"date\": \"2010-06-30\"}, \"line\": 26842, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 273}, {\"relation\": \"increases\", \"evidence\": \"ABCA2 expression promoted b-secretase (BACE1) cleavage of APP not at the common Asp1 amino acid site (beta-site) of Abeta in APP but at the Glu11 site (beta'-site) to increase C89 carboxyl-terminal fragment levels (beta'-CTF/C89). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"20704561\", \"authors\": [\"Davis W Jr\"], \"date\": \"2010-11-01\"}, \"line\": 26849, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 274}, {\"relation\": \"increases\", \"evidence\": \"We discovered a nonpeptidic compound, TAK-070, that inhibited BACE1, a rate-limiting protease for the generation of Abeta peptides that are considered causative for Alzheimer's disease (AD), in a noncompetitive manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"20720123\", \"authors\": [\"Fukumoto H\", \"Takahashi H\", \"Tarui N\", \"Matsui J\", \"Tomita T\", \"Hirode M\", \"Sagayama M\", \"Maeda R\", \"Kawamoto M\", \"Hirai K\", \"Terauchi J\", \"Sakura Y\", \"Kakihana M\", \"Kato K\", \"Iwatsubo T\", \"Miyamoto M\"], \"date\": \"2010-08-18\"}, \"line\": 26861, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 275}, {\"relation\": \"increases\", \"evidence\": \"We discovered a nonpeptidic compound, TAK-070, that inhibited BACE1, a rate-limiting protease for the generation of Abeta peptides that are considered causative for Alzheimer's disease (AD), in a noncompetitive manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"20720123\", \"authors\": [\"Fukumoto H\", \"Takahashi H\", \"Tarui N\", \"Matsui J\", \"Tomita T\", \"Hirode M\", \"Sagayama M\", \"Maeda R\", \"Kawamoto M\", \"Hirai K\", \"Terauchi J\", \"Sakura Y\", \"Kakihana M\", \"Kato K\", \"Iwatsubo T\", \"Miyamoto M\"], \"date\": \"2010-08-18\"}, \"line\": 26861, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 276}, {\"relation\": \"increases\", \"evidence\": \"Western blot analyses detected increased levels of BACE1 protein and beta-site-cleavage amyloid precursor protein C-terminal fragments in plaque-bearing human and monkey cortex relative to controls. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The European journal of neuroscience\", \"reference\": \"20726888\", \"authors\": [\"Cai Y\", \"Xiong K\", \"Zhang XM\", \"Cai H\", \"Luo XG\", \"Feng JC\", \"Clough RW\", \"Struble RG\", \"Patrylo PR\", \"Chu Y\", \"Kordower JH\", \"Yan XX\"], \"date\": \"2010-10-01\"}, \"line\": 26869, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 277}, {\"relation\": \"increases\", \"evidence\": \"beta-Site APP-cleaving enzyme 1 (BACE1) initiates amyloid-beta (Abeta) generation and thus represents a prime therapeutic target in treating Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"20886088\", \"authors\": [\"Devi L\", \"Ohno M\"], \"date\": \"2010-09-23\"}, \"line\": 26877, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 278}, {\"relation\": \"increases\", \"evidence\": \"Importantly, recent evidence reveals that expression and activity levels of the beta-site APP cleaving enzyme 1 (BACE1), which initiates amyloid-beta (Abeta) production, are elevated in AD brains.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular brain\", \"reference\": \"21059265\", \"authors\": [\"Devi L\", \"Alldred MJ\", \"Ginsberg SD\", \"Ohno M\"], \"date\": \"2010-11-08\"}, \"line\": 26887, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 279}, {\"relation\": \"increases\", \"evidence\": \"Importantly, recent evidence reveals that expression and activity levels of the beta-site APP cleaving enzyme 1 (BACE1), which initiates amyloid-beta (Abeta) production, are elevated in AD brains.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular brain\", \"reference\": \"21059265\", \"authors\": [\"Devi L\", \"Alldred MJ\", \"Ginsberg SD\", \"Ohno M\"], \"date\": \"2010-11-08\"}, \"line\": 26887, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 280}, {\"relation\": \"increases\", \"evidence\": \"BACE1 (beta-site beta-amyloid precursor protein (APP)-cleaving enzyme 1) mediates the first proteolytic cleavage of APP, leading to amyloid beta-peptide (Abeta) production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"21245145\", \"authors\": [\"Finan GM\", \"Okada H\", \"Kim TW\"], \"date\": \"2011-04-08\"}, \"line\": 26895, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 281}, {\"relation\": \"increases\", \"evidence\": \"Increased beta-secretase cleavage of APP after introduction of the Swedish double mutation causes apical missorting of about 20% of beta-secretase-cleaved APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"7876155\", \"authors\": [\"Greenberg B\", \"Craessaerts K\", \"Van den Berghe H\", \"Moechars D\", \"Van Leuven F\", \"De Strooper B\", \"Dewachter I\"], \"date\": \"1995-02-24\"}, \"line\": 26902, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 282}, {\"relation\": \"increases\", \"evidence\": \"Taken together, the data suggest that the processing pathway for betaPP is similar for both betaPP-wt and betaPP-sw cells and that increased Abeta production by betaPP-sw cells arises from enhanced cleavage of mutant betaPP by beta-secretase, the as-yet unidentified enzyme(s) that cleaves at the NH2 terminus of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"8621560\", \"authors\": [\"Perez RG\", \"Squazzo SL\", \"Koo EH\"], \"date\": \"1996-04-12\"}, \"line\": 26913, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 283}, {\"relation\": \"increases\", \"evidence\": \"Taken together, the data suggest that the processing pathway for betaPP is similar for both betaPP-wt and betaPP-sw cells and that increased Abeta production by betaPP-sw cells arises from enhanced cleavage of mutant betaPP by beta-secretase, the as-yet unidentified enzyme(s) that cleaves at the NH2 terminus of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"8621560\", \"authors\": [\"Perez RG\", \"Squazzo SL\", \"Koo EH\"], \"date\": \"1996-04-12\"}, \"line\": 26913, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 284}, {\"relation\": \"increases\", \"evidence\": \"Heparin promotes beta-secretase cleavage of the Alzheimer's amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurochemistry international\", \"reference\": \"9152995\", \"authors\": [\"Mistretta S\", \"Leveugle B\", \"Fillit HM\", \"Greenberg BD\", \"Siman R\", \"Durkin JT\", \"Ding W\", \"Matic M\", \"Eisle J\"], \"date\": \"1997-06-01\"}, \"line\": 26920, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 285}, {\"relation\": \"increases\", \"evidence\": \"Sequential proteolytic cleavage of APP by beta-secretase and gamma-secretase liberates Abeta from APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biological chemistry\", \"reference\": \"20731541\", \"authors\": [\"Klaver DW\", \"Wilce MC\", \"Cui H\", \"Hung AC\", \"Gasperini R\", \"Foa L\", \"Small DH\"], \"date\": \"2010-08-01\"}, \"line\": 26930, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 286}, {\"relation\": \"increases\", \"evidence\": \"Sequential proteolytic cleavage of APP by beta-secretase and gamma-secretase liberates Abeta from APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biological chemistry\", \"reference\": \"20731541\", \"authors\": [\"Klaver DW\", \"Wilce MC\", \"Cui H\", \"Hung AC\", \"Gasperini R\", \"Foa L\", \"Small DH\"], \"date\": \"2010-08-01\"}, \"line\": 26930, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 287}, {\"relation\": \"increases\", \"evidence\": \"Formation of stable complexes between two Alzheimer's disease gene products: presenilin-2 and beta-amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature medicine\", \"reference\": \"9055862\", \"authors\": [\"Dürrwang U\", \"Paliga K\", \"Evin G\", \"Masters CL\", \"Czech C\", \"Weidemann A\", \"Beyreuther K\"], \"date\": \"1997-03-01\"}, \"line\": 26939, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 288}, {\"relation\": \"increases\", \"evidence\": \"Formation of stable complexes between two Alzheimer's disease gene products: presenilin-2 and beta-amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature medicine\", \"reference\": \"9055862\", \"authors\": [\"Dürrwang U\", \"Paliga K\", \"Evin G\", \"Masters CL\", \"Czech C\", \"Weidemann A\", \"Beyreuther K\"], \"date\": \"1997-03-01\"}, \"line\": 26939, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 289}, {\"relation\": \"increases\", \"evidence\": \"The novel transmembrane aspartic protease BACE (for Beta-site APP Cleaving Enzyme) is the beta-secretase that cleaves amyloid precursor protein to initiate beta-amyloid formation. As such, BACE is a prime therapeutic target for the treatment of Alzheimer's disease. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"10956649\", \"authors\": [\"Haniu M\", \"Citron M\", \"Vassar R\", \"Louis JC\", \"Bennett BD\", \"Denis P\", \"Teplow DB\", \"Kahn S\"], \"date\": \"2000-12-01\"}, \"line\": 27314, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {}, \"source\": 55, \"target\": 1, \"key\": 290}, {\"relation\": \"increases\", \"evidence\": \"beta-Secretase (BACE) carries out the first of two proteolysis steps to generate the amyloid-beta peptides that accumulate in the senile plaques in Alzheimer's disease (AD). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of cell science\", \"reference\": \"15466887\", \"authors\": [\"von Arnim CA\", \"Tangredi MM\", \"Peltan ID\", \"Lee BM\", \"Irizarry MC\", \"Kinoshita A\", \"Hyman BT\"], \"date\": \"2004-10-15\"}, \"line\": 27339, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 291}, {\"relation\": \"increases\", \"evidence\": \"beta-Secretase (BACE) carries out the first of two proteolysis steps to generate the amyloid-beta peptides that accumulate in the senile plaques in Alzheimer's disease (AD). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of cell science\", \"reference\": \"15466887\", \"authors\": [\"von Arnim CA\", \"Tangredi MM\", \"Peltan ID\", \"Lee BM\", \"Irizarry MC\", \"Kinoshita A\", \"Hyman BT\"], \"date\": \"2004-10-15\"}, \"line\": 27339, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 292}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartyl protease that specifically cleaves amyloid precursor protein (APP) at the beta-secretase site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular biology\", \"reference\": \"16979658\", \"authors\": [\"He W\", \"Hu X\", \"Shi Q\", \"Zhou X\", \"Lu Y\", \"Fisher C\", \"Yan R\"], \"date\": \"2006-10-27\"}, \"line\": 27402, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 293}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartyl protease that specifically cleaves amyloid precursor protein (APP) at the beta-secretase site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular biology\", \"reference\": \"16979658\", \"authors\": [\"He W\", \"Hu X\", \"Shi Q\", \"Zhou X\", \"Lu Y\", \"Fisher C\", \"Yan R\"], \"date\": \"2006-10-27\"}, \"line\": 27402, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 294}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartic protease that cleaves the amyloid precursor protein (APP) at the beta-secretase site, a critical step in the Alzheimer disease pathogenesis. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12473667\", \"authors\": [\"Kitazume S\", \"Tachida Y\", \"Oka R\", \"Kotani N\", \"Ogawa K\", \"Suzuki M\", \"Dohmae N\", \"Takio K\", \"Saido TC\", \"Hashimoto Y\"], \"date\": \"2003-04-25\"}, \"line\": 27422, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 295}, {\"relation\": \"increases\", \"evidence\": \"BACE1 is a membrane-bound aspartic protease that cleaves the amyloid precursor protein (APP) at the beta-secretase site, a critical step in the Alzheimer disease pathogenesis. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12473667\", \"authors\": [\"Kitazume S\", \"Tachida Y\", \"Oka R\", \"Kotani N\", \"Ogawa K\", \"Suzuki M\", \"Dohmae N\", \"Takio K\", \"Saido TC\", \"Hashimoto Y\"], \"date\": \"2003-04-25\"}, \"line\": 27422, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 296}, {\"relation\": \"increases\", \"evidence\": \"LRRTM3 promotes processing of amyloid-precursor protein by BACE1 \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17098871\", \"authors\": [\"Majercak J\", \"Ray WJ\", \"Espeseth A\", \"Simon A\", \"Shi XP\", \"Wolffe C\", \"Getty K\", \"Marine S\", \"Stec E\", \"Ferrer M\", \"Strulovici B\", \"Bartz S\", \"Gates A\", \"Xu M\", \"Huang Q\", \"Ma L\", \"Shughrue P\", \"Burchard J\", \"Colussi D\", \"Pietrak B\", \"Kahana J\", \"Beher D\", \"Rosahl T\", \"Shearman M\", \"Hazuda D\", \"Sachs AB\", \"Koblan KS\", \"Seabrook GR\", \"Stone DJ\"], \"date\": \"2006-11-21\"}, \"line\": 29749, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 297}, {\"relation\": \"increases\", \"evidence\": \"The complex mediates the intramembraneous proteolysis of beta-secretase cleaved beta-amyloid precursor protein (APP) leading to the secretion of the Alzheimer's disease-associated amyloid beta-peptide (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"15189355\", \"authors\": [\"Shirotani K\", \"Edbauer D\", \"Kostka M\", \"Steiner H\", \"Haass C\"], \"date\": \"2004-06-01\"}, \"line\": 29994, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 298}, {\"relation\": \"increases\", \"evidence\": \"The complex mediates the intramembraneous proteolysis of beta-secretase cleaved beta-amyloid precursor protein (APP) leading to the secretion of the Alzheimer's disease-associated amyloid beta-peptide (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"15189355\", \"authors\": [\"Shirotani K\", \"Edbauer D\", \"Kostka M\", \"Steiner H\", \"Haass C\"], \"date\": \"2004-06-01\"}, \"line\": 29994, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 299}, {\"relation\": \"increases\", \"evidence\": \"The familial Alzheimer's disease gene product amyloid beta precursor protein (APP) is sequentially processed by beta- and gamma-secretases to generate the Abeta peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11724784\", \"authors\": [\"Scheinfeld MH\", \"D'Adamio L\", \"Roncarati R\", \"Abdallah M\", \"Vito P\", \"Lopez PA\"], \"date\": \"2002-02-01\"}, \"line\": 30964, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 300}, {\"relation\": \"increases\", \"evidence\": \"The familial Alzheimer's disease gene product amyloid beta precursor protein (APP) is sequentially processed by beta- and gamma-secretases to generate the Abeta peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11724784\", \"authors\": [\"Scheinfeld MH\", \"D'Adamio L\", \"Roncarati R\", \"Abdallah M\", \"Vito P\", \"Lopez PA\"], \"date\": \"2002-02-01\"}, \"line\": 30964, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 301}, {\"relation\": \"increases\", \"evidence\": \"The level of Abeta-site APP-cleaving enzyme 1 (BACE1) has been documented to increase in the brains of patients with Alzheimer's disease, which has resulted in elevation of Abeta-amyloid (Abeta) peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"European journal of pharmacology\", \"reference\": \"22166376\", \"authors\": [\"Zhang J\", \"Chen XC\", \"Pan XD\", \"Lin N\", \"Zhu YG\", \"Lin ZY\", \"Chen LM\"], \"date\": \"2012-01-30\"}, \"line\": 41945, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 302}, {\"relation\": \"increases\", \"evidence\": \"The level of Abeta-site APP-cleaving enzyme 1 (BACE1) has been documented to increase in the brains of patients with Alzheimer's disease, which has resulted in elevation of Abeta-amyloid (Abeta) peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"European journal of pharmacology\", \"reference\": \"22166376\", \"authors\": [\"Zhang J\", \"Chen XC\", \"Pan XD\", \"Lin N\", \"Zhu YG\", \"Lin ZY\", \"Chen LM\"], \"date\": \"2012-01-30\"}, \"line\": 41945, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 1, \"key\": 303}, {\"relation\": \"increases\", \"evidence\": \"Down regulation of DNMT results in hypomethylation of BACE1 and APP which are involved in Abeta production and causes upregulation of their protein expression; in turn SP1 transcription factor increases which finally results in Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Free radical biology &amp; medicine\", \"reference\": \"19245828\", \"authors\": [\"Zawia NH\", \"Lahiri DK\", \"Cardozo-Pelaez F\"], \"date\": \"2009-05-01\"}, \"line\": 42554, \"annotations\": {\"Species\": \"10090\"}, \"source\": 55, \"target\": 1, \"key\": 304}, {\"relation\": \"increases\", \"evidence\": \"Physiological levels of ABeta¸ also have trophic and neuroprotective actions in trophic deprived conditions [95]. Many ABeta¸ has a physiological role in normal synapse function. In organotypic hippocampal slices, BACE activity is increased by synaptic activity and the resulting ABeta¸ peptides depress excitatory transmission through a-amino-3-hydroxyl-5-methyl-4-isoxazole-propionate (AMPA) and N-methyl-D-aspartic acid (NMDA) receptors, suggesting a role for ABeta¸ in homeostatic plasticity [29]. ABeta¸ may have an important physiological role in synapse elimination during brain development\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34927, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 55, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Physiological levels of ABeta¸ also have trophic and neuroprotective actions in trophic deprived conditions [95]. Many ABeta¸ has a physiological role in normal synapse function. In organotypic hippocampal slices, BACE activity is increased by synaptic activity and the resulting ABeta¸ peptides depress excitatory transmission through a-amino-3-hydroxyl-5-methyl-4-isoxazole-propionate (AMPA) and N-methyl-D-aspartic acid (NMDA) receptors, suggesting a role for ABeta¸ in homeostatic plasticity [29]. ABeta¸ may have an important physiological role in synapse elimination during brain development\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34927, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 55, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Insulin modulates metabolism of β-amyloid precursor protein (APP) in neurons, decreasing the intracellular accumulation of β-amyloid (Abeta) peptides, which are pivotal in AD pathogenesis. The present study investigates whether the widely prescribed insulin-sensitizing drug, metformin (GlucophageR), affects APP metabolism and Abeta generation in various cell models. We demonstrate that metformin, at doses that lead to activation of the AMP-activated protein kinase (AMPK), significantly increases the generation of both intracellular and extracellular Abeta species\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"19237574\", \"authors\": [\"Chen Y\", \"Zhou K\", \"Wang R\", \"Liu Y\", \"Kwak YD\", \"Ma T\", \"Thompson RC\", \"Zhao Y\", \"Smith L\", \"Gasparini L\", \"Luo Z\", \"Xu H\", \"Liao FF\"], \"date\": \"2009-03-10\"}, \"line\": 7645, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"Insulin signal transduction\"}, \"source\": 56, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Insulin modulates metabolism of β-amyloid precursor protein (APP) in neurons, decreasing the intracellular accumulation of β-amyloid (Abeta) peptides, which are pivotal in AD pathogenesis. The present study investigates whether the widely prescribed insulin-sensitizing drug, metformin (GlucophageR), affects APP metabolism and Abeta generation in various cell models. We demonstrate that metformin, at doses that lead to activation of the AMP-activated protein kinase (AMPK), significantly increases the generation of both intracellular and extracellular Abeta species\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"19237574\", \"authors\": [\"Chen Y\", \"Zhou K\", \"Wang R\", \"Liu Y\", \"Kwak YD\", \"Ma T\", \"Thompson RC\", \"Zhao Y\", \"Smith L\", \"Gasparini L\", \"Luo Z\", \"Xu H\", \"Liao FF\"], \"date\": \"2009-03-10\"}, \"line\": 7645, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 56, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Insulin and insulin receptors were shown to decrease in a normal brain with aging, but increase in AD brains [29]. Several basic science studies have explored and shown the relationship between the increased insulin and AD pathology in the aspects other than Abeta degradation alone. For example, insulin increases the secretion of Abeta into extracellular space [31], stimulates tau phosphorylation to form neurofibrillary tangles, and impairs insulin signal transduction [32] and [40] (reviewed by Gasparini and Hoyer). Insulin also affected APP processing in vivo, a critical molecular step in generating Abeta, to secrete sAPP [13], [16] and [81]. In addition, Abeta reduces insulin binding to insulin receptors [95].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"16399206\", \"authors\": [\"Qiu WQ\", \"Folstein MF\"], \"date\": \"2006-02-01\"}, \"line\": 8675, \"object\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"GOCC\", \"name\": \"intracellular\"}, \"toLoc\": {\"namespace\": \"GOCC\", \"name\": \"extracellular space\"}}}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Insulin signal transduction\"}, \"source\": 56, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Insulin and insulin receptors were shown to decrease in a normal brain with aging, but increase in AD brains [29]. Several basic science studies have explored and shown the relationship between the increased insulin and AD pathology in the aspects other than Abeta degradation alone. For example, insulin increases the secretion of Abeta into extracellular space [31], stimulates tau phosphorylation to form neurofibrillary tangles, and impairs insulin signal transduction [32] and [40] (reviewed by Gasparini and Hoyer). Insulin also affected APP processing in vivo, a critical molecular step in generating Abeta, to secrete sAPP [13], [16] and [81]. In addition, Abeta reduces insulin binding to insulin receptors [95].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"16399206\", \"authors\": [\"Qiu WQ\", \"Folstein MF\"], \"date\": \"2006-02-01\"}, \"line\": 8675, \"object\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"GOCC\", \"name\": \"intracellular\"}, \"toLoc\": {\"namespace\": \"GOCC\", \"name\": \"extracellular space\"}}}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 56, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular medicine\", \"reference\": \"22367557\", \"authors\": [\"Yan LJ\", \"Quazi SH\", \"Cai Z\", \"Zhao B\", \"Li K\"], \"date\": \"2012-03-01\"}, \"line\": 3133, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Akt subgraph\"}, \"source\": 57, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular medicine\", \"reference\": \"22367557\", \"authors\": [\"Yan LJ\", \"Quazi SH\", \"Cai Z\", \"Zhao B\", \"Li K\"], \"date\": \"2012-03-01\"}, \"line\": 3133, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 57, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular medicine\", \"reference\": \"22367557\", \"authors\": [\"Yan LJ\", \"Quazi SH\", \"Cai Z\", \"Zhao B\", \"Li K\"], \"date\": \"2012-03-01\"}, \"line\": 3133, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 57, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by Abeta (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"21994399\", \"authors\": [\"Chan C\", \"Geekiyanage H\"], \"date\": \"2011-10-12\"}, \"line\": 2719, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 58, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by Abeta (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"21994399\", \"authors\": [\"Chan C\", \"Geekiyanage H\"], \"date\": \"2011-10-12\"}, \"line\": 2719, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 58, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The phosphotyrosine binding domain of the neuronal protein X11alpha/mint-1 binds to the C-terminus of amyloid precursor protein (APP) and inhibits catabolism to beta-amyloid (Abeta), but the mechanism of this effect is unclear.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"14756819\", \"authors\": [\"King GD\", \"Cherian K\", \"Turner RS\"], \"date\": \"2004-02-01\"}, \"line\": 32275, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 59, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The phosphotyrosine binding domain of the neuronal protein X11alpha/mint-1 binds to the C-terminus of amyloid precursor protein (APP) and inhibits catabolism to beta-amyloid (Abeta), but the mechanism of this effect is unclear.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"14756819\", \"authors\": [\"King GD\", \"Cherian K\", \"Turner RS\"], \"date\": \"2004-02-01\"}, \"line\": 32275, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 59, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The neuronal adaptor X11alpha interacts with the conserved -GYENPTY- sequence in the C-terminus of amyloid precursor protein (APP) or its Swedish mutation (APPswe) to inhibit Abeta40 and Abeta42 secretion.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience\", \"reference\": \"12849748\", \"authors\": [\"King GD\", \"Perez RG\", \"Steinhilb ML\", \"Gaut JR\", \"Turner RS\"], \"date\": \"2003-01-01\"}, \"line\": 32291, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 59, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"On the contrary, X11 stabilizes ABeta¸PP conformation in membrane, inhibiting ABeta¸ secretion in cultured cells [45], likely impairing ABeta¸PP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptosis. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABeta¸PP. The expression of JIP1b stabilizes immature ABeta¸PP and decreases the ABeta¸PP ectodomain, ABeta¸40/ÃƒÅ¸42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 36362, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 59, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"On the contrary, X11 stabilizes ABeta¸PP conformation in membrane, inhibiting ABeta¸ secretion in cultured cells [45], likely impairing ABeta¸PP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptosis. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABeta¸PP. The expression of JIP1b stabilizes immature ABeta¸PP and decreases the ABeta¸PP ectodomain, ABeta¸40/ÃƒÅ¸42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 36362, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 59, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Both in vitro and in vivo, overexpression of the sortilin-related VPS10 domain containing receptor 1 (SORCS1) protein lowers amyloid-β generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Psychogeriatrics : the official journal of the Japanese Psychogeriatric Society\", \"reference\": \"23279143\", \"authors\": [\"Yu G\", \"Fang Z\", \"He Y\"], \"date\": \"2012-12-01\"}, \"line\": 11707, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 60, \"target\": 1, \"key\": 0}, {\"relation\": \"directlyDecreases\", \"evidence\": \"We previously reported that genetic variants in SORCS1 increase the risk of AD, that over-expression of SorCS1 reduces gamma-secretase activity and ABeta¸ levels, and that SorCS1 suppression increases gamma-secretase processing of APP and ABeta¸ levels. We now explored the effect of variation in SORCS1 on memory.Variation in intron 1 in SORCS1 is associated with memory changes in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"22046233\", \"authors\": [\"Reitz C\", \"Mayeux R\", \"Lee JH\", \"Rogers RS\"], \"date\": \"2011-01-01\"}, \"line\": 36593, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 60, \"target\": 4, \"key\": 0}, {\"relation\": \"directlyDecreases\", \"evidence\": \"We previously reported that genetic variants in SORCS1 increase the risk of AD, that over-expression of SorCS1 reduces gamma-secretase activity and ABeta¸ levels, and that SorCS1 suppression increases gamma-secretase processing of APP and ABeta¸ levels. We now explored the effect of variation in SORCS1 on memory.Variation in intron 1 in SORCS1 is associated with memory changes in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"22046233\", \"authors\": [\"Reitz C\", \"Mayeux R\", \"Lee JH\", \"Rogers RS\"], \"date\": \"2011-01-01\"}, \"line\": 36593, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 60, \"target\": 4, \"key\": 1}, {\"relation\": \"directlyDecreases\", \"evidence\": \"Sorting mechanisms that cause the amyloid precursor protein (APP) and the ÃƒÅ¸-secretases and gamma-secretases to colocalize in the same compartment play an important role in the regulation of ABeta¸ production in Alzheimer's disease (AD). We and others have reported that genetic variants in the Sortilin-related receptor (SORL1) increased the risk of AD, that SORL1 is involved in trafficking of APP, and that underexpression of SORL1 leads to overproduction of ABeta¸. Here we explored the role of one of its homologs, the sortilin-related VPS10 domain containing receptor 1 (SORCS1), in AD.SorCS1 influenced APP processing. While overexpression of SorCS1 reduced gamma-secretase activity and ABeta¸ levels, the suppression of SorCS1 increased gamma-secretase processing of APP and the levels of ABeta¸.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Annals of neurology\", \"reference\": \"21280075\", \"authors\": [\"Reitz C\", \"Tokuhiro S\", \"Clark LN\", \"Conrad C\", \"Vonsattel JP\", \"Hazrati LN\", \"Palotás A\", \"Lantigua R\", \"Medrano M\", \"Z Jiménez-Velázquez I\", \"Vardarajan B\", \"Simkin I\", \"Haines JL\", \"Pericak-Vance MA\", \"Farrer LA\", \"Lee JH\", \"Rogaeva E\", \"George-Hyslop PS\", \"Mayeux R\"], \"date\": \"2011-01-01\"}, \"line\": 34515, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 60, \"target\": 4, \"key\": 2}, {\"relation\": \"directlyDecreases\", \"evidence\": \"Sorting mechanisms that cause the amyloid precursor protein (APP) and the ÃƒÅ¸-secretases and gamma-secretases to colocalize in the same compartment play an important role in the regulation of ABeta¸ production in Alzheimer's disease (AD). We and others have reported that genetic variants in the Sortilin-related receptor (SORL1) increased the risk of AD, that SORL1 is involved in trafficking of APP, and that underexpression of SORL1 leads to overproduction of ABeta¸. Here we explored the role of one of its homologs, the sortilin-related VPS10 domain containing receptor 1 (SORCS1), in AD.SorCS1 influenced APP processing. While overexpression of SorCS1 reduced gamma-secretase activity and ABeta¸ levels, the suppression of SorCS1 increased gamma-secretase processing of APP and the levels of ABeta¸.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Annals of neurology\", \"reference\": \"21280075\", \"authors\": [\"Reitz C\", \"Tokuhiro S\", \"Clark LN\", \"Conrad C\", \"Vonsattel JP\", \"Hazrati LN\", \"Palotás A\", \"Lantigua R\", \"Medrano M\", \"Z Jiménez-Velázquez I\", \"Vardarajan B\", \"Simkin I\", \"Haines JL\", \"Pericak-Vance MA\", \"Farrer LA\", \"Lee JH\", \"Rogaeva E\", \"George-Hyslop PS\", \"Mayeux R\"], \"date\": \"2011-01-01\"}, \"line\": 34515, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 60, \"target\": 4, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"PEN-2 is an integral membrane protein that is a necessary component of the gamma-secretase complex, which is central in the pathogenesis of Alzheimer's disease and is also required for Notch signaling. In the absence of PEN-2, Notch signaling fails to guide normal development in Caenorhabditis elegans, and amyloid beta peptide is not generated from the amyloid precursor protein\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12639958\", \"authors\": [\"Crystal AS\", \"Morais VA\", \"Pierson TC\", \"Pijak DS\", \"Carlin D\", \"Lee VM\", \"Doms RW\"], \"date\": \"2003-05-30\"}, \"line\": 12030, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Species\": \"6239\", \"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 61, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"PEN-2 is an integral membrane protein that is a necessary component of the gamma-secretase complex, which is central in the pathogenesis of Alzheimer's disease and is also required for Notch signaling. In the absence of PEN-2, Notch signaling fails to guide normal development in Caenorhabditis elegans, and amyloid beta peptide is not generated from the amyloid precursor protein\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12639958\", \"authors\": [\"Crystal AS\", \"Morais VA\", \"Pierson TC\", \"Pijak DS\", \"Carlin D\", \"Lee VM\", \"Doms RW\"], \"date\": \"2003-05-30\"}, \"line\": 12030, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Species\": \"6239\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 61, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"PEN-2 is an integral membrane protein that is a necessary component of the gamma-secretase complex, which is central in the pathogenesis of Alzheimer's disease and is also required for Notch signaling. In the absence of PEN-2, Notch signaling fails to guide normal development in Caenorhabditis elegans, and amyloid beta peptide is not generated from the amyloid precursor protein\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12639958\", \"authors\": [\"Crystal AS\", \"Morais VA\", \"Pierson TC\", \"Pijak DS\", \"Carlin D\", \"Lee VM\", \"Doms RW\"], \"date\": \"2003-05-30\"}, \"line\": 12030, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Species\": \"6239\", \"Subgraph\": \"Notch signaling subgraph\"}, \"source\": 61, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that -amyloid accumulation is the critical event in development of disease (I 49, I 50]. Lots of research has been done on the formation and accumulation of Al3, however, in the last years the mechanism' of amyloid clearance came into focus. For Al3 clearance several mechanisms are known (Fig. 2): i) EnzyÂ­ matic degradation by activated microglia or by insulin deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and angiotensin converting enzyme (ACE); ii) Receptor-mediated transport across the blood brain barrier (BBB) by binding to the low-density lipoprotein receptorÂ­ related protein (LRP) either directly or after binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M) to be delivered to peripheral sites of degradation, e.g., liver and kidney. (Review in [151 ]). Concerning insulin resistance it has been shown that IDE expression is stimulated by the IR/lGF-1 R cascade (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19519303\", \"authors\": [\"Freude S\", \"Schilbach K\", \"Schubert M\"], \"date\": \"2009-06-01\"}, \"line\": 8592, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 62, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that -amyloid accumulation is the critical event in development of disease (I 49, I 50]. Lots of research has been done on the formation and accumulation of Al3, however, in the last years the mechanism' of amyloid clearance came into focus. For Al3 clearance several mechanisms are known (Fig. 2): i) EnzyÂ­ matic degradation by activated microglia or by insulin deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and angiotensin converting enzyme (ACE); ii) Receptor-mediated transport across the blood brain barrier (BBB) by binding to the low-density lipoprotein receptorÂ­ related protein (LRP) either directly or after binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M) to be delivered to peripheral sites of degradation, e.g., liver and kidney. (Review in [151 ]). Concerning insulin resistance it has been shown that IDE expression is stimulated by the IR/lGF-1 R cascade (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19519303\", \"authors\": [\"Freude S\", \"Schilbach K\", \"Schubert M\"], \"date\": \"2009-06-01\"}, \"line\": 8592, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 62, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that -amyloid accumulation is the critical event in development of disease (I 49, I 50]. Lots of research has been done on the formation and accumulation of Al3, however, in the last years the mechanism' of amyloid clearance came into focus. For Al3 clearance several mechanisms are known (Fig. 2): i) EnzyÂ­ matic degradation by activated microglia or by insulin deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and angiotensin converting enzyme (ACE); ii) Receptor-mediated transport across the blood brain barrier (BBB) by binding to the low-density lipoprotein receptorÂ­ related protein (LRP) either directly or after binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M) to be delivered to peripheral sites of degradation, e.g., liver and kidney. (Review in [151 ]). Concerning insulin resistance it has been shown that IDE expression is stimulated by the IR/lGF-1 R cascade (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19519303\", \"authors\": [\"Freude S\", \"Schilbach K\", \"Schubert M\"], \"date\": \"2009-06-01\"}, \"line\": 8592, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Endothelin subgraph\"}, \"source\": 62, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Since Abeta is secreted extracellularly, and deposits outside the neuronal cells in the AD brain, we took an approach to screen any secreted protease(s) in neuronal and non-neuronal cell culture media for the ability to degrade Abeta. Among all secreted proteases from the cells, only IDE degraded Abeta. We found that under physiological conditions IDE is secreted at high levels from the microglial cells, and degrades Abeta extracellularly [73]. Purified IDE from rat liver and brain was shown to degrade Abeta effectively. IDE is present in the soluble fractions from human brains, and binds and degrades Abeta specifically [57] and [71]. Primary cultured neurons were also shown to clear Abeta via extracellular IDE as well as IDE on the cell surface [90]. IDE from brain homogenates degrades different forms of Abeta: Abeta40, Abeta42 and an Abeta mutant in one type of AD (Dutch Variant 1-40 Q) [61] and [71]. Abeta42 is the longer form of Abeta and more abundant in the AD brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"16399206\", \"authors\": [\"Qiu WQ\", \"Folstein MF\"], \"date\": \"2006-02-01\"}, \"line\": 8635, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 62, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Since Abeta is secreted extracellularly, and deposits outside the neuronal cells in the AD brain, we took an approach to screen any secreted protease(s) in neuronal and non-neuronal cell culture media for the ability to degrade Abeta. Among all secreted proteases from the cells, only IDE degraded Abeta. We found that under physiological conditions IDE is secreted at high levels from the microglial cells, and degrades Abeta extracellularly [73]. Purified IDE from rat liver and brain was shown to degrade Abeta effectively. IDE is present in the soluble fractions from human brains, and binds and degrades Abeta specifically [57] and [71]. Primary cultured neurons were also shown to clear Abeta via extracellular IDE as well as IDE on the cell surface [90]. IDE from brain homogenates degrades different forms of Abeta: Abeta40, Abeta42 and an Abeta mutant in one type of AD (Dutch Variant 1-40 Q) [61] and [71]. Abeta42 is the longer form of Abeta and more abundant in the AD brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"16399206\", \"authors\": [\"Qiu WQ\", \"Folstein MF\"], \"date\": \"2006-02-01\"}, \"line\": 8635, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 62, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that NOS2 upregulation impairs amyloid β degradation through negative regulation of IDE activity and thus loss of NOS2 activity will positively influence amyloid β clearance.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroimmune pharmacology : the official journal of the Society on NeuroImmune Pharmacology\", \"reference\": \"22227962\", \"authors\": [\"Heneka MT\", \"Axt D\", \"Hermes M\", \"Kummer MP\", \"Hülsmann C\"], \"date\": \"2012-03-01\"}, \"line\": 21607, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 62, \"target\": 1, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that NOS2 upregulation impairs amyloid β degradation through negative regulation of IDE activity and thus loss of NOS2 activity will positively influence amyloid β clearance.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroimmune pharmacology : the official journal of the Society on NeuroImmune Pharmacology\", \"reference\": \"22227962\", \"authors\": [\"Heneka MT\", \"Axt D\", \"Hermes M\", \"Kummer MP\", \"Hülsmann C\"], \"date\": \"2012-03-01\"}, \"line\": 21607, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 62, \"target\": 1, \"key\": 6}, {\"relation\": \"directlyDecreases\", \"evidence\": \"Lower-expression of PS1 and over-expression of IDE or NEP may be helpful in potentially lowering brain Abeta levels in subjects with AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19355846\", \"authors\": [\"Tian J\", \"Shi J\", \"Zhang L\", \"Yin J\", \"Hu Q\", \"Xu Y\", \"Sheng S\", \"Wang P\", \"Ren Y\", \"Wang R\", \"Wang Y\"], \"date\": \"2009-04-01\"}, \"line\": 36516, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 62, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that NOS2 upregulation impairs amyloid beta degradation through negative regulation of IDE activity and thus loss of NOS2 activity will positively influence amyloid beta clearance.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroimmune pharmacology : the official journal of the Society on NeuroImmune Pharmacology\", \"reference\": \"22227962\", \"authors\": [\"Heneka MT\", \"Axt D\", \"Hermes M\", \"Kummer MP\", \"Hülsmann C\"], \"date\": \"2012-03-01\"}, \"line\": 37111, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 62, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3525, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 63, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3525, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 63, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3528, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 63, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3528, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 63, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17428983\", \"authors\": [\"He X\", \"Cooley K\", \"Chung CH\", \"Dashti N\", \"Tang J\"], \"date\": \"2007-04-11\"}, \"line\": 4780, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 64, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17428983\", \"authors\": [\"He X\", \"Cooley K\", \"Chung CH\", \"Dashti N\", \"Tang J\"], \"date\": \"2007-04-11\"}, \"line\": 4780, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 64, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17428983\", \"authors\": [\"He X\", \"Cooley K\", \"Chung CH\", \"Dashti N\", \"Tang J\"], \"date\": \"2007-04-11\"}, \"line\": 4780, \"subject\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Cell Surface Extensions\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Endosomes\"}}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 64, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that BACE2 contributes to Abeta production in individuals bearing the Flemish mutation, and that selective inhibition of these highly similar proteases may be feasible and therapeutically advantageous.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"10931940\", \"authors\": [\"Farzan M\", \"Schnitzler CE\", \"Vasilieva N\", \"Leung D\", \"Choe H\"], \"date\": \"2000-08-15\"}, \"line\": 25173, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 64, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that BACE2 contributes to Abeta production in individuals bearing the Flemish mutation, and that selective inhibition of these highly similar proteases may be feasible and therapeutically advantageous.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"10931940\", \"authors\": [\"Farzan M\", \"Schnitzler CE\", \"Vasilieva N\", \"Leung D\", \"Choe H\"], \"date\": \"2000-08-15\"}, \"line\": 25173, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 64, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"BACE2 cleaves the amyloid precursor protein at the beta-secretase site and is thought to contribute to amyloid beta protein production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"12052539\", \"authors\": [\"Motonaga K\", \"Itoh M\", \"Becker LE\", \"Takashima S\", \"Goto Y\"], \"date\": \"2002-06-21\"}, \"line\": 25235, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 64, \"target\": 1, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"BACE2 cleaves the amyloid precursor protein at the beta-secretase site and is thought to contribute to amyloid beta protein production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"12052539\", \"authors\": [\"Motonaga K\", \"Itoh M\", \"Becker LE\", \"Takashima S\", \"Goto Y\"], \"date\": \"2002-06-21\"}, \"line\": 25235, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 64, \"target\": 1, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"Mapping to the Down syndrome critical region (chromosome 21) and identified as a homologue of BACE1, BACE2 also cleaves amyloid precursor protein at the beta-site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of proteome research\", \"reference\": \"15473697\", \"authors\": [\"Chou KC\"], \"date\": \"2004-09-01\"}, \"line\": 26392, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 64, \"target\": 1, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"Mapping to the Down syndrome critical region (chromosome 21) and identified as a homologue of BACE1, BACE2 also cleaves amyloid precursor protein at the beta-site.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of proteome research\", \"reference\": \"15473697\", \"authors\": [\"Chou KC\"], \"date\": \"2004-09-01\"}, \"line\": 26392, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 64, \"target\": 1, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \"Physiological levels of ABeta¸ also have trophic and neuroprotective actions in trophic deprived conditions [95]. Many ABeta¸ has a physiological role in normal synapse function. In organotypic hippocampal slices, BACE activity is increased by synaptic activity and the resulting ABeta¸ peptides depress excitatory transmission through a-amino-3-hydroxyl-5-methyl-4-isoxazole-propionate (AMPA) and N-methyl-D-aspartic acid (NMDA) receptors, suggesting a role for ABeta¸ in homeostatic plasticity [29]. ABeta¸ may have an important physiological role in synapse elimination during brain development\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34929, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 64, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Physiological levels of ABeta¸ also have trophic and neuroprotective actions in trophic deprived conditions [95]. Many ABeta¸ has a physiological role in normal synapse function. In organotypic hippocampal slices, BACE activity is increased by synaptic activity and the resulting ABeta¸ peptides depress excitatory transmission through a-amino-3-hydroxyl-5-methyl-4-isoxazole-propionate (AMPA) and N-methyl-D-aspartic acid (NMDA) receptors, suggesting a role for ABeta¸ in homeostatic plasticity [29]. ABeta¸ may have an important physiological role in synapse elimination during brain development\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34929, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 64, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Insulin and IGF-I may pmodulate brain levels of insulin degrading enzyme, which would also lead to an accumulation of Abeta amyloid.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neural plasticity\", \"reference\": \"16444902\", \"authors\": [\"Messier C\", \"Teutenberg K\"], \"date\": \"2005-01-01\"}, \"line\": 31531, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 65, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Insulin and IGF-I may pmodulate brain levels of insulin degrading enzyme, which would also lead to an accumulation of Abeta amyloid.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neural plasticity\", \"reference\": \"16444902\", \"authors\": [\"Messier C\", \"Teutenberg K\"], \"date\": \"2005-01-01\"}, \"line\": 31531, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 65, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In addition, NF-ÎºB p65 expression leads to up-regulated β-secretase cleavage and Abeta production, while non-steroidal anti-inflammatory drugs (NSAIDs) inhibited BACE1 transcriptional activation induced by strong NF-ÎºB activator tumour necrosis factor-alpha (TNF-Î±).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The international journal of neuropsychopharmacology\", \"reference\": \"21329555\", \"authors\": [\"Chen CH\", \"Zhou W\", \"Liu S\", \"Deng Y\", \"Cai F\", \"Tone M\", \"Tone Y\", \"Tong Y\", \"Song W\"], \"date\": \"2012-02-01\"}, \"line\": 14969, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 66, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In addition, NF-ÎºB p65 expression leads to up-regulated β-secretase cleavage and Abeta production, while non-steroidal anti-inflammatory drugs (NSAIDs) inhibited BACE1 transcriptional activation induced by strong NF-ÎºB activator tumour necrosis factor-alpha (TNF-Î±).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The international journal of neuropsychopharmacology\", \"reference\": \"21329555\", \"authors\": [\"Chen CH\", \"Zhou W\", \"Liu S\", \"Deng Y\", \"Cai F\", \"Tone M\", \"Tone Y\", \"Tong Y\", \"Song W\"], \"date\": \"2012-02-01\"}, \"line\": 14969, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Nuclear factor Kappa beta subgraph\"}, \"source\": 66, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that overexpression of endogenous calpain inhibitor calpastatin (CAST) under the control of the calcium/calpmodulin-dependent protein kinase II promoter in APP/PS1 mice caused a remarkable decrease of amyloid plaque burdens and prevented Tau phosphorylation and the loss of synapses.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20595388\", \"authors\": [\"Liang B\", \"Duan BY\", \"Zhou XP\", \"Gong JX\", \"Luo ZG\"], \"date\": \"2010-09-03\"}, \"line\": 25957, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 67, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"We found that overexpression of endogenous calpain inhibitor calpastatin (CAST) under the control of the calcium/calpmodulin-dependent protein kinase II promoter in APP/PS1 mice caused a remarkable decrease of amyloid plaque burdens and prevented Tau phosphorylation and the loss of synapses.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20595388\", \"authors\": [\"Liang B\", \"Duan BY\", \"Zhou XP\", \"Gong JX\", \"Luo ZG\"], \"date\": \"2010-09-03\"}, \"line\": 25957, \"annotations\": {\"Subgraph\": \"Calpastatin-calpain subgraph\"}, \"source\": 67, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20595388\", \"authors\": [\"Liang B\", \"Duan BY\", \"Zhou XP\", \"Gong JX\", \"Luo ZG\"], \"date\": \"2010-09-03\"}, \"line\": 25980, \"annotations\": {\"Subgraph\": \"Calpastatin-calpain subgraph\"}, \"source\": 67, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20595388\", \"authors\": [\"Liang B\", \"Duan BY\", \"Zhou XP\", \"Gong JX\", \"Luo ZG\"], \"date\": \"2010-09-03\"}, \"line\": 25980, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 67, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20595388\", \"authors\": [\"Liang B\", \"Duan BY\", \"Zhou XP\", \"Gong JX\", \"Luo ZG\"], \"date\": \"2010-09-03\"}, \"line\": 25980, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 67, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20595388\", \"authors\": [\"Liang B\", \"Duan BY\", \"Zhou XP\", \"Gong JX\", \"Luo ZG\"], \"date\": \"2010-09-03\"}, \"line\": 25982, \"annotations\": {\"Subgraph\": \"Calpastatin-calpain subgraph\"}, \"source\": 67, \"target\": 1, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20595388\", \"authors\": [\"Liang B\", \"Duan BY\", \"Zhou XP\", \"Gong JX\", \"Luo ZG\"], \"date\": \"2010-09-03\"}, \"line\": 25982, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 67, \"target\": 1, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20595388\", \"authors\": [\"Liang B\", \"Duan BY\", \"Zhou XP\", \"Gong JX\", \"Luo ZG\"], \"date\": \"2010-09-03\"}, \"line\": 25982, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 67, \"target\": 1, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"In addition, we show that some NF-kappaB inhibitors decrease sAPPbeta and APP-CTFbeta suggesting that they reduce the beta-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"17223266\", \"authors\": [\"Paris D\", \"Patel N\", \"Quadros A\", \"Linan M\", \"Bakshi P\", \"Ait-Ghezala G\", \"Mullan M\"], \"date\": \"2007-03-19\"}, \"line\": 26574, \"annotations\": {\"Subgraph\": \"Nuclear factor Kappa beta subgraph\"}, \"source\": 68, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In addition, we show that some NF-kappaB inhibitors decrease sAPPbeta and APP-CTFbeta suggesting that they reduce the beta-secretase cleavage of APP.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"17223266\", \"authors\": [\"Paris D\", \"Patel N\", \"Quadros A\", \"Linan M\", \"Bakshi P\", \"Ait-Ghezala G\", \"Mullan M\"], \"date\": \"2007-03-19\"}, \"line\": 26574, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 68, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Under physiological conditions, NFkappaB lowers the transcriptional activity of the promoters of betaAPP, beta-secretase (beta-site APP-cleaving enzyme 1, BACE1), and of the four protein components (Aph-1, Pen-2, nicastrin, presenilin-1, or presenilin-2) of the γ-secretase in HEK293 cells. This was accompanied by a reduction of both protein levels and enzymatic activities, thereby ultimately yielding lower amounts of Abeta and AICD (APP intracellular domain).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22654105\", \"authors\": [\"Peyron JF\", \"Checler F\", \"Buggia-Prévot V\", \"Chami L\", \"Duplan E\", \"Del Prete D\", \"Chami M\"], \"date\": \"2012-07-13\"}, \"line\": 36484, \"annotations\": {\"Subgraph\": \"Nuclear factor Kappa beta subgraph\"}, \"source\": 68, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Under physiological conditions, NFkappaB lowers the transcriptional activity of the promoters of betaAPP, beta-secretase (beta-site APP-cleaving enzyme 1, BACE1), and of the four protein components (Aph-1, Pen-2, nicastrin, presenilin-1, or presenilin-2) of the γ-secretase in HEK293 cells. This was accompanied by a reduction of both protein levels and enzymatic activities, thereby ultimately yielding lower amounts of Abeta and AICD (APP intracellular domain).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22654105\", \"authors\": [\"Peyron JF\", \"Checler F\", \"Buggia-Prévot V\", \"Chami L\", \"Duplan E\", \"Del Prete D\", \"Chami M\"], \"date\": \"2012-07-13\"}, \"line\": 36484, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 68, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Although the mechanism of Ab action in the pathogenesis of Alzheimer’s disease (AD) has remained elusive, it is known to increase the expression of the antagonist of canonical wnt signalling, Dickkopf-1 (Dkk1), whereas the silencing of Dkk1 blocks Ab neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"23164821\", \"authors\": [\"To AW\", \"Brion JP\", \"Reynolds CH\", \"Leroy K\", \"Noble W\", \"Robinson M\", \"Williamson R\", \"Hooper C\", \"Causevic M\", \"Breen G\", \"Lin K\", \"Ribe EM\", \"Wroe R\", \"Malik B\", \"Kellie S\", \"Lovestone S\", \"Usardi A\", \"Stephenson J\", \"Killick R\", \"Hodges A\", \"Lunnon K\", \"Bazenet C\", \"Simons JP\", \"Furney S\", \"Nolan PM\", \"Fernandes C\", \"Dobson R\", \"Lourdusamy A\", \"Al-Shawi R\"], \"date\": \"2014-01-01\"}, \"line\": 54917, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Confidence\": \"High\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 69, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Although the mechanism of Ab action in the pathogenesis of Alzheimer’s disease (AD) has remained elusive, it is known to increase the expression of the antagonist of canonical wnt signalling, Dickkopf-1 (Dkk1), whereas the silencing of Dkk1 blocks Ab neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"23164821\", \"authors\": [\"To AW\", \"Brion JP\", \"Reynolds CH\", \"Leroy K\", \"Noble W\", \"Robinson M\", \"Williamson R\", \"Hooper C\", \"Causevic M\", \"Breen G\", \"Lin K\", \"Ribe EM\", \"Wroe R\", \"Malik B\", \"Kellie S\", \"Lovestone S\", \"Usardi A\", \"Stephenson J\", \"Killick R\", \"Hodges A\", \"Lunnon K\", \"Bazenet C\", \"Simons JP\", \"Furney S\", \"Nolan PM\", \"Fernandes C\", \"Dobson R\", \"Lourdusamy A\", \"Al-Shawi R\"], \"date\": \"2014-01-01\"}, \"line\": 54917, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Confidence\": \"High\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"DKK1 subgraph\"}, \"source\": 69, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Caricasole et al demonstrated that the Aβ peptide fragment, Aβ25-35, induces neuronal expression of the wnt antagonist Dkk1 and that silencing of DKK1 blocks Aβ neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"23164821\", \"authors\": [\"To AW\", \"Brion JP\", \"Reynolds CH\", \"Leroy K\", \"Noble W\", \"Robinson M\", \"Williamson R\", \"Hooper C\", \"Causevic M\", \"Breen G\", \"Lin K\", \"Ribe EM\", \"Wroe R\", \"Malik B\", \"Kellie S\", \"Lovestone S\", \"Usardi A\", \"Stephenson J\", \"Killick R\", \"Hodges A\", \"Lunnon K\", \"Bazenet C\", \"Simons JP\", \"Furney S\", \"Nolan PM\", \"Fernandes C\", \"Dobson R\", \"Lourdusamy A\", \"Al-Shawi R\"], \"date\": \"2014-01-01\"}, \"line\": 54930, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Confidence\": \"High\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 69, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Caricasole et al demonstrated that the Aβ peptide fragment, Aβ25-35, induces neuronal expression of the wnt antagonist Dkk1 and that silencing of DKK1 blocks Aβ neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"23164821\", \"authors\": [\"To AW\", \"Brion JP\", \"Reynolds CH\", \"Leroy K\", \"Noble W\", \"Robinson M\", \"Williamson R\", \"Hooper C\", \"Causevic M\", \"Breen G\", \"Lin K\", \"Ribe EM\", \"Wroe R\", \"Malik B\", \"Kellie S\", \"Lovestone S\", \"Usardi A\", \"Stephenson J\", \"Killick R\", \"Hodges A\", \"Lunnon K\", \"Bazenet C\", \"Simons JP\", \"Furney S\", \"Nolan PM\", \"Fernandes C\", \"Dobson R\", \"Lourdusamy A\", \"Al-Shawi R\"], \"date\": \"2014-01-01\"}, \"line\": 54930, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Confidence\": \"High\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"DKK1 subgraph\"}, \"source\": 69, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In this study, activation of the Wnt pathway by overexpression of the agonist Wnt3a or β-catenin or by inhibition of glycogen kinase synthase-3 in N2a cells resulted in a reduction in Aβ levels and in the activity and expression of BACE1 (β-APP cleaving enzyme). Conversely, inhibition of the pathway by transfection of the antagonists secreted frizzled receptor protein-1 or dickkopf-1 produced the opposite effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"25384422\", \"authors\": [\"Sastre M\", \"Mirzaei N\", \"Parr C\", \"Christian M\"], \"date\": \"2015-02-01\"}, \"line\": 55212, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 69, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"In this study, activation of the Wnt pathway by overexpression of the agonist Wnt3a or β-catenin or by inhibition of glycogen kinase synthase-3 in N2a cells resulted in a reduction in Aβ levels and in the activity and expression of BACE1 (β-APP cleaving enzyme). Conversely, inhibition of the pathway by transfection of the antagonists secreted frizzled receptor protein-1 or dickkopf-1 produced the opposite effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"25384422\", \"authors\": [\"Sastre M\", \"Mirzaei N\", \"Parr C\", \"Christian M\"], \"date\": \"2015-02-01\"}, \"line\": 55212, \"annotations\": {\"Subgraph\": \"DKK1 subgraph\"}, \"source\": 69, \"target\": 1, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"More importantly, persistent activation of Wnt signaling through Wnt ligands, or inhibition of negative regulators of Wnt signaling, such as Dickkopf-1 (DKK-1) and glycogen synthase kinase-3 β (GSK-3 β ) that are hyperactive in the disease state, is able to protect against A β toxicity and ameliorate cognitive performance in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24883305\", \"authors\": [\"Kalionis B\", \"Xia S\", \"Wan W\", \"Li Y\", \"Liu L\"], \"date\": \"2014-01-01\"}, \"line\": 55251, \"annotations\": {\"Subgraph\": \"DKK1 subgraph\"}, \"source\": 69, \"target\": 1, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \"Seladin-1 was considered a novel neuroprotective factor, because of its anti-apoptotic activity. Subsequently, it was demonstrated that seladin-1 has also enzymatic activity [3-β-hydroxysterol delta-24-reductase, (DHCR24)], which catalyzes the synthesis of cholesterol from desmosterol. The amount of membrane cholesterol may play an important role both in protecting neuronal cells against toxic insults and in inhibiting the production of β-amyloid. We demonstrated that seladin-1 overexpression increases the amount of membrane cholesterol and induces resistance against β-amyloid aggregates in neuroblastoma cells, whereas a specific inhibitor of DHCR24 increased cell vulnerability. We also hypothesized that seladin-1 might be a mediator of the neuroprotective effects of estrogens. We first demonstrated that, in human fetal neuroepithelial cells (FNC), 17β-estradiol, raloxifene, and tamoxifen exert protective effects against β-amyloid toxicity and oxidative stress. In addition, these molecules significantly increased the expression of seladin-1 and the amount of cell cholesterol. Then, we showed that, upon seladin-1 silencing, the protective effects of estrogens were abolished, thus indicating this factor as a fundamental mediator of estrogen-mediated neuroprotection, at least in FNC cells. Furthermore, we detected the presence of functionally active half-palindromic estrogen responsive elements upstream the coding region of the seladin-1 gene. Overall, our results indicate that seladin-1 may be viewed as a multi-faceted protein, which conjugates both the neuroprotective properties of estrogens and the important functions of cholesterol in maintaining brain homeostasis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience\", \"reference\": \"21396986\", \"authors\": [\"Peri A\", \"Benvenuti S\", \"Luciani P\", \"Deledda C\", \"Cellai I\"], \"date\": \"2011-09-15\"}, \"line\": 12148, \"annotations\": {\"UserdefinedCellLine\": \"Neuroblastoma cell\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 70, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Seladin-1 was considered a novel neuroprotective factor, because of its anti-apoptotic activity. Subsequently, it was demonstrated that seladin-1 has also enzymatic activity [3-β-hydroxysterol delta-24-reductase, (DHCR24)], which catalyzes the synthesis of cholesterol from desmosterol. The amount of membrane cholesterol may play an important role both in protecting neuronal cells against toxic insults and in inhibiting the production of β-amyloid. We demonstrated that seladin-1 overexpression increases the amount of membrane cholesterol and induces resistance against β-amyloid aggregates in neuroblastoma cells, whereas a specific inhibitor of DHCR24 increased cell vulnerability. We also hypothesized that seladin-1 might be a mediator of the neuroprotective effects of estrogens. We first demonstrated that, in human fetal neuroepithelial cells (FNC), 17β-estradiol, raloxifene, and tamoxifen exert protective effects against β-amyloid toxicity and oxidative stress. In addition, these molecules significantly increased the expression of seladin-1 and the amount of cell cholesterol. Then, we showed that, upon seladin-1 silencing, the protective effects of estrogens were abolished, thus indicating this factor as a fundamental mediator of estrogen-mediated neuroprotection, at least in FNC cells. Furthermore, we detected the presence of functionally active half-palindromic estrogen responsive elements upstream the coding region of the seladin-1 gene. Overall, our results indicate that seladin-1 may be viewed as a multi-faceted protein, which conjugates both the neuroprotective properties of estrogens and the important functions of cholesterol in maintaining brain homeostasis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience\", \"reference\": \"21396986\", \"authors\": [\"Peri A\", \"Benvenuti S\", \"Luciani P\", \"Deledda C\", \"Cellai I\"], \"date\": \"2011-09-15\"}, \"line\": 12148, \"annotations\": {\"UserdefinedCellLine\": \"Neuroblastoma cell\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 70, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The role of miR-124 on the expression of Abeta-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the Abeta-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by Abeta neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Toxicology letters\", \"reference\": \"22178568\", \"authors\": [\"Fang M\", \"Wang J\", \"Han S\", \"Hu Z\", \"Zhang X\", \"Geng Y\", \"Rudd JA\", \"Ling S\", \"Chen W\"], \"date\": \"2012-02-25\"}, \"line\": 3187, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 71, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The role of miR-124 on the expression of Abeta-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the Abeta-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by Abeta neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Toxicology letters\", \"reference\": \"22178568\", \"authors\": [\"Fang M\", \"Wang J\", \"Han S\", \"Hu Z\", \"Zhang X\", \"Geng Y\", \"Rudd JA\", \"Ling S\", \"Chen W\"], \"date\": \"2012-02-25\"}, \"line\": 3187, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"miRNA subgraph\"}, \"source\": 71, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The role of miR-124 on the expression of β-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the β-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by Abeta neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Toxicology letters\", \"reference\": \"22178568\", \"authors\": [\"Fang M\", \"Wang J\", \"Han S\", \"Hu Z\", \"Zhang X\", \"Geng Y\", \"Rudd JA\", \"Ling S\", \"Chen W\"], \"date\": \"2012-02-25\"}, \"line\": 9633, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 71, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The role of miR-124 on the expression of β-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the β-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by Abeta neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Toxicology letters\", \"reference\": \"22178568\", \"authors\": [\"Fang M\", \"Wang J\", \"Han S\", \"Hu Z\", \"Zhang X\", \"Geng Y\", \"Rudd JA\", \"Ling S\", \"Chen W\"], \"date\": \"2012-02-25\"}, \"line\": 9633, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 71, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"The role of miR-124 on the expression of β-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the β-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by Abeta neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Toxicology letters\", \"reference\": \"22178568\", \"authors\": [\"Fang M\", \"Wang J\", \"Han S\", \"Hu Z\", \"Zhang X\", \"Geng Y\", \"Rudd JA\", \"Ling S\", \"Chen W\"], \"date\": \"2012-02-25\"}, \"line\": 9633, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 71, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Our results revealed that histone H3 acetylation in PS1 and BACE1 promoters is markedly increased in N2a/APPswe cells\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"25051175\", \"authors\": [\"Wang L\", \"Guo X\", \"Yu C\", \"Yu G\", \"Cao H\", \"Lu X\", \"Deng Y\", \"Yu D\", \"Zhang Y\", \"Liu L\"], \"date\": \"2014-01-01\"}, \"line\": 43812, \"annotations\": {\"Encode_Feature_Types\": \"Promoter\"}, \"source\": 71, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" IL-1 levels are elevated in Alzheimer brain, and overexpression of IL-1 is associated with beta-amyloid plaque progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"10850859\", \"authors\": [\"Mrak RE\", \"Griffin WS\"], \"date\": \"2000-06-01\"}, \"line\": 23162, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 72, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" IL-1 levels are elevated in Alzheimer brain, and overexpression of IL-1 is associated with beta-amyloid plaque progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"10850859\", \"authors\": [\"Mrak RE\", \"Griffin WS\"], \"date\": \"2000-06-01\"}, \"line\": 23162, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 72, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" IL-1 levels are elevated in Alzheimer brain, and overexpression of IL-1 is associated with beta-amyloid plaque progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"10850859\", \"authors\": [\"Mrak RE\", \"Griffin WS\"], \"date\": \"2000-06-01\"}, \"line\": 23444, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 72, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" IL-1 levels are elevated in Alzheimer brain, and overexpression of IL-1 is associated with beta-amyloid plaque progression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuropathology and experimental neurology\", \"reference\": \"10850859\", \"authors\": [\"Mrak RE\", \"Griffin WS\"], \"date\": \"2000-06-01\"}, \"line\": 23444, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 72, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18021299\", \"authors\": [\"Tachida Y\", \"Nakagawa K\", \"Saito T\", \"Saido TC\", \"Honda T\", \"Saito Y\", \"Murayama S\", \"Endo T\", \"Sakaguchi G\", \"Kato A\", \"Kitazume S\", \"Hashimoto Y\"], \"date\": \"2008-03-01\"}, \"line\": 23651, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 72, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18021299\", \"authors\": [\"Tachida Y\", \"Nakagawa K\", \"Saito T\", \"Saido TC\", \"Honda T\", \"Saito Y\", \"Murayama S\", \"Endo T\", \"Sakaguchi G\", \"Kato A\", \"Kitazume S\", \"Hashimoto Y\"], \"date\": \"2008-03-01\"}, \"line\": 23651, \"annotations\": {\"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 72, \"target\": 1, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18021299\", \"authors\": [\"Tachida Y\", \"Nakagawa K\", \"Saito T\", \"Saido TC\", \"Honda T\", \"Saito Y\", \"Murayama S\", \"Endo T\", \"Sakaguchi G\", \"Kato A\", \"Kitazume S\", \"Hashimoto Y\"], \"date\": \"2008-03-01\"}, \"line\": 23651, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 72, \"target\": 1, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18021299\", \"authors\": [\"Tachida Y\", \"Nakagawa K\", \"Saito T\", \"Saido TC\", \"Honda T\", \"Saito Y\", \"Murayama S\", \"Endo T\", \"Sakaguchi G\", \"Kato A\", \"Kitazume S\", \"Hashimoto Y\"], \"date\": \"2008-03-01\"}, \"line\": 24164, \"annotations\": {\"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 72, \"target\": 1, \"key\": 7}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18021299\", \"authors\": [\"Tachida Y\", \"Nakagawa K\", \"Saito T\", \"Saido TC\", \"Honda T\", \"Saito Y\", \"Murayama S\", \"Endo T\", \"Sakaguchi G\", \"Kato A\", \"Kitazume S\", \"Hashimoto Y\"], \"date\": \"2008-03-01\"}, \"line\": 24164, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 72, \"target\": 1, \"key\": 8}, {\"relation\": \"decreases\", \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18021299\", \"authors\": [\"Tachida Y\", \"Nakagawa K\", \"Saito T\", \"Saido TC\", \"Honda T\", \"Saito Y\", \"Murayama S\", \"Endo T\", \"Sakaguchi G\", \"Kato A\", \"Kitazume S\", \"Hashimoto Y\"], \"date\": \"2008-03-01\"}, \"line\": 24164, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 72, \"target\": 1, \"key\": 9}, {\"relation\": \"decreases\", \"evidence\": \"We show that knock-down of ATXN1 significantly increases the levels of both Abeta40 and Abeta42. This effect could be rescued with concurrent overexpression of ATXN1. Moreover, overexpression of ATXN1 decreased Abeta levels. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature reviews. Neurology\", \"reference\": \"20139999\", \"authors\": [\"De Strooper B\", \"Vassar R\", \"Golde T\"], \"date\": \"2010-02-01\"}, \"line\": 26809, \"annotations\": {\"Subgraph\": \"Akt subgraph\"}, \"source\": 73, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"We show that knock-down of ATXN1 significantly increases the levels of both Abeta40 and Abeta42. This effect could be rescued with concurrent overexpression of ATXN1. Moreover, overexpression of ATXN1 decreased Abeta levels. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature reviews. Neurology\", \"reference\": \"20139999\", \"authors\": [\"De Strooper B\", \"Vassar R\", \"Golde T\"], \"date\": \"2010-02-01\"}, \"line\": 26809, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 73, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by Abeta (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"21994399\", \"authors\": [\"Chan C\", \"Geekiyanage H\"], \"date\": \"2011-10-12\"}, \"line\": 2721, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 74, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by Abeta (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"21994399\", \"authors\": [\"Chan C\", \"Geekiyanage H\"], \"date\": \"2011-10-12\"}, \"line\": 2721, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 74, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 6393, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 75, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 6393, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 75, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Our laboratory has previously shown that EGCG can increase non-amyloidogenic processing of APP through promotion of the beta-secretase ADAM10, which consequently reduced Abeta deposition and improved cognition in AD mice\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS letters\", \"reference\": \"20849853\", \"authors\": [\"Fernandez JW\", \"Rezai-Zadeh K\", \"Obregon D\", \"Tan J\"], \"date\": \"2010-10-08\"}, \"line\": 24039, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 75, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Our laboratory has previously shown that EGCG can increase non-amyloidogenic processing of APP through promotion of the beta-secretase ADAM10, which consequently reduced Abeta deposition and improved cognition in AD mice\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS letters\", \"reference\": \"20849853\", \"authors\": [\"Fernandez JW\", \"Rezai-Zadeh K\", \"Obregon D\", \"Tan J\"], \"date\": \"2010-10-08\"}, \"line\": 24039, \"annotations\": {\"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 75, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"The γ-secretase complex comprises presenilins (PS1 or PS2), nicastrin, APH-1, and PEN-2. Herein, we find that PEN-2 can interact with ferritin light chain (FTL), an important component of the iron storage protein ferritin. In addition, we show that overexpression of FTL increases the protein levels of PEN-2 and PS1 amino-terminal fragment (NTF) and promotes γ-secretase activity for more production of Abeta and notch intracellular domain (NICD). Furthermore, iron treatments increase the levels of FTL, PEN-2 and PS1 NTF and promote γ-secretase-mediated NICD production. Moreover, downregulation of FTL decreases the levels of PEN-2 and PS1 NTF. Together, our results suggest that iron can increase γ-secretase activity through promoting the level of FTL that interacts with and stabilizes PEN-2, providing a new molecular link between iron, PEN-2/γ-secretase and Abeta generation in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"23685131\", \"authors\": [\"Li X\", \"Bu G\", \"Yao G\", \"Liu Y\", \"Zheng Q\", \"Zhang YW\", \"Xu H\", \"Cheng P\"], \"date\": \"2013-08-26\"}, \"line\": 12052, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 76, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The γ-secretase complex comprises presenilins (PS1 or PS2), nicastrin, APH-1, and PEN-2. Herein, we find that PEN-2 can interact with ferritin light chain (FTL), an important component of the iron storage protein ferritin. In addition, we show that overexpression of FTL increases the protein levels of PEN-2 and PS1 amino-terminal fragment (NTF) and promotes γ-secretase activity for more production of Abeta and notch intracellular domain (NICD). Furthermore, iron treatments increase the levels of FTL, PEN-2 and PS1 NTF and promote γ-secretase-mediated NICD production. Moreover, downregulation of FTL decreases the levels of PEN-2 and PS1 NTF. Together, our results suggest that iron can increase γ-secretase activity through promoting the level of FTL that interacts with and stabilizes PEN-2, providing a new molecular link between iron, PEN-2/γ-secretase and Abeta generation in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"23685131\", \"authors\": [\"Li X\", \"Bu G\", \"Yao G\", \"Liu Y\", \"Zheng Q\", \"Zhang YW\", \"Xu H\", \"Cheng P\"], \"date\": \"2013-08-26\"}, \"line\": 12052, \"annotations\": {\"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 76, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Activated gadd153 can generate oxidative damage and reactive oxygen species (ROS), increase β-amyloid (Abeta) levels, disturb iron homeostasis and induce inflammation as well as cell death, which are all pathological hallmarks of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"22046282\", \"authors\": [\"Ghribi O\", \"Larson T\", \"Schommer J\", \"Prasanthi JR\"], \"date\": \"2011-01-01\"}, \"line\": 19791, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"CellStructure\": \"Endoplasmic Reticulum\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 77, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Activated gadd153 can generate oxidative damage and reactive oxygen species (ROS), increase β-amyloid (Abeta) levels, disturb iron homeostasis and induce inflammation as well as cell death, which are all pathological hallmarks of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"22046282\", \"authors\": [\"Ghribi O\", \"Larson T\", \"Schommer J\", \"Prasanthi JR\"], \"date\": \"2011-01-01\"}, \"line\": 19791, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"CellStructure\": \"Endoplasmic Reticulum\", \"Subgraph\": \"Unfolded protein response subgraph\"}, \"source\": 77, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Here, we show that siRNA-mediated loss of calsyntenin-1 in cultured neurons alters APP processing to increase production of Abeta. We also show that calsyntenin-1 is reduced in Alzheimer's disease brains and that the extent of this reduction correlates with increased Abeta levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human molecular genetics\", \"reference\": \"22434822\", \"authors\": [\"Francis PT\", \"Perkinton MS\", \"Gray EH\", \"Vagnoni A\", \"Noble W\", \"Miller CC\"], \"date\": \"2012-07-01\"}, \"line\": 172, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 78, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Here, we show that siRNA-mediated loss of calsyntenin-1 in cultured neurons alters APP processing to increase production of Abeta. We also show that calsyntenin-1 is reduced in Alzheimer's disease brains and that the extent of this reduction correlates with increased Abeta levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human molecular genetics\", \"reference\": \"22434822\", \"authors\": [\"Francis PT\", \"Perkinton MS\", \"Gray EH\", \"Vagnoni A\", \"Noble W\", \"Miller CC\"], \"date\": \"2012-07-01\"}, \"line\": 172, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Calsyntenin subgraph\"}, \"source\": 78, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Here, we show that siRNA-mediated loss of calsyntenin-1 in cultured neurons alters APP processing to increase production of Abeta. We also show that calsyntenin-1 is reduced in Alzheimer's disease brains and that the extent of this reduction correlates with increased Abeta levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human molecular genetics\", \"reference\": \"22434822\", \"authors\": [\"Francis PT\", \"Perkinton MS\", \"Gray EH\", \"Vagnoni A\", \"Noble W\", \"Miller CC\"], \"date\": \"2012-07-01\"}, \"line\": 172, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Axonal transport subgraph\"}, \"source\": 78, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Understanding the mechanisms that control processing of the amyloid precursor protein (APP) to produce amyloid-ÃƒÅ¸ (ABeta¸) peptide represents a key area of Alzheimer's disease research. We also show that calsyntenin-1 is reduced in Alzheimer's disease brains and that the extent of this reduction correlates with increased ABeta¸ levels. Calsyntenin-1 is a ligand for kinesin-1 light chains and APP is transported through axons on kinesin-1 molecular motors.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human molecular genetics\", \"reference\": \"22434822\", \"authors\": [\"Francis PT\", \"Perkinton MS\", \"Gray EH\", \"Vagnoni A\", \"Noble W\", \"Miller CC\"], \"date\": \"2012-07-01\"}, \"line\": 34447, \"annotations\": {\"MeSHAnatomy\": \"Axons\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 78, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Understanding the mechanisms that control processing of the amyloid precursor protein (APP) to produce amyloid-ÃƒÅ¸ (ABeta¸) peptide represents a key area of Alzheimer's disease research. We also show that calsyntenin-1 is reduced in Alzheimer's disease brains and that the extent of this reduction correlates with increased ABeta¸ levels. Calsyntenin-1 is a ligand for kinesin-1 light chains and APP is transported through axons on kinesin-1 molecular motors.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human molecular genetics\", \"reference\": \"22434822\", \"authors\": [\"Francis PT\", \"Perkinton MS\", \"Gray EH\", \"Vagnoni A\", \"Noble W\", \"Miller CC\"], \"date\": \"2012-07-01\"}, \"line\": 34447, \"annotations\": {\"MeSHAnatomy\": \"Axons\", \"Subgraph\": \"Calsyntenin subgraph\"}, \"source\": 78, \"target\": 4, \"key\": 1}, {\"relation\": \"directlyIncreases\", \"evidence\": \"Differentiated neural cultures derived from KLC1-suppressed hESC contained neural rosettes but further differentiation revealed obvious morphological changes along with reduced levels of microtubule-associated neural proteins, including Tau and less secreted Abeta, supporting the previously established connection between KLC1, Tau and Abeta\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"22272245\", \"authors\": [\"Goldstein LS\", \"Herrera CM\", \"Almenar-Queralt A\", \"Flippin JD\", \"Killian RL\"], \"date\": \"2012-01-01\"}, \"line\": 1087, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 79, \"target\": 1, \"key\": 0}, {\"relation\": \"directlyIncreases\", \"evidence\": \"Differentiated neural cultures derived from KLC1-suppressed hESC contained neural rosettes but further differentiation revealed obvious morphological changes along with reduced levels of microtubule-associated neural proteins, including Tau and less secreted Abeta, supporting the previously established connection between KLC1, Tau and Abeta\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"22272245\", \"authors\": [\"Goldstein LS\", \"Herrera CM\", \"Almenar-Queralt A\", \"Flippin JD\", \"Killian RL\"], \"date\": \"2012-01-01\"}, \"line\": 1087, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Tau protein subgraph\"}, \"source\": 79, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Using a novel, unbiased genetic screen, Morihara et al. identified kinesin light chain-1 splice variant E (KLC1vE) as a modifier of Aβ accumulation...First, KLC1vE reduces APP transport, leading to Aβ accumulation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioEssays : news and reviews in molecular, cellular and developmental biology\", \"reference\": \"25394182\", \"authors\": [\"Gan KJ\", \"Silverman MA\", \"Morihara T\"], \"date\": \"2015-02-01\"}, \"line\": 55479, \"annotations\": {\"Subgraph\": \"Axonal transport subgraph\"}, \"source\": 79, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Using a novel, unbiased genetic screen, Morihara et al. identified kinesin light chain-1 splice variant E (KLC1vE) as a modifier of Aβ accumulation...First, KLC1vE reduces APP transport, leading to Aβ accumulation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioEssays : news and reviews in molecular, cellular and developmental biology\", \"reference\": \"25394182\", \"authors\": [\"Gan KJ\", \"Silverman MA\", \"Morihara T\"], \"date\": \"2015-02-01\"}, \"line\": 55479, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 79, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Using a novel, unbiased genetic screen, Morihara et al. identified kinesin light chain-1 splice variant E (KLC1vE) as a modifier of Aβ accumulation...First, KLC1vE reduces APP transport, leading to Aβ accumulation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioEssays : news and reviews in molecular, cellular and developmental biology\", \"reference\": \"25394182\", \"authors\": [\"Gan KJ\", \"Silverman MA\", \"Morihara T\"], \"date\": \"2015-02-01\"}, \"line\": 55479, \"annotations\": {\"Subgraph\": \"Innate immune system subgraph\"}, \"source\": 79, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Similar to the loss of presenilin or nicastrin, the inactivation of endogenous mAPH-1 using small interfering RNAs results in the decrease of presenilin levels, accumulation of gamma-secretase substrates (APP carboxyl-terminal fragments), and reduction of gamma-secretase products (amyloid-beta peptides and the intracellular domains of APP and Notch). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12297508\", \"authors\": [\"Li H\", \"Yu C\", \"Yu G\", \"Shah S\", \"Han W\", \"Lee SF\"], \"date\": \"2002-11-22\"}, \"line\": 32384, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 80, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Similar to the loss of presenilin or nicastrin, the inactivation of endogenous mAPH-1 using small interfering RNAs results in the decrease of presenilin levels, accumulation of gamma-secretase substrates (APP carboxyl-terminal fragments), and reduction of gamma-secretase products (amyloid-beta peptides and the intracellular domains of APP and Notch). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12297508\", \"authors\": [\"Li H\", \"Yu C\", \"Yu G\", \"Shah S\", \"Han W\", \"Lee SF\"], \"date\": \"2002-11-22\"}, \"line\": 32384, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 80, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Similar to the loss of presenilin or nicastrin, the inactivation of endogenous mAPH-1 using small interfering RNAs results in the decrease of presenilin levels, accumulation of gamma-secretase substrates (APP carboxyl-terminal fragments), and reduction of gamma-secretase products (amyloid-beta peptides and the intracellular domains of APP and Notch). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12297508\", \"authors\": [\"Li H\", \"Yu C\", \"Yu G\", \"Shah S\", \"Han W\", \"Lee SF\"], \"date\": \"2002-11-22\"}, \"line\": 32384, \"annotations\": {\"Subgraph\": \"Notch signaling subgraph\"}, \"source\": 80, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" In this issue of Nature Medicine, Thathiah et al.4 now provide provocative evidence that the adaptor protein β mediates the Abeta-altering effects of these GPCRs by promoting Abeta generation. This newly uncovered function of β-arrestin 2 suggests it could be targeted to decrease amyloid pathology in patients with Alzheimer's disease. Production of the amyloid-β peptide in Alzheimer's disease by the γ-secretase complex can be regulated by certain G proteinâ€“coupled receptors. This regulation seems to be mediated by β-arrestin-2, whose expression was found to be elevated in Alzheimer's disease brains.Recruitment of β-arrestin 2 to a GPCR leads to interaction with the γ-secretase complex via the Aph-1 subunit. Other members of the complex include presenilin-1 (PS-1), nicastrin (Nct) and Pen-2. The complex then moves laterally into lipid rafts, where γ-secretase activation is enhanced. Internalization may also occur to localize γ-secretase to late endosomes, where its activation is also increased. Cleavage of APP by β-secretase (BACE1) to release soluble APP (sAPPb) followed by γ-secretase produces Abeta and APP intracellular domain (AICD). Increased production and secretion of Abeta from cells can lead to extracellular Abeta aggregation in the form of plaques. Mutagenesis of GPR3 in regions of the protein that specifically interact with either G protein or β-arrestin 2 further showed that β-arrestin 2, not G protein, mediates the ability of GPR3 to increase Abeta levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature medicine\", \"reference\": \"23296004\", \"authors\": [\"Wolfe MS\"], \"date\": \"2013-01-01\"}, \"line\": 274, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 81, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" In this issue of Nature Medicine, Thathiah et al.4 now provide provocative evidence that the adaptor protein β mediates the Abeta-altering effects of these GPCRs by promoting Abeta generation. This newly uncovered function of β-arrestin 2 suggests it could be targeted to decrease amyloid pathology in patients with Alzheimer's disease. Production of the amyloid-β peptide in Alzheimer's disease by the γ-secretase complex can be regulated by certain G proteinâ€“coupled receptors. This regulation seems to be mediated by β-arrestin-2, whose expression was found to be elevated in Alzheimer's disease brains.Recruitment of β-arrestin 2 to a GPCR leads to interaction with the γ-secretase complex via the Aph-1 subunit. Other members of the complex include presenilin-1 (PS-1), nicastrin (Nct) and Pen-2. The complex then moves laterally into lipid rafts, where γ-secretase activation is enhanced. Internalization may also occur to localize γ-secretase to late endosomes, where its activation is also increased. Cleavage of APP by β-secretase (BACE1) to release soluble APP (sAPPb) followed by γ-secretase produces Abeta and APP intracellular domain (AICD). Increased production and secretion of Abeta from cells can lead to extracellular Abeta aggregation in the form of plaques. Mutagenesis of GPR3 in regions of the protein that specifically interact with either G protein or β-arrestin 2 further showed that β-arrestin 2, not G protein, mediates the ability of GPR3 to increase Abeta levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature medicine\", \"reference\": \"23296004\", \"authors\": [\"Wolfe MS\"], \"date\": \"2013-01-01\"}, \"line\": 274, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 81, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" In this issue of Nature Medicine, Thathiah et al.4 now provide provocative evidence that the adaptor protein β mediates the Abeta-altering effects of these GPCRs by promoting Abeta generation. This newly uncovered function of β-arrestin 2 suggests it could be targeted to decrease amyloid pathology in patients with Alzheimer's disease. Production of the amyloid-β peptide in Alzheimer's disease by the γ-secretase complex can be regulated by certain G proteinâ€“coupled receptors. This regulation seems to be mediated by β-arrestin-2, whose expression was found to be elevated in Alzheimer's disease brains.Recruitment of β-arrestin 2 to a GPCR leads to interaction with the γ-secretase complex via the Aph-1 subunit. Other members of the complex include presenilin-1 (PS-1), nicastrin (Nct) and Pen-2. The complex then moves laterally into lipid rafts, where γ-secretase activation is enhanced. Internalization may also occur to localize γ-secretase to late endosomes, where its activation is also increased. Cleavage of APP by β-secretase (BACE1) to release soluble APP (sAPPb) followed by γ-secretase produces Abeta and APP intracellular domain (AICD). Increased production and secretion of Abeta from cells can lead to extracellular Abeta aggregation in the form of plaques. Mutagenesis of GPR3 in regions of the protein that specifically interact with either G protein or β-arrestin 2 further showed that β-arrestin 2, not G protein, mediates the ability of GPR3 to increase Abeta levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nature medicine\", \"reference\": \"23296004\", \"authors\": [\"Wolfe MS\"], \"date\": \"2013-01-01\"}, \"line\": 274, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"G-protein-mediated signaling\"}, \"source\": 81, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"alpha(2)M) is an abundant plasma/extracellular space protein implicated in clearance of amyloid beta (Abeta)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"14678766\", \"authors\": [\"Hope C\", \"Mettenburg J\", \"Gonias SL\", \"DeKosky ST\", \"Kamboh MI\", \"Chu CT\"], \"date\": \"2003-12-01\"}, \"line\": 23112, \"annotations\": {\"Subgraph\": \"Alpha 2 macroglobulin subgraph\"}, \"source\": 82, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"alpha(2)M) is an abundant plasma/extracellular space protein implicated in clearance of amyloid beta (Abeta)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"14678766\", \"authors\": [\"Hope C\", \"Mettenburg J\", \"Gonias SL\", \"DeKosky ST\", \"Kamboh MI\", \"Chu CT\"], \"date\": \"2003-12-01\"}, \"line\": 23112, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 82, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, we investigated whether alpha2-macroglobulin (alpha2M), a protein present in neuritic plaques and elevated in Alzheimer's disease brain, is a potential regulatory factor for A beta fibril formation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"9489740\", \"authors\": [\"Little SP\", \"Horn JW\", \"Glinn MA\", \"Bales KR\", \"Paul SM\", \"Liu X\", \"Dodel RC\", \"Du Y\"], \"date\": \"1998-03-01\"}, \"line\": 23150, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 82, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present study, we investigated whether alpha2-macroglobulin (alpha2M), a protein present in neuritic plaques and elevated in Alzheimer's disease brain, is a potential regulatory factor for A beta fibril formation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"9489740\", \"authors\": [\"Little SP\", \"Horn JW\", \"Glinn MA\", \"Bales KR\", \"Paul SM\", \"Liu X\", \"Dodel RC\", \"Du Y\"], \"date\": \"1998-03-01\"}, \"line\": 23150, \"annotations\": {\"Subgraph\": \"Alpha 2 macroglobulin subgraph\"}, \"source\": 82, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"alpha(2)-Macroglobulin (alpha(2)M) is an abundant plasma/extracellular space protein implicated in clearance of amyloid beta (Abeta)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"14678766\", \"authors\": [\"Hope C\", \"Mettenburg J\", \"Gonias SL\", \"DeKosky ST\", \"Kamboh MI\", \"Chu CT\"], \"date\": \"2003-12-01\"}, \"line\": 23411, \"annotations\": {\"Subgraph\": \"Alpha 2 macroglobulin subgraph\"}, \"source\": 82, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"alpha(2)-Macroglobulin (alpha(2)M) is an abundant plasma/extracellular space protein implicated in clearance of amyloid beta (Abeta)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"14678766\", \"authors\": [\"Hope C\", \"Mettenburg J\", \"Gonias SL\", \"DeKosky ST\", \"Kamboh MI\", \"Chu CT\"], \"date\": \"2003-12-01\"}, \"line\": 23411, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 82, \"target\": 1, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"Humanin (HN) is a short neuroprotective peptide abolishing Abeta neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"18813209\", \"authors\": [\"Chiba T\", \"Yamada M\", \"Sasabe J\", \"Terashita K\", \"Shimoda M\", \"Matsuoka M\", \"Aiso S\"], \"date\": \"2009-02-01\"}, \"line\": 767, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 83, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Humanin (HN) is a short neuroprotective peptide abolishing Abeta neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"18813209\", \"authors\": [\"Chiba T\", \"Yamada M\", \"Sasabe J\", \"Terashita K\", \"Shimoda M\", \"Matsuoka M\", \"Aiso S\"], \"date\": \"2009-02-01\"}, \"line\": 36493, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 83, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Humanin (HN) is a short neuroprotective peptide abolishing Abeta neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular psychiatry\", \"reference\": \"18813209\", \"authors\": [\"Chiba T\", \"Yamada M\", \"Sasabe J\", \"Terashita K\", \"Shimoda M\", \"Matsuoka M\", \"Aiso S\"], \"date\": \"2009-02-01\"}, \"line\": 36493, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 83, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We report here that (i) a single amino acid mutation at the Thr-668 residue of APP695, located 14 amino acids toward the amino-terminal end from the (682)YENPTY(687) motif, reduced the interaction between members of the Fe65 family of proteins and APP, whereas interaction of APP with the phosphotyrosine interaction domain of other APP binders such as X11-like and mammalian disabled-1 was not influenced by this mutation; (ii) the phosphorylation of APP at Thr-668 diminished the interaction of APP with Fe65 by causing a conformational change in the cytoplasmic domain that contains the Fe65-binding motif, YENPTY; and (iii) the expression of Fe65 slightly suppressed maturation of APP and decreased production of beta-amyloid (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11517218\", \"authors\": [\"Suzuki T\", \"Iijima KI\", \"Elliott JI\", \"Ando K\", \"Kirino Y\"], \"date\": \"2001-10-26\"}, \"line\": 31612, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 84, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \" Overexpression of neuronal adaptor protein X11β has been shown to decrease the production of amyloid-β\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroreport\", \"reference\": \"22222501\", \"authors\": [\"Chai KH\", \"Hao Y\", \"McLoughlin DM\", \"Lau KF\", \"Chan HY\"], \"date\": \"2012-02-15\"}, \"line\": 43243, \"annotations\": {}, \"source\": 85, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"On the contrary, X11 stabilizes ABeta¸PP conformation in membrane, inhibiting ABeta¸ secretion in cultured cells [45], likely impairing ABeta¸PP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptosis. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABeta¸PP. The expression of JIP1b stabilizes immature ABeta¸PP and decreases the ABeta¸PP ectodomain, ABeta¸40/ÃƒÅ¸42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 36364, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 85, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"On the contrary, X11 stabilizes ABeta¸PP conformation in membrane, inhibiting ABeta¸ secretion in cultured cells [45], likely impairing ABeta¸PP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptosis. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABeta¸PP. The expression of JIP1b stabilizes immature ABeta¸PP and decreases the ABeta¸PP ectodomain, ABeta¸40/ÃƒÅ¸42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 36364, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 85, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our in vivo studies using transgenic mice have shown that overexpression of LRP in central nervous system (CNS) neurons increases soluble brain Abeta and this increase correlates with deficits in memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Annals of the New York Academy of Sciences\", \"reference\": \"17185504\", \"authors\": [\"Bu G\", \"Cam J\", \"Zerbinatti C\"], \"date\": \"2006-11-01\"}, \"line\": 25029, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 86, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our in vivo studies using transgenic mice have shown that overexpression of LRP in central nervous system (CNS) neurons increases soluble brain Abeta and this increase correlates with deficits in memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Annals of the New York Academy of Sciences\", \"reference\": \"17185504\", \"authors\": [\"Bu G\", \"Cam J\", \"Zerbinatti C\"], \"date\": \"2006-11-01\"}, \"line\": 25029, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 86, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Furthermore, LDLR overexpression dramatically reduced A beta aggregation and enhanced A beta clearance from the brain extracellular space. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"20005821\", \"authors\": [\"Kim J\", \"Castellano JM\", \"Jiang H\", \"Basak JM\", \"Parsadanian M\", \"Pham V\", \"Mason SM\", \"Paul SM\", \"Holtzman DM\"], \"date\": \"2009-12-10\"}, \"line\": 32128, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 86, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Furthermore, LDLR overexpression dramatically reduced A beta aggregation and enhanced A beta clearance from the brain extracellular space. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"20005821\", \"authors\": [\"Kim J\", \"Castellano JM\", \"Jiang H\", \"Basak JM\", \"Parsadanian M\", \"Pham V\", \"Mason SM\", \"Paul SM\", \"Holtzman DM\"], \"date\": \"2009-12-10\"}, \"line\": 32128, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 86, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Furthermore, LDLR overexpression dramatically reduced A beta aggregation and enhanced A beta clearance from the brain extracellular space. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"20005821\", \"authors\": [\"Kim J\", \"Castellano JM\", \"Jiang H\", \"Basak JM\", \"Parsadanian M\", \"Pham V\", \"Mason SM\", \"Paul SM\", \"Holtzman DM\"], \"date\": \"2009-12-10\"}, \"line\": 32129, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 86, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Furthermore, LDLR overexpression dramatically reduced A beta aggregation and enhanced A beta clearance from the brain extracellular space. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"20005821\", \"authors\": [\"Kim J\", \"Castellano JM\", \"Jiang H\", \"Basak JM\", \"Parsadanian M\", \"Pham V\", \"Mason SM\", \"Paul SM\", \"Holtzman DM\"], \"date\": \"2009-12-10\"}, \"line\": 32129, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 86, \"target\": 1, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34668, \"object\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"COP-Coated Vesicles\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Lysosomes\"}}}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 86, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34668, \"object\": {\"modifier\": \"Translocation\", \"effect\": {\"fromLoc\": {\"namespace\": \"MESHCS\", \"name\": \"COP-Coated Vesicles\"}, \"toLoc\": {\"namespace\": \"MESHCS\", \"name\": \"Lysosomes\"}}}, \"annotations\": {\"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 86, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer disease (AD) is characterized by the presence of senile plaques of amyloid-Abeta (Abeta) peptides derived from amyloid precursor protein (APP) and neurofibrillary tangles made of hyperphosphorylated Tau. Increasing APP gene dosage or expression has been shown to cause familial early-onset AD. However, whether and how protein stability of APP is regulated is unclear. The prolyl isomerase Pin1 and glycogen synthase kinase-3Abeta (GSK3Abeta) have been shown to have the opposite effects on APP processing and Tau hyperphosphorylation, relevant to the pathogenesis of AD. However, nothing is known about their relationship. In this study, we found that Pin1 binds to the pT330-P motif in GSK3Abeta to inhibit its kinase activity. Furthermore, Pin1 promotes protein turnover of APP by inhibiting GSK3Abeta activity. A point mutation either at Thr-330, the Pin1-binding site in GSK3Abeta, or at Thr-668, the GSK3Abeta phosphorylation site in APP, abolished the regulation of GSK3Abeta activity, Thr-668 phosphorylation, and APP stability by Pin1, resulting in reduced non-amyloidogenic APP processing and increased APP levels. These results uncover a novel role of Pin1 in inhibiting GSK3Abeta kinase activity to reduce APP protein levels, providing a previously unrecognized mechanism by which Pin1 protects against Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22184106\", \"authors\": [\"Lu KP\", \"Pastorino L\", \"Zhou XZ\", \"Ma SL\"], \"date\": \"2012-03-02\"}, \"line\": 2600, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 87, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer disease (AD) is characterized by the presence of senile plaques of amyloid-Abeta (Abeta) peptides derived from amyloid precursor protein (APP) and neurofibrillary tangles made of hyperphosphorylated Tau. Increasing APP gene dosage or expression has been shown to cause familial early-onset AD. However, whether and how protein stability of APP is regulated is unclear. The prolyl isomerase Pin1 and glycogen synthase kinase-3Abeta (GSK3Abeta) have been shown to have the opposite effects on APP processing and Tau hyperphosphorylation, relevant to the pathogenesis of AD. However, nothing is known about their relationship. In this study, we found that Pin1 binds to the pT330-P motif in GSK3Abeta to inhibit its kinase activity. Furthermore, Pin1 promotes protein turnover of APP by inhibiting GSK3Abeta activity. A point mutation either at Thr-330, the Pin1-binding site in GSK3Abeta, or at Thr-668, the GSK3Abeta phosphorylation site in APP, abolished the regulation of GSK3Abeta activity, Thr-668 phosphorylation, and APP stability by Pin1, resulting in reduced non-amyloidogenic APP processing and increased APP levels. These results uncover a novel role of Pin1 in inhibiting GSK3Abeta kinase activity to reduce APP protein levels, providing a previously unrecognized mechanism by which Pin1 protects against Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22184106\", \"authors\": [\"Lu KP\", \"Pastorino L\", \"Zhou XZ\", \"Ma SL\"], \"date\": \"2012-03-02\"}, \"line\": 2600, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 87, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"First, Pin1 inhibits the production of Abeta, and enhances the activity of eNOS. Second, Abeta and eNOS form a mutual inhibition system. Third, the well-balanced feedback signaling loop avoids the development of AD, HTN, and CAA by inhibiting the frequent pathological characteristics of these diseases, including Abeta deposition in cerebral microvessels and cerebral microbleeds.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"24332564\", \"authors\": [\"Xu ZX\", \"Du WT\", \"Wang JZ\", \"Sun XW\", \"Zhu WD\", \"Zhang HY\", \"Wang XH\"], \"date\": \"2014-02-01\"}, \"line\": 16991, \"annotations\": {\"Anatomy\": \"cerebral blood vessel\", \"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 87, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"First, Pin1 inhibits the production of Abeta, and enhances the activity of eNOS. Second, Abeta and eNOS form a mutual inhibition system. Third, the well-balanced feedback signaling loop avoids the development of AD, HTN, and CAA by inhibiting the frequent pathological characteristics of these diseases, including Abeta deposition in cerebral microvessels and cerebral microbleeds.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"24332564\", \"authors\": [\"Xu ZX\", \"Du WT\", \"Wang JZ\", \"Sun XW\", \"Zhu WD\", \"Zhang HY\", \"Wang XH\"], \"date\": \"2014-02-01\"}, \"line\": 16991, \"annotations\": {\"Anatomy\": \"cerebral blood vessel\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 87, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"To explore the molecular mechanism underlying AD, HTN, and CAA, we hypothesize a feedback signaling loop consisted of Pin1, eNOS, and Abeta. Pin1 and eNOS mainly inhibit Abeta deposition in cerebral vessels, cerebral microbleeds, and elevation of blood pressure, preventing the development of AD, HTN, and CAA, however, Abeta plays an opposite role and aggravates these diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"24332564\", \"authors\": [\"Xu ZX\", \"Du WT\", \"Wang JZ\", \"Sun XW\", \"Zhu WD\", \"Zhang HY\", \"Wang XH\"], \"date\": \"2014-02-01\"}, \"line\": 17025, \"annotations\": {\"Anatomy\": \"cerebral blood vessel\", \"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 87, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"To explore the molecular mechanism underlying AD, HTN, and CAA, we hypothesize a feedback signaling loop consisted of Pin1, eNOS, and Abeta. Pin1 and eNOS mainly inhibit Abeta deposition in cerebral vessels, cerebral microbleeds, and elevation of blood pressure, preventing the development of AD, HTN, and CAA, however, Abeta plays an opposite role and aggravates these diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"24332564\", \"authors\": [\"Xu ZX\", \"Du WT\", \"Wang JZ\", \"Sun XW\", \"Zhu WD\", \"Zhang HY\", \"Wang XH\"], \"date\": \"2014-02-01\"}, \"line\": 17025, \"annotations\": {\"Anatomy\": \"cerebral blood vessel\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 87, \"target\": 1, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"Alzheimer disease (AD) is characterized by the presence of senile plaques of amyloid-Abeta (Abeta) peptides derived from amyloid precursor protein (APP) and neurofibrillary tangles made of hyperphosphorylated Tau. Increasing APP gene dosage or expression has been shown to cause familial early-onset AD. However, whether and how protein stability of APP is regulated is unclear. The prolyl isomerase Pin1 and glycogen synthase kinase-3Abeta (GSK3Abeta) have been shown to have the opposite effects on APP processing and Tau hyperphosphorylation, relevant to the pathogenesis of AD. However, nothing is known about their relationship. In this study, we found that Pin1 binds to the pT330-P motif in GSK3Abeta to inhibit its kinase activity. Furthermore, Pin1 promotes protein turnover of APP by inhibiting GSK3Abeta activity. A point mutation either at Thr-330, the Pin1-binding site in GSK3Abeta, or at Thr-668, the GSK3Abeta phosphorylation site in APP, abolished the regulation of GSK3Abeta activity, Thr-668 phosphorylation, and APP stability by Pin1, resulting in reduced non-amyloidogenic APP processing and increased APP levels. These results uncover a novel role of Pin1 in inhibiting GSK3Abeta kinase activity to reduce APP protein levels, providing a previously unrecognized mechanism by which Pin1 protects against Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22184106\", \"authors\": [\"Lu KP\", \"Pastorino L\", \"Zhou XZ\", \"Ma SL\"], \"date\": \"2012-03-02\"}, \"line\": 2598, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 87, \"target\": 8, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Alzheimer disease (AD) is characterized by the presence of senile plaques of amyloid-Abeta (Abeta) peptides derived from amyloid precursor protein (APP) and neurofibrillary tangles made of hyperphosphorylated Tau. Increasing APP gene dosage or expression has been shown to cause familial early-onset AD. However, whether and how protein stability of APP is regulated is unclear. The prolyl isomerase Pin1 and glycogen synthase kinase-3Abeta (GSK3Abeta) have been shown to have the opposite effects on APP processing and Tau hyperphosphorylation, relevant to the pathogenesis of AD. However, nothing is known about their relationship. In this study, we found that Pin1 binds to the pT330-P motif in GSK3Abeta to inhibit its kinase activity. Furthermore, Pin1 promotes protein turnover of APP by inhibiting GSK3Abeta activity. A point mutation either at Thr-330, the Pin1-binding site in GSK3Abeta, or at Thr-668, the GSK3Abeta phosphorylation site in APP, abolished the regulation of GSK3Abeta activity, Thr-668 phosphorylation, and APP stability by Pin1, resulting in reduced non-amyloidogenic APP processing and increased APP levels. These results uncover a novel role of Pin1 in inhibiting GSK3Abeta kinase activity to reduce APP protein levels, providing a previously unrecognized mechanism by which Pin1 protects against Alzheimer disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22184106\", \"authors\": [\"Lu KP\", \"Pastorino L\", \"Zhou XZ\", \"Ma SL\"], \"date\": \"2012-03-02\"}, \"line\": 2598, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 87, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by Abeta (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"21994399\", \"authors\": [\"Chan C\", \"Geekiyanage H\"], \"date\": \"2011-10-12\"}, \"line\": 2720, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 88, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by Abeta (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"21994399\", \"authors\": [\"Chan C\", \"Geekiyanage H\"], \"date\": \"2011-10-12\"}, \"line\": 2720, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 88, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Ex vivo studies show that the levels of BACE1 are tightly regulated by the ATases. Specifically, up-regulation of ATase1 and ATase2 increases the levels of BACE1 and the generation of Abeta while siRNA-mediated down-regulation of either transferase achieves the opposite effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22267734\", \"authors\": [\"Ding Y\", \"Pehar M\", \"Ko MH\", \"Luo Y\", \"Puglielli L\", \"Kotch F\", \"Peters NR\", \"Salamat SM\"], \"date\": \"2012-03-09\"}, \"line\": 3042, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"MeSHAnatomy\": \"Neurons\", \"Cell\": \"microglial cell\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 89, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Ex vivo studies show that the levels of BACE1 are tightly regulated by the ATases. Specifically, up-regulation of ATase1 and ATase2 increases the levels of BACE1 and the generation of Abeta while siRNA-mediated down-regulation of either transferase achieves the opposite effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22267734\", \"authors\": [\"Ding Y\", \"Pehar M\", \"Ko MH\", \"Luo Y\", \"Puglielli L\", \"Kotch F\", \"Peters NR\", \"Salamat SM\"], \"date\": \"2012-03-09\"}, \"line\": 3042, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"MeSHAnatomy\": \"Neurons\", \"Cell\": \"microglial cell\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 89, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"RNAi silencing of GGA3 also elevated levels of BACE and Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"17553422\", \"authors\": [\"Tesco G\", \"Koh YH\", \"Kang EL\", \"Cameron AN\", \"Das S\", \"Sena-Esteves M\", \"Hiltunen M\", \"Yang SH\", \"Zhong Z\", \"Shen Y\", \"Simpkins JW\", \"Tanzi RE\"], \"date\": \"2007-06-07\"}, \"line\": 26618, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 90, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Cellular overexpression of PrP(C) inhibited the beta-secretase cleavage of APP and reduced Abeta formation. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17573534\", \"authors\": [\"Parkin ET\", \"Watt NT\", \"Hussain I\", \"Eckman EA\", \"Eckman CB\", \"Manson JC\", \"Baybutt HN\", \"Turner AJ\", \"Hooper NM\"], \"date\": \"2007-06-26\"}, \"line\": 25505, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 91, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In our opinion, albeit based on limited available data, a future potential therapeutic strategy is to mimic the mechanism by which the normal cellular form of the prion protein inhibits the beta-secretase beta-site amyloid precursor protein cleaving enzyme-1 (BACE1), and hence the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Expert opinion on therapeutic targets\", \"reference\": \"18479216\", \"authors\": [\"Griffiths HH\", \"Morten IJ\", \"Hooper NM\"], \"date\": \"2008-06-01\"}, \"line\": 26689, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 91, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"In our opinion, albeit based on limited available data, a future potential therapeutic strategy is to mimic the mechanism by which the normal cellular form of the prion protein inhibits the beta-secretase beta-site amyloid precursor protein cleaving enzyme-1 (BACE1), and hence the production of Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Expert opinion on therapeutic targets\", \"reference\": \"18479216\", \"authors\": [\"Griffiths HH\", \"Morten IJ\", \"Hooper NM\"], \"date\": \"2008-06-01\"}, \"line\": 26689, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 91, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Cellular prion protein regulates beta-secretase cleavage of the Alzheimer's amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17573534\", \"authors\": [\"Parkin ET\", \"Watt NT\", \"Hussain I\", \"Eckman EA\", \"Eckman CB\", \"Manson JC\", \"Baybutt HN\", \"Turner AJ\", \"Hooper NM\"], \"date\": \"2007-06-26\"}, \"line\": 30315, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 91, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Cellular prion protein regulates beta-secretase cleavage of the Alzheimer's amyloid precursor protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17573534\", \"authors\": [\"Parkin ET\", \"Watt NT\", \"Hussain I\", \"Eckman EA\", \"Eckman CB\", \"Manson JC\", \"Baybutt HN\", \"Turner AJ\", \"Hooper NM\"], \"date\": \"2007-06-26\"}, \"line\": 30315, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 91, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"PrP(C) decreases amyloid-beta (Abeta) production, which is involved in AD pathogenesis, by inhibiting beta-secretase (BACE1) activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"23577068\", \"authors\": [\"Miners JS\", \"Whitehouse IJ\", \"Kehoe PG\", \"Hooper NM\", \"Kellett KA\", \"Love S\", \"Glennon EB\"], \"date\": \"2013-01-01\"}, \"line\": 30346, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 91, \"target\": 1, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"PrP(C) decreases amyloid-beta (Abeta) production, which is involved in AD pathogenesis, by inhibiting beta-secretase (BACE1) activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"23577068\", \"authors\": [\"Miners JS\", \"Whitehouse IJ\", \"Kehoe PG\", \"Hooper NM\", \"Kellett KA\", \"Love S\", \"Glennon EB\"], \"date\": \"2013-01-01\"}, \"line\": 30346, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 91, \"target\": 1, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \"PrP(C) decreases amyloid-beta (Abeta) production, which is involved in AD pathogenesis, by inhibiting beta-secretase (BACE1) activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"23577068\", \"authors\": [\"Miners JS\", \"Whitehouse IJ\", \"Kehoe PG\", \"Hooper NM\", \"Kellett KA\", \"Love S\", \"Glennon EB\"], \"date\": \"2013-01-01\"}, \"line\": 30347, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 91, \"target\": 1, \"key\": 7}, {\"relation\": \"decreases\", \"evidence\": \"PrP(C) decreases amyloid-beta (Abeta) production, which is involved in AD pathogenesis, by inhibiting beta-secretase (BACE1) activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PloS one\", \"reference\": \"23577068\", \"authors\": [\"Miners JS\", \"Whitehouse IJ\", \"Kehoe PG\", \"Hooper NM\", \"Kellett KA\", \"Love S\", \"Glennon EB\"], \"date\": \"2013-01-01\"}, \"line\": 30347, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 91, \"target\": 1, \"key\": 8}, {\"relation\": \"increases\", \"evidence\": \" This report identifies ABCA2 as a key regulator of endogenous APP expression and processing and suggests a possible biochemical mechanism linking ABCA2 expression, APP processing and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"20704561\", \"authors\": [\"Davis W Jr\"], \"date\": \"2010-11-01\"}, \"line\": 4252, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 92, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" This report identifies ABCA2 as a key regulator of endogenous APP expression and processing and suggests a possible biochemical mechanism linking ABCA2 expression, APP processing and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"20704561\", \"authors\": [\"Davis W Jr\"], \"date\": \"2010-11-01\"}, \"line\": 4252, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"ATP binding cassette transport subgraph\"}, \"source\": 92, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"It is believed that amyloid-beta peptide (Abeta) plays a central role in the pathogenesis of Alzheimer's disease (AD). Thus, the process of amyloid precursor protein (APP) cleavage is a key event and has raised much attention in the field of AD research. It is proposed that APP, beta- and gamma-secretases are all located on the lipid raft, and the meeting of them is an indispensable step for Abeta generation. Endocytosis can lead to clustering of APP, beta- and gamma-secretases from separate smaller lipid rafts into a larger one. On the other hand, for myristoylated alanine-rich C kinase substrate (MARCKS), phosphorylation by protein kinase C (PKC) or interaction with Ca(2+) can lead to its release from membrane into cytoplasm. This process induces the release of actins and phosphatidylinositol 4, 5-bisphosphate (PIP2), which are important factors for endocytosis. Thus, the present review proposes that MARCKS may be implicated in Abeta generation, by modulating free PIP2 level and actin movement, causing endocytosis\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience bulletin\", \"reference\": \"20651816\", \"authors\": [\"Su R\", \"Han ZY\", \"Fan JP\", \"Zhang YL\"], \"date\": \"2010-08-01\"}, \"line\": 4322, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Phosphatidylinositol 3 subgraph\"}, \"source\": 93, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \" We observed that the neuronal cells primarily produce IL-34 and that microglia express its receptor, colony-stimulating factor 1 receptor. IL-34 promoted microglial proliferation and clearance of soluble oligomeric amyloid-β (oAbeta), which mediates synaptic dysfunction and neuronal damage in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"21872563\", \"authors\": [\"Mizoguchi H\", \"Jin S\", \"Doi Y\", \"Mizuno T\", \"Sonobe Y\", \"Takeuchi H\", \"Noda M\", \"Suzumura A\"], \"date\": \"2011-10-01\"}, \"line\": 4943, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 94, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \" We observed that the neuronal cells primarily produce IL-34 and that microglia express its receptor, colony-stimulating factor 1 receptor. IL-34 promoted microglial proliferation and clearance of soluble oligomeric amyloid-β (oAbeta), which mediates synaptic dysfunction and neuronal damage in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"21872563\", \"authors\": [\"Mizoguchi H\", \"Jin S\", \"Doi Y\", \"Mizuno T\", \"Sonobe Y\", \"Takeuchi H\", \"Noda M\", \"Suzumura A\"], \"date\": \"2011-10-01\"}, \"line\": 4943, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 94, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 6392, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 95, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 6392, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 95, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 6394, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 96, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing Aï¿½ levels [65]. Stimulation with muscarinic agonists or\\\\ activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage\\\\ of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 6394, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"ADAM Metallopeptidase subgraph\"}, \"source\": 96, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Cathepsin B cleaves the wild-type beta-secretase site sequence in AbetaPP to produce Abeta, and cathepsin B inhibitors administered to animal pmodels expressing AbetaPP containing the wild-type beta-secretase site sequence reduce brain Abeta in a manner consistent with beta-secretase inhibition. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"21613740\", \"authors\": [\"Kindy M\", \"Hook G\", \"Hook V\"], \"date\": \"2011-01-01\"}, \"line\": 28619, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 97, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Impaired degradation of amyloid beta (Abeta) peptides could lead to Abeta accumulation, an early trigger of Alzheimer's disease (AD). How Abeta-degrading enzymes are regulated remains largely unknown. Cystatin C (CysC, CST3) is an endogenous inhibitor of cysteine proteases, including cathepsin B (CatB), a recently discovered Abeta-degrading enzyme\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"18957217\", \"authors\": [\"Sun B\", \"Zhou Y\", \"Halabisky B\", \"Lo I\", \"Cho SH\", \"Mueller-Steiner S\", \"Devidze N\", \"Wang X\", \"Grubb A\", \"Gan L\"], \"date\": \"2008-10-23\"}, \"line\": 36579, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 97, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Impaired degradation of amyloid beta (Abeta) peptides could lead to Abeta accumulation, an early trigger of Alzheimer's disease (AD). How Abeta-degrading enzymes are regulated remains largely unknown. Cystatin C (CysC, CST3) is an endogenous inhibitor of cysteine proteases, including cathepsin B (CatB), a recently discovered Abeta-degrading enzyme\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"18957217\", \"authors\": [\"Sun B\", \"Zhou Y\", \"Halabisky B\", \"Lo I\", \"Cho SH\", \"Mueller-Steiner S\", \"Devidze N\", \"Wang X\", \"Grubb A\", \"Gan L\"], \"date\": \"2008-10-23\"}, \"line\": 36579, \"annotations\": {\"Subgraph\": \"Caspase subgraph\"}, \"source\": 97, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Lysosomal dysfunction may be the earliest histological change in AD. Amyloid plaques are full of active lysosomal hydrolases, implying that plaques may originate from lysosomal rupture. Cathepsins D and E (intracellular aspartyl proteases) are considered to influence Abeta peptides generation within the endosomalâ€“lysosomal pathway because they exhibit β- and γ-secretase like-activity [32]. For this reason, the endosomalâ€“lysosomal pathway is a site for cleavage of the APP into smaller β-amyloid-containing peptides, which are then degraded by cathepsins. Inhibition of cathepsins activity causes a rapid and pronounced build-up of potentially amyloidogenic protein fragments [33]. On the other hand, a failure to degrade aggregated Abeta_42 in late endosomes or secondary lysosomes was a mechanism that contributed to intracellular accumulation of Abeta in AD. The cysteine protease cathepsin B in lysosomes degrades A peptides, especially the aggregation-prone species Abeta_42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta biochimica et biophysica Sinica\", \"reference\": \"19499146\", \"authors\": [\"Zhang L\", \"Sheng R\", \"Qin Z\"], \"date\": \"2009-06-01\"}, \"line\": 6796, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 98, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Lysosomal dysfunction may be the earliest histological change in AD. Amyloid plaques are full of active lysosomal hydrolases, implying that plaques may originate from lysosomal rupture. Cathepsins D and E (intracellular aspartyl proteases) are considered to influence Abeta peptides generation within the endosomalâ€“lysosomal pathway because they exhibit β- and γ-secretase like-activity [32]. For this reason, the endosomalâ€“lysosomal pathway is a site for cleavage of the APP into smaller β-amyloid-containing peptides, which are then degraded by cathepsins. Inhibition of cathepsins activity causes a rapid and pronounced build-up of potentially amyloidogenic protein fragments [33]. On the other hand, a failure to degrade aggregated Abeta_42 in late endosomes or secondary lysosomes was a mechanism that contributed to intracellular accumulation of Abeta in AD. The cysteine protease cathepsin B in lysosomes degrades A peptides, especially the aggregation-prone species Abeta_42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta biochimica et biophysica Sinica\", \"reference\": \"19499146\", \"authors\": [\"Zhang L\", \"Sheng R\", \"Qin Z\"], \"date\": \"2009-06-01\"}, \"line\": 6796, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 98, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Pharmacological agents, such as dipeptidyl peptidase-4 (DPP-4) inhibitors, which increase the level of glucagon-like peptide-1 (GLP-1) and ameliorate T2D, have become valuable candidates as disease modifying agents in the treatment of AD. In addition, endogenous GLP-1 levels decrease amyloid beta (Abeta) peptide and tau phosphorylation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropharmacology\", \"reference\": \"23603201\", \"authors\": [\"Holsinger RM\", \"Basavan D\", \"Madhunapantula VS\", \"Chinni S\", \"Dubala A\", \"Khatwal RB\", \"Gali CC\", \"Kosaraju J\", \"Muthureddy Nataraj SK\"], \"date\": \"2013-09-01\"}, \"line\": 11969, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 99, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Pharmacological agents, such as dipeptidyl peptidase-4 (DPP-4) inhibitors, which increase the level of glucagon-like peptide-1 (GLP-1) and ameliorate T2D, have become valuable candidates as disease modifying agents in the treatment of AD. In addition, endogenous GLP-1 levels decrease amyloid beta (Abeta) peptide and tau phosphorylation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropharmacology\", \"reference\": \"23603201\", \"authors\": [\"Holsinger RM\", \"Basavan D\", \"Madhunapantula VS\", \"Chinni S\", \"Dubala A\", \"Khatwal RB\", \"Gali CC\", \"Kosaraju J\", \"Muthureddy Nataraj SK\"], \"date\": \"2013-09-01\"}, \"line\": 11969, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Tau protein subgraph\"}, \"source\": 99, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Pharmacological agents, such as dipeptidyl peptidase-4 (DPP-4) inhibitors, which increase the level of glucagon-like peptide-1 (GLP-1) and ameliorate T2D, have become valuable candidates as disease modifying agents in the treatment of AD. In addition, endogenous GLP-1 levels decrease amyloid beta (Abeta) peptide and tau phosphorylation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropharmacology\", \"reference\": \"23603201\", \"authors\": [\"Holsinger RM\", \"Basavan D\", \"Madhunapantula VS\", \"Chinni S\", \"Dubala A\", \"Khatwal RB\", \"Gali CC\", \"Kosaraju J\", \"Muthureddy Nataraj SK\"], \"date\": \"2013-09-01\"}, \"line\": 11969, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Insulin signal transduction\"}, \"source\": 99, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Pharmacological agents, such as dipeptidyl peptidase-4 (DPP-4) inhibitors, which increase the level of glucagon-like peptide-1 (GLP-1) and ameliorate T2D, have become valuable candidates as disease modifying agents in the treatment of AD. In addition, endogenous GLP-1 levels decrease amyloid beta (Abeta) peptide and tau phosphorylation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropharmacology\", \"reference\": \"23603201\", \"authors\": [\"Holsinger RM\", \"Basavan D\", \"Madhunapantula VS\", \"Chinni S\", \"Dubala A\", \"Khatwal RB\", \"Gali CC\", \"Kosaraju J\", \"Muthureddy Nataraj SK\"], \"date\": \"2013-09-01\"}, \"line\": 11969, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Glucagon subgraph\"}, \"source\": 99, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that -amyloid accumulation is the critical event in development of disease (I 49, I 50]. Lots of research has been done on the formation and accumulation of Al3, however, in the last years the mechanism' of amyloid clearance came into focus. For Al3 clearance several mechanisms are known (Fig. 2): i) EnzyÂ­ matic degradation by activated microglia or by insulin deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and angiotensin converting enzyme (ACE); ii) Receptor-mediated transport across the blood brain barrier (BBB) by binding to the low-density lipoprotein receptorÂ­ related protein (LRP) either directly or after binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M) to be delivered to peripheral sites of degradation, e.g., liver and kidney. (Review in [151 ]). Concerning insulin resistance it has been shown that IDE expression is stimulated by the IR/lGF-1 R cascade (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19519303\", \"authors\": [\"Freude S\", \"Schilbach K\", \"Schubert M\"], \"date\": \"2009-06-01\"}, \"line\": 8596, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 100, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that -amyloid accumulation is the critical event in development of disease (I 49, I 50]. Lots of research has been done on the formation and accumulation of Al3, however, in the last years the mechanism' of amyloid clearance came into focus. For Al3 clearance several mechanisms are known (Fig. 2): i) EnzyÂ­ matic degradation by activated microglia or by insulin deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and angiotensin converting enzyme (ACE); ii) Receptor-mediated transport across the blood brain barrier (BBB) by binding to the low-density lipoprotein receptorÂ­ related protein (LRP) either directly or after binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M) to be delivered to peripheral sites of degradation, e.g., liver and kidney. (Review in [151 ]). Concerning insulin resistance it has been shown that IDE expression is stimulated by the IR/lGF-1 R cascade (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19519303\", \"authors\": [\"Freude S\", \"Schilbach K\", \"Schubert M\"], \"date\": \"2009-06-01\"}, \"line\": 8596, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 100, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that -amyloid accumulation is the critical event in development of disease (I 49, I 50]. Lots of research has been done on the formation and accumulation of Al3, however, in the last years the mechanism' of amyloid clearance came into focus. For Al3 clearance several mechanisms are known (Fig. 2): i) EnzyÂ­ matic degradation by activated microglia or by insulin deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and angiotensin converting enzyme (ACE); ii) Receptor-mediated transport across the blood brain barrier (BBB) by binding to the low-density lipoprotein receptorÂ­ related protein (LRP) either directly or after binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M) to be delivered to peripheral sites of degradation, e.g., liver and kidney. (Review in [151 ]). Concerning insulin resistance it has been shown that IDE expression is stimulated by the IR/lGF-1 R cascade (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19519303\", \"authors\": [\"Freude S\", \"Schilbach K\", \"Schubert M\"], \"date\": \"2009-06-01\"}, \"line\": 8596, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Endothelin subgraph\"}, \"source\": 100, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The amyloid hypothesis of AD suggests that -amyloid accumulation is the critical event in development of disease (I 49, I 50]. Lots of research has been done on the formation and accumulation of Al3, however, in the last years the mechanism' of amyloid clearance came into focus. For Al3 clearance several mechanisms are known (Fig. 2): i) EnzyÂ­ matic degradation by activated microglia or by insulin deÂ­ grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and angiotensin converting enzyme (ACE); ii) Receptor-mediated transport across the blood brain barrier (BBB) by binding to the low-density lipoprotein receptorÂ­ related protein (LRP) either directly or after binding to apolÂ­ ipoprotein E (ApoE) and/or a2-macroglobulin (a2M) to be delivered to peripheral sites of degradation, e.g., liver and kidney. (Review in [151 ]). Concerning insulin resistance it has been shown that IDE expression is stimulated by the IR/lGF-1 R cascade (152]. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19519303\", \"authors\": [\"Freude S\", \"Schilbach K\", \"Schubert M\"], \"date\": \"2009-06-01\"}, \"line\": 8598, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Lower angiotensin-converting enzyme (ACE) activity could increase the risk of Alzheimer's disease (AD) as ACE functions to degrade amyloid-β (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Alzheimer's research &amp; therapy\", \"reference\": \"24987467\", \"authors\": [\"Kehoe PG\", \"Teunissen CE\", \"Jochemsen HM\", \"Scheltens P\", \"Jones RE\", \"Ashby EL\", \"Muller M\", \"Geerlings MI\", \"van der Flier WM\"], \"date\": \"2014-01-01\"}, \"line\": 20404, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Lower angiotensin-converting enzyme (ACE) activity could increase the risk of Alzheimer's disease (AD) as ACE functions to degrade amyloid-β (Abeta).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Alzheimer's research &amp; therapy\", \"reference\": \"24987467\", \"authors\": [\"Kehoe PG\", \"Teunissen CE\", \"Jochemsen HM\", \"Scheltens P\", \"Jones RE\", \"Ashby EL\", \"Muller M\", \"Geerlings MI\", \"van der Flier WM\"], \"date\": \"2014-01-01\"}, \"line\": 20404, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Lower CSF ACE protein level, and to a lesser extent serum ACE protein level and CSF ACE activity, were associated with lower CSF Abeta, indicating more brain Abeta pathology; adjusted regression coefficients (B) (95% CI) per SD increase were 0.09 (0.04; 0.15), 0.06 (0.00; 0.12) and 0.05 (0.00; 0.11), respectively. Further, lower CSF ACE protein level was associated with lower CSF tau and ptau levels; adjusted B's (95% CI) per SD increase were 0.15 (0.06; 0.25) and 0.17 (0.10; 0.25), respectively.These results strengthen the hypothesis that ACE degrades Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Alzheimer's research &amp; therapy\", \"reference\": \"24987467\", \"authors\": [\"Kehoe PG\", \"Teunissen CE\", \"Jochemsen HM\", \"Scheltens P\", \"Jones RE\", \"Ashby EL\", \"Muller M\", \"Geerlings MI\", \"van der Flier WM\"], \"date\": \"2014-01-01\"}, \"line\": 20428, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Lower CSF ACE protein level, and to a lesser extent serum ACE protein level and CSF ACE activity, were associated with lower CSF Abeta, indicating more brain Abeta pathology; adjusted regression coefficients (B) (95% CI) per SD increase were 0.09 (0.04; 0.15), 0.06 (0.00; 0.12) and 0.05 (0.00; 0.11), respectively. Further, lower CSF ACE protein level was associated with lower CSF tau and ptau levels; adjusted B's (95% CI) per SD increase were 0.15 (0.06; 0.25) and 0.17 (0.10; 0.25), respectively.These results strengthen the hypothesis that ACE degrades Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Alzheimer's research &amp; therapy\", \"reference\": \"24987467\", \"authors\": [\"Kehoe PG\", \"Teunissen CE\", \"Jochemsen HM\", \"Scheltens P\", \"Jones RE\", \"Ashby EL\", \"Muller M\", \"Geerlings MI\", \"van der Flier WM\"], \"date\": \"2014-01-01\"}, \"line\": 20428, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"ACE was found to significantly inhibit A beta aggregation in a dose response manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11604391\", \"authors\": [\"Igarashi A\", \"Hu J\", \"Kamata M\", \"Nakagawa H\"], \"date\": \"2001-12-21\"}, \"line\": 23307, \"annotations\": {\"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"ACE was found to significantly inhibit A beta aggregation in a dose response manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11604391\", \"authors\": [\"Igarashi A\", \"Hu J\", \"Kamata M\", \"Nakagawa H\"], \"date\": \"2001-12-21\"}, \"line\": 23307, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \"These findings led to the hypothesis that ACE may affect susceptibility to AD by degrading A beta and preventing the accumulation of amyloid plaques in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11604391\", \"authors\": [\"Igarashi A\", \"Hu J\", \"Kamata M\", \"Nakagawa H\"], \"date\": \"2001-12-21\"}, \"line\": 23313, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 7}, {\"relation\": \"decreases\", \"evidence\": \"These findings led to the hypothesis that ACE may affect susceptibility to AD by degrading A beta and preventing the accumulation of amyloid plaques in vivo.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11604391\", \"authors\": [\"Igarashi A\", \"Hu J\", \"Kamata M\", \"Nakagawa H\"], \"date\": \"2001-12-21\"}, \"line\": 23313, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 8}, {\"relation\": \"decreases\", \"evidence\": \"The most striking fact was that ACE degraded A beta by cleaving A beta-(1-40) at the site Asp(7)-Ser(8).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11604391\", \"authors\": [\"Igarashi A\", \"Hu J\", \"Kamata M\", \"Nakagawa H\"], \"date\": \"2001-12-21\"}, \"line\": 23317, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 9}, {\"relation\": \"decreases\", \"evidence\": \"The most striking fact was that ACE degraded A beta by cleaving A beta-(1-40) at the site Asp(7)-Ser(8).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11604391\", \"authors\": [\"Igarashi A\", \"Hu J\", \"Kamata M\", \"Nakagawa H\"], \"date\": \"2001-12-21\"}, \"line\": 23317, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 10}, {\"relation\": \"decreases\", \"evidence\": \"ACE was found to significantly inhibit A beta aggregation in a dose response manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11604391\", \"authors\": [\"Igarashi A\", \"Hu J\", \"Kamata M\", \"Nakagawa H\"], \"date\": \"2001-12-21\"}, \"line\": 23503, \"annotations\": {\"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 11}, {\"relation\": \"decreases\", \"evidence\": \"ACE was found to significantly inhibit A beta aggregation in a dose response manner.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11604391\", \"authors\": [\"Igarashi A\", \"Hu J\", \"Kamata M\", \"Nakagawa H\"], \"date\": \"2001-12-21\"}, \"line\": 23503, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 12}, {\"relation\": \"decreases\", \"evidence\": \"The most striking fact was that ACE degraded A beta by cleaving A beta-(1-40) at the site Asp(7)-Ser(8).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11604391\", \"authors\": [\"Igarashi A\", \"Hu J\", \"Kamata M\", \"Nakagawa H\"], \"date\": \"2001-12-21\"}, \"line\": 23512, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Renin-angiotensin subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 13}, {\"relation\": \"decreases\", \"evidence\": \"The most striking fact was that ACE degraded A beta by cleaving A beta-(1-40) at the site Asp(7)-Ser(8).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"11604391\", \"authors\": [\"Igarashi A\", \"Hu J\", \"Kamata M\", \"Nakagawa H\"], \"date\": \"2001-12-21\"}, \"line\": 23512, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 101, \"target\": 1, \"key\": 14}, {\"relation\": \"increases\", \"evidence\": \"for the first time it was shown that the Abeta40 and Abeta42 are substrates of DPPIV. DPPIV prohibits the fibrillation of peptides and promotes disaggregation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurochemistry international\", \"reference\": \"23579020\", \"authors\": [\"Antonyan A\", \"Hovnanyan N\", \"Movsisyan N\", \"Sharoyan S\", \"Hovnanyan K\", \"Harutyunyan H\", \"Mardanyan S\"], \"date\": \"2013-06-01\"}, \"line\": 11992, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 102, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Human bleomycin hydrolase (hBH) is a neutral cysteine protease genetically associated with increased risk for Alzheimer disease. We show here that ectopic expression of hBH in 293APPwt and CHOAPPsw cells altered the processing of amyloid precursor protein (APP) and increased significantly the release of its proteolytic fragment, beta amyloid (Abeta). We also found that hBH interacted and colocalized with APP as determined by subcellular fractionation, in vitro binding assay, and confocal immunolocalization. Metabolic labeling and pulse-chase experiments showed that ectopic hBH expression increased secretion of soluble APPalpha/beta products without changing the half-life of cellular APP. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"10973933\", \"authors\": [\"Koldamova RP\", \"Lazo JS\", \"Lefterov IM\"], \"date\": \"2000-09-01\"}, \"line\": 12169, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\", \"UserdefinedCellLine\": \"CHOAPPsw\"}, \"source\": 103, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Human bleomycin hydrolase (hBH) is a neutral cysteine protease genetically associated with increased risk for Alzheimer disease. We show here that ectopic expression of hBH in 293APPwt and CHOAPPsw cells altered the processing of amyloid precursor protein (APP) and increased significantly the release of its proteolytic fragment, beta amyloid (Abeta). We also found that hBH interacted and colocalized with APP as determined by subcellular fractionation, in vitro binding assay, and confocal immunolocalization. Metabolic labeling and pulse-chase experiments showed that ectopic hBH expression increased secretion of soluble APPalpha/beta products without changing the half-life of cellular APP. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"10973933\", \"authors\": [\"Koldamova RP\", \"Lazo JS\", \"Lefterov IM\"], \"date\": \"2000-09-01\"}, \"line\": 12169, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\", \"UserdefinedCellLine\": \"293APPwt\"}, \"source\": 103, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Human serum albumin (HSA) binds 95% of Abeta in blood plasma and is thought to inhibit plaque formation in peripheral tissue. However, the role of albumin in binding Abeta in the cerebrospinal fluid has been largely overlooked. Here we investigate the effect of HSA on both Abeta(1-40) and Abeta(1-42) fibril growth. We show that at micromolar cerebrospinal fluid levels, HSA inhibits the kinetics of Abeta fibrillization, significantly increasing the lag time and decreasing the total amount of fibrils produced\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22718756\", \"authors\": [\"Stanyon HF\", \"Viles JH\"], \"date\": \"2012-08-10\"}, \"line\": 12230, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"MeSHAnatomy\": \"Cerebrospinal Fluid\", \"Subgraph\": \"Albumin subgraph\"}, \"source\": 104, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Human serum albumin (HSA) binds 95% of Abeta in blood plasma and is thought to inhibit plaque formation in peripheral tissue. However, the role of albumin in binding Abeta in the cerebrospinal fluid has been largely overlooked. Here we investigate the effect of HSA on both Abeta(1-40) and Abeta(1-42) fibril growth. We show that at micromolar cerebrospinal fluid levels, HSA inhibits the kinetics of Abeta fibrillization, significantly increasing the lag time and decreasing the total amount of fibrils produced\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22718756\", \"authors\": [\"Stanyon HF\", \"Viles JH\"], \"date\": \"2012-08-10\"}, \"line\": 12230, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"MeSHAnatomy\": \"Cerebrospinal Fluid\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 104, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Human serum albumin (HSA) is a potent inhibitor of Abeta self-association and this novel, to our knowledge, function of HSA is of potential therapeutic interest for the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biophysical journal\", \"reference\": \"24094411\", \"authors\": [\"Zhang W\", \"Melacini G\", \"Jafari N\", \"Milojevic J\", \"Algamal M\"], \"date\": \"2013-10-01\"}, \"line\": 20793, \"annotations\": {\"MeSHAnatomy\": \"Serum\", \"Species\": \"9606\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Albumin subgraph\"}, \"source\": 104, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Human serum albumin (HSA) is a potent inhibitor of Abeta self-association and this novel, to our knowledge, function of HSA is of potential therapeutic interest for the treatment of Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biophysical journal\", \"reference\": \"24094411\", \"authors\": [\"Zhang W\", \"Melacini G\", \"Jafari N\", \"Milojevic J\", \"Algamal M\"], \"date\": \"2013-10-01\"}, \"line\": 20793, \"annotations\": {\"MeSHAnatomy\": \"Serum\", \"Species\": \"9606\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 104, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"19429119\", \"authors\": [\"Greco SJ\", \"Sarkar S\", \"Casadesus G\", \"Zhu X\", \"Smith MA\", \"Ashford JW\", \"Johnston JM\", \"Tezapsidis N\"], \"date\": \"2009-05-22\"}, \"line\": 12426, \"annotations\": {\"Subgraph\": \"Leptin subgraph\"}, \"source\": 105, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"19429119\", \"authors\": [\"Greco SJ\", \"Sarkar S\", \"Casadesus G\", \"Zhu X\", \"Smith MA\", \"Ashford JW\", \"Johnston JM\", \"Tezapsidis N\"], \"date\": \"2009-05-22\"}, \"line\": 12426, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 105, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Treatment with leptin reversed the 27-OHC-induced increase in Abeta and phosphorylated tau by decreasing the levels of BACE-1 and GSK-3beta respectively.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20157255\", \"authors\": [\"Marwarha G\", \"Dasari B\", \"Prasanthi JR\", \"Schommer J\", \"Ghribi O\"], \"date\": \"2010-01-01\"}, \"line\": 29451, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 105, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"19429119\", \"authors\": [\"Greco SJ\", \"Sarkar S\", \"Casadesus G\", \"Zhu X\", \"Smith MA\", \"Ashford JW\", \"Johnston JM\", \"Tezapsidis N\"], \"date\": \"2009-05-22\"}, \"line\": 12432, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Leptin subgraph\"}, \"source\": 105, \"target\": 8, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"19429119\", \"authors\": [\"Greco SJ\", \"Sarkar S\", \"Casadesus G\", \"Zhu X\", \"Smith MA\", \"Ashford JW\", \"Johnston JM\", \"Tezapsidis N\"], \"date\": \"2009-05-22\"}, \"line\": 12432, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 105, \"target\": 8, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"19429119\", \"authors\": [\"Greco SJ\", \"Sarkar S\", \"Casadesus G\", \"Zhu X\", \"Smith MA\", \"Ashford JW\", \"Johnston JM\", \"Tezapsidis N\"], \"date\": \"2009-05-22\"}, \"line\": 12432, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 105, \"target\": 8, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Treatment with leptin reversed the 27-OHC-induced increase in Abeta and phosphorylated tau by decreasing the levels of BACE-1 and GSK-3beta respectively.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20157255\", \"authors\": [\"Marwarha G\", \"Dasari B\", \"Prasanthi JR\", \"Schommer J\", \"Ghribi O\"], \"date\": \"2010-01-01\"}, \"line\": 29455, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 105, \"target\": 8, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Estrogen activates matrix metalloproteinases-2 and -9 to increase beta amyloid degradation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular neurosciences\", \"reference\": \"22402435\", \"authors\": [\"Sortino MA\", \"Merlo S\"], \"date\": \"2012-04-01\"}, \"line\": 15778, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 106, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Estrogen activates matrix metalloproteinases-2 and -9 to increase beta amyloid degradation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular neurosciences\", \"reference\": \"22402435\", \"authors\": [\"Sortino MA\", \"Merlo S\"], \"date\": \"2012-04-01\"}, \"line\": 15778, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 106, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"OBJECTIVE: Urinary-type plasminogen activator (uPA) binding to uPA receptor (uPAR) promotes the activation of matrix metalloproteinase-9 (MMP-9), which degrades amyloid beta protein (Abeta) in vitro.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Clinical neuropathology\", \"reference\": \"11327298\", \"authors\": [\"Yoshiyama Y\", \"Hattori T\", \"Asahina M\"], \"date\": \"2001-03-01\"}, \"line\": 29759, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 106, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"OBJECTIVE: Urinary-type plasminogen activator (uPA) binding to uPA receptor (uPAR) promotes the activation of matrix metalloproteinase-9 (MMP-9), which degrades amyloid beta protein (Abeta) in vitro.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Clinical neuropathology\", \"reference\": \"11327298\", \"authors\": [\"Yoshiyama Y\", \"Hattori T\", \"Asahina M\"], \"date\": \"2001-03-01\"}, \"line\": 29759, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 106, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that MMP-9 can degrade fAbeta and may contribute to ongoing clearance of plaques from amyloid-laden brains.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"16787929\", \"authors\": [\"Yan P\", \"Hu X\", \"Song H\", \"Yin K\", \"Bateman RJ\", \"Cirrito JR\", \"Xiao Q\", \"Hsu FF\", \"Turk JW\", \"Xu J\", \"Hsu CY\", \"Holtzman DM\", \"Lee JM\"], \"date\": \"2006-08-25\"}, \"line\": 29885, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {}, \"source\": 106, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Our aim was to test if this enzyme could also degrade the insoluble 40-42 residues long A beta peptides purified from Alzheimer Disease brain. Our results indicate that MMP2 hydrolyzes A beta 1-40 and A beta 1-42 peptides at Lys 16-Leu 17, at Leu 34-Met 35, and Met 35-Val 36 peptide bonds.These results suggest that MMP2 has the ability of degrading A beta of AD in vitro.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemical and biophysical research communications\", \"reference\": \"7811262\", \"authors\": [\"Roher AE\", \"Kasunic TC\", \"Woods AS\", \"Fridman R\", \"Ball MJ\", \"Cotter RJ\"], \"date\": \"1994-12-30\"}, \"line\": 15716, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 107, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our aim was to test if this enzyme could also degrade the insoluble 40-42 residues long A beta peptides purified from Alzheimer Disease brain. Our results indicate that MMP2 hydrolyzes A beta 1-40 and A beta 1-42 peptides at Lys 16-Leu 17, at Leu 34-Met 35, and Met 35-Val 36 peptide bonds.These results suggest that MMP2 has the ability of degrading A beta of AD in vitro.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemical and biophysical research communications\", \"reference\": \"7811262\", \"authors\": [\"Roher AE\", \"Kasunic TC\", \"Woods AS\", \"Fridman R\", \"Ball MJ\", \"Cotter RJ\"], \"date\": \"1994-12-30\"}, \"line\": 15716, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 107, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Estrogen activates matrix metalloproteinases-2 and -9 to increase beta amyloid degradation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular neurosciences\", \"reference\": \"22402435\", \"authors\": [\"Sortino MA\", \"Merlo S\"], \"date\": \"2012-04-01\"}, \"line\": 15776, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 107, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Estrogen activates matrix metalloproteinases-2 and -9 to increase beta amyloid degradation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular neurosciences\", \"reference\": \"22402435\", \"authors\": [\"Sortino MA\", \"Merlo S\"], \"date\": \"2012-04-01\"}, \"line\": 15776, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 107, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Cells overexpressing Bcl-xL were significantly protected from beta-amyloid neurotoxicity and staurosporine-induced apoptosis compared to vector-transfected controls.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience\", \"reference\": \"11226677\", \"authors\": [\"Poppe M\", \"Krohn AJ\", \"Bui NT\", \"Lankiewicz S\", \"Luetjens CM\", \"Prehn JH\"], \"date\": \"2001-01-01\"}, \"line\": 16336, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 108, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Cells overexpressing Bcl-xL were significantly protected from beta-amyloid neurotoxicity and staurosporine-induced apoptosis compared to vector-transfected controls.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience\", \"reference\": \"11226677\", \"authors\": [\"Poppe M\", \"Krohn AJ\", \"Bui NT\", \"Lankiewicz S\", \"Luetjens CM\", \"Prehn JH\"], \"date\": \"2001-01-01\"}, \"line\": 16336, \"annotations\": {\"Subgraph\": \"Bcl-2 subgraph\"}, \"source\": 108, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Deletion of Abca1 increases Abeta deposition\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"16207708\", \"authors\": [\"Wahrle SE\", \"Jiang H\", \"Parsadanian M\", \"Hartman RE\", \"Bales KR\", \"Paul SM\", \"Holtzman DM\"], \"date\": \"2005-12-30\"}, \"line\": 23877, \"annotations\": {\"Subgraph\": \"ATP binding cassette transport subgraph\"}, \"source\": 109, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Deletion of Abca1 increases Abeta deposition\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"16207708\", \"authors\": [\"Wahrle SE\", \"Jiang H\", \"Parsadanian M\", \"Hartman RE\", \"Bales KR\", \"Paul SM\", \"Holtzman DM\"], \"date\": \"2005-12-30\"}, \"line\": 23877, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 109, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"These data indicate that ABCA1 and ABCG1 play a significant role in the regulation of neuronal cholesterol efflux to apoE discs and in suppression of APP processing to generate Abeta peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"17121837\", \"authors\": [\"Kim WS\", \"Rahmanto AS\", \"Kamili A\", \"Rye KA\", \"Guillemin GJ\", \"Gelissen IC\", \"Jessup W\", \"Hill AF\", \"Garner B\"], \"date\": \"2007-02-02\"}, \"line\": 36542, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\"}, \"source\": 109, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In contrast, enhanced FasL shedding, by recombinant MMP-7, completely protected neurons from Abeta neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current biology : CB\", \"reference\": \"12372252\", \"authors\": [\"Ethell DW\", \"Kinloch R\", \"Green DR\"], \"date\": \"2002-09-17\"}, \"line\": 18104, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 110, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In contrast, enhanced FasL shedding, by recombinant MMP-7, completely protected neurons from Abeta neurotoxicity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current biology : CB\", \"reference\": \"12372252\", \"authors\": [\"Ethell DW\", \"Kinloch R\", \"Green DR\"], \"date\": \"2002-09-17\"}, \"line\": 18104, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 110, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Thus, endogenous tPA, preferentially its aggregate form, could degrade Abeta molecules and maintain low levels of brain Abeta, resulting in the delay of AD pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"24126163\", \"authors\": [\"Koh JY\", \"Carmeliet P\", \"Jo DG\", \"Yun JH\", \"Oh SB\", \"Byun CJ\", \"Lee JY\"], \"date\": \"2014-03-01\"}, \"line\": 18548, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Plasminogen activator subgraph\"}, \"source\": 111, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Thus, endogenous tPA, preferentially its aggregate form, could degrade Abeta molecules and maintain low levels of brain Abeta, resulting in the delay of AD pathogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"24126163\", \"authors\": [\"Koh JY\", \"Carmeliet P\", \"Jo DG\", \"Yun JH\", \"Oh SB\", \"Byun CJ\", \"Lee JY\"], \"date\": \"2014-03-01\"}, \"line\": 18548, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 111, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Subsequent studies showed that it was plasmin, the product of tPA activation of plasminogen, that specifically cleaved A beta 1-40 in the amino terminal region between Arg5 and His6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"10471309\", \"authors\": [\"Van Nostrand WE\", \"Porter M\"], \"date\": \"1999-08-31\"}, \"line\": 18627, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Plasminogen activator subgraph\"}, \"source\": 111, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Subsequent studies showed that it was plasmin, the product of tPA activation of plasminogen, that specifically cleaved A beta 1-40 in the amino terminal region between Arg5 and His6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"10471309\", \"authors\": [\"Van Nostrand WE\", \"Porter M\"], \"date\": \"1999-08-31\"}, \"line\": 18627, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 111, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Additionally, plasmin (activated by tPA) attenuates Abeta neurotoxicity by degrading the peptide and rendering it inactive.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular neuroscience : MN\", \"reference\": \"14501010\", \"authors\": [\"Melchor JP\", \"Pawlak R\", \"Chen Z\", \"Strickland S\"], \"date\": \"2003-01-01\"}, \"line\": 29730, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Plasminogen activator subgraph\"}, \"source\": 111, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Additionally, plasmin (activated by tPA) attenuates Abeta neurotoxicity by degrading the peptide and rendering it inactive.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of molecular neuroscience : MN\", \"reference\": \"14501010\", \"authors\": [\"Melchor JP\", \"Pawlak R\", \"Chen Z\", \"Strickland S\"], \"date\": \"2003-01-01\"}, \"line\": 29730, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 111, \"target\": 1, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Thus, neuroserpin inhibition of tissue plasminogen activator activity leads to reduced plasmin and may be responsible for reduced clearance of amyloid-beta in the Alzheimer disease brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"19222708\", \"authors\": [\"Fabbro S\", \"Seeds NW\"], \"date\": \"2009-04-01\"}, \"line\": 18689, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Species\": \"9606\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 112, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Thus, neuroserpin inhibition of tissue plasminogen activator activity leads to reduced plasmin and may be responsible for reduced clearance of amyloid-beta in the Alzheimer disease brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"19222708\", \"authors\": [\"Fabbro S\", \"Seeds NW\"], \"date\": \"2009-04-01\"}, \"line\": 18689, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Species\": \"9606\", \"Subgraph\": \"Plasminogen activator subgraph\"}, \"source\": 112, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"This suggests that increased expression of uPA may initially serve as a protective mechanism leading to localized degradation and clearance of the pathogenic stimulus A beta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12754271\", \"authors\": [\"Davis J\", \"Wagner MR\", \"Zhang W\", \"Xu F\", \"Van Nostrand WE\"], \"date\": \"2003-05-23\"}, \"line\": 20286, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Anatomy\": \"cerebral blood vessel\", \"Subgraph\": \"Plasminogen activator subgraph\"}, \"source\": 113, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"This suggests that increased expression of uPA may initially serve as a protective mechanism leading to localized degradation and clearance of the pathogenic stimulus A beta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"12754271\", \"authors\": [\"Davis J\", \"Wagner MR\", \"Zhang W\", \"Xu F\", \"Van Nostrand WE\"], \"date\": \"2003-05-23\"}, \"line\": 20286, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Anatomy\": \"cerebral blood vessel\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 113, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"An increase of the Abeta binding protein transthyretin suggests that increased clearance of Abeta underlies the reduction in plaques.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemical and biophysical research communications\", \"reference\": \"19523444\", \"authors\": [\"Killick R\", \"Scales G\", \"Leroy K\", \"Causevic M\", \"Hooper C\", \"Irvine EE\", \"Choudhury AI\", \"Drinkwater L\", \"Kerr F\", \"Al-Qassab H\", \"Stephenson J\", \"Yilmaz Z\", \"Giese KP\", \"Brion JP\", \"Withers DJ\", \"Lovestone S\"], \"date\": \"2009-08-14\"}, \"line\": 27270, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 114, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"18941241\", \"authors\": [\"Shimizu E\", \"Kawahara K\", \"Kajizono M\", \"Sawada M\", \"Nakayama H\"], \"date\": \"2008-11-01\"}, \"line\": 29275, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 115, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"18941241\", \"authors\": [\"Shimizu E\", \"Kawahara K\", \"Kajizono M\", \"Sawada M\", \"Nakayama H\"], \"date\": \"2008-11-01\"}, \"line\": 29275, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 115, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"18941241\", \"authors\": [\"Shimizu E\", \"Kawahara K\", \"Kajizono M\", \"Sawada M\", \"Nakayama H\"], \"date\": \"2008-11-01\"}, \"line\": 29275, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 115, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Finally, we show that PAT1a promotes APP/APLPs processing, resulting in increased secretion of beta-amyloid peptide\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"17050537\", \"authors\": [\"Kuan YH\", \"Gruebl T\", \"Soba P\", \"Eggert S\", \"Nesic I\", \"Back S\", \"Kirsch J\", \"Beyreuther K\", \"Kins S\"], \"date\": \"2006-12-29\"}, \"line\": 25119, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 116, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"PAR-4 (prostate apoptosis response-4) is a leucine zipper protein that was initially identified to be associated with neuronal degeneration and aberrant Abeta production in pmodels of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"15671026\", \"authors\": [\"Xie J\", \"Guo Q\"], \"date\": \"2005-04-08\"}, \"line\": 25335, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 117, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"PAR-4 (prostate apoptosis response-4) is a leucine zipper protein that was initially identified to be associated with neuronal degeneration and aberrant Abeta production in pmodels of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"15671026\", \"authors\": [\"Xie J\", \"Guo Q\"], \"date\": \"2005-04-08\"}, \"line\": 25335, \"annotations\": {\"Subgraph\": \"Apoptosis signaling subgraph\"}, \"source\": 117, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" Overexpression of constitutively active CK1epsilon, one of the CK1 isoforms expressed in brain, leads to an increase in Abeta peptide production. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17360493\", \"authors\": [\"Flajolet M\", \"He G\", \"Heiman M\", \"Lin A\", \"Nairn AC\", \"Greengard P\"], \"date\": \"2007-03-06\"}, \"line\": 25479, \"annotations\": {\"Subgraph\": \"Wnt signaling subgraph\"}, \"source\": 118, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Here we show that glutaminyl cyclase (QC) catalyzes the formation of Abeta 3(pE)-40/42 after amyloidogenic processing of APP in two different cell lines, applying specific ELISAs and Western blotting based on urea-PAGE.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"18570439\", \"authors\": [\"Cynis H\", \"Scheel E\", \"Saido TC\", \"Schilling S\", \"Demuth HU\"], \"date\": \"2008-07-15\"}, \"line\": 25609, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 119, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Here we show that glutaminyl cyclase (QC) catalyzes the formation of Abeta 3(pE)-40/42 after amyloidogenic processing of APP in two different cell lines, applying specific ELISAs and Western blotting based on urea-PAGE.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"18570439\", \"authors\": [\"Cynis H\", \"Scheel E\", \"Saido TC\", \"Schilling S\", \"Demuth HU\"], \"date\": \"2008-07-15\"}, \"line\": 25609, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 119, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Here we show that glutaminyl cyclase (QC) catalyzes the formation of Abeta 3(pE)-40/42 after amyloidogenic processing of APP in two different cell lines, applying specific ELISAs and Western blotting based on urea-PAGE. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry\", \"reference\": \"18570439\", \"authors\": [\"Cynis H\", \"Scheel E\", \"Saido TC\", \"Schilling S\", \"Demuth HU\"], \"date\": \"2008-07-15\"}, \"line\": 26953, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 119, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"urther, overexpression of the SUMO E2 enzyme ubc9 along with SUMO-1 results in decreased levels of Abeta aggregates in cells transfected with the familial Alzheimer's disease-associated V642F mutant APP, indicating the potential of up-regulating activity of the cellular sumoylation machinery as an approach against Alzheimer's disease. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemical and biophysical research communications\", \"reference\": \"18675254\", \"authors\": [\"Zhang YQ\", \"Sarge KD\"], \"date\": \"2008-10-03\"}, \"line\": 25631, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 120, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \" Importantly, knockdown of endogenous RanBP9 significantly reduced Abeta generation in Chinese hamster ovary cells and in primary neurons, demonstrating its physiological role in BACE1 cleavage of APP. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"19251705\", \"authors\": [\"Lakshmana MK\", \"Yoon IS\", \"Chen E\", \"Bianchi E\", \"Koo EH\", \"Kang DE\"], \"date\": \"2009-05-01\"}, \"line\": 25674, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 121, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Consistently, overexpression of calpain in heterologous APP expressing cells up-regulated the level of BACE1 and increased Abeta production. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20595388\", \"authors\": [\"Liang B\", \"Duan BY\", \"Zhou XP\", \"Gong JX\", \"Luo ZG\"], \"date\": \"2010-09-03\"}, \"line\": 25989, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 122, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Consistently, overexpression of calpain in heterologous APP expressing cells up-regulated the level of BACE1 and increased Abeta production. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20595388\", \"authors\": [\"Liang B\", \"Duan BY\", \"Zhou XP\", \"Gong JX\", \"Luo ZG\"], \"date\": \"2010-09-03\"}, \"line\": 25989, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 122, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Using high-throughput siRNA screening technology, we assessed 15,200 genes for their role in Abeta42 secretion and identified leucine-rich repeat transmembrane 3 (LRRTM3) as a neuronal gene that promotes APP processing by BACE1. siRNAs targeting LRRTM3 inhibit the secretion of Abeta40, Abeta42, and sAPPbeta, the N-terminal APP fragment produced by BACE1 cleavage, from cultured cells and primary neurons by up to 60%, whereas overexpression increases Abeta secretion.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17098871\", \"authors\": [\"Majercak J\", \"Ray WJ\", \"Espeseth A\", \"Simon A\", \"Shi XP\", \"Wolffe C\", \"Getty K\", \"Marine S\", \"Stec E\", \"Ferrer M\", \"Strulovici B\", \"Bartz S\", \"Gates A\", \"Xu M\", \"Huang Q\", \"Ma L\", \"Shughrue P\", \"Burchard J\", \"Colussi D\", \"Pietrak B\", \"Kahana J\", \"Beher D\", \"Rosahl T\", \"Shearman M\", \"Hazuda D\", \"Sachs AB\", \"Koblan KS\", \"Seabrook GR\", \"Stone DJ\"], \"date\": \"2006-11-21\"}, \"line\": 26537, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 123, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Using high-throughput siRNA screening technology, we assessed 15,200 genes for their role in Abeta42 secretion and identified leucine-rich repeat transmembrane 3 (LRRTM3) as a neuronal gene that promotes APP processing by BACE1. siRNAs targeting LRRTM3 inhibit the secretion of Abeta40, Abeta42, and sAPPbeta, the N-terminal APP fragment produced by BACE1 cleavage, from cultured cells and primary neurons by up to 60%, whereas overexpression increases Abeta secretion.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Proceedings of the National Academy of Sciences of the United States of America\", \"reference\": \"17098871\", \"authors\": [\"Majercak J\", \"Ray WJ\", \"Espeseth A\", \"Simon A\", \"Shi XP\", \"Wolffe C\", \"Getty K\", \"Marine S\", \"Stec E\", \"Ferrer M\", \"Strulovici B\", \"Bartz S\", \"Gates A\", \"Xu M\", \"Huang Q\", \"Ma L\", \"Shughrue P\", \"Burchard J\", \"Colussi D\", \"Pietrak B\", \"Kahana J\", \"Beher D\", \"Rosahl T\", \"Shearman M\", \"Hazuda D\", \"Sachs AB\", \"Koblan KS\", \"Seabrook GR\", \"Stone DJ\"], \"date\": \"2006-11-21\"}, \"line\": 26537, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 123, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Overexpression of NgR decreases Abeta production in neuroblastoma culture, and targeted disruption of NgR expression increases transgenic mouse brain Abeta levels, Abeta plaque deposition, and dystrophic neurites. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"16452662\", \"authors\": [\"Park JH\", \"Gimbel DA\", \"GrandPre T\", \"Lee JK\", \"Kim JE\", \"Li W\", \"Lee DH\", \"Strittmatter SM\"], \"date\": \"2006-02-01\"}, \"line\": 26974, \"annotations\": {\"Subgraph\": \"Axonal guidance subgraph\"}, \"source\": 124, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Overexpression of NgR decreases Abeta production in neuroblastoma culture, and targeted disruption of NgR expression increases transgenic mouse brain Abeta levels, Abeta plaque deposition, and dystrophic neurites. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"16452662\", \"authors\": [\"Park JH\", \"Gimbel DA\", \"GrandPre T\", \"Lee JK\", \"Kim JE\", \"Li W\", \"Lee DH\", \"Strittmatter SM\"], \"date\": \"2006-02-01\"}, \"line\": 26974, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 124, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Overexpression of NgR decreases Abeta production in neuroblastoma culture, and targeted disruption of NgR expression increases transgenic mouse brain Abeta levels, Abeta plaque deposition, and dystrophic neurites. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"16452662\", \"authors\": [\"Park JH\", \"Gimbel DA\", \"GrandPre T\", \"Lee JK\", \"Kim JE\", \"Li W\", \"Lee DH\", \"Strittmatter SM\"], \"date\": \"2006-02-01\"}, \"line\": 26974, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 124, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \" Deletion of NgR expression increases Abeta plaque deposition in transgenic mice, while excess soluble NgR treatment reduces Abeta plaque deposition in mice.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"18220524\", \"authors\": [\"Park JH\", \"Strittmatter SM\"], \"date\": \"2007-12-01\"}, \"line\": 26995, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 124, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"18941241\", \"authors\": [\"Shimizu E\", \"Kawahara K\", \"Kajizono M\", \"Sawada M\", \"Nakayama H\"], \"date\": \"2008-11-01\"}, \"line\": 29274, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 125, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"18941241\", \"authors\": [\"Shimizu E\", \"Kawahara K\", \"Kajizono M\", \"Sawada M\", \"Nakayama H\"], \"date\": \"2008-11-01\"}, \"line\": 29274, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 125, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"18941241\", \"authors\": [\"Shimizu E\", \"Kawahara K\", \"Kajizono M\", \"Sawada M\", \"Nakayama H\"], \"date\": \"2008-11-01\"}, \"line\": 29274, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 125, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Down regulation of DNMT results in hypomethylation of BACE1 and APP which are involved in Abeta production and causes upregulation of their protein expression; in turn SP1 transcription factor increases which finally results in Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Free radical biology &amp; medicine\", \"reference\": \"19245828\", \"authors\": [\"Zawia NH\", \"Lahiri DK\", \"Cardozo-Pelaez F\"], \"date\": \"2009-05-01\"}, \"line\": 42556, \"annotations\": {\"Species\": \"10090\"}, \"source\": 126, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In CAA, and probably in AD, corticosteroid treatment suppresses the secondarily induced vascular inflammation (Chang et al., 2012, Kloppenborg et al., 2010 and Machida et al., 2012) and the activation of S100A9/Mrp14 (Gebhardt et al., 2002 and Kametani and Ikeda, 2013). Recently, it has been reported that anti-Abeta autoantibodies were progressively reduced in the cerebrospinal fluid of a patient with CAA-related inflammation, following intravenous steroid administration (Piazza et al., 2013). Reduction of S100A9/Mrp14 might induce suppression of amyloid fibril formation (Zhang et al., 2012) and increase phagocytosis of fibrillar Abeta in microglia cells (Kummer et al., 2012), resulting in reduced Abeta amyloid fibril deposition.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"24262203\", \"authors\": [\"Kametani F\"], \"date\": \"2014-04-01\"}, \"line\": 41708, \"annotations\": {\"Subgraph\": \"Metabolism of steroid hormones subgraph\"}, \"source\": 127, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In CAA, and probably in AD, corticosteroid treatment suppresses the secondarily induced vascular inflammation (Chang et al., 2012, Kloppenborg et al., 2010 and Machida et al., 2012) and the activation of S100A9/Mrp14 (Gebhardt et al., 2002 and Kametani and Ikeda, 2013). Recently, it has been reported that anti-Abeta autoantibodies were progressively reduced in the cerebrospinal fluid of a patient with CAA-related inflammation, following intravenous steroid administration (Piazza et al., 2013). Reduction of S100A9/Mrp14 might induce suppression of amyloid fibril formation (Zhang et al., 2012) and increase phagocytosis of fibrillar Abeta in microglia cells (Kummer et al., 2012), resulting in reduced Abeta amyloid fibril deposition.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"24262203\", \"authors\": [\"Kametani F\"], \"date\": \"2014-04-01\"}, \"line\": 41708, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 127, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Pro-inflammatory S100A9 protein is increasingly recognized as an important contributor to inflammation-related neurodegeneration. Here, we provide insights into S100A9 specific mechanisms of action in Alzheimer's disease (AD). Due to its inherent amyloidogenicity S100A9 contributes to amyloid plaque formation together with Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta neuropathologica\", \"reference\": \"24240735\", \"authors\": [\"Narayana VK\", \"Shankar SK\", \"Olofsson A\", \"Zhao L\", \"Gräslund A\", \"Mu Y\", \"Brännström T\", \"Jia X\", \"Wang C\", \"Klechikov AG\", \"Gharibyan AL\", \"Wärmländer SK\", \"Jarvet J\", \"Morozova-Roche LA\"], \"date\": \"2014-04-01\"}, \"line\": 41732, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 127, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Pro-inflammatory S100A9 protein is increasingly recognized as an important contributor to inflammation-related neurodegeneration. Here, we provide insights into S100A9 specific mechanisms of action in Alzheimer's disease (AD). Due to its inherent amyloidogenicity S100A9 contributes to amyloid plaque formation together with Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta neuropathologica\", \"reference\": \"24240735\", \"authors\": [\"Narayana VK\", \"Shankar SK\", \"Olofsson A\", \"Zhao L\", \"Gräslund A\", \"Mu Y\", \"Brännström T\", \"Jia X\", \"Wang C\", \"Klechikov AG\", \"Gharibyan AL\", \"Wärmländer SK\", \"Jarvet J\", \"Morozova-Roche LA\"], \"date\": \"2014-04-01\"}, \"line\": 41732, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\"}, \"source\": 127, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"We therefore conclude that Mrp14 promotes APP processing and Abeta accumulation under neuroinflammatory conditions.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"23223301\", \"authors\": [\"Vieira-Saecker A\", \"Vogl T\", \"Griep A\", \"Roth J\", \"Gelpi E\", \"Heneka MT\", \"Axt D\", \"Jessen F\", \"Kummer MP\"], \"date\": \"2012-12-05\"}, \"line\": 41772, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 127, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"We therefore conclude that Mrp14 promotes APP processing and Abeta accumulation under neuroinflammatory conditions.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"23223301\", \"authors\": [\"Vieira-Saecker A\", \"Vogl T\", \"Griep A\", \"Roth J\", \"Gelpi E\", \"Heneka MT\", \"Axt D\", \"Jessen F\", \"Kummer MP\"], \"date\": \"2012-12-05\"}, \"line\": 41772, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\"}, \"source\": 127, \"target\": 1, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"The 5-HT4 receptor may play a role in memory and learning and 5-HT4 receptor activation has been suggested to modulate acetylcholine release and to reduce amyloid-β (Abeta) accumulation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"21673407\", \"authors\": [\"Knudsen GM\", \"Haahr MT\", \"Hasselbalch SG\", \"Madsen K\", \"Lehel S\", \"Neumann WJ\", \"Holst K\", \"Marner L\"], \"date\": \"2011-01-01\"}, \"line\": 45152, \"annotations\": {}, \"source\": 128, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The formation of endogenous inflammatory lipid mediators, leukotrienes, is initiated by 5-lipoxygenase (5-LOX), which is also expressed in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical hypotheses\", \"reference\": \"10790729\", \"authors\": [\"Manev H\"], \"date\": \"2000-01-01\"}, \"line\": 21051, \"annotations\": {\"MeSHAnatomy\": \"Neurons\"}, \"source\": 129, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Here we report that the levels of a human brain-enriched miRNA-125b are up-regulated in interleukin-6 (IL-6)-stressed normal human astrocytes (NHA), a treatment known to induce astrogliosis. In vitro, anti-miRNA-125b added exogenously to IL-6-stressed NHA cultures attenuated both glial cell proliferation and increased the expression of the cyclin-dependent kinase inhibitor 2A (CDKN2A), a miRNA-125b target and negative regulator of cell growth. A strong positive correlation between miRNA-125b abundance and the glial cell markers glial fibrillary acidic protein (GFAP) and vimentin, and CDKN2A down-regulation was noted in advanced Alzheimer's disease (AD) and in Down's syndrome (DS) brain, chronic neurological disorders associated with astrogliosis. The results suggest that miRNA-125b up-regulation contributes to astrogliosis and to defects in the cell cycle that are characteristic of degenerating brain tissues.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"20347935\", \"authors\": [\"Pogue AI\", \"Cui JG\", \"Li YY\", \"Zhao Y\", \"Culicchia F\", \"Lukiw WJ\"], \"date\": \"2010-05-26\"}, \"line\": 9197, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 130, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Here we report that the levels of a human brain-enriched miRNA-125b are up-regulated in interleukin-6 (IL-6)-stressed normal human astrocytes (NHA), a treatment known to induce astrogliosis. In vitro, anti-miRNA-125b added exogenously to IL-6-stressed NHA cultures attenuated both glial cell proliferation and increased the expression of the cyclin-dependent kinase inhibitor 2A (CDKN2A), a miRNA-125b target and negative regulator of cell growth. A strong positive correlation between miRNA-125b abundance and the glial cell markers glial fibrillary acidic protein (GFAP) and vimentin, and CDKN2A down-regulation was noted in advanced Alzheimer's disease (AD) and in Down's syndrome (DS) brain, chronic neurological disorders associated with astrogliosis. The results suggest that miRNA-125b up-regulation contributes to astrogliosis and to defects in the cell cycle that are characteristic of degenerating brain tissues.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"20347935\", \"authors\": [\"Pogue AI\", \"Cui JG\", \"Li YY\", \"Zhao Y\", \"Culicchia F\", \"Lukiw WJ\"], \"date\": \"2010-05-26\"}, \"line\": 9198, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 131, \"target\": 3, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"we found that the Nogo-66 receptor (NgR) interacts physically with both Abeta and the amyloid precursor protein (APP). The inverse correlation of Abeta levels with NgR levels within the brain may reflect regulation of Abeta production and/or Abeta clearance.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17182778\", \"authors\": [\"Park JH\", \"Widi GA\", \"Gimbel DA\", \"Harel NY\", \"Lee DH\", \"Strittmatter SM\"], \"date\": \"2006-12-20\"}, \"line\": 36506, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 132, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"we found that the Nogo-66 receptor (NgR) interacts physically with both Abeta and the amyloid precursor protein (APP). The inverse correlation of Abeta levels with NgR levels within the brain may reflect regulation of Abeta production and/or Abeta clearance.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"17182778\", \"authors\": [\"Park JH\", \"Widi GA\", \"Gimbel DA\", \"Harel NY\", \"Lee DH\", \"Strittmatter SM\"], \"date\": \"2006-12-20\"}, \"line\": 36506, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 132, \"target\": 4, \"key\": 1}, {\"relation\": \"directlyDecreases\", \"evidence\": \"Lower-expression of PS1 and over-expression of IDE or NEP may be helpful in potentially lowering brain Abeta levels in subjects with AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"19355846\", \"authors\": [\"Tian J\", \"Shi J\", \"Zhang L\", \"Yin J\", \"Hu Q\", \"Xu Y\", \"Sheng S\", \"Wang P\", \"Ren Y\", \"Wang R\", \"Wang Y\"], \"date\": \"2009-04-01\"}, \"line\": 36517, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 133, \"target\": 4, \"key\": 0}, {\"relation\": \"directlyDecreases\", \"evidence\": \"Using screening approaches in primary neurons, we identified brain-derived neurotrophic factor (BDNF) as a major inducer of Sorla that activates receptor gene transcription through the ERK (extracellular regulated kinase) pathway.These findings demonstrate that the beneficial effects ascribed to BDNF in APP metabolism act through induction of Sorla that encodes a negative regulator of neuronal APP processing\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"20007471\", \"authors\": [\"Rohe M\", \"Synowitz M\", \"Glass R\", \"Paul SM\", \"Nykjaer A\", \"Willnow TE\"], \"date\": \"2009-12-09\"}, \"line\": 36563, \"annotations\": {\"Subgraph\": \"MAPK-ERK subgraph\"}, \"source\": 134, \"target\": 4, \"key\": 0}, {\"relation\": \"directlyDecreases\", \"evidence\": \"Using screening approaches in primary neurons, we identified brain-derived neurotrophic factor (BDNF) as a major inducer of Sorla that activates receptor gene transcription through the ERK (extracellular regulated kinase) pathway.These findings demonstrate that the beneficial effects ascribed to BDNF in APP metabolism act through induction of Sorla that encodes a negative regulator of neuronal APP processing\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"20007471\", \"authors\": [\"Rohe M\", \"Synowitz M\", \"Glass R\", \"Paul SM\", \"Nykjaer A\", \"Willnow TE\"], \"date\": \"2009-12-09\"}, \"line\": 36563, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 134, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Impaired degradation of amyloid beta (Abeta) peptides could lead to Abeta accumulation, an early trigger of Alzheimer's disease (AD). How Abeta-degrading enzymes are regulated remains largely unknown. Cystatin C (CysC, CST3) is an endogenous inhibitor of cysteine proteases, including cathepsin B (CatB), a recently discovered Abeta-degrading enzyme\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"18957217\", \"authors\": [\"Sun B\", \"Zhou Y\", \"Halabisky B\", \"Lo I\", \"Cho SH\", \"Mueller-Steiner S\", \"Devidze N\", \"Wang X\", \"Grubb A\", \"Gan L\"], \"date\": \"2008-10-23\"}, \"line\": 36580, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 135, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Impaired degradation of amyloid beta (Abeta) peptides could lead to Abeta accumulation, an early trigger of Alzheimer's disease (AD). How Abeta-degrading enzymes are regulated remains largely unknown. Cystatin C (CysC, CST3) is an endogenous inhibitor of cysteine proteases, including cathepsin B (CatB), a recently discovered Abeta-degrading enzyme\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"18957217\", \"authors\": [\"Sun B\", \"Zhou Y\", \"Halabisky B\", \"Lo I\", \"Cho SH\", \"Mueller-Steiner S\", \"Devidze N\", \"Wang X\", \"Grubb A\", \"Gan L\"], \"date\": \"2008-10-23\"}, \"line\": 36580, \"annotations\": {\"Subgraph\": \"Caspase subgraph\"}, \"source\": 135, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"These studies indicate that ABCA7 has the capacity to stimulate cellular cholesterol efflux to apoE discs and regulate APP processing resulting in an inhibition of Abeta production\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18429932\", \"authors\": [\"Chan SL\", \"Kim WS\", \"Kwok JB\", \"Hill AF\", \"Cappai R\", \"Rye KA\", \"Garner B\"], \"date\": \"2008-07-01\"}, \"line\": 36529, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 136, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"These studies indicate that ABCA7 has the capacity to stimulate cellular cholesterol efflux to apoE discs and regulate APP processing resulting in an inhibition of Abeta production\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18429932\", \"authors\": [\"Chan SL\", \"Kim WS\", \"Kwok JB\", \"Hill AF\", \"Cappai R\", \"Rye KA\", \"Garner B\"], \"date\": \"2008-07-01\"}, \"line\": 36529, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"ATP binding cassette transport subgraph\"}, \"source\": 136, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"These studies indicate that ABCA7 has the capacity to stimulate cellular cholesterol efflux to apoE discs and regulate APP processing resulting in an inhibition of Abeta production\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"18429932\", \"authors\": [\"Chan SL\", \"Kim WS\", \"Kwok JB\", \"Hill AF\", \"Cappai R\", \"Rye KA\", \"Garner B\"], \"date\": \"2008-07-01\"}, \"line\": 36529, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 136, \"target\": 4, \"key\": 2}, {\"relation\": \"directlyDecreases\", \"evidence\": \"In the brain, megalin is expressed in brain capillaries, ependymal cells and choroid plexus, where it participates in the clearance of brain amyloid beta-peptide (Abeta) complex.Additionally, given that FE65 mediates the interaction between the low density lipoprotein receptor-related protein-1 and the amyloid precursor protein (APP) to modulate the rate of APP internalization from the cell surface, we hypothesize that megalin could also interact with APP in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular neurosciences\", \"reference\": \"20637285\", \"authors\": [\"Alvira-Botero X\", \"Pérez-Gonzalez R\", \"Spuch C\", \"Vargas T\", \"Antequera D\", \"Garzón M\", \"Bermejo-Pareja F\", \"Carro E\"], \"date\": \"2010-11-01\"}, \"line\": 36721, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 137, \"target\": 4, \"key\": 0}, {\"relation\": \"directlyDecreases\", \"evidence\": \"In the brain, megalin is expressed in brain capillaries, ependymal cells and choroid plexus, where it participates in the clearance of brain amyloid beta-peptide (Abeta) complex.Additionally, given that FE65 mediates the interaction between the low density lipoprotein receptor-related protein-1 and the amyloid precursor protein (APP) to modulate the rate of APP internalization from the cell surface, we hypothesize that megalin could also interact with APP in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Molecular and cellular neurosciences\", \"reference\": \"20637285\", \"authors\": [\"Alvira-Botero X\", \"Pérez-Gonzalez R\", \"Spuch C\", \"Vargas T\", \"Antequera D\", \"Garzón M\", \"Bermejo-Pareja F\", \"Carro E\"], \"date\": \"2010-11-01\"}, \"line\": 36721, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 137, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34884, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 138, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34884, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 138, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34911, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 138, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34911, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 138, \"target\": 4, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"it has been demonstrated that LRP1 might bind to APP independent of the KPI domain in APP. This APP - LRP1 interaction is facilitated through a trimeric complex of APP-FE65-LRP1, which has a functional role in APP processing. Along with LRP1, APP is transported from the early secretory compartments to the cell surface and subsequently internalised into the endosomal / lysosomal compartments. Recent investigations indicate that ApoER2 and SorLA fulfil a similar role in shifting APP localisation in the cell, which affects APP processing and the production of the APP derived amyloid beta-peptide (Abeta)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"18288927\", \"authors\": [\"Jaeger S\", \"Pietrzik CU\"], \"date\": \"2008-02-01\"}, \"line\": 34570, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 139, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"it has been demonstrated that LRP1 might bind to APP independent of the KPI domain in APP. This APP - LRP1 interaction is facilitated through a trimeric complex of APP-FE65-LRP1, which has a functional role in APP processing. Along with LRP1, APP is transported from the early secretory compartments to the cell surface and subsequently internalised into the endosomal / lysosomal compartments. Recent investigations indicate that ApoER2 and SorLA fulfil a similar role in shifting APP localisation in the cell, which affects APP processing and the production of the APP derived amyloid beta-peptide (Abeta)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"18288927\", \"authors\": [\"Jaeger S\", \"Pietrzik CU\"], \"date\": \"2008-02-01\"}, \"line\": 34570, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 139, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Following this export Abeta is degraded in the liver, where LRP1 potentially conducts the removal of Abeta from the blood stream\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"18288927\", \"authors\": [\"Jaeger S\", \"Pietrzik CU\"], \"date\": \"2008-02-01\"}, \"line\": 34577, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Liver\", \"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 139, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Following this export Abeta is degraded in the liver, where LRP1 potentially conducts the removal of Abeta from the blood stream\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"18288927\", \"authors\": [\"Jaeger S\", \"Pietrzik CU\"], \"date\": \"2008-02-01\"}, \"line\": 34577, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Liver\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 139, \"target\": 4, \"key\": 3}, {\"relation\": \"directlyDecreases\", \"evidence\": \"Furthermore, overexpression of miR-195 in N2a/APP decreased the level of Abeta, while inhibition of miR-195 resulted in an increase of Abeta. Thus, we demonstrated that miR-195 could downregulate the level of Abeta by inhibiting the translation of BACE1. We conclude that miR-195 might provide a therapeutic strategy for AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain research bulletin\", \"reference\": \"22721728\", \"authors\": [\"Song B\", \"Wang M\", \"Tan S\", \"Wang LM\", \"Zhu HC\", \"Teng JF\", \"Duan DX\"], \"date\": \"2012-09-01\"}, \"line\": 36645, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 140, \"target\": 4, \"key\": 0}, {\"relation\": \"directlyDecreases\", \"evidence\": \"Furthermore, overexpression of miR-195 in N2a/APP decreased the level of Abeta, while inhibition of miR-195 resulted in an increase of Abeta. Thus, we demonstrated that miR-195 could downregulate the level of Abeta by inhibiting the translation of BACE1. We conclude that miR-195 might provide a therapeutic strategy for AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Brain research bulletin\", \"reference\": \"22721728\", \"authors\": [\"Song B\", \"Wang M\", \"Tan S\", \"Wang LM\", \"Zhu HC\", \"Teng JF\", \"Duan DX\"], \"date\": \"2012-09-01\"}, \"line\": 36645, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 140, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Impaired degradation of amyloid beta (Abeta) peptides could lead to Abeta accumulation, an early trigger of Alzheimer's disease (AD). How Abeta-degrading enzymes are regulated remains largely unknown. Cystatin C (CysC, CST3) is an endogenous inhibitor of cysteine proteases, including cathepsin B (CatB), a recently discovered Abeta-degrading enzyme\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"18957217\", \"authors\": [\"Sun B\", \"Zhou Y\", \"Halabisky B\", \"Lo I\", \"Cho SH\", \"Mueller-Steiner S\", \"Devidze N\", \"Wang X\", \"Grubb A\", \"Gan L\"], \"date\": \"2008-10-23\"}, \"line\": 36577, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 141, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Impaired degradation of amyloid beta (Abeta) peptides could lead to Abeta accumulation, an early trigger of Alzheimer's disease (AD). How Abeta-degrading enzymes are regulated remains largely unknown. Cystatin C (CysC, CST3) is an endogenous inhibitor of cysteine proteases, including cathepsin B (CatB), a recently discovered Abeta-degrading enzyme\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron\", \"reference\": \"18957217\", \"authors\": [\"Sun B\", \"Zhou Y\", \"Halabisky B\", \"Lo I\", \"Cho SH\", \"Mueller-Steiner S\", \"Devidze N\", \"Wang X\", \"Grubb A\", \"Gan L\"], \"date\": \"2008-10-23\"}, \"line\": 36577, \"annotations\": {\"Subgraph\": \"Caspase subgraph\"}, \"source\": 141, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In cells, CRF treatment increases Abeta production and triggers CRF receptor 1 (CRFR1) and γ-secretase internalization. Co-immunoprecipitation studies establish that γ-secretase associates with CRFR1; this is mediated by β-arrestin binding motifs. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"The EMBO journal\", \"reference\": \"25964433\", \"authors\": [\"Daaka Y\", \"Ryabinin AE\", \"Han C\", \"Ran Y\", \"Park HJ\", \"Holmes O\", \"Felsenstein KM\", \"Ceballos-Diaz C\", \"Hauger RL\", \"Jung JI\", \"Kim SH\", \"Golde TE\", \"Wolfe MS\", \"Price AR\", \"Smithson L\"], \"date\": \"2015-06-12\"}, \"line\": 54770, \"annotations\": {\"Disease\": \"Alzheimer's disease\"}, \"source\": 142, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta deposits in AD, parenchymal as well as (cap)CAA and dyshoric angiopathy, are associated with a local inflammatory reaction, including activation of microglial cells and astrocytes that, among others, produce cytokines and reactive oxygen species. This neuroinflammatory reaction may account for at least part of the cognitive decline. In previous studies we observed that small heat shock proteins (sHsps) are associated with Abeta deposits in AD. In this study the molecular chaperones Hsp20, HspB8 and HspB2B3 were found to colocalize with CAA and capCAA in AD brains. In addition, Hsp20, HspB8 and HspB2B3 colocalized with intercellular adhesion molecule 1 (ICAM-1) in capCAA-associated dyshoric angiopathy. Furthermore, we demonstrated that Hsp20, HspB8 and HspB2B3 induced production of interleukin 8, soluble ICAM-1 and monocyte chemoattractant protein 1 by human leptomeningeal smooth muscle cells and human brain astrocytes in vitro and that Hsp27 inhibited production of transforming growth factor beta 1 and CD40 ligand. Our results suggest a central role for sHsps in the neuroinflammatory reaction in AD and CAA and thus in contributing to cognitive decline.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"21849559\", \"authors\": [\"Veerhuis R\", \"Verbeek MM\", \"de Waal RM\", \"Boelens W\", \"Versleijen AA\", \"Bruinsma IB\", \"de Jager M\", \"Carrano A\", \"Rozemuller AJ\"], \"date\": \"2011-08-17\"}, \"line\": 38092, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 143, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta deposits in AD, parenchymal as well as (cap)CAA and dyshoric angiopathy, are associated with a local inflammatory reaction, including activation of microglial cells and astrocytes that, among others, produce cytokines and reactive oxygen species. This neuroinflammatory reaction may account for at least part of the cognitive decline. In previous studies we observed that small heat shock proteins (sHsps) are associated with Abeta deposits in AD. In this study the molecular chaperones Hsp20, HspB8 and HspB2B3 were found to colocalize with CAA and capCAA in AD brains. In addition, Hsp20, HspB8 and HspB2B3 colocalized with intercellular adhesion molecule 1 (ICAM-1) in capCAA-associated dyshoric angiopathy. Furthermore, we demonstrated that Hsp20, HspB8 and HspB2B3 induced production of interleukin 8, soluble ICAM-1 and monocyte chemoattractant protein 1 by human leptomeningeal smooth muscle cells and human brain astrocytes in vitro and that Hsp27 inhibited production of transforming growth factor beta 1 and CD40 ligand. Our results suggest a central role for sHsps in the neuroinflammatory reaction in AD and CAA and thus in contributing to cognitive decline.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of neuroscience : the official journal of the Society for Neuroscience\", \"reference\": \"21849559\", \"authors\": [\"Veerhuis R\", \"Verbeek MM\", \"de Waal RM\", \"Boelens W\", \"Versleijen AA\", \"Bruinsma IB\", \"de Jager M\", \"Carrano A\", \"Rozemuller AJ\"], \"date\": \"2011-08-17\"}, \"line\": 38092, \"annotations\": {\"Subgraph\": \"Chaperone subgraph\"}, \"source\": 143, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"On the contrary, X11 stabilizes ABeta¸PP conformation in membrane, inhibiting ABeta¸ secretion in cultured cells [45], likely impairing ABeta¸PP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptosis. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABeta¸PP. The expression of JIP1b stabilizes immature ABeta¸PP and decreases the ABeta¸PP ectodomain, ABeta¸40/ÃƒÅ¸42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 36366, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 144, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"On the contrary, X11 stabilizes ABeta¸PP conformation in membrane, inhibiting ABeta¸ secretion in cultured cells [45], likely impairing ABeta¸PP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptosis. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABeta¸PP. The expression of JIP1b stabilizes immature ABeta¸PP and decreases the ABeta¸PP ectodomain, ABeta¸40/ÃƒÅ¸42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 36366, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 144, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34881, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 145, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34881, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 145, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34908, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 145, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34908, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 145, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34880, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 146, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34880, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 146, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34907, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 146, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34907, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 146, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35272, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 147, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35272, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 147, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1β could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptosis in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGFβ1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGFβ2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37444, \"annotations\": {\"Subgraph\": \"TGF-Beta subgraph\"}, \"source\": 148, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34882, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 149, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34882, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 149, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34909, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 149, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34909, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 149, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34883, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 150, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34883, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 150, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34910, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 150, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34910, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 150, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34885, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 151, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta¸ levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta¸ levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34885, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 151, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34912, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 151, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The lower concentrations of aged ABeta¸42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta¸ production while lowering synaptic activity minimizes ABeta¸ production. Similarly specific stimulation of NMDA receptors promotes ABeta¸ production and ABeta¸ in turn depresses synaptic activity. Thus indirectly ABeta¸ also plays a role in suppressing excessive glutamate release. Interestingly, ABeta¸ may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta¸ have a role in controlling synaptic activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 34912, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Glutamatergic subgraph\"}, \"source\": 151, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"The pharmacological blockage of autophagy resulted in a dramatic increase of mutant SOD1 aggregates. Immunoprecipitation studies, performed during autophagic flux blockage, demonstrated that mutant SOD1 interacts with the HspB8/Bag3/Hsc70/CHIP multiheteromeric complex, known to selectively activate autophagic removal of misfolded proteins. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human molecular genetics\", \"reference\": \"20570967\", \"authors\": [\"Crippa V\", \"Sau D\", \"Rusmini P\", \"Boncoraglio A\", \"Onesto E\", \"Bolzoni E\", \"Galbiati M\", \"Fontana E\", \"Marino M\", \"Carra S\", \"Bendotti C\", \"De Biasi S\", \"Poletti A\"], \"date\": \"2010-09-01\"}, \"line\": 36624, \"annotations\": {\"Subgraph\": \"Chaperone subgraph\"}, \"source\": 152, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The pharmacological blockage of autophagy resulted in a dramatic increase of mutant SOD1 aggregates. Immunoprecipitation studies, performed during autophagic flux blockage, demonstrated that mutant SOD1 interacts with the HspB8/Bag3/Hsc70/CHIP multiheteromeric complex, known to selectively activate autophagic removal of misfolded proteins. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Human molecular genetics\", \"reference\": \"20570967\", \"authors\": [\"Crippa V\", \"Sau D\", \"Rusmini P\", \"Boncoraglio A\", \"Onesto E\", \"Bolzoni E\", \"Galbiati M\", \"Fontana E\", \"Marino M\", \"Carra S\", \"Bendotti C\", \"De Biasi S\", \"Poletti A\"], \"date\": \"2010-09-01\"}, \"line\": 36624, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 152, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35270, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 153, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35270, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 153, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35271, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 154, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35271, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 154, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35273, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 155, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35273, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 155, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35274, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 156, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35274, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 156, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35275, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 157, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35275, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 157, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35277, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 158, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35277, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 158, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35278, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 159, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35278, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 159, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35269, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 160, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of AlzheimerÃ¢â‚¬â„¢s disease [150]. ABeta¸ is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta¸ binds to the calcium permeable nAChRs with high affinity [152]. ABeta¸42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta¸-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta¸ enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35269, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 160, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"On the contrary, X11 stabilizes ABeta¸PP conformation in membrane, inhibiting ABeta¸ secretion in cultured cells [45], likely impairing ABeta¸PP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptosis. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABeta¸PP. The expression of JIP1b stabilizes immature ABeta¸PP and decreases the ABeta¸PP ectodomain, ABeta¸40/ÃƒÅ¸42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 36359, \"annotations\": {\"Subgraph\": \"MAPK-ERK subgraph\"}, \"source\": 161, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"On the contrary, X11 stabilizes ABeta¸PP conformation in membrane, inhibiting ABeta¸ secretion in cultured cells [45], likely impairing ABeta¸PP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptosis. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABeta¸PP. The expression of JIP1b stabilizes immature ABeta¸PP and decreases the ABeta¸PP ectodomain, ABeta¸40/ÃƒÅ¸42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 36359, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 161, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"On the contrary, X11 stabilizes ABeta¸PP conformation in membrane, inhibiting ABeta¸ secretion in cultured cells [45], likely impairing ABeta¸PP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptosis. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABeta¸PP. The expression of JIP1b stabilizes immature ABeta¸PP and decreases the ABeta¸PP ectodomain, ABeta¸40/ÃƒÅ¸42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of toxicology\", \"reference\": \"22496686\", \"authors\": [\"Corsaro A\", \"Thellung S\", \"Villa V\", \"Porcile C\", \"Russo C\", \"Nizzari M\", \"Florio T\", \"Pagano A\"], \"date\": \"2012-01-01\"}, \"line\": 36359, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 161, \"target\": 4, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"These data indicate that ABCA1 and ABCG1 play a significant role in the regulation of neuronal cholesterol efflux to apoE discs and in suppression of APP processing to generate Abeta peptides.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"17121837\", \"authors\": [\"Kim WS\", \"Rahmanto AS\", \"Kamili A\", \"Rye KA\", \"Guillemin GJ\", \"Gelissen IC\", \"Jessup W\", \"Hill AF\", \"Garner B\"], \"date\": \"2007-02-02\"}, \"line\": 36544, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\"}, \"source\": 162, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Here, we report a novel function of glutamate carboxypeptidase II (GCPII) in Abeta degradation in brain, which is a peptidase involved in N-acetylaspartylglutamate cleavage, folate metabolism, and prostate tumorigenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"20624932\", \"authors\": [\"Kim MJ\", \"Chae SS\", \"Koh YH\", \"Lee SK\", \"Jo SA\"], \"date\": \"2010-11-01\"}, \"line\": 36748, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 163, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Here, we report a novel function of glutamate carboxypeptidase II (GCPII) in Abeta degradation in brain, which is a peptidase involved in N-acetylaspartylglutamate cleavage, folate metabolism, and prostate tumorigenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"reference\": \"20624932\", \"authors\": [\"Kim MJ\", \"Chae SS\", \"Koh YH\", \"Lee SK\", \"Jo SA\"], \"date\": \"2010-11-01\"}, \"line\": 36749, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 163, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of cellular and molecular medicine\", \"reference\": \"18494933\", \"authors\": [\"Esposito G\", \"Scuderi C\", \"Lu J\", \"Savani C\", \"De Filippis D\", \"Iuvone T\", \"Steardo L Jr\", \"Sheen V\", \"Steardo L\"], \"date\": \"2008-06-01\"}, \"line\": 37627, \"annotations\": {}, \"source\": 164, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of cellular and molecular medicine\", \"reference\": \"18494933\", \"authors\": [\"Esposito G\", \"Scuderi C\", \"Lu J\", \"Savani C\", \"De Filippis D\", \"Iuvone T\", \"Steardo L Jr\", \"Sheen V\", \"Steardo L\"], \"date\": \"2008-06-01\"}, \"line\": 37618, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {}, \"source\": 164, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Stromelysin-1 (MMP-3) plays a central role in activating latent-type MMPs, which are originally secreted as proenzymes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta neuropathologica\", \"reference\": \"10672313\", \"authors\": [\"Yoshiyama Y\", \"Hattori T\", \"Asahina M\"], \"date\": \"2000-02-01\"}, \"line\": 18429, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 165, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta¸ is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane ÃƒÅ¸-subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the ÃƒÅ¸-subunit, and causes autophosphorylation of the ÃƒÅ¸-subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta¸ toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3ÃƒÅ¸ and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3ÃƒÅ¸ signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta¸ activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta¸ monomers may bind to IGF-1/insulin receptors, as already shown for ABeta¸ oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 166, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta¸ is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane ÃƒÅ¸-subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the ÃƒÅ¸-subunit, and causes autophosphorylation of the ÃƒÅ¸-subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta¸ toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3ÃƒÅ¸ and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3ÃƒÅ¸ signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta¸ activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta¸ monomers may bind to IGF-1/insulin receptors, as already shown for ABeta¸ oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Regulation of cytoskeleton subgraph\"}, \"source\": 166, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta¸ is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane ÃƒÅ¸-subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the ÃƒÅ¸-subunit, and causes autophosphorylation of the ÃƒÅ¸-subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta¸ toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3ÃƒÅ¸ and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3ÃƒÅ¸ signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta¸ activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta¸ monomers may bind to IGF-1/insulin receptors, as already shown for ABeta¸ oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35051, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 166, \"target\": 8, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta¸ is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane ÃƒÅ¸-subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the ÃƒÅ¸-subunit, and causes autophosphorylation of the ÃƒÅ¸-subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta¸ toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3ÃƒÅ¸ and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3ÃƒÅ¸ signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta¸ activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta¸ monomers may bind to IGF-1/insulin receptors, as already shown for ABeta¸ oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35051, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Regulation of cytoskeleton subgraph\"}, \"source\": 166, \"target\": 8, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"The PI3K/Akt signaling: PI3K, a membrane-associated second messenger protein, and its downstream kinase, Akt, are associated with neuronal survival [111] and plasticity [112] via activation of transcription pathways and protein synthesis. PI3K pathway, which required the activation of IGF-1/insulin receptors, is the most convincing prosurvival effect of ABeta¸42 monomers. The PI3K signaling pathway is important in the transmission of survival signals in many cell types including neurons [113, 114]. The PI3K-Akt signaling cascade, initiated by IRS, is phosphorylated by stimulated insulin- andIGF-receptor tyrosine kinases [115]. One of the kinases known to lie downstream of PI3K is Akt, which can be directly activated by products of PI3K [116] by promoting its phosphorylation at Ser473 and Thr308 [117]. Activated Akt, in turn, phosphorylates a wide range of substrates activating anti-apoptotic (survival) factors and inactivating pro-apoptotic factors [114, 117]. Certain proapoptotic mediators, such as the transcription factor forkhead (FOXO), the tau kinase GSK-3ÃƒÅ¸, and the Bcl2 antagonist BAD proteins, are inactivated by Akt [118, 119]. Akt downregulates the activities of GSK-3a and GSK-3ÃƒÅ¸ by phosphorylating the former at Ser21 and the latter at Ser9 [118]. Phosphorylation/inactivation of GSK-3ÃƒÅ¸, suppresses GSK-3ÃƒÅ¸-dependent phosphorylation of tau at residues overphosphorylated in AD and prevents apoptosis of confluent cells. Treatment of cortical neurons with ABeta¸42 monomers increased Ser9 phosphorylation (inhibition) of the Akt substrate, GSK- 3ÃƒÅ¸ [97]. Inhibition of GSK-3ÃƒÅ¸ promotes cell survival through a variety of mechanisms including a reduced degradation of ÃƒÅ¸-catenin, which then translocates into the nucleus and activates the transcription of protective genes [126].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35161, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Beta-Catenin subgraph\"}, \"source\": 167, \"target\": 8, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The PI3K/Akt signaling: PI3K, a membrane-associated second messenger protein, and its downstream kinase, Akt, are associated with neuronal survival [111] and plasticity [112] via activation of transcription pathways and protein synthesis. PI3K pathway, which required the activation of IGF-1/insulin receptors, is the most convincing prosurvival effect of ABeta¸42 monomers. The PI3K signaling pathway is important in the transmission of survival signals in many cell types including neurons [113, 114]. The PI3K-Akt signaling cascade, initiated by IRS, is phosphorylated by stimulated insulin- andIGF-receptor tyrosine kinases [115]. One of the kinases known to lie downstream of PI3K is Akt, which can be directly activated by products of PI3K [116] by promoting its phosphorylation at Ser473 and Thr308 [117]. Activated Akt, in turn, phosphorylates a wide range of substrates activating anti-apoptotic (survival) factors and inactivating pro-apoptotic factors [114, 117]. Certain proapoptotic mediators, such as the transcription factor forkhead (FOXO), the tau kinase GSK-3ÃƒÅ¸, and the Bcl2 antagonist BAD proteins, are inactivated by Akt [118, 119]. Akt downregulates the activities of GSK-3a and GSK-3ÃƒÅ¸ by phosphorylating the former at Ser21 and the latter at Ser9 [118]. Phosphorylation/inactivation of GSK-3ÃƒÅ¸, suppresses GSK-3ÃƒÅ¸-dependent phosphorylation of tau at residues overphosphorylated in AD and prevents apoptosis of confluent cells. Treatment of cortical neurons with ABeta¸42 monomers increased Ser9 phosphorylation (inhibition) of the Akt substrate, GSK- 3ÃƒÅ¸ [97]. Inhibition of GSK-3ÃƒÅ¸ promotes cell survival through a variety of mechanisms including a reduced degradation of ÃƒÅ¸-catenin, which then translocates into the nucleus and activates the transcription of protective genes [126].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35161, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 167, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In situ, FRAT-2 significantly increased GSK3 beta-mediated phosphorylation of tau at a primed epitope while not significantly affecting the phosphorylation of unprimed sites.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"15522877\", \"authors\": [\"Stoothoff WH\", \"Cho JH\", \"McDonald RP\", \"Johnson GV\"], \"date\": \"2005-01-07\"}, \"line\": 29142, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 168, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In situ, FRAT-2 significantly increased GSK3 beta-mediated phosphorylation of tau at a primed epitope while not significantly affecting the phosphorylation of unprimed sites.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"15522877\", \"authors\": [\"Stoothoff WH\", \"Cho JH\", \"McDonald RP\", \"Johnson GV\"], \"date\": \"2005-01-07\"}, \"line\": 29142, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 168, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta¸ is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane ÃƒÅ¸-subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the ÃƒÅ¸-subunit, and causes autophosphorylation of the ÃƒÅ¸-subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta¸ toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3ÃƒÅ¸ and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3ÃƒÅ¸ signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta¸ activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta¸ monomers may bind to IGF-1/insulin receptors, as already shown for ABeta¸ oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35035, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 169, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta¸ is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane ÃƒÅ¸-subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the ÃƒÅ¸-subunit, and causes autophosphorylation of the ÃƒÅ¸-subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta¸ toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3ÃƒÅ¸ and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3ÃƒÅ¸ signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta¸ activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta¸ monomers may bind to IGF-1/insulin receptors, as already shown for ABeta¸ oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20847424\", \"authors\": [\"Parihar MS\", \"Brewer GJ\"], \"date\": \"2010-01-01\"}, \"line\": 35035, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Regulation of cytoskeleton subgraph\"}, \"source\": 169, \"target\": 8, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \" Here we provide evidence in AD brains of a specific up-regulation of an NF-kappaB-sensitive miRNA-146a highly complementary to the 3'-untranslated region of complement factor H (CFH), an important repressor of the inflammatory response of the brain.Transfection of HN cells using an NF-kappaB-containing pre-miRNA-146a promoter-luciferase reporter construct in stressed HN cells showed significant up-regulation of luciferase activity that paralleled decreases in CFH gene expression. Up-regulation of miRNA-146a coupled to down-regulation of CFH was observed in AD brain and in interleukin-1beta, Abeta42, and/or oxidatively stressed human neural (HN) cells in primary culture.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18801740\", \"authors\": [\"Lukiw WJ\", \"Zhao Y\", \"Cui JG\"], \"date\": \"2008-11-14\"}, \"line\": 8975, \"annotations\": {\"Subgraph\": \"Complement system subgraph\"}, \"source\": 170, \"target\": 11, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \" Here we provide evidence in AD brains of a specific up-regulation of an NF-kappaB-sensitive miRNA-146a highly complementary to the 3'-untranslated region of complement factor H (CFH), an important repressor of the inflammatory response of the brain.Transfection of HN cells using an NF-kappaB-containing pre-miRNA-146a promoter-luciferase reporter construct in stressed HN cells showed significant up-regulation of luciferase activity that paralleled decreases in CFH gene expression. Up-regulation of miRNA-146a coupled to down-regulation of CFH was observed in AD brain and in interleukin-1beta, Abeta42, and/or oxidatively stressed human neural (HN) cells in primary culture.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"18801740\", \"authors\": [\"Lukiw WJ\", \"Zhao Y\", \"Cui JG\"], \"date\": \"2008-11-14\"}, \"line\": 8975, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 170, \"target\": 11, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"MicroRNA-146a (miRNA-146a) is an inducible, 22 nucleotide, small RNA over-expressed in Alzheimer's disease (AD) brain. Up-regulated miRNA-146a targets several inflammation-related and membrane-associated messenger RNAs (mRNAs), including those encoding complement factor-H (CFH) and the interleukin-1 receptor associated kinase-1 (IRAK-1), resulting in significant decreases in their expression (p<0.05, ANOVA). In this study we assayed miRNA-146a, CFH, IRAK-1 and tetraspanin-12 (TSPAN12), abundances in primary human neuronal-glial (HNG) co-cultures, in human astroglial (HAG) and microglial (HMG) cells stressed with Abeta42 peptide and tumor necrosis factor alpha (TNFalpha). The results indicate a consistent inverse relationship between miRNA-146a and CFH, IRAK-1 and TSPAN12 expression levels, and indicate that HNG, HAG and HMG cell types each respond differently to Abeta42-peptide+TNFalpha-triggered stress. While the strongest miRNA-146a-IRAK-1 response was found in HAG cells, the largest miRNA-146a-TSPAN12 response was found in HNG cells, and the most significant miRNA-146a-CFH changes were found in HMG cells, the 'resident scavenging macrophages' of the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"21640790\", \"authors\": [\"Cui JG\", \"Lukiw WJ\", \"Pogue AI\", \"Li YY\", \"Bhattacharjee S\", \"Dua P\"], \"date\": \"2011-07-20\"}, \"line\": 9493, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 170, \"target\": 11, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"MicroRNA-146a (miRNA-146a) is an inducible, 22 nucleotide, small RNA over-expressed in Alzheimer's disease (AD) brain. Up-regulated miRNA-146a targets several inflammation-related and membrane-associated messenger RNAs (mRNAs), including those encoding complement factor-H (CFH) and the interleukin-1 receptor associated kinase-1 (IRAK-1), resulting in significant decreases in their expression (p<0.05, ANOVA). In this study we assayed miRNA-146a, CFH, IRAK-1 and tetraspanin-12 (TSPAN12), abundances in primary human neuronal-glial (HNG) co-cultures, in human astroglial (HAG) and microglial (HMG) cells stressed with Abeta42 peptide and tumor necrosis factor alpha (TNFalpha). The results indicate a consistent inverse relationship between miRNA-146a and CFH, IRAK-1 and TSPAN12 expression levels, and indicate that HNG, HAG and HMG cell types each respond differently to Abeta42-peptide+TNFalpha-triggered stress. While the strongest miRNA-146a-IRAK-1 response was found in HAG cells, the largest miRNA-146a-TSPAN12 response was found in HNG cells, and the most significant miRNA-146a-CFH changes were found in HMG cells, the 'resident scavenging macrophages' of the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"21640790\", \"authors\": [\"Cui JG\", \"Lukiw WJ\", \"Pogue AI\", \"Li YY\", \"Bhattacharjee S\", \"Dua P\"], \"date\": \"2011-07-20\"}, \"line\": 9493, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 170, \"target\": 11, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Here we provide evidence in human neural (HN) cells of an aluminum-sulfate- and reactive oxygen species (ROS)-mediated up-regulation of an NF-kappaB-sensitive miRNA-146a that down-regulates the expression of complement factor H (CFH), an important repressor of inflammation. This NF-kappaB-miRNA-146a-CFH signaling circuit is known to be similarly affected by Abeta42 peptides and in AD brain. These aluminum-sulfate-inducible events were not observed in parallel experiments using iron-, magnesium-, or zinc-sulfate-stressed HN cells. An NF-kappaB-containing miRNA-146a-promoter-luciferase reporter construct transfected into HN cells showed significant up-regulation of miRNA-146a after aluminum-sulfate treatment that corresponded to decreased CFH gene expression. These data suggest that (1) as in AD brain, NF-kappaB-sensitive, miRNA-146a-mediated, modulation of CFH gene expression may contribute to inflammatory responses in aluminum-stressed HN cells, and (2) underscores the potential of nanomolar aluminum to drive genotoxic mechanisms characteristic of neurodegenerative disease processes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of inorganic biochemistry\", \"reference\": \"19540598\", \"authors\": [\"Pogue AI\", \"Li YY\", \"Cui JG\", \"Zhao Y\", \"Kruck TP\", \"Percy ME\", \"Tarr MA\", \"Lukiw WJ\"], \"date\": \"2009-11-01\"}, \"line\": 9015, \"annotations\": {\"Subgraph\": \"Complement system subgraph\"}, \"source\": 170, \"target\": 11, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"Here we provide evidence in human neural (HN) cells of an aluminum-sulfate- and reactive oxygen species (ROS)-mediated up-regulation of an NF-kappaB-sensitive miRNA-146a that down-regulates the expression of complement factor H (CFH), an important repressor of inflammation. This NF-kappaB-miRNA-146a-CFH signaling circuit is known to be similarly affected by Abeta42 peptides and in AD brain. These aluminum-sulfate-inducible events were not observed in parallel experiments using iron-, magnesium-, or zinc-sulfate-stressed HN cells. An NF-kappaB-containing miRNA-146a-promoter-luciferase reporter construct transfected into HN cells showed significant up-regulation of miRNA-146a after aluminum-sulfate treatment that corresponded to decreased CFH gene expression. These data suggest that (1) as in AD brain, NF-kappaB-sensitive, miRNA-146a-mediated, modulation of CFH gene expression may contribute to inflammatory responses in aluminum-stressed HN cells, and (2) underscores the potential of nanomolar aluminum to drive genotoxic mechanisms characteristic of neurodegenerative disease processes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of inorganic biochemistry\", \"reference\": \"19540598\", \"authors\": [\"Pogue AI\", \"Li YY\", \"Cui JG\", \"Zhao Y\", \"Kruck TP\", \"Percy ME\", \"Tarr MA\", \"Lukiw WJ\"], \"date\": \"2009-11-01\"}, \"line\": 9015, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 170, \"target\": 11, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \" We report that infection of human primary neural cells with a high phenotypic reactivator HSV-1 (17syn+) induces upregulation of a brain-enriched microRNA (miRNA)-146a that is associated with proinflammatory signaling in stressed brain cells and Alzheimer's disease. Expression of cytoplasmic phospholipase A2, the inducible prostaglandin synthase cyclooxygenase-2, and the neuroinflammatory cytokine interleukin-1beta were each upregulated. A known miRNA-146a target in the brain, complement factor H, was downregulated. These data suggest a role for HSV-1-induced miRNA-146a in the evasion of HSV-1 from the complement system, and the activation of key elements of the arachidonic acid cascade known to contribute to Alzheimer-type neuropathological change.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroreport\", \"reference\": \"19801956\", \"authors\": [\"Hill JM\", \"Zhao Y\", \"Clement C\", \"Neumann DM\", \"Lukiw WJ\"], \"date\": \"2009-10-28\"}, \"line\": 9041, \"annotations\": {\"Subgraph\": \"Complement system subgraph\"}, \"source\": 170, \"target\": 11, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \" We report that infection of human primary neural cells with a high phenotypic reactivator HSV-1 (17syn+) induces upregulation of a brain-enriched microRNA (miRNA)-146a that is associated with proinflammatory signaling in stressed brain cells and Alzheimer's disease. Expression of cytoplasmic phospholipase A2, the inducible prostaglandin synthase cyclooxygenase-2, and the neuroinflammatory cytokine interleukin-1beta were each upregulated. A known miRNA-146a target in the brain, complement factor H, was downregulated. These data suggest a role for HSV-1-induced miRNA-146a in the evasion of HSV-1 from the complement system, and the activation of key elements of the arachidonic acid cascade known to contribute to Alzheimer-type neuropathological change.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroreport\", \"reference\": \"19801956\", \"authors\": [\"Hill JM\", \"Zhao Y\", \"Clement C\", \"Neumann DM\", \"Lukiw WJ\"], \"date\": \"2009-10-28\"}, \"line\": 9041, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 170, \"target\": 11, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \" We report that infection of human primary neural cells with a high phenotypic reactivator HSV-1 (17syn+) induces upregulation of a brain-enriched microRNA (miRNA)-146a that is associated with proinflammatory signaling in stressed brain cells and Alzheimer's disease. Expression of cytoplasmic phospholipase A2, the inducible prostaglandin synthase cyclooxygenase-2, and the neuroinflammatory cytokine interleukin-1beta were each upregulated. A known miRNA-146a target in the brain, complement factor H, was downregulated. These data suggest a role for HSV-1-induced miRNA-146a in the evasion of HSV-1 from the complement system, and the activation of key elements of the arachidonic acid cascade known to contribute to Alzheimer-type neuropathological change.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroreport\", \"reference\": \"19801956\", \"authors\": [\"Hill JM\", \"Zhao Y\", \"Clement C\", \"Neumann DM\", \"Lukiw WJ\"], \"date\": \"2009-10-28\"}, \"line\": 9037, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 170, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" We report that infection of human primary neural cells with a high phenotypic reactivator HSV-1 (17syn+) induces upregulation of a brain-enriched microRNA (miRNA)-146a that is associated with proinflammatory signaling in stressed brain cells and Alzheimer's disease. Expression of cytoplasmic phospholipase A2, the inducible prostaglandin synthase cyclooxygenase-2, and the neuroinflammatory cytokine interleukin-1beta were each upregulated. A known miRNA-146a target in the brain, complement factor H, was downregulated. These data suggest a role for HSV-1-induced miRNA-146a in the evasion of HSV-1 from the complement system, and the activation of key elements of the arachidonic acid cascade known to contribute to Alzheimer-type neuropathological change.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroreport\", \"reference\": \"19801956\", \"authors\": [\"Hill JM\", \"Zhao Y\", \"Clement C\", \"Neumann DM\", \"Lukiw WJ\"], \"date\": \"2009-10-28\"}, \"line\": 9037, \"annotations\": {\"Subgraph\": \"Prostaglandin subgraph\"}, \"source\": 170, \"target\": 12, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" We report that infection of human primary neural cells with a high phenotypic reactivator HSV-1 (17syn+) induces upregulation of a brain-enriched microRNA (miRNA)-146a that is associated with proinflammatory signaling in stressed brain cells and Alzheimer's disease. Expression of cytoplasmic phospholipase A2, the inducible prostaglandin synthase cyclooxygenase-2, and the neuroinflammatory cytokine interleukin-1beta were each upregulated. A known miRNA-146a target in the brain, complement factor H, was downregulated. These data suggest a role for HSV-1-induced miRNA-146a in the evasion of HSV-1 from the complement system, and the activation of key elements of the arachidonic acid cascade known to contribute to Alzheimer-type neuropathological change.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroreport\", \"reference\": \"19801956\", \"authors\": [\"Hill JM\", \"Zhao Y\", \"Clement C\", \"Neumann DM\", \"Lukiw WJ\"], \"date\": \"2009-10-28\"}, \"line\": 9037, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 170, \"target\": 12, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"MicroRNA-146a (miRNA-146a) is an inducible, 22 nucleotide, small RNA over-expressed in Alzheimer's disease (AD) brain. Up-regulated miRNA-146a targets several inflammation-related and membrane-associated messenger RNAs (mRNAs), including those encoding complement factor-H (CFH) and the interleukin-1 receptor associated kinase-1 (IRAK-1), resulting in significant decreases in their expression (p<0.05, ANOVA). In this study we assayed miRNA-146a, CFH, IRAK-1 and tetraspanin-12 (TSPAN12), abundances in primary human neuronal-glial (HNG) co-cultures, in human astroglial (HAG) and microglial (HMG) cells stressed with Abeta42 peptide and tumor necrosis factor alpha (TNFalpha). The results indicate a consistent inverse relationship between miRNA-146a and CFH, IRAK-1 and TSPAN12 expression levels, and indicate that HNG, HAG and HMG cell types each respond differently to Abeta42-peptide+TNFalpha-triggered stress. While the strongest miRNA-146a-IRAK-1 response was found in HAG cells, the largest miRNA-146a-TSPAN12 response was found in HNG cells, and the most significant miRNA-146a-CFH changes were found in HMG cells, the 'resident scavenging macrophages' of the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"21640790\", \"authors\": [\"Cui JG\", \"Lukiw WJ\", \"Pogue AI\", \"Li YY\", \"Bhattacharjee S\", \"Dua P\"], \"date\": \"2011-07-20\"}, \"line\": 9494, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 170, \"target\": 13, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"MicroRNA-146a (miRNA-146a) is an inducible, 22 nucleotide, small RNA over-expressed in Alzheimer's disease (AD) brain. Up-regulated miRNA-146a targets several inflammation-related and membrane-associated messenger RNAs (mRNAs), including those encoding complement factor-H (CFH) and the interleukin-1 receptor associated kinase-1 (IRAK-1), resulting in significant decreases in their expression (p<0.05, ANOVA). In this study we assayed miRNA-146a, CFH, IRAK-1 and tetraspanin-12 (TSPAN12), abundances in primary human neuronal-glial (HNG) co-cultures, in human astroglial (HAG) and microglial (HMG) cells stressed with Abeta42 peptide and tumor necrosis factor alpha (TNFalpha). The results indicate a consistent inverse relationship between miRNA-146a and CFH, IRAK-1 and TSPAN12 expression levels, and indicate that HNG, HAG and HMG cell types each respond differently to Abeta42-peptide+TNFalpha-triggered stress. While the strongest miRNA-146a-IRAK-1 response was found in HAG cells, the largest miRNA-146a-TSPAN12 response was found in HNG cells, and the most significant miRNA-146a-CFH changes were found in HMG cells, the 'resident scavenging macrophages' of the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience letters\", \"reference\": \"21640790\", \"authors\": [\"Cui JG\", \"Lukiw WJ\", \"Pogue AI\", \"Li YY\", \"Bhattacharjee S\", \"Dua P\"], \"date\": \"2011-07-20\"}, \"line\": 9494, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 170, \"target\": 13, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9366, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 170, \"target\": 13, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9366, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 170, \"target\": 13, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9368, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 170, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-κB-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1β (IL-1β) and amyloid-β-42 (Abeta42) peptide-stressed HAG cells. NF-κB-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-κB inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1β+Abeta42 peptide-stressed HAG cells and that an inducible, NF-κB-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-κB-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-κB and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-κB inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"20937840\", \"authors\": [\"Cui JG\", \"Li YY\", \"Zhao Y\", \"Bhattacharjee S\", \"Lukiw WJ\"], \"date\": \"2010-12-10\"}, \"line\": 9368, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 170, \"target\": 14, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Nuclear factor-kappaB activation regulates cyclooxygenase-2 induction in human astrocytes in response to CXCL12: role in neuronal toxicity\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"20180883\", \"authors\": [\"Alvarez S\", \"Blanco A\", \"Fresno M\", \"Muñoz-Fernández MA\"], \"date\": \"2010-05-01\"}, \"line\": 4335, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"MeSHAnatomy\": \"Astrocytes\", \"Subgraph\": \"Nuclear factor Kappa beta subgraph\"}, \"source\": 171, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Nuclear factor-kappaB activation regulates cyclooxygenase-2 induction in human astrocytes in response to CXCL12: role in neuronal toxicity\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"20180883\", \"authors\": [\"Alvarez S\", \"Blanco A\", \"Fresno M\", \"Muñoz-Fernández MA\"], \"date\": \"2010-05-01\"}, \"line\": 4335, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"MeSHAnatomy\": \"Astrocytes\", \"Subgraph\": \"Prostaglandin subgraph\"}, \"source\": 171, \"target\": 12, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Nuclear factor-kappaB activation regulates cyclooxygenase-2 induction in human astrocytes in response to CXCL12: role in neuronal toxicity\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"20180883\", \"authors\": [\"Alvarez S\", \"Blanco A\", \"Fresno M\", \"Muñoz-Fernández MA\"], \"date\": \"2010-05-01\"}, \"line\": 34029, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Nuclear factor Kappa beta subgraph\"}, \"source\": 171, \"target\": 12, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Nuclear factor-kappaB activation regulates cyclooxygenase-2 induction in human astrocytes in response to CXCL12: role in neuronal toxicity\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neurochemistry\", \"reference\": \"20180883\", \"authors\": [\"Alvarez S\", \"Blanco A\", \"Fresno M\", \"Muñoz-Fernández MA\"], \"date\": \"2010-05-01\"}, \"line\": 34029, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Prostaglandin subgraph\"}, \"source\": 171, \"target\": 12, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 23754, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 172, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24346, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 172, \"target\": 15, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24346, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 172, \"target\": 15, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"CCAAT/enhancer binding protein delta (CEBPD) elevating PTX3 expression inhibits macrophage-mediated phagocytosis of dying neuron cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"21112127\", \"authors\": [\"Ko CY\", \"Chang LH\", \"Lee YC\", \"Sterneck E\", \"Cheng CP\", \"Chen SH\", \"Huang AM\", \"Tseng JT\", \"Wang JM\"], \"date\": \"2012-02-01\"}, \"line\": 37250, \"annotations\": {\"Subgraph\": \"Innate immune system subgraph\"}, \"source\": 173, \"target\": 16, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We propose that IL-1 and the IL-6 family of cytokines regulate YKL-40 expression during sterile inflammation via both STAT3 and RelB/p50 complexes\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45245, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 174, \"target\": 33, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We propose that IL-1 and the IL-6 family of cytokines regulate YKL-40 expression during sterile inflammation via both STAT3 and RelB/p50 complexes\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45245, \"annotations\": {\"Subgraph\": \"JAK-STAT signaling subgraph\"}, \"source\": 174, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The secreted protein, YKL-40, has been proposed as a biomarker of a variety of human diseases characterized by ongoing inflammation, including chronic neurologic pathologies such as multiple sclerosis and Alzheimer's disease. However, inflammatory mediators and the molecular mechanism responsible for enhanced expression of YKL-40 remained elusive. Using several mouse models of inflammation, we now show that YKL-40 expression correlated with increased expression of both IL-1 and IL-6. Furthermore, IL-1 together with IL-6 or the IL-6 family cytokine, oncostatin M, synergistically upregulated YKL-40 expression in both primary human and mouse astrocytes in vitro. The robust cytokine-driven expression of YKL-40 in astrocytes required both STAT3 and NF-kB binding elements of the YKL-40 promoter. In addition, YKL-40 expression was enhanced by constitutively active STAT3 and inhibited by dominant-negative IkBalpha. Surprisingly, cytokine-driven expression of YKL-40 in astrocytes was independent of the p65 subunit of NF-kB and instead required subunits RelB and p50. Mechanistically, we show that IL-1-induced RelB/p50 complex formation was further promoted by oncostatin M and that these complexes directly bound to the YKL-40 promoter. Moreover, we found that expression of RelB was strongly upregulated during inflammation in vivo and by IL-1 in astrocytes in vitro. We propose that IL-1 and the IL-6 family of cytokines regulate YKL-40 expression during sterile inflammation via both STAT3 and RelB/p50 complexes. These results suggest that IL-1 may regulate the expression of specific anti-inflammatory genes in nonlymphoid tissues via the canonical activation of the RelB/p50 complexes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 42333, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 175, \"target\": 33, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The secreted protein, YKL-40, has been proposed as a biomarker of a variety of human diseases characterized by ongoing inflammation, including chronic neurologic pathologies such as multiple sclerosis and Alzheimer's disease. However, inflammatory mediators and the molecular mechanism responsible for enhanced expression of YKL-40 remained elusive. Using several mouse models of inflammation, we now show that YKL-40 expression correlated with increased expression of both IL-1 and IL-6. Furthermore, IL-1 together with IL-6 or the IL-6 family cytokine, oncostatin M, synergistically upregulated YKL-40 expression in both primary human and mouse astrocytes in vitro. The robust cytokine-driven expression of YKL-40 in astrocytes required both STAT3 and NF-kB binding elements of the YKL-40 promoter. In addition, YKL-40 expression was enhanced by constitutively active STAT3 and inhibited by dominant-negative IkBalpha. Surprisingly, cytokine-driven expression of YKL-40 in astrocytes was independent of the p65 subunit of NF-kB and instead required subunits RelB and p50. Mechanistically, we show that IL-1-induced RelB/p50 complex formation was further promoted by oncostatin M and that these complexes directly bound to the YKL-40 promoter. Moreover, we found that expression of RelB was strongly upregulated during inflammation in vivo and by IL-1 in astrocytes in vitro. We propose that IL-1 and the IL-6 family of cytokines regulate YKL-40 expression during sterile inflammation via both STAT3 and RelB/p50 complexes. These results suggest that IL-1 may regulate the expression of specific anti-inflammatory genes in nonlymphoid tissues via the canonical activation of the RelB/p50 complexes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 42333, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 175, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In addition, IL-6 and OSM moderately upregulate YKL-40 expression in human astrocytes [...] demonstrate that YKL-40 expression correlates with the expression of both IL-1β and IL-6\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45258, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 175, \"target\": 33, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In addition, IL-6 and OSM moderately upregulate YKL-40 expression in human astrocytes [...] demonstrate that YKL-40 expression correlates with the expression of both IL-1β and IL-6\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45258, \"annotations\": {\"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 175, \"target\": 33, \"key\": 3}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pbt.summary.print_summary(subgraph)\n",
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This entire procedure can be run automatically with [pbt.generation.generate_mechanism](http://pybel-tools.readthedocs.io/en/latest/generation.html#pybel_tools.generation.generate_mechanism)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Running"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "runner = cmpa.Runner(subgraph, inflamation_node, label)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "While the algorithm can be immediately run with [pbt.analysis.cmpa.Runner.run](http://pybel-tools.readthedocs.io/en/latest/cmpa.html#pybel_tools.analysis.cmpa.Runner.run), the process is outlined to show the evolution of the graph throughouts its steps. For animation purposes, the functin [pbt.analysis.cmpa.Runner.run_with_graph_transformation](http://pybel-tools.readthedocs.io/en/latest/cmpa.html#pybel_tools.analysis.cmpa.Runner.run_with_graph_transformation) yields a new `BELGraph` at each step through the process as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{('Abundance', 'CHEBI', 'leukotriene'),\n",
       " ('Abundance', 'MESHCS', 'Neurofibrillary Tangles'),\n",
       " ('Gene', 'HGNC', 'CFH'),\n",
       " ('Gene', 'HGNC', 'CHI3L1'),\n",
       " ('Gene', 'HGNC', 'IRAK1'),\n",
       " ('Gene', 'HGNC', 'IRAK2'),\n",
       " ('Gene', 'HGNC', 'PTGS2')}"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"dtzjxqobeyqvgrmp\";\n",
       "        var width = 1000;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.7.3-dev\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"description\": \"This gene encodes a cell surface receptor and transmembrane precursor protein that is cleaved by secretases to form a number of peptides. Some of these peptides are secreted and can bind to the acetyltransferase complex APBB1/TIP60 to promote transcriptional activation, while others form the protein basis of the amyloid plaques found in the brains of patients with Alzheimer disease. In addition, two of the peptides are antimicrobial peptides, having been shown to have bacteriocidal and antifungal activities. Mutations in this gene have been implicated in autosomal dominant Alzheimer disease and cerebroarterial amyloidosis (cerebral amyloid angiopathy). Multiple transcript variants encoding several different isoforms have been found for this gene. [provided by RefSeq, Aug 2014]\", \"label\": \"amyloid beta precursor protein\", \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Gliosis\", \"cname\": \"Gliosis\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"]}, {\"function\": \"Protein\", \"namespace\": \"ADO\", \"name\": \"Amyloid Beta\", \"cname\": \"Amyloid Beta\", \"id\": [\"Protein\", \"ADO\", \"Amyloid Beta\"]}, {\"function\": \"Protein\", \"namespace\": \"SFAM\", \"name\": \"MMP Family\", \"cname\": \"MMP Family\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"]}, {\"function\": \"Abundance\", \"namespace\": \"BRCO\", \"name\": \"Astrocyte\", \"cname\": \"Astrocyte\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"description\": \"The protein encoded by this gene is a serine-threonine kinase belonging to the glycogen synthase kinase subfamily. It is a negative regulator of glucose homeostasis and is involved in energy metabolism, inflammation, ER-stress, mitochondrial dysfunction, and apoptotic pathways. Defects in this gene have been associated with Parkinson disease and Alzheimer disease. [provided by RefSeq, Aug 2017]\", \"label\": \"glycogen synthase kinase 3 beta\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"description\": \"The advanced glycosylation end product (AGE) receptor encoded by this gene is a member of the immunoglobulin superfamily of cell surface receptors. It is a multiligand receptor, and besides AGE, interacts with other molecules implicated in homeostasis, development, and inflammation, and certain diseases, such as diabetes and Alzheimer's disease. Many alternatively spliced transcript variants encoding different isoforms, as well as non-protein-coding variants, have been described for this gene (PMID:18089847). [provided by RefSeq, May 2011]\", \"label\": \"advanced glycosylation end-product specific receptor\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PTX3\", \"description\": \"This gene encodes a member of the pentraxin protein family. The expression of this protein is induced by inflammatory cytokines in response to inflammatory stimuli in several mesenchymal and epithelial cell types, particularly endothelial cells and mononuclear phagocytes. The protein promotes fibrocyte differentiation and is involved in regulating inflammation and complement activation. It also plays a role in angiogenesis and tissue remodeling. The protein serves as a biomarker for several inflammatory conditions. [provided by RefSeq, Jun 2016]\", \"label\": \"pentraxin 3\", \"weight\": -1.00547039, \"cname\": \"PTX3\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL23A\", \"description\": \"This gene encodes a subunit of the heterodimeric cytokine interleukin 23 (IL23). IL23 is composed of this protein and the p40 subunit of interleukin 12 (IL12B). The receptor of IL23 is formed by the beta 1 subunit of IL12 (IL12RB1) and an IL23 specific subunit, IL23R. Both IL23 and IL12 can activate the transcription activator STAT4, and stimulate the production of interferon-gamma (IFNG). In contrast to IL12, which acts mainly on naive CD4(+) T cells, IL23 preferentially acts on memory CD4(+) T cells. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 23 subunit alpha\", \"weight\": -0.28486748, \"cname\": \"IL23A\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"description\": null, \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"description\": \"The protein encoded by this gene is a scavenger receptor that is expressed in endothelial cells. It regulates the uptake of chemically modified low density lipoproteins, including acetylated low density lipoprotein (Ac-LDL), and it may be involved in atherogenesis. This gene is regulated by the transcription factors ZNF444/EZF-2 and SP1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2013]\", \"label\": \"scavenger receptor class F member 1\", \"weight\": 0.18225965, \"cname\": \"SCARF1\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"description\": \"The protein encoded by this gene is a plasma membrane receptor for high density lipoprotein cholesterol (HDL). The encoded protein mediates cholesterol transfer to and from HDL. In addition, this protein is a receptor for hepatitis C virus glycoprotein E2. Two transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Mar 2011]\", \"label\": \"scavenger receptor class B member 1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"description\": null, \"label\": \"scavenger receptor class A member 5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"description\": \"This gene encodes a subunit of interleukin 12, a cytokine that acts on T and natural killer cells, and has a broad array of biological activities. Interleukin 12 is a disulfide-linked heterodimer composed of the 40 kD cytokine receptor like subunit encoded by this gene, and a 35 kD subunit encoded by IL12A. This cytokine is expressed by activated macrophages that serve as an essential inducer of Th1 cells development. This cytokine has been found to be important for sustaining a sufficient number of memory/effector Th1 cells to mediate long-term protection to an intracellular pathogen. Overexpression of this gene was observed in the central nervous system of patients with multiple sclerosis (MS), suggesting a role of this cytokine in the pathogenesis of the disease. The promoter polymorphism of this gene has been reported to be associated with the severity of atopic and non-atopic asthma in children. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 12B\", \"weight\": -1.61550531, \"cname\": \"IL12B\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. They recognize pathogen-associated molecular patterns that are expressed on infectious agents, and mediate the production of cytokines necessary for the development of effective immunity. The various TLRs exhibit different patterns of expression. This receptor has been implicated in signal transduction events induced by lipopolysaccharide (LPS) found in most gram-negative bacteria. Mutations in this gene have been associated with differences in LPS responsiveness. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012]\", \"label\": \"toll like receptor 4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR2\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. This protein is a cell-surface protein that can form heterodimers with other TLR family members to recognize conserved molecules derived from microorganisms known as pathogen-associated molecular patterns (PAMPs). Activation of TLRs by PAMPs leads to an up-regulation of signaling pathways to modulate the host's inflammatory response. This gene is also thought to promote apoptosis in response to bacterial lipoproteins. This gene has been implicated in the pathogenesis of several autoimmune diseases. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]\", \"label\": \"toll like receptor 2\", \"weight\": 0.13520554, \"cname\": \"TLR2\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"description\": \"The protein encoded by this gene is a serine/threonine protein kinase that is activated by autophosphorylation after binding to dsRNA. The activated form of the encoded protein can phosphorylate translation initiation factor EIF2S1, which in turn inhibits protein synthesis. This protein is also activated by manganese ions and heparin. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Oct 2011]\", \"label\": \"eukaryotic translation initiation factor 2 alpha kinase 2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CD14\", \"description\": \"The protein encoded by this gene is a surface antigen that is preferentially expressed on monocytes/macrophages. It cooperates with other proteins to mediate the innate immune response to bacterial lipopolysaccharide. Alternative splicing results in multiple transcript variants encoding the same protein. [provided by RefSeq, Mar 2010]\", \"label\": \"CD14 molecule\", \"weight\": -0.73001192, \"cname\": \"CD14\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR1\", \"description\": \"This gene encodes a G protein-coupled receptor of mammalian phagocytic cells that is a member of the G-protein coupled receptor 1 family. The protein mediates the response of phagocytic cells to invasion of the host by microorganisms and is important in host defense and inflammation.[provided by RefSeq, Jul 2010]\", \"label\": \"formyl peptide receptor 1\", \"weight\": -0.5889405, \"cname\": \"FPR1\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NOD2\", \"description\": \"This gene is a member of the Nod1/Apaf-1 family and encodes a protein with two caspase recruitment (CARD) domains and six leucine-rich repeats (LRRs). The protein is primarily expressed in the peripheral blood leukocytes. It plays a role in the immune response to intracellular bacterial lipopolysaccharides (LPS) by recognizing the muramyl dipeptide (MDP) derived from them and activating the NFKB protein. Mutations in this gene have been associated with Crohn disease and Blau syndrome. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Jun 2014]\", \"label\": \"nucleotide binding oligomerization domain containing 2\", \"weight\": -0.62084437, \"cname\": \"NOD2\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR2\", \"description\": null, \"label\": \"formyl peptide receptor 2\", \"weight\": 5.3e-05, \"cname\": \"FPR2\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MYD88\", \"description\": \"This gene encodes a cytosolic adapter protein that plays a central role in the innate and adaptive immune response. This protein functions as an essential signal transducer in the interleukin-1 and Toll-like receptor signaling pathways. These pathways regulate that activation of numerous proinflammatory genes. The encoded protein consists of an N-terminal death domain and a C-terminal Toll-interleukin1 receptor domain. Patients with defects in this gene have an increased susceptibility to pyogenic bacterial infections. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010]\", \"label\": \"myeloid differentiation primary response 88\", \"weight\": 0.981683, \"cname\": \"MYD88\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"The serum IL-1β and TNF-Î± level were significantly increased, and the expressions of TLR4 and NF-ÎºB p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of developmental neuroscience : the official journal of the International Society for Developmental Neuroscience\", \"reference\": \"21515354\", \"authors\": [\"Ma WW\", \"Xiao R\", \"Zhou X\", \"He LL\", \"Ding BJ\", \"Feng JF\", \"Yu HL\", \"Yuan LH\"], \"date\": \"2011-08-01\"}, \"line\": 15007, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"MeSHAnatomy\": \"Serum\"}, \"source\": 0, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2746, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2746, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2749, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2749, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 23756, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24348, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24348, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1192, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1216, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 11, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1217, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1218, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis : an international journal on programmed cell death\", \"reference\": \"16532272\", \"authors\": [\"Yu MS\", \"Suen KC\", \"Kwok NS\", \"So KF\", \"Hugon J\", \"Chang RC\"], \"date\": \"2006-05-01\"}, \"line\": 20045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 17, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis : an international journal on programmed cell death\", \"reference\": \"16532272\", \"authors\": [\"Yu MS\", \"Suen KC\", \"Kwok NS\", \"So KF\", \"Hugon J\", \"Chang RC\"], \"date\": \"2006-05-01\"}, \"line\": 20045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 1, \"target\": 17, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37424, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1β could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptosis in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGFβ1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGFβ2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37434, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroimmunology\", \"reference\": \"18353443\", \"authors\": [\"Choi K\", \"Zhuang H\", \"Crain B\", \"Doré S\"], \"date\": \"2008-03-01\"}, \"line\": 37319, \"annotations\": {\"Subgraph\": \"Prostaglandin subgraph\"}, \"source\": 3, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41420, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41420, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"8287928\", \"authors\": [\"Perry G\", \"Wujek JR\", \"DeWitt DA\", \"McKeon RJ\", \"Silver J\", \"Canning DR\", \"Frederickson RC\"], \"date\": \"1993-12-01\"}, \"line\": 41576, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"8287928\", \"authors\": [\"Perry G\", \"Wujek JR\", \"DeWitt DA\", \"McKeon RJ\", \"Silver J\", \"Canning DR\", \"Frederickson RC\"], \"date\": \"1993-12-01\"}, \"line\": 41576, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 33907, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 33913, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37595, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37601, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37700, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37597, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37599, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37698, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37585, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37589, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 15, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37583, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37711, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37707, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 18, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37591, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 19, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37587, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 20, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37593, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 21, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37709, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 22, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"16822591\", \"authors\": [\"Helbecque N\", \"Cottel D\", \"Hermant X\", \"Amouyel P\"], \"date\": \"2007-08-01\"}, \"line\": 18450, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20413894\", \"authors\": [\"Wang HM\", \"Zhao YX\", \"Zhang S\", \"Liu GD\", \"Kang WY\", \"Tang HD\", \"Ding JQ\", \"Chen SD\"], \"date\": \"2010-01-01\"}, \"line\": 41354, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3529, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3529, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 6121, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 6121, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34666, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34666, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"APOE subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In addition, primary human astrocytes stimulated with the RIG-1 ligand 5'ppp RNA showed increased expression of amyloid precursor protein (APP) and amyloid-β (Abeta), supporting the idea that RIG-1 is involved in the pathology of MCI associated with early progression to AD.These findings suggest that RIG-1 may play a critical role in incipient AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"24694234\", \"authors\": [\"Brand FJ 3rd\", \"Keane RW\", \"de Rivero Vaccari JP\", \"Dietrich WD\", \"Mash DC\", \"Sedaghat C\"], \"date\": \"2014-04-02\"}, \"line\": 38742, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\", \"MeSHAnatomy\": \"Astrocytes\"}, \"source\": 5, \"target\": 3, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41421, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41421, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of virology\", \"reference\": \"19710140\", \"authors\": [\"Tribouillard-Tanvier D\", \"Striebel JF\", \"Peterson KE\", \"Chesebro B\"], \"date\": \"2009-11-01\"}, \"line\": 4834, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 5, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reviews in the neurosciences\", \"reference\": \"22718609\", \"authors\": [\"Liu Y\", \"Gao C\", \"Hölscher C\", \"Li L\"], \"date\": \"2011-12-21\"}, \"line\": 11905, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reviews in the neurosciences\", \"reference\": \"22718609\", \"authors\": [\"Liu Y\", \"Gao C\", \"Hölscher C\", \"Li L\"], \"date\": \"2011-12-21\"}, \"line\": 11905, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37596, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37602, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37701, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37598, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1203, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1203, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 11, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 12, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37600, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1205, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1205, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 13, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37699, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 14, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37586, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37590, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37584, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37712, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science signaling\", \"reference\": \"23092889\", \"authors\": [\"Tang D\", \"Kang R\"], \"date\": \"2012-10-23\"}, \"line\": 37518, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37708, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37592, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 19, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37588, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 20, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37594, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 21, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37710, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 22, \"target\": 0, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.cmpa:no leaves.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "set()"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"ybjfvhurdgpoczts\";\n",
       "        var width = 1000;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.7.3-dev\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"description\": \"This gene encodes a cell surface receptor and transmembrane precursor protein that is cleaved by secretases to form a number of peptides. Some of these peptides are secreted and can bind to the acetyltransferase complex APBB1/TIP60 to promote transcriptional activation, while others form the protein basis of the amyloid plaques found in the brains of patients with Alzheimer disease. In addition, two of the peptides are antimicrobial peptides, having been shown to have bacteriocidal and antifungal activities. Mutations in this gene have been implicated in autosomal dominant Alzheimer disease and cerebroarterial amyloidosis (cerebral amyloid angiopathy). Multiple transcript variants encoding several different isoforms have been found for this gene. [provided by RefSeq, Aug 2014]\", \"label\": \"amyloid beta precursor protein\", \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Gliosis\", \"cname\": \"Gliosis\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"]}, {\"function\": \"Protein\", \"namespace\": \"ADO\", \"name\": \"Amyloid Beta\", \"cname\": \"Amyloid Beta\", \"id\": [\"Protein\", \"ADO\", \"Amyloid Beta\"]}, {\"function\": \"Protein\", \"namespace\": \"SFAM\", \"name\": \"MMP Family\", \"cname\": \"MMP Family\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"]}, {\"function\": \"Abundance\", \"namespace\": \"BRCO\", \"name\": \"Astrocyte\", \"cname\": \"Astrocyte\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"description\": \"The protein encoded by this gene is a serine-threonine kinase belonging to the glycogen synthase kinase subfamily. It is a negative regulator of glucose homeostasis and is involved in energy metabolism, inflammation, ER-stress, mitochondrial dysfunction, and apoptotic pathways. Defects in this gene have been associated with Parkinson disease and Alzheimer disease. [provided by RefSeq, Aug 2017]\", \"label\": \"glycogen synthase kinase 3 beta\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"description\": \"The advanced glycosylation end product (AGE) receptor encoded by this gene is a member of the immunoglobulin superfamily of cell surface receptors. It is a multiligand receptor, and besides AGE, interacts with other molecules implicated in homeostasis, development, and inflammation, and certain diseases, such as diabetes and Alzheimer's disease. Many alternatively spliced transcript variants encoding different isoforms, as well as non-protein-coding variants, have been described for this gene (PMID:18089847). [provided by RefSeq, May 2011]\", \"label\": \"advanced glycosylation end-product specific receptor\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PTX3\", \"description\": \"This gene encodes a member of the pentraxin protein family. The expression of this protein is induced by inflammatory cytokines in response to inflammatory stimuli in several mesenchymal and epithelial cell types, particularly endothelial cells and mononuclear phagocytes. The protein promotes fibrocyte differentiation and is involved in regulating inflammation and complement activation. It also plays a role in angiogenesis and tissue remodeling. The protein serves as a biomarker for several inflammatory conditions. [provided by RefSeq, Jun 2016]\", \"label\": \"pentraxin 3\", \"weight\": -1.00547039, \"cname\": \"PTX3\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL23A\", \"description\": \"This gene encodes a subunit of the heterodimeric cytokine interleukin 23 (IL23). IL23 is composed of this protein and the p40 subunit of interleukin 12 (IL12B). The receptor of IL23 is formed by the beta 1 subunit of IL12 (IL12RB1) and an IL23 specific subunit, IL23R. Both IL23 and IL12 can activate the transcription activator STAT4, and stimulate the production of interferon-gamma (IFNG). In contrast to IL12, which acts mainly on naive CD4(+) T cells, IL23 preferentially acts on memory CD4(+) T cells. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 23 subunit alpha\", \"weight\": -0.28486748, \"cname\": \"IL23A\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"description\": null, \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"description\": \"The protein encoded by this gene is a scavenger receptor that is expressed in endothelial cells. It regulates the uptake of chemically modified low density lipoproteins, including acetylated low density lipoprotein (Ac-LDL), and it may be involved in atherogenesis. This gene is regulated by the transcription factors ZNF444/EZF-2 and SP1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2013]\", \"label\": \"scavenger receptor class F member 1\", \"weight\": 0.18225965, \"cname\": \"SCARF1\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"description\": \"The protein encoded by this gene is a plasma membrane receptor for high density lipoprotein cholesterol (HDL). The encoded protein mediates cholesterol transfer to and from HDL. In addition, this protein is a receptor for hepatitis C virus glycoprotein E2. Two transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Mar 2011]\", \"label\": \"scavenger receptor class B member 1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"description\": null, \"label\": \"scavenger receptor class A member 5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"description\": \"This gene encodes a subunit of interleukin 12, a cytokine that acts on T and natural killer cells, and has a broad array of biological activities. Interleukin 12 is a disulfide-linked heterodimer composed of the 40 kD cytokine receptor like subunit encoded by this gene, and a 35 kD subunit encoded by IL12A. This cytokine is expressed by activated macrophages that serve as an essential inducer of Th1 cells development. This cytokine has been found to be important for sustaining a sufficient number of memory/effector Th1 cells to mediate long-term protection to an intracellular pathogen. Overexpression of this gene was observed in the central nervous system of patients with multiple sclerosis (MS), suggesting a role of this cytokine in the pathogenesis of the disease. The promoter polymorphism of this gene has been reported to be associated with the severity of atopic and non-atopic asthma in children. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 12B\", \"weight\": -1.61550531, \"cname\": \"IL12B\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. They recognize pathogen-associated molecular patterns that are expressed on infectious agents, and mediate the production of cytokines necessary for the development of effective immunity. The various TLRs exhibit different patterns of expression. This receptor has been implicated in signal transduction events induced by lipopolysaccharide (LPS) found in most gram-negative bacteria. Mutations in this gene have been associated with differences in LPS responsiveness. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012]\", \"label\": \"toll like receptor 4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR2\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. This protein is a cell-surface protein that can form heterodimers with other TLR family members to recognize conserved molecules derived from microorganisms known as pathogen-associated molecular patterns (PAMPs). Activation of TLRs by PAMPs leads to an up-regulation of signaling pathways to modulate the host's inflammatory response. This gene is also thought to promote apoptosis in response to bacterial lipoproteins. This gene has been implicated in the pathogenesis of several autoimmune diseases. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]\", \"label\": \"toll like receptor 2\", \"weight\": 0.13520554, \"cname\": \"TLR2\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"description\": \"The protein encoded by this gene is a serine/threonine protein kinase that is activated by autophosphorylation after binding to dsRNA. The activated form of the encoded protein can phosphorylate translation initiation factor EIF2S1, which in turn inhibits protein synthesis. This protein is also activated by manganese ions and heparin. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Oct 2011]\", \"label\": \"eukaryotic translation initiation factor 2 alpha kinase 2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CD14\", \"description\": \"The protein encoded by this gene is a surface antigen that is preferentially expressed on monocytes/macrophages. It cooperates with other proteins to mediate the innate immune response to bacterial lipopolysaccharide. Alternative splicing results in multiple transcript variants encoding the same protein. [provided by RefSeq, Mar 2010]\", \"label\": \"CD14 molecule\", \"weight\": -0.73001192, \"cname\": \"CD14\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR1\", \"description\": \"This gene encodes a G protein-coupled receptor of mammalian phagocytic cells that is a member of the G-protein coupled receptor 1 family. The protein mediates the response of phagocytic cells to invasion of the host by microorganisms and is important in host defense and inflammation.[provided by RefSeq, Jul 2010]\", \"label\": \"formyl peptide receptor 1\", \"weight\": -0.5889405, \"cname\": \"FPR1\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NOD2\", \"description\": \"This gene is a member of the Nod1/Apaf-1 family and encodes a protein with two caspase recruitment (CARD) domains and six leucine-rich repeats (LRRs). The protein is primarily expressed in the peripheral blood leukocytes. It plays a role in the immune response to intracellular bacterial lipopolysaccharides (LPS) by recognizing the muramyl dipeptide (MDP) derived from them and activating the NFKB protein. Mutations in this gene have been associated with Crohn disease and Blau syndrome. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Jun 2014]\", \"label\": \"nucleotide binding oligomerization domain containing 2\", \"weight\": -0.62084437, \"cname\": \"NOD2\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR2\", \"description\": null, \"label\": \"formyl peptide receptor 2\", \"weight\": 5.3e-05, \"cname\": \"FPR2\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MYD88\", \"description\": \"This gene encodes a cytosolic adapter protein that plays a central role in the innate and adaptive immune response. This protein functions as an essential signal transducer in the interleukin-1 and Toll-like receptor signaling pathways. These pathways regulate that activation of numerous proinflammatory genes. The encoded protein consists of an N-terminal death domain and a C-terminal Toll-interleukin1 receptor domain. Patients with defects in this gene have an increased susceptibility to pyogenic bacterial infections. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010]\", \"label\": \"myeloid differentiation primary response 88\", \"weight\": 0.981683, \"cname\": \"MYD88\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"The serum IL-1β and TNF-Î± level were significantly increased, and the expressions of TLR4 and NF-ÎºB p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of developmental neuroscience : the official journal of the International Society for Developmental Neuroscience\", \"reference\": \"21515354\", \"authors\": [\"Ma WW\", \"Xiao R\", \"Zhou X\", \"He LL\", \"Ding BJ\", \"Feng JF\", \"Yu HL\", \"Yuan LH\"], \"date\": \"2011-08-01\"}, \"line\": 15007, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"MeSHAnatomy\": \"Serum\"}, \"source\": 0, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2746, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2746, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2749, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2749, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 23756, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24348, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24348, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1192, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1216, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 11, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1217, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1218, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis : an international journal on programmed cell death\", \"reference\": \"16532272\", \"authors\": [\"Yu MS\", \"Suen KC\", \"Kwok NS\", \"So KF\", \"Hugon J\", \"Chang RC\"], \"date\": \"2006-05-01\"}, \"line\": 20045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 17, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis : an international journal on programmed cell death\", \"reference\": \"16532272\", \"authors\": [\"Yu MS\", \"Suen KC\", \"Kwok NS\", \"So KF\", \"Hugon J\", \"Chang RC\"], \"date\": \"2006-05-01\"}, \"line\": 20045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 1, \"target\": 17, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37424, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1β could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptosis in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGFβ1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGFβ2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37434, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroimmunology\", \"reference\": \"18353443\", \"authors\": [\"Choi K\", \"Zhuang H\", \"Crain B\", \"Doré S\"], \"date\": \"2008-03-01\"}, \"line\": 37319, \"annotations\": {\"Subgraph\": \"Prostaglandin subgraph\"}, \"source\": 3, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41420, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41420, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"8287928\", \"authors\": [\"Perry G\", \"Wujek JR\", \"DeWitt DA\", \"McKeon RJ\", \"Silver J\", \"Canning DR\", \"Frederickson RC\"], \"date\": \"1993-12-01\"}, \"line\": 41576, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"8287928\", \"authors\": [\"Perry G\", \"Wujek JR\", \"DeWitt DA\", \"McKeon RJ\", \"Silver J\", \"Canning DR\", \"Frederickson RC\"], \"date\": \"1993-12-01\"}, \"line\": 41576, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 33907, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 33913, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37595, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37601, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37700, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37597, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37599, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37698, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37585, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37589, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 15, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37583, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37711, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37707, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 18, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37591, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 19, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37587, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 20, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37593, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 21, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37709, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 22, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"16822591\", \"authors\": [\"Helbecque N\", \"Cottel D\", \"Hermant X\", \"Amouyel P\"], \"date\": \"2007-08-01\"}, \"line\": 18450, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20413894\", \"authors\": [\"Wang HM\", \"Zhao YX\", \"Zhang S\", \"Liu GD\", \"Kang WY\", \"Tang HD\", \"Ding JQ\", \"Chen SD\"], \"date\": \"2010-01-01\"}, \"line\": 41354, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3529, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3529, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 6121, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 6121, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34666, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34666, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"APOE subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In addition, primary human astrocytes stimulated with the RIG-1 ligand 5'ppp RNA showed increased expression of amyloid precursor protein (APP) and amyloid-β (Abeta), supporting the idea that RIG-1 is involved in the pathology of MCI associated with early progression to AD.These findings suggest that RIG-1 may play a critical role in incipient AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"24694234\", \"authors\": [\"Brand FJ 3rd\", \"Keane RW\", \"de Rivero Vaccari JP\", \"Dietrich WD\", \"Mash DC\", \"Sedaghat C\"], \"date\": \"2014-04-02\"}, \"line\": 38742, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\", \"MeSHAnatomy\": \"Astrocytes\"}, \"source\": 5, \"target\": 3, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41421, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41421, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of virology\", \"reference\": \"19710140\", \"authors\": [\"Tribouillard-Tanvier D\", \"Striebel JF\", \"Peterson KE\", \"Chesebro B\"], \"date\": \"2009-11-01\"}, \"line\": 4834, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 5, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reviews in the neurosciences\", \"reference\": \"22718609\", \"authors\": [\"Liu Y\", \"Gao C\", \"Hölscher C\", \"Li L\"], \"date\": \"2011-12-21\"}, \"line\": 11905, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reviews in the neurosciences\", \"reference\": \"22718609\", \"authors\": [\"Liu Y\", \"Gao C\", \"Hölscher C\", \"Li L\"], \"date\": \"2011-12-21\"}, \"line\": 11905, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37596, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37602, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37701, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37598, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1203, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1203, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 11, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 12, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37600, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1205, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1205, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 13, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37699, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 14, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37586, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37590, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37584, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37712, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science signaling\", \"reference\": \"23092889\", \"authors\": [\"Tang D\", \"Kang R\"], \"date\": \"2012-10-23\"}, \"line\": 37518, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37708, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37592, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 19, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37588, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 20, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37594, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 21, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37710, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 22, \"target\": 0, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.cmpa:no leaves.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "set()"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"kvgqyqdpahbmtsjx\";\n",
       "        var width = 1000;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.7.3-dev\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"description\": \"This gene encodes a cell surface receptor and transmembrane precursor protein that is cleaved by secretases to form a number of peptides. Some of these peptides are secreted and can bind to the acetyltransferase complex APBB1/TIP60 to promote transcriptional activation, while others form the protein basis of the amyloid plaques found in the brains of patients with Alzheimer disease. In addition, two of the peptides are antimicrobial peptides, having been shown to have bacteriocidal and antifungal activities. Mutations in this gene have been implicated in autosomal dominant Alzheimer disease and cerebroarterial amyloidosis (cerebral amyloid angiopathy). Multiple transcript variants encoding several different isoforms have been found for this gene. [provided by RefSeq, Aug 2014]\", \"label\": \"amyloid beta precursor protein\", \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Gliosis\", \"cname\": \"Gliosis\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"]}, {\"function\": \"Protein\", \"namespace\": \"ADO\", \"name\": \"Amyloid Beta\", \"cname\": \"Amyloid Beta\", \"id\": [\"Protein\", \"ADO\", \"Amyloid Beta\"]}, {\"function\": \"Protein\", \"namespace\": \"SFAM\", \"name\": \"MMP Family\", \"cname\": \"MMP Family\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"]}, {\"function\": \"Abundance\", \"namespace\": \"BRCO\", \"name\": \"Astrocyte\", \"cname\": \"Astrocyte\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"description\": \"The protein encoded by this gene is a serine-threonine kinase belonging to the glycogen synthase kinase subfamily. It is a negative regulator of glucose homeostasis and is involved in energy metabolism, inflammation, ER-stress, mitochondrial dysfunction, and apoptotic pathways. Defects in this gene have been associated with Parkinson disease and Alzheimer disease. [provided by RefSeq, Aug 2017]\", \"label\": \"glycogen synthase kinase 3 beta\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"description\": \"The advanced glycosylation end product (AGE) receptor encoded by this gene is a member of the immunoglobulin superfamily of cell surface receptors. It is a multiligand receptor, and besides AGE, interacts with other molecules implicated in homeostasis, development, and inflammation, and certain diseases, such as diabetes and Alzheimer's disease. Many alternatively spliced transcript variants encoding different isoforms, as well as non-protein-coding variants, have been described for this gene (PMID:18089847). [provided by RefSeq, May 2011]\", \"label\": \"advanced glycosylation end-product specific receptor\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PTX3\", \"description\": \"This gene encodes a member of the pentraxin protein family. The expression of this protein is induced by inflammatory cytokines in response to inflammatory stimuli in several mesenchymal and epithelial cell types, particularly endothelial cells and mononuclear phagocytes. The protein promotes fibrocyte differentiation and is involved in regulating inflammation and complement activation. It also plays a role in angiogenesis and tissue remodeling. The protein serves as a biomarker for several inflammatory conditions. [provided by RefSeq, Jun 2016]\", \"label\": \"pentraxin 3\", \"weight\": -1.00547039, \"cname\": \"PTX3\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL23A\", \"description\": \"This gene encodes a subunit of the heterodimeric cytokine interleukin 23 (IL23). IL23 is composed of this protein and the p40 subunit of interleukin 12 (IL12B). The receptor of IL23 is formed by the beta 1 subunit of IL12 (IL12RB1) and an IL23 specific subunit, IL23R. Both IL23 and IL12 can activate the transcription activator STAT4, and stimulate the production of interferon-gamma (IFNG). In contrast to IL12, which acts mainly on naive CD4(+) T cells, IL23 preferentially acts on memory CD4(+) T cells. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 23 subunit alpha\", \"weight\": -0.28486748, \"cname\": \"IL23A\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"description\": null, \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"description\": \"The protein encoded by this gene is a scavenger receptor that is expressed in endothelial cells. It regulates the uptake of chemically modified low density lipoproteins, including acetylated low density lipoprotein (Ac-LDL), and it may be involved in atherogenesis. This gene is regulated by the transcription factors ZNF444/EZF-2 and SP1. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Apr 2013]\", \"label\": \"scavenger receptor class F member 1\", \"weight\": 0.18225965, \"cname\": \"SCARF1\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"description\": \"The protein encoded by this gene is a plasma membrane receptor for high density lipoprotein cholesterol (HDL). The encoded protein mediates cholesterol transfer to and from HDL. In addition, this protein is a receptor for hepatitis C virus glycoprotein E2. Two transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Mar 2011]\", \"label\": \"scavenger receptor class B member 1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"description\": null, \"label\": \"scavenger receptor class A member 5\", \"weight\": 0.49422828, \"cname\": \"SCARA5\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"description\": \"This gene encodes a subunit of interleukin 12, a cytokine that acts on T and natural killer cells, and has a broad array of biological activities. Interleukin 12 is a disulfide-linked heterodimer composed of the 40 kD cytokine receptor like subunit encoded by this gene, and a 35 kD subunit encoded by IL12A. This cytokine is expressed by activated macrophages that serve as an essential inducer of Th1 cells development. This cytokine has been found to be important for sustaining a sufficient number of memory/effector Th1 cells to mediate long-term protection to an intracellular pathogen. Overexpression of this gene was observed in the central nervous system of patients with multiple sclerosis (MS), suggesting a role of this cytokine in the pathogenesis of the disease. The promoter polymorphism of this gene has been reported to be associated with the severity of atopic and non-atopic asthma in children. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 12B\", \"weight\": -1.61550531, \"cname\": \"IL12B\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. They recognize pathogen-associated molecular patterns that are expressed on infectious agents, and mediate the production of cytokines necessary for the development of effective immunity. The various TLRs exhibit different patterns of expression. This receptor has been implicated in signal transduction events induced by lipopolysaccharide (LPS) found in most gram-negative bacteria. Mutations in this gene have been associated with differences in LPS responsiveness. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012]\", \"label\": \"toll like receptor 4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR2\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. This protein is a cell-surface protein that can form heterodimers with other TLR family members to recognize conserved molecules derived from microorganisms known as pathogen-associated molecular patterns (PAMPs). Activation of TLRs by PAMPs leads to an up-regulation of signaling pathways to modulate the host's inflammatory response. This gene is also thought to promote apoptosis in response to bacterial lipoproteins. This gene has been implicated in the pathogenesis of several autoimmune diseases. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]\", \"label\": \"toll like receptor 2\", \"weight\": 0.13520554, \"cname\": \"TLR2\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"description\": \"The protein encoded by this gene is a serine/threonine protein kinase that is activated by autophosphorylation after binding to dsRNA. The activated form of the encoded protein can phosphorylate translation initiation factor EIF2S1, which in turn inhibits protein synthesis. This protein is also activated by manganese ions and heparin. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Oct 2011]\", \"label\": \"eukaryotic translation initiation factor 2 alpha kinase 2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CD14\", \"description\": \"The protein encoded by this gene is a surface antigen that is preferentially expressed on monocytes/macrophages. It cooperates with other proteins to mediate the innate immune response to bacterial lipopolysaccharide. Alternative splicing results in multiple transcript variants encoding the same protein. [provided by RefSeq, Mar 2010]\", \"label\": \"CD14 molecule\", \"weight\": -0.73001192, \"cname\": \"CD14\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR1\", \"description\": \"This gene encodes a G protein-coupled receptor of mammalian phagocytic cells that is a member of the G-protein coupled receptor 1 family. The protein mediates the response of phagocytic cells to invasion of the host by microorganisms and is important in host defense and inflammation.[provided by RefSeq, Jul 2010]\", \"label\": \"formyl peptide receptor 1\", \"weight\": -0.5889405, \"cname\": \"FPR1\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NOD2\", \"description\": \"This gene is a member of the Nod1/Apaf-1 family and encodes a protein with two caspase recruitment (CARD) domains and six leucine-rich repeats (LRRs). The protein is primarily expressed in the peripheral blood leukocytes. It plays a role in the immune response to intracellular bacterial lipopolysaccharides (LPS) by recognizing the muramyl dipeptide (MDP) derived from them and activating the NFKB protein. Mutations in this gene have been associated with Crohn disease and Blau syndrome. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Jun 2014]\", \"label\": \"nucleotide binding oligomerization domain containing 2\", \"weight\": -0.62084437, \"cname\": \"NOD2\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR2\", \"description\": null, \"label\": \"formyl peptide receptor 2\", \"weight\": 5.3e-05, \"cname\": \"FPR2\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MYD88\", \"description\": \"This gene encodes a cytosolic adapter protein that plays a central role in the innate and adaptive immune response. This protein functions as an essential signal transducer in the interleukin-1 and Toll-like receptor signaling pathways. These pathways regulate that activation of numerous proinflammatory genes. The encoded protein consists of an N-terminal death domain and a C-terminal Toll-interleukin1 receptor domain. Patients with defects in this gene have an increased susceptibility to pyogenic bacterial infections. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010]\", \"label\": \"myeloid differentiation primary response 88\", \"weight\": 0.981683, \"cname\": \"MYD88\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"The serum IL-1β and TNF-Î± level were significantly increased, and the expressions of TLR4 and NF-ÎºB p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of developmental neuroscience : the official journal of the International Society for Developmental Neuroscience\", \"reference\": \"21515354\", \"authors\": [\"Ma WW\", \"Xiao R\", \"Zhou X\", \"He LL\", \"Ding BJ\", \"Feng JF\", \"Yu HL\", \"Yuan LH\"], \"date\": \"2011-08-01\"}, \"line\": 15007, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"MeSHAnatomy\": \"Serum\"}, \"source\": 0, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2746, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2746, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2749, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2749, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 23756, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24348, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24348, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1192, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1217, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1218, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis : an international journal on programmed cell death\", \"reference\": \"16532272\", \"authors\": [\"Yu MS\", \"Suen KC\", \"Kwok NS\", \"So KF\", \"Hugon J\", \"Chang RC\"], \"date\": \"2006-05-01\"}, \"line\": 20045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 17, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis : an international journal on programmed cell death\", \"reference\": \"16532272\", \"authors\": [\"Yu MS\", \"Suen KC\", \"Kwok NS\", \"So KF\", \"Hugon J\", \"Chang RC\"], \"date\": \"2006-05-01\"}, \"line\": 20045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 1, \"target\": 17, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37424, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1β could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptosis in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGFβ1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGFβ2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37434, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroimmunology\", \"reference\": \"18353443\", \"authors\": [\"Choi K\", \"Zhuang H\", \"Crain B\", \"Doré S\"], \"date\": \"2008-03-01\"}, \"line\": 37319, \"annotations\": {\"Subgraph\": \"Prostaglandin subgraph\"}, \"source\": 3, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41420, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41420, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"8287928\", \"authors\": [\"Perry G\", \"Wujek JR\", \"DeWitt DA\", \"McKeon RJ\", \"Silver J\", \"Canning DR\", \"Frederickson RC\"], \"date\": \"1993-12-01\"}, \"line\": 41576, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"8287928\", \"authors\": [\"Perry G\", \"Wujek JR\", \"DeWitt DA\", \"McKeon RJ\", \"Silver J\", \"Canning DR\", \"Frederickson RC\"], \"date\": \"1993-12-01\"}, \"line\": 41576, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 33907, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 33913, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37595, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37601, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37700, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37597, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37599, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37698, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37585, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37589, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 15, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37583, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37711, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37707, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 18, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37591, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 19, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37587, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 20, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37593, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 21, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37709, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 22, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"16822591\", \"authors\": [\"Helbecque N\", \"Cottel D\", \"Hermant X\", \"Amouyel P\"], \"date\": \"2007-08-01\"}, \"line\": 18450, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20413894\", \"authors\": [\"Wang HM\", \"Zhao YX\", \"Zhang S\", \"Liu GD\", \"Kang WY\", \"Tang HD\", \"Ding JQ\", \"Chen SD\"], \"date\": \"2010-01-01\"}, \"line\": 41354, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3529, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3529, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 6121, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 6121, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34666, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34666, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"APOE subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In addition, primary human astrocytes stimulated with the RIG-1 ligand 5'ppp RNA showed increased expression of amyloid precursor protein (APP) and amyloid-β (Abeta), supporting the idea that RIG-1 is involved in the pathology of MCI associated with early progression to AD.These findings suggest that RIG-1 may play a critical role in incipient AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"24694234\", \"authors\": [\"Brand FJ 3rd\", \"Keane RW\", \"de Rivero Vaccari JP\", \"Dietrich WD\", \"Mash DC\", \"Sedaghat C\"], \"date\": \"2014-04-02\"}, \"line\": 38742, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\", \"MeSHAnatomy\": \"Astrocytes\"}, \"source\": 5, \"target\": 3, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41421, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41421, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of virology\", \"reference\": \"19710140\", \"authors\": [\"Tribouillard-Tanvier D\", \"Striebel JF\", \"Peterson KE\", \"Chesebro B\"], \"date\": \"2009-11-01\"}, \"line\": 4834, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 5, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reviews in the neurosciences\", \"reference\": \"22718609\", \"authors\": [\"Liu Y\", \"Gao C\", \"Hölscher C\", \"Li L\"], \"date\": \"2011-12-21\"}, \"line\": 11905, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reviews in the neurosciences\", \"reference\": \"22718609\", \"authors\": [\"Liu Y\", \"Gao C\", \"Hölscher C\", \"Li L\"], \"date\": \"2011-12-21\"}, \"line\": 11905, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37596, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37602, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37701, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37598, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1203, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1203, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 11, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 12, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37600, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1205, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1205, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 13, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37699, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 14, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37586, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37590, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37584, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37712, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science signaling\", \"reference\": \"23092889\", \"authors\": [\"Tang D\", \"Kang R\"], \"date\": \"2012-10-23\"}, \"line\": 37518, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37708, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37592, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 19, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37588, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 20, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37594, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 21, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37710, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 22, \"target\": 0, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.remove_random_edge()\n",
    "\n",
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{('Gene', 'HGNC', 'SCARF1')}"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.cmpa:no leaves.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "set()"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.remove_random_edge()\n",
    "leaves = runner.score_leaves()\n",
    "len(leaves)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"gzcpevkqtuqsfyjh\";\n",
       "        var width = 1000;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.7.3-dev\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"description\": \"This gene encodes a cell surface receptor and transmembrane precursor protein that is cleaved by secretases to form a number of peptides. Some of these peptides are secreted and can bind to the acetyltransferase complex APBB1/TIP60 to promote transcriptional activation, while others form the protein basis of the amyloid plaques found in the brains of patients with Alzheimer disease. In addition, two of the peptides are antimicrobial peptides, having been shown to have bacteriocidal and antifungal activities. Mutations in this gene have been implicated in autosomal dominant Alzheimer disease and cerebroarterial amyloidosis (cerebral amyloid angiopathy). Multiple transcript variants encoding several different isoforms have been found for this gene. [provided by RefSeq, Aug 2014]\", \"label\": \"amyloid beta precursor protein\", \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Gliosis\", \"cname\": \"Gliosis\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"]}, {\"function\": \"Protein\", \"namespace\": \"ADO\", \"name\": \"Amyloid Beta\", \"cname\": \"Amyloid Beta\", \"id\": [\"Protein\", \"ADO\", \"Amyloid Beta\"]}, {\"function\": \"Protein\", \"namespace\": \"SFAM\", \"name\": \"MMP Family\", \"cname\": \"MMP Family\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"]}, {\"function\": \"Abundance\", \"namespace\": \"BRCO\", \"name\": \"Astrocyte\", \"cname\": \"Astrocyte\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"description\": \"The protein encoded by this gene is a serine-threonine kinase belonging to the glycogen synthase kinase subfamily. It is a negative regulator of glucose homeostasis and is involved in energy metabolism, inflammation, ER-stress, mitochondrial dysfunction, and apoptotic pathways. Defects in this gene have been associated with Parkinson disease and Alzheimer disease. [provided by RefSeq, Aug 2017]\", \"label\": \"glycogen synthase kinase 3 beta\", \"weight\": 0.32860467, \"cname\": \"GSK3B\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"description\": \"The advanced glycosylation end product (AGE) receptor encoded by this gene is a member of the immunoglobulin superfamily of cell surface receptors. It is a multiligand receptor, and besides AGE, interacts with other molecules implicated in homeostasis, development, and inflammation, and certain diseases, such as diabetes and Alzheimer's disease. Many alternatively spliced transcript variants encoding different isoforms, as well as non-protein-coding variants, have been described for this gene (PMID:18089847). [provided by RefSeq, May 2011]\", \"label\": \"advanced glycosylation end-product specific receptor\", \"weight\": -0.52367838, \"cname\": \"AGER\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PTX3\", \"description\": \"This gene encodes a member of the pentraxin protein family. The expression of this protein is induced by inflammatory cytokines in response to inflammatory stimuli in several mesenchymal and epithelial cell types, particularly endothelial cells and mononuclear phagocytes. The protein promotes fibrocyte differentiation and is involved in regulating inflammation and complement activation. It also plays a role in angiogenesis and tissue remodeling. The protein serves as a biomarker for several inflammatory conditions. [provided by RefSeq, Jun 2016]\", \"label\": \"pentraxin 3\", \"weight\": -1.00547039, \"cname\": \"PTX3\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL23A\", \"description\": \"This gene encodes a subunit of the heterodimeric cytokine interleukin 23 (IL23). IL23 is composed of this protein and the p40 subunit of interleukin 12 (IL12B). The receptor of IL23 is formed by the beta 1 subunit of IL12 (IL12RB1) and an IL23 specific subunit, IL23R. Both IL23 and IL12 can activate the transcription activator STAT4, and stimulate the production of interferon-gamma (IFNG). In contrast to IL12, which acts mainly on naive CD4(+) T cells, IL23 preferentially acts on memory CD4(+) T cells. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 23 subunit alpha\", \"weight\": -0.28486748, \"cname\": \"IL23A\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"description\": null, \"weight\": 0.60114332, \"cname\": \"MSR1\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"description\": \"The protein encoded by this gene is a plasma membrane receptor for high density lipoprotein cholesterol (HDL). The encoded protein mediates cholesterol transfer to and from HDL. In addition, this protein is a receptor for hepatitis C virus glycoprotein E2. Two transcript variants encoding different isoforms have been found for this gene.[provided by RefSeq, Mar 2011]\", \"label\": \"scavenger receptor class B member 1\", \"weight\": -0.36691523, \"cname\": \"SCARB1\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"description\": \"This gene encodes a subunit of interleukin 12, a cytokine that acts on T and natural killer cells, and has a broad array of biological activities. Interleukin 12 is a disulfide-linked heterodimer composed of the 40 kD cytokine receptor like subunit encoded by this gene, and a 35 kD subunit encoded by IL12A. This cytokine is expressed by activated macrophages that serve as an essential inducer of Th1 cells development. This cytokine has been found to be important for sustaining a sufficient number of memory/effector Th1 cells to mediate long-term protection to an intracellular pathogen. Overexpression of this gene was observed in the central nervous system of patients with multiple sclerosis (MS), suggesting a role of this cytokine in the pathogenesis of the disease. The promoter polymorphism of this gene has been reported to be associated with the severity of atopic and non-atopic asthma in children. [provided by RefSeq, Jul 2008]\", \"label\": \"interleukin 12B\", \"weight\": -1.61550531, \"cname\": \"IL12B\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. They recognize pathogen-associated molecular patterns that are expressed on infectious agents, and mediate the production of cytokines necessary for the development of effective immunity. The various TLRs exhibit different patterns of expression. This receptor has been implicated in signal transduction events induced by lipopolysaccharide (LPS) found in most gram-negative bacteria. Mutations in this gene have been associated with differences in LPS responsiveness. Multiple transcript variants encoding different isoforms have been found for this gene. [provided by RefSeq, Jan 2012]\", \"label\": \"toll like receptor 4\", \"weight\": 0.11823175, \"cname\": \"TLR4\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TLR2\", \"description\": \"The protein encoded by this gene is a member of the Toll-like receptor (TLR) family which plays a fundamental role in pathogen recognition and activation of innate immunity. TLRs are highly conserved from Drosophila to humans and share structural and functional similarities. This protein is a cell-surface protein that can form heterodimers with other TLR family members to recognize conserved molecules derived from microorganisms known as pathogen-associated molecular patterns (PAMPs). Activation of TLRs by PAMPs leads to an up-regulation of signaling pathways to modulate the host's inflammatory response. This gene is also thought to promote apoptosis in response to bacterial lipoproteins. This gene has been implicated in the pathogenesis of several autoimmune diseases. Alternative splicing results in multiple transcript variants. [provided by RefSeq, Jan 2016]\", \"label\": \"toll like receptor 2\", \"weight\": 0.13520554, \"cname\": \"TLR2\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"description\": \"The protein encoded by this gene is a serine/threonine protein kinase that is activated by autophosphorylation after binding to dsRNA. The activated form of the encoded protein can phosphorylate translation initiation factor EIF2S1, which in turn inhibits protein synthesis. This protein is also activated by manganese ions and heparin. Three transcript variants encoding two different isoforms have been found for this gene. [provided by RefSeq, Oct 2011]\", \"label\": \"eukaryotic translation initiation factor 2 alpha kinase 2\", \"weight\": 0.47307381, \"cname\": \"EIF2AK2\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CD14\", \"description\": \"The protein encoded by this gene is a surface antigen that is preferentially expressed on monocytes/macrophages. It cooperates with other proteins to mediate the innate immune response to bacterial lipopolysaccharide. Alternative splicing results in multiple transcript variants encoding the same protein. [provided by RefSeq, Mar 2010]\", \"label\": \"CD14 molecule\", \"weight\": -0.73001192, \"cname\": \"CD14\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR1\", \"description\": \"This gene encodes a G protein-coupled receptor of mammalian phagocytic cells that is a member of the G-protein coupled receptor 1 family. The protein mediates the response of phagocytic cells to invasion of the host by microorganisms and is important in host defense and inflammation.[provided by RefSeq, Jul 2010]\", \"label\": \"formyl peptide receptor 1\", \"weight\": -0.5889405, \"cname\": \"FPR1\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"NOD2\", \"description\": \"This gene is a member of the Nod1/Apaf-1 family and encodes a protein with two caspase recruitment (CARD) domains and six leucine-rich repeats (LRRs). The protein is primarily expressed in the peripheral blood leukocytes. It plays a role in the immune response to intracellular bacterial lipopolysaccharides (LPS) by recognizing the muramyl dipeptide (MDP) derived from them and activating the NFKB protein. Mutations in this gene have been associated with Crohn disease and Blau syndrome. Alternatively spliced transcript variants encoding distinct isoforms have been found for this gene. [provided by RefSeq, Jun 2014]\", \"label\": \"nucleotide binding oligomerization domain containing 2\", \"weight\": -0.62084437, \"cname\": \"NOD2\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"FPR2\", \"description\": null, \"label\": \"formyl peptide receptor 2\", \"weight\": 5.3e-05, \"cname\": \"FPR2\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MYD88\", \"description\": \"This gene encodes a cytosolic adapter protein that plays a central role in the innate and adaptive immune response. This protein functions as an essential signal transducer in the interleukin-1 and Toll-like receptor signaling pathways. These pathways regulate that activation of numerous proinflammatory genes. The encoded protein consists of an N-terminal death domain and a C-terminal Toll-interleukin1 receptor domain. Patients with defects in this gene have an increased susceptibility to pyogenic bacterial infections. Alternate splicing results in multiple transcript variants. [provided by RefSeq, Feb 2010]\", \"label\": \"myeloid differentiation primary response 88\", \"weight\": 0.981683, \"cname\": \"MYD88\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"The serum IL-1β and TNF-Î± level were significantly increased, and the expressions of TLR4 and NF-ÎºB p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"International journal of developmental neuroscience : the official journal of the International Society for Developmental Neuroscience\", \"reference\": \"21515354\", \"authors\": [\"Ma WW\", \"Xiao R\", \"Zhou X\", \"He LL\", \"Ding BJ\", \"Feng JF\", \"Yu HL\", \"Yuan LH\"], \"date\": \"2011-08-01\"}, \"line\": 15007, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"MeSHAnatomy\": \"Serum\"}, \"source\": 0, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"10222124\", \"authors\": [\"Parker TA\", \"Paris D\", \"Town T\", \"Humphrey J\", \"Tan J\", \"Mullan M\", \"Crawford F\"], \"date\": \"1999-05-01\"}, \"line\": 12568, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of immunology (Baltimore, Md. : 1950)\", \"reference\": \"25681350\", \"authors\": [\"Surace MJ\", \"Waters MR\", \"Biswas DD\", \"Hauser KF\", \"Singh SK\", \"Yester JW\", \"Yao Z\", \"Bhardwaj R\", \"Boyce BF\", \"Kordula T\"], \"date\": \"2015-03-15\"}, \"line\": 45369, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56005, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 1, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2746, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2746, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2749, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 2749, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"16914869\", \"authors\": [\"Takashima A\"], \"date\": \"2006-01-01\"}, \"line\": 31381, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 1, \"target\": 6, \"key\": 6}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 23756, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24348, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current medicinal chemistry\", \"reference\": \"16842191\", \"authors\": [\"Geroldi D\", \"Falcone C\", \"Emanuele E\"], \"date\": \"2006-01-01\"}, \"line\": 24348, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 1, \"target\": 7, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1192, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1217, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 11, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis : an international journal on programmed cell death\", \"reference\": \"16532272\", \"authors\": [\"Yu MS\", \"Suen KC\", \"Kwok NS\", \"So KF\", \"Hugon J\", \"Chang RC\"], \"date\": \"2006-05-01\"}, \"line\": 20045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 1, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis : an international journal on programmed cell death\", \"reference\": \"16532272\", \"authors\": [\"Yu MS\", \"Suen KC\", \"Kwok NS\", \"So KF\", \"Hugon J\", \"Chang RC\"], \"date\": \"2006-05-01\"}, \"line\": 20045, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 1, \"target\": 15, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37424, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1β could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptosis in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGFβ1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGFβ2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Current Alzheimer research\", \"reference\": \"21143158\", \"authors\": [\"Li C\", \"Zhao R\", \"Gao K\", \"Wei Z\", \"Yin MY\", \"Lau LT\", \"Chui D\", \"Yu AC\"], \"date\": \"2011-02-01\"}, \"line\": 37434, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroimmunology\", \"reference\": \"18353443\", \"authors\": [\"Choi K\", \"Zhuang H\", \"Crain B\", \"Doré S\"], \"date\": \"2008-03-01\"}, \"line\": 37319, \"annotations\": {\"Subgraph\": \"Prostaglandin subgraph\"}, \"source\": 3, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41420, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41420, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"8287928\", \"authors\": [\"Perry G\", \"Wujek JR\", \"DeWitt DA\", \"McKeon RJ\", \"Silver J\", \"Canning DR\", \"Frederickson RC\"], \"date\": \"1993-12-01\"}, \"line\": 41576, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Experimental neurology\", \"reference\": \"8287928\", \"authors\": [\"Perry G\", \"Wujek JR\", \"DeWitt DA\", \"McKeon RJ\", \"Silver J\", \"Canning DR\", \"Frederickson RC\"], \"date\": \"1993-12-01\"}, \"line\": 41576, \"annotations\": {\"Subgraph\": \"Autophagy signaling subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 33907, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of disease\", \"reference\": \"21945540\", \"authors\": [\"Hudry E\", \"Hyman BT\", \"Rodríguez M\", \"Gómez-Isla T\", \"Sánchez MB\", \"Barroeta I\", \"Serenó L\", \"Hashimoto T\", \"Coma M\", \"Fan Z\", \"Sánchez-Ferrer P\", \"Siao M\", \"Pérez-Nievas BG\", \"Scotton TC\", \"DaRocha-Souto B\"], \"date\": \"2012-01-01\"}, \"line\": 33913, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37595, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37601, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37700, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37597, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37599, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 11, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37698, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37585, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37589, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 13, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37583, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37711, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 14, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37707, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37591, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 17, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37587, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 18, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37593, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 19, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37709, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 20, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BioMed research international\", \"reference\": \"24455696\", \"authors\": [\"Hsieh HL\", \"Yang CM\"], \"date\": \"2013-01-01\"}, \"line\": 38532, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of aging\", \"reference\": \"16822591\", \"authors\": [\"Helbecque N\", \"Cottel D\", \"Hermant X\", \"Amouyel P\"], \"date\": \"2007-08-01\"}, \"line\": 18450, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid-β (Aβ). It has also been shown that Aβ enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"23792694\", \"authors\": [\"Barcikowska M\", \"Groblewska M\", \"Mroczko B\"], \"date\": \"2013-01-01\"}, \"line\": 56007, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Matrix metalloproteinase subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Alzheimer's disease : JAD\", \"reference\": \"20413894\", \"authors\": [\"Wang HM\", \"Zhao YX\", \"Zhang S\", \"Liu GD\", \"Kang WY\", \"Tang HD\", \"Ding JQ\", \"Chen SD\"], \"date\": \"2010-01-01\"}, \"line\": 41354, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3529, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of learning and memory\", \"reference\": \"19394434\", \"authors\": [\"Lee YK\", \"Kwak DH\", \"Oh KW\", \"Nam SY\", \"Lee BJ\", \"Yun YW\", \"Kim YB\", \"Han SB\", \"Hong JT\"], \"date\": \"2009-10-01\"}, \"line\": 3529, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 6121, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"β-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. β-Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 6121, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34666, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Low density lipoprotein subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta¸ may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta¸ by astrocytes. Deletion of LDLR caused a decrease in ABeta¸ uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta¸. Increasing LDLR levels also enhanced the cellular degradation of ABeta¸ and facilitated the vesicular transport of ABeta¸ to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta¸ uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta¸ can directly bind to LDLR, suggesting an interaction between LDLR and ABeta¸ could be responsible for LDLR-mediated ABeta¸ uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta¸ uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta¸ degradation within the brain\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The Journal of biological chemistry\", \"reference\": \"22383525\", \"authors\": [\"Holtzman DM\", \"Yoon H\", \"Verghese PB\", \"Basak JM\", \"Kim J\"], \"date\": \"2012-04-20\"}, \"line\": 34666, \"object\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"APOE subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In addition, primary human astrocytes stimulated with the RIG-1 ligand 5'ppp RNA showed increased expression of amyloid precursor protein (APP) and amyloid-β (Abeta), supporting the idea that RIG-1 is involved in the pathology of MCI associated with early progression to AD.These findings suggest that RIG-1 may play a critical role in incipient AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"24694234\", \"authors\": [\"Brand FJ 3rd\", \"Keane RW\", \"de Rivero Vaccari JP\", \"Dietrich WD\", \"Mash DC\", \"Sedaghat C\"], \"date\": \"2014-04-02\"}, \"line\": 38742, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\", \"MeSHAnatomy\": \"Astrocytes\"}, \"source\": 5, \"target\": 3, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41421, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Cytokines including TNF-Î±+IFN-γ increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and β-secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of neuroinflammation\", \"reference\": \"22047170\", \"authors\": [\"O'Connor T\", \"Zhao J\", \"Vassar R\"], \"date\": \"2011-11-02\"}, \"line\": 41421, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 5, \"target\": 3, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of virology\", \"reference\": \"19710140\", \"authors\": [\"Tribouillard-Tanvier D\", \"Striebel JF\", \"Peterson KE\", \"Chesebro B\"], \"date\": \"2009-11-01\"}, \"line\": 4834, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 5, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 5989, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"The American journal of pathology\", \"reference\": \"20056834\", \"authors\": [\"Ramirez SH\", \"Fan S\", \"Zhang M\", \"Papugani A\", \"Reichenbach N\", \"Dykstra H\", \"Mercer AJ\", \"Tuma RF\", \"Persidsky Y\"], \"date\": \"2010-02-01\"}, \"line\": 6001, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reviews in the neurosciences\", \"reference\": \"22718609\", \"authors\": [\"Liu Y\", \"Gao C\", \"Hölscher C\", \"Li L\"], \"date\": \"2011-12-21\"}, \"line\": 11905, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reviews in the neurosciences\", \"reference\": \"22718609\", \"authors\": [\"Liu Y\", \"Gao C\", \"Hölscher C\", \"Li L\"], \"date\": \"2011-12-21\"}, \"line\": 11905, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37596, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37602, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37701, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37598, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotoxicity research\", \"reference\": \"22237943\", \"authors\": [\"von Bernhardi R\", \"Godoy B\", \"Murgas P\"], \"date\": \"2012-07-01\"}, \"line\": 1204, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 11, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37600, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37699, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37586, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37590, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37584, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 14, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37712, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 14, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor ÎºB,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science signaling\", \"reference\": \"23092889\", \"authors\": [\"Tang D\", \"Kang R\"], \"date\": \"2012-10-23\"}, \"line\": 37518, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37708, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37592, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37588, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Progress in neurobiology\", \"reference\": \"19388207\", \"authors\": [\"Salminen A\", \"Ojala J\", \"Kauppinen A\", \"Kaarniranta K\", \"Suuronen T\"], \"date\": \"2009-02-01\"}, \"line\": 37594, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 19, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR research\", \"reference\": \"18615183\", \"authors\": [\"Xu J\", \"Barger SW\", \"Drew PD\"], \"date\": \"2008-01-01\"}, \"line\": 37710, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 20, \"target\": 0, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "runner.run()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "247.93679084000192"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.get_final_score()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Robustness Analysis\n",
    "\n",
    "The algorithm is random, so it can be run multiple times to assess the robustness of the final scores using [pbt.analysis.cmpa.multirun](http://pybel-tools.readthedocs.io/en/latest/cmpa.html#pybel_tools.analysis.cmpa.multirun)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 39.5 s, sys: 713 ms, total: 40.2 s\n",
      "Wall time: 40.8 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "runners = list(cmpa.multirun(subgraph, inflamation_node, label, default_score=0, runs=500))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"274pt\" version=\"1.1\" viewBox=\"0 0 388 274\" width=\"388pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 274.853533 \n",
       "L 388.267074 274.853533 \n",
       "L 388.267074 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 41.999682 238.94218 \n",
       "L 376.799682 238.94218 \n",
       "L 376.799682 21.50218 \n",
       "L 41.999682 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#p9bae311fce)\" d=\"M 57.217864 238.94218 \n",
       "L 87.654228 238.94218 \n",
       "L 87.654228 231.176466 \n",
       "L 57.217864 231.176466 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#p9bae311fce)\" d=\"M 87.654228 238.94218 \n",
       "L 118.090592 238.94218 \n",
       "L 118.090592 231.176466 \n",
       "L 87.654228 231.176466 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#p9bae311fce)\" d=\"M 118.090592 238.94218 \n",
       "L 148.526955 238.94218 \n",
       "L 148.526955 191.053608 \n",
       "L 118.090592 191.053608 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#p9bae311fce)\" d=\"M 148.526955 238.94218 \n",
       "L 178.963319 238.94218 \n",
       "L 178.963319 179.405037 \n",
       "L 148.526955 179.405037 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#p9bae311fce)\" d=\"M 178.963319 238.94218 \n",
       "L 209.399682 238.94218 \n",
       "L 209.399682 127.633608 \n",
       "L 178.963319 127.633608 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#p9bae311fce)\" d=\"M 209.399682 238.94218 \n",
       "L 239.836046 238.94218 \n",
       "L 239.836046 31.856466 \n",
       "L 209.399682 31.856466 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#p9bae311fce)\" d=\"M 239.836046 238.94218 \n",
       "L 270.27241 238.94218 \n",
       "L 270.27241 126.339323 \n",
       "L 239.836046 126.339323 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#p9bae311fce)\" d=\"M 270.27241 238.94218 \n",
       "L 300.708773 238.94218 \n",
       "L 300.708773 178.110751 \n",
       "L 270.27241 178.110751 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#p9bae311fce)\" d=\"M 300.708773 238.94218 \n",
       "L 331.145137 238.94218 \n",
       "L 331.145137 214.350751 \n",
       "L 300.708773 214.350751 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#p9bae311fce)\" d=\"M 331.145137 238.94218 \n",
       "L 361.581501 238.94218 \n",
       "L 361.581501 231.176466 \n",
       "L 331.145137 231.176466 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"meb8fae3b2a\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"58.567077\" xlink:href=\"#meb8fae3b2a\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- $0$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "z\n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "z\n",
       "\" id=\"CMR17-48\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(56.07641 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"111.071743\" xlink:href=\"#meb8fae3b2a\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- $100$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(103.59974 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"163.576409\" xlink:href=\"#meb8fae3b2a\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- $200$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(156.104406 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"216.081074\" xlink:href=\"#meb8fae3b2a\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- $300$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 22.09375 33.703125 \n",
       "C 31 33.703125 34.90625 25.96875 34.90625 17.03125 \n",
       "C 34.90625 5 28.5 0.390625 22.703125 0.390625 \n",
       "C 17.40625 0.390625 8.796875 3.03125 6.09375 10.828125 \n",
       "C 6.59375 10.625 7.09375 10.625 7.59375 10.625 \n",
       "C 10 10.625 11.796875 12.21875 11.796875 14.8125 \n",
       "C 11.796875 17.703125 9.59375 19 7.59375 19 \n",
       "C 5.90625 19 3.296875 18.203125 3.296875 14.46875 \n",
       "C 3.296875 5.234375 12.296875 -2 22.90625 -2 \n",
       "C 34 -2 42.5 6.71875 42.5 16.953125 \n",
       "C 42.5 26.671875 34.5 33.703125 25 34.796875 \n",
       "C 32.59375 36.375 39.90625 43.0625 39.90625 52.015625 \n",
       "C 39.90625 59.6875 32 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 59.828125 5.90625 52.015625 \n",
       "C 5.90625 48.59375 8.5 48 9.796875 48 \n",
       "C 11.90625 48 13.703125 49.296875 13.703125 51.890625 \n",
       "C 13.703125 54.46875 11.90625 55.765625 9.796875 55.765625 \n",
       "C 9.40625 55.765625 8.90625 55.765625 8.5 55.578125 \n",
       "C 11.40625 61.859375 19.296875 63 22.796875 63 \n",
       "C 26.296875 63 32.90625 61.328125 32.90625 51.875 \n",
       "C 32.90625 49.109375 32.5 44.1875 29.09375 39.84375 \n",
       "C 26.09375 36 22.703125 36 19.40625 35.6875 \n",
       "C 18.90625 35.6875 16.59375 35.453125 16.203125 35.453125 \n",
       "C 15.5 35.359375 15.09375 35.265625 15.09375 34.5 \n",
       "C 15.09375 33.796875 15.203125 33.703125 17.203125 33.703125 \n",
       "z\n",
       "\" id=\"CMR17-51\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(208.609071 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-51\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"268.58574\" xlink:href=\"#meb8fae3b2a\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- $400$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(261.113737 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"321.090405\" xlink:href=\"#meb8fae3b2a\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_6\">\n",
       "      <!-- $500$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 11.40625 57.6875 \n",
       "C 12.40625 57.28125 16.5 56 20.703125 56 \n",
       "C 30 56 35.09375 61.109375 38 64.0625 \n",
       "C 38 64.953125 38 65.5 37.40625 65.5 \n",
       "C 37.296875 65.5 37.09375 65.5 36.296875 65.046875 \n",
       "C 32.796875 63.40625 28.703125 62.078125 23.703125 62.078125 \n",
       "C 20.703125 62.078125 16.203125 62.484375 11.296875 64.671875 \n",
       "C 10.203125 65.171875 10 65.171875 9.90625 65.171875 \n",
       "C 9.40625 65.171875 9.296875 65.0625 9.296875 63.078125 \n",
       "L 9.296875 34.421875 \n",
       "C 9.296875 32.640625 9.296875 32.140625 10.296875 32.140625 \n",
       "C 10.796875 32.140625 11 32.34375 11.5 33.03125 \n",
       "C 14.703125 37.515625 19.09375 39.40625 24.09375 39.40625 \n",
       "C 27.59375 39.40625 35.09375 37.21875 35.09375 20.140625 \n",
       "C 35.09375 16.953125 35.09375 11.171875 32.09375 6.578125 \n",
       "C 29.59375 2.484375 25.703125 0.390625 21.40625 0.390625 \n",
       "C 14.796875 0.390625 8.09375 5 6.296875 12.71875 \n",
       "C 6.703125 12.609375 7.5 12.421875 7.90625 12.421875 \n",
       "C 9.203125 12.421875 11.703125 13.125 11.703125 16.21875 \n",
       "C 11.703125 18.90625 9.796875 20 7.90625 20 \n",
       "C 5.59375 20 4.09375 18.59375 4.09375 15.796875 \n",
       "C 4.09375 7.09375 11 -2 21.59375 -2 \n",
       "C 31.90625 -2 41.703125 6.875 41.703125 19.75 \n",
       "C 41.703125 31.71875 33.90625 41 24.203125 41 \n",
       "C 19.09375 41 14.796875 39.109375 11.40625 35.53125 \n",
       "z\n",
       "\" id=\"CMR17-53\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(313.618403 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_7\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"373.595071\" xlink:href=\"#meb8fae3b2a\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- $600$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "z\n",
       "\" id=\"CMR17-54\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(366.123068 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_8\">\n",
       "     <!-- Score -->\n",
       "     <defs>\n",
       "      <path d=\"M 43.796875 67.03125 \n",
       "C 43.796875 68.8125 43.703125 68.921875 43.09375 68.921875 \n",
       "C 42.5 68.921875 42.296875 68.515625 41.90625 67.625 \n",
       "L 38.90625 61.046875 \n",
       "C 35.09375 66.515625 29.296875 69 23 69 \n",
       "C 12.796875 69 4.59375 60.953125 4.59375 50.828125 \n",
       "C 4.59375 47.75 5.296875 43.375 8.703125 39.21875 \n",
       "C 12.40625 34.65625 15.90625 33.859375 23.90625 31.765625 \n",
       "C 26.90625 30.96875 31.90625 29.671875 32.90625 29.28125 \n",
       "C 38.59375 26.90625 41.59375 20.953125 41.59375 15.390625 \n",
       "C 41.59375 8.234375 36.5 0.59375 27.796875 0.59375 \n",
       "C 17.09375 0.59375 7.09375 6.546875 6.40625 20.484375 \n",
       "C 6.296875 22.078125 6.203125 22.171875 5.5 22.171875 \n",
       "C 4.703125 22.171875 4.59375 22.078125 4.59375 20.078125 \n",
       "L 4.59375 -0.015625 \n",
       "C 4.59375 -1.796875 4.703125 -1.90625 5.296875 -1.90625 \n",
       "C 6 -1.90625 6.203125 -1.40625 7.59375 1.6875 \n",
       "C 7.703125 1.984375 7.703125 2.1875 9.59375 5.953125 \n",
       "C 14.59375 -0.609375 23 -2 27.796875 -2 \n",
       "C 38.703125 -2 46.40625 7.046875 46.40625 17.5625 \n",
       "C 46.40625 22.71875 44.5 26.6875 43.40625 28.484375 \n",
       "C 39.203125 34.84375 34.703125 36.03125 29.90625 37.21875 \n",
       "C 29.203125 37.53125 29 37.53125 23.40625 38.90625 \n",
       "C 18 40.296875 15.703125 41 13.203125 43.6875 \n",
       "C 10.40625 46.75 9.40625 49.9375 9.40625 53.109375 \n",
       "C 9.40625 59.5625 14.59375 66.609375 23.09375 66.609375 \n",
       "C 33.703125 66.609375 40.296875 59.15625 41.796875 46.765625 \n",
       "C 42.09375 44.96875 42.203125 44.875 42.90625 44.875 \n",
       "C 43.796875 44.875 43.796875 45.171875 43.796875 46.859375 \n",
       "z\n",
       "\" id=\"CMR17-83\"/>\n",
       "      <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "z\n",
       "\" id=\"CMR17-99\"/>\n",
       "      <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "z\n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "z\n",
       "\" id=\"CMR17-111\"/>\n",
       "      <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "      <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(197.762748 265.716355)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(91.38103 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(137.071507 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(172.352275 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"m9888e76678\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m9888e76678\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(30.018347 242.401429)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m9888e76678\" y=\"213.056466\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- $20$ -->\n",
       "      <g transform=\"translate(25.037012 216.515715)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m9888e76678\" y=\"187.170751\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- $40$ -->\n",
       "      <g transform=\"translate(25.037012 190.63)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m9888e76678\" y=\"161.285037\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_12\">\n",
       "      <!-- $60$ -->\n",
       "      <g transform=\"translate(25.037012 164.744286)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m9888e76678\" y=\"135.399323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_13\">\n",
       "      <!-- $80$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 27.203125 35.375 \n",
       "C 33.5 38.546875 39.90625 43.328125 39.90625 50.984375 \n",
       "C 39.90625 60.015625 31.09375 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 58.734375 5.90625 49.6875 \n",
       "C 5.90625 47.203125 6.5 42.921875 10.40625 39.140625 \n",
       "C 11.40625 38.15625 15.59375 35.171875 18.296875 33.28125 \n",
       "C 13.796875 30.984375 3.296875 25.53125 3.296875 14.59375 \n",
       "C 3.296875 4.359375 13.09375 -2 22.796875 -2 \n",
       "C 33.5 -2 42.5 5.65625 42.5 15.78125 \n",
       "C 42.5 24.84375 36.40625 29.015625 32.40625 31.703125 \n",
       "z\n",
       "M 14.09375 44.09375 \n",
       "C 13.296875 44.59375 9.296875 47.671875 9.296875 52.359375 \n",
       "C 9.296875 58.421875 15.59375 63 22.796875 63 \n",
       "C 30.703125 63 36.5 57.4375 36.5 50.96875 \n",
       "C 36.5 41.703125 26.09375 36.421875 25.59375 36.421875 \n",
       "C 25.5 36.421875 25.40625 36.421875 24.59375 37.03125 \n",
       "z\n",
       "M 32.5 23.734375 \n",
       "C 34 22.640625 38.796875 19.375 38.796875 13.296875 \n",
       "C 38.796875 5.953125 31.40625 0.390625 23 0.390625 \n",
       "C 13.90625 0.390625 7 6.84375 7 14.6875 \n",
       "C 7 22.546875 13.09375 29.09375 20 32.1875 \n",
       "z\n",
       "\" id=\"CMR17-56\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(25.037012 138.858572)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m9888e76678\" y=\"109.513608\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_14\">\n",
       "      <!-- $100$ -->\n",
       "      <g transform=\"translate(20.055677 112.972858)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_7\">\n",
       "     <g id=\"line2d_14\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m9888e76678\" y=\"83.627894\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- $120$ -->\n",
       "      <g transform=\"translate(20.055677 87.087143)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_8\">\n",
       "     <g id=\"line2d_15\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m9888e76678\" y=\"57.74218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- $140$ -->\n",
       "      <g transform=\"translate(20.055677 61.201429)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_9\">\n",
       "     <g id=\"line2d_16\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m9888e76678\" y=\"31.856466\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- $160$ -->\n",
       "      <g transform=\"translate(20.055677 35.315715)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_18\">\n",
       "     <!-- Frequency -->\n",
       "     <defs>\n",
       "      <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      <path d=\"M 41.09375 43.78125 \n",
       "L 39.5 43.78125 \n",
       "L 36.5 33.9375 \n",
       "C 34.203125 39.203125 30 43.78125 23.296875 43.78125 \n",
       "C 13.09375 43.78125 3.296875 34.734375 3.296875 21.609375 \n",
       "C 3.296875 8.5625 12.5 -0.59375 22.5 -0.59375 \n",
       "C 30.203125 -0.59375 34.5 5.28125 35.796875 7.46875 \n",
       "L 35.796875 -12.328125 \n",
       "C 35.796875 -16.109375 35.40625 -16.609375 29 -16.609375 \n",
       "L 29 -19.1875 \n",
       "C 31.40625 -19 35.796875 -19 38.40625 -19 \n",
       "C 41 -19 45.5 -19 47.90625 -19.1875 \n",
       "L 47.90625 -16.609375 \n",
       "C 41.5 -16.609375 41.09375 -16.203125 41.09375 -12.328125 \n",
       "z\n",
       "M 36 13.84375 \n",
       "C 36 9.5625 32.40625 5.578125 32.09375 5.28125 \n",
       "C 29 1.796875 25.40625 1 23 1 \n",
       "C 15.5 1 9.703125 10.453125 9.703125 21.484375 \n",
       "C 9.703125 33.140625 16.296875 42 24 42 \n",
       "C 32 42 36 32.9375 36 27.765625 \n",
       "z\n",
       "\" id=\"CMR17-113\"/>\n",
       "      <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "      <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "      <path d=\"M 39 34.328125 \n",
       "C 41.5 40.40625 46 40.40625 47.40625 40.40625 \n",
       "L 47.40625 43 \n",
       "C 45.5 43 43 43 41.09375 43 \n",
       "C 39 43 35.703125 43 33.703125 43.09375 \n",
       "L 33.703125 40.5 \n",
       "C 37.5 40.203125 37.59375 37.40625 37.59375 36.609375 \n",
       "C 37.59375 35.625 37.40625 35.125 36.90625 33.921875 \n",
       "L 26 7.015625 \n",
       "L 14.09375 36.03125 \n",
       "C 13.59375 37.21875 13.59375 37.90625 13.59375 38.015625 \n",
       "C 13.59375 40.203125 15.90625 40.40625 18.296875 40.40625 \n",
       "L 18.296875 43.09375 \n",
       "C 15.90625 43 11.59375 43 9.09375 43 \n",
       "C 6.40625 43 3.203125 43 1 43.09375 \n",
       "L 1 40.40625 \n",
       "C 6.40625 40.40625 7 39.90625 8.296875 36.71875 \n",
       "L 23.203125 0.234375 \n",
       "C 18.703125 -11.53125 16.09375 -18.40625 9.796875 -18.40625 \n",
       "C 8.703125 -18.40625 6.203125 -18.109375 4.40625 -16.3125 \n",
       "C 6.703125 -16.109375 7.703125 -14.71875 7.703125 -13.015625 \n",
       "C 7.703125 -11.328125 6.5 -9.828125 4.5 -9.828125 \n",
       "C 2.296875 -9.828125 1.203125 -11.328125 1.203125 -13.125 \n",
       "C 1.203125 -17.109375 5.296875 -20 9.796875 -20 \n",
       "C 15.59375 -20 19.203125 -14.40625 21.203125 -9.4375 \n",
       "z\n",
       "\" id=\"CMR17-121\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(14.118498 152.444442)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(52.176591 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(87.457359 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(127.943019 0)scale(0.996264)\" xlink:href=\"#CMR17-113\"/>\n",
       "      <use transform=\"translate(176.235935 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(227.131305 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(267.616966 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      <use transform=\"translate(318.512335 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(358.997996 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 41.999682 238.94218 \n",
       "L 41.999682 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 376.799682 238.94218 \n",
       "L 376.799682 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 41.999682 238.94218 \n",
       "L 376.799682 238.94218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 41.999682 21.50218 \n",
       "L 376.799682 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_19\">\n",
       "    <!-- Histogram of CMPA scores for inflammatory response -->\n",
       "    <defs>\n",
       "     <path d=\"M 56.09375 61.015625 \n",
       "C 56.09375 65.5 57.203125 65.5 64.09375 65.5 \n",
       "L 64.09375 68 \n",
       "C 61.203125 68 55.90625 68 52.796875 68 \n",
       "C 49.703125 68 44.40625 68 41.5 68 \n",
       "L 41.5 65.5 \n",
       "C 48.40625 65.5 49.5 65.5 49.5 61 \n",
       "L 49.5 36.578125 \n",
       "L 19.703125 36.578125 \n",
       "L 19.703125 61 \n",
       "C 19.703125 65.5 20.796875 65.5 27.703125 65.5 \n",
       "L 27.703125 68 \n",
       "C 24.796875 68 19.5 68 16.40625 68 \n",
       "C 13.296875 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 6.984375 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.296875 0 16.40625 0 \n",
       "C 19.5 0 24.796875 0 27.703125 0 \n",
       "L 27.703125 2.5 \n",
       "C 20.796875 2.5 19.703125 2.5 19.703125 6.96875 \n",
       "L 19.703125 34 \n",
       "L 49.5 34 \n",
       "L 49.5 6.96875 \n",
       "C 49.5 2.5 48.40625 2.5 41.5 2.5 \n",
       "L 41.5 0 \n",
       "C 44.40625 0 49.703125 0 52.796875 0 \n",
       "C 55.90625 0 61.203125 0 64.09375 0 \n",
       "L 64.09375 2.5 \n",
       "C 57.203125 2.5 56.09375 2.5 56.09375 6.984375 \n",
       "z\n",
       "\" id=\"CMR17-72\"/>\n",
       "     <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "z\n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "     <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "     <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "     <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "z\n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "z\n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "z\n",
       "\" id=\"CMR17-103\"/>\n",
       "     <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "     <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "     <path d=\"M 15.296875 40 \n",
       "L 26.703125 40 \n",
       "L 26.703125 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 54.59375 \n",
       "C 15.09375 62.84375 19.296875 68 23.59375 68 \n",
       "C 24.90625 68 26.40625 67.609375 27.40625 66.90625 \n",
       "C 26.59375 66.703125 24.5 66.015625 24.5 63.546875 \n",
       "C 24.5 60.953125 26.5 60.15625 27.90625 60.15625 \n",
       "C 29.296875 60.15625 31.296875 60.953125 31.296875 63.546875 \n",
       "C 31.296875 67.3125 27.703125 69.59375 23.703125 69.59375 \n",
       "C 18 69.59375 10.203125 64.9375 10.203125 54.390625 \n",
       "L 10.203125 42.578125 \n",
       "L 2.203125 42.578125 \n",
       "L 2.203125 40 \n",
       "L 10.203125 40 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.40625 0 13 0 \n",
       "C 15.90625 0 21 0 23.703125 -0.203125 \n",
       "L 23.703125 2.390625 \n",
       "C 16.5 2.390625 15.296875 2.390625 15.296875 6.859375 \n",
       "z\n",
       "\" id=\"CMR17-102\"/>\n",
       "     <path d=\"M 62.09375 67.015625 \n",
       "C 62.09375 68.796875 62 68.90625 61.40625 68.90625 \n",
       "C 61 68.90625 60.90625 68.796875 60.203125 67.609375 \n",
       "L 55.796875 59.25 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 12.90625 20.09375 -2 37.296875 -2 \n",
       "C 52.59375 -2 62.09375 11.40625 62.09375 22.828125 \n",
       "C 62.09375 23.8125 62.09375 24.21875 61.203125 24.21875 \n",
       "C 60.40625 24.21875 60.40625 23.921875 60.296875 23.03125 \n",
       "C 59.5 9.140625 49.5 0.59375 38.703125 0.59375 \n",
       "C 28.5 0.59375 12.203125 7.734375 12.203125 33.546875 \n",
       "C 12.203125 59.453125 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.6875 59.90625 42.890625 \n",
       "C 60.09375 41.6875 60.09375 41.484375 61 41.484375 \n",
       "C 62.09375 41.484375 62.09375 41.6875 62.09375 43.484375 \n",
       "z\n",
       "\" id=\"CMR17-67\"/>\n",
       "     <path d=\"M 20.90625 66 \n",
       "C 20.296875 67.5 20.203125 67.5 18.09375 67.5 \n",
       "L 5.296875 67.5 \n",
       "L 5.296875 65 \n",
       "C 12.203125 65 13.296875 65 13.296875 60.546875 \n",
       "L 13.296875 9.53125 \n",
       "C 13.296875 7.046875 13.296875 2.390625 5.296875 2.390625 \n",
       "L 5.296875 -0.203125 \n",
       "C 7.59375 0 11.796875 0 14.296875 0 \n",
       "C 16.796875 0 21 0 23.296875 -0.203125 \n",
       "L 23.296875 2.390625 \n",
       "C 15.296875 2.390625 15.296875 7.046875 15.296875 9.53125 \n",
       "L 15.296875 64.609375 \n",
       "L 15.40625 64.609375 \n",
       "L 38.90625 1.390625 \n",
       "C 39.296875 0.390625 39.5 -0.203125 40.296875 -0.203125 \n",
       "C 41.09375 -0.203125 41.296875 0.390625 41.703125 1.390625 \n",
       "L 65.40625 65.09375 \n",
       "L 65.5 65.09375 \n",
       "L 65.5 6.84375 \n",
       "C 65.5 2.390625 64.40625 2.390625 57.5 2.390625 \n",
       "L 57.5 -0.203125 \n",
       "C 60.203125 0 65.59375 0 68.5 0 \n",
       "C 71.40625 0 76.90625 0 79.59375 -0.203125 \n",
       "L 79.59375 2.390625 \n",
       "C 72.703125 2.390625 71.59375 2.390625 71.59375 6.84375 \n",
       "L 71.59375 60.546875 \n",
       "C 71.59375 65 72.703125 65 79.59375 65 \n",
       "L 79.59375 67.5 \n",
       "L 66.90625 67.5 \n",
       "C 64.796875 67.5 64.703125 67.40625 64.09375 65.90625 \n",
       "L 42.5 7.84375 \n",
       "z\n",
       "\" id=\"CMR17-77\"/>\n",
       "     <path d=\"M 19.796875 31.90625 \n",
       "L 35.796875 31.90625 \n",
       "C 48.703125 31.90625 58.09375 40.28125 58.09375 49.75 \n",
       "C 58.09375 59.234375 48.90625 68 35.796875 68 \n",
       "L 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.921875 \n",
       "L 13.203125 6.96875 \n",
       "C 13.203125 2.5 12.09375 2.5 5.203125 2.5 \n",
       "L 5.203125 0 \n",
       "C 8.09375 0 13.40625 0 16.5 0 \n",
       "C 19.59375 0 24.90625 0 27.796875 0 \n",
       "L 27.796875 2.5 \n",
       "C 20.90625 2.5 19.796875 2.5 19.796875 6.96875 \n",
       "z\n",
       "M 19.59375 34 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 33.90625 65.40625 \n",
       "C 46.796875 65.40625 50.5 58.03125 50.5 49.75 \n",
       "C 50.5 40.6875 46.09375 34 33.90625 34 \n",
       "z\n",
       "\" id=\"CMR17-80\"/>\n",
       "     <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "     <path d=\"M 39.90625 68.890625 \n",
       "C 38.796875 68.6875 37.90625 68.5 36.59375 68.203125 \n",
       "C 33.703125 69.6875 29.90625 69.6875 28.90625 69.6875 \n",
       "C 20.40625 69.6875 9.90625 64.90625 9.90625 54.25 \n",
       "L 9.90625 42.578125 \n",
       "L 1.90625 42.578125 \n",
       "L 1.90625 40 \n",
       "L 9.90625 40 \n",
       "L 9.90625 6.671875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 9.90625 0 12.40625 0 \n",
       "C 15 0 19.40625 0 21.796875 -0.203125 \n",
       "L 21.796875 2.390625 \n",
       "C 15.40625 2.390625 15 2.78125 15 6.671875 \n",
       "L 15 40 \n",
       "L 34.796875 40 \n",
       "L 34.796875 6.671875 \n",
       "C 34.796875 2.875 34.40625 2.390625 28 2.390625 \n",
       "L 28 -0.203125 \n",
       "C 30.40625 0 34.796875 0 37.296875 0 \n",
       "C 39.90625 0 44.296875 0 46.703125 -0.203125 \n",
       "L 46.703125 2.390625 \n",
       "C 40.296875 2.390625 39.90625 2.78125 39.90625 6.671875 \n",
       "z\n",
       "M 14.796875 42.578125 \n",
       "L 14.796875 54.203125 \n",
       "C 14.796875 62.9375 21.40625 68 28.703125 68 \n",
       "C 29.203125 68 34.703125 68 34.703125 66.90625 \n",
       "C 34.703125 66.796875 34.59375 66.703125 34.09375 66.203125 \n",
       "C 33.203125 65.609375 32.796875 64.609375 32.796875 63.609375 \n",
       "C 32.796875 62.203125 33.5 61.015625 34.796875 60.515625 \n",
       "L 34.796875 42.578125 \n",
       "z\n",
       "\" id=\"CMR17-13\"/>\n",
       "     <path d=\"M 22 -16.609375 \n",
       "C 15.59375 -16.609375 15.203125 -16.203125 15.203125 -12.328125 \n",
       "L 15.203125 6.28125 \n",
       "C 18.09375 2.09375 22.296875 -0.59375 27.59375 -0.59375 \n",
       "C 37.59375 -0.59375 47.703125 8.171875 47.703125 21.6875 \n",
       "C 47.703125 34.234375 39 43.78125 28.703125 43.78125 \n",
       "C 22.703125 43.78125 17.796875 40.5 15 36.53125 \n",
       "L 15 43.78125 \n",
       "L 3.09375 42.6875 \n",
       "L 3.09375 40.109375 \n",
       "C 9 40.109375 9.90625 39.5 9.90625 34.734375 \n",
       "L 9.90625 -12.328125 \n",
       "C 9.90625 -16.109375 9.5 -16.609375 3.09375 -16.609375 \n",
       "L 3.09375 -19.1875 \n",
       "C 5.5 -19 9.90625 -19 12.5 -19 \n",
       "C 15.09375 -19 19.59375 -19 22 -19.1875 \n",
       "z\n",
       "M 15.203125 31.4375 \n",
       "C 15.203125 32.84375 15.203125 34.921875 19.09375 38.609375 \n",
       "C 19.59375 39.015625 23 42 28 42 \n",
       "C 35.296875 42 41.296875 32.9375 41.296875 21.59375 \n",
       "C 41.296875 10.25 34.90625 1 27 1 \n",
       "C 23.40625 1 19.5 2.6875 16.5 7.375 \n",
       "C 15.203125 9.5625 15.203125 10.15625 15.203125 11.75 \n",
       "z\n",
       "\" id=\"CMR17-112\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(70.967464 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-72\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(93.943488 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(129.744826 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(165.025594 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(210.716071 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(256.406548 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(291.687316 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(337.377793 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(444.373349 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(490.063826 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(547.613305 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "     <use transform=\"translate(614.123186 0)scale(0.996264)\" xlink:href=\"#CMR17-77\"/>\n",
       "     <use transform=\"translate(698.810126 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "     <use transform=\"translate(753.589171 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "     <use transform=\"translate(852.737567 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(888.538905 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "     <use transform=\"translate(929.024565 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(974.715043 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(1009.99581 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(1050.481471 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(1116.358775 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(1143.832287 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(1189.522764 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(1254.879499 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(1279.750557 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(1330.645927 0)scale(0.996264)\" xlink:href=\"#CMR17-13\"/>\n",
       "     <use transform=\"translate(1380.38815 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(1426.078627 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(1502.998217 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(1579.917807 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(1625.608284 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(1660.889052 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(1706.579529 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(1741.860297 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "     <use transform=\"translate(1820.229179 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(1855.509947 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(1895.995607 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(1931.796945 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "     <use transform=\"translate(1985.294753 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(2030.985231 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(2081.8806 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(2117.681938 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"p9bae311fce\">\n",
       "   <rect height=\"217.44\" width=\"334.8\" x=\"41.999682\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x115557f28>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.title('Histogram of CMPA scores for {}'.format(calculate_canonical_name(subgraph, inflamation_node)))\n",
    "plt.ylabel('Frequency')\n",
    "plt.xlabel('Score')\n",
    "plt.hist([runner.get_final_score() for runner in runners])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Full Workflow\n",
    "\n",
    "This workflow can all be run with [pbt.analysis.cmpa.workflow_all_average](http://pybel-tools.readthedocs.io/en/latest/cmpa.html#pybel_tools.analysis.cmpa.workflow_all_average) but is outlined here for a deeper understanding.\n",
    "\n",
    "### Generate Candidate Mechanisms\n",
    "\n",
    "Candidate mechanism are generated with the aforementioned function [pbt.generation.generate_mechanism](http://pybel-tools.readthedocs.io/en/latest/generation.html#pybel_tools.generation.generate_mechanism) to assess the upstream causal subgraphs of all biological processes."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "all_bioprocesses = pbt.filters.get_nodes_by_function(graph, BIOPROCESS)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 558 ms, sys: 13.8 ms, total: 572 ms\n",
      "Wall time: 574 ms\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "candidate_mechanisms = {\n",
    "    node: pbt.generation.generate_mechanism(graph, node, label) \n",
    "    for node in all_bioprocesses\n",
    "}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The algorithm is run over all candidate subgraphs that are nontrivial (have more than one node in them) and a pandas DataFrame is prepared."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.cmpa:Unable to run CMPA on ('BiologicalProcess', 'GOBP', 'rhythmic excitation')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 9min 35s, sys: 7.56 s, total: 9min 42s\n",
      "Wall time: 9min 47s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "scores = {}\n",
    "\n",
    "for node, subgraph in candidate_mechanisms.items():\n",
    "    \n",
    "    fneighbors = subgraph.in_degree(node)\n",
    "    fneighbors = 0 if isinstance(fneighbors, dict) else fneighbors\n",
    "    size = subgraph.number_of_nodes()\n",
    "    \n",
    "    if size <= 1:  # Can't calculate a score for empty subgraphs\n",
    "        average_score = None\n",
    "    else:\n",
    "        average_score = cmpa.workflow_average(subgraph, node, label)\n",
    "    \n",
    "    scores[node] = (average_score, fneighbors, size)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(401, 221)"
      ]
     },
     "execution_count": 49,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(candidate_mechanisms), sum(1 for score, _, _ in scores.values() if score)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Optionally, calcualte \"cute names\" for the nodes\n",
    "# scores = {calculate_canonical_name(graph, n): s for n, s in scores.items()}\n",
    "\n",
    "scores_df = pd.DataFrame.from_items(scores.items(), orient='index', columns=['Average CMPA Score','Number First Neighbors','Subgraph Size'])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "A scatter matrix is shown below. The distribution of average CMPA scores is shown to be mostly normal. The following scatter plots show small, non-linear relationships between average CMPA score and the subgraph size and the number of first neighbors of the seed node. The trivial positive correlation between number of first neighbors is also shown."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/usr/local/lib/python3.6/site-packages/ipykernel_launcher.py:1: FutureWarning: 'pandas.tools.plotting.scatter_matrix' is deprecated, import 'pandas.plotting.scatter_matrix' instead.\n",
      "  \"\"\"Entry point for launching an IPython kernel.\n"
     ]
    },
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"433pt\" version=\"1.1\" viewBox=\"0 0 608 433\" width=\"608pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 433.330549 \n",
       "L 608.458225 433.330549 \n",
       "L 608.458225 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 39.758225 137.54 \n",
       "L 225.758225 137.54 \n",
       "L 225.758225 10.7 \n",
       "L 39.758225 10.7 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#p05c76295e5)\" d=\"M 44.186797 137.54 \n",
       "L 61.901082 137.54 \n",
       "L 61.901082 136.923673 \n",
       "L 44.186797 136.923673 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#p05c76295e5)\" d=\"M 61.901082 137.54 \n",
       "L 79.615368 137.54 \n",
       "L 79.615368 137.54 \n",
       "L 61.901082 137.54 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#p05c76295e5)\" d=\"M 79.615368 137.54 \n",
       "L 97.329654 137.54 \n",
       "L 97.329654 124.597143 \n",
       "L 79.615368 124.597143 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#p05c76295e5)\" d=\"M 97.329654 137.54 \n",
       "L 115.04394 137.54 \n",
       "L 115.04394 16.74 \n",
       "L 97.329654 16.74 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#p05c76295e5)\" d=\"M 115.04394 137.54 \n",
       "L 132.758225 137.54 \n",
       "L 132.758225 134.458367 \n",
       "L 115.04394 134.458367 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#p05c76295e5)\" d=\"M 132.758225 137.54 \n",
       "L 150.472511 137.54 \n",
       "L 150.472511 136.923673 \n",
       "L 132.758225 136.923673 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#p05c76295e5)\" d=\"M 150.472511 137.54 \n",
       "L 168.186797 137.54 \n",
       "L 168.186797 137.54 \n",
       "L 150.472511 137.54 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#p05c76295e5)\" d=\"M 168.186797 137.54 \n",
       "L 185.901082 137.54 \n",
       "L 185.901082 137.54 \n",
       "L 168.186797 137.54 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#p05c76295e5)\" d=\"M 185.901082 137.54 \n",
       "L 203.615368 137.54 \n",
       "L 203.615368 137.54 \n",
       "L 185.901082 137.54 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#p05c76295e5)\" d=\"M 203.615368 137.54 \n",
       "L 221.329654 137.54 \n",
       "L 221.329654 136.923673 \n",
       "L 203.615368 136.923673 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"mf5d37b9f9f\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"117.149618\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- -20 -->\n",
       "      <defs>\n",
       "       <path d=\"M 25.5 19 \n",
       "L 25.5 23.375 \n",
       "L 0.40625 23.375 \n",
       "L 0.40625 19 \n",
       "z\n",
       "\" id=\"CMR17-45\"/>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "z\n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "z\n",
       "\" id=\"CMR17-48\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(21.467071 119.917019)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "       <use transform=\"translate(30.075966 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(75.766443 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"94.046366\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- 0 -->\n",
       "      <g transform=\"translate(28.524042 96.813767)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"70.943115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- 20 -->\n",
       "      <g transform=\"translate(24.28986 73.710515)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"47.839863\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- 40 -->\n",
       "      <defs>\n",
       "       <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(24.28986 50.607264)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"24.736612\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- 60 -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "z\n",
       "\" id=\"CMR17-54\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(24.28986 27.504012)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_6\">\n",
       "     <!-- Average CMPA Score -->\n",
       "     <defs>\n",
       "      <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "      <path d=\"M 39.09375 33.796875 \n",
       "C 40 36 41.59375 40 47.40625 40 \n",
       "L 47.40625 42.59375 \n",
       "C 45.5 42.59375 43 42.59375 41.09375 42.59375 \n",
       "C 39 42.59375 35.703125 42.59375 33.703125 42.6875 \n",
       "L 33.703125 40.09375 \n",
       "C 36.203125 39.890625 37.59375 38.703125 37.59375 36.109375 \n",
       "C 37.59375 35.203125 37.59375 35 37 33.609375 \n",
       "L 26 5.890625 \n",
       "L 14.09375 35.71875 \n",
       "C 13.5 37.015625 13.5 37.203125 13.5 37.609375 \n",
       "C 13.5 40 16.796875 40 18.296875 40 \n",
       "L 18.296875 42.6875 \n",
       "C 15.90625 42.59375 11.5 42.59375 9 42.59375 \n",
       "C 6.296875 42.59375 3.203125 42.59375 1 42.6875 \n",
       "L 1 40 \n",
       "C 4.796875 40 6.90625 40 8.09375 36.90625 \n",
       "L 22.59375 0.6875 \n",
       "C 23.203125 -0.796875 23.296875 -1 24.203125 -1 \n",
       "C 25.09375 -1 25.203125 -0.796875 25.796875 0.6875 \n",
       "z\n",
       "\" id=\"CMR17-118\"/>\n",
       "      <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "      <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "      <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "      <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "z\n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "z\n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "z\n",
       "\" id=\"CMR17-103\"/>\n",
       "      <path d=\"M 62.09375 67.015625 \n",
       "C 62.09375 68.796875 62 68.90625 61.40625 68.90625 \n",
       "C 61 68.90625 60.90625 68.796875 60.203125 67.609375 \n",
       "L 55.796875 59.25 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 12.90625 20.09375 -2 37.296875 -2 \n",
       "C 52.59375 -2 62.09375 11.40625 62.09375 22.828125 \n",
       "C 62.09375 23.8125 62.09375 24.21875 61.203125 24.21875 \n",
       "C 60.40625 24.21875 60.40625 23.921875 60.296875 23.03125 \n",
       "C 59.5 9.140625 49.5 0.59375 38.703125 0.59375 \n",
       "C 28.5 0.59375 12.203125 7.734375 12.203125 33.546875 \n",
       "C 12.203125 59.453125 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.6875 59.90625 42.890625 \n",
       "C 60.09375 41.6875 60.09375 41.484375 61 41.484375 \n",
       "C 62.09375 41.484375 62.09375 41.6875 62.09375 43.484375 \n",
       "z\n",
       "\" id=\"CMR17-67\"/>\n",
       "      <path d=\"M 20.90625 66 \n",
       "C 20.296875 67.5 20.203125 67.5 18.09375 67.5 \n",
       "L 5.296875 67.5 \n",
       "L 5.296875 65 \n",
       "C 12.203125 65 13.296875 65 13.296875 60.546875 \n",
       "L 13.296875 9.53125 \n",
       "C 13.296875 7.046875 13.296875 2.390625 5.296875 2.390625 \n",
       "L 5.296875 -0.203125 \n",
       "C 7.59375 0 11.796875 0 14.296875 0 \n",
       "C 16.796875 0 21 0 23.296875 -0.203125 \n",
       "L 23.296875 2.390625 \n",
       "C 15.296875 2.390625 15.296875 7.046875 15.296875 9.53125 \n",
       "L 15.296875 64.609375 \n",
       "L 15.40625 64.609375 \n",
       "L 38.90625 1.390625 \n",
       "C 39.296875 0.390625 39.5 -0.203125 40.296875 -0.203125 \n",
       "C 41.09375 -0.203125 41.296875 0.390625 41.703125 1.390625 \n",
       "L 65.40625 65.09375 \n",
       "L 65.5 65.09375 \n",
       "L 65.5 6.84375 \n",
       "C 65.5 2.390625 64.40625 2.390625 57.5 2.390625 \n",
       "L 57.5 -0.203125 \n",
       "C 60.203125 0 65.59375 0 68.5 0 \n",
       "C 71.40625 0 76.90625 0 79.59375 -0.203125 \n",
       "L 79.59375 2.390625 \n",
       "C 72.703125 2.390625 71.59375 2.390625 71.59375 6.84375 \n",
       "L 71.59375 60.546875 \n",
       "C 71.59375 65 72.703125 65 79.59375 65 \n",
       "L 79.59375 67.5 \n",
       "L 66.90625 67.5 \n",
       "C 64.796875 67.5 64.703125 67.40625 64.09375 65.90625 \n",
       "L 42.5 7.84375 \n",
       "z\n",
       "\" id=\"CMR17-77\"/>\n",
       "      <path d=\"M 19.796875 31.90625 \n",
       "L 35.796875 31.90625 \n",
       "C 48.703125 31.90625 58.09375 40.28125 58.09375 49.75 \n",
       "C 58.09375 59.234375 48.90625 68 35.796875 68 \n",
       "L 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.921875 \n",
       "L 13.203125 6.96875 \n",
       "C 13.203125 2.5 12.09375 2.5 5.203125 2.5 \n",
       "L 5.203125 0 \n",
       "C 8.09375 0 13.40625 0 16.5 0 \n",
       "C 19.59375 0 24.90625 0 27.796875 0 \n",
       "L 27.796875 2.5 \n",
       "C 20.90625 2.5 19.796875 2.5 19.796875 6.96875 \n",
       "z\n",
       "M 19.59375 34 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 33.90625 65.40625 \n",
       "C 46.796875 65.40625 50.5 58.03125 50.5 49.75 \n",
       "C 50.5 40.6875 46.09375 34 33.90625 34 \n",
       "z\n",
       "\" id=\"CMR17-80\"/>\n",
       "      <path d=\"M 43.796875 67.03125 \n",
       "C 43.796875 68.8125 43.703125 68.921875 43.09375 68.921875 \n",
       "C 42.5 68.921875 42.296875 68.515625 41.90625 67.625 \n",
       "L 38.90625 61.046875 \n",
       "C 35.09375 66.515625 29.296875 69 23 69 \n",
       "C 12.796875 69 4.59375 60.953125 4.59375 50.828125 \n",
       "C 4.59375 47.75 5.296875 43.375 8.703125 39.21875 \n",
       "C 12.40625 34.65625 15.90625 33.859375 23.90625 31.765625 \n",
       "C 26.90625 30.96875 31.90625 29.671875 32.90625 29.28125 \n",
       "C 38.59375 26.90625 41.59375 20.953125 41.59375 15.390625 \n",
       "C 41.59375 8.234375 36.5 0.59375 27.796875 0.59375 \n",
       "C 17.09375 0.59375 7.09375 6.546875 6.40625 20.484375 \n",
       "C 6.296875 22.078125 6.203125 22.171875 5.5 22.171875 \n",
       "C 4.703125 22.171875 4.59375 22.078125 4.59375 20.078125 \n",
       "L 4.59375 -0.015625 \n",
       "C 4.59375 -1.796875 4.703125 -1.90625 5.296875 -1.90625 \n",
       "C 6 -1.90625 6.203125 -1.40625 7.59375 1.6875 \n",
       "C 7.703125 1.984375 7.703125 2.1875 9.59375 5.953125 \n",
       "C 14.59375 -0.609375 23 -2 27.796875 -2 \n",
       "C 38.703125 -2 46.40625 7.046875 46.40625 17.5625 \n",
       "C 46.40625 22.71875 44.5 26.6875 43.40625 28.484375 \n",
       "C 39.203125 34.84375 34.703125 36.03125 29.90625 37.21875 \n",
       "C 29.203125 37.53125 29 37.53125 23.40625 38.90625 \n",
       "C 18 40.296875 15.703125 41 13.203125 43.6875 \n",
       "C 10.40625 46.75 9.40625 49.9375 9.40625 53.109375 \n",
       "C 9.40625 59.5625 14.59375 66.609375 23.09375 66.609375 \n",
       "C 33.703125 66.609375 40.296875 59.15625 41.796875 46.765625 \n",
       "C 42.09375 44.96875 42.203125 44.875 42.90625 44.875 \n",
       "C 43.796875 44.875 43.796875 45.171875 43.796875 46.859375 \n",
       "z\n",
       "\" id=\"CMR17-83\"/>\n",
       "      <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "z\n",
       "\" id=\"CMR17-99\"/>\n",
       "      <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "z\n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "z\n",
       "\" id=\"CMR17-111\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(15.529893 121.539467)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "      <use transform=\"translate(114.762892 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(155.248552 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(190.52932 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(236.219797 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(281.910274 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(352.471901 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "      <use transform=\"translate(418.981782 0)scale(0.996264)\" xlink:href=\"#CMR17-77\"/>\n",
       "      <use transform=\"translate(503.668722 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "      <use transform=\"translate(558.447767 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      <use transform=\"translate(657.596163 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(708.491533 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(748.977194 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(794.667671 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(829.948439 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 39.758225 137.54 \n",
       "L 39.758225 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 225.758225 137.54 \n",
       "L 225.758225 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 39.758225 137.54 \n",
       "L 225.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 39.758225 10.7 \n",
       "L 225.758225 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_2\">\n",
       "   <g id=\"patch_17\">\n",
       "    <path d=\"M 225.758225 137.54 \n",
       "L 411.758225 137.54 \n",
       "L 411.758225 10.7 \n",
       "L 225.758225 10.7 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_1\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C0_0_79f0254025\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"261.67886\" xlink:href=\"#C0_0_79f0254025\" y=\"100.244856\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"89.250237\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"89.250237\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"87.626196\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"249.869336\" xlink:href=\"#C0_0_79f0254025\" y=\"96.949074\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"348.282035\" xlink:href=\"#C0_0_79f0254025\" y=\"13.72\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"98.842495\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"364.028067\" xlink:href=\"#C0_0_79f0254025\" y=\"103.666279\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"277.424892\" xlink:href=\"#C0_0_79f0254025\" y=\"110.02958\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"407.329654\" xlink:href=\"#C0_0_79f0254025\" y=\"100.522774\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"304.980447\" xlink:href=\"#C0_0_79f0254025\" y=\"90.152935\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"91.73516\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"97.036234\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"98.864658\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.955756\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"92.860388\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"92.892234\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"99.545689\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"92.838425\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"304.980447\" xlink:href=\"#C0_0_79f0254025\" y=\"81.954361\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"285.297908\" xlink:href=\"#C0_0_79f0254025\" y=\"93.58209\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"95.753337\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"93.809072\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.888092\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"96.796027\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"85.759356\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"84.263891\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"91.296705\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"89.250237\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"96.038852\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"95.316097\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.415389\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"93.763743\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.049442\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.04329\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.04329\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.509989\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"95.292342\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"88.607187\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"95.088289\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"281.3614\" xlink:href=\"#C0_0_79f0254025\" y=\"88.322418\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"93.989901\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"90.5625\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"99.577534\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"102.121607\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"91.728312\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"261.67886\" xlink:href=\"#C0_0_79f0254025\" y=\"98.424198\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"88.812174\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.714379\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.714379\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"94.954947\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"96.439673\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"92.935777\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"94.541587\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"91.671483\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.444746\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"87.400601\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.594994\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"261.67886\" xlink:href=\"#C0_0_79f0254025\" y=\"97.730054\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"93.398988\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"95.457134\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"100.764046\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"91.296705\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.132422\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"95.044209\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"92.081043\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"106.092432\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"265.615368\" xlink:href=\"#C0_0_79f0254025\" y=\"110.245848\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"93.582503\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"95.175657\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"97.89088\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"92.986918\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"94.760854\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"88.630432\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"96.730976\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.090991\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.884654\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.305988\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.395396\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"90.578761\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"91.304438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"93.931038\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"78.268008\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"96.796027\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.110357\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"93.718795\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"95.992512\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"95.234698\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.032283\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"293.170924\" xlink:href=\"#C0_0_79f0254025\" y=\"93.738637\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"91.728312\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"265.615368\" xlink:href=\"#C0_0_79f0254025\" y=\"95.41168\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"94.352637\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.615452\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"98.467085\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.770982\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"90.217115\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"92.892234\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"87.221943\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"261.67886\" xlink:href=\"#C0_0_79f0254025\" y=\"82.492318\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.305988\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.305988\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.256287\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"93.748539\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"94.550653\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.782761\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"93.543563\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.046366\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.048558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C0_0_79f0254025\" y=\"100.041708\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.425958\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.666774\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.082842\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.197055\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.710401\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.044143\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"95.171943\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.98271\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"95.331611\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.402302\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"92.688843\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"86.210568\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"253.805844\" xlink:href=\"#C0_0_79f0254025\" y=\"90.684199\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"88.812174\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"253.805844\" xlink:href=\"#C0_0_79f0254025\" y=\"87.581126\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.782761\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.252431\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"95.178727\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"277.424892\" xlink:href=\"#C0_0_79f0254025\" y=\"88.352714\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"95.056941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C0_0_79f0254025\" y=\"89.945867\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"92.039102\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"99.714734\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"91.296705\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"96.796027\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"95.056941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"94.896889\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.295615\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"91.728312\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.847527\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"265.615368\" xlink:href=\"#C0_0_79f0254025\" y=\"104.596992\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.208078\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"98.56551\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"96.045168\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"91.728312\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.509989\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"98.788187\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"95.072057\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.624329\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.11786\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"94.541575\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.606463\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.688091\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"90.85184\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"95.178727\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"94.207913\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"96.045168\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.415389\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.415389\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.677344\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.837218\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.005216\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.772824\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.209935\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.085236\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.046366\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"94.256388\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.232442\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.196214\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"97.036234\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.65729\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.65729\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.65729\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"87.428316\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"86.41959\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"253.805844\" xlink:href=\"#C0_0_79f0254025\" y=\"73.060954\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"87.428316\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.966033\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"94.231849\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.567327\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"100.185768\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.863449\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"257.742352\" xlink:href=\"#C0_0_79f0254025\" y=\"134.52\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"97.879889\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"91.999899\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"97.658055\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"95.511582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"94.847308\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.497739\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.046366\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"95.145108\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C0_0_79f0254025\" y=\"95.310814\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.209935\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"100.664416\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.882798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.184894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.013347\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.046366\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"92.453852\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"94.207913\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.175958\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.175958\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.509989\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"92.453852\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C0_0_79f0254025\" y=\"94.973611\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"95.989646\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"87.428316\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.866231\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.299619\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.299619\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.299619\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.299619\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.293741\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.208936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.119214\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.3897\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"93.813234\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"94.011342\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p3d2d5e247e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C0_0_79f0254025\" y=\"95.018766\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_18\">\n",
       "    <path d=\"M 225.758225 137.54 \n",
       "L 225.758225 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_19\">\n",
       "    <path d=\"M 411.758225 137.54 \n",
       "L 411.758225 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_20\">\n",
       "    <path d=\"M 225.758225 137.54 \n",
       "L 411.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_21\">\n",
       "    <path d=\"M 225.758225 10.7 \n",
       "L 411.758225 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_3\">\n",
       "   <g id=\"patch_22\">\n",
       "    <path d=\"M 411.758225 137.54 \n",
       "L 597.758225 137.54 \n",
       "L 597.758225 10.7 \n",
       "L 411.758225 10.7 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_2\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C1_0_eea7825ce2\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.75341\" xlink:href=\"#C1_0_eea7825ce2\" y=\"100.244856\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.75341\" xlink:href=\"#C1_0_eea7825ce2\" y=\"89.250237\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.75341\" xlink:href=\"#C1_0_eea7825ce2\" y=\"89.250237\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"452.013442\" xlink:href=\"#C1_0_eea7825ce2\" y=\"87.626196\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.949074\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"553.52227\" xlink:href=\"#C1_0_eea7825ce2\" y=\"13.72\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.75341\" xlink:href=\"#C1_0_eea7825ce2\" y=\"98.842495\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"593.329654\" xlink:href=\"#C1_0_eea7825ce2\" y=\"103.666279\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"509.734148\" xlink:href=\"#C1_0_eea7825ce2\" y=\"110.02958\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"534.613763\" xlink:href=\"#C1_0_eea7825ce2\" y=\"100.522774\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"510.729333\" xlink:href=\"#C1_0_eea7825ce2\" y=\"90.152935\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"486.844903\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.73516\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C1_0_eea7825ce2\" y=\"97.036234\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C1_0_eea7825ce2\" y=\"98.864658\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.153089\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.955756\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.143458\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.860388\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.892234\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"486.844903\" xlink:href=\"#C1_0_eea7825ce2\" y=\"99.545689\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.838425\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"551.531901\" xlink:href=\"#C1_0_eea7825ce2\" y=\"81.954361\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"532.623394\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.58209\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"432.10975\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.753337\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.809072\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.888092\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.796027\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"508.738964\" xlink:href=\"#C1_0_eea7825ce2\" y=\"85.759356\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"507.743779\" xlink:href=\"#C1_0_eea7825ce2\" y=\"84.263891\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.296705\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.75341\" xlink:href=\"#C1_0_eea7825ce2\" y=\"89.250237\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.153089\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.038852\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.316097\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.415389\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.763743\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.049442\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.04329\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.04329\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.509989\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.292342\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"453.008627\" xlink:href=\"#C1_0_eea7825ce2\" y=\"88.607187\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.088289\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"486.844903\" xlink:href=\"#C1_0_eea7825ce2\" y=\"88.322418\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.989901\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"486.844903\" xlink:href=\"#C1_0_eea7825ce2\" y=\"90.5625\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"493.811195\" xlink:href=\"#C1_0_eea7825ce2\" y=\"99.577534\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"512.719702\" xlink:href=\"#C1_0_eea7825ce2\" y=\"102.121607\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"435.095304\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.728312\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"492.81601\" xlink:href=\"#C1_0_eea7825ce2\" y=\"98.424198\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"442.061596\" xlink:href=\"#C1_0_eea7825ce2\" y=\"88.812174\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.714379\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.714379\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.954947\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.119381\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.439673\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.935777\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.541587\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"451.018257\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.671483\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.444746\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"449.027888\" xlink:href=\"#C1_0_eea7825ce2\" y=\"87.400601\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.594994\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"447.037519\" xlink:href=\"#C1_0_eea7825ce2\" y=\"97.730054\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.398988\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.124196\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.457134\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"464.950842\" xlink:href=\"#C1_0_eea7825ce2\" y=\"100.764046\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.296705\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.132422\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.044209\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.157904\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.081043\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"504.758225\" xlink:href=\"#C1_0_eea7825ce2\" y=\"106.092432\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"455.99418\" xlink:href=\"#C1_0_eea7825ce2\" y=\"110.245848\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.153089\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.582503\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.175657\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.157904\" xlink:href=\"#C1_0_eea7825ce2\" y=\"97.89088\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.986918\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.760854\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"496.796749\" xlink:href=\"#C1_0_eea7825ce2\" y=\"88.630432\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.148273\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.730976\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.090991\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.884654\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.305988\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.395396\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"455.99418\" xlink:href=\"#C1_0_eea7825ce2\" y=\"90.578761\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.304438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.153089\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.931038\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"451.018257\" xlink:href=\"#C1_0_eea7825ce2\" y=\"78.268008\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.796027\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.110357\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.718795\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.157904\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.992512\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"431.114566\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.234698\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.032283\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"470.921949\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.738637\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"435.095304\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.728312\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.124196\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.41168\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.153089\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.352637\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.615452\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"435.095304\" xlink:href=\"#C1_0_eea7825ce2\" y=\"98.467085\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.770982\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C1_0_eea7825ce2\" y=\"90.217115\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"487.840087\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.892234\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"455.99418\" xlink:href=\"#C1_0_eea7825ce2\" y=\"87.221943\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"473.907503\" xlink:href=\"#C1_0_eea7825ce2\" y=\"82.492318\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.305988\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.305988\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.256287\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.748539\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.550653\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.782761\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.148273\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.543563\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.046366\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.048558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"467.936395\" xlink:href=\"#C1_0_eea7825ce2\" y=\"100.041708\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.425958\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.666774\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.082842\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.197055\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.710401\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.044143\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.171943\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.98271\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.331611\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.402302\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.688843\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"450.023073\" xlink:href=\"#C1_0_eea7825ce2\" y=\"86.210568\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"455.99418\" xlink:href=\"#C1_0_eea7825ce2\" y=\"90.684199\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"442.061596\" xlink:href=\"#C1_0_eea7825ce2\" y=\"88.812174\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"461.965288\" xlink:href=\"#C1_0_eea7825ce2\" y=\"87.581126\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.782761\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.252431\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.178727\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"484.854533\" xlink:href=\"#C1_0_eea7825ce2\" y=\"88.352714\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.056941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"509.734148\" xlink:href=\"#C1_0_eea7825ce2\" y=\"89.945867\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"494.806379\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.039102\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"488.835272\" xlink:href=\"#C1_0_eea7825ce2\" y=\"99.714734\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.296705\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.796027\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.056941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.896889\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.295615\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"435.095304\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.728312\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.847527\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"494.806379\" xlink:href=\"#C1_0_eea7825ce2\" y=\"104.596992\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.208078\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.143458\" xlink:href=\"#C1_0_eea7825ce2\" y=\"98.56551\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.119381\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.045168\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"435.095304\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.728312\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.509989\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C1_0_eea7825ce2\" y=\"98.788187\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.072057\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.624329\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.11786\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.541575\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.606463\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.688091\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C1_0_eea7825ce2\" y=\"90.85184\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.178727\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.207913\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.119381\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.045168\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.415389\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.415389\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.677344\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.837218\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.005216\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.772824\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.209935\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.085236\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.046366\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.256388\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.232442\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.196214\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C1_0_eea7825ce2\" y=\"97.036234\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.65729\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.65729\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.65729\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"447.037519\" xlink:href=\"#C1_0_eea7825ce2\" y=\"87.428316\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C1_0_eea7825ce2\" y=\"86.41959\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"470.921949\" xlink:href=\"#C1_0_eea7825ce2\" y=\"73.060954\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"447.037519\" xlink:href=\"#C1_0_eea7825ce2\" y=\"87.428316\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.124196\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.966033\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.231849\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.567327\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"445.04715\" xlink:href=\"#C1_0_eea7825ce2\" y=\"100.185768\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"445.04715\" xlink:href=\"#C1_0_eea7825ce2\" y=\"96.092834\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.863449\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"452.013442\" xlink:href=\"#C1_0_eea7825ce2\" y=\"134.52\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.143458\" xlink:href=\"#C1_0_eea7825ce2\" y=\"97.879889\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C1_0_eea7825ce2\" y=\"91.999899\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"445.04715\" xlink:href=\"#C1_0_eea7825ce2\" y=\"97.658055\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.511582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.847308\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.497739\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.046366\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.145108\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.119381\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.310814\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.209935\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"447.037519\" xlink:href=\"#C1_0_eea7825ce2\" y=\"100.664416\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.882798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.184894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.013347\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.046366\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.453852\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.207913\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.175958\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.175958\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.509989\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C1_0_eea7825ce2\" y=\"92.453852\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.973611\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.989646\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"447.037519\" xlink:href=\"#C1_0_eea7825ce2\" y=\"87.428316\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.866231\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.299619\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.299619\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.299619\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.299619\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.293741\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.208936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.119214\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.3897\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"93.813234\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C1_0_eea7825ce2\" y=\"94.011342\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pead633cdef)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C1_0_eea7825ce2\" y=\"95.018766\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_23\">\n",
       "    <path d=\"M 411.758225 137.54 \n",
       "L 411.758225 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_24\">\n",
       "    <path d=\"M 597.758225 137.54 \n",
       "L 597.758225 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_25\">\n",
       "    <path d=\"M 411.758225 137.54 \n",
       "L 597.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_26\">\n",
       "    <path d=\"M 411.758225 10.7 \n",
       "L 597.758225 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_4\">\n",
       "   <g id=\"patch_27\">\n",
       "    <path d=\"M 39.758225 264.38 \n",
       "L 225.758225 264.38 \n",
       "L 225.758225 137.54 \n",
       "L 39.758225 137.54 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_3\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C2_0_ccdf15e529\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"94.44836\" xlink:href=\"#C2_0_ccdf15e529\" y=\"239.884444\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"110.571028\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"110.571028\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"112.952545\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.281343\" xlink:href=\"#C2_0_ccdf15e529\" y=\"247.937778\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"221.329654\" xlink:href=\"#C2_0_ccdf15e529\" y=\"180.826667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"96.504803\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"89.431137\" xlink:href=\"#C2_0_ccdf15e529\" y=\"170.088889\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"80.0999\" xlink:href=\"#C2_0_ccdf15e529\" y=\"229.146667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"94.040818\" xlink:href=\"#C2_0_ccdf15e529\" y=\"140.56\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"109.247299\" xlink:href=\"#C2_0_ccdf15e529\" y=\"210.355556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.927101\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.15353\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"96.472303\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.204373\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.277051\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.230352\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"95.47363\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.309258\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"121.269805\" xlink:href=\"#C2_0_ccdf15e529\" y=\"210.355556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.218736\" xlink:href=\"#C2_0_ccdf15e529\" y=\"223.777778\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.034789\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.885886\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.770011\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.505773\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"115.690107\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"117.883077\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"107.570058\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"110.571028\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.616106\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.675964\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.996776\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.952358\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.533405\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.542426\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.542426\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.858053\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.710799\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.514006\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.010025\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.931595\" xlink:href=\"#C2_0_ccdf15e529\" y=\"226.462222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.620716\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"108.646707\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"95.426931\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"91.696266\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.937143\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"97.1182\" xlink:href=\"#C2_0_ccdf15e529\" y=\"239.884444\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.21341\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.558331\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.558331\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.20556\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.028337\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.1665\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.811717\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"107.020479\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.953725\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.28336\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.199815\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"98.136102\" xlink:href=\"#C2_0_ccdf15e529\" y=\"239.884444\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.487241\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.469145\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"93.687013\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"107.570058\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.878137\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.074664\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.419894\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"85.873391\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"79.782762\" xlink:href=\"#C2_0_ccdf15e529\" y=\"237.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.218131\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.881908\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"97.900265\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.091505\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.49018\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.479919\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.601165\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.472476\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.775052\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.157202\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.492507\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"108.622862\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"107.558719\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.707035\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"126.675527\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.505773\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.444079\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.018271\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.684059\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.795329\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.024982\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.989173\" xlink:href=\"#C2_0_ccdf15e529\" y=\"218.408889\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.937143\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.5358\" xlink:href=\"#C2_0_ccdf15e529\" y=\"237.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.088795\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.703399\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"97.05531\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.475329\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"109.153184\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.230352\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.545347\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"120.480938\" xlink:href=\"#C2_0_ccdf15e529\" y=\"239.884444\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.157202\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.157202\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.230085\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.974653\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.798423\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.92447\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.275234\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.537915\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.534702\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"94.74626\" xlink:href=\"#C2_0_ccdf15e529\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.981277\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.094554\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.484427\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.316944\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.030579\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.541176\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.887353\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.631262\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.653214\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.48238\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.528606\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"115.028443\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"108.468246\" xlink:href=\"#C2_0_ccdf15e529\" y=\"245.253333\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.21341\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.018637\" xlink:href=\"#C2_0_ccdf15e529\" y=\"245.253333\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.92447\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.235739\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.877405\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.887169\" xlink:href=\"#C2_0_ccdf15e529\" y=\"229.146667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.055993\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"109.550947\" xlink:href=\"#C2_0_ccdf15e529\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.481396\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"95.22574\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"107.570058\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.505773\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.055993\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.290696\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.638828\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.937143\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.829496\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"88.066326\" xlink:href=\"#C2_0_ccdf15e529\" y=\"237.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.300779\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"96.910978\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.606844\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.937143\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.858053\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"96.584441\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.033828\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.690383\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.433075\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.811734\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.182995\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.063295\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"108.222414\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.877405\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.301021\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.606844\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.996776\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.996776\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.079055\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.844613\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.598259\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.939042\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.764471\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.480916\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.537915\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.229936\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.265051\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.318177\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.15353\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.642047\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.642047\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.642047\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.242719\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"114.72193\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"134.311226\" xlink:href=\"#C2_0_ccdf15e529\" y=\"245.253333\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.242719\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.189303\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.265921\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.77397\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"94.535008\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.339734\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"44.186797\" xlink:href=\"#C2_0_ccdf15e529\" y=\"242.568889\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"97.916383\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.538885\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"98.241683\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.389301\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.363402\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.876016\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.537915\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.926705\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.683711\" xlink:href=\"#C2_0_ccdf15e529\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.764471\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"93.833112\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.31136\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.334776\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.586334\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.537915\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.873202\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.301021\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.34788\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.34788\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.858053\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.873202\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.17819\" xlink:href=\"#C2_0_ccdf15e529\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.688262\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.242719\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.802068\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.166542\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.166542\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.166542\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.166542\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.175162\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.765935\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.431091\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.500861\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.879784\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.589275\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p21334c744c)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.111974\" xlink:href=\"#C2_0_ccdf15e529\" y=\"261.36\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"264.044444\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(28.524042 266.811845)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_7\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"237.2\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- $10$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(24.28986 239.967401)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_8\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"210.355556\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- $20$ -->\n",
       "      <g transform=\"translate(24.28986 213.122956)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_9\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"183.511111\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- $30$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 22.09375 33.703125 \n",
       "C 31 33.703125 34.90625 25.96875 34.90625 17.03125 \n",
       "C 34.90625 5 28.5 0.390625 22.703125 0.390625 \n",
       "C 17.40625 0.390625 8.796875 3.03125 6.09375 10.828125 \n",
       "C 6.59375 10.625 7.09375 10.625 7.59375 10.625 \n",
       "C 10 10.625 11.796875 12.21875 11.796875 14.8125 \n",
       "C 11.796875 17.703125 9.59375 19 7.59375 19 \n",
       "C 5.90625 19 3.296875 18.203125 3.296875 14.46875 \n",
       "C 3.296875 5.234375 12.296875 -2 22.90625 -2 \n",
       "C 34 -2 42.5 6.71875 42.5 16.953125 \n",
       "C 42.5 26.671875 34.5 33.703125 25 34.796875 \n",
       "C 32.59375 36.375 39.90625 43.0625 39.90625 52.015625 \n",
       "C 39.90625 59.6875 32 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 59.828125 5.90625 52.015625 \n",
       "C 5.90625 48.59375 8.5 48 9.796875 48 \n",
       "C 11.90625 48 13.703125 49.296875 13.703125 51.890625 \n",
       "C 13.703125 54.46875 11.90625 55.765625 9.796875 55.765625 \n",
       "C 9.40625 55.765625 8.90625 55.765625 8.5 55.578125 \n",
       "C 11.40625 61.859375 19.296875 63 22.796875 63 \n",
       "C 26.296875 63 32.90625 61.328125 32.90625 51.875 \n",
       "C 32.90625 49.109375 32.5 44.1875 29.09375 39.84375 \n",
       "C 26.09375 36 22.703125 36 19.40625 35.6875 \n",
       "C 18.90625 35.6875 16.59375 35.453125 16.203125 35.453125 \n",
       "C 15.5 35.359375 15.09375 35.265625 15.09375 34.5 \n",
       "C 15.09375 33.796875 15.203125 33.703125 17.203125 33.703125 \n",
       "z\n",
       "\" id=\"CMR17-51\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(24.28986 186.278512)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-51\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_10\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"156.666667\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- $40$ -->\n",
       "      <g transform=\"translate(24.28986 159.434068)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_12\">\n",
       "     <!-- Number First Neighbors -->\n",
       "     <defs>\n",
       "      <path d=\"M 20.5 66.40625 \n",
       "C 19.796875 67.5 19.703125 67.5 17.796875 67.5 \n",
       "L 5.09375 67.5 \n",
       "L 5.09375 65 \n",
       "L 7 65 \n",
       "C 11 65 12.796875 64.515625 13.09375 64.40625 \n",
       "L 13.09375 9.53125 \n",
       "C 13.09375 7.046875 13.09375 2.390625 5.09375 2.390625 \n",
       "L 5.09375 -0.203125 \n",
       "C 7.40625 0 11.59375 0 14.09375 0 \n",
       "C 16.59375 0 20.796875 0 23.09375 -0.203125 \n",
       "L 23.09375 2.390625 \n",
       "C 15.09375 2.390625 15.09375 7.046875 15.09375 9.53125 \n",
       "L 15.09375 63.015625 \n",
       "C 15.796875 62.421875 15.796875 62.21875 16.5 61.234375 \n",
       "L 53.59375 1.1875 \n",
       "C 54.296875 0 54.59375 0 55.09375 0 \n",
       "C 56.09375 0 56.09375 0.296875 56.09375 2.171875 \n",
       "L 56.09375 57.890625 \n",
       "C 56.09375 60.359375 56.09375 65 64.09375 65 \n",
       "L 64.09375 67.59375 \n",
       "C 61.796875 67.5 57.59375 67.5 55.09375 67.5 \n",
       "C 52.59375 67.5 48.40625 67.5 46.09375 67.59375 \n",
       "L 46.09375 65 \n",
       "C 54.09375 65 54.09375 60.34375 54.09375 57.859375 \n",
       "L 54.09375 11.921875 \n",
       "z\n",
       "\" id=\"CMR17-78\"/>\n",
       "      <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "      <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "      <path d=\"M 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 -0.203125 \n",
       "L 11.703125 -0.203125 \n",
       "L 14.59375 6.890625 \n",
       "C 17.5 2.203125 21.796875 -1 27.59375 -1 \n",
       "C 37.59375 -1 47.703125 7.8125 47.703125 21.40625 \n",
       "C 47.703125 34.1875 38.703125 43.59375 28.59375 43.59375 \n",
       "C 22.40625 43.59375 18 40.296875 15 36.40625 \n",
       "z\n",
       "M 15.203125 31.3125 \n",
       "C 15.203125 33.09375 15.203125 33.40625 16.59375 35.5 \n",
       "C 20.5 41.515625 26.09375 42 28 42 \n",
       "C 31 42 41.296875 40.390625 41.296875 21.5 \n",
       "C 41.296875 1.6875 29.5 0.59375 27.09375 0.59375 \n",
       "C 24 0.59375 19.5 1.78125 16.296875 7.59375 \n",
       "C 15.203125 9.5 15.203125 9.703125 15.203125 11.484375 \n",
       "z\n",
       "\" id=\"CMR17-98\"/>\n",
       "      <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "z\n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "      <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "      <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "      <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.5 43.59375 16.5 37 15.296875 33.703125 \n",
       "L 15.203125 33.703125 \n",
       "L 15.203125 68.6875 \n",
       "L 3.296875 67.59375 \n",
       "L 3.296875 65 \n",
       "C 9.203125 65 10.09375 64.40625 10.09375 59.640625 \n",
       "L 10.09375 6.65625 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-104\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(18.352681 254.121857)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "      <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "      <use transform=\"translate(194.284936 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(247.782744 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(288.268404 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(353.625139 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(413.608985 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(438.480044 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(473.760812 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      <use transform=\"translate(509.562149 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      <use transform=\"translate(574.918883 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "      <use transform=\"translate(643.991313 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(684.476974 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(709.348032 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(755.038509 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(803.331425 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(856.829234 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(902.519711 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(937.800478 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_28\">\n",
       "    <path d=\"M 39.758225 264.38 \n",
       "L 39.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_29\">\n",
       "    <path d=\"M 225.758225 264.38 \n",
       "L 225.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_30\">\n",
       "    <path d=\"M 39.758225 264.38 \n",
       "L 225.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_31\">\n",
       "    <path d=\"M 39.758225 137.54 \n",
       "L 225.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_5\">\n",
       "   <g id=\"patch_32\">\n",
       "    <path d=\"M 225.758225 264.38 \n",
       "L 411.758225 264.38 \n",
       "L 411.758225 137.54 \n",
       "L 225.758225 137.54 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_33\">\n",
       "    <path clip-path=\"url(#p763dbc43fb)\" d=\"M 230.186797 264.38 \n",
       "L 247.901082 264.38 \n",
       "L 247.901082 143.58 \n",
       "L 230.186797 143.58 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_34\">\n",
       "    <path clip-path=\"url(#p763dbc43fb)\" d=\"M 247.901082 264.38 \n",
       "L 265.615368 264.38 \n",
       "L 265.615368 259.024335 \n",
       "L 247.901082 259.024335 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_35\">\n",
       "    <path clip-path=\"url(#p763dbc43fb)\" d=\"M 265.615368 264.38 \n",
       "L 283.329654 264.38 \n",
       "L 283.329654 260.809557 \n",
       "L 265.615368 260.809557 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_36\">\n",
       "    <path clip-path=\"url(#p763dbc43fb)\" d=\"M 283.329654 264.38 \n",
       "L 301.04394 264.38 \n",
       "L 301.04394 263.189852 \n",
       "L 283.329654 263.189852 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_37\">\n",
       "    <path clip-path=\"url(#p763dbc43fb)\" d=\"M 301.04394 264.38 \n",
       "L 318.758225 264.38 \n",
       "L 318.758225 263.189852 \n",
       "L 301.04394 263.189852 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_38\">\n",
       "    <path clip-path=\"url(#p763dbc43fb)\" d=\"M 318.758225 264.38 \n",
       "L 336.472511 264.38 \n",
       "L 336.472511 264.38 \n",
       "L 318.758225 264.38 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_39\">\n",
       "    <path clip-path=\"url(#p763dbc43fb)\" d=\"M 336.472511 264.38 \n",
       "L 354.186797 264.38 \n",
       "L 354.186797 263.784926 \n",
       "L 336.472511 263.784926 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_40\">\n",
       "    <path clip-path=\"url(#p763dbc43fb)\" d=\"M 354.186797 264.38 \n",
       "L 371.901082 264.38 \n",
       "L 371.901082 263.784926 \n",
       "L 354.186797 263.784926 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_41\">\n",
       "    <path clip-path=\"url(#p763dbc43fb)\" d=\"M 371.901082 264.38 \n",
       "L 389.615368 264.38 \n",
       "L 389.615368 264.38 \n",
       "L 371.901082 264.38 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_42\">\n",
       "    <path clip-path=\"url(#p763dbc43fb)\" d=\"M 389.615368 264.38 \n",
       "L 407.329654 264.38 \n",
       "L 407.329654 263.784926 \n",
       "L 389.615368 263.784926 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_43\">\n",
       "    <path d=\"M 225.758225 264.38 \n",
       "L 225.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_44\">\n",
       "    <path d=\"M 411.758225 264.38 \n",
       "L 411.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_45\">\n",
       "    <path d=\"M 225.758225 264.38 \n",
       "L 411.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_46\">\n",
       "    <path d=\"M 225.758225 137.54 \n",
       "L 411.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_6\">\n",
       "   <g id=\"patch_47\">\n",
       "    <path d=\"M 411.758225 264.38 \n",
       "L 597.758225 264.38 \n",
       "L 597.758225 137.54 \n",
       "L 411.758225 137.54 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_4\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C3_0_85fe3a0087\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.75341\" xlink:href=\"#C3_0_85fe3a0087\" y=\"239.884444\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.75341\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.75341\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"452.013442\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C3_0_85fe3a0087\" y=\"247.937778\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"553.52227\" xlink:href=\"#C3_0_85fe3a0087\" y=\"180.826667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.75341\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"593.329654\" xlink:href=\"#C3_0_85fe3a0087\" y=\"170.088889\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"509.734148\" xlink:href=\"#C3_0_85fe3a0087\" y=\"229.146667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"534.613763\" xlink:href=\"#C3_0_85fe3a0087\" y=\"140.56\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"510.729333\" xlink:href=\"#C3_0_85fe3a0087\" y=\"210.355556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"486.844903\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.153089\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.143458\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"486.844903\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"551.531901\" xlink:href=\"#C3_0_85fe3a0087\" y=\"210.355556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"532.623394\" xlink:href=\"#C3_0_85fe3a0087\" y=\"223.777778\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"432.10975\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"508.738964\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"507.743779\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.75341\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.153089\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"453.008627\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"486.844903\" xlink:href=\"#C3_0_85fe3a0087\" y=\"226.462222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"486.844903\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"493.811195\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"512.719702\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"435.095304\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"492.81601\" xlink:href=\"#C3_0_85fe3a0087\" y=\"239.884444\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"442.061596\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.119381\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"451.018257\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"449.027888\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"447.037519\" xlink:href=\"#C3_0_85fe3a0087\" y=\"239.884444\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.124196\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"464.950842\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.157904\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"504.758225\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"455.99418\" xlink:href=\"#C3_0_85fe3a0087\" y=\"237.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.153089\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.157904\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"496.796749\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.148273\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"455.99418\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.153089\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"451.018257\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.157904\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"431.114566\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"470.921949\" xlink:href=\"#C3_0_85fe3a0087\" y=\"218.408889\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"435.095304\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.124196\" xlink:href=\"#C3_0_85fe3a0087\" y=\"237.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.153089\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"435.095304\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"427.133827\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"487.840087\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"455.99418\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"473.907503\" xlink:href=\"#C3_0_85fe3a0087\" y=\"239.884444\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.148273\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"467.936395\" xlink:href=\"#C3_0_85fe3a0087\" y=\"250.622222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"450.023073\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"455.99418\" xlink:href=\"#C3_0_85fe3a0087\" y=\"245.253333\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"442.061596\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"461.965288\" xlink:href=\"#C3_0_85fe3a0087\" y=\"245.253333\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"484.854533\" xlink:href=\"#C3_0_85fe3a0087\" y=\"229.146667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"509.734148\" xlink:href=\"#C3_0_85fe3a0087\" y=\"253.306667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"494.806379\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"488.835272\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"485.849718\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"435.095304\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"494.806379\" xlink:href=\"#C3_0_85fe3a0087\" y=\"237.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.143458\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.119381\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"435.095304\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.119381\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"447.037519\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"470.921949\" xlink:href=\"#C3_0_85fe3a0087\" y=\"245.253333\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"447.037519\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.124196\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.17235\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"445.04715\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"445.04715\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"452.013442\" xlink:href=\"#C3_0_85fe3a0087\" y=\"242.568889\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.143458\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.056781\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"445.04715\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.119381\" xlink:href=\"#C3_0_85fe3a0087\" y=\"255.991111\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"447.037519\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.16272\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"258.675556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.167535\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"447.037519\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"418.177166\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"416.186797\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p45ec344943)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"417.181981\" xlink:href=\"#C3_0_85fe3a0087\" y=\"261.36\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_48\">\n",
       "    <path d=\"M 411.758225 264.38 \n",
       "L 411.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_49\">\n",
       "    <path d=\"M 597.758225 264.38 \n",
       "L 597.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_50\">\n",
       "    <path d=\"M 411.758225 264.38 \n",
       "L 597.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_51\">\n",
       "    <path d=\"M 411.758225 137.54 \n",
       "L 597.758225 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_7\">\n",
       "   <g id=\"patch_52\">\n",
       "    <path d=\"M 39.758225 391.22 \n",
       "L 225.758225 391.22 \n",
       "L 225.758225 264.38 \n",
       "L 39.758225 264.38 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_5\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C4_0_91dfff0daa\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"94.44836\" xlink:href=\"#C4_0_91dfff0daa\" y=\"327.121348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"110.571028\" xlink:href=\"#C4_0_91dfff0daa\" y=\"327.121348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"110.571028\" xlink:href=\"#C4_0_91dfff0daa\" y=\"327.121348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"112.952545\" xlink:href=\"#C4_0_91dfff0daa\" y=\"363.768539\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.281343\" xlink:href=\"#C4_0_91dfff0daa\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"221.329654\" xlink:href=\"#C4_0_91dfff0daa\" y=\"294.546067\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"96.504803\" xlink:href=\"#C4_0_91dfff0daa\" y=\"327.121348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"89.431137\" xlink:href=\"#C4_0_91dfff0daa\" y=\"267.4\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"80.0999\" xlink:href=\"#C4_0_91dfff0daa\" y=\"324.406742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"94.040818\" xlink:href=\"#C4_0_91dfff0daa\" y=\"307.440449\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"109.247299\" xlink:href=\"#C4_0_91dfff0daa\" y=\"323.72809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.927101\" xlink:href=\"#C4_0_91dfff0daa\" y=\"340.01573\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.15353\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"96.472303\" xlink:href=\"#C4_0_91dfff0daa\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.204373\" xlink:href=\"#C4_0_91dfff0daa\" y=\"383.449438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.277051\" xlink:href=\"#C4_0_91dfff0daa\" y=\"382.092135\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.230352\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"95.47363\" xlink:href=\"#C4_0_91dfff0daa\" y=\"340.01573\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.309258\" xlink:href=\"#C4_0_91dfff0daa\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"121.269805\" xlink:href=\"#C4_0_91dfff0daa\" y=\"295.903371\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.218736\" xlink:href=\"#C4_0_91dfff0daa\" y=\"308.797753\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.034789\" xlink:href=\"#C4_0_91dfff0daa\" y=\"377.341573\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.885886\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.770011\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.505773\" xlink:href=\"#C4_0_91dfff0daa\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"115.690107\" xlink:href=\"#C4_0_91dfff0daa\" y=\"325.085393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"117.883077\" xlink:href=\"#C4_0_91dfff0daa\" y=\"325.764045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"107.570058\" xlink:href=\"#C4_0_91dfff0daa\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"110.571028\" xlink:href=\"#C4_0_91dfff0daa\" y=\"327.121348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.616106\" xlink:href=\"#C4_0_91dfff0daa\" y=\"383.449438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.675964\" xlink:href=\"#C4_0_91dfff0daa\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.996776\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.952358\" xlink:href=\"#C4_0_91dfff0daa\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.533405\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.542426\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.542426\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.858053\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.710799\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.514006\" xlink:href=\"#C4_0_91dfff0daa\" y=\"363.089888\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.010025\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.931595\" xlink:href=\"#C4_0_91dfff0daa\" y=\"340.01573\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.620716\" xlink:href=\"#C4_0_91dfff0daa\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"108.646707\" xlink:href=\"#C4_0_91dfff0daa\" y=\"340.01573\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"95.426931\" xlink:href=\"#C4_0_91dfff0daa\" y=\"335.265169\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"91.696266\" xlink:href=\"#C4_0_91dfff0daa\" y=\"322.370787\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.937143\" xlink:href=\"#C4_0_91dfff0daa\" y=\"375.305618\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"97.1182\" xlink:href=\"#C4_0_91dfff0daa\" y=\"335.94382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.21341\" xlink:href=\"#C4_0_91dfff0daa\" y=\"370.555056\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.558331\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.558331\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.20556\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.028337\" xlink:href=\"#C4_0_91dfff0daa\" y=\"378.698876\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.1665\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.811717\" xlink:href=\"#C4_0_91dfff0daa\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"107.020479\" xlink:href=\"#C4_0_91dfff0daa\" y=\"364.447191\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.953725\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.28336\" xlink:href=\"#C4_0_91dfff0daa\" y=\"365.804494\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.199815\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"98.136102\" xlink:href=\"#C4_0_91dfff0daa\" y=\"367.161798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.487241\" xlink:href=\"#C4_0_91dfff0daa\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.469145\" xlink:href=\"#C4_0_91dfff0daa\" y=\"379.377528\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"93.687013\" xlink:href=\"#C4_0_91dfff0daa\" y=\"354.946067\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"107.570058\" xlink:href=\"#C4_0_91dfff0daa\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.878137\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.074664\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.419894\" xlink:href=\"#C4_0_91dfff0daa\" y=\"384.12809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"85.873391\" xlink:href=\"#C4_0_91dfff0daa\" y=\"327.8\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"79.782762\" xlink:href=\"#C4_0_91dfff0daa\" y=\"361.053933\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.218131\" xlink:href=\"#C4_0_91dfff0daa\" y=\"383.449438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.881908\" xlink:href=\"#C4_0_91dfff0daa\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"97.900265\" xlink:href=\"#C4_0_91dfff0daa\" y=\"384.12809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.091505\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.49018\" xlink:href=\"#C4_0_91dfff0daa\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.479919\" xlink:href=\"#C4_0_91dfff0daa\" y=\"333.229213\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.601165\" xlink:href=\"#C4_0_91dfff0daa\" y=\"382.770787\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.472476\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.775052\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.157202\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.492507\" xlink:href=\"#C4_0_91dfff0daa\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"108.622862\" xlink:href=\"#C4_0_91dfff0daa\" y=\"361.053933\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"107.558719\" xlink:href=\"#C4_0_91dfff0daa\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.707035\" xlink:href=\"#C4_0_91dfff0daa\" y=\"383.449438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"126.675527\" xlink:href=\"#C4_0_91dfff0daa\" y=\"364.447191\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.505773\" xlink:href=\"#C4_0_91dfff0daa\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.444079\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.018271\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.684059\" xlink:href=\"#C4_0_91dfff0daa\" y=\"384.12809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.795329\" xlink:href=\"#C4_0_91dfff0daa\" y=\"378.020225\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.024982\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.989173\" xlink:href=\"#C4_0_91dfff0daa\" y=\"350.874157\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.937143\" xlink:href=\"#C4_0_91dfff0daa\" y=\"375.305618\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.5358\" xlink:href=\"#C4_0_91dfff0daa\" y=\"379.377528\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.088795\" xlink:href=\"#C4_0_91dfff0daa\" y=\"383.449438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.703399\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"97.05531\" xlink:href=\"#C4_0_91dfff0daa\" y=\"375.305618\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.475329\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"109.153184\" xlink:href=\"#C4_0_91dfff0daa\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.230352\" xlink:href=\"#C4_0_91dfff0daa\" y=\"339.337079\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.545347\" xlink:href=\"#C4_0_91dfff0daa\" y=\"361.053933\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"120.480938\" xlink:href=\"#C4_0_91dfff0daa\" y=\"348.838202\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.157202\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.157202\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.230085\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.974653\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.798423\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.92447\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.275234\" xlink:href=\"#C4_0_91dfff0daa\" y=\"382.770787\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.537915\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.534702\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"94.74626\" xlink:href=\"#C4_0_91dfff0daa\" y=\"352.910112\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.981277\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.094554\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.484427\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.316944\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.030579\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.541176\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.887353\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.631262\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.653214\" xlink:href=\"#C4_0_91dfff0daa\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C4_0_91dfff0daa\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.48238\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.528606\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"115.028443\" xlink:href=\"#C4_0_91dfff0daa\" y=\"365.125843\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"108.468246\" xlink:href=\"#C4_0_91dfff0daa\" y=\"361.053933\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.21341\" xlink:href=\"#C4_0_91dfff0daa\" y=\"370.555056\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.018637\" xlink:href=\"#C4_0_91dfff0daa\" y=\"356.982022\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.92447\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.235739\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.877405\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"111.887169\" xlink:href=\"#C4_0_91dfff0daa\" y=\"341.373034\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.055993\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"109.550947\" xlink:href=\"#C4_0_91dfff0daa\" y=\"324.406742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.481396\" xlink:href=\"#C4_0_91dfff0daa\" y=\"334.586517\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"95.22574\" xlink:href=\"#C4_0_91dfff0daa\" y=\"338.658427\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"107.570058\" xlink:href=\"#C4_0_91dfff0daa\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.505773\" xlink:href=\"#C4_0_91dfff0daa\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.055993\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.290696\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.638828\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.937143\" xlink:href=\"#C4_0_91dfff0daa\" y=\"375.305618\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.829496\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"88.066326\" xlink:href=\"#C4_0_91dfff0daa\" y=\"334.586517\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.300779\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"96.910978\" xlink:href=\"#C4_0_91dfff0daa\" y=\"382.092135\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.606844\" xlink:href=\"#C4_0_91dfff0daa\" y=\"378.698876\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.937143\" xlink:href=\"#C4_0_91dfff0daa\" y=\"375.305618\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.858053\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"96.584441\" xlink:href=\"#C4_0_91dfff0daa\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.033828\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C4_0_91dfff0daa\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.690383\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.433075\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.811734\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.182995\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.063295\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"108.222414\" xlink:href=\"#C4_0_91dfff0daa\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.877405\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.301021\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.606844\" xlink:href=\"#C4_0_91dfff0daa\" y=\"378.698876\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.996776\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.996776\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.079055\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.844613\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.598259\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.939042\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.764471\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.480916\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.537915\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.229936\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.265051\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.318177\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C4_0_91dfff0daa\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"99.15353\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.642047\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.642047\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.642047\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.242719\" xlink:href=\"#C4_0_91dfff0daa\" y=\"367.161798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"114.72193\" xlink:href=\"#C4_0_91dfff0daa\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"134.311226\" xlink:href=\"#C4_0_91dfff0daa\" y=\"350.874157\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.242719\" xlink:href=\"#C4_0_91dfff0daa\" y=\"367.161798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.189303\" xlink:href=\"#C4_0_91dfff0daa\" y=\"379.377528\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C4_0_91dfff0daa\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.265921\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.77397\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C4_0_91dfff0daa\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"94.535008\" xlink:href=\"#C4_0_91dfff0daa\" y=\"368.519101\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.536946\" xlink:href=\"#C4_0_91dfff0daa\" y=\"368.519101\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.339734\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"44.186797\" xlink:href=\"#C4_0_91dfff0daa\" y=\"363.768539\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"97.916383\" xlink:href=\"#C4_0_91dfff0daa\" y=\"382.092135\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"106.538885\" xlink:href=\"#C4_0_91dfff0daa\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"98.241683\" xlink:href=\"#C4_0_91dfff0daa\" y=\"368.519101\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.389301\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.363402\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.876016\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.537915\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.926705\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"101.683711\" xlink:href=\"#C4_0_91dfff0daa\" y=\"378.698876\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.764471\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"93.833112\" xlink:href=\"#C4_0_91dfff0daa\" y=\"367.161798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.31136\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.334776\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.586334\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.537915\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.873202\" xlink:href=\"#C4_0_91dfff0daa\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.301021\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.34788\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.34788\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.858053\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"105.873202\" xlink:href=\"#C4_0_91dfff0daa\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.17819\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"100.688262\" xlink:href=\"#C4_0_91dfff0daa\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"113.242719\" xlink:href=\"#C4_0_91dfff0daa\" y=\"367.161798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.802068\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.166542\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.166542\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.166542\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.166542\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.175162\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.765935\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.431091\" xlink:href=\"#C4_0_91dfff0daa\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"104.500861\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.879784\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"103.589275\" xlink:href=\"#C4_0_91dfff0daa\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pfe8400baf3)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"102.111974\" xlink:href=\"#C4_0_91dfff0daa\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_3\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"m651f423596\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"69.658975\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_13\">\n",
       "      <!-- $-20$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 65.90625 23 \n",
       "C 67.59375 23 69.40625 23 69.40625 25 \n",
       "C 69.40625 27 67.59375 27 65.90625 27 \n",
       "L 11.796875 27 \n",
       "C 10.09375 27 8.296875 27 8.296875 25 \n",
       "C 8.296875 23 10.09375 23 11.796875 23 \n",
       "z\n",
       "\" id=\"CMSY10-0\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(71.651503 413.274872)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(123.177945 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"103.537915\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_14\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(105.530443 402.454183)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"137.416856\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- $20$ -->\n",
       "      <g transform=\"translate(139.409384 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_14\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"171.295796\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- $40$ -->\n",
       "      <g transform=\"translate(173.288324 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_15\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"205.174736\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- $60$ -->\n",
       "      <g transform=\"translate(207.167264 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_18\">\n",
       "     <!-- Average CMPA Score -->\n",
       "     <g transform=\"translate(85.338758 424.193371)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "      <use transform=\"translate(114.762892 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(155.248552 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(190.52932 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(236.219797 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(281.910274 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(352.471901 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "      <use transform=\"translate(418.981782 0)scale(0.996264)\" xlink:href=\"#CMR17-77\"/>\n",
       "      <use transform=\"translate(503.668722 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "      <use transform=\"translate(558.447767 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      <use transform=\"translate(657.596163 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(708.491533 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(748.977194 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(794.667671 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(829.948439 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_4\">\n",
       "    <g id=\"ytick_11\">\n",
       "     <g id=\"line2d_16\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"389.557303\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_19\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(28.524042 392.324704)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_12\">\n",
       "     <g id=\"line2d_17\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"355.624719\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_20\">\n",
       "      <!-- $50$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 11.40625 57.6875 \n",
       "C 12.40625 57.28125 16.5 56 20.703125 56 \n",
       "C 30 56 35.09375 61.109375 38 64.0625 \n",
       "C 38 64.953125 38 65.5 37.40625 65.5 \n",
       "C 37.296875 65.5 37.09375 65.5 36.296875 65.046875 \n",
       "C 32.796875 63.40625 28.703125 62.078125 23.703125 62.078125 \n",
       "C 20.703125 62.078125 16.203125 62.484375 11.296875 64.671875 \n",
       "C 10.203125 65.171875 10 65.171875 9.90625 65.171875 \n",
       "C 9.40625 65.171875 9.296875 65.0625 9.296875 63.078125 \n",
       "L 9.296875 34.421875 \n",
       "C 9.296875 32.640625 9.296875 32.140625 10.296875 32.140625 \n",
       "C 10.796875 32.140625 11 32.34375 11.5 33.03125 \n",
       "C 14.703125 37.515625 19.09375 39.40625 24.09375 39.40625 \n",
       "C 27.59375 39.40625 35.09375 37.21875 35.09375 20.140625 \n",
       "C 35.09375 16.953125 35.09375 11.171875 32.09375 6.578125 \n",
       "C 29.59375 2.484375 25.703125 0.390625 21.40625 0.390625 \n",
       "C 14.796875 0.390625 8.09375 5 6.296875 12.71875 \n",
       "C 6.703125 12.609375 7.5 12.421875 7.90625 12.421875 \n",
       "C 9.203125 12.421875 11.703125 13.125 11.703125 16.21875 \n",
       "C 11.703125 18.90625 9.796875 20 7.90625 20 \n",
       "C 5.59375 20 4.09375 18.59375 4.09375 15.796875 \n",
       "C 4.09375 7.09375 11 -2 21.59375 -2 \n",
       "C 31.90625 -2 41.703125 6.875 41.703125 19.75 \n",
       "C 41.703125 31.71875 33.90625 41 24.203125 41 \n",
       "C 19.09375 41 14.796875 39.109375 11.40625 35.53125 \n",
       "z\n",
       "\" id=\"CMR17-53\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(24.28986 358.39212)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_13\">\n",
       "     <g id=\"line2d_18\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"321.692135\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_21\">\n",
       "      <!-- $100$ -->\n",
       "      <g transform=\"translate(20.055677 324.459536)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_14\">\n",
       "     <g id=\"line2d_19\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"39.758225\" xlink:href=\"#mf5d37b9f9f\" y=\"287.759551\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_22\">\n",
       "      <!-- $150$ -->\n",
       "      <g transform=\"translate(20.055677 290.526951)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_23\">\n",
       "     <!-- Subgraph Size -->\n",
       "     <defs>\n",
       "      <path d=\"M 22 -16.609375 \n",
       "C 15.59375 -16.609375 15.203125 -16.203125 15.203125 -12.328125 \n",
       "L 15.203125 6.28125 \n",
       "C 18.09375 2.09375 22.296875 -0.59375 27.59375 -0.59375 \n",
       "C 37.59375 -0.59375 47.703125 8.171875 47.703125 21.6875 \n",
       "C 47.703125 34.234375 39 43.78125 28.703125 43.78125 \n",
       "C 22.703125 43.78125 17.796875 40.5 15 36.53125 \n",
       "L 15 43.78125 \n",
       "L 3.09375 42.6875 \n",
       "L 3.09375 40.109375 \n",
       "C 9 40.109375 9.90625 39.5 9.90625 34.734375 \n",
       "L 9.90625 -12.328125 \n",
       "C 9.90625 -16.109375 9.5 -16.609375 3.09375 -16.609375 \n",
       "L 3.09375 -19.1875 \n",
       "C 5.5 -19 9.90625 -19 12.5 -19 \n",
       "C 15.09375 -19 19.59375 -19 22 -19.1875 \n",
       "z\n",
       "M 15.203125 31.4375 \n",
       "C 15.203125 32.84375 15.203125 34.921875 19.09375 38.609375 \n",
       "C 19.59375 39.015625 23 42 28 42 \n",
       "C 35.296875 42 41.296875 32.9375 41.296875 21.59375 \n",
       "C 41.296875 10.25 34.90625 1 27 1 \n",
       "C 23.40625 1 19.5 2.6875 16.5 7.375 \n",
       "C 15.203125 9.5625 15.203125 10.15625 15.203125 11.75 \n",
       "z\n",
       "\" id=\"CMR17-112\"/>\n",
       "      <path d=\"M 36.09375 40.703125 \n",
       "C 36.90625 41.71875 36.90625 41.90625 36.90625 42.109375 \n",
       "C 36.90625 43 36.5 43 34.703125 43 \n",
       "L 4.296875 43 \n",
       "L 3.296875 27.5 \n",
       "L 5.09375 27.5 \n",
       "C 5.796875 37.203125 6.90625 41.40625 18.40625 41.40625 \n",
       "L 30.59375 41.40625 \n",
       "L 2 1.5 \n",
       "C 2 0 2.09375 0 4.203125 0 \n",
       "L 35.796875 0 \n",
       "L 37.203125 18 \n",
       "L 35.40625 18 \n",
       "C 34.59375 6.90625 33.296875 1.796875 21.09375 1.796875 \n",
       "L 8.296875 1.796875 \n",
       "z\n",
       "\" id=\"CMR17-122\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(14.118498 358.808786)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(101.79074 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(152.686109 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(198.376586 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(233.657354 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(279.347831 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "      <use transform=\"translate(330.243201 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(411.214537 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(462.109907 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(486.980966 0)scale(0.996264)\" xlink:href=\"#CMR17-122\"/>\n",
       "      <use transform=\"translate(527.466626 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_53\">\n",
       "    <path d=\"M 39.758225 391.22 \n",
       "L 39.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_54\">\n",
       "    <path d=\"M 225.758225 391.22 \n",
       "L 225.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_55\">\n",
       "    <path d=\"M 39.758225 391.22 \n",
       "L 225.758225 391.22 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_56\">\n",
       "    <path d=\"M 39.758225 264.38 \n",
       "L 225.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_8\">\n",
       "   <g id=\"patch_57\">\n",
       "    <path d=\"M 225.758225 391.22 \n",
       "L 411.758225 391.22 \n",
       "L 411.758225 264.38 \n",
       "L 225.758225 264.38 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_6\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C5_0_2e66841502\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"261.67886\" xlink:href=\"#C5_0_2e66841502\" y=\"327.121348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"327.121348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"327.121348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"363.768539\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"249.869336\" xlink:href=\"#C5_0_2e66841502\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"348.282035\" xlink:href=\"#C5_0_2e66841502\" y=\"294.546067\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"327.121348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"364.028067\" xlink:href=\"#C5_0_2e66841502\" y=\"267.4\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"277.424892\" xlink:href=\"#C5_0_2e66841502\" y=\"324.406742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"407.329654\" xlink:href=\"#C5_0_2e66841502\" y=\"307.440449\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"304.980447\" xlink:href=\"#C5_0_2e66841502\" y=\"323.72809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"340.01573\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"383.449438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"382.092135\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"340.01573\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"304.980447\" xlink:href=\"#C5_0_2e66841502\" y=\"295.903371\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"285.297908\" xlink:href=\"#C5_0_2e66841502\" y=\"308.797753\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"377.341573\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"325.085393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"325.764045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"327.121348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"383.449438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"363.089888\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"281.3614\" xlink:href=\"#C5_0_2e66841502\" y=\"340.01573\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"340.01573\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"335.265169\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"322.370787\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"375.305618\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"261.67886\" xlink:href=\"#C5_0_2e66841502\" y=\"335.94382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"370.555056\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"378.698876\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"364.447191\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"365.804494\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"261.67886\" xlink:href=\"#C5_0_2e66841502\" y=\"367.161798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"379.377528\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"354.946067\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"384.12809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"327.8\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"265.615368\" xlink:href=\"#C5_0_2e66841502\" y=\"361.053933\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"383.449438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"384.12809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"333.229213\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"382.770787\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"361.053933\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"383.449438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"364.447191\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"384.12809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"378.020225\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"293.170924\" xlink:href=\"#C5_0_2e66841502\" y=\"350.874157\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"375.305618\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"265.615368\" xlink:href=\"#C5_0_2e66841502\" y=\"379.377528\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"383.449438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"375.305618\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"380.734831\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"339.337079\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"361.053933\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"261.67886\" xlink:href=\"#C5_0_2e66841502\" y=\"348.838202\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"382.770787\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"245.932828\" xlink:href=\"#C5_0_2e66841502\" y=\"352.910112\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"365.125843\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"253.805844\" xlink:href=\"#C5_0_2e66841502\" y=\"361.053933\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"370.555056\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"253.805844\" xlink:href=\"#C5_0_2e66841502\" y=\"356.982022\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"277.424892\" xlink:href=\"#C5_0_2e66841502\" y=\"341.373034\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"241.99632\" xlink:href=\"#C5_0_2e66841502\" y=\"324.406742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"334.586517\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"338.658427\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"340.694382\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"375.305618\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"265.615368\" xlink:href=\"#C5_0_2e66841502\" y=\"334.586517\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"382.092135\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"378.698876\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"375.305618\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"378.698876\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"367.161798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"253.805844\" xlink:href=\"#C5_0_2e66841502\" y=\"350.874157\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"367.161798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"379.377528\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"386.164045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"368.519101\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"368.519101\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"257.742352\" xlink:href=\"#C5_0_2e66841502\" y=\"363.768539\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"382.092135\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"369.876404\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"368.519101\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.059813\" xlink:href=\"#C5_0_2e66841502\" y=\"378.698876\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"367.161798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"384.806742\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"234.123305\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"385.485393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"367.161798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"386.842697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p0735f583f2)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"230.186797\" xlink:href=\"#C5_0_2e66841502\" y=\"387.521348\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_5\">\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_20\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"226.250289\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_24\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(228.242817 402.454183)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_7\">\n",
       "     <g id=\"line2d_21\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"265.615368\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_25\">\n",
       "      <!-- $10$ -->\n",
       "      <g transform=\"translate(267.607896 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_8\">\n",
       "     <g id=\"line2d_22\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"304.980447\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_26\">\n",
       "      <!-- $20$ -->\n",
       "      <g transform=\"translate(306.972976 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_9\">\n",
       "     <g id=\"line2d_23\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"344.345527\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_27\">\n",
       "      <!-- $30$ -->\n",
       "      <g transform=\"translate(346.338055 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-51\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_10\">\n",
       "     <g id=\"line2d_24\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"383.710606\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_28\">\n",
       "      <!-- $40$ -->\n",
       "      <g transform=\"translate(385.703134 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_29\">\n",
       "     <!-- Number First Neighbors -->\n",
       "     <g transform=\"translate(265.596369 417.606864)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "      <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "      <use transform=\"translate(194.284936 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(247.782744 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(288.268404 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(353.625139 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(413.608985 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(438.480044 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(473.760812 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      <use transform=\"translate(509.562149 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      <use transform=\"translate(574.918883 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "      <use transform=\"translate(643.991313 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(684.476974 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(709.348032 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(755.038509 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(803.331425 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(856.829234 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(902.519711 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(937.800478 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_58\">\n",
       "    <path d=\"M 225.758225 391.22 \n",
       "L 225.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_59\">\n",
       "    <path d=\"M 411.758225 391.22 \n",
       "L 411.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_60\">\n",
       "    <path d=\"M 225.758225 391.22 \n",
       "L 411.758225 391.22 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_61\">\n",
       "    <path d=\"M 225.758225 264.38 \n",
       "L 411.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_9\">\n",
       "   <g id=\"patch_62\">\n",
       "    <path d=\"M 411.758225 391.22 \n",
       "L 597.758225 391.22 \n",
       "L 597.758225 264.38 \n",
       "L 411.758225 264.38 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_63\">\n",
       "    <path clip-path=\"url(#pafef0475b3)\" d=\"M 416.186797 391.22 \n",
       "L 433.901082 391.22 \n",
       "L 433.901082 270.42 \n",
       "L 416.186797 270.42 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_64\">\n",
       "    <path clip-path=\"url(#pafef0475b3)\" d=\"M 433.901082 391.22 \n",
       "L 451.615368 391.22 \n",
       "L 451.615368 371.351579 \n",
       "L 433.901082 371.351579 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_65\">\n",
       "    <path clip-path=\"url(#pafef0475b3)\" d=\"M 451.615368 391.22 \n",
       "L 469.329654 391.22 \n",
       "L 469.329654 383.272632 \n",
       "L 451.615368 383.272632 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_66\">\n",
       "    <path clip-path=\"url(#pafef0475b3)\" d=\"M 469.329654 391.22 \n",
       "L 487.04394 391.22 \n",
       "L 487.04394 380.093684 \n",
       "L 469.329654 380.093684 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_67\">\n",
       "    <path clip-path=\"url(#pafef0475b3)\" d=\"M 487.04394 391.22 \n",
       "L 504.758225 391.22 \n",
       "L 504.758225 385.656842 \n",
       "L 487.04394 385.656842 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_68\">\n",
       "    <path clip-path=\"url(#pafef0475b3)\" d=\"M 504.758225 391.22 \n",
       "L 522.472511 391.22 \n",
       "L 522.472511 381.683158 \n",
       "L 504.758225 381.683158 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_69\">\n",
       "    <path clip-path=\"url(#pafef0475b3)\" d=\"M 522.472511 391.22 \n",
       "L 540.186797 391.22 \n",
       "L 540.186797 389.630526 \n",
       "L 522.472511 389.630526 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_70\">\n",
       "    <path clip-path=\"url(#pafef0475b3)\" d=\"M 540.186797 391.22 \n",
       "L 557.901082 391.22 \n",
       "L 557.901082 389.630526 \n",
       "L 540.186797 389.630526 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_71\">\n",
       "    <path clip-path=\"url(#pafef0475b3)\" d=\"M 557.901082 391.22 \n",
       "L 575.615368 391.22 \n",
       "L 575.615368 391.22 \n",
       "L 557.901082 391.22 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_72\">\n",
       "    <path clip-path=\"url(#pafef0475b3)\" d=\"M 575.615368 391.22 \n",
       "L 593.329654 391.22 \n",
       "L 593.329654 390.425263 \n",
       "L 575.615368 390.425263 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_6\">\n",
       "    <g id=\"xtick_11\">\n",
       "     <g id=\"line2d_25\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"414.196428\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_30\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(416.188956 402.454183)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_12\">\n",
       "     <g id=\"line2d_26\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"463.955657\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_31\">\n",
       "      <!-- $50$ -->\n",
       "      <g transform=\"translate(465.948185 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_13\">\n",
       "     <g id=\"line2d_27\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"513.714887\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_32\">\n",
       "      <!-- $100$ -->\n",
       "      <g transform=\"translate(515.707415 410.922549)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_14\">\n",
       "     <g id=\"line2d_28\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"563.474116\" xlink:href=\"#m651f423596\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_33\">\n",
       "      <!-- $150$ -->\n",
       "      <g transform=\"translate(565.466644 410.922549)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_34\">\n",
       "     <!-- Subgraph Size -->\n",
       "     <g transform=\"translate(473.74944 421.841047)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(101.79074 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(152.686109 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(198.376586 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(233.657354 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(279.347831 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "      <use transform=\"translate(330.243201 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(411.214537 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(462.109907 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(486.980966 0)scale(0.996264)\" xlink:href=\"#CMR17-122\"/>\n",
       "      <use transform=\"translate(527.466626 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_73\">\n",
       "    <path d=\"M 411.758225 391.22 \n",
       "L 411.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_74\">\n",
       "    <path d=\"M 597.758225 391.22 \n",
       "L 597.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_75\">\n",
       "    <path d=\"M 411.758225 391.22 \n",
       "L 597.758225 391.22 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_76\">\n",
       "    <path d=\"M 411.758225 264.38 \n",
       "L 597.758225 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"p05c76295e5\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"39.758225\" y=\"10.7\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p3d2d5e247e\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"225.758225\" y=\"10.7\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pead633cdef\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"411.758225\" y=\"10.7\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p21334c744c\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"39.758225\" y=\"137.54\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p763dbc43fb\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"225.758225\" y=\"137.54\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p45ec344943\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"411.758225\" y=\"137.54\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pfe8400baf3\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"39.758225\" y=\"264.38\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p0735f583f2\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"225.758225\" y=\"264.38\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pafef0475b3\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"411.758225\" y=\"264.38\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x1155de320>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "scatter_matrix(scores_df[scores_df['Average CMPA Score'].notnull()], figsize=(10, 7))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "scores_df.to_csv(os.path.expanduser('~/Desktop/cmpa_demo.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Average CMPA Score</th>\n",
       "      <th>Number First Neighbors</th>\n",
       "      <th>Subgraph Size</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, glial cell differentiation)</th>\n",
       "      <td>-35.037175</td>\n",
       "      <td>8</td>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, production of molecular mediator involved in inflammatory response)</th>\n",
       "      <td>-14.023552</td>\n",
       "      <td>10</td>\n",
       "      <td>42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell death)</th>\n",
       "      <td>-13.836333</td>\n",
       "      <td>13</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, chronic inflammatory response)</th>\n",
       "      <td>-10.428026</td>\n",
       "      <td>5</td>\n",
       "      <td>91</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, beta-amyloid formation)</th>\n",
       "      <td>-9.133455</td>\n",
       "      <td>10</td>\n",
       "      <td>81</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, neuron apoptotic process)</th>\n",
       "      <td>-8.327757</td>\n",
       "      <td>35</td>\n",
       "      <td>180</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of neuron projection regeneration)</th>\n",
       "      <td>-6.990566</td>\n",
       "      <td>3</td>\n",
       "      <td>99</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of synaptic activity)</th>\n",
       "      <td>-5.815355</td>\n",
       "      <td>4</td>\n",
       "      <td>51</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, positive regulation of synaptic transmission, cholinergic)</th>\n",
       "      <td>-5.729107</td>\n",
       "      <td>1</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, apoptotic process)</th>\n",
       "      <td>-5.606490</td>\n",
       "      <td>46</td>\n",
       "      <td>121</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, response to oxidative stress)</th>\n",
       "      <td>-5.365903</td>\n",
       "      <td>9</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell-matrix adhesion)</th>\n",
       "      <td>-5.314752</td>\n",
       "      <td>3</td>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell migration)</th>\n",
       "      <td>-5.190042</td>\n",
       "      <td>5</td>\n",
       "      <td>54</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, vasoconstriction)</th>\n",
       "      <td>-4.906987</td>\n",
       "      <td>3</td>\n",
       "      <td>75</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of calcium-mediated signaling)</th>\n",
       "      <td>-4.788216</td>\n",
       "      <td>4</td>\n",
       "      <td>80</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, MESHPP, Lipid Peroxidation)</th>\n",
       "      <td>-4.760648</td>\n",
       "      <td>2</td>\n",
       "      <td>73</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, response to endoplasmic reticulum stress)</th>\n",
       "      <td>-4.171094</td>\n",
       "      <td>4</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, reactive oxygen species metabolic process)</th>\n",
       "      <td>-4.151908</td>\n",
       "      <td>2</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, central nervous system development)</th>\n",
       "      <td>-4.104895</td>\n",
       "      <td>3</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, beta-amyloid clearance)</th>\n",
       "      <td>-3.912128</td>\n",
       "      <td>3</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, phagocytosis)</th>\n",
       "      <td>-3.826924</td>\n",
       "      <td>5</td>\n",
       "      <td>21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, positive regulation of neuron apoptotic process)</th>\n",
       "      <td>-3.789797</td>\n",
       "      <td>9</td>\n",
       "      <td>79</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, immune response)</th>\n",
       "      <td>-3.328115</td>\n",
       "      <td>1</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, gliogenesis)</th>\n",
       "      <td>-3.318600</td>\n",
       "      <td>1</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, microglial cell activation)</th>\n",
       "      <td>-3.188892</td>\n",
       "      <td>9</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, DNA biosynthetic process)</th>\n",
       "      <td>-3.126563</td>\n",
       "      <td>3</td>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, stem cell proliferation)</th>\n",
       "      <td>-2.588266</td>\n",
       "      <td>1</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, neuronal signal transduction)</th>\n",
       "      <td>-2.588266</td>\n",
       "      <td>1</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, microglial cell activation involved in immune response)</th>\n",
       "      <td>-2.512813</td>\n",
       "      <td>6</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, leukotriene production involved in inflammatory response)</th>\n",
       "      <td>-2.380324</td>\n",
       "      <td>1</td>\n",
       "      <td>72</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of neuronal synaptic plasticity)</th>\n",
       "      <td>3.001833</td>\n",
       "      <td>4</td>\n",
       "      <td>42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, reelin-mediated signaling pathway)</th>\n",
       "      <td>3.015910</td>\n",
       "      <td>2</td>\n",
       "      <td>73</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, MESHPP, Insulin Resistance)</th>\n",
       "      <td>3.314902</td>\n",
       "      <td>4</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, memory)</th>\n",
       "      <td>3.370462</td>\n",
       "      <td>20</td>\n",
       "      <td>97</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, blood circulation)</th>\n",
       "      <td>3.549716</td>\n",
       "      <td>4</td>\n",
       "      <td>96</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, electron transport chain)</th>\n",
       "      <td>4.151908</td>\n",
       "      <td>2</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, mitochondrial calcium ion homeostasis)</th>\n",
       "      <td>4.151908</td>\n",
       "      <td>2</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, protein import into mitochondrial matrix)</th>\n",
       "      <td>4.151908</td>\n",
       "      <td>2</td>\n",
       "      <td>92</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of microtubule depolymerization)</th>\n",
       "      <td>4.531130</td>\n",
       "      <td>1</td>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, generation of neurons)</th>\n",
       "      <td>4.531130</td>\n",
       "      <td>1</td>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, long-term synaptic potentiation)</th>\n",
       "      <td>4.688461</td>\n",
       "      <td>4</td>\n",
       "      <td>83</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, ERK1 and ERK2 cascade)</th>\n",
       "      <td>4.708583</td>\n",
       "      <td>4</td>\n",
       "      <td>39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, MESHPP, Apoptosis)</th>\n",
       "      <td>4.928875</td>\n",
       "      <td>13</td>\n",
       "      <td>71</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, calcium ion import)</th>\n",
       "      <td>4.955102</td>\n",
       "      <td>14</td>\n",
       "      <td>73</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, calcium ion homeostasis)</th>\n",
       "      <td>5.557807</td>\n",
       "      <td>3</td>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, neuron development)</th>\n",
       "      <td>5.596823</td>\n",
       "      <td>7</td>\n",
       "      <td>48</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, positive regulation of calcium-mediated signaling)</th>\n",
       "      <td>5.729107</td>\n",
       "      <td>1</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, glutamate secretion)</th>\n",
       "      <td>5.729107</td>\n",
       "      <td>1</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, hormone secretion)</th>\n",
       "      <td>5.729107</td>\n",
       "      <td>1</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, long term synaptic depression)</th>\n",
       "      <td>5.753099</td>\n",
       "      <td>3</td>\n",
       "      <td>35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of catalytic activity)</th>\n",
       "      <td>5.907760</td>\n",
       "      <td>3</td>\n",
       "      <td>42</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, energy reserve metabolic process)</th>\n",
       "      <td>6.602340</td>\n",
       "      <td>3</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, neurogenesis)</th>\n",
       "      <td>6.783286</td>\n",
       "      <td>3</td>\n",
       "      <td>36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, mitochondria-nucleus signaling pathway)</th>\n",
       "      <td>7.173891</td>\n",
       "      <td>5</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, mitochondrion organization)</th>\n",
       "      <td>8.468483</td>\n",
       "      <td>5</td>\n",
       "      <td>94</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, MESHPP, Oxidative Stress)</th>\n",
       "      <td>10.002097</td>\n",
       "      <td>9</td>\n",
       "      <td>60</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cognition)</th>\n",
       "      <td>10.467795</td>\n",
       "      <td>20</td>\n",
       "      <td>138</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, brain development)</th>\n",
       "      <td>13.658994</td>\n",
       "      <td>2</td>\n",
       "      <td>37</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of neuron apoptotic process)</th>\n",
       "      <td>18.166631</td>\n",
       "      <td>7</td>\n",
       "      <td>57</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, inflammatory response)</th>\n",
       "      <td>69.536849</td>\n",
       "      <td>31</td>\n",
       "      <td>140</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>225 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/latex": [
       "\\begin{center}{\\begin{tabular}{lrrr}\n",
       "\\toprule\n",
       "{} &  Average CMPA Score &  Number First Neighbors &  Subgraph Size \\\\\n",
       "\\midrule\n",
       "(BiologicalProcess, GOBP, glial cell differenti... &          -35.037175 &                       8 &             38 \\\\\n",
       "(BiologicalProcess, GOBP, production of molecul... &          -14.023552 &                      10 &             42 \\\\\n",
       "(BiologicalProcess, GOBP, cell death)              &          -13.836333 &                      13 &             96 \\\\\n",
       "(BiologicalProcess, GOBP, chronic inflammatory ... &          -10.428026 &                       5 &             91 \\\\\n",
       "(BiologicalProcess, GOBP, beta-amyloid formation)  &           -9.133455 &                      10 &             81 \\\\\n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc... &           -8.327757 &                      35 &            180 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           -6.990566 &                       3 &             99 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of synapti... &           -5.815355 &                       4 &             51 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -5.729107 &                       1 &             33 \\\\\n",
       "(BiologicalProcess, GOBP, apoptotic process)       &           -5.606490 &                      46 &            121 \\\\\n",
       "(BiologicalProcess, GOBP, response to oxidative... &           -5.365903 &                       9 &             92 \\\\\n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)    &           -5.314752 &                       3 &             31 \\\\\n",
       "(BiologicalProcess, GOBP, cell migration)          &           -5.190042 &                       5 &             54 \\\\\n",
       "(BiologicalProcess, GOBP, vasoconstriction)        &           -4.906987 &                       3 &             75 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           -4.788216 &                       4 &             80 \\\\\n",
       "(BiologicalProcess, MESHPP, Lipid Peroxidation)    &           -4.760648 &                       2 &             73 \\\\\n",
       "(BiologicalProcess, GOBP, response to endoplasm... &           -4.171094 &                       4 &             13 \\\\\n",
       "(BiologicalProcess, GOBP, reactive oxygen speci... &           -4.151908 &                       2 &             92 \\\\\n",
       "(BiologicalProcess, GOBP, central nervous syste... &           -4.104895 &                       3 &              7 \\\\\n",
       "(BiologicalProcess, GOBP, beta-amyloid clearance)  &           -3.912128 &                       3 &             11 \\\\\n",
       "(BiologicalProcess, GOBP, phagocytosis)            &           -3.826924 &                       5 &             21 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -3.789797 &                       9 &             79 \\\\\n",
       "(BiologicalProcess, GOBP, immune response)         &           -3.328115 &                       1 &              8 \\\\\n",
       "(BiologicalProcess, GOBP, gliogenesis)             &           -3.318600 &                       1 &             11 \\\\\n",
       "(BiologicalProcess, GOBP, microglial cell activ... &           -3.188892 &                       9 &             33 \\\\\n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc... &           -3.126563 &                       3 &             31 \\\\\n",
       "(BiologicalProcess, GOBP, stem cell proliferation) &           -2.588266 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, neuronal signal trans... &           -2.588266 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, microglial cell activ... &           -2.512813 &                       6 &             13 \\\\\n",
       "(BiologicalProcess, GOBP, leukotriene productio... &           -2.380324 &                       1 &             72 \\\\\n",
       "(BiologicalProcess, GOBP, copper ion export)       &           -2.380324 &                       1 &             72 \\\\\n",
       "(BiologicalProcess, GOBP, response to reactive ... &           -2.380324 &                       1 &             72 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of synapti... &           -2.324010 &                       5 &             10 \\\\\n",
       "(BiologicalProcess, GOBP, astrocyte activation)    &           -2.071835 &                       3 &             16 \\\\\n",
       "(BiologicalProcess, GOBP, RNA metabolic process)   &           -1.771584 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, synaptic transmission... &           -1.771584 &                       2 &             31 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of growth)    &           -1.771584 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, cell adhesion)           &           -1.771584 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, growth)                  &           -1.771584 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, cell-cell signaling)     &           -1.771584 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, lymphocyte activation)   &           -1.730321 &                       2 &             16 \\\\\n",
       "(BiologicalProcess, GOBP, natural killer cell a... &           -1.730321 &                       2 &             16 \\\\\n",
       "(BiologicalProcess, GOBP, lipid oxidation)         &           -1.724853 &                       3 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -1.684737 &                       1 &              8 \\\\\n",
       "(BiologicalProcess, MESHPP, Mitochondrial Swell... &           -1.682256 &                       1 &              6 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -1.477688 &                       3 &             18 \\\\\n",
       "(BiologicalProcess, GOBP, cell cycle process)      &           -1.268407 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of microtu... &           -1.221272 &                       1 &             15 \\\\\n",
       "(BiologicalProcess, GOBP, clathrin-dependent en... &           -1.181924 &                      10 &             15 \\\\\n",
       "(BiologicalProcess, GOBP, synaptic transmission... &           -1.112609 &                       3 &              7 \\\\\n",
       "(BiologicalProcess, GOBP, cell cycle)              &           -1.099180 &                       5 &              6 \\\\\n",
       "(BiologicalProcess, GOBP, pyroptosis)              &           -1.094606 &                       3 &             16 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of cell cy... &           -1.078615 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -1.028714 &                       4 &             17 \\\\\n",
       "(BiologicalProcess, GOBP, stress response to co... &           -0.980261 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, mitochondrial depolar... &           -0.980261 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, cholesterol efflux)      &           -0.977603 &                       4 &              6 \\\\\n",
       "(BiologicalProcess, GOBP, activation of MAPKK a... &           -0.974388 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, phagocytosis, engulfm... &           -0.951158 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, neuron migration)        &           -0.901970 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, apoptotic signaling p... &           -0.887919 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, metabolic process)       &           -0.874834 &                       3 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, behavior)                &           -0.874834 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, calcium ion transport)   &           -0.863812 &                       3 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, iron assimilation by ... &           -0.841787 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of dendrit... &           -0.802697 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of short-t... &           -0.796136 &                       1 &             15 \\\\\n",
       "(BiologicalProcess, GOBP, response to unfolded ... &           -0.787240 &                       1 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, cell redox homeostasis)  &           -0.786539 &                       2 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, xenobiotic metabolic ... &           -0.736280 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, axonogenesis)            &           -0.724081 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, potassium ion import)    &           -0.707331 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, neuron remodeling)       &           -0.693359 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -0.627284 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, stress-activated prot... &           -0.618517 &                       3 &             13 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -0.578285 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, protein catabolic pro... &           -0.578285 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, response to misfolded... &           -0.528864 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, detection of misfolde... &           -0.528864 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, receptor-mediated end... &           -0.528864 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, cell cycle checkpoint)   &           -0.500330 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, plasma membrane fusion)  &           -0.492646 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, actin crosslink forma... &           -0.450985 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, insulin receptor sign... &           -0.436550 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, neuron projection dev... &           -0.428702 &                       4 &             13 \\\\\n",
       "(BiologicalProcess, GOBP, iron ion homeostasis)    &           -0.428692 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of inflamm... &           -0.401348 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, immune response to tu... &           -0.401348 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of phagocy... &           -0.401348 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, ERK5 cascade)            &           -0.390744 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, neuron differentiation)  &           -0.344869 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of cAMP-de... &           -0.328605 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, cellular response to ... &           -0.319455 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -0.319455 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, cell communication)      &           -0.319455 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -0.265133 &                       4 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, telencephalon microgl... &           -0.224749 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           -0.224749 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           -0.224749 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, viral entry into host... &           -0.219236 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -0.219236 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, MESHPP, Cell Transformation... &           -0.219236 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, MESHPP, Virus Internalization) &           -0.219236 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of cAMP bi... &           -0.214147 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, T cell activation)       &           -0.181812 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, insulin receptor sign... &           -0.181724 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, cholesterol biosynthe... &           -0.178386 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, MESHPP, Neurogenesis)          &           -0.161082 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of neuron ... &           -0.160568 &                       3 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, cell cycle arrest)       &           -0.139991 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, ATP biosynthetic proc... &           -0.139847 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           -0.139847 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, MESHPP, Longevity)             &           -0.130448 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, cAMP catabolic process)  &           -0.129720 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, T cell tolerance indu... &           -0.119921 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, PTS, AKT signaling)            &           -0.112185 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, PTS, ERK cascade)              &           -0.112185 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, cell cycle G1/S phase... &           -0.063063 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, T cell proliferation)    &           -0.061891 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, cellular response to ... &           -0.055395 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, transmission of nerve... &           -0.038631 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, ADO, loss\\_of\\_synapses)         &           -0.033649 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, insulin secretion inv... &           -0.031576 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, Notch signaling pathway) &           -0.002663 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, response to insulin)     &           -0.001897 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, necrotic cell death)     &            0.000000 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, MESHPP, Endoplasmic Reticul... &            0.000000 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of adenosi... &            0.000000 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, hydrogen peroxide-med... &            0.000000 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, MAPK cascade)            &            0.001925 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, calcium-dependent cel... &            0.002663 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of cell-ce... &            0.002663 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, JAK-STAT cascade)        &            0.028584 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, reactive oxygen speci... &            0.030319 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, PTS, APP processing pathway)   &            0.035623 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of long-te... &            0.048880 &                       3 &              6 \\\\\n",
       "(BiologicalProcess, GOBP, TOR signaling)           &            0.055106 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, MESHPP, Wnt Signaling Pathway) &            0.099837 &                       5 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of mitocho... &            0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, cell growth)             &            0.139991 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, actin polymerization ... &            0.155939 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, angiogenesis)            &            0.172131 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of MAPK ca... &            0.181055 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, focal adhesion assembly) &            0.201818 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, autophagy)               &            0.205420 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, acetylcholine secretion) &            0.228197 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &            0.228197 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, PTS, amyloid pathway)          &            0.236800 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, cell proliferation)      &            0.244661 &                       3 &              6 \\\\\n",
       "(BiologicalProcess, MESHPP, Energy Metabolism)     &            0.257822 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &            0.266394 &                      17 &             57 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &            0.283572 &                       2 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, activation of GTPase ... &            0.290838 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, lipid transport)         &            0.310151 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, immune system process)   &            0.319455 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, dopamine receptor sig... &            0.328605 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &            0.380815 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, pre-B cell receptor e... &            0.390744 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, monocyte aggregation)    &            0.401556 &                       2 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, neuron death)            &            0.401914 &                      15 &            119 \\\\\n",
       "(BiologicalProcess, GOBP, glucose import)          &            0.435266 &                       5 &             10 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &            0.557552 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, endocytosis)             &            0.560422 &                       5 &             13 \\\\\n",
       "(BiologicalProcess, GOBP, synapse assembly)        &            0.563531 &                       1 &              7 \\\\\n",
       "(BiologicalProcess, GOBP, SNARE complex assembly)  &            0.568462 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, prolactin secretion)     &            0.649910 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, PTS, amyloidogenic pathway)    &            0.724081 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, lactic acid secretion)   &            0.724081 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, PTS, Regulation of Wnt sign... &            0.724946 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, membrane lipid catabo... &            0.791182 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, long-term memory)        &            0.877871 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, synaptic vesicle budd... &            0.917142 &                       2 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, glucose metabolic pro... &            0.961414 &                       3 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of N-methy... &            0.999108 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, MESHPP, Long-Term Potentiat... &            0.999108 &                       2 &             74 \\\\\n",
       "(BiologicalProcess, GOBP, endoplasmic reticulum... &            1.026676 &                       2 &             11 \\\\\n",
       "(BiologicalProcess, GOBP, cholesterol homeostasis) &            1.045690 &                       1 &              6 \\\\\n",
       "(BiologicalProcess, GOBP, cellular senescence)     &            1.175179 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, cytokine secretion)      &            1.378606 &                       2 &              7 \\\\\n",
       "(BiologicalProcess, GOBP, cytokine production)     &            1.378606 &                       2 &              7 \\\\\n",
       "(BiologicalProcess, GOBP, chemokine biosyntheti... &            1.701339 &                       3 &              8 \\\\\n",
       "(BiologicalProcess, GOBP, vasodilation)            &            1.737646 &                       2 &             81 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of cellula... &            1.771584 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of synapti... &            2.000763 &                       2 &             73 \\\\\n",
       "(BiologicalProcess, GOBP, leukocyte mediated im... &            2.006691 &                       1 &             21 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &            2.006691 &                       1 &             21 \\\\\n",
       "(BiologicalProcess, GOBP, cAMP-mediated signaling) &            2.006691 &                       1 &             21 \\\\\n",
       "(BiologicalProcess, GOBP, zinc ion homeostasis)    &            2.006691 &                       1 &             21 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of synapti... &            2.055887 &                       5 &             37 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of dendrit... &            2.373630 &                       1 &              7 \\\\\n",
       "(BiologicalProcess, GOBP, nitric oxide-cGMP-med... &            2.380324 &                       1 &             72 \\\\\n",
       "(BiologicalProcess, GOBP, mitochondrial transport) &            2.380324 &                       1 &             72 \\\\\n",
       "(BiologicalProcess, GOBP, sodium ion homeostasis)  &            2.380324 &                       1 &             72 \\\\\n",
       "(BiologicalProcess, GOBP, amyloid fibril format... &            2.765434 &                       1 &              6 \\\\\n",
       "(BiologicalProcess, GOBP, learning)                &            2.910558 &                       7 &             42 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of neurona... &            3.001833 &                       4 &             42 \\\\\n",
       "(BiologicalProcess, GOBP, reelin-mediated signa... &            3.015910 &                       2 &             73 \\\\\n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)    &            3.314902 &                       4 &             13 \\\\\n",
       "(BiologicalProcess, GOBP, memory)                  &            3.370462 &                      20 &             97 \\\\\n",
       "(BiologicalProcess, GOBP, blood circulation)       &            3.549716 &                       4 &             96 \\\\\n",
       "(BiologicalProcess, GOBP, electron transport ch... &            4.151908 &                       2 &             92 \\\\\n",
       "(BiologicalProcess, GOBP, mitochondrial calcium... &            4.151908 &                       2 &             92 \\\\\n",
       "(BiologicalProcess, GOBP, protein import into m... &            4.151908 &                       2 &             92 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &            4.531130 &                       1 &             28 \\\\\n",
       "(BiologicalProcess, GOBP, generation of neurons)   &            4.531130 &                       1 &             28 \\\\\n",
       "(BiologicalProcess, GOBP, long-term synaptic po... &            4.688461 &                       4 &             83 \\\\\n",
       "(BiologicalProcess, GOBP, ERK1 and ERK2 cascade)   &            4.708583 &                       4 &             39 \\\\\n",
       "(BiologicalProcess, MESHPP, Apoptosis)             &            4.928875 &                      13 &             71 \\\\\n",
       "(BiologicalProcess, GOBP, calcium ion import)      &            4.955102 &                      14 &             73 \\\\\n",
       "(BiologicalProcess, GOBP, calcium ion homeostasis) &            5.557807 &                       3 &             38 \\\\\n",
       "(BiologicalProcess, GOBP, neuron development)      &            5.596823 &                       7 &             48 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &            5.729107 &                       1 &             33 \\\\\n",
       "(BiologicalProcess, GOBP, glutamate secretion)     &            5.729107 &                       1 &             33 \\\\\n",
       "(BiologicalProcess, GOBP, hormone secretion)       &            5.729107 &                       1 &             33 \\\\\n",
       "(BiologicalProcess, GOBP, long term synaptic de... &            5.753099 &                       3 &             35 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of catalyt... &            5.907760 &                       3 &             42 \\\\\n",
       "(BiologicalProcess, GOBP, energy reserve metabo... &            6.602340 &                       3 &              7 \\\\\n",
       "(BiologicalProcess, GOBP, neurogenesis)            &            6.783286 &                       3 &             36 \\\\\n",
       "(BiologicalProcess, GOBP, mitochondria-nucleus ... &            7.173891 &                       5 &             95 \\\\\n",
       "(BiologicalProcess, GOBP, mitochondrion organiz... &            8.468483 &                       5 &             94 \\\\\n",
       "(BiologicalProcess, MESHPP, Oxidative Stress)      &           10.002097 &                       9 &             60 \\\\\n",
       "(BiologicalProcess, GOBP, cognition)               &           10.467795 &                      20 &            138 \\\\\n",
       "(BiologicalProcess, GOBP, brain development)       &           13.658994 &                       2 &             37 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           18.166631 &                       7 &             57 \\\\\n",
       "(BiologicalProcess, GOBP, inflammatory response)   &           69.536849 &                      31 &            140 \\\\\n",
       "\\bottomrule\n",
       "\\end{tabular}\n",
       "}\\end{center}"
      ],
      "text/plain": [
       "                                                    Average CMPA Score  \\\n",
       "(BiologicalProcess, GOBP, glial cell differenti...          -35.037175   \n",
       "(BiologicalProcess, GOBP, production of molecul...          -14.023552   \n",
       "(BiologicalProcess, GOBP, cell death)                       -13.836333   \n",
       "(BiologicalProcess, GOBP, chronic inflammatory ...          -10.428026   \n",
       "(BiologicalProcess, GOBP, beta-amyloid formation)            -9.133455   \n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc...           -8.327757   \n",
       "(BiologicalProcess, GOBP, negative regulation o...           -6.990566   \n",
       "(BiologicalProcess, GOBP, regulation of synapti...           -5.815355   \n",
       "(BiologicalProcess, GOBP, positive regulation o...           -5.729107   \n",
       "(BiologicalProcess, GOBP, apoptotic process)                 -5.606490   \n",
       "(BiologicalProcess, GOBP, response to oxidative...           -5.365903   \n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)              -5.314752   \n",
       "(BiologicalProcess, GOBP, cell migration)                    -5.190042   \n",
       "(BiologicalProcess, GOBP, vasoconstriction)                  -4.906987   \n",
       "(BiologicalProcess, GOBP, negative regulation o...           -4.788216   \n",
       "(BiologicalProcess, MESHPP, Lipid Peroxidation)              -4.760648   \n",
       "(BiologicalProcess, GOBP, response to endoplasm...           -4.171094   \n",
       "(BiologicalProcess, GOBP, reactive oxygen speci...           -4.151908   \n",
       "(BiologicalProcess, GOBP, central nervous syste...           -4.104895   \n",
       "(BiologicalProcess, GOBP, beta-amyloid clearance)            -3.912128   \n",
       "(BiologicalProcess, GOBP, phagocytosis)                      -3.826924   \n",
       "(BiologicalProcess, GOBP, positive regulation o...           -3.789797   \n",
       "(BiologicalProcess, GOBP, immune response)                   -3.328115   \n",
       "(BiologicalProcess, GOBP, gliogenesis)                       -3.318600   \n",
       "(BiologicalProcess, GOBP, microglial cell activ...           -3.188892   \n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc...           -3.126563   \n",
       "(BiologicalProcess, GOBP, stem cell proliferation)           -2.588266   \n",
       "(BiologicalProcess, GOBP, neuronal signal trans...           -2.588266   \n",
       "(BiologicalProcess, GOBP, microglial cell activ...           -2.512813   \n",
       "(BiologicalProcess, GOBP, leukotriene productio...           -2.380324   \n",
       "...                                                                ...   \n",
       "(BiologicalProcess, GOBP, regulation of neurona...            3.001833   \n",
       "(BiologicalProcess, GOBP, reelin-mediated signa...            3.015910   \n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)               3.314902   \n",
       "(BiologicalProcess, GOBP, memory)                             3.370462   \n",
       "(BiologicalProcess, GOBP, blood circulation)                  3.549716   \n",
       "(BiologicalProcess, GOBP, electron transport ch...            4.151908   \n",
       "(BiologicalProcess, GOBP, mitochondrial calcium...            4.151908   \n",
       "(BiologicalProcess, GOBP, protein import into m...            4.151908   \n",
       "(BiologicalProcess, GOBP, negative regulation o...            4.531130   \n",
       "(BiologicalProcess, GOBP, generation of neurons)              4.531130   \n",
       "(BiologicalProcess, GOBP, long-term synaptic po...            4.688461   \n",
       "(BiologicalProcess, GOBP, ERK1 and ERK2 cascade)              4.708583   \n",
       "(BiologicalProcess, MESHPP, Apoptosis)                        4.928875   \n",
       "(BiologicalProcess, GOBP, calcium ion import)                 4.955102   \n",
       "(BiologicalProcess, GOBP, calcium ion homeostasis)            5.557807   \n",
       "(BiologicalProcess, GOBP, neuron development)                 5.596823   \n",
       "(BiologicalProcess, GOBP, positive regulation o...            5.729107   \n",
       "(BiologicalProcess, GOBP, glutamate secretion)                5.729107   \n",
       "(BiologicalProcess, GOBP, hormone secretion)                  5.729107   \n",
       "(BiologicalProcess, GOBP, long term synaptic de...            5.753099   \n",
       "(BiologicalProcess, GOBP, regulation of catalyt...            5.907760   \n",
       "(BiologicalProcess, GOBP, energy reserve metabo...            6.602340   \n",
       "(BiologicalProcess, GOBP, neurogenesis)                       6.783286   \n",
       "(BiologicalProcess, GOBP, mitochondria-nucleus ...            7.173891   \n",
       "(BiologicalProcess, GOBP, mitochondrion organiz...            8.468483   \n",
       "(BiologicalProcess, MESHPP, Oxidative Stress)                10.002097   \n",
       "(BiologicalProcess, GOBP, cognition)                         10.467795   \n",
       "(BiologicalProcess, GOBP, brain development)                 13.658994   \n",
       "(BiologicalProcess, GOBP, negative regulation o...           18.166631   \n",
       "(BiologicalProcess, GOBP, inflammatory response)             69.536849   \n",
       "\n",
       "                                                    Number First Neighbors  \\\n",
       "(BiologicalProcess, GOBP, glial cell differenti...                       8   \n",
       "(BiologicalProcess, GOBP, production of molecul...                      10   \n",
       "(BiologicalProcess, GOBP, cell death)                                   13   \n",
       "(BiologicalProcess, GOBP, chronic inflammatory ...                       5   \n",
       "(BiologicalProcess, GOBP, beta-amyloid formation)                       10   \n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc...                      35   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       3   \n",
       "(BiologicalProcess, GOBP, regulation of synapti...                       4   \n",
       "(BiologicalProcess, GOBP, positive regulation o...                       1   \n",
       "(BiologicalProcess, GOBP, apoptotic process)                            46   \n",
       "(BiologicalProcess, GOBP, response to oxidative...                       9   \n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)                          3   \n",
       "(BiologicalProcess, GOBP, cell migration)                                5   \n",
       "(BiologicalProcess, GOBP, vasoconstriction)                              3   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       4   \n",
       "(BiologicalProcess, MESHPP, Lipid Peroxidation)                          2   \n",
       "(BiologicalProcess, GOBP, response to endoplasm...                       4   \n",
       "(BiologicalProcess, GOBP, reactive oxygen speci...                       2   \n",
       "(BiologicalProcess, GOBP, central nervous syste...                       3   \n",
       "(BiologicalProcess, GOBP, beta-amyloid clearance)                        3   \n",
       "(BiologicalProcess, GOBP, phagocytosis)                                  5   \n",
       "(BiologicalProcess, GOBP, positive regulation o...                       9   \n",
       "(BiologicalProcess, GOBP, immune response)                               1   \n",
       "(BiologicalProcess, GOBP, gliogenesis)                                   1   \n",
       "(BiologicalProcess, GOBP, microglial cell activ...                       9   \n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc...                       3   \n",
       "(BiologicalProcess, GOBP, stem cell proliferation)                       1   \n",
       "(BiologicalProcess, GOBP, neuronal signal trans...                       1   \n",
       "(BiologicalProcess, GOBP, microglial cell activ...                       6   \n",
       "(BiologicalProcess, GOBP, leukotriene productio...                       1   \n",
       "...                                                                    ...   \n",
       "(BiologicalProcess, GOBP, regulation of neurona...                       4   \n",
       "(BiologicalProcess, GOBP, reelin-mediated signa...                       2   \n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)                          4   \n",
       "(BiologicalProcess, GOBP, memory)                                       20   \n",
       "(BiologicalProcess, GOBP, blood circulation)                             4   \n",
       "(BiologicalProcess, GOBP, electron transport ch...                       2   \n",
       "(BiologicalProcess, GOBP, mitochondrial calcium...                       2   \n",
       "(BiologicalProcess, GOBP, protein import into m...                       2   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       1   \n",
       "(BiologicalProcess, GOBP, generation of neurons)                         1   \n",
       "(BiologicalProcess, GOBP, long-term synaptic po...                       4   \n",
       "(BiologicalProcess, GOBP, ERK1 and ERK2 cascade)                         4   \n",
       "(BiologicalProcess, MESHPP, Apoptosis)                                  13   \n",
       "(BiologicalProcess, GOBP, calcium ion import)                           14   \n",
       "(BiologicalProcess, GOBP, calcium ion homeostasis)                       3   \n",
       "(BiologicalProcess, GOBP, neuron development)                            7   \n",
       "(BiologicalProcess, GOBP, positive regulation o...                       1   \n",
       "(BiologicalProcess, GOBP, glutamate secretion)                           1   \n",
       "(BiologicalProcess, GOBP, hormone secretion)                             1   \n",
       "(BiologicalProcess, GOBP, long term synaptic de...                       3   \n",
       "(BiologicalProcess, GOBP, regulation of catalyt...                       3   \n",
       "(BiologicalProcess, GOBP, energy reserve metabo...                       3   \n",
       "(BiologicalProcess, GOBP, neurogenesis)                                  3   \n",
       "(BiologicalProcess, GOBP, mitochondria-nucleus ...                       5   \n",
       "(BiologicalProcess, GOBP, mitochondrion organiz...                       5   \n",
       "(BiologicalProcess, MESHPP, Oxidative Stress)                            9   \n",
       "(BiologicalProcess, GOBP, cognition)                                    20   \n",
       "(BiologicalProcess, GOBP, brain development)                             2   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       7   \n",
       "(BiologicalProcess, GOBP, inflammatory response)                        31   \n",
       "\n",
       "                                                    Subgraph Size  \n",
       "(BiologicalProcess, GOBP, glial cell differenti...             38  \n",
       "(BiologicalProcess, GOBP, production of molecul...             42  \n",
       "(BiologicalProcess, GOBP, cell death)                          96  \n",
       "(BiologicalProcess, GOBP, chronic inflammatory ...             91  \n",
       "(BiologicalProcess, GOBP, beta-amyloid formation)              81  \n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc...            180  \n",
       "(BiologicalProcess, GOBP, negative regulation o...             99  \n",
       "(BiologicalProcess, GOBP, regulation of synapti...             51  \n",
       "(BiologicalProcess, GOBP, positive regulation o...             33  \n",
       "(BiologicalProcess, GOBP, apoptotic process)                  121  \n",
       "(BiologicalProcess, GOBP, response to oxidative...             92  \n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)                31  \n",
       "(BiologicalProcess, GOBP, cell migration)                      54  \n",
       "(BiologicalProcess, GOBP, vasoconstriction)                    75  \n",
       "(BiologicalProcess, GOBP, negative regulation o...             80  \n",
       "(BiologicalProcess, MESHPP, Lipid Peroxidation)                73  \n",
       "(BiologicalProcess, GOBP, response to endoplasm...             13  \n",
       "(BiologicalProcess, GOBP, reactive oxygen speci...             92  \n",
       "(BiologicalProcess, GOBP, central nervous syste...              7  \n",
       "(BiologicalProcess, GOBP, beta-amyloid clearance)              11  \n",
       "(BiologicalProcess, GOBP, phagocytosis)                        21  \n",
       "(BiologicalProcess, GOBP, positive regulation o...             79  \n",
       "(BiologicalProcess, GOBP, immune response)                      8  \n",
       "(BiologicalProcess, GOBP, gliogenesis)                         11  \n",
       "(BiologicalProcess, GOBP, microglial cell activ...             33  \n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc...             31  \n",
       "(BiologicalProcess, GOBP, stem cell proliferation)              5  \n",
       "(BiologicalProcess, GOBP, neuronal signal trans...              5  \n",
       "(BiologicalProcess, GOBP, microglial cell activ...             13  \n",
       "(BiologicalProcess, GOBP, leukotriene productio...             72  \n",
       "...                                                           ...  \n",
       "(BiologicalProcess, GOBP, regulation of neurona...             42  \n",
       "(BiologicalProcess, GOBP, reelin-mediated signa...             73  \n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)                13  \n",
       "(BiologicalProcess, GOBP, memory)                              97  \n",
       "(BiologicalProcess, GOBP, blood circulation)                   96  \n",
       "(BiologicalProcess, GOBP, electron transport ch...             92  \n",
       "(BiologicalProcess, GOBP, mitochondrial calcium...             92  \n",
       "(BiologicalProcess, GOBP, protein import into m...             92  \n",
       "(BiologicalProcess, GOBP, negative regulation o...             28  \n",
       "(BiologicalProcess, GOBP, generation of neurons)               28  \n",
       "(BiologicalProcess, GOBP, long-term synaptic po...             83  \n",
       "(BiologicalProcess, GOBP, ERK1 and ERK2 cascade)               39  \n",
       "(BiologicalProcess, MESHPP, Apoptosis)                         71  \n",
       "(BiologicalProcess, GOBP, calcium ion import)                  73  \n",
       "(BiologicalProcess, GOBP, calcium ion homeostasis)             38  \n",
       "(BiologicalProcess, GOBP, neuron development)                  48  \n",
       "(BiologicalProcess, GOBP, positive regulation o...             33  \n",
       "(BiologicalProcess, GOBP, glutamate secretion)                 33  \n",
       "(BiologicalProcess, GOBP, hormone secretion)                   33  \n",
       "(BiologicalProcess, GOBP, long term synaptic de...             35  \n",
       "(BiologicalProcess, GOBP, regulation of catalyt...             42  \n",
       "(BiologicalProcess, GOBP, energy reserve metabo...              7  \n",
       "(BiologicalProcess, GOBP, neurogenesis)                        36  \n",
       "(BiologicalProcess, GOBP, mitochondria-nucleus ...             95  \n",
       "(BiologicalProcess, GOBP, mitochondrion organiz...             94  \n",
       "(BiologicalProcess, MESHPP, Oxidative Stress)                  60  \n",
       "(BiologicalProcess, GOBP, cognition)                          138  \n",
       "(BiologicalProcess, GOBP, brain development)                   37  \n",
       "(BiologicalProcess, GOBP, negative regulation o...             57  \n",
       "(BiologicalProcess, GOBP, inflammatory response)              140  \n",
       "\n",
       "[225 rows x 3 columns]"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "scores_df[scores_df['Average CMPA Score'].notnull()].sort_values('Average CMPA Score')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Conclusions\n",
    "\n",
    "## Problems\n",
    "\n",
    "This algorithm overcomes deterministic problems with cyclic graphs by randomly removing edges. It can be run multiple times to assess the stability of the score with `pbt.analysis.cmpa.average_cmpa_run`. The effect of the default score can also be checked with a grid search.\n",
    "\n",
    "## Looking Forward\n",
    "\n",
    "While this new algorithm is generally applicable and overcomes the original algorithm, it must throw away information to do so. Further algorithmic development, such as heat diffusion approaches could allow for a more thorough investigation of the propogation of effects of the differential gene expression.\n",
    "\n",
    "This algorithm can be run over the mechanisms generated for each biological process. Further investigation into the upstream mechanism generation can yield bias-free candidate mechanisms. They can also be matched to canonical mechanisms in NeuroMMSigDB to identify concordance between the biological dogma and all possible mechanisms, then to identify previously unknown cross-talk between mechanisms."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
