{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Abstract\n",
    "\n",
    "**Author:** [Charles Tapley Hoyt](https://github.com/cthoyt)\n",
    "\n",
    "**Estimated Run Time:** 1 minute\n",
    "\n",
    "This notebook demonstrates the utilities in [PyBEL Tools](http://pybel-tools.readthedocs.io) that facilitate the exploration and expansion of subgraphs to allow for easier interpretation and contextualization of their underlying mechanisms. The data used in this notebook comes from the [AETIONOMY](http://www.aetionomy.eu/) Alzheimer's Disease (AD) knowledge assembly that has been annotated with the [NeuromMMSig Database](http://neurommsig.scai.fraunhofer.de/pathways/introduction)."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Notebook Setup"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import logging\n",
    "import os\n",
    "import sys\n",
    "import time\n",
    "from collections import Counter, defaultdict\n",
    "from operator import itemgetter\n",
    "\n",
    "import matplotlib.pyplot as plt\n",
    "import networkx as nx\n",
    "\n",
    "import pybel\n",
    "import pybel_tools as pbt\n",
    "from pybel.constants import *\n",
    "from pybel_tools.visualization import to_jupyter\n",
    "from pybel_tools.utils import barh, barv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%config InlineBackend.figure_format = 'svg'\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Notebook Provenance\n",
    "\n",
    "The time of execution and the versions of the software packegs used are displayed explicitly."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Tue Apr 25 11:27:10 2017'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "time.asctime()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.5.4-dev'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel.__version__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.1.8-dev'"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pbt.__version__"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Local Path Definitions\n",
    "\n",
    "To make this notebook interoperable across many machines, locations to the repositories that contain the data used in this notebook are referenced from the environment, set in `~/.bashrc` to point to the place where the repositories have been cloned. Assuming the repositories have been `git clone`'d into the `~/dev` folder, the entries in `~/.bashrc` should look like:\n",
    "\n",
    "```bash\n",
    "...\n",
    "export BMS_BASE=~/dev/bms\n",
    "...\n",
    "```\n",
    "\n",
    "#### BMS \n",
    "\n",
    "The biological model store (BMS) is the internal Fraunhofer SCAI repository for keeping BEL models under version control. It can be downloaded from https://tor-2.scai.fraunhofer.de/gf/project/bms/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "bms_base = os.environ['BMS_BASE']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Data\n",
    "\n",
    "The Alzheimer's Disease Knowledge Assembly has been precompiled with the following command line script, and will be loaded from this format for improved performance. In general, derived data, such as the gpickle representation of a BEL script, are not saved under version control to ensure that the most up-to-date data is always used.\n",
    "\n",
    "```sh\n",
    "pybel convert --path \"$BMS_BASE/aetionomy/alzheimers.bel\" --pickle \"$BMS_BASE/aetionomy/alzheimers.gpickle\"\n",
    "```\n",
    "\n",
    "The BEL script can also be compiled from inside this notebook with the following python code:\n",
    "\n",
    "```python\n",
    ">>> import os\n",
    ">>> import pybel\n",
    ">>> # Input from BEL script\n",
    ">>> bel_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.bel')\n",
    ">>> graph = pybel.from_path(bel_path)\n",
    ">>> # Output to gpickle for fast loading later\n",
    ">>> pickle_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.gpickle')\n",
    ">>> pybel.to_pickle(graph, pickle_path)\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pickle_path = os.path.join(bms_base, 'aetionomy', 'alzheimers', 'alzheimers.gpickle')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "graph = pybel.from_pickle(pickle_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'3.0.2'"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "graph.version"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Add all canonical names for later\n",
    "pbt.mutation.add_canonical_names(graph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Connecting Components\n",
    "\n",
    "The GABA Subgraph is explored in this example. This subgraph contains a representative group of genes, RNAs, proteins, biological processes, and pathologies; and all of their relations. It is extracted with [pbt.selection.get_subgraph_by_annotation](http://pybel-tools.readthedocs.io/en/latest/selection.html#pybel_tools.selection.group_nodes_by_annotation_filtered)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "example_subgraph_name = 'GABA subgraph'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of nodes: 61\n",
      "Number of edges: 154\n",
      "Network density: 0.04207650273224044\n",
      "Number weakly connected components: 7\n",
      "Average in-degree: 2.5245901639344264\n",
      "Average out-degree: 2.5245901639344264\n"
     ]
    }
   ],
   "source": [
    "subgraph = pbt.selection.get_subgraph_by_annotation_value(graph, annotation='Subgraph', value=example_subgraph_name)\n",
    "\n",
    "pbt.summary.print_summary(subgraph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"vgqfeaoyrdtkcsbh\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {\"name\": \"Alzheimer's Disease Knowledge Assembly - (Subgraph: GABA subgraph)\"}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"cname\": \"APP\", \"id\": [\"Protein\", \"HGNC\", \"APP\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}}], \"cname\": \"p(HGNC:GRIN2B, pmod(Ph))\", \"id\": [\"Protein\", \"HGNC\", \"GRIN2B\", [\"pmod\", [\"bel\", \"Ph\"]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"EGR1\", \"cname\": \"EGR1\", \"id\": [\"Protein\", \"HGNC\", \"EGR1\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Alzheimer Disease\", \"cname\": \"Alzheimer Disease\", \"id\": [\"Pathology\", \"MESHD\", \"Alzheimer Disease\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"synaptic transmission, cholinergic\", \"cname\": \"synaptic transmission, cholinergic\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"synaptic transmission, cholinergic\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PSEN2\", \"cname\": \"PSEN2\", \"id\": [\"Protein\", \"HGNC\", \"PSEN2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TNF\", \"cname\": \"TNF\", \"id\": [\"Protein\", \"HGNC\", \"TNF\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA1\", \"cname\": \"GABRA1\", \"id\": [\"Protein\", \"HGNC\", \"GABRA1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA2\", \"cname\": \"GABRA2\", \"id\": [\"Protein\", \"HGNC\", \"GABRA2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA3\", \"cname\": \"GABRA3\", \"id\": [\"Protein\", \"HGNC\", \"GABRA3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA4\", \"cname\": \"GABRA4\", \"id\": [\"Protein\", \"HGNC\", \"GABRA4\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA5\", \"cname\": \"GABRA5\", \"id\": [\"Protein\", \"HGNC\", \"GABRA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA6\", \"cname\": \"GABRA6\", \"id\": [\"Protein\", \"HGNC\", \"GABRA6\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB1\", \"cname\": \"GABRB1\", \"id\": [\"Protein\", \"HGNC\", \"GABRB1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB2\", \"cname\": \"GABRB2\", \"id\": [\"Protein\", \"HGNC\", \"GABRB2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB3\", \"cname\": \"GABRB3\", \"id\": [\"Protein\", \"HGNC\", \"GABRB3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG1\", \"cname\": \"GABRG1\", \"id\": [\"Protein\", \"HGNC\", \"GABRG1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG2\", \"cname\": \"GABRG2\", \"id\": [\"Protein\", \"HGNC\", \"GABRG2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG3\", \"cname\": \"GABRG3\", \"id\": [\"Protein\", \"HGNC\", \"GABRG3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRD\", \"cname\": \"GABRD\", \"id\": [\"Protein\", \"HGNC\", \"GABRD\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRE\", \"cname\": \"GABRE\", \"id\": [\"Protein\", \"HGNC\", \"GABRE\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRP\", \"cname\": \"GABRP\", \"id\": [\"Protein\", \"HGNC\", \"GABRP\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRQ\", \"cname\": \"GABRQ\", \"id\": [\"Protein\", \"HGNC\", \"GABRQ\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABBR1\", \"cname\": \"GABBR1\", \"id\": [\"Protein\", \"HGNC\", \"GABBR1\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\", \"cname\": \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of synaptic plasticity\", \"cname\": \"regulation of synaptic plasticity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of synaptic plasticity\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"cname\": \"GRIN2B\", \"id\": [\"Protein\", \"HGNC\", \"GRIN2B\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"cognition\", \"cname\": \"cognition\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"cognition\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"learning\", \"cname\": \"learning\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"learning\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"memory\", \"cname\": \"memory\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"memory\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA4\", \"cname\": \"CHRNA4\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA4\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"gamma-aminobutyric acid\", \"cname\": \"gamma-aminobutyric acid\", \"id\": [\"Abundance\", \"CHEBI\", \"gamma-aminobutyric acid\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"glutamate(1-)\", \"cname\": \"glutamate(1-)\", \"id\": [\"Abundance\", \"CHEBI\", \"glutamate(1-)\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB2\", \"cname\": \"CHRNB2\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB2\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of neuronal synaptic plasticity\", \"cname\": \"regulation of neuronal synaptic plasticity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of neuronal synaptic plasticity\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA2\", \"cname\": \"CHRNA2\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA3\", \"cname\": \"CHRNA3\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA5\", \"cname\": \"CHRNA5\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA6\", \"cname\": \"CHRNA6\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA6\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA7\", \"cname\": \"CHRNA7\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA7\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA9\", \"cname\": \"CHRNA9\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA9\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA10\", \"cname\": \"CHRNA10\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA10\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB3\", \"cname\": \"CHRNB3\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB4\", \"cname\": \"CHRNB4\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB4\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"galanthamine\", \"cname\": \"galanthamine\", \"id\": [\"Abundance\", \"CHEBI\", \"galanthamine\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"1-(3-chlorophenyl)piperazine\", \"cname\": \"1-(3-chlorophenyl)piperazine\", \"id\": [\"Abundance\", \"CHEBI\", \"1-(3-chlorophenyl)piperazine\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABBR2\", \"cname\": \"GABBR2\", \"id\": [\"Protein\", \"HGNC\", \"GABBR2\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"cAMP catabolic process\", \"cname\": \"cAMP catabolic process\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"cAMP catabolic process\"]}, {\"function\": \"Complex\", \"namespace\": \"SCOMP\", \"name\": \"CAMK2 Complex\", \"cname\": \"CAMK2 Complex\", \"id\": [\"Complex\", \"SCOMP\", \"CAMK2 Complex\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIA1\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 831}], \"cname\": \"p(HGNC:GRIA1, pmod(Ph, Ser, 831))\", \"id\": [\"Protein\", \"HGNC\", \"GRIA1\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 831]]}, {\"function\": \"Complex\", \"namespace\": \"GOCC\", \"name\": \"proteasome complex\", \"cname\": \"proteasome complex\", \"id\": [\"Complex\", \"GOCC\", \"proteasome complex\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GPHN\", \"cname\": \"GPHN\", \"id\": [\"Protein\", \"HGNC\", \"GPHN\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Glaucoma\", \"cname\": \"Glaucoma\", \"id\": [\"Pathology\", \"MESHD\", \"Glaucoma\"]}, {\"function\": \"Abundance\", \"namespace\": \"SCHEM\", \"name\": \"Glatiramer acetate\", \"cname\": \"Glatiramer acetate\", \"id\": [\"Abundance\", \"SCHEM\", \"Glatiramer acetate\"]}, {\"function\": \"Protein\", \"namespace\": \"MGI\", \"name\": \"Egr1\", \"cname\": \"Egr1\", \"id\": [\"Protein\", \"MGI\", \"Egr1\"]}, {\"function\": \"Complex\", \"cname\": \"complex(p(HGNC:EGR1), p(HGNC:PSEN2))\", \"id\": [\"Complex\", [\"Protein\", \"HGNC\", \"EGR1\"], [\"Protein\", \"HGNC\", \"PSEN2\"]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"estradiol\", \"cname\": \"estradiol\", \"id\": [\"Abundance\", \"CHEBI\", \"estradiol\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA4\", \"cname\": \"GABRA4\", \"id\": [\"RNA\", \"HGNC\", \"GABRA4\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA3\", \"cname\": \"GABRA3\", \"id\": [\"RNA\", \"HGNC\", \"GABRA3\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"Midazolam\", \"cname\": \"Midazolam\", \"id\": [\"Abundance\", \"CHEBI\", \"Midazolam\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"In immunoblotting analysis, nefiracetam treatment increased the PKCalpha activity with a bell-shaped dose-response relationship peaking at 10 nM, thereby increasing phosphorylation of PKC substrate and NMDA receptor. Nefiracetam treatment did not affect the PKA activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Pharmacol. 2007 Feb;71(2):580-7\", \"reference\": \"17095583\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"negativeCorrelation\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 4, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 4, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"Most forms of neuronal plasticity are associated with the induction of the transcription factor zif268 (egr1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 35, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 8, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 9, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 10, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 11, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 12, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 13, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 14, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 15, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 17, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 18, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 19, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 20, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 21, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 22, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 23, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 25, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Induction of zif268 in neurons leads to altered expression of proteasome subunit and proteasome-regulatory genes, thereby changing the capacity of the neuron to degrade synaptic proteins, including receptors and receptor subunits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 51, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In addition, zif268 alters the transcription of genes associated with GABA(A)R expression and trafficking, such as ubiquilin and gephyrin.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 52, \"key\": 0}, {\"relation\": \"negativeCorrelation\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 2, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 53, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, it identifies EGR1 transcription factor as a potential mediator for GA-induced neuroprotection in both glaucoma and AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Alzheimer Disease\"}, \"source\": 3, \"target\": 54, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, it identifies EGR1 transcription factor as a potential mediator for GA-induced neuroprotection in both glaucoma and AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Glaucoma\"}, \"source\": 3, \"target\": 54, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The nefiracetam-induced increase in the frequency of mEPSCs and mIPSCs over and above the level achieved by ACh alone would contribute to the therapeutic effect of nefiracetam as the cholinergic system is known to be downregulated in the brain of Alzheimer's disease patients.The therapeutic effects of galantamine are ascribed to the modest potentiation of nACh receptor and NMDA receptor activities in addition to the modest inhibition of acetylcholinesterase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 5, \"key\": 3}, {\"relation\": \"association\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 3, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 3, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 8, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 9, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 10, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 11, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 12, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 13, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 14, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 15, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 16, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 17, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 18, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 19, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 20, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 21, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 22, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 23, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 24, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"1-(3-chlorophenyl)piperazine affects the reaction [gamma-Aminobutyric Acid results in increased activity of [GABRA1 protein co-treated with GABBR2 protein co-treated with GABRG2 protein]]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"21729720\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 46, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 9, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 9, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 10, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 10, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 11, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 11, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 12, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 12, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 13, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 13, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 14, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 14, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 15, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 15, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 16, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 16, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 17, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 17, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 18, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 18, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 19, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 19, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 20, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 20, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 21, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 21, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 22, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 22, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 23, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 23, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 25, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 25, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 8, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 9, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 10, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 11, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 12, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 13, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 14, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 15, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 16, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 17, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 18, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 19, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 20, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 21, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 22, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 23, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 25, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 28, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 29, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 30, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 31, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 31, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 34, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 34, \"target\": 33, \"key\": 3}, {\"relation\": \"association\", \"evidence\": \"Most forms of neuronal plasticity are associated with the induction of the transcription factor zif268 (egr1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 35, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 36, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 36, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 37, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 37, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 38, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 38, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 39, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 39, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 40, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 40, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 41, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 41, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 42, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 42, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 43, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 43, \"target\": 33, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 44, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 44, \"target\": 33, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 36, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 37, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 31, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 38, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 39, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 40, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 41, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 42, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 34, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 43, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 44, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"1-(3-chlorophenyl)piperazine affects the reaction [gamma-Aminobutyric Acid results in increased activity of [GABRA1 protein co-treated with GABBR2 protein co-treated with GABRG2 protein]]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"21729720\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 46, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In this work, we describe a novel non-coding (nc) RNA (named 17A) RNA polymerase (pol) III-dependent embedded in the human G-protein-coupled receptor 51 gene (GPR51, GABA B2 receptor). The stable expression of 17A in SHSY5Y neuroblastoma cells induces the synthesis of an alternative splicing isoform that abolish GABA B2 intracellular signaling (i.e., inhibition of cAMP accumulation and activation of K(+) channels). Indeed, 17A is expressed in human brain, and we report that it is upregulated in cerebral tissues derived from Alzheimer disease patients. We demonstrate that 17A expression in neuroblastoma cells enhances the secretion of amyloid β peptide (Abeta) and the Abeta x-42/Î‘β x-40 peptide ratio and that its synthesis is induced in response to inflammatory stimuli.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2011 Feb;41(2):308-17. doi: 10.1016/j.nbd.2010.09.019. Epub 2010 Oct 1.\", \"reference\": \"20888417\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 47, \"target\": 48, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Nefiracetam treatment significantly enhanced long-term potentiation (LTP) with the same bell-shaped dose-response curve. Furthermore, nefiracetam-induced LTP enhancement was closely associated with calcium/calmodulin-dependent protein kinase II (CaMKII) activation with increase in phosphorylation of AMPA-type glutamate receptor subunit 1 (GluA1) (Ser-831) as a postsynaptic CaMKII substrate. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 49, \"target\": 50, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Induction of zif268 in neurons leads to altered expression of proteasome subunit and proteasome-regulatory genes, thereby changing the capacity of the neuron to degrade synaptic proteins, including receptors and receptor subunits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 51, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In addition, zif268 alters the transcription of genes associated with GABA(A)R expression and trafficking, such as ubiquilin and gephyrin.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 52, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 53, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Egr1 expression is induced following glatiramer acetate immunotherapy in rodent models of glaucoma and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Alzheimer Disease\"}, \"source\": 54, \"target\": 55, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Egr1 expression is induced following glatiramer acetate immunotherapy in rodent models of glaucoma and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Glaucoma\"}, \"source\": 54, \"target\": 55, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 54, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We show that Egr-1 binds to PSEN-P2, and that PSEN-P2 activity is increased threefold by overexpression of Egr-1, and by 12-O-tetradecanoylphorbol-13-acetate (TPA), which induces physiological Egr-1 levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 56, \"target\": 6, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Estradiol affects the expression of GABRA4 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 57, \"target\": 58, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Estradiol results in increased expression of GABRA3 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 57, \"target\": 59, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Estradiol affects the expression of GABRA4 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 58, \"target\": 57, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"GABRA6 protein mutant form results in decreased susceptibility to Midazolam\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"12031754\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 60, \"target\": 13, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The subgraph also contains elements with important unqualified edges, like the relationships between complexes and their members. These relationships can be enriched from the original graph using the function [pbt.mutation.enrich_unqualified](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.enrich_unqualified). For example, the connection between `complex(p(HGNC:EGR1), p(HGNC:PSEN2))` and `p(HGNC:PSEN2)` is added during this process. The connection between the `p(HGNC:APP)` and `p(HGNC:APP, frag(672_713))` is also recovered."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of nodes: 62\n",
      "Number of edges: 159\n",
      "Network density: 0.042041248016922265\n",
      "Number weakly connected components: 5\n",
      "Average in-degree: 2.564516129032258\n",
      "Average out-degree: 2.564516129032258\n"
     ]
    }
   ],
   "source": [
    "pbt.mutation.enrich_unqualified(graph, subgraph)\n",
    "\n",
    "pbt.summary.print_summary(subgraph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"mohdzayjrsfxvupb\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {\"name\": \"Alzheimer's Disease Knowledge Assembly - (Subgraph: GABA subgraph)\"}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"cname\": \"APP\", \"id\": [\"Protein\", \"HGNC\", \"APP\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}}], \"cname\": \"p(HGNC:GRIN2B, pmod(Ph))\", \"id\": [\"Protein\", \"HGNC\", \"GRIN2B\", [\"pmod\", [\"bel\", \"Ph\"]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"EGR1\", \"cname\": \"EGR1\", \"id\": [\"Protein\", \"HGNC\", \"EGR1\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Alzheimer Disease\", \"cname\": \"Alzheimer Disease\", \"id\": [\"Pathology\", \"MESHD\", \"Alzheimer Disease\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"synaptic transmission, cholinergic\", \"cname\": \"synaptic transmission, cholinergic\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"synaptic transmission, cholinergic\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PSEN2\", \"cname\": \"PSEN2\", \"id\": [\"Protein\", \"HGNC\", \"PSEN2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TNF\", \"cname\": \"TNF\", \"id\": [\"Protein\", \"HGNC\", \"TNF\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA1\", \"cname\": \"GABRA1\", \"id\": [\"Protein\", \"HGNC\", \"GABRA1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA2\", \"cname\": \"GABRA2\", \"id\": [\"Protein\", \"HGNC\", \"GABRA2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA3\", \"cname\": \"GABRA3\", \"id\": [\"Protein\", \"HGNC\", \"GABRA3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA4\", \"cname\": \"GABRA4\", \"id\": [\"Protein\", \"HGNC\", \"GABRA4\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA5\", \"cname\": \"GABRA5\", \"id\": [\"Protein\", \"HGNC\", \"GABRA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA6\", \"cname\": \"GABRA6\", \"id\": [\"Protein\", \"HGNC\", \"GABRA6\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB1\", \"cname\": \"GABRB1\", \"id\": [\"Protein\", \"HGNC\", \"GABRB1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB2\", \"cname\": \"GABRB2\", \"id\": [\"Protein\", \"HGNC\", \"GABRB2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB3\", \"cname\": \"GABRB3\", \"id\": [\"Protein\", \"HGNC\", \"GABRB3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG1\", \"cname\": \"GABRG1\", \"id\": [\"Protein\", \"HGNC\", \"GABRG1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG2\", \"cname\": \"GABRG2\", \"id\": [\"Protein\", \"HGNC\", \"GABRG2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG3\", \"cname\": \"GABRG3\", \"id\": [\"Protein\", \"HGNC\", \"GABRG3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRD\", \"cname\": \"GABRD\", \"id\": [\"Protein\", \"HGNC\", \"GABRD\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRE\", \"cname\": \"GABRE\", \"id\": [\"Protein\", \"HGNC\", \"GABRE\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRP\", \"cname\": \"GABRP\", \"id\": [\"Protein\", \"HGNC\", \"GABRP\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRQ\", \"cname\": \"GABRQ\", \"id\": [\"Protein\", \"HGNC\", \"GABRQ\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABBR1\", \"cname\": \"GABBR1\", \"id\": [\"Protein\", \"HGNC\", \"GABBR1\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\", \"cname\": \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of synaptic plasticity\", \"cname\": \"regulation of synaptic plasticity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of synaptic plasticity\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"cname\": \"GRIN2B\", \"id\": [\"Protein\", \"HGNC\", \"GRIN2B\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"cognition\", \"cname\": \"cognition\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"cognition\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"learning\", \"cname\": \"learning\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"learning\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"memory\", \"cname\": \"memory\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"memory\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA4\", \"cname\": \"CHRNA4\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA4\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"gamma-aminobutyric acid\", \"cname\": \"gamma-aminobutyric acid\", \"id\": [\"Abundance\", \"CHEBI\", \"gamma-aminobutyric acid\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"glutamate(1-)\", \"cname\": \"glutamate(1-)\", \"id\": [\"Abundance\", \"CHEBI\", \"glutamate(1-)\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB2\", \"cname\": \"CHRNB2\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB2\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of neuronal synaptic plasticity\", \"cname\": \"regulation of neuronal synaptic plasticity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of neuronal synaptic plasticity\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA2\", \"cname\": \"CHRNA2\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA3\", \"cname\": \"CHRNA3\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA5\", \"cname\": \"CHRNA5\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA6\", \"cname\": \"CHRNA6\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA6\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA7\", \"cname\": \"CHRNA7\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA7\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA9\", \"cname\": \"CHRNA9\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA9\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA10\", \"cname\": \"CHRNA10\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA10\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB3\", \"cname\": \"CHRNB3\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB4\", \"cname\": \"CHRNB4\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB4\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"galanthamine\", \"cname\": \"galanthamine\", \"id\": [\"Abundance\", \"CHEBI\", \"galanthamine\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"1-(3-chlorophenyl)piperazine\", \"cname\": \"1-(3-chlorophenyl)piperazine\", \"id\": [\"Abundance\", \"CHEBI\", \"1-(3-chlorophenyl)piperazine\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABBR2\", \"cname\": \"GABBR2\", \"id\": [\"Protein\", \"HGNC\", \"GABBR2\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"cAMP catabolic process\", \"cname\": \"cAMP catabolic process\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"cAMP catabolic process\"]}, {\"function\": \"Complex\", \"namespace\": \"SCOMP\", \"name\": \"CAMK2 Complex\", \"cname\": \"CAMK2 Complex\", \"id\": [\"Complex\", \"SCOMP\", \"CAMK2 Complex\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIA1\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 831}], \"cname\": \"p(HGNC:GRIA1, pmod(Ph, Ser, 831))\", \"id\": [\"Protein\", \"HGNC\", \"GRIA1\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 831]]}, {\"function\": \"Complex\", \"namespace\": \"GOCC\", \"name\": \"proteasome complex\", \"cname\": \"proteasome complex\", \"id\": [\"Complex\", \"GOCC\", \"proteasome complex\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GPHN\", \"cname\": \"GPHN\", \"id\": [\"Protein\", \"HGNC\", \"GPHN\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Glaucoma\", \"cname\": \"Glaucoma\", \"id\": [\"Pathology\", \"MESHD\", \"Glaucoma\"]}, {\"function\": \"Abundance\", \"namespace\": \"SCHEM\", \"name\": \"Glatiramer acetate\", \"cname\": \"Glatiramer acetate\", \"id\": [\"Abundance\", \"SCHEM\", \"Glatiramer acetate\"]}, {\"function\": \"Protein\", \"namespace\": \"MGI\", \"name\": \"Egr1\", \"cname\": \"Egr1\", \"id\": [\"Protein\", \"MGI\", \"Egr1\"]}, {\"function\": \"Complex\", \"cname\": \"complex(p(HGNC:EGR1), p(HGNC:PSEN2))\", \"id\": [\"Complex\", [\"Protein\", \"HGNC\", \"EGR1\"], [\"Protein\", \"HGNC\", \"PSEN2\"]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"estradiol\", \"cname\": \"estradiol\", \"id\": [\"Abundance\", \"CHEBI\", \"estradiol\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA4\", \"cname\": \"GABRA4\", \"id\": [\"RNA\", \"HGNC\", \"GABRA4\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA3\", \"cname\": \"GABRA3\", \"id\": [\"RNA\", \"HGNC\", \"GABRA3\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"Midazolam\", \"cname\": \"Midazolam\", \"id\": [\"Abundance\", \"CHEBI\", \"Midazolam\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIA1\", \"cname\": \"GRIA1\", \"id\": [\"Protein\", \"HGNC\", \"GRIA1\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"In immunoblotting analysis, nefiracetam treatment increased the PKCalpha activity with a bell-shaped dose-response relationship peaking at 10 nM, thereby increasing phosphorylation of PKC substrate and NMDA receptor. Nefiracetam treatment did not affect the PKA activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Pharmacol. 2007 Feb;71(2):580-7\", \"reference\": \"17095583\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 0, \"target\": 2, \"key\": -4}, {\"relation\": \"negativeCorrelation\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 4, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 4, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"Most forms of neuronal plasticity are associated with the induction of the transcription factor zif268 (egr1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 35, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 8, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 9, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 10, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 11, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 12, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 13, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 14, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 15, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 17, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 18, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 19, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 20, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 21, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 22, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 23, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 25, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Induction of zif268 in neurons leads to altered expression of proteasome subunit and proteasome-regulatory genes, thereby changing the capacity of the neuron to degrade synaptic proteins, including receptors and receptor subunits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 51, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In addition, zif268 alters the transcription of genes associated with GABA(A)R expression and trafficking, such as ubiquilin and gephyrin.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 52, \"key\": 0}, {\"relation\": \"negativeCorrelation\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 2, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 53, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, it identifies EGR1 transcription factor as a potential mediator for GA-induced neuroprotection in both glaucoma and AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Alzheimer Disease\"}, \"source\": 3, \"target\": 54, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, it identifies EGR1 transcription factor as a potential mediator for GA-induced neuroprotection in both glaucoma and AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Glaucoma\"}, \"source\": 3, \"target\": 54, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The nefiracetam-induced increase in the frequency of mEPSCs and mIPSCs over and above the level achieved by ACh alone would contribute to the therapeutic effect of nefiracetam as the cholinergic system is known to be downregulated in the brain of Alzheimer's disease patients.The therapeutic effects of galantamine are ascribed to the modest potentiation of nACh receptor and NMDA receptor activities in addition to the modest inhibition of acetylcholinesterase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 5, \"key\": 3}, {\"relation\": \"association\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 3, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 3, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 8, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 9, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 10, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 11, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 12, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 13, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 14, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 15, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 16, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 17, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 18, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 19, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 20, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 21, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 22, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 23, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 24, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"1-(3-chlorophenyl)piperazine affects the reaction [gamma-Aminobutyric Acid results in increased activity of [GABRA1 protein co-treated with GABBR2 protein co-treated with GABRG2 protein]]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"21729720\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 46, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 9, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 9, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 10, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 10, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 11, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 11, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 12, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 12, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 13, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 13, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 14, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 14, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 15, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 15, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 16, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 16, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 17, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 17, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 18, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 18, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 19, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 19, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 20, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 20, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 21, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 21, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 22, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 22, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 23, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 23, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 25, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 25, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 8, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 9, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 10, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 11, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 12, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 13, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 14, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 15, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 16, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 17, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 18, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 19, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 20, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 21, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 22, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 23, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 25, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 28, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 29, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 30, \"key\": 1}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 27, \"target\": 1, \"key\": -4}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 31, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 31, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 34, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 34, \"target\": 33, \"key\": 3}, {\"relation\": \"association\", \"evidence\": \"Most forms of neuronal plasticity are associated with the induction of the transcription factor zif268 (egr1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 35, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 36, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 36, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 37, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 37, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 38, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 38, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 39, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 39, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 40, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 40, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 41, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 41, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 42, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 42, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 43, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 43, \"target\": 33, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 44, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 44, \"target\": 33, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 36, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 37, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 31, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 38, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 39, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 40, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 41, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 42, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 34, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 43, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 44, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"1-(3-chlorophenyl)piperazine affects the reaction [gamma-Aminobutyric Acid results in increased activity of [GABRA1 protein co-treated with GABBR2 protein co-treated with GABRG2 protein]]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"21729720\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 46, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In this work, we describe a novel non-coding (nc) RNA (named 17A) RNA polymerase (pol) III-dependent embedded in the human G-protein-coupled receptor 51 gene (GPR51, GABA B2 receptor). The stable expression of 17A in SHSY5Y neuroblastoma cells induces the synthesis of an alternative splicing isoform that abolish GABA B2 intracellular signaling (i.e., inhibition of cAMP accumulation and activation of K(+) channels). Indeed, 17A is expressed in human brain, and we report that it is upregulated in cerebral tissues derived from Alzheimer disease patients. We demonstrate that 17A expression in neuroblastoma cells enhances the secretion of amyloid β peptide (Abeta) and the Abeta x-42/Î‘β x-40 peptide ratio and that its synthesis is induced in response to inflammatory stimuli.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2011 Feb;41(2):308-17. doi: 10.1016/j.nbd.2010.09.019. Epub 2010 Oct 1.\", \"reference\": \"20888417\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 47, \"target\": 48, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Nefiracetam treatment significantly enhanced long-term potentiation (LTP) with the same bell-shaped dose-response curve. Furthermore, nefiracetam-induced LTP enhancement was closely associated with calcium/calmodulin-dependent protein kinase II (CaMKII) activation with increase in phosphorylation of AMPA-type glutamate receptor subunit 1 (GluA1) (Ser-831) as a postsynaptic CaMKII substrate. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 49, \"target\": 50, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Induction of zif268 in neurons leads to altered expression of proteasome subunit and proteasome-regulatory genes, thereby changing the capacity of the neuron to degrade synaptic proteins, including receptors and receptor subunits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 51, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In addition, zif268 alters the transcription of genes associated with GABA(A)R expression and trafficking, such as ubiquilin and gephyrin.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 52, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 53, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Egr1 expression is induced following glatiramer acetate immunotherapy in rodent models of glaucoma and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Alzheimer Disease\"}, \"source\": 54, \"target\": 55, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Egr1 expression is induced following glatiramer acetate immunotherapy in rodent models of glaucoma and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Glaucoma\"}, \"source\": 54, \"target\": 55, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 54, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We show that Egr-1 binds to PSEN-P2, and that PSEN-P2 activity is increased threefold by overexpression of Egr-1, and by 12-O-tetradecanoylphorbol-13-acetate (TPA), which induces physiological Egr-1 levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 56, \"target\": 6, \"key\": 1}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 56, \"target\": 6, \"key\": -3}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 56, \"target\": 3, \"key\": -3}, {\"relation\": \"association\", \"evidence\": \"Estradiol affects the expression of GABRA4 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 57, \"target\": 58, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Estradiol results in increased expression of GABRA3 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 57, \"target\": 59, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Estradiol affects the expression of GABRA4 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 58, \"target\": 57, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"GABRA6 protein mutant form results in decreased susceptibility to Midazolam\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"12031754\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 60, \"target\": 13, \"key\": 0}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 61, \"target\": 50, \"key\": -4}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The graph also contains some related nodes, like `r(HGNC:GABRA5)` and `p(HGNC:GABRA5)` that are disconnected. Inferring the translation and transcriptional relationships between genes, RNAs, and proteins allows for connecting parts of the graph without much information. This can be accomplished with [pbt.mutation.infer_central_dogma](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.infer_central_dogma)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of nodes: 134\n",
      "Number of edges: 233\n",
      "Network density: 0.013073729098866569\n",
      "Number weakly connected components: 4\n",
      "Average in-degree: 1.7388059701492538\n",
      "Average out-degree: 1.7388059701492538\n"
     ]
    }
   ],
   "source": [
    "pbt.mutation.infer_central_dogma(subgraph)\n",
    "\n",
    "pbt.summary.print_summary(subgraph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"kgmhbeqjfuqzdpvc\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {\"name\": \"Alzheimer's Disease Knowledge Assembly - (Subgraph: GABA subgraph)\"}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"cname\": \"APP\", \"id\": [\"Protein\", \"HGNC\", \"APP\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}}], \"cname\": \"p(HGNC:GRIN2B, pmod(Ph))\", \"id\": [\"Protein\", \"HGNC\", \"GRIN2B\", [\"pmod\", [\"bel\", \"Ph\"]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"EGR1\", \"cname\": \"EGR1\", \"id\": [\"Protein\", \"HGNC\", \"EGR1\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Alzheimer Disease\", \"cname\": \"Alzheimer Disease\", \"id\": [\"Pathology\", \"MESHD\", \"Alzheimer Disease\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"synaptic transmission, cholinergic\", \"cname\": \"synaptic transmission, cholinergic\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"synaptic transmission, cholinergic\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PSEN2\", \"cname\": \"PSEN2\", \"id\": [\"Protein\", \"HGNC\", \"PSEN2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TNF\", \"cname\": \"TNF\", \"id\": [\"Protein\", \"HGNC\", \"TNF\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA1\", \"cname\": \"GABRA1\", \"id\": [\"Protein\", \"HGNC\", \"GABRA1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA2\", \"cname\": \"GABRA2\", \"id\": [\"Protein\", \"HGNC\", \"GABRA2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA3\", \"cname\": \"GABRA3\", \"id\": [\"Protein\", \"HGNC\", \"GABRA3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA4\", \"cname\": \"GABRA4\", \"id\": [\"Protein\", \"HGNC\", \"GABRA4\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA5\", \"cname\": \"GABRA5\", \"id\": [\"Protein\", \"HGNC\", \"GABRA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA6\", \"cname\": \"GABRA6\", \"id\": [\"Protein\", \"HGNC\", \"GABRA6\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB1\", \"cname\": \"GABRB1\", \"id\": [\"Protein\", \"HGNC\", \"GABRB1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB2\", \"cname\": \"GABRB2\", \"id\": [\"Protein\", \"HGNC\", \"GABRB2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB3\", \"cname\": \"GABRB3\", \"id\": [\"Protein\", \"HGNC\", \"GABRB3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG1\", \"cname\": \"GABRG1\", \"id\": [\"Protein\", \"HGNC\", \"GABRG1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG2\", \"cname\": \"GABRG2\", \"id\": [\"Protein\", \"HGNC\", \"GABRG2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG3\", \"cname\": \"GABRG3\", \"id\": [\"Protein\", \"HGNC\", \"GABRG3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRD\", \"cname\": \"GABRD\", \"id\": [\"Protein\", \"HGNC\", \"GABRD\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRE\", \"cname\": \"GABRE\", \"id\": [\"Protein\", \"HGNC\", \"GABRE\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRP\", \"cname\": \"GABRP\", \"id\": [\"Protein\", \"HGNC\", \"GABRP\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRQ\", \"cname\": \"GABRQ\", \"id\": [\"Protein\", \"HGNC\", \"GABRQ\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABBR1\", \"cname\": \"GABBR1\", \"id\": [\"Protein\", \"HGNC\", \"GABBR1\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\", \"cname\": \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of synaptic plasticity\", \"cname\": \"regulation of synaptic plasticity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of synaptic plasticity\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"cname\": \"GRIN2B\", \"id\": [\"Protein\", \"HGNC\", \"GRIN2B\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"cognition\", \"cname\": \"cognition\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"cognition\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"learning\", \"cname\": \"learning\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"learning\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"memory\", \"cname\": \"memory\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"memory\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA4\", \"cname\": \"CHRNA4\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA4\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"gamma-aminobutyric acid\", \"cname\": \"gamma-aminobutyric acid\", \"id\": [\"Abundance\", \"CHEBI\", \"gamma-aminobutyric acid\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"glutamate(1-)\", \"cname\": \"glutamate(1-)\", \"id\": [\"Abundance\", \"CHEBI\", \"glutamate(1-)\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB2\", \"cname\": \"CHRNB2\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB2\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of neuronal synaptic plasticity\", \"cname\": \"regulation of neuronal synaptic plasticity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of neuronal synaptic plasticity\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA2\", \"cname\": \"CHRNA2\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA3\", \"cname\": \"CHRNA3\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA5\", \"cname\": \"CHRNA5\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA6\", \"cname\": \"CHRNA6\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA6\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA7\", \"cname\": \"CHRNA7\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA7\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA9\", \"cname\": \"CHRNA9\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA9\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA10\", \"cname\": \"CHRNA10\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA10\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB3\", \"cname\": \"CHRNB3\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB4\", \"cname\": \"CHRNB4\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB4\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"galanthamine\", \"cname\": \"galanthamine\", \"id\": [\"Abundance\", \"CHEBI\", \"galanthamine\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"1-(3-chlorophenyl)piperazine\", \"cname\": \"1-(3-chlorophenyl)piperazine\", \"id\": [\"Abundance\", \"CHEBI\", \"1-(3-chlorophenyl)piperazine\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABBR2\", \"cname\": \"GABBR2\", \"id\": [\"Protein\", \"HGNC\", \"GABBR2\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"cAMP catabolic process\", \"cname\": \"cAMP catabolic process\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"cAMP catabolic process\"]}, {\"function\": \"Complex\", \"namespace\": \"SCOMP\", \"name\": \"CAMK2 Complex\", \"cname\": \"CAMK2 Complex\", \"id\": [\"Complex\", \"SCOMP\", \"CAMK2 Complex\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIA1\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 831}], \"cname\": \"p(HGNC:GRIA1, pmod(Ph, Ser, 831))\", \"id\": [\"Protein\", \"HGNC\", \"GRIA1\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 831]]}, {\"function\": \"Complex\", \"namespace\": \"GOCC\", \"name\": \"proteasome complex\", \"cname\": \"proteasome complex\", \"id\": [\"Complex\", \"GOCC\", \"proteasome complex\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GPHN\", \"cname\": \"GPHN\", \"id\": [\"Protein\", \"HGNC\", \"GPHN\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Glaucoma\", \"cname\": \"Glaucoma\", \"id\": [\"Pathology\", \"MESHD\", \"Glaucoma\"]}, {\"function\": \"Abundance\", \"namespace\": \"SCHEM\", \"name\": \"Glatiramer acetate\", \"cname\": \"Glatiramer acetate\", \"id\": [\"Abundance\", \"SCHEM\", \"Glatiramer acetate\"]}, {\"function\": \"Protein\", \"namespace\": \"MGI\", \"name\": \"Egr1\", \"cname\": \"Egr1\", \"id\": [\"Protein\", \"MGI\", \"Egr1\"]}, {\"function\": \"Complex\", \"cname\": \"complex(p(HGNC:EGR1), p(HGNC:PSEN2))\", \"id\": [\"Complex\", [\"Protein\", \"HGNC\", \"EGR1\"], [\"Protein\", \"HGNC\", \"PSEN2\"]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"estradiol\", \"cname\": \"estradiol\", \"id\": [\"Abundance\", \"CHEBI\", \"estradiol\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA4\", \"cname\": \"GABRA4\", \"id\": [\"RNA\", \"HGNC\", \"GABRA4\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA3\", \"cname\": \"GABRA3\", \"id\": [\"RNA\", \"HGNC\", \"GABRA3\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"Midazolam\", \"cname\": \"Midazolam\", \"id\": [\"Abundance\", \"CHEBI\", \"Midazolam\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIA1\", \"cname\": \"GRIA1\", \"id\": [\"Protein\", \"HGNC\", \"GRIA1\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"cname\": \"APP\", \"id\": [\"RNA\", \"HGNC\", \"APP\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"EGR1\", \"cname\": \"EGR1\", \"id\": [\"RNA\", \"HGNC\", \"EGR1\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"PSEN2\", \"cname\": \"PSEN2\", \"id\": [\"RNA\", \"HGNC\", \"PSEN2\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"TNF\", \"cname\": \"TNF\", \"id\": [\"RNA\", \"HGNC\", \"TNF\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA1\", \"cname\": \"GABRA1\", \"id\": [\"RNA\", \"HGNC\", \"GABRA1\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA2\", \"cname\": \"GABRA2\", \"id\": [\"RNA\", \"HGNC\", \"GABRA2\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA5\", \"cname\": \"GABRA5\", \"id\": [\"RNA\", \"HGNC\", \"GABRA5\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA6\", \"cname\": \"GABRA6\", \"id\": [\"RNA\", \"HGNC\", \"GABRA6\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRB1\", \"cname\": \"GABRB1\", \"id\": [\"RNA\", \"HGNC\", \"GABRB1\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRB2\", \"cname\": \"GABRB2\", \"id\": [\"RNA\", \"HGNC\", \"GABRB2\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRB3\", \"cname\": \"GABRB3\", \"id\": [\"RNA\", \"HGNC\", \"GABRB3\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRG1\", \"cname\": \"GABRG1\", \"id\": [\"RNA\", \"HGNC\", \"GABRG1\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRG2\", \"cname\": \"GABRG2\", \"id\": [\"RNA\", \"HGNC\", \"GABRG2\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRG3\", \"cname\": \"GABRG3\", \"id\": [\"RNA\", \"HGNC\", \"GABRG3\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRD\", \"cname\": \"GABRD\", \"id\": [\"RNA\", \"HGNC\", \"GABRD\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRE\", \"cname\": \"GABRE\", \"id\": [\"RNA\", \"HGNC\", \"GABRE\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRP\", \"cname\": \"GABRP\", \"id\": [\"RNA\", \"HGNC\", \"GABRP\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRQ\", \"cname\": \"GABRQ\", \"id\": [\"RNA\", \"HGNC\", \"GABRQ\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABBR1\", \"cname\": \"GABBR1\", \"id\": [\"RNA\", \"HGNC\", \"GABBR1\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"cname\": \"GRIN2B\", \"id\": [\"RNA\", \"HGNC\", \"GRIN2B\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNA4\", \"cname\": \"CHRNA4\", \"id\": [\"RNA\", \"HGNC\", \"CHRNA4\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNB2\", \"cname\": \"CHRNB2\", \"id\": [\"RNA\", \"HGNC\", \"CHRNB2\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNA2\", \"cname\": \"CHRNA2\", \"id\": [\"RNA\", \"HGNC\", \"CHRNA2\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNA3\", \"cname\": \"CHRNA3\", \"id\": [\"RNA\", \"HGNC\", \"CHRNA3\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNA5\", \"cname\": \"CHRNA5\", \"id\": [\"RNA\", \"HGNC\", \"CHRNA5\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNA6\", \"cname\": \"CHRNA6\", \"id\": [\"RNA\", \"HGNC\", \"CHRNA6\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNA7\", \"cname\": \"CHRNA7\", \"id\": [\"RNA\", \"HGNC\", \"CHRNA7\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNA9\", \"cname\": \"CHRNA9\", \"id\": [\"RNA\", \"HGNC\", \"CHRNA9\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNA10\", \"cname\": \"CHRNA10\", \"id\": [\"RNA\", \"HGNC\", \"CHRNA10\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNB3\", \"cname\": \"CHRNB3\", \"id\": [\"RNA\", \"HGNC\", \"CHRNB3\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CHRNB4\", \"cname\": \"CHRNB4\", \"id\": [\"RNA\", \"HGNC\", \"CHRNB4\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABBR2\", \"cname\": \"GABBR2\", \"id\": [\"RNA\", \"HGNC\", \"GABBR2\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GPHN\", \"cname\": \"GPHN\", \"id\": [\"RNA\", \"HGNC\", \"GPHN\"]}, {\"function\": \"RNA\", \"namespace\": \"MGI\", \"name\": \"Egr1\", \"cname\": \"Egr1\", \"id\": [\"RNA\", \"MGI\", \"Egr1\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GRIA1\", \"cname\": \"GRIA1\", \"id\": [\"RNA\", \"HGNC\", \"GRIA1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRA4\", \"cname\": \"GABRA4\", \"id\": [\"Gene\", \"HGNC\", \"GABRA4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRA3\", \"cname\": \"GABRA3\", \"id\": [\"Gene\", \"HGNC\", \"GABRA3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"cname\": \"APP\", \"id\": [\"Gene\", \"HGNC\", \"APP\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EGR1\", \"cname\": \"EGR1\", \"id\": [\"Gene\", \"HGNC\", \"EGR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSEN2\", \"cname\": \"PSEN2\", \"id\": [\"Gene\", \"HGNC\", \"PSEN2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"TNF\", \"cname\": \"TNF\", \"id\": [\"Gene\", \"HGNC\", \"TNF\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRA1\", \"cname\": \"GABRA1\", \"id\": [\"Gene\", \"HGNC\", \"GABRA1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRA2\", \"cname\": \"GABRA2\", \"id\": [\"Gene\", \"HGNC\", \"GABRA2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRA5\", \"cname\": \"GABRA5\", \"id\": [\"Gene\", \"HGNC\", \"GABRA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRA6\", \"cname\": \"GABRA6\", \"id\": [\"Gene\", \"HGNC\", \"GABRA6\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRB1\", \"cname\": \"GABRB1\", \"id\": [\"Gene\", \"HGNC\", \"GABRB1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRB2\", \"cname\": \"GABRB2\", \"id\": [\"Gene\", \"HGNC\", \"GABRB2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRB3\", \"cname\": \"GABRB3\", \"id\": [\"Gene\", \"HGNC\", \"GABRB3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRG1\", \"cname\": \"GABRG1\", \"id\": [\"Gene\", \"HGNC\", \"GABRG1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRG2\", \"cname\": \"GABRG2\", \"id\": [\"Gene\", \"HGNC\", \"GABRG2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRG3\", \"cname\": \"GABRG3\", \"id\": [\"Gene\", \"HGNC\", \"GABRG3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRD\", \"cname\": \"GABRD\", \"id\": [\"Gene\", \"HGNC\", \"GABRD\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRE\", \"cname\": \"GABRE\", \"id\": [\"Gene\", \"HGNC\", \"GABRE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRP\", \"cname\": \"GABRP\", \"id\": [\"Gene\", \"HGNC\", \"GABRP\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABRQ\", \"cname\": \"GABRQ\", \"id\": [\"Gene\", \"HGNC\", \"GABRQ\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABBR1\", \"cname\": \"GABBR1\", \"id\": [\"Gene\", \"HGNC\", \"GABBR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"cname\": \"GRIN2B\", \"id\": [\"Gene\", \"HGNC\", \"GRIN2B\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA4\", \"cname\": \"CHRNA4\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNB2\", \"cname\": \"CHRNB2\", \"id\": [\"Gene\", \"HGNC\", \"CHRNB2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA2\", \"cname\": \"CHRNA2\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA3\", \"cname\": \"CHRNA3\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA5\", \"cname\": \"CHRNA5\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA5\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA6\", \"cname\": \"CHRNA6\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA6\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA7\", \"cname\": \"CHRNA7\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA7\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA9\", \"cname\": \"CHRNA9\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA9\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNA10\", \"cname\": \"CHRNA10\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA10\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNB3\", \"cname\": \"CHRNB3\", \"id\": [\"Gene\", \"HGNC\", \"CHRNB3\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CHRNB4\", \"cname\": \"CHRNB4\", \"id\": [\"Gene\", \"HGNC\", \"CHRNB4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GABBR2\", \"cname\": \"GABBR2\", \"id\": [\"Gene\", \"HGNC\", \"GABBR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GPHN\", \"cname\": \"GPHN\", \"id\": [\"Gene\", \"HGNC\", \"GPHN\"]}, {\"function\": \"Gene\", \"namespace\": \"MGI\", \"name\": \"Egr1\", \"cname\": \"Egr1\", \"id\": [\"Gene\", \"MGI\", \"Egr1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GRIA1\", \"cname\": \"GRIA1\", \"id\": [\"Gene\", \"HGNC\", \"GRIA1\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"In immunoblotting analysis, nefiracetam treatment increased the PKCalpha activity with a bell-shaped dose-response relationship peaking at 10 nM, thereby increasing phosphorylation of PKC substrate and NMDA receptor. Nefiracetam treatment did not affect the PKA activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Pharmacol. 2007 Feb;71(2):580-7\", \"reference\": \"17095583\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 0, \"target\": 2, \"key\": -4}, {\"relation\": \"negativeCorrelation\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 4, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 4, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"Most forms of neuronal plasticity are associated with the induction of the transcription factor zif268 (egr1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 35, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 8, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 9, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 10, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 11, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 12, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 13, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 14, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 15, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 17, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 18, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 19, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 20, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 21, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 22, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 23, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 25, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Induction of zif268 in neurons leads to altered expression of proteasome subunit and proteasome-regulatory genes, thereby changing the capacity of the neuron to degrade synaptic proteins, including receptors and receptor subunits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 51, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In addition, zif268 alters the transcription of genes associated with GABA(A)R expression and trafficking, such as ubiquilin and gephyrin.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 52, \"key\": 0}, {\"relation\": \"negativeCorrelation\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 2, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 53, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, it identifies EGR1 transcription factor as a potential mediator for GA-induced neuroprotection in both glaucoma and AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Alzheimer Disease\"}, \"source\": 3, \"target\": 54, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, it identifies EGR1 transcription factor as a potential mediator for GA-induced neuroprotection in both glaucoma and AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Glaucoma\"}, \"source\": 3, \"target\": 54, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The nefiracetam-induced increase in the frequency of mEPSCs and mIPSCs over and above the level achieved by ACh alone would contribute to the therapeutic effect of nefiracetam as the cholinergic system is known to be downregulated in the brain of Alzheimer's disease patients.The therapeutic effects of galantamine are ascribed to the modest potentiation of nACh receptor and NMDA receptor activities in addition to the modest inhibition of acetylcholinesterase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 5, \"key\": 3}, {\"relation\": \"association\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 3, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 3, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 8, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 9, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 10, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 11, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 12, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 13, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 14, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 15, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 16, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 17, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 18, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 19, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 20, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 21, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 22, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 23, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 24, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"1-(3-chlorophenyl)piperazine affects the reaction [gamma-Aminobutyric Acid results in increased activity of [GABRA1 protein co-treated with GABBR2 protein co-treated with GABRG2 protein]]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"21729720\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 46, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 9, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 9, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 10, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 10, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 11, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 11, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 12, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 12, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 13, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 13, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 14, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 14, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 15, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 15, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 16, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 16, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 17, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 17, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 18, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 18, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 19, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 19, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 20, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 20, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 21, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 21, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 22, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 22, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 23, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 23, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 25, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 25, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 8, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 9, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 10, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 11, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 12, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 13, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 14, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 15, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 16, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 17, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 18, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 19, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 20, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 21, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 22, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 23, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 25, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 28, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 29, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 30, \"key\": 1}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 27, \"target\": 1, \"key\": -4}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 31, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 31, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 34, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 34, \"target\": 33, \"key\": 3}, {\"relation\": \"association\", \"evidence\": \"Most forms of neuronal plasticity are associated with the induction of the transcription factor zif268 (egr1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 35, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 36, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 36, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 37, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 37, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 38, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 38, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 39, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 39, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 40, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 40, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 41, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 41, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 42, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 42, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 43, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 43, \"target\": 33, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 44, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 44, \"target\": 33, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 36, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 37, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 31, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 38, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 39, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 40, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 41, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 42, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 34, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 43, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 44, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"1-(3-chlorophenyl)piperazine affects the reaction [gamma-Aminobutyric Acid results in increased activity of [GABRA1 protein co-treated with GABBR2 protein co-treated with GABRG2 protein]]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"21729720\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 46, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In this work, we describe a novel non-coding (nc) RNA (named 17A) RNA polymerase (pol) III-dependent embedded in the human G-protein-coupled receptor 51 gene (GPR51, GABA B2 receptor). The stable expression of 17A in SHSY5Y neuroblastoma cells induces the synthesis of an alternative splicing isoform that abolish GABA B2 intracellular signaling (i.e., inhibition of cAMP accumulation and activation of K(+) channels). Indeed, 17A is expressed in human brain, and we report that it is upregulated in cerebral tissues derived from Alzheimer disease patients. We demonstrate that 17A expression in neuroblastoma cells enhances the secretion of amyloid β peptide (Abeta) and the Abeta x-42/Î‘β x-40 peptide ratio and that its synthesis is induced in response to inflammatory stimuli.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2011 Feb;41(2):308-17. doi: 10.1016/j.nbd.2010.09.019. Epub 2010 Oct 1.\", \"reference\": \"20888417\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 47, \"target\": 48, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Nefiracetam treatment significantly enhanced long-term potentiation (LTP) with the same bell-shaped dose-response curve. Furthermore, nefiracetam-induced LTP enhancement was closely associated with calcium/calmodulin-dependent protein kinase II (CaMKII) activation with increase in phosphorylation of AMPA-type glutamate receptor subunit 1 (GluA1) (Ser-831) as a postsynaptic CaMKII substrate. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 49, \"target\": 50, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Induction of zif268 in neurons leads to altered expression of proteasome subunit and proteasome-regulatory genes, thereby changing the capacity of the neuron to degrade synaptic proteins, including receptors and receptor subunits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 51, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In addition, zif268 alters the transcription of genes associated with GABA(A)R expression and trafficking, such as ubiquilin and gephyrin.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 52, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 53, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Egr1 expression is induced following glatiramer acetate immunotherapy in rodent models of glaucoma and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Alzheimer Disease\"}, \"source\": 54, \"target\": 55, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Egr1 expression is induced following glatiramer acetate immunotherapy in rodent models of glaucoma and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Glaucoma\"}, \"source\": 54, \"target\": 55, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 54, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We show that Egr-1 binds to PSEN-P2, and that PSEN-P2 activity is increased threefold by overexpression of Egr-1, and by 12-O-tetradecanoylphorbol-13-acetate (TPA), which induces physiological Egr-1 levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 56, \"target\": 6, \"key\": 1}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 56, \"target\": 6, \"key\": -3}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 56, \"target\": 3, \"key\": -3}, {\"relation\": \"association\", \"evidence\": \"Estradiol affects the expression of GABRA4 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 57, \"target\": 58, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Estradiol results in increased expression of GABRA3 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 57, \"target\": 59, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Estradiol affects the expression of GABRA4 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 58, \"target\": 57, \"key\": 0}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 58, \"target\": 11, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 59, \"target\": 10, \"key\": -6}, {\"relation\": \"decreases\", \"evidence\": \"GABRA6 protein mutant form results in decreased susceptibility to Midazolam\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"12031754\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 60, \"target\": 13, \"key\": 0}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 61, \"target\": 50, \"key\": -4}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 62, \"target\": 0, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 63, \"target\": 3, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 64, \"target\": 6, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 65, \"target\": 7, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 66, \"target\": 8, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 67, \"target\": 9, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 68, \"target\": 12, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 69, \"target\": 13, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 70, \"target\": 14, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 71, \"target\": 15, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 72, \"target\": 16, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 73, \"target\": 17, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 74, \"target\": 18, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 75, \"target\": 19, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 76, \"target\": 20, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 77, \"target\": 21, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 78, \"target\": 22, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 79, \"target\": 23, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 80, \"target\": 24, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 81, \"target\": 27, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 82, \"target\": 31, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 83, \"target\": 34, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 84, \"target\": 36, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 85, \"target\": 37, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 86, \"target\": 38, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 87, \"target\": 39, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 88, \"target\": 40, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 89, \"target\": 41, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 90, \"target\": 42, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 91, \"target\": 43, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 92, \"target\": 44, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 93, \"target\": 47, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 94, \"target\": 52, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 95, \"target\": 55, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 96, \"target\": 61, \"key\": -6}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 97, \"target\": 58, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 98, \"target\": 59, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 99, \"target\": 62, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 100, \"target\": 63, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 101, \"target\": 64, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 102, \"target\": 65, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 103, \"target\": 66, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 104, \"target\": 67, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 105, \"target\": 68, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 106, \"target\": 69, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 107, \"target\": 70, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 108, \"target\": 71, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 109, \"target\": 72, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 110, \"target\": 73, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 111, \"target\": 74, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 112, \"target\": 75, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 113, \"target\": 76, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 114, \"target\": 77, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 115, \"target\": 78, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 116, \"target\": 79, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 117, \"target\": 80, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 118, \"target\": 81, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 119, \"target\": 82, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 120, \"target\": 83, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 121, \"target\": 84, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 122, \"target\": 85, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 123, \"target\": 86, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 124, \"target\": 87, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 125, \"target\": 88, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 126, \"target\": 89, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 127, \"target\": 90, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 128, \"target\": 91, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 129, \"target\": 92, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 130, \"target\": 93, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 131, \"target\": 94, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 132, \"target\": 95, \"key\": -5}, {\"relation\": \"transcribedTo\", \"annotations\": {}, \"source\": 133, \"target\": 96, \"key\": -5}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Finally, some of the genes and RNAs that have been added have no connections, and can be removed with [pbt.mutation.prune_central_dogma](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.prune_central_dogma)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of nodes: 62\n",
      "Number of edges: 161\n",
      "Network density: 0.04257006874669487\n",
      "Number weakly connected components: 4\n",
      "Average in-degree: 2.596774193548387\n",
      "Average out-degree: 2.596774193548387\n"
     ]
    }
   ],
   "source": [
    "pbt.mutation.prune_central_dogma(subgraph)\n",
    "\n",
    "pbt.summary.print_summary(subgraph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"egsmptzuvqrbhkoy\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {\"name\": \"Alzheimer's Disease Knowledge Assembly - (Subgraph: GABA subgraph)\"}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"cname\": \"APP\", \"id\": [\"Protein\", \"HGNC\", \"APP\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}}], \"cname\": \"p(HGNC:GRIN2B, pmod(Ph))\", \"id\": [\"Protein\", \"HGNC\", \"GRIN2B\", [\"pmod\", [\"bel\", \"Ph\"]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"EGR1\", \"cname\": \"EGR1\", \"id\": [\"Protein\", \"HGNC\", \"EGR1\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Alzheimer Disease\", \"cname\": \"Alzheimer Disease\", \"id\": [\"Pathology\", \"MESHD\", \"Alzheimer Disease\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"synaptic transmission, cholinergic\", \"cname\": \"synaptic transmission, cholinergic\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"synaptic transmission, cholinergic\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PSEN2\", \"cname\": \"PSEN2\", \"id\": [\"Protein\", \"HGNC\", \"PSEN2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TNF\", \"cname\": \"TNF\", \"id\": [\"Protein\", \"HGNC\", \"TNF\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA1\", \"cname\": \"GABRA1\", \"id\": [\"Protein\", \"HGNC\", \"GABRA1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA2\", \"cname\": \"GABRA2\", \"id\": [\"Protein\", \"HGNC\", \"GABRA2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA3\", \"cname\": \"GABRA3\", \"id\": [\"Protein\", \"HGNC\", \"GABRA3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA4\", \"cname\": \"GABRA4\", \"id\": [\"Protein\", \"HGNC\", \"GABRA4\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA5\", \"cname\": \"GABRA5\", \"id\": [\"Protein\", \"HGNC\", \"GABRA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRA6\", \"cname\": \"GABRA6\", \"id\": [\"Protein\", \"HGNC\", \"GABRA6\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB1\", \"cname\": \"GABRB1\", \"id\": [\"Protein\", \"HGNC\", \"GABRB1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB2\", \"cname\": \"GABRB2\", \"id\": [\"Protein\", \"HGNC\", \"GABRB2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRB3\", \"cname\": \"GABRB3\", \"id\": [\"Protein\", \"HGNC\", \"GABRB3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG1\", \"cname\": \"GABRG1\", \"id\": [\"Protein\", \"HGNC\", \"GABRG1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG2\", \"cname\": \"GABRG2\", \"id\": [\"Protein\", \"HGNC\", \"GABRG2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRG3\", \"cname\": \"GABRG3\", \"id\": [\"Protein\", \"HGNC\", \"GABRG3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRD\", \"cname\": \"GABRD\", \"id\": [\"Protein\", \"HGNC\", \"GABRD\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRE\", \"cname\": \"GABRE\", \"id\": [\"Protein\", \"HGNC\", \"GABRE\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRP\", \"cname\": \"GABRP\", \"id\": [\"Protein\", \"HGNC\", \"GABRP\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABRQ\", \"cname\": \"GABRQ\", \"id\": [\"Protein\", \"HGNC\", \"GABRQ\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABBR1\", \"cname\": \"GABBR1\", \"id\": [\"Protein\", \"HGNC\", \"GABBR1\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\", \"cname\": \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of N-methyl-D-aspartate selective glutamate receptor activity\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of synaptic plasticity\", \"cname\": \"regulation of synaptic plasticity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of synaptic plasticity\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIN2B\", \"cname\": \"GRIN2B\", \"id\": [\"Protein\", \"HGNC\", \"GRIN2B\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"cognition\", \"cname\": \"cognition\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"cognition\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"learning\", \"cname\": \"learning\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"learning\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"memory\", \"cname\": \"memory\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"memory\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA4\", \"cname\": \"CHRNA4\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA4\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"gamma-aminobutyric acid\", \"cname\": \"gamma-aminobutyric acid\", \"id\": [\"Abundance\", \"CHEBI\", \"gamma-aminobutyric acid\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"glutamate(1-)\", \"cname\": \"glutamate(1-)\", \"id\": [\"Abundance\", \"CHEBI\", \"glutamate(1-)\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB2\", \"cname\": \"CHRNB2\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB2\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"regulation of neuronal synaptic plasticity\", \"cname\": \"regulation of neuronal synaptic plasticity\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"regulation of neuronal synaptic plasticity\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA2\", \"cname\": \"CHRNA2\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA3\", \"cname\": \"CHRNA3\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA5\", \"cname\": \"CHRNA5\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA6\", \"cname\": \"CHRNA6\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA6\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA7\", \"cname\": \"CHRNA7\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA7\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA9\", \"cname\": \"CHRNA9\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA9\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNA10\", \"cname\": \"CHRNA10\", \"id\": [\"Protein\", \"HGNC\", \"CHRNA10\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB3\", \"cname\": \"CHRNB3\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHRNB4\", \"cname\": \"CHRNB4\", \"id\": [\"Protein\", \"HGNC\", \"CHRNB4\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"galanthamine\", \"cname\": \"galanthamine\", \"id\": [\"Abundance\", \"CHEBI\", \"galanthamine\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"1-(3-chlorophenyl)piperazine\", \"cname\": \"1-(3-chlorophenyl)piperazine\", \"id\": [\"Abundance\", \"CHEBI\", \"1-(3-chlorophenyl)piperazine\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GABBR2\", \"cname\": \"GABBR2\", \"id\": [\"Protein\", \"HGNC\", \"GABBR2\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"cAMP catabolic process\", \"cname\": \"cAMP catabolic process\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"cAMP catabolic process\"]}, {\"function\": \"Complex\", \"namespace\": \"SCOMP\", \"name\": \"CAMK2 Complex\", \"cname\": \"CAMK2 Complex\", \"id\": [\"Complex\", \"SCOMP\", \"CAMK2 Complex\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIA1\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 831}], \"cname\": \"p(HGNC:GRIA1, pmod(Ph, Ser, 831))\", \"id\": [\"Protein\", \"HGNC\", \"GRIA1\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 831]]}, {\"function\": \"Complex\", \"namespace\": \"GOCC\", \"name\": \"proteasome complex\", \"cname\": \"proteasome complex\", \"id\": [\"Complex\", \"GOCC\", \"proteasome complex\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GPHN\", \"cname\": \"GPHN\", \"id\": [\"Protein\", \"HGNC\", \"GPHN\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Glaucoma\", \"cname\": \"Glaucoma\", \"id\": [\"Pathology\", \"MESHD\", \"Glaucoma\"]}, {\"function\": \"Abundance\", \"namespace\": \"SCHEM\", \"name\": \"Glatiramer acetate\", \"cname\": \"Glatiramer acetate\", \"id\": [\"Abundance\", \"SCHEM\", \"Glatiramer acetate\"]}, {\"function\": \"Protein\", \"namespace\": \"MGI\", \"name\": \"Egr1\", \"cname\": \"Egr1\", \"id\": [\"Protein\", \"MGI\", \"Egr1\"]}, {\"function\": \"Complex\", \"cname\": \"complex(p(HGNC:EGR1), p(HGNC:PSEN2))\", \"id\": [\"Complex\", [\"Protein\", \"HGNC\", \"EGR1\"], [\"Protein\", \"HGNC\", \"PSEN2\"]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"estradiol\", \"cname\": \"estradiol\", \"id\": [\"Abundance\", \"CHEBI\", \"estradiol\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA4\", \"cname\": \"GABRA4\", \"id\": [\"RNA\", \"HGNC\", \"GABRA4\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"GABRA3\", \"cname\": \"GABRA3\", \"id\": [\"RNA\", \"HGNC\", \"GABRA3\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"Midazolam\", \"cname\": \"Midazolam\", \"id\": [\"Abundance\", \"CHEBI\", \"Midazolam\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GRIA1\", \"cname\": \"GRIA1\", \"id\": [\"Protein\", \"HGNC\", \"GRIA1\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"In immunoblotting analysis, nefiracetam treatment increased the PKCalpha activity with a bell-shaped dose-response relationship peaking at 10 nM, thereby increasing phosphorylation of PKC substrate and NMDA receptor. Nefiracetam treatment did not affect the PKA activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Pharmacol. 2007 Feb;71(2):580-7\", \"reference\": \"17095583\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 0, \"target\": 2, \"key\": -4}, {\"relation\": \"negativeCorrelation\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 4, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 4, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"Most forms of neuronal plasticity are associated with the induction of the transcription factor zif268 (egr1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 35, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 8, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 9, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 10, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 11, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 12, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 13, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 14, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 15, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 16, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 17, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 18, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 19, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 20, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 21, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 22, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 23, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 25, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Induction of zif268 in neurons leads to altered expression of proteasome subunit and proteasome-regulatory genes, thereby changing the capacity of the neuron to degrade synaptic proteins, including receptors and receptor subunits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 51, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In addition, zif268 alters the transcription of genes associated with GABA(A)R expression and trafficking, such as ubiquilin and gephyrin.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 52, \"key\": 0}, {\"relation\": \"negativeCorrelation\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 2, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 53, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, it identifies EGR1 transcription factor as a potential mediator for GA-induced neuroprotection in both glaucoma and AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Alzheimer Disease\"}, \"source\": 3, \"target\": 54, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, it identifies EGR1 transcription factor as a potential mediator for GA-induced neuroprotection in both glaucoma and AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Glaucoma\"}, \"source\": 3, \"target\": 54, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 3, \"target\": 6, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The nefiracetam-induced increase in the frequency of mEPSCs and mIPSCs over and above the level achieved by ACh alone would contribute to the therapeutic effect of nefiracetam as the cholinergic system is known to be downregulated in the brain of Alzheimer's disease patients.The therapeutic effects of galantamine are ascribed to the modest potentiation of nACh receptor and NMDA receptor activities in addition to the modest inhibition of acetylcholinesterase.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 5, \"key\": 3}, {\"relation\": \"association\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 3, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 3, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Egr-1 upregulates the Alzheimer's disease presenilin-2 gene in neuronal cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 8, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 9, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 10, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 11, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 12, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 13, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 14, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 15, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 16, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 17, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 18, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 19, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 20, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 21, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 22, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 23, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"In addition, TNF- Î± can potentiate glutamate-mediated cytotoxicity by two complementary mechanisms: indirectly, by inhibiting glutamate transport on astrocytes, and directly, by rapidly triggering the surface expression of Ca+2 permeable-AMPA receptors and NMDA receptors, while decreasing inhibitory GABAA receptors on neurons.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mediators Inflamm. 2014;2014:861231\", \"reference\": \"24966471\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 7, \"target\": 24, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"1-(3-chlorophenyl)piperazine affects the reaction [gamma-Aminobutyric Acid results in increased activity of [GABRA1 protein co-treated with GABBR2 protein co-treated with GABRG2 protein]]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"21729720\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 8, \"target\": 46, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 9, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 9, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 10, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 10, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 11, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 11, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 12, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 12, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 13, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 13, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 14, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 14, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 15, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 15, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 16, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 16, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 17, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 17, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 18, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 18, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 19, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 19, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 20, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 20, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 21, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 21, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 22, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 22, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 23, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 23, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 25, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Hence, it is predicted that zif268 may regulate transcription of genes associated with glutamate receptors and/or GABA(A)Rs.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 25, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 8, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 9, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 10, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 11, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 12, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 13, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 14, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 15, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 16, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 17, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 18, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 19, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 20, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 21, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 22, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 23, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"The regulation of synaptic glutamate receptor and GABA(A)R (gamma-aminobutyric acid subtype A receptor) levels is a key component of synaptic plasticity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 26, \"target\": 25, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 28, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 29, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In conclusion, nefiracetam enhances NMDA-receptor function through stimulation of its glycine binding site and nefiracetam-induced CaMKII activation likely contributes to improvement of cognition, learning, and memory.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 27, \"target\": 30, \"key\": 1}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 27, \"target\": 1, \"key\": -4}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 31, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 31, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 34, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 34, \"target\": 33, \"key\": 3}, {\"relation\": \"association\", \"evidence\": \"Most forms of neuronal plasticity are associated with the induction of the transcription factor zif268 (egr1).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 35, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 36, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 36, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 37, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 37, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 38, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 38, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 39, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 39, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 40, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 40, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 41, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 41, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 42, \"target\": 32, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 42, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 43, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 43, \"target\": 33, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 44, \"target\": 33, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 44, \"target\": 33, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 36, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 37, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 31, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 38, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 39, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 40, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 41, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 42, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 34, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 43, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Galantamine and nefiracetam have been shown to potentiate the phasic activity of nicotinic acetylcholine receptors (nAChRss) in the brain. Stimulation of nAChRss is also known to cause release of various neurotransmitters including glutamate and gamma-aminobutyric acid (GABA).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuroscience. 2009 May 5;160(2):484-91\", \"reference\": \"19272425\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 45, \"target\": 44, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"1-(3-chlorophenyl)piperazine affects the reaction [gamma-Aminobutyric Acid results in increased activity of [GABRA1 protein co-treated with GABBR2 protein co-treated with GABRG2 protein]]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"21729720\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 46, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In this work, we describe a novel non-coding (nc) RNA (named 17A) RNA polymerase (pol) III-dependent embedded in the human G-protein-coupled receptor 51 gene (GPR51, GABA B2 receptor). The stable expression of 17A in SHSY5Y neuroblastoma cells induces the synthesis of an alternative splicing isoform that abolish GABA B2 intracellular signaling (i.e., inhibition of cAMP accumulation and activation of K(+) channels). Indeed, 17A is expressed in human brain, and we report that it is upregulated in cerebral tissues derived from Alzheimer disease patients. We demonstrate that 17A expression in neuroblastoma cells enhances the secretion of amyloid β peptide (Abeta) and the Abeta x-42/Î‘β x-40 peptide ratio and that its synthesis is induced in response to inflammatory stimuli.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2011 Feb;41(2):308-17. doi: 10.1016/j.nbd.2010.09.019. Epub 2010 Oct 1.\", \"reference\": \"20888417\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 47, \"target\": 48, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Nefiracetam treatment significantly enhanced long-term potentiation (LTP) with the same bell-shaped dose-response curve. Furthermore, nefiracetam-induced LTP enhancement was closely associated with calcium/calmodulin-dependent protein kinase II (CaMKII) activation with increase in phosphorylation of AMPA-type glutamate receptor subunit 1 (GluA1) (Ser-831) as a postsynaptic CaMKII substrate. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2011;117(1):6-11\", \"reference\": \"21821968\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 49, \"target\": 50, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Induction of zif268 in neurons leads to altered expression of proteasome subunit and proteasome-regulatory genes, thereby changing the capacity of the neuron to degrade synaptic proteins, including receptors and receptor subunits.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 51, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"In addition, zif268 alters the transcription of genes associated with GABA(A)R expression and trafficking, such as ubiquilin and gephyrin.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochem Soc Trans. 2009 Dec;37(Pt 6):1375-7\", \"reference\": \"19909279\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 52, \"target\": 3, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Further, EGR1 levels were negatively correlated with hippocampal amyloid-β plaque burden.This study presents global gene expression profiles associated with GA immunization in a glaucoma rat model.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHDisease\": \"Glaucoma\", \"MeSHAnatomy\": \"Hippocampus\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 53, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Egr1 expression is induced following glatiramer acetate immunotherapy in rodent models of glaucoma and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Alzheimer Disease\"}, \"source\": 54, \"target\": 55, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Egr1 expression is induced following glatiramer acetate immunotherapy in rodent models of glaucoma and Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\", \"MeSHDisease\": \"Glaucoma\"}, \"source\": 54, \"target\": 55, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In AD-Tg mice, a significant increase in hippocampal EGR1 protein levels was also found in response to GA immunization.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Invest Ophthalmol Vis Sci. 2011 Nov 21;52(12):9033-46\", \"reference\": \"21969301\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"MeSHAnatomy\": \"Hippocampus\", \"Species\": \"10090\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 54, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We show that Egr-1 binds to PSEN-P2, and that PSEN-P2 activity is increased threefold by overexpression of Egr-1, and by 12-O-tetradecanoylphorbol-13-acetate (TPA), which induces physiological Egr-1 levels.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Gene. 2003 Oct 30;318:113-24\", \"reference\": \"14585504\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 56, \"target\": 6, \"key\": 1}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 56, \"target\": 6, \"key\": -3}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 56, \"target\": 3, \"key\": -3}, {\"relation\": \"association\", \"evidence\": \"Estradiol affects the expression of GABRA4 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 57, \"target\": 58, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Estradiol results in increased expression of GABRA3 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Subgraph\": \"GABA subgraph\"}, \"source\": 57, \"target\": 59, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Estradiol affects the expression of GABRA4 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16000229\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 58, \"target\": 57, \"key\": 0}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 58, \"target\": 11, \"key\": -6}, {\"relation\": \"translatedTo\", \"annotations\": {}, \"source\": 59, \"target\": 10, \"key\": -6}, {\"relation\": \"decreases\", \"evidence\": \"GABRA6 protein mutant form results in decreased susceptibility to Midazolam\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"12031754\"}, \"annotations\": {\"Species\": \"9606\", \"Subgraph\": \"GABA subgraph\"}, \"source\": 60, \"target\": 13, \"key\": 0}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 61, \"target\": 50, \"key\": -4}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The concept of expansion then contraction is commonly called \"opening\" in the domain of image processing. Inference of the central dogma then removal of leaf genes and RNAs is such a standard operation that both steps can be run by [pbt.mutation.opening_on_central_dogma](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.opening_on_central_dogma).\n",
    "\n",
    "### Further Consideration\n",
    "\n",
    "The fact that a subgraph contains more than one connected component probably means that there were errors in the original BEL script. There is an entire module devoted to analyzing the errors produced during compilation called `pbt.summary.error_summary`\n",
    "\n",
    "However, it's also possible that the connections are due to lack of knowledge in the literature. In the curation process for the NeuroMMSig Database, many entitity types were not considered. We've developed an algorithm for inferring additional members of a subgraph, including chemicals that occur as intermediates in biochemical processes, and higher level entities such as biological processes. The set of tools for running the algorithm are avaliable in the `pbt.mutations.subgraph_expansion` submodule (see [pbt.mutation.fill_subgraph](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.fill_subgraph)). "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Expanding on the Periphery\n",
    "\n",
    "In this example, we'll look at the Estrogen Subgraph. The subgraph is enriched with unqualified edges and opened with the central dogma."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "example_subgraph_name = 'Estrogen subgraph'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of nodes: 25\n",
      "Number of edges: 39\n",
      "Network density: 0.065\n",
      "Number weakly connected components: 5\n",
      "Average in-degree: 1.56\n",
      "Average out-degree: 1.56\n"
     ]
    }
   ],
   "source": [
    "subgraph = pbt.selection.get_subgraph_by_annotation_value(graph, annotation='Subgraph', value=example_subgraph_name)\n",
    "\n",
    "pbt.mutation.enrich_unqualified(graph, subgraph)\n",
    "pbt.mutation.opening_on_central_dogma(subgraph)\n",
    "\n",
    "pbt.summary.print_summary(subgraph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"dsfbejqokvgrqchu\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {\"name\": \"Alzheimer's Disease Knowledge Assembly - (Subgraph: Estrogen subgraph)\"}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Alzheimer Disease\", \"cname\": \"Alzheimer Disease\", \"id\": [\"Pathology\", \"MESHD\", \"Alzheimer Disease\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"ESR1\", \"cname\": \"ESR1\", \"id\": [\"Protein\", \"HGNC\", \"ESR1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"cname\": \"GSK3B\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"17beta-estradiol\", \"cname\": \"17beta-estradiol\", \"id\": [\"Abundance\", \"CHEBI\", \"17beta-estradiol\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Parkinson Disease\", \"cname\": \"Parkinson Disease\", \"id\": [\"Pathology\", \"MESHD\", \"Parkinson Disease\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 9}], \"cname\": \"p(HGNC:GSK3B, pmod(Ph, Ser, 9))\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 9]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CYP3A4\", \"cname\": \"CYP3A4\", \"id\": [\"Protein\", \"HGNC\", \"CYP3A4\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"metabolic process\", \"cname\": \"metabolic process\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"metabolic process\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Thr\", \"pos\": 231}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Thr, 231))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Thr\", 231]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Thr\", \"pos\": 205}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Thr, 205))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Thr\", 205]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 396}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Ser, 396))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 396]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 404}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Ser, 404))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 404]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 199}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Ser, 199))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 199]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 202}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Ser, 202))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 202]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"resveratrol\", \"cname\": \"resveratrol\", \"id\": [\"Abundance\", \"CHEBI\", \"resveratrol\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"ESR2\", \"cname\": \"ESR2\", \"id\": [\"Protein\", \"HGNC\", \"ESR2\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"estradiol\", \"cname\": \"estradiol\", \"id\": [\"Abundance\", \"CHEBI\", \"estradiol\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"S100A9\", \"cname\": \"S100A9\", \"id\": [\"Protein\", \"HGNC\", \"S100A9\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PPARG\", \"cname\": \"PPARG\", \"id\": [\"Protein\", \"HGNC\", \"PPARG\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"TFF1\", \"cname\": \"TFF1\", \"id\": [\"RNA\", \"HGNC\", \"TFF1\"]}, {\"function\": \"Complex\", \"cname\": \"complex(a(CHEBI:\\\"24-hydroxycholesterol\\\"), p(HGNC:ESR2))\", \"id\": [\"Complex\", [\"Abundance\", \"CHEBI\", \"24-hydroxycholesterol\"], [\"Protein\", \"HGNC\", \"ESR2\"]]}, {\"function\": \"Complex\", \"cname\": \"complex(a(CHEBI:\\\"medroxyprogesterone acetate\\\"), p(HGNC:ESR2))\", \"id\": [\"Complex\", [\"Abundance\", \"CHEBI\", \"medroxyprogesterone acetate\"], [\"Protein\", \"HGNC\", \"ESR2\"]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"24-hydroxycholesterol\", \"cname\": \"24-hydroxycholesterol\", \"id\": [\"Abundance\", \"CHEBI\", \"24-hydroxycholesterol\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"medroxyprogesterone acetate\", \"cname\": \"medroxyprogesterone acetate\", \"id\": [\"Abundance\", \"CHEBI\", \"medroxyprogesterone acetate\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"cname\": \"MAPT\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\"]}], \"links\": [{\"relation\": \"association\", \"evidence\": \"Herein, we conducted a meta-analysis to clarify the association between ESR1 polymorphisms and the occurrence of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2014 Jun 27;574:41-6\", \"reference\": \"24857745\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Herein, we conducted a meta-analysis to clarify the association between ESR1 polymorphisms and the occurrence of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2014 Jun 27;574:41-6\", \"reference\": \"24857745\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Positive association between an estrogen receptor gene polymorphism and Parkinson's disease with dementia.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurol. 1999 Jul;6(4):431-5\", \"reference\": \"10362895\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\", \"Species\": \"9606\", \"MeSHDisease\": \"Dementia\"}, \"source\": 1, \"target\": 4, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Positive association between an estrogen receptor gene polymorphism and Parkinson's disease with dementia.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurol. 1999 Jul;6(4):431-5\", \"reference\": \"10362895\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\", \"Species\": \"9606\", \"MeSHDisease\": \"Parkinson Disease\"}, \"source\": 1, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 10}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 13}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 16}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 2, \"target\": 5, \"key\": -4}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 8, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 9, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 10, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 11, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 12, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 13, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Simultaneously, it increased the level of Ser9-phosphorylated (inactive) GSK-3beta\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Positive association between an estrogen receptor gene polymorphism and Parkinson's disease with dementia.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurol. 1999 Jul;6(4):431-5\", \"reference\": \"10362895\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\", \"Species\": \"9606\", \"MeSHDisease\": \"Dementia\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Positive association between an estrogen receptor gene polymorphism and Parkinson's disease with dementia.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurol. 1999 Jul;6(4):431-5\", \"reference\": \"10362895\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\", \"Species\": \"9606\", \"MeSHDisease\": \"Parkinson Disease\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Simultaneously, it increased the level of Ser9-phosphorylated (inactive) GSK-3beta\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 5, \"target\": 2, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Donepezil is metabolized via CYP-related enzymes, especially CYP2D6, CYP3A4, and CYP1A2.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropsychiatr Dis Treat. 2007 Jun;3(3):303-33\", \"reference\": \"19300564\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 6, \"target\": 7, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"resveratrol inhibits the reaction [ESR2 protein binds to CAV1 protein]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"18296501\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 14, \"target\": 15, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"resveratrol inhibits the reaction [ESR2 protein binds to CAV1 protein]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"18296501\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 15, \"target\": 14, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"[Estradiol co-treated with Tetrachlorodibenzodioxin] results in decreased expression of S100A9 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"19484750\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 16, \"target\": 17, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Estradiol results in increased expression of PPARG protein\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"15964169\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 16, \"target\": 18, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"2-(2-amino-3-methoxyphenyl)-4H-1-benzopyran-4-one inhibits the reaction [Estradiol results in increased expression of TFF1 mRNA]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"11331657\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 16, \"target\": 19, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"24-hydroxycholesterol binds to and results in decreased activity of ESR2 protein\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"17873880\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 20, \"target\": 15, \"key\": 1}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 20, \"target\": 15, \"key\": -3}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 20, \"target\": 22, \"key\": -3}, {\"relation\": \"increases\", \"evidence\": \"Activation;Binding;Medroxyprogesterone acetate intracellular binds to and activates ESR2 .\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"9464360\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 21, \"target\": 15, \"key\": 1}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 21, \"target\": 15, \"key\": -3}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 21, \"target\": 23, \"key\": -3}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 8, \"key\": -4}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 9, \"key\": -4}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 10, \"key\": -4}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 11, \"key\": -4}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 12, \"key\": -4}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 13, \"key\": -4}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The nodes along the periphery of this subgraph can be investigated with [pbt.mutation.get_subgraph_peripheral_nodes](). Below, it is used to output which nodes which aren't already in the Estrogen Subgraph, and how many in- and out-edges they have to it."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pnd = pbt.mutation.get_subgraph_peripheral_nodes(graph, subgraph, node_filters=pbt.filters.exclude_pathology_filter)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('Protein', 'HGNC', 'APP', ('frag', (672, 713))) 7 2 7\n",
      "('Protein', 'HGNC', 'MAPT', ('pmod', ('bel', 'Glyco'))) 1 5 6\n"
     ]
    }
   ],
   "source": [
    "for node in sorted(pnd, key=lambda k: len(set(pnd[k]['successor']) | set(pnd[k]['predecessor'])), reverse=True):\n",
    "    pred_d = pnd[node]['predecessor']\n",
    "    succ_d = pnd[node]['successor']\n",
    "\n",
    "    if 0 == len(pred_d) or 0 == len(succ_d):\n",
    "        continue\n",
    "    \n",
    "    periphery = set(pred_d) | set(succ_d)\n",
    "    \n",
    "    if 4 > len(periphery):\n",
    "        continue\n",
    "    \n",
    "    print(node, len(pred_d), len(succ_d), len(periphery))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The function [pbt.mutation.expand_periphery](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.expand_periphery) automatically handles these calcuations and allows for the specification of a threshold for how \"confident\" it should be to add a node to the subgraph. Filters to exclude pathologies (which have many connections to everything). The inferred edges are limited to only causal edges, to avoid adding many low confidence relations. Luckily, the Estrogen Subgraph is small and doesn't become unmanagable after expanding along the periphery. Other, larger subgraphs might have this issue. If the subgraph becomes too complicated, it might be useful to extract the causal subgraph using  [pbt.selection.get_causal_subgraph](http://pybel-tools.readthedocs.io/en/latest/selection.html#pybel_tools.selection.get_causal_subgraph)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of nodes: 33\n",
      "Number of edges: 137\n",
      "Network density: 0.12973484848484848\n",
      "Number weakly connected components: 2\n",
      "Average in-degree: 4.151515151515151\n",
      "Average out-degree: 4.151515151515151\n"
     ]
    }
   ],
   "source": [
    "pbt.mutation.expand_periphery(\n",
    "    graph, \n",
    "    subgraph, \n",
    "    node_filters=pbt.filters.exclude_pathology_filter, \n",
    "    threshold=3)\n",
    "\n",
    "pbt.summary.print_summary(subgraph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"sarqbvegocqktpum\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {\"name\": \"Alzheimer's Disease Knowledge Assembly - (Subgraph: Estrogen subgraph)\"}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Alzheimer Disease\", \"cname\": \"Alzheimer Disease\", \"id\": [\"Pathology\", \"MESHD\", \"Alzheimer Disease\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"ESR1\", \"cname\": \"ESR1\", \"id\": [\"Protein\", \"HGNC\", \"ESR1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"cname\": \"GSK3B\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"17beta-estradiol\", \"cname\": \"17beta-estradiol\", \"id\": [\"Abundance\", \"CHEBI\", \"17beta-estradiol\"]}, {\"function\": \"Pathology\", \"namespace\": \"MESHD\", \"name\": \"Parkinson Disease\", \"cname\": \"Parkinson Disease\", \"id\": [\"Pathology\", \"MESHD\", \"Parkinson Disease\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 9}], \"cname\": \"p(HGNC:GSK3B, pmod(Ph, Ser, 9))\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 9]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CYP3A4\", \"cname\": \"CYP3A4\", \"id\": [\"Protein\", \"HGNC\", \"CYP3A4\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"metabolic process\", \"cname\": \"metabolic process\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"metabolic process\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Thr\", \"pos\": 231}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Thr, 231))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Thr\", 231]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Thr\", \"pos\": 205}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Thr, 205))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Thr\", 205]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 396}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Ser, 396))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 396]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 404}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Ser, 404))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 404]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 199}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Ser, 199))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 199]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 202}], \"cname\": \"p(HGNC:MAPT, pmod(Ph, Ser, 202))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 202]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"resveratrol\", \"cname\": \"resveratrol\", \"id\": [\"Abundance\", \"CHEBI\", \"resveratrol\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"ESR2\", \"cname\": \"ESR2\", \"id\": [\"Protein\", \"HGNC\", \"ESR2\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"estradiol\", \"cname\": \"estradiol\", \"id\": [\"Abundance\", \"CHEBI\", \"estradiol\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"S100A9\", \"cname\": \"S100A9\", \"id\": [\"Protein\", \"HGNC\", \"S100A9\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PPARG\", \"cname\": \"PPARG\", \"id\": [\"Protein\", \"HGNC\", \"PPARG\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"TFF1\", \"cname\": \"TFF1\", \"id\": [\"RNA\", \"HGNC\", \"TFF1\"]}, {\"function\": \"Complex\", \"cname\": \"complex(a(CHEBI:\\\"24-hydroxycholesterol\\\"), p(HGNC:ESR2))\", \"id\": [\"Complex\", [\"Abundance\", \"CHEBI\", \"24-hydroxycholesterol\"], [\"Protein\", \"HGNC\", \"ESR2\"]]}, {\"function\": \"Complex\", \"cname\": \"complex(a(CHEBI:\\\"medroxyprogesterone acetate\\\"), p(HGNC:ESR2))\", \"id\": [\"Complex\", [\"Abundance\", \"CHEBI\", \"medroxyprogesterone acetate\"], [\"Protein\", \"HGNC\", \"ESR2\"]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"24-hydroxycholesterol\", \"cname\": \"24-hydroxycholesterol\", \"id\": [\"Abundance\", \"CHEBI\", \"24-hydroxycholesterol\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"medroxyprogesterone acetate\", \"cname\": \"medroxyprogesterone acetate\", \"id\": [\"Abundance\", \"CHEBI\", \"medroxyprogesterone acetate\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"cname\": \"MAPT\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"cname\": \"p(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"donepezil\", \"cname\": \"donepezil\", \"id\": [\"Abundance\", \"CHEBI\", \"donepezil\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"cname\": \"APP\", \"id\": [\"Protein\", \"HGNC\", \"APP\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"TIMP3\", \"cname\": \"TIMP3\", \"id\": [\"RNA\", \"HGNC\", \"TIMP3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Glyco\"}}], \"cname\": \"p(HGNC:MAPT, pmod(Glyco))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Glyco\"]]]}, {\"function\": \"Abundance\", \"namespace\": \"SCHEM\", \"name\": \"Calyculin A\", \"cname\": \"Calyculin A\", \"id\": [\"Abundance\", \"SCHEM\", \"Calyculin A\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CDK5\", \"cname\": \"CDK5\", \"id\": [\"Protein\", \"HGNC\", \"CDK5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MAPK14\", \"cname\": \"MAPK14\", \"id\": [\"Protein\", \"HGNC\", \"MAPK14\"]}], \"links\": [{\"relation\": \"association\", \"evidence\": \"Herein, we conducted a meta-analysis to clarify the association between ESR1 polymorphisms and the occurrence of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2014 Jun 27;574:41-6\", \"reference\": \"24857745\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Herein, we conducted a meta-analysis to clarify the association between ESR1 polymorphisms and the occurrence of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurosci Lett. 2014 Jun 27;574:41-6\", \"reference\": \"24857745\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Positive association between an estrogen receptor gene polymorphism and Parkinson's disease with dementia.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurol. 1999 Jul;6(4):431-5\", \"reference\": \"10362895\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\", \"Species\": \"9606\", \"MeSHDisease\": \"Dementia\"}, \"source\": 1, \"target\": 4, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Positive association between an estrogen receptor gene polymorphism and Parkinson's disease with dementia.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurol. 1999 Jul;6(4):431-5\", \"reference\": \"10362895\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\", \"Species\": \"9606\", \"MeSHDisease\": \"Parkinson Disease\"}, \"source\": 1, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 7}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 10}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 13}, {\"relation\": \"increases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 2, \"target\": 3, \"key\": 16}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 2, \"target\": 5, \"key\": -4}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 25, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of β-amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 2, \"target\": 25, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 8, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 9, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 10, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 11, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 12, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"We found that betaE2 could attenuate tau hyperphosphorylation at multiple AD-related sites, including Ser396/404, Thr231, Thr205, and Ser199/202, induced by Wort/GFX or transient overexpression of GSK-3beta. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 13, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Simultaneously, it increased the level of Ser9-phosphorylated (inactive) GSK-3beta\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 3, \"target\": 5, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Positive association between an estrogen receptor gene polymorphism and Parkinson's disease with dementia.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurol. 1999 Jul;6(4):431-5\", \"reference\": \"10362895\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\", \"Species\": \"9606\", \"MeSHDisease\": \"Dementia\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Positive association between an estrogen receptor gene polymorphism and Parkinson's disease with dementia.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Eur J Neurol. 1999 Jul;6(4):431-5\", \"reference\": \"10362895\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\", \"Species\": \"9606\", \"MeSHDisease\": \"Parkinson Disease\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Simultaneously, it increased the level of Ser9-phosphorylated (inactive) GSK-3beta\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 5, \"target\": 2, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Donepezil is metabolized via CYP-related enzymes, especially CYP2D6, CYP3A4, and CYP1A2.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropsychiatr Dis Treat. 2007 Jun;3(3):303-33\", \"reference\": \"19300564\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 6, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Some cholinesterase inhibitors (tacrine, donepezil, galantamine) are metabolized via CYP-related enzymes, especially CYP2D6, CYP3A4, and CYP1A2.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2007 Sep;4(4):479-500\", \"reference\": \"17908053\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 6, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Donepezil is metabolized via CYP-related enzymes, especially CYP2D6, CYP3A4, and CYP1A2.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropsychiatr Dis Treat. 2007 Jun;3(3):303-33\", \"reference\": \"19300564\"}, \"annotations\": {\"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 7, \"target\": 26, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Cytochrome P450 (CYP) 2D6 enzyme is the major responsible for the metabolism of donepezil, an inhibitor of acetyl cholinesterase currently used for the symptomatic treatment of mild-to-moderate Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Pharmacogenet Genomics. 2011 Apr;21(4):225-30\", \"reference\": \"20859244\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Paroxetine subgraph\"}, \"source\": 7, \"target\": 26, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Over the last several years a number of reports have emerged suggesting that at least some ChEI might take part in betaAPP metabolism, influencing its secretion and Abeta differential cleavage. Moreover, we (Sob6w and Kloszewska 2005) and others (Basun et al. 2002, Zimmermann et al. 2005) have shown that the treatment with ChEI might influence BAPP metabolism in AD patients as measured by changes in plasma (including platelet-derived) metabolites. In our previous pilot study we have demonstrated that short-term treatment with ChEI rivastigmine exhibits a significant effect on plasma concentrations of Abeta-42 (mean increase after treatment reached 7.8 Â± 8.4 pg/ml) with a negative correlation to patients age, while no changes in Abeta-40 levels were detected.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Neurobiol Exp (Wars). 2007;67(2):131-9\", \"reference\": \"17691220\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 7, \"target\": 27, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Rivastigmine treatment induced an elevation in both metabolic activity and APP secretion, and differentially impacted two isoforms of sAPP. In our cultures, low molecular weight sAPP corresponded closely to neuronal viability, and high molecular weight sAPP corresponded with glial proliferation. Our results suggest that within the mixed culture system used, neurons are the primary source of sAPP, and rivastigmine's actions are mediated principally through neuronal, rather than glial, targets. Importantly, rivastigmine was found to increase the neurotrophic sAPPÎ± and decrease Abeta secretion, suggesting a mechanism for the previously observed neuropreservation effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2011;6(7):e21954. doi: 10.1371/journal.pone.0021954. Epub 2011 Jul 22\", \"reference\": \"21799757\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 7, \"target\": 27, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 8, \"target\": 25, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Akt subgraph\"}, \"source\": 8, \"target\": 25, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 8, \"target\": 25, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 10, \"target\": 25, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Akt subgraph\"}, \"source\": 10, \"target\": 25, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 10, \"target\": 25, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 11, \"target\": 25, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Akt subgraph\"}, \"source\": 11, \"target\": 25, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 11, \"target\": 25, \"key\": 2}, {\"relation\": \"association\", \"evidence\": \"resveratrol inhibits the reaction [ESR2 protein binds to CAV1 protein]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"18296501\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 14, \"target\": 15, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"[Coumestrol co-treated with resveratrol] results in decreased expression of TIMP3 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"19167446\"}, \"annotations\": {}, \"source\": 14, \"target\": 28, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"resveratrol inhibits the reaction [ESR2 protein binds to CAV1 protein]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"18296501\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 15, \"target\": 14, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"[Estradiol co-treated with Tetrachlorodibenzodioxin] results in decreased expression of S100A9 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"19484750\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 16, \"target\": 17, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Estradiol results in increased expression of PPARG protein\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"15964169\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 16, \"target\": 18, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"2-(2-amino-3-methoxyphenyl)-4H-1-benzopyran-4-one inhibits the reaction [Estradiol results in increased expression of TFF1 mRNA]\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"11331657\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 16, \"target\": 19, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"[Estradiol binds to ESR2 protein] which results in increased expression of TIMP3 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16474171\"}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 16, \"target\": 28, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In CAA, and probably in AD, corticosteroid treatment suppresses the secondarily induced vascular inflammation (Chang et al., 2012, Kloppenborg et al., 2010 and Machida et al., 2012) and the activation of S100A9/Mrp14 (Gebhardt et al., 2002 and Kametani and Ikeda, 2013). Recently, it has been reported that anti-Abeta autoantibodies were progressively reduced in the cerebrospinal fluid of a patient with CAA-related inflammation, following intravenous steroid administration (Piazza et al., 2013). Reduction of S100A9/Mrp14 might induce suppression of amyloid fibril formation (Zhang et al., 2012) and increase phagocytosis of fibrillar Abeta in microglia cells (Kummer et al., 2012), resulting in reduced Abeta amyloid fibril deposition.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2014 Apr;35(4):e11-2\", \"reference\": \"24262203\"}, \"annotations\": {\"Subgraph\": \"Metabolism of steroid hormones subgraph\"}, \"source\": 17, \"target\": 25, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In CAA, and probably in AD, corticosteroid treatment suppresses the secondarily induced vascular inflammation (Chang et al., 2012, Kloppenborg et al., 2010 and Machida et al., 2012) and the activation of S100A9/Mrp14 (Gebhardt et al., 2002 and Kametani and Ikeda, 2013). Recently, it has been reported that anti-Abeta autoantibodies were progressively reduced in the cerebrospinal fluid of a patient with CAA-related inflammation, following intravenous steroid administration (Piazza et al., 2013). Reduction of S100A9/Mrp14 might induce suppression of amyloid fibril formation (Zhang et al., 2012) and increase phagocytosis of fibrillar Abeta in microglia cells (Kummer et al., 2012), resulting in reduced Abeta amyloid fibril deposition.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2014 Apr;35(4):e11-2\", \"reference\": \"24262203\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 25, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Pro-inflammatory S100A9 protein is increasingly recognized as an important contributor to inflammation-related neurodegeneration. Here, we provide insights into S100A9 specific mechanisms of action in Alzheimer's disease (AD). Due to its inherent amyloidogenicity S100A9 contributes to amyloid plaque formation together with Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Neuropathol. 2014 Apr;127(4):507-22\", \"reference\": \"24240735\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\"}, \"source\": 17, \"target\": 25, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Pro-inflammatory S100A9 protein is increasingly recognized as an important contributor to inflammation-related neurodegeneration. Here, we provide insights into S100A9 specific mechanisms of action in Alzheimer's disease (AD). Due to its inherent amyloidogenicity S100A9 contributes to amyloid plaque formation together with Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Neuropathol. 2014 Apr;127(4):507-22\", \"reference\": \"24240735\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 17, \"target\": 25, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"We therefore conclude that Mrp14 promotes APP processing and Abeta accumulation under neuroinflammatory conditions.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2012 Dec 5;32(49):17824-9.\", \"reference\": \"23223301\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\"}, \"source\": 17, \"target\": 25, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"We therefore conclude that Mrp14 promotes APP processing and Abeta accumulation under neuroinflammatory conditions.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurosci. 2012 Dec 5;32(49):17824-9.\", \"reference\": \"23223301\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 17, \"target\": 25, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Abeta) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2011 Mar;32(3):547.e1-6. Epub 2009 Aug 6.\", \"reference\": \"19660836\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 18, \"target\": 25, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Abeta) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2011 Mar;32(3):547.e1-6. Epub 2009 Aug 6.\", \"reference\": \"19660836\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 18, \"target\": 25, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical Hypotheses2006\", \"reference\": \"16828233\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 18, \"target\": 25, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical Hypotheses2006\", \"reference\": \"16828233\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 18, \"target\": 25, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Abeta) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2011 Mar;32(3):547.e1-6. Epub 2009 Aug 6.\", \"reference\": \"19660836\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 18, \"target\": 25, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"The peroxisome proliferator-activated receptor-γ (PPAR-γ) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Abeta) production through promoting cholesterol efflux from glial cells. PPAR-γ agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) ε4 allele\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2011 Mar;32(3):547.e1-6. Epub 2009 Aug 6.\", \"reference\": \"19660836\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 18, \"target\": 25, \"key\": 5}, {\"relation\": \"decreases\", \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical Hypotheses2006\", \"reference\": \"16828233\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 18, \"target\": 25, \"key\": 6}, {\"relation\": \"decreases\", \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Medical Hypotheses2006\", \"reference\": \"16828233\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 18, \"target\": 25, \"key\": 7}, {\"relation\": \"decreases\", \"evidence\": \"24-hydroxycholesterol binds to and results in decreased activity of ESR2 protein\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"17873880\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 20, \"target\": 15, \"key\": 1}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 20, \"target\": 15, \"key\": -3}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 20, \"target\": 22, \"key\": -3}, {\"relation\": \"increases\", \"evidence\": \"Activation;Binding;Medroxyprogesterone acetate intracellular binds to and activates ESR2 .\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"9464360\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 21, \"target\": 15, \"key\": 1}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 21, \"target\": 15, \"key\": -3}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 21, \"target\": 23, \"key\": -3}, {\"relation\": \"decreases\", \"evidence\": \"Medroxyprogesterone Acetate results in decreased expression of TIMP3 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"20843944\"}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 23, \"target\": 28, \"key\": 0}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 8, \"key\": -4}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 9, \"key\": -4}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 10, \"key\": -4}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 11, \"key\": -4}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 12, \"key\": -4}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 13, \"key\": -4}, {\"relation\": \"association\", \"evidence\": \"Compared to vehicle, Abeta increased GSK3 activity, and was associated with elevations in levels of ptau, caspase-3, the tau kinase phospho-c-jun N-terminal kinase (pJNK), neuronal DNA fragmentation, and gliosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of Disease2009\", \"reference\": \"19038340\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 24, \"target\": 25, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Compared to vehicle, Abeta increased GSK3 activity, and was associated with elevations in levels of ptau, caspase-3, the tau kinase phospho-c-jun N-terminal kinase (pJNK), neuronal DNA fragmentation, and gliosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of Disease2009\", \"reference\": \"19038340\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 24, \"target\": 25, \"key\": 1}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 24, \"target\": 29, \"key\": -4}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 25, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 25, \"target\": 2, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 25, \"target\": 2, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 25, \"target\": 2, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 25, \"target\": 2, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 25, \"target\": 2, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 25, \"target\": 2, \"key\": 6}, {\"relation\": \"association\", \"evidence\": \"Compared to vehicle, Abeta increased GSK3 activity, and was associated with elevations in levels of ptau, caspase-3, the tau kinase phospho-c-jun N-terminal kinase (pJNK), neuronal DNA fragmentation, and gliosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of Disease2009\", \"reference\": \"19038340\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 25, \"target\": 24, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Compared to vehicle, Abeta increased GSK3 activity, and was associated with elevations in levels of ptau, caspase-3, the tau kinase phospho-c-jun N-terminal kinase (pJNK), neuronal DNA fragmentation, and gliosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiology of Disease2009\", \"reference\": \"19038340\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 25, \"target\": 24, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Taken together, our results suggest that the neuroprotective effects of donepezil against Abeta42-induced neurotoxicity are mediated through activation of PP2A, but its additional mechanisms including regulation of GSK-3beta and nAChRs activity would partially contribute to its effects. This observation led us to assume that additional mechanisms of donepezil, including its inhibitory effect on GSK-3beta activity and/or the activation role of nicotinic acetylcholine receptors (nAChRs), might be involved. Donepezil increased neuronal viability with reduced p-tau by enhancing PP2A activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2013 Nov;127(4):562-74\", \"reference\": \"23711227\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Acetylcholine signaling subgraph\"}, \"source\": 26, \"target\": 2, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Taken together, our results suggest that the neuroprotective effects of donepezil against Abeta42-induced neurotoxicity are mediated through activation of PP2A, but its additional mechanisms including regulation of GSK-3beta and nAChRs activity would partially contribute to its effects. This observation led us to assume that additional mechanisms of donepezil, including its inhibitory effect on GSK-3beta activity and/or the activation role of nicotinic acetylcholine receptors (nAChRs), might be involved. Donepezil increased neuronal viability with reduced p-tau by enhancing PP2A activity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2013 Nov;127(4):562-74\", \"reference\": \"23711227\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 26, \"target\": 2, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"Donepezil is metabolized via CYP-related enzymes, especially CYP2D6, CYP3A4, and CYP1A2.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuropsychiatr Dis Treat. 2007 Jun;3(3):303-33\", \"reference\": \"19300564\"}, \"annotations\": {\"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 26, \"target\": 7, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Cytochrome P450 (CYP) 2D6 enzyme is the major responsible for the metabolism of donepezil, an inhibitor of acetyl cholinesterase currently used for the symptomatic treatment of mild-to-moderate Alzheimer's disease (AD).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Pharmacogenet Genomics. 2011 Apr;21(4):225-30\", \"reference\": \"20859244\"}, \"annotations\": {\"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Paroxetine subgraph\"}, \"source\": 26, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Although the role of AICD as a gene transcription inducer is still controversial, several targets of AICD- or APP-mediated transcriptional activation are reportedly regulated by APP nuclear signaling, including APP itself, BACE, Tip60, GSK-3ÃƒÅ¸, p53, Mn-SOD, KAI1, Neprilysin, transgelin, a2actin, S100a9, and other genes\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMB Rep. 2010 Oct;43(10):656-63.\", \"reference\": \"21034527\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 27, \"target\": 2, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Over the last several years a number of reports have emerged suggesting that at least some ChEI might take part in betaAPP metabolism, influencing its secretion and Abeta differential cleavage. Moreover, we (Sob6w and Kloszewska 2005) and others (Basun et al. 2002, Zimmermann et al. 2005) have shown that the treatment with ChEI might influence BAPP metabolism in AD patients as measured by changes in plasma (including platelet-derived) metabolites. In our previous pilot study we have demonstrated that short-term treatment with ChEI rivastigmine exhibits a significant effect on plasma concentrations of Abeta-42 (mean increase after treatment reached 7.8 Â± 8.4 pg/ml) with a negative correlation to patients age, while no changes in Abeta-40 levels were detected.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Neurobiol Exp (Wars). 2007;67(2):131-9\", \"reference\": \"17691220\"}, \"annotations\": {\"Species\": \"10116\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 27, \"target\": 7, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"Rivastigmine treatment induced an elevation in both metabolic activity and APP secretion, and differentially impacted two isoforms of sAPP. In our cultures, low molecular weight sAPP corresponded closely to neuronal viability, and high molecular weight sAPP corresponded with glial proliferation. Our results suggest that within the mixed culture system used, neurons are the primary source of sAPP, and rivastigmine's actions are mediated principally through neuronal, rather than glial, targets. Importantly, rivastigmine was found to increase the neurotrophic sAPPÎ± and decrease Abeta secretion, suggesting a mechanism for the previously observed neuropreservation effects.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2011;6(7):e21954. doi: 10.1371/journal.pone.0021954. Epub 2011 Jul 22\", \"reference\": \"21799757\"}, \"annotations\": {\"Species\": \"10116\", \"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 27, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Although the role of AICD as a gene transcription inducer is still controversial, several targets of AICD- or APP-mediated transcriptional activation are reportedly regulated by APP nuclear signaling, including APP itself, BACE, Tip60, GSK-3ÃƒÅ¸, p53, Mn-SOD, KAI1, Neprilysin, transgelin, a2actin, S100a9, and other genes\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMB Rep. 2010 Oct;43(10):656-63.\", \"reference\": \"21034527\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {}, \"source\": 27, \"target\": 17, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Although the role of AICD as a gene transcription inducer is still controversial, several targets of AICD- or APP-mediated transcriptional activation are reportedly regulated by APP nuclear signaling, including APP itself, BACE, Tip60, GSK-3ÃƒÅ¸, p53, Mn-SOD, KAI1, Neprilysin, transgelin, a2actin, S100a9, and other genes\", \"citation\": {\"type\": \"PubMed\", \"name\": \"BMB Rep. 2010 Oct;43(10):656-63.\", \"reference\": \"21034527\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {}, \"source\": 27, \"target\": 17, \"key\": 1}, {\"relation\": \"association\", \"evidence\": \"[Coumestrol co-treated with resveratrol] results in decreased expression of TIMP3 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"19167446\"}, \"annotations\": {}, \"source\": 28, \"target\": 14, \"key\": 0}, {\"relation\": \"association\", \"evidence\": \"[Estradiol binds to ESR2 protein] which results in increased expression of TIMP3 mRNA\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Reference from AD_HBP Data\", \"reference\": \"16474171\"}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 28, \"target\": 16, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that aberrant glycosylation of tau in AD might be involved in neurofibrillary degeneration by promoting abnormal hyperphosphorylation by cdk5 and GSK-3beta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 29, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that aberrant glycosylation of tau in AD might be involved in neurofibrillary degeneration by promoting abnormal hyperphosphorylation by cdk5 and GSK-3beta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 29, \"target\": 2, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Deglycosylation by glycosidases depressed the subsequent phosphorylation of AD-tau (i) with cdk5 at Thr-181, Ser-199, Ser-202, Thr-205, and Ser-404, but not at Thr-212; and (ii) with GSK-3beta at Thr-181, Ser-202, Thr-205, Ser-217, and Ser-404, but not at Ser-199, Thr-212, Thr-231, or Ser-396. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 29, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Deglycosylation by glycosidases depressed the subsequent phosphorylation of AD-tau (i) with cdk5 at Thr-181, Ser-199, Ser-202, Thr-205, and Ser-404, but not at Thr-212; and (ii) with GSK-3beta at Thr-181, Ser-202, Thr-205, Ser-217, and Ser-404, but not at Ser-199, Thr-212, Thr-231, or Ser-396. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 29, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Deglycosylation by glycosidases depressed the subsequent phosphorylation of AD-tau (i) with cdk5 at Thr-181, Ser-199, Ser-202, Thr-205, and Ser-404, but not at Thr-212; and (ii) with GSK-3beta at Thr-181, Ser-202, Thr-205, Ser-217, and Ser-404, but not at Ser-199, Thr-212, Thr-231, or Ser-396. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 29, \"target\": 11, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Deglycosylation by glycosidases depressed the subsequent phosphorylation of AD-tau (i) with cdk5 at Thr-181, Ser-199, Ser-202, Thr-205, and Ser-404, but not at Thr-212; and (ii) with GSK-3beta at Thr-181, Ser-202, Thr-205, Ser-217, and Ser-404, but not at Ser-199, Thr-212, Thr-231, or Ser-396. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 29, \"target\": 11, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Deglycosylation by glycosidases depressed the subsequent phosphorylation of AD-tau (i) with cdk5 at Thr-181, Ser-199, Ser-202, Thr-205, and Ser-404, but not at Thr-212; and (ii) with GSK-3beta at Thr-181, Ser-202, Thr-205, Ser-217, and Ser-404, but not at Ser-199, Thr-212, Thr-231, or Ser-396. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 29, \"target\": 11, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Deglycosylation by glycosidases depressed the subsequent phosphorylation of AD-tau (i) with cdk5 at Thr-181, Ser-199, Ser-202, Thr-205, and Ser-404, but not at Thr-212; and (ii) with GSK-3beta at Thr-181, Ser-202, Thr-205, Ser-217, and Ser-404, but not at Ser-199, Thr-212, Thr-231, or Ser-396. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 29, \"target\": 11, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Deglycosylation by glycosidases depressed the subsequent phosphorylation of AD-tau (i) with cdk5 at Thr-181, Ser-199, Ser-202, Thr-205, and Ser-404, but not at Thr-212; and (ii) with GSK-3beta at Thr-181, Ser-202, Thr-205, Ser-217, and Ser-404, but not at Ser-199, Thr-212, Thr-231, or Ser-396. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 29, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Deglycosylation by glycosidases depressed the subsequent phosphorylation of AD-tau (i) with cdk5 at Thr-181, Ser-199, Ser-202, Thr-205, and Ser-404, but not at Thr-212; and (ii) with GSK-3beta at Thr-181, Ser-202, Thr-205, Ser-217, and Ser-404, but not at Ser-199, Thr-212, Thr-231, or Ser-396. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 29, \"target\": 12, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Deglycosylation by glycosidases depressed the subsequent phosphorylation of AD-tau (i) with cdk5 at Thr-181, Ser-199, Ser-202, Thr-205, and Ser-404, but not at Thr-212; and (ii) with GSK-3beta at Thr-181, Ser-202, Thr-205, Ser-217, and Ser-404, but not at Ser-199, Thr-212, Thr-231, or Ser-396. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 29, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Deglycosylation by glycosidases depressed the subsequent phosphorylation of AD-tau (i) with cdk5 at Thr-181, Ser-199, Ser-202, Thr-205, and Ser-404, but not at Thr-212; and (ii) with GSK-3beta at Thr-181, Ser-202, Thr-205, Ser-217, and Ser-404, but not at Ser-199, Thr-212, Thr-231, or Ser-396. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 29, \"target\": 13, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"We found a significant reduction of calyculin A-induced tau hyperphosphorylation at Ser198/199/202, Ser396, Ser404, Thr205, and Thr231 24 h after treatment with 20 Î¼g/ml berberine. Berberine also restored protein phosphates 2A activity and reversed glycogen synthase kinase-3β (GSK-3β) activation, as determined by phosphatase activity assay and GSK-3β phosphorylation at Tyr216 and Ser9, respectively. Furthermore, berberine reversed both the increase of malondialdehyde and the decrease of superoxide dismutase activity induced by calyculin A, indicating its role in anti-oxidative stress.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2011;24(3):525-35\", \"reference\": \"21297267\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 30, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found a significant reduction of calyculin A-induced tau hyperphosphorylation at Ser198/199/202, Ser396, Ser404, Thr205, and Thr231 24 h after treatment with 20 Î¼g/ml berberine. Berberine also restored protein phosphates 2A activity and reversed glycogen synthase kinase-3β (GSK-3β) activation, as determined by phosphatase activity assay and GSK-3β phosphorylation at Tyr216 and Ser9, respectively. Furthermore, berberine reversed both the increase of malondialdehyde and the decrease of superoxide dismutase activity induced by calyculin A, indicating its role in anti-oxidative stress.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2011;24(3):525-35\", \"reference\": \"21297267\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 30, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found a significant reduction of calyculin A-induced tau hyperphosphorylation at Ser198/199/202, Ser396, Ser404, Thr205, and Thr231 24 h after treatment with 20 Î¼g/ml berberine. Berberine also restored protein phosphates 2A activity and reversed glycogen synthase kinase-3β (GSK-3β) activation, as determined by phosphatase activity assay and GSK-3β phosphorylation at Tyr216 and Ser9, respectively. Furthermore, berberine reversed both the increase of malondialdehyde and the decrease of superoxide dismutase activity induced by calyculin A, indicating its role in anti-oxidative stress.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2011;24(3):525-35\", \"reference\": \"21297267\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 30, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found a significant reduction of calyculin A-induced tau hyperphosphorylation at Ser198/199/202, Ser396, Ser404, Thr205, and Thr231 24 h after treatment with 20 Î¼g/ml berberine. Berberine also restored protein phosphates 2A activity and reversed glycogen synthase kinase-3β (GSK-3β) activation, as determined by phosphatase activity assay and GSK-3β phosphorylation at Tyr216 and Ser9, respectively. Furthermore, berberine reversed both the increase of malondialdehyde and the decrease of superoxide dismutase activity induced by calyculin A, indicating its role in anti-oxidative stress.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2011;24(3):525-35\", \"reference\": \"21297267\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 30, \"target\": 11, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found a significant reduction of calyculin A-induced tau hyperphosphorylation at Ser198/199/202, Ser396, Ser404, Thr205, and Thr231 24 h after treatment with 20 Î¼g/ml berberine. Berberine also restored protein phosphates 2A activity and reversed glycogen synthase kinase-3β (GSK-3β) activation, as determined by phosphatase activity assay and GSK-3β phosphorylation at Tyr216 and Ser9, respectively. Furthermore, berberine reversed both the increase of malondialdehyde and the decrease of superoxide dismutase activity induced by calyculin A, indicating its role in anti-oxidative stress.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2011;24(3):525-35\", \"reference\": \"21297267\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 30, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found a significant reduction of calyculin A-induced tau hyperphosphorylation at Ser198/199/202, Ser396, Ser404, Thr205, and Thr231 24 h after treatment with 20 Î¼g/ml berberine. Berberine also restored protein phosphates 2A activity and reversed glycogen synthase kinase-3β (GSK-3β) activation, as determined by phosphatase activity assay and GSK-3β phosphorylation at Tyr216 and Ser9, respectively. Furthermore, berberine reversed both the increase of malondialdehyde and the decrease of superoxide dismutase activity induced by calyculin A, indicating its role in anti-oxidative stress.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2011;24(3):525-35\", \"reference\": \"21297267\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 30, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 31, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 31, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 31, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 31, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"we conclude that tau protein kinase II (cdk5/p20) can phosphorylate human tau at Thr(181), Thr(205), Thr(212), Thr(217), Ser(396) and Ser(404).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2001\", \"reference\": \"11181841\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 31, \"target\": 9, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"we conclude that tau protein kinase II (cdk5/p20) can phosphorylate human tau at Thr(181), Thr(205), Thr(212), Thr(217), Ser(396) and Ser(404).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2001\", \"reference\": \"11181841\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 31, \"target\": 9, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 31, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 31, \"target\": 10, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"we conclude that tau protein kinase II (cdk5/p20) can phosphorylate human tau at Thr(181), Thr(205), Thr(212), Thr(217), Ser(396) and Ser(404).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2001\", \"reference\": \"11181841\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 31, \"target\": 10, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"we conclude that tau protein kinase II (cdk5/p20) can phosphorylate human tau at Thr(181), Thr(205), Thr(212), Thr(217), Ser(396) and Ser(404).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2001\", \"reference\": \"11181841\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 31, \"target\": 10, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 31, \"target\": 11, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 31, \"target\": 11, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"we conclude that tau protein kinase II (cdk5/p20) can phosphorylate human tau at Thr(181), Thr(205), Thr(212), Thr(217), Ser(396) and Ser(404).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2001\", \"reference\": \"11181841\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 31, \"target\": 11, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"we conclude that tau protein kinase II (cdk5/p20) can phosphorylate human tau at Thr(181), Thr(205), Thr(212), Thr(217), Ser(396) and Ser(404).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neurochemistry2001\", \"reference\": \"11181841\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 31, \"target\": 11, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 31, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 31, \"target\": 12, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 31, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We found that cdk5 phosphorylated tau(441) at Thr-181, Ser-199, Ser-202, Thr-205, Thr-212, Ser-214, Thr-217, Thr-231, Ser-235, Ser-396, and Ser-404, but not at Ser-262, Ser-400, Thr-403, Ser-409, Ser-413, or Ser-422.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 31, \"target\": 13, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Western blots with phosphorylation-sensitive antibodies showed that p38, like ERK2 and SAP kinase-beta/JNK, phosphorylated tau at sites found phosphorylated physiologically (Thr181, Ser202, Thr205, and Ser396) and also at Ser422, which is phosphorylated in neurofibrillary tangles but not in normal adult or foetal brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1997\", \"reference\": \"9202310\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 32, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Western blots with phosphorylation-sensitive antibodies showed that p38, like ERK2 and SAP kinase-beta/JNK, phosphorylated tau at sites found phosphorylated physiologically (Thr181, Ser202, Thr205, and Ser396) and also at Ser422, which is phosphorylated in neurofibrillary tangles but not in normal adult or foetal brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1997\", \"reference\": \"9202310\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 32, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Western blots with phosphorylation-sensitive antibodies showed that p38, like ERK2 and SAP kinase-beta/JNK, phosphorylated tau at sites found phosphorylated physiologically (Thr181, Ser202, Thr205, and Ser396) and also at Ser422, which is phosphorylated in neurofibrillary tangles but not in normal adult or foetal brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1997\", \"reference\": \"9202310\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 32, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Western blots with phosphorylation-sensitive antibodies showed that p38, like ERK2 and SAP kinase-beta/JNK, phosphorylated tau at sites found phosphorylated physiologically (Thr181, Ser202, Thr205, and Ser396) and also at Ser422, which is phosphorylated in neurofibrillary tangles but not in normal adult or foetal brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1997\", \"reference\": \"9202310\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 32, \"target\": 10, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Western blots with phosphorylation-sensitive antibodies showed that p38, like ERK2 and SAP kinase-beta/JNK, phosphorylated tau at sites found phosphorylated physiologically (Thr181, Ser202, Thr205, and Ser396) and also at Ser422, which is phosphorylated in neurofibrillary tangles but not in normal adult or foetal brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1997\", \"reference\": \"9202310\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 32, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Western blots with phosphorylation-sensitive antibodies showed that p38, like ERK2 and SAP kinase-beta/JNK, phosphorylated tau at sites found phosphorylated physiologically (Thr181, Ser202, Thr205, and Ser396) and also at Ser422, which is phosphorylated in neurofibrillary tangles but not in normal adult or foetal brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem1997\", \"reference\": \"9202310\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 32, \"target\": 13, \"key\": 1}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In this case, we were able to infer connections that not only gave the Estrogen subgraph more context, but also connected the individual components"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Conclusions\n",
    "\n",
    "A final touch to the subgraph might be to infer connections between nodes that have just been added. This can be done with [pbt.mutation.expand_internal](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.expand_internal) which allows for specification of edge filters, or with [pbt.mutation.expand_internal_causal](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.expand_internal_causal) that is a thin wrapper, giving the edge filter [pbt.filters.keep_causal_edges](http://pybel-tools.readthedocs.io/en/latest/filters.html#pybel_tools.filters.edge_filters.keep_causal_edges). Again, expansion on unqualified edges and opening with the central dogma can make this expanded subgraph easier to interpret.\n",
    "\n",
    "Further, an unbiased expansion method could allow for annotations of entities to subgraphs such as chemicals and bioprocesses, and allow for more exotic enrichment algorithms to be implemented similar to NeuroMMSigDB."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
